-- MySQL dump 10.13  Distrib 5.1.63, for debian-linux-gnu (x86_64)
--
-- Host: reactomecurator.oicr.on.ca    Database: gk_central
-- ------------------------------------------------------
-- Server version	5.0.51a-24+lenny5-log

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Not dumping tablespaces as no INFORMATION_SCHEMA.FILES table on this server
--

--
-- Table structure for table `Ontology`
--

DROP TABLE IF EXISTS `Ontology`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `Ontology` (
  `ontology` longblob
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Ontology`
--

LOCK TABLES `Ontology` WRITE;
/*!40000 ALTER TABLE `Ontology` DISABLE KEYS */;
INSERT INTO `Ontology` VALUES ('\rGKB::Ontology\0\0\0\0\0\0a\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\Z\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0Å\0\0\0CellType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0\0\0Å\0\0\0StableIdentifier\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0stableIdentifier\0\0\0\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\nVARCHAR(64)\0\0\0db_col_type\nDatabaseObject\0\0\0origin\n\nother_type\0\0\0type\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\n	TIMESTAMP\0\0\0db_col_type\nDatabaseObject\0\0\0origin\n\nother_type\0\0\0type\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_internal_id_type\0\0\0typeÄ\0\0\0multiple\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\nVARCHAR(255)\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\nENUM(\'TRUE\')\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0\n__is_ghost\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0Disease\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0cellTypeÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0compartmentÅ\0\0\0editedÅ\0\0\0crossReferenceÅ\0\0\0goCellularComponentÅ\0\0\0nameÅ\0\0\0inferredFromÅ\0\0\0diseaseÅ\0\0\0authoredÅ\0\0\0\ninferredToÅ\0\0\0revised\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0cellType\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0Å\0\0\0cellTypeÅ\0\0\0\ninferredToÅ\0\0\0inferredFromÅ\0\0\0disease\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0PhysicalEntity\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nDomain\0\0\0ancestor_order\0\0\0Å\0\0\0Domain\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nDomain\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0\rComplexDomainÅ\0\0\0SequenceDomain\0\0\0allowed\n\rGenericDomain\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasInstance\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Domain\0\0\0ancestor\0\0\0Å\0\0\0hasInstance\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0hasInstance\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0hasInstance\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\rGenericDomain\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nDomain\0\0\0ancestor_order\0\0\0Å\0\0\0Domain\0\0\0parent\0\0\0\r\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nSequenceDomain\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rendCoordinate\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nSequenceDomain\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceEntity\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nSequenceDomain\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0startCoordinate\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Domain\0\0\0ancestor\0\0\0Å\0\0\0referenceEntityÅ\0\0\0\rendCoordinateÅ\0\0\0startCoordinate\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceEntityÅ\0\0\0\rendCoordinateÅ\0\0\0startCoordinate\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\rendCoordinateÅ\0\0\0startCoordinate\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceEntity\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0SequenceDomain\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\nPublication\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0author\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPublication\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0title\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0titleÅ\0\0\0author\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0authorÅ\0\0\0title\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0Publication\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nControlledVocabulary\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nControlledVocabulary\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0nameÅ\0\0\0\ndefinition\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0ControlledVocabulary\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\nTranslationalModification\0\0\0ancestor_order\0\0\0Å\0\0\0TranslationalModification\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PsiMod\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0psiMod\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nAbstractModifiedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceSequence\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0TranslationalModificationÅ\0\0\0DatabaseObjectÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0\ncoordinateÅ\0\0\0referenceSequenceÅ\0\0\0psiMod\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\ncoordinate\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0psiMod\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0ModifiedResidue\0\0\0Ç\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0ControlledVocabulary\0\0\0ancestor\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0˛\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\r\0\0\0modified\0\0\0\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\0\0\0\0\rown_attribute\0\0\0\nDatabaseObject\nControlledVocabulary\0\0\0ancestor_order\0\0\0Å\0\0\0ControlledVocabulary\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0DeletedControlledVocabulary\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0RegulationType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulationType\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\n\nRegulation\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0containedInPathway\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	regulator\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulatedEntity\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\rÅ\0\0\0releaseDateÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0nameÅ\0\0\0	summationÅ\0\0\0literatureReferenceÅ\0\0\0regulationTypeÅ\0\0\0containedInPathwayÅ\0\0\0	regulatorÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0regulatedEntityÅ\0\0\0edited\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0Å\0\0\0containedInPathwayÅ\0\0\0name\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0\nRegulation\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nExternalOntology\0\0\0ancestor_order\0\0\0Å\0\0\0ExternalOntology\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0synonym\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0CellType\0\0\0allowed\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0ExternalOntology\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0Å\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0CellType\0\0\0Ç\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItem\0\0\0ancestor\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEdge\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0pointCoordinates\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PathwayDiagram\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEdge\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0pathwayDiagram\0\0\0\0\0\0Å\0\0\0Vertex\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEdge\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0targetVertex\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEdge\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0edgeType\0\0\0\0\0\0Å\0\0\0Vertex\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEdge\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0sourceVertex\0\0\0	attribute\0\0\0\nDatabaseObject\nPathwayDiagramItem\0\0\0ancestor_order\0\0\0Å\0\0\0pathwayDiagramÅ\0\0\0targetVertexÅ\0\0\0pointCoordinatesÅ\0\0\0sourceVertexÅ\0\0\0edgeType\0\0\0\rown_attribute\0\0\0Å\0\0\0PathwayDiagramItem\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0Edge\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\n\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0name\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0Domain\0\0\0É\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItemÅ\0\0\0Vertex\0\0\0ancestor\0\0\0\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0width\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0x\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0height\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0representedInstance\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0y\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0PathwayDiagram\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0pathwayDiagram\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\0\0\0\0\rown_attribute\0\0\0\nDatabaseObject\nPathwayDiagramItem\nVertex\0\0\0ancestor_order\0\0\0Å\0\0\0Vertex\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0pathwayDiagramÅ\0\0\0representedInstance\0\0\0all\0\0\0check\0\0\0\rPathwayVertex\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nEvent\0\0\0ancestor_order\0\0\0Å\0\0\0Event\0\0\0parent\0\0\0\"\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nENUM(\'TRUE\',\'FALSE\')\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPathway\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0isCanonical\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nPathway\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasEvent\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\nVARCHAR(64)\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPathway\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0doi\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0Disease\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\nENUM(\'TRUE\',\'FALSE\')\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0\n_doRelease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0relatedSpecies\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rreleaseStatus\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Event\0\0\0ancestor\0\0\0Å\0\0\0hasEventÅ\0\0\0isCanonicalÅ\0\0\0doi\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0hasEvent\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0reviewedÅ\0\0\0hasEventÅ\0\0\0compartmentÅ\0\0\0authoredÅ\0\0\0edited\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0releaseDateÅ\0\0\0	summationÅ\0\0\0nameÅ\0\0\0species\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0goBiologicalProcessÅ\0\0\0isCanonicalÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0evidenceTypeÅ\0\0\0revisedÅ\0\0\0precedingEventÅ\0\0\0orthologousEventÅ\0\0\0doi\0\0\0OPTIONAL\0\0\0category\0\0\0Pathway\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\0\0\0ancestor_order\0\0\0Å\0\0\0PhysicalEntity\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0ReferenceMolecule\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nSimpleEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceEntity\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nSimpleEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0species\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0referenceEntityÅ\0\0\0species\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceEntityÅ\0\0\0cellTypeÅ\0\0\0compartmentÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceEntityÅ\0\0\0compartmentÅ\0\0\0name\0\0\0	MANDATORY\0\0\0\rÅ\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0speciesÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0SimpleEntity\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_Release\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_Release\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rreleaseNumber\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0\rreleaseNumberÅ\0\0\0releaseDate\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0releaseDateÅ\0\0\0\rreleaseNumber\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0_Release\0\0\0Ñ\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\nTranslationalModification\nCrosslinkedResidue\0\0\0ancestor_order\0\0\0Å\0\0\0CrosslinkedResidue\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PsiMod\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0psiMod\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\nCrosslinkedResidue\0\0\0origin\ndb_integer_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondCoordinate\0\0\0\0T\0\0\0\n_timestamp\0\0\0b\0\0\0created\0\0\0\0\0\0Å\0\0\0PolymerÅ\0\0\0ReferenceGroupÅ\0\0\0	EntitySet\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nCrosslinkedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0modification\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0n\0\0\0referenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0TranslationalModificationÅ\0\0\0DatabaseObjectÅ\0\0\0CrosslinkedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0modificationÅ\0\0\0referenceSequenceÅ\0\0\0\ncoordinateÅ\0\0\0psiModÅ\0\0\0secondCoordinate\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0secondCoordinate\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0IntraChainCrosslinkedResidue\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0otherIdentifier\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0crossReferenceÅ\0\0\0\nidentifierÅ\0\0\0referenceDatabaseÅ\0\0\0nameÅ\0\0\0otherIdentifier\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0role\0\0\0ReferenceEntity\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nDomain\0\0\0ancestor_order\0\0\0Å\0\0\0Domain\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\n\rComplexDomain\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasComponent\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Domain\0\0\0ancestor\0\0\0Å\0\0\0hasComponent\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0hasComponent\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0hasComponent\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\rComplexDomain\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nAffiliation\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nAffiliation\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0address\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0nameÅ\0\0\0address\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0name\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0address\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0Affiliation\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nReferenceEntity\0\0\0ancestor_order\0\0\0Å\0\0\0ReferenceEntity\0\0\0parent\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceMolecule\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0atomicConnectivity\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceMolecule\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0formula\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0ReferenceEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0atomicConnectivityÅ\0\0\0formula\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0otherIdentifierÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0atomicConnectivityÅ\0\0\0crossReferenceÅ\0\0\0formula\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceMolecule\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\n\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nFigure\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0url\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0url\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0url\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0url\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0Figure\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nFunctionalStatusType\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nFunctionalStatusType\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0nameÅ\0\0\0\ndefinition\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0FunctionalStatusType\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\0\0\0ancestor_order\0\0\0Å\0\0\0PhysicalEntity\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0SpeciesÅ\0\0\0Taxon\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nGenomeEncodedEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0species\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0species\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0Å\0\0\0cellTypeÅ\0\0\0compartmentÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0compartmentÅ\0\0\0nameÅ\0\0\0species\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0GenomeEncodedEntity\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\n	EntitySet\0\0\0ancestor_order\0\0\0Å\0\0\0	EntitySet\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Domain\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nCandidateSet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasCandidate\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n	EntitySet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasMember\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\n	EntitySet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObjectÅ\0\0\0	EntitySet\0\0\0ancestor\0\0\0Å\0\0\0hasCandidate\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0cellTypeÅ\0\0\0	hasMemberÅ\0\0\0hasCandidate\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0	hasMember\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0compartmentÅ\0\0\0hasCandidate\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0CandidateSet\0\0\0Ç\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Taxon\0\0\0ancestor\0\0\0\r\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nTaxon\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nSpecies\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nvarchar(255)\0\0\0db_col_type\nTaxon\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Taxon\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTaxon\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nsuperTaxon\0\0\0	attribute\0\0\0\nDatabaseObject\nTaxon\0\0\0ancestor_order\0\0\0Å\0\0\0figure\0\0\0\rown_attribute\0\0\0Å\0\0\0Taxon\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\nsuperTaxon\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0crossReference\0\0\0	MANDATORY\0\0\0Å\0\0\0figure\0\0\0OPTIONAL\0\0\0category\0\0\0Species\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nReferenceDatabase\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceDatabase\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0url\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceDatabase\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accessUrl\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0urlÅ\0\0\0nameÅ\0\0\0	accessUrl\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0urlÅ\0\0\0	accessUrl\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceDatabase\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nGO_CellularComponent\nCompartment\0\0\0ancestor_order\0\0\0Å\0\0\0Compartment\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0componentOf\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasPart\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accession\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0GO_CellularComponentÅ\0\0\0Compartment\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0	accession\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0componentOfÅ\0\0\0\ninstanceOf\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0	accession\0\0\0	MANDATORY\0\0\0Å\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0EntityCompartment\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nEvidenceType\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvidenceType\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\nEvidenceType\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0nameÅ\0\0\0\ninstanceOfÅ\0\0\0\ndefinition\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0\ndefinitionÅ\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0EvidenceType\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\n	Summation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n	Summation\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0text\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0textÅ\0\0\0literatureReference\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0text\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0text\0\0\0	MANDATORY\0\0\0Å\0\0\0literatureReference\0\0\0OPTIONAL\0\0\0category\0\0\0	Summation\0\0\0É\0\0\0ancestor_count\0\0\0Å\0\0\0ReferenceSequenceÅ\0\0\0ReferenceEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0keyword\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0checksum\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0\0\0Å\0\0\0ReferenceDNASequence\0\0\0allowed\nReferenceRNASequence\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rreferenceGene\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0isSequenceChanged\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceSequence\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0sequenceLength\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nReferenceSequence\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0species\0\0\0\0ó\0\0\0_displayName\0\0\0\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0description\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0geneName\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0comment\0\0\0\nReferenceSequence\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondaryIdentifier\0\0\0	attribute\0\0\0\nDatabaseObject\nReferenceEntity\nReferenceSequence\0\0\0ancestor_order\0\0\0Å\0\0\0\rreferenceGene\0\0\0\rown_attribute\0\0\0Å\0\0\0ReferenceSequence\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0\rreferenceGene\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceRNASequence\0\0\0É\0\0\0ancestor_count\0\0\0Å\0\0\0EventÅ\0\0\0DatabaseObjectÅ\0\0\0ReactionlikeEvent\0\0\0ancestor\0\0\0\'\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0requiredInputComponent\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0normalReaction\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0output\0\0\0\r\0\0\0disease\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0\nENUM(\'TRUE\',\'FALSE\')\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionlikeEvent\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nisChimeric\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0E\0\0\0relatedSpecies\0\0\0M\0\0\0\rreleaseStatus\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0\0\0Å\0\0\0EntityFunctionalStatus\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0entityFunctionalStatus\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0entityOnOtherCell\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0catalystActivity\0\0\0	attribute\0\0\0\0\0\0\0\rown_attribute\0\0\0\nDatabaseObject\nEvent\nReactionlikeEvent\0\0\0ancestor_order\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0entityFunctionalStatusÅ\0\0\0inputÅ\0\0\0catalystActivity\0\0\0all\0\0\0check\0\0\0FailedReaction\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\0\0\0ancestor_order\0\0\0Å\0\0\0PhysicalEntity\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\nENUM(\'TRUE\',\'FALSE\')\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nComplex\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nisChimeric\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Domain\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nComplex\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0includedLocation\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nComplex\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0entityOnOtherCell\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nComplex\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nComplex\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasComponent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0\nisChimericÅ\0\0\0entityOnOtherCellÅ\0\0\0includedLocationÅ\0\0\0speciesÅ\0\0\0hasComponent\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0cellTypeÅ\0\0\0hasComponent\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0species\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0includedLocationÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0compartmentÅ\0\0\0nameÅ\0\0\0hasComponent\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0\nisChimericÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0entityOnOtherCellÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0Complex\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nDatabaseIdentifier\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nDatabaseIdentifier\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\nvarchar(20)\0\0\0db_col_type\0\0\0max_cardinality\n1\0\0\0min_cardinality\nDatabaseIdentifier\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0crossReferenceÅ\0\0\0\nidentifierÅ\0\0\0referenceDatabase\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReference\0\0\0OPTIONAL\0\0\0category\0\0\0DatabaseIdentifier\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0l\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0componentOf\0\0\0{\0\0\0hasPart\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accession\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0nameÅ\0\0\0componentOfÅ\0\0\0hasPartÅ\0\0\0\ninstanceOfÅ\0\0\0	accessionÅ\0\0\0\ndefinition\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0	accession\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0componentOfÅ\0\0\0\ninstanceOf\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0nameÅ\0\0\0	accession\0\0\0	MANDATORY\0\0\0Å\0\0\0hasPartÅ\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0GO_CellularComponent\0\0\0	É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\nTranslationalModification\0\0\0ancestor_order\0\0\0Å\0\0\0TranslationalModification\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0’\0\0\0psiMod\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0Z\0\0\0modified\0\0\0\nCrosslinkedResidue\0\0\0origin\ndb_integer_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondCoordinate\0\0\0\0T\0\0\0\n_timestamp\0\0\0b\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0Í\0\0\0modification\0\0\0n\0\0\0referenceSequence\0\0\0˜\0\0\0\ncoordinate\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0TranslationalModificationÅ\0\0\0DatabaseObjectÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0modificationÅ\0\0\0secondCoordinate\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0modificationÅ\0\0\0referenceSequenceÅ\0\0\0\ncoordinateÅ\0\0\0psiModÅ\0\0\0secondCoordinate\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\ncoordinateÅ\0\0\0secondCoordinate\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0Å\0\0\0modification\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0CrosslinkedResidue\0\0\0Ñ\0\0\0ancestor_count\0\0\0Å\0\0\0FragmentModificationÅ\0\0\0DatabaseObjectÅ\0\0\0\ZGeneticallyModifiedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nFragmentModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0endPositionInReferenceSequence\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nFragmentModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0 startPositionInReferenceSequence\0\0\0n\0\0\0referenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\0\0\0\0\rown_attribute\0\0\0\nDatabaseObject\nAbstractModifiedResidue\n\ZGeneticallyModifiedResidue\nFragmentModification\0\0\0ancestor_order\0\0\0Å\0\0\0FragmentModification\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceSequenceÅ\0\0\0 startPositionInReferenceSequenceÅ\0\0\0endPositionInReferenceSequence\0\0\0all\0\0\0check\0\0\0FragmentDeletionModification\0\0\0É\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItemÅ\0\0\0Vertex\0\0\0ancestor\0\0\0\0\0\0L\0\0\0width\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionVertex\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0pointCoordinates\0\0\0T\0\0\0x\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0]\0\0\0height\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0representedInstance\0\0\0p\0\0\0y\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0y\0\0\0pathwayDiagram\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\nPathwayDiagramItem\nVertex\0\0\0ancestor_order\0\0\0Å\0\0\0pointCoordinates\0\0\0\rown_attribute\0\0\0Å\0\0\0Vertex\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0pathwayDiagramÅ\0\0\0representedInstance\0\0\0all\0\0\0check\0\0\0ReactionVertex\0\0\0Ñ\0\0\0ancestor_count\0\0\0\nDatabaseObject\nReferenceEntity\nReferenceSequence\nReferenceGeneProduct\0\0\0ancestor_order\0\0\0Å\0\0\0ReferenceGeneProduct\0\0\0parent\0\0\0\0\0\0}\0\0\0keyword\0\0\0\0)\0\0\0stableIdentifier\0\0\0É\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0â\0\0\0checksum\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0ReferenceGeneProduct\0\0\0allowed\nReferenceIsoform\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\risoformParent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceIsoform\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0variantIdentifier\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0ReferenceRNASequence\0\0\0allowed\nReferenceGeneProduct\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0referenceTranscript\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0\0\0Å\0\0\0ReferenceDNASequence\0\0\0allowed\nReferenceGeneProduct\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rreferenceGene\0\0\0¥\0\0\0isSequenceChanged\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceGeneProduct\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0_chainChangeLog\0\0\0ª\0\0\0name\0\0\0¿\0\0\0sequenceLength\0\0\0\0ó\0\0\0_displayName\0\0\0»\0\0\0species\0\0\0\0è\0\0\0\n__is_ghost\0\0\0”\0\0\0description\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0‡\0\0\0geneName\0\0\0\nReferenceGeneProduct\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0chain\0\0\0Ù\0\0\0secondaryIdentifier\0\0\0Ô\0\0\0comment\0\0\0	attribute\0\0\0Å\0\0\0ReferenceSequenceÅ\0\0\0ReferenceEntityÅ\0\0\0DatabaseObjectÅ\0\0\0ReferenceGeneProduct\0\0\0ancestor\0\0\0Å\0\0\0\risoformParentÅ\0\0\0variantIdentifier\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0variantIdentifierÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0\risoformParentÅ\0\0\0variantIdentifier\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceIsoform\0\0\0	Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\0\0\0ancestor_order\0\0\0Å\0\0\0AbstractModifiedResidue\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0’\0\0\0psiMod\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0n\0\0\0referenceSequence\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0\ncoordinateÅ\0\0\0psiMod\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceSequenceÅ\0\0\0\ncoordinateÅ\0\0\0psiMod\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\ncoordinate\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0psiMod\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0TranslationalModification\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0FunctionalStatus\0\0\0allowed\nEntityFunctionalStatus\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0functionalStatus\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEntityFunctionalStatus\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0physicalEntity\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0functionalStatusÅ\0\0\0physicalEntity\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0functionalStatusÅ\0\0\0physicalEntity\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0functionalStatusÅ\0\0\0physicalEntity\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0EntityFunctionalStatus\0\0\0Ñ\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\n\ZGeneticallyModifiedResidue\nFragmentModification\0\0\0ancestor_order\0\0\0Å\0\0\0FragmentModification\0\0\0parent\0\0\0\r\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nFragmentReplacedModification\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0alteredAminoAcidFragment\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\00\0\0\0endPositionInReferenceSequence\0\0\07\0\0\0 startPositionInReferenceSequence\0\0\0n\0\0\0referenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0FragmentModificationÅ\0\0\0DatabaseObjectÅ\0\0\0\ZGeneticallyModifiedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0alteredAminoAcidFragment\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0alteredAminoAcidFragmentÅ\0\0\0referenceSequenceÅ\0\0\0 startPositionInReferenceSequenceÅ\0\0\0endPositionInReferenceSequence\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0alteredAminoAcidFragment\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0FragmentReplacedModification\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0SequenceOntology\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nFunctionalStatus\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0structuralVariant\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0FunctionalStatusType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nFunctionalStatus\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0functionalStatusType\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0functionalStatusTypeÅ\0\0\0structuralVariant\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0functionalStatusTypeÅ\0\0\0structuralVariant\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0functionalStatusTypeÅ\0\0\0structuralVariant\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0FunctionalStatus\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\r\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0DeletedControlledVocabulary\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_Deleted\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0reason\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\n_Deleted\0\0\0origin\ndb_integer_type\0\0\0typeÅ\0\0\0multiple\0\0\0deletedInstanceDB_ID\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_Deleted\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0curatorComment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0EventÅ\0\0\0\nRegulation\0\0\0allowed\n_Deleted\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0replacementInstances\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0reasonÅ\0\0\0curatorCommentÅ\0\0\0replacementInstancesÅ\0\0\0deletedInstanceDB_ID\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0deletedInstanceDB_ID\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0_Deleted\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nExternalOntology\0\0\0ancestor_order\0\0\0Å\0\0\0ExternalOntology\0\0\0parent\0\0\0\0\0\0w\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0â\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0õ\0\0\0synonym\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0SequenceOntology\0\0\0allowed\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0ExternalOntology\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0\nidentifier\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0\nidentifierÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0SequenceOntology\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nExternalOntology\0\0\0ancestor_order\0\0\0Å\0\0\0ExternalOntology\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0synonym\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0PsiMod\0\0\0allowed\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0ExternalOntology\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0synonymÅ\0\0\0\ndefinitionÅ\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0PsiMod\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\0\0\0ancestor_order\0\0\0Å\0\0\0AbstractModifiedResidue\0\0\0parent\0\0\0\n\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0n\0\0\0referenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceSequence\0\0\0all\0\0\0check\0\0\0\0\0\0\0role\0\0\0\ZGeneticallyModifiedResidue\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\n	EntitySet\0\0\0ancestor_order\0\0\0Å\0\0\0	EntitySet\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Domain\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0ReferenceEntity\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nOpenSet\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceEntity\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n	EntitySet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasMember\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\n5\0\0\0name\0\0\0\n:\0\0\0species\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObjectÅ\0\0\0	EntitySet\0\0\0ancestor\0\0\0Å\0\0\0referenceEntity\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceEntityÅ\0\0\0cellTypeÅ\0\0\0compartment\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0referenceEntity\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0compartment\0\0\0	MANDATORY\0\0\0\rÅ\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0	hasMemberÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0OpenSet\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nReferenceEntity\0\0\0ancestor_order\0\0\0Å\0\0\0ReferenceEntity\0\0\0parent\0\0\0\0\0\0}\0\0\0keyword\0\0\0\0)\0\0\0stableIdentifier\0\0\0É\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0â\0\0\0checksum\0\0\0§\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0ƒ\0\0\0crossReference\0\0\0Ã\0\0\0referenceDatabase\0\0\0¥\0\0\0isSequenceChanged\0\0\0ª\0\0\0name\0\0\0¿\0\0\0sequenceLength\0\0\0\0ó\0\0\0_displayName\0\0\0»\0\0\0species\0\0\0\0è\0\0\0\n__is_ghost\0\0\0”\0\0\0description\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0‡\0\0\0geneName\0\0\0Ù\0\0\0secondaryIdentifier\0\0\0Ô\0\0\0comment\0\0\0	attribute\0\0\0Å\0\0\0ReferenceEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0	Å\0\0\0keywordÅ\0\0\0isSequenceChangedÅ\0\0\0sequenceLengthÅ\0\0\0speciesÅ\0\0\0descriptionÅ\0\0\0checksumÅ\0\0\0geneNameÅ\0\0\0commentÅ\0\0\0secondaryIdentifier\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0checksumÅ\0\0\0otherIdentifier\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0species\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0keywordÅ\0\0\0isSequenceChangedÅ\0\0\0nameÅ\0\0\0descriptionÅ\0\0\0geneNameÅ\0\0\0commentÅ\0\0\0secondaryIdentifier\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceSequence\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nEvent\nReactionlikeEvent\0\0\0ancestor_order\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0parent\0\0\0)\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0requiredInputComponent\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\n\rBlackBoxEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasEvent\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\rä\0\0\0normalReaction\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\r\0\0\0disease\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0output\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\rBlackBoxEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rtemplateEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0E\0\0\0relatedSpecies\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\rµ\0\0\0\nisChimeric\0\0\0M\0\0\0\rreleaseStatus\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\rœ\0\0\0entityFunctionalStatus\0\0\0\rÿ\0\0\0entityOnOtherCell\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0catalystActivity\0\0\0	attribute\0\0\0Å\0\0\0EventÅ\0\0\0DatabaseObjectÅ\0\0\0ReactionlikeEvent\0\0\0ancestor\0\0\0Å\0\0\0hasEventÅ\0\0\0\rtemplateEvent\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0entityFunctionalStatusÅ\0\0\0inputÅ\0\0\0output\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0reviewedÅ\0\0\0compartmentÅ\0\0\0inputÅ\0\0\0authoredÅ\0\0\0outputÅ\0\0\0literatureReferenceÅ\0\0\0edited\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDate\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0	summationÅ\0\0\0nameÅ\0\0\0species\0\0\0	MANDATORY\0\0\0\rÅ\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0goBiologicalProcessÅ\0\0\0inferredFromÅ\0\0\0\ndefinitionÅ\0\0\0requiredInputComponentÅ\0\0\0hasEventÅ\0\0\0\rtemplateEventÅ\0\0\0evidenceTypeÅ\0\0\0revisedÅ\0\0\0precedingEventÅ\0\0\0catalystActivityÅ\0\0\0orthologousEvent\0\0\0OPTIONAL\0\0\0category\0\0\0\rBlackBoxEvent\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\n\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\n	FrontPage\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rfrontPageItem\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0\rfrontPageItem\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0\rfrontPageItem\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0	FrontPage\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0componentOf\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0regulate\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_BiologicalProcess\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0negativelyRegulate\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0positivelyRegulate\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\nGO_BiologicalProcess\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_BiologicalProcess\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasPart\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_BiologicalProcess\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accession\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\nÅ\0\0\0negativelyRegulateÅ\0\0\0referenceDatabaseÅ\0\0\0positivelyRegulateÅ\0\0\0nameÅ\0\0\0componentOfÅ\0\0\0hasPartÅ\0\0\0	accessionÅ\0\0\0\ndefinitionÅ\0\0\0regulateÅ\0\0\0\ninstanceOf\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0	accession\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0componentOfÅ\0\0\0\ninstanceOf\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0nameÅ\0\0\0	accession\0\0\0	MANDATORY\0\0\0Å\0\0\0negativelyRegulateÅ\0\0\0positivelyRegulateÅ\0\0\0hasPartÅ\0\0\0regulateÅ\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0GO_BiologicalProcess\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPublication\0\0\0ancestor_order\0\0\0Å\0\0\0Publication\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0¬\0\0\0author\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nURL\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0uniformResourceLocator\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0œ\0\0\0title\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Publication\0\0\0ancestor\0\0\0Å\0\0\0uniformResourceLocator\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0uniformResourceLocator\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0uniformResourceLocator\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0URL\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\r\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\n	TIMESTAMP\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nInstanceEdit\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0dateTime\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\nInstanceEdit\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0author\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nInstanceEdit\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0note\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nInstanceEdit\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\Z_applyToAllEditedInstances\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0authorÅ\0\0\0dateTimeÅ\0\0\0\Z_applyToAllEditedInstancesÅ\0\0\0note\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0InstanceEdit\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0componentOf\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0regulate\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_MolecularFunction\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0GO_MolecularFunction\0\0\0allowed\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0negativelyRegulate\0\0\0\nGO_MolecularFunction\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0ecNumber\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0positivelyRegulate\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\nGO_MolecularFunction\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nGO_MolecularFunction\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasPart\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_MolecularFunction\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accession\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0ecNumberÅ\0\0\0negativelyRegulateÅ\0\0\0referenceDatabaseÅ\0\0\0positivelyRegulateÅ\0\0\0nameÅ\0\0\0componentOfÅ\0\0\0hasPartÅ\0\0\0	accessionÅ\0\0\0\ndefinitionÅ\0\0\0regulateÅ\0\0\0\ninstanceOf\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0	accession\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0componentOfÅ\0\0\0\ninstanceOf\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0nameÅ\0\0\0	accession\0\0\0	MANDATORY\0\0\0Å\0\0\0ecNumberÅ\0\0\0negativelyRegulateÅ\0\0\0positivelyRegulateÅ\0\0\0hasPartÅ\0\0\0regulateÅ\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0GO_MolecularFunction\0\0\0Ñ\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\n\ZGeneticallyModifiedResidue\nFragmentModification\0\0\0ancestor_order\0\0\0Å\0\0\0FragmentModification\0\0\0parent\0\0\0\r\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\00\0\0\0endPositionInReferenceSequence\0\0\07\0\0\0 startPositionInReferenceSequence\0\0\0n\0\0\0referenceSequence\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nFragmentInsertionModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0FragmentModificationÅ\0\0\0DatabaseObjectÅ\0\0\0\ZGeneticallyModifiedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0\ncoordinate\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceSequenceÅ\0\0\0 startPositionInReferenceSequenceÅ\0\0\0endPositionInReferenceSequenceÅ\0\0\0\ncoordinate\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\ncoordinate\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0FragmentInsertionModification\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\n\nRegulation\0\0\0ancestor_order\0\0\0Å\0\0\0\nRegulation\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0RegulationType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulationType\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0	\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0containedInPathway\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	regulator\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulatedEntity\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0\nRegulation\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDateÅ\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0	MANDATORY\0\0\0Å\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0	summationÅ\0\0\0literatureReferenceÅ\0\0\0regulationTypeÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0NegativeRegulation\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nTaxon\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0)\0\0\0stableIdentifier\0\0\0\n∏\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Taxon\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTaxon\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nsuperTaxon\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0crossReferenceÅ\0\0\0nameÅ\0\0\0\nsuperTaxon\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\nsuperTaxon\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0crossReferenceÅ\0\0\0name\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0Taxon\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nEvent\nReactionlikeEvent\0\0\0ancestor_order\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0parent\0\0\0\'\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0requiredInputComponent\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\rä\0\0\0normalReaction\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\r\0\0\0disease\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0output\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\rµ\0\0\0\nisChimeric\0\0\0E\0\0\0relatedSpecies\0\0\0M\0\0\0\rreleaseStatus\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\rœ\0\0\0entityFunctionalStatus\0\0\0\rÿ\0\0\0entityOnOtherCell\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0catalystActivity\0\0\0	attribute\0\0\0Å\0\0\0EventÅ\0\0\0DatabaseObjectÅ\0\0\0ReactionlikeEvent\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0entityFunctionalStatusÅ\0\0\0inputÅ\0\0\0catalystActivityÅ\0\0\0output\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0reviewedÅ\0\0\0compartmentÅ\0\0\0authoredÅ\0\0\0catalystActivityÅ\0\0\0literatureReferenceÅ\0\0\0edited\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDate\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0inputÅ\0\0\0	summationÅ\0\0\0nameÅ\0\0\0speciesÅ\0\0\0output\0\0\0	MANDATORY\0\0\0\nÅ\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0goBiologicalProcessÅ\0\0\0inferredFromÅ\0\0\0\ndefinitionÅ\0\0\0requiredInputComponentÅ\0\0\0evidenceTypeÅ\0\0\0revisedÅ\0\0\0precedingEventÅ\0\0\0orthologousEvent\0\0\0OPTIONAL\0\0\0category\0\0\0Depolymerisation\0\0\0Ñ\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\nTranslationalModification\nCrosslinkedResidue\0\0\0ancestor_order\0\0\0Å\0\0\0CrosslinkedResidue\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0InterChainCrosslinkedResidue\0\0\0allowed\nInterChainCrosslinkedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0equivalentTo\0\0\0\0)\0\0\0stableIdentifier\0\0\0’\0\0\0psiMod\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0Ó\0\0\0secondCoordinate\0\0\0Z\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0Í\0\0\0modification\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\nInterChainCrosslinkedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondReferenceSequence\0\0\0˜\0\0\0\ncoordinate\0\0\0n\0\0\0referenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0TranslationalModificationÅ\0\0\0DatabaseObjectÅ\0\0\0CrosslinkedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0equivalentToÅ\0\0\0secondReferenceSequence\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0modificationÅ\0\0\0secondReferenceSequenceÅ\0\0\0referenceSequenceÅ\0\0\0\ncoordinateÅ\0\0\0psiModÅ\0\0\0secondCoordinate\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0equivalentToÅ\0\0\0secondReferenceSequence\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0InterChainCrosslinkedResidue\0\0\0É\0\0\0ancestor_count\0\0\0Å\0\0\0ReferenceSequenceÅ\0\0\0ReferenceEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0}\0\0\0keyword\0\0\0\0)\0\0\0stableIdentifier\0\0\0É\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0â\0\0\0checksum\0\0\0\0T\0\0\0\n_timestamp\0\0\0§\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0º\0\0\0referenceTranscript\0\0\0ƒ\0\0\0crossReference\0\0\0Ã\0\0\0referenceDatabase\0\0\0÷\0\0\0\rreferenceGene\0\0\0¥\0\0\0isSequenceChanged\0\0\0¿\0\0\0sequenceLength\0\0\0ª\0\0\0name\0\0\0ﬂ\0\0\0_chainChangeLog\0\0\0\0è\0\0\0\n__is_ghost\0\0\0»\0\0\0species\0\0\0\0ó\0\0\0_displayName\0\0\0”\0\0\0description\0\0\0\0Æ\0\0\0modified\0\0\0‡\0\0\0geneName\0\0\0\0∂\0\0\0created\0\0\0Ô\0\0\0chain\0\0\0Ô\0\0\0comment\0\0\0Ù\0\0\0secondaryIdentifier\0\0\0	attribute\0\0\0\nDatabaseObject\nReferenceEntity\nReferenceSequence\0\0\0ancestor_order\0\0\0Å\0\0\0chainÅ\0\0\0\rreferenceGeneÅ\0\0\0_chainChangeLogÅ\0\0\0referenceTranscript\0\0\0\rown_attribute\0\0\0Å\0\0\0ReferenceSequence\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0Å\0\0\0chainÅ\0\0\0_chainChangeLog\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceGeneProduct\0\0\0\0\0\0\0\0\0\0ancestorÄ\0\0\0ancestor_count\0\0\0	\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0	Å\0\0\0stableIdentifierÅ\0\0\0\n__is_ghostÅ\0\0\0_displayNameÅ\0\0\0modifiedÅ\0\0\0_classÅ\0\0\0\n_timestampÅ\0\0\0createdÅ\0\0\0DB_IDÅ\0\0\0_Protege_id\0\0\0\rown_attribute\0\0\0\0\0\0\0ancestor_order\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0stableIdentifierÅ\0\0\0DB_IDÅ\0\0\0_displayNameÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0DatabaseObject\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\n\nRegulation\0\0\0ancestor_order\0\0\0Å\0\0\0\nRegulation\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0RegulationType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulationType\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0	\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0containedInPathway\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	regulator\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulatedEntity\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0\nRegulation\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDateÅ\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0	MANDATORY\0\0\0Å\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0	summationÅ\0\0\0literatureReferenceÅ\0\0\0regulationTypeÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0PositiveRegulation\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\n\ZGeneticallyModifiedResidue\0\0\0ancestor_order\0\0\0Å\0\0\0\ZGeneticallyModifiedResidue\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PsiMod\0\0\0allowed\nReplacedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0psiMod\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nAbstractModifiedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceSequence\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReplacedResidue\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0\ZGeneticallyModifiedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0\ncoordinateÅ\0\0\0psiMod\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0\ncoordinateÅ\0\0\0referenceSequence\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\ncoordinate\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceSequenceÅ\0\0\0psiMod\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0ReplacedResidue\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPublication\0\0\0ancestor_order\0\0\0Å\0\0\0Publication\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\nPublication\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0author\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nBook\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0ISBN\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Affiliation\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nBook\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	publisher\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nBook\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0chapterTitle\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPublication\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0title\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\nBook\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0chapterAuthors\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nBook\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0pages\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nBook\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0year\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Publication\0\0\0ancestor\0\0\0Å\0\0\0ISBNÅ\0\0\0chapterAuthorsÅ\0\0\0	publisherÅ\0\0\0yearÅ\0\0\0pagesÅ\0\0\0chapterTitle\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0ISBNÅ\0\0\0titleÅ\0\0\0chapterTitle\0\0\0any\0\0\0Å\0\0\0author\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0authorÅ\0\0\0titleÅ\0\0\0year\0\0\0	MANDATORY\0\0\0Å\0\0\0ISBNÅ\0\0\0	publisherÅ\0\0\0chapterAuthorsÅ\0\0\0pagesÅ\0\0\0chapterTitle\0\0\0OPTIONAL\0\0\0category\0\0\0Book\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPublication\0\0\0ancestor_order\0\0\0Å\0\0\0Publication\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0¬\0\0\0author\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nLiteratureReference\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0pubMedIdentifier\0\0\0\nvarchar(255)\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nLiteratureReference\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0journal\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nLiteratureReference\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0volume\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0œ\0\0\0title\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nLiteratureReference\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0pages\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nLiteratureReference\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0year\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Publication\0\0\0ancestor\0\0\0Å\0\0\0volumeÅ\0\0\0pubMedIdentifierÅ\0\0\0yearÅ\0\0\0pagesÅ\0\0\0journal\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0pubMedIdentifier\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0pubMedIdentifierÅ\0\0\0journal\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0year\0\0\0	MANDATORY\0\0\0Å\0\0\0volumeÅ\0\0\0pages\0\0\0OPTIONAL\0\0\0category\0\0\0LiteratureReference\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nStableIdentifier\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0identifierVersion\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nStableIdentifier\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nStableIdentifier\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0identifierVersionÅ\0\0\0\nidentifierÅ\0\0\0referenceDatabase\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0identifierVersionÅ\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0StableIdentifier\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_AttributeValueBeforeChange\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0changedAttribute\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\n_AttributeValueBeforeChange\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rpreviousValue\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0changedAttributeÅ\0\0\0\rpreviousValue\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0_AttributeValueBeforeChange\0\0\0	Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\0\0\0ancestor_order\0\0\0Å\0\0\0PhysicalEntity\0\0\0parent\0\0\0\0\0\0\0\0\0\0figure\0\0\0\0!\0\0\0reviewed\0\0\0\0\0\0\0cellType\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Domain\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\08\0\0\0	summation\0\0\0\0@\0\0\0literatureReference\0\0\0\0M\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n	EntitySet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasMember\0\0\0\0`\0\0\0compartment\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0p\0\0\0edited\0\0\0\0x\0\0\0crossReference\0\0\0\0Ä\0\0\0goCellularComponent\0\0\0\n5\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\n:\0\0\0species\0\0\0\0¶\0\0\0disease\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\0¿\0\0\0authored\0\0\0\0“\0\0\0revised\0\0\0\0 \0\0\0\ninferredTo\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0	hasMemberÅ\0\0\0species\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0cellType\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0species\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0	EntitySet\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\r\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\nCatalystActivity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\nactiveUnit\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0GO_MolecularFunction\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nCatalystActivity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0activity\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nCatalystActivity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nCatalystActivity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0physicalEntity\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0\nactiveUnitÅ\0\0\0physicalEntityÅ\0\0\0literatureReferenceÅ\0\0\0activity\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0physicalEntityÅ\0\0\0activity\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0activity\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0physicalEntity\0\0\0	MANDATORY\0\0\0Å\0\0\0\nactiveUnitÅ\0\0\0literatureReference\0\0\0OPTIONAL\0\0\0category\0\0\0CatalystActivity\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\nTranslationalModification\0\0\0ancestor_order\0\0\0Å\0\0\0TranslationalModification\0\0\0parent\0\0\0\r\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PsiMod\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0psiMod\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0Z\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0b\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0ReferenceGroupÅ\0\0\0PolymerÅ\0\0\0	EntitySet\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nGroupModifiedResidue\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0modification\0\0\0n\0\0\0referenceSequence\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nTranslationalModification\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0TranslationalModificationÅ\0\0\0DatabaseObjectÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0modification\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0modificationÅ\0\0\0\ncoordinateÅ\0\0\0referenceSequenceÅ\0\0\0psiMod\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\ncoordinate\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0modificationÅ\0\0\0psiMod\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0GroupModifiedResidue\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nVARCHAR(255)\0\0\0db_col_type\nRegulationType\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0RegulationType\0\0\0allowed\nRegulationType\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0nameÅ\0\0\0\ninstanceOf\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0RegulationType\0\0\0	É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nAbstractModifiedResidue\n\ZGeneticallyModifiedResidue\0\0\0ancestor_order\0\0\0Å\0\0\0\ZGeneticallyModifiedResidue\0\0\0parent\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0Z\0\0\0modified\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0n\0\0\0referenceSequence\0\0\07\0\0\0 startPositionInReferenceSequence\0\0\00\0\0\0endPositionInReferenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0\ZGeneticallyModifiedResidueÅ\0\0\0AbstractModifiedResidue\0\0\0ancestor\0\0\0Å\0\0\0endPositionInReferenceSequenceÅ\0\0\0 startPositionInReferenceSequence\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceSequenceÅ\0\0\0endPositionInReferenceSequenceÅ\0\0\0 startPositionInReferenceSequence\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0endPositionInReferenceSequenceÅ\0\0\0 startPositionInReferenceSequence\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0FragmentModification\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\0\0\0ancestor_order\0\0\0Å\0\0\0PhysicalEntity\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPolymer\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxUnitCount\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPolymer\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minUnitCount\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nPolymer\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPolymer\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0repeatedUnit\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0maxUnitCountÅ\0\0\0minUnitCountÅ\0\0\0speciesÅ\0\0\0repeatedUnit\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0cellTypeÅ\0\0\0repeatedUnit\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0species\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0compartmentÅ\0\0\0nameÅ\0\0\0repeatedUnit\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0maxUnitCountÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0minUnitCountÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0Polymer\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\n	EntitySet\0\0\0ancestor_order\0\0\0Å\0\0\0	EntitySet\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Domain\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n	EntitySet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasMember\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\n5\0\0\0name\0\0\0\n:\0\0\0species\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObjectÅ\0\0\0	EntitySet\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0cellTypeÅ\0\0\0	hasMember\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0	hasMemberÅ\0\0\0compartment\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0\nDefinedSet\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPerson\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPerson\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	firstname\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPerson\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0project\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPerson\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0url\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPerson\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\nvarchar(255)\0\0\0db_col_type\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPerson\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0surname\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\nvarchar(10)\0\0\0db_col_type\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPerson\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0initial\0\0\0\0\0\0Å\0\0\0Affiliation\0\0\0allowed\nPerson\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0affiliation\0\0\0\nvarchar(255)\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPerson\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0eMailAddress\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0	Å\0\0\0crossReferenceÅ\0\0\0	firstnameÅ\0\0\0figureÅ\0\0\0projectÅ\0\0\0surnameÅ\0\0\0urlÅ\0\0\0eMailAddressÅ\0\0\0initialÅ\0\0\0affiliation\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0	firstnameÅ\0\0\0projectÅ\0\0\0initialÅ\0\0\0surname\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0initialÅ\0\0\0surname\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0	firstnameÅ\0\0\0projectÅ\0\0\0urlÅ\0\0\0eMailAddressÅ\0\0\0affiliation\0\0\0OPTIONAL\0\0\0category\0\0\0Person\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nGO_CellularComponent\0\0\0ancestor_order\0\0\0Å\0\0\0GO_CellularComponent\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0l\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0componentOf\0\0\0{\0\0\0hasPart\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nGO_CellularComponent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accession\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\nGO_CellularComponent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0GO_CellularComponent\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0	accession\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0componentOfÅ\0\0\0\ninstanceOf\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0	accession\0\0\0	MANDATORY\0\0\0Å\0\0\0\ndefinition\0\0\0OPTIONAL\0\0\0category\0\0\0Compartment\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nReferenceEntity\0\0\0ancestor_order\0\0\0Å\0\0\0ReferenceEntity\0\0\0parent\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceGroup\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0atomicConnectivity\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceGroup\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0formula\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0ReferenceEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0atomicConnectivityÅ\0\0\0formula\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0otherIdentifierÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0atomicConnectivityÅ\0\0\0crossReferenceÅ\0\0\0formula\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceGroup\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\0\0\0ancestor_order\0\0\0Å\0\0\0PhysicalEntity\0\0\0parent\0\0\0\Z\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0Å\0\0\0name\0\0\0any\0\0\0Å\0\0\0cellTypeÅ\0\0\0compartment\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0compartmentÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0OtherEntity\0\0\0É\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItemÅ\0\0\0Vertex\0\0\0ancestor\0\0\0\0\0\0L\0\0\0width\0\0\0\0\0\0Å\0\0\0EntityVertex\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEntityVertex\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0containedInEntityVertex\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0x\0\0\0]\0\0\0height\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0representedInstance\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0y\0\0\0y\0\0\0pathwayDiagram\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\nPathwayDiagramItem\nVertex\0\0\0ancestor_order\0\0\0Å\0\0\0containedInEntityVertex\0\0\0\rown_attribute\0\0\0Å\0\0\0Vertex\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0yÅ\0\0\0pathwayDiagramÅ\0\0\0xÅ\0\0\0representedInstance\0\0\0all\0\0\0check\0\0\0EntityVertex\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\r¨\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0\rΩ\0\0\0reviewed\0\0\0E\0\0\0relatedSpecies\0\0\0\0)\0\0\0stableIdentifier\0\0\0M\0\0\0\rreleaseStatus\0\0\0\r0\0\0\0	summation\0\0\0\r8\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\r»\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\r@\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\rS\0\0\0precedingEvent\0\0\0\r[\0\0\0edited\0\0\0\rk\0\0\0crossReference\0\0\0\rc\0\0\0releaseDate\0\0\0\rs\0\0\0goBiologicalProcess\0\0\0\r}\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\r‡\0\0\0species\0\0\0\rÇ\0\0\0inferredFrom\0\0\0\r\0\0\0disease\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\rÎ\0\0\0authored\0\0\0\rÛ\0\0\0evidenceType\0\0\0\rú\0\0\0revised\0\0\0\r§\0\0\0orthologousEvent\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0\n_doReleaseÅ\0\0\0figureÅ\0\0\0relatedSpeciesÅ\0\0\0reviewedÅ\0\0\0\rreleaseStatusÅ\0\0\0	summationÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0compartmentÅ\0\0\0precedingEventÅ\0\0\0editedÅ\0\0\0crossReferenceÅ\0\0\0releaseDateÅ\0\0\0goBiologicalProcessÅ\0\0\0nameÅ\0\0\0speciesÅ\0\0\0inferredFromÅ\0\0\0diseaseÅ\0\0\0evidenceTypeÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0orthologousEvent\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0\rreleaseStatus\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0releaseDate\0\0\0	MANDATORY\0\0\0Å\0\0\0\n_doReleaseÅ\0\0\0relatedSpeciesÅ\0\0\0authoredÅ\0\0\0disease\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0Event\0\0\0Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0	\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0role\0\0\0PathwayDiagramItem\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\n\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0Z\0\0\0modified\0\0\0b\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0n\0\0\0referenceSequence\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0referenceSequence\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceSequence\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceSequence\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0AbstractModifiedResidue\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nReferenceEntity\nReferenceSequence\0\0\0ancestor_order\0\0\0Å\0\0\0ReferenceSequence\0\0\0parent\0\0\0\0\0\0}\0\0\0keyword\0\0\0\0)\0\0\0stableIdentifier\0\0\0É\0\0\0otherIdentifier\0\0\0\0H\0\0\0_class\0\0\0â\0\0\0checksum\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReferenceEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0¥\0\0\0isSequenceChanged\0\0\0ª\0\0\0name\0\0\0¿\0\0\0sequenceLength\0\0\0\0è\0\0\0\n__is_ghost\0\0\0»\0\0\0species\0\0\0\0ó\0\0\0_displayName\0\0\0”\0\0\0description\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0‡\0\0\0geneName\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0Ô\0\0\0comment\0\0\0Ù\0\0\0secondaryIdentifier\0\0\0	attribute\0\0\0Å\0\0\0ReferenceSequenceÅ\0\0\0ReferenceEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0species\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifier\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReference\0\0\0OPTIONAL\0\0\0category\0\0\0ReferenceDNASequence\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nEvent\nReactionlikeEvent\0\0\0ancestor_order\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0parent\0\0\0\'\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0requiredInputComponent\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\rä\0\0\0normalReaction\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\r\0\0\0disease\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0output\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\rµ\0\0\0\nisChimeric\0\0\0E\0\0\0relatedSpecies\0\0\0M\0\0\0\rreleaseStatus\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\rœ\0\0\0entityFunctionalStatus\0\0\0\rÿ\0\0\0entityOnOtherCell\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0∂\0\0\0created\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0catalystActivity\0\0\0	attribute\0\0\0Å\0\0\0EventÅ\0\0\0DatabaseObjectÅ\0\0\0ReactionlikeEvent\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0entityFunctionalStatusÅ\0\0\0inputÅ\0\0\0catalystActivityÅ\0\0\0output\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0reviewedÅ\0\0\0compartmentÅ\0\0\0authoredÅ\0\0\0catalystActivityÅ\0\0\0literatureReferenceÅ\0\0\0edited\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDate\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0inputÅ\0\0\0	summationÅ\0\0\0nameÅ\0\0\0speciesÅ\0\0\0output\0\0\0	MANDATORY\0\0\0\nÅ\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0goBiologicalProcessÅ\0\0\0inferredFromÅ\0\0\0\ndefinitionÅ\0\0\0requiredInputComponentÅ\0\0\0evidenceTypeÅ\0\0\0revisedÅ\0\0\0precedingEventÅ\0\0\0orthologousEvent\0\0\0OPTIONAL\0\0\0category\0\0\0Polymerisation\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nConcurrentEventSet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0concurrentEvents\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nConcurrentEventSet\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0focusEntity\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0concurrentEventsÅ\0\0\0focusEntity\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0concurrentEventsÅ\0\0\0focusEntity\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0ConcurrentEventSet\0\0\0	Å\0\0\0ancestor_count\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0w\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0Ç\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0â\0\0\0\ndefinition\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0î\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0õ\0\0\0synonym\0\0\0\0h\0\0\0_Protege_id\0\0\0\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0\nidentifierÅ\0\0\0referenceDatabaseÅ\0\0\0synonymÅ\0\0\0nameÅ\0\0\0\ninstanceOfÅ\0\0\0\ndefinition\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0synonymÅ\0\0\0\ndefinitionÅ\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0ExternalOntology\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0sourceX\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nReactionCoordinates\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0locatedEvent\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionCoordinates\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0locationContext\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0targetX\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0targetY\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0sourceY\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0targetYÅ\0\0\0locatedEventÅ\0\0\0sourceXÅ\0\0\0locationContextÅ\0\0\0sourceYÅ\0\0\0targetX\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0targetYÅ\0\0\0sourceXÅ\0\0\0locatedEventÅ\0\0\0sourceYÅ\0\0\0targetX\0\0\0	MANDATORY\0\0\0Å\0\0\0locationContext\0\0\0OPTIONAL\0\0\0category\0\0\0ReactionCoordinates\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nEvent\nReactionlikeEvent\0\0\0ancestor_order\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0parent\0\0\0(\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0\0\0Å\0\0\0Reaction\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReaction\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0reverseReaction\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0requiredInputComponent\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\rä\0\0\0normalReaction\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\r\0\0\0disease\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0output\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0E\0\0\0relatedSpecies\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\rµ\0\0\0\nisChimeric\0\0\0M\0\0\0\rreleaseStatus\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\rœ\0\0\0entityFunctionalStatus\0\0\0\rÿ\0\0\0entityOnOtherCell\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0catalystActivity\0\0\0	attribute\0\0\0Å\0\0\0EventÅ\0\0\0DatabaseObjectÅ\0\0\0ReactionlikeEvent\0\0\0ancestor\0\0\0Å\0\0\0reverseReaction\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0entityFunctionalStatusÅ\0\0\0inputÅ\0\0\0catalystActivityÅ\0\0\0output\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0reviewedÅ\0\0\0compartmentÅ\0\0\0authoredÅ\0\0\0catalystActivityÅ\0\0\0literatureReferenceÅ\0\0\0edited\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDateÅ\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0inputÅ\0\0\0	summationÅ\0\0\0nameÅ\0\0\0speciesÅ\0\0\0output\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0goBiologicalProcessÅ\0\0\0inferredFromÅ\0\0\0\ndefinitionÅ\0\0\0reverseReactionÅ\0\0\0requiredInputComponentÅ\0\0\0evidenceTypeÅ\0\0\0revisedÅ\0\0\0precedingEventÅ\0\0\0orthologousEvent\0\0\0OPTIONAL\0\0\0category\0\0\0Reaction\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nExternalOntology\0\0\0ancestor_order\0\0\0Å\0\0\0ExternalOntology\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0)\0\0\0stableIdentifier\0\0\0\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0∂\0\0\0created\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\0Y\0\0\0DB_ID\0\0\0\nExternalOntology\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0synonym\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0Disease\0\0\0allowed\nExternalOntology\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0ExternalOntology\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0\nidentifier\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceDatabaseÅ\0\0\0\nidentifierÅ\0\0\0name\0\0\0	MANDATORY\0\0\0Å\0\0\0synonymÅ\0\0\0\ndefinitionÅ\0\0\0\ninstanceOf\0\0\0OPTIONAL\0\0\0category\0\0\0Disease\0\0\0Ç\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItem\0\0\0ancestor\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0ReferenceEntityÅ\0\0\0DatabaseIdentifierÅ\0\0\0EventÅ\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\nVertexSearchableTerm\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0termProvider\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0SpeciesÅ\0\0\0Taxon\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertexSearchableTerm\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0species\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertexSearchableTerm\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rproviderCount\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertexSearchableTerm\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0vertexCount\0\0\0\0h\0\0\0_Protege_id\0\0\0\nVARCHAR(255)\0\0\0db_col_type\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertexSearchableTerm\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0searchableTerm\0\0\0\0\0\0Å\0\0\0Vertex\0\0\0allowed\nVertexSearchableTerm\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0vertex\0\0\0	attribute\0\0\0\nDatabaseObject\nPathwayDiagramItem\0\0\0ancestor_order\0\0\0Å\0\0\0\rproviderCountÅ\0\0\0vertexCountÅ\0\0\0termProviderÅ\0\0\0speciesÅ\0\0\0vertexÅ\0\0\0searchableTerm\0\0\0\rown_attribute\0\0\0Å\0\0\0PathwayDiagramItem\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0VertexSearchableTerm\0\0\0Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPathwayDiagramItem\0\0\0ancestor_order\0\0\0Å\0\0\0PathwayDiagramItem\0\0\0parent\0\0\0\r\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayDiagram\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0width\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayDiagram\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0height\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\nLONGBLOB\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPathwayDiagram\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0storedATXML\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\nPathwayDiagram\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0representedPathway\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItem\0\0\0ancestor\0\0\0Å\0\0\0widthÅ\0\0\0storedATXMLÅ\0\0\0representedPathwayÅ\0\0\0height\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0representedPathway\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0widthÅ\0\0\0storedATXMLÅ\0\0\0height\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0representedPathway\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0PathwayDiagram\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\nPhysicalEntity\nGenomeEncodedEntity\0\0\0ancestor_order\0\0\0Å\0\0\0GenomeEncodedEntity\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0\0cellType\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0SequenceDomain\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nEntityWithAccessionedSequence\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceEntity\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEntityWithAccessionedSequence\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0startCoordinate\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0AbstractModifiedResidue\0\0\0allowed\nEntityWithAccessionedSequence\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasModifiedResidue\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nEntityWithAccessionedSequence\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rendCoordinate\0\0\0\nPhysicalEntity\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nGenomeEncodedEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0species\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0¶\0\0\0disease\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0 \0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nPhysicalEntity\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0	attribute\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0GenomeEncodedEntityÅ\0\0\0DatabaseObject\0\0\0ancestor\0\0\0Å\0\0\0referenceEntityÅ\0\0\0\rendCoordinateÅ\0\0\0hasModifiedResidueÅ\0\0\0startCoordinate\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0referenceEntityÅ\0\0\0cellTypeÅ\0\0\0\rendCoordinateÅ\0\0\0hasModifiedResidueÅ\0\0\0compartmentÅ\0\0\0startCoordinate\0\0\0all\0\0\0check\0\0\0\0\0\0Å\0\0\0\rendCoordinateÅ\0\0\0hasModifiedResidueÅ\0\0\0startCoordinate\0\0\0REQUIRED\0\0\0Å\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0referenceEntityÅ\0\0\0compartmentÅ\0\0\0nameÅ\0\0\0species\0\0\0	MANDATORY\0\0\0Å\0\0\0crossReferenceÅ\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0goCellularComponentÅ\0\0\0	hasDomainÅ\0\0\0	summationÅ\0\0\0inferredFromÅ\0\0\0literatureReferenceÅ\0\0\0\ndefinitionÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0EntityWithAccessionedSequence\0\0\0É\0\0\0ancestor_count\0\0\0\nDatabaseObject\n\nRegulation\nPositiveRegulation\0\0\0ancestor_order\0\0\0Å\0\0\0PositiveRegulation\0\0\0parent\0\0\0\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0H\0\0\0_class\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0Å\0\0\0RegulationType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulationType\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\ndate\0\0\0db_col_type\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\n\nother_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0	\0\0\0name\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\nDatabaseObject\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0containedInPathway\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	regulator\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n\nRegulation\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulatedEntity\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0\nRegulationÅ\0\0\0PositiveRegulation\0\0\0ancestor\0\0\0\0\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0Å\0\0\0releaseDateÅ\0\0\0createdÅ\0\0\0modified\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0	regulatorÅ\0\0\0regulatedEntity\0\0\0	MANDATORY\0\0\0Å\0\0\0figureÅ\0\0\0reviewedÅ\0\0\0	summationÅ\0\0\0literatureReferenceÅ\0\0\0regulationTypeÅ\0\0\0authoredÅ\0\0\0revisedÅ\0\0\0edited\0\0\0OPTIONAL\0\0\0category\0\0\0Requirement\0\0\0	Ç\0\0\0ancestor_count\0\0\0\nDatabaseObject\nEvent\0\0\0ancestor_order\0\0\0Å\0\0\0Event\0\0\0parent\0\0\0\'\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\r0\0\0\0	summation\0\0\0\r8\0\0\0literatureReference\0\0\0\r@\0\0\0compartment\0\0\0\0Y\0\0\0DB_ID\0\0\0\rI\0\0\0requiredInputComponent\0\0\0\rS\0\0\0precedingEvent\0\0\0\0h\0\0\0_Protege_id\0\0\0\r[\0\0\0edited\0\0\0\rc\0\0\0releaseDate\0\0\0\rk\0\0\0crossReference\0\0\0\rs\0\0\0goBiologicalProcess\0\0\0\r}\0\0\0name\0\0\0\rÇ\0\0\0inferredFrom\0\0\0\rä\0\0\0normalReaction\0\0\0\r\0\0\0disease\0\0\0\rí\0\0\0output\0\0\0\0Æ\0\0\0modified\0\0\0\rú\0\0\0revised\0\0\0\r§\0\0\0orthologousEvent\0\0\0\r¨\0\0\0figure\0\0\05\0\0\0\n_doRelease\0\0\0\rµ\0\0\0\nisChimeric\0\0\0\rΩ\0\0\0reviewed\0\0\0E\0\0\0relatedSpecies\0\0\0M\0\0\0\rreleaseStatus\0\0\0\0H\0\0\0_class\0\0\0\r»\0\0\0\ndefinition\0\0\0\rœ\0\0\0entityFunctionalStatus\0\0\0\0T\0\0\0\n_timestamp\0\0\0\rÿ\0\0\0entityOnOtherCell\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\r‡\0\0\0species\0\0\0\0∂\0\0\0created\0\0\0\rÛ\0\0\0evidenceType\0\0\0\rÎ\0\0\0authored\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\nReactionlikeEvent\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0catalystActivity\0\0\0	attribute\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0Event\0\0\0ancestor\0\0\0Å\0\0\0\nisChimericÅ\0\0\0inputÅ\0\0\0entityOnOtherCellÅ\0\0\0normalReactionÅ\0\0\0outputÅ\0\0\0entityFunctionalStatusÅ\0\0\0requiredInputComponentÅ\0\0\0catalystActivity\0\0\0\rown_attribute\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0entityFunctionalStatus\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0\0\0\0\0	MANDATORY\0\0\0Å\0\0\0entityFunctionalStatusÅ\0\0\0\nisChimericÅ\0\0\0entityOnOtherCellÅ\0\0\0normalReaction\0\0\0OPTIONAL\0\0\0category\0\0\0\0\0\0\0role\0\0\0ReactionlikeEvent\0\0\0Ç\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObjectÅ\0\0\0PathwayDiagramItem\0\0\0ancestor\0\0\0\0\0\0L\0\0\0width\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0ó\0\0\0_displayName\0\0\0\0è\0\0\0\n__is_ghost\0\0\0T\0\0\0x\0\0\0]\0\0\0height\0\0\0\0H\0\0\0_class\0\0\0\0Æ\0\0\0modified\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nVertex\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0representedInstance\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0p\0\0\0y\0\0\0\0Y\0\0\0DB_ID\0\0\0y\0\0\0pathwayDiagram\0\0\0\0h\0\0\0_Protege_id\0\0\0	attribute\0\0\0\nDatabaseObject\nPathwayDiagramItem\0\0\0ancestor_order\0\0\0Å\0\0\0yÅ\0\0\0widthÅ\0\0\0pathwayDiagramÅ\0\0\0xÅ\0\0\0heightÅ\0\0\0representedInstance\0\0\0\rown_attribute\0\0\0Å\0\0\0PathwayDiagramItem\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0Å\0\0\0pathwayDiagramÅ\0\0\0representedInstance\0\0\0all\0\0\0check\0\0\0Vertex\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayCoordinates\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0locatedEvent\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxX\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0Y\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minX\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minY\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\nPathwayCoordinates\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxY\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0locatedEventÅ\0\0\0minXÅ\0\0\0maxXÅ\0\0\0minYÅ\0\0\0maxY\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0\0\0\0\0\0\0\0REQUIRED\0\0\0\0\0\0\0NOMANUALEDIT\0\0\0Å\0\0\0locatedEventÅ\0\0\0maxXÅ\0\0\0minXÅ\0\0\0minYÅ\0\0\0maxY\0\0\0	MANDATORY\0\0\0\0\0\0\0OPTIONAL\0\0\0category\0\0\0PathwayCoordinates\0\0\0Å\0\0\0ancestor_count\0\0\0Å\0\0\0DatabaseObject\0\0\0ancestor\0\0\0\0\0\0\0)\0\0\0stableIdentifier\0\0\0\0è\0\0\0\n__is_ghost\0\0\0\0ó\0\0\0_displayName\0\0\0\0Æ\0\0\0modified\0\0\0\0H\0\0\0_class\0\0\0\0\0\0Å\0\0\0_AttributeValueBeforeChange\0\0\0allowed\n_InstanceBeforeChange\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0attributeValuesBeforeChange\0\0\0\0∂\0\0\0created\0\0\0\0T\0\0\0\n_timestamp\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_InstanceBeforeChange\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0changedInstanceDB_ID\0\0\0\0Y\0\0\0DB_ID\0\0\0\0h\0\0\0_Protege_id\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n_InstanceBeforeChange\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0instanceEdit\0\0\0	attribute\0\0\0\nDatabaseObject\0\0\0ancestor_order\0\0\0Å\0\0\0changedInstanceDB_IDÅ\0\0\0instanceEditÅ\0\0\0attributeValuesBeforeChange\0\0\0\rown_attribute\0\0\0Å\0\0\0DatabaseObject\0\0\0parent\0\0\0\0\0\0\0\0\0\0any\0\0\0\0\0\0\0all\0\0\0check\0\0\0_InstanceBeforeChange\0\0\0classes\0\0π; Wed Feb 05 17:58:55 PST 2014\n; \n;+ (version \"3.4.7\")\n;+ (build \"Build 620\")\n\n([BROWSER_SLOT_NAMES] of  Property_List\n\n	(properties\n		[Reactome_109607_30201_ProjectKB_Class105]\n		[Reactome_109607_30201_ProjectKB_Class106]\n		[Reactome_109607_30201_ProjectKB_Class107]\n		[Reactome_109607_30201_ProjectKB_Class108]\n		[Reactome_109607_30201_ProjectKB_Class109]))\n\n([CLSES_TAB] of  Widget\n\n	(label \"Classes\")\n	(property_list [ProjectKB_4766_00016])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ClsesTab\"))\n\n([FORMS_TAB] of  Widget\n\n	(label \"Forms\")\n	(property_list [ProjectKB_4766_00045])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormsTab\"))\n\n([gk_central_20060615_ProjectKB_Instance_5] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"attributeCategory\")\n	(property_list [gk_central_20060615_ProjectKB_Instance_6])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ComboBoxWidget\")\n	(width 250)\n	(x 350)\n	(y 350))\n\n([gk_central_20060615_ProjectKB_Instance_6] of  Property_List\n)\n\n([GKB_core_ProjectKB_00103] of  String\n\n	(name \"instances_file_name\")\n	(string_value \"Reactome_109607_30201.pins\"))\n\n([GKB_core_ProjectKB_00203] of  Widget\n\n	(height 0)\n	(is_hidden FALSE)\n	(name \"__is_ghost\")\n	(property_list [GKB_core_ProjectKB_00303])\n	(width 0)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_00254] of  Widget\n\n	(height 435)\n	(is_hidden FALSE)\n	(name \":EXTENDED-SLOT\")\n	(property_list [GKB_core_ProjectKB_00484])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 758)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_00303] of  Property_List\n)\n\n([GKB_core_ProjectKB_00304] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"slotDisplayName\")\n	(property_list [GKB_core_ProjectKB_00305])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 0)\n	(y 330))\n\n([GKB_core_ProjectKB_00305] of  Property_List\n)\n\n([GKB_core_ProjectKB_00484] of  Property_List\n\n	(name \"slot widget properties\")\n	(properties\n		[GKB_core_ProjectKB_00485]\n		[GKB_core_ProjectKB_00486]\n		[GKB_core_ProjectKB_00487]\n		[GKB_core_ProjectKB_00488]\n		[GKB_core_ProjectKB_00489]\n		[GKB_core_ProjectKB_00495]\n		[GKB_core_ProjectKB_00496]\n		[GKB_core_ProjectKB_00497]\n		[GKB_core_ProjectKB_00503]\n		[GKB_core_ProjectKB_00504]\n		[GKB_core_ProjectKB_00505]\n		[GKB_core_ProjectKB_00506]\n		[GKB_core_ProjectKB_00507]\n		[GKB_core_ProjectKB_00508]\n		[GKB_core_ProjectKB_00509]\n		[GKB_core_ProjectKB_00510]\n		[GKB_core_ProjectKB_00801]\n		[GKB_core_ProjectKB_00920]\n		[GKB_core_ProjectKB_00304]\n		[PROJECT_3770_ProjectKB_Instance_59]\n		[gk_central_20060615_ProjectKB_Instance_5]))\n\n([GKB_core_ProjectKB_00485] of  Widget\n\n	(height 60)\n	(label \"Cardinality\")\n	(name \":SLOT-MINIMUM-CARDINALITY\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_60])\n	(widget_class_name \"edu.stanford.smi.protege.widget.MinimumCardinalityWidget\")\n	(width 200)\n	(x 200)\n	(y 120))\n\n([GKB_core_ProjectKB_00486] of  Widget\n\n	(height 35)\n	(name \":SLOT-MAXIMUM-CARDINALITY\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_61])\n	(widget_class_name \"edu.stanford.smi.protege.widget.MaximumCardinalityWidget\")\n	(width 200)\n	(x 200)\n	(y 175))\n\n([GKB_core_ProjectKB_00487] of  Widget\n\n	(name \":SLOT-CONSTRAINTS\"))\n\n([GKB_core_ProjectKB_00488] of  Widget\n\n	(name \":DIRECT-TYPE\"))\n\n([GKB_core_ProjectKB_00489] of  Widget\n\n	(height 120)\n	(label \"Template Values\")\n	(name \":SLOT-VALUES\")\n	(property_list [GKB_core_ProjectKB_00490])\n	(widget_class_name \"edu.stanford.smi.protege.widget.SlotValuesWidget\")\n	(width 200)\n	(x 400)\n	(y 0))\n\n([GKB_core_ProjectKB_00490] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_00491]\n		[GKB_core_ProjectKB_00492]\n		[GKB_core_ProjectKB_00493]\n		[GKB_core_ProjectKB_00494]))\n\n([GKB_core_ProjectKB_00491] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View the selected value\"))\n\n([GKB_core_ProjectKB_00492] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create a new value\"))\n\n([GKB_core_ProjectKB_00493] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_00494] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_00495] of  Widget\n\n	(name \":DIRECT-SUPERSLOTS\"))\n\n([GKB_core_ProjectKB_00496] of  Widget\n\n	(name \":DIRECT-SUBSLOTS\"))\n\n([GKB_core_ProjectKB_00497] of  Widget\n\n	(height 150)\n	(label \"Default\")\n	(name \":SLOT-DEFAULTS\")\n	(property_list [GKB_core_ProjectKB_00498])\n	(widget_class_name \"edu.stanford.smi.protege.widget.DefaultValuesWidget\")\n	(width 200)\n	(x 400)\n	(y 120))\n\n([GKB_core_ProjectKB_00498] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_00499]\n		[GKB_core_ProjectKB_00500]\n		[GKB_core_ProjectKB_00501]\n		[GKB_core_ProjectKB_00502]))\n\n([GKB_core_ProjectKB_00499] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View the selected value\"))\n\n([GKB_core_ProjectKB_00500] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create a new value\"))\n\n([GKB_core_ProjectKB_00501] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_00502] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_00503] of  Widget\n\n	(height 120)\n	(label \"Documentation\")\n	(name \":DOCUMENTATION\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_54])\n	(widget_class_name \"edu.stanford.smi.protege.widget.DocumentationWidget\")\n	(width 200)\n	(x 200)\n	(y 0))\n\n([GKB_core_ProjectKB_00504] of  Widget\n\n	(height 60)\n	(label \"Maximum\")\n	(name \":SLOT-NUMERIC-MAXIMUM\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_55])\n	(widget_class_name \"edu.stanford.smi.protege.widget.NumericMaximumWidget\")\n	(width 100)\n	(x 100)\n	(y 210))\n\n([GKB_core_ProjectKB_00505] of  Widget\n\n	(height 60)\n	(label \"Minimum\")\n	(name \":SLOT-NUMERIC-MINIMUM\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_56])\n	(widget_class_name \"edu.stanford.smi.protege.widget.NumericMinimumWidget\")\n	(width 100)\n	(x 0)\n	(y 210))\n\n([GKB_core_ProjectKB_00506] of  Widget\n\n	(name \":ASSOCIATED-FACET\"))\n\n([GKB_core_ProjectKB_00507] of  Widget\n\n	(height 60)\n	(label \"Name\")\n	(name \":NAME\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_57])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceNameWidget\")\n	(width 200)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_00508] of  Widget\n\n	(height 60)\n	(label \"Inverse Slot\")\n	(name \":SLOT-INVERSE\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_62])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InverseSlotWidget\")\n	(width 200)\n	(x 200)\n	(y 210))\n\n([GKB_core_ProjectKB_00509] of  Widget\n\n	(height 150)\n	(label \"Value Type\")\n	(name \":SLOT-VALUE-TYPE\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_63])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ValueTypeWidget\")\n	(width 200)\n	(x 0)\n	(y 60))\n\n([GKB_core_ProjectKB_00510] of  Property_List\n\n	(name \"layout properties\"))\n\n([GKB_core_ProjectKB_00801] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"identityDefinedBy\")\n	(property_list [GKB_core_ProjectKB_00901])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ComboBoxWidget\")\n	(width 150)\n	(x 0)\n	(y 270))\n\n([GKB_core_ProjectKB_00901] of  Property_List\n)\n\n([GKB_core_ProjectKB_00920] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"databaseColumnSpecification\")\n	(property_list [GKB_core_ProjectKB_00921])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 350)\n	(y 270))\n\n([GKB_core_ProjectKB_00921] of  Property_List\n)\n\n([GKB_core_ProjectKB_00989] of  Widget\n\n	(height 990)\n	(is_hidden FALSE)\n	(name \"Summation\")\n	(property_list [GKB_core_ProjectKB_00990])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 613)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_00990] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_00991]\n		[GKB_core_ProjectKB_00993]\n		[GKB_core_ProjectKB_01001]\n		[GKB_core_ProjectKB_01009]\n		[GKB_core_ProjectKB_01017]\n		[GKB_core_ProjectKB_01027]\n		[GKB_core_ProjectKB_01137]\n		[GKB_core_ProjectKB_01235]\n		[GKB_core_ProjectKB_01243]\n		[GKB_core_ProjectKB_01251]\n		[GKB_core_ProjectKB_01583]\n		[GKB_core_ProjectKB_01660]\n		[GKB_core_ProjectKB_06062]))\n\n([GKB_core_ProjectKB_00991] of  Widget\n\n	(height 170)\n	(is_hidden FALSE)\n	(name \"text\")\n	(property_list [GKB_core_ProjectKB_00992])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextAreaWidget\")\n	(width 510)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_00992] of  Property_List\n)\n\n([GKB_core_ProjectKB_00993] of  Widget\n\n	(height 120)\n	(is_hidden FALSE)\n	(name \"literatureReference\")\n	(property_list [GKB_core_ProjectKB_00994])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceListWidget\")\n	(width 510)\n	(x 0)\n	(y 170))\n\n([GKB_core_ProjectKB_00994] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_00995]\n		[GKB_core_ProjectKB_00996]\n		[GKB_core_ProjectKB_00997]\n		[GKB_core_ProjectKB_00998]\n		[GKB_core_ProjectKB_00999]\n		[GKB_core_ProjectKB_01000]))\n\n([GKB_core_ProjectKB_00995] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Selected Instances\"))\n\n([GKB_core_ProjectKB_00996] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_00997] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_00998] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_00999] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01000] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Selected Instances\"))\n\n([GKB_core_ProjectKB_01001] of  Widget\n\n	(height 90)\n	(is_hidden FALSE)\n	(name \"editor\")\n	(property_list [GKB_core_ProjectKB_01002])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceListWidget\")\n	(width 250)\n	(x 0)\n	(y 480))\n\n([GKB_core_ProjectKB_01002] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01003]\n		[GKB_core_ProjectKB_01004]\n		[GKB_core_ProjectKB_01005]\n		[GKB_core_ProjectKB_01006]\n		[GKB_core_ProjectKB_01007]\n		[GKB_core_ProjectKB_01008]))\n\n([GKB_core_ProjectKB_01003] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Selected Instances\"))\n\n([GKB_core_ProjectKB_01004] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_01005] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_01006] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_01007] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01008] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Selected Instances\"))\n\n([GKB_core_ProjectKB_01009] of  Widget\n\n	(height 100)\n	(is_hidden FALSE)\n	(name \"reviewer\")\n	(property_list [GKB_core_ProjectKB_01010])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceListWidget\")\n	(width 250)\n	(x 260)\n	(y 470))\n\n([GKB_core_ProjectKB_01010] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01011]\n		[GKB_core_ProjectKB_01012]\n		[GKB_core_ProjectKB_01013]\n		[GKB_core_ProjectKB_01014]\n		[GKB_core_ProjectKB_01015]\n		[GKB_core_ProjectKB_01016]))\n\n([GKB_core_ProjectKB_01011] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Selected Instances\"))\n\n([GKB_core_ProjectKB_01012] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_01013] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_01014] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_01015] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01016] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Selected Instances\"))\n\n([GKB_core_ProjectKB_01017] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"dateSubmitted\")\n	(property_list [GKB_core_ProjectKB_01018])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 260)\n	(y 290))\n\n([GKB_core_ProjectKB_01018] of  Property_List\n)\n\n([GKB_core_ProjectKB_01027] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"dateRevised\")\n	(property_list [GKB_core_ProjectKB_01028])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 260)\n	(y 410))\n\n([GKB_core_ProjectKB_01028] of  Property_List\n)\n\n([GKB_core_ProjectKB_01137] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"dateAccepted\")\n	(property_list [GKB_core_ProjectKB_01138])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 260)\n	(y 350))\n\n([GKB_core_ProjectKB_01138] of  Property_List\n)\n\n([GKB_core_ProjectKB_01234] of  Widget\n\n	(height 634)\n	(is_hidden FALSE)\n	(name \"DatabaseObject\")\n	(property_list [GKB_core_ProjectKB_01435])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 774)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_01235] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"_Protege_id\")\n	(property_list [GKB_core_ProjectKB_01236])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 0)\n	(y 570))\n\n([GKB_core_ProjectKB_01236] of  Property_List\n)\n\n([GKB_core_ProjectKB_01243] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"_displayName\")\n	(property_list [GKB_core_ProjectKB_01244])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 0)\n	(y 630))\n\n([GKB_core_ProjectKB_01244] of  Property_List\n)\n\n([GKB_core_ProjectKB_01251] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"created\")\n	(property_list [GKB_core_ProjectKB_01252])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceFieldWidget\")\n	(width 250)\n	(x 0)\n	(y 690))\n\n([GKB_core_ProjectKB_01252] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01559]\n		[GKB_core_ProjectKB_01560]\n		[GKB_core_ProjectKB_01561]\n		[GKB_core_ProjectKB_01562]\n		[GKB_core_ProjectKB_01563]\n		[GKB_core_ProjectKB_01564]))\n\n([GKB_core_ProjectKB_01435] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01535]\n		[GKB_core_ProjectKB_01537]\n		[GKB_core_ProjectKB_01745]\n		[GKB_core_ProjectKB_01749]\n		[GKB_core_ProjectKB_01757]\n		[GKB_core_ProjectKB_00203]\n		[PROJECT_3770_ProjectKB_Instance_106]\n		[Reactome_75244_8601_ProjectKB_Instance_203]))\n\n([GKB_core_ProjectKB_01535] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"DB_ID\")\n	(property_list [GKB_core_ProjectKB_01536])\n	(widget_class_name \"edu.stanford.smi.protege.widget.IntegerFieldWidget\")\n	(width 125)\n	(x 125)\n	(y 120))\n\n([GKB_core_ProjectKB_01536] of  Property_List\n)\n\n([GKB_core_ProjectKB_01537] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"created\")\n	(property_list [GKB_core_ProjectKB_01538])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceFieldWidget\")\n	(width 250)\n	(x 250)\n	(y 0))\n\n([GKB_core_ProjectKB_01538] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01640]\n		[GKB_core_ProjectKB_01740]\n		[GKB_core_ProjectKB_01741]\n		[GKB_core_ProjectKB_01742]\n		[GKB_core_ProjectKB_01743]\n		[GKB_core_ProjectKB_01744]))\n\n([GKB_core_ProjectKB_01559] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Instance\"))\n\n([GKB_core_ProjectKB_01560] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_01561] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_01562] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_01563] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01564] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Instance\"))\n\n([GKB_core_ProjectKB_01583] of  Widget\n\n	(height 120)\n	(is_hidden FALSE)\n	(name \"modified\")\n	(property_list [GKB_core_ProjectKB_01584])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceListWidget\")\n	(width 250)\n	(x 0)\n	(y 750))\n\n([GKB_core_ProjectKB_01584] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01615]\n		[GKB_core_ProjectKB_01616]\n		[GKB_core_ProjectKB_01617]\n		[GKB_core_ProjectKB_01618]\n		[GKB_core_ProjectKB_01619]\n		[GKB_core_ProjectKB_01620]))\n\n([GKB_core_ProjectKB_01615] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Selected Instances\"))\n\n([GKB_core_ProjectKB_01616] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_01617] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_01618] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_01619] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01620] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Selected Instances\"))\n\n([GKB_core_ProjectKB_01640] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Instance\"))\n\n([GKB_core_ProjectKB_01660] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"DB_ID\")\n	(property_list [GKB_core_ProjectKB_01661])\n	(widget_class_name \"edu.stanford.smi.protege.widget.IntegerFieldWidget\")\n	(width 125)\n	(x 0)\n	(y 870))\n\n([GKB_core_ProjectKB_01661] of  Property_List\n)\n\n([GKB_core_ProjectKB_01740] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_01741] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_01742] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_01743] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01744] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Instance\"))\n\n([GKB_core_ProjectKB_01745] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"_Protege_id\")\n	(property_list [GKB_core_ProjectKB_01746])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 0)\n	(y 60))\n\n([GKB_core_ProjectKB_01746] of  Property_List\n)\n\n([GKB_core_ProjectKB_01749] of  Widget\n\n	(height 120)\n	(is_hidden FALSE)\n	(name \"modified\")\n	(property_list [GKB_core_ProjectKB_01750])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceListWidget\")\n	(width 250)\n	(x 250)\n	(y 60))\n\n([GKB_core_ProjectKB_01750] of  Property_List\n\n	(properties\n		[GKB_core_ProjectKB_01751]\n		[GKB_core_ProjectKB_01752]\n		[GKB_core_ProjectKB_01753]\n		[GKB_core_ProjectKB_01754]\n		[GKB_core_ProjectKB_01755]\n		[GKB_core_ProjectKB_01756]))\n\n([GKB_core_ProjectKB_01751] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Selected Instances\"))\n\n([GKB_core_ProjectKB_01752] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([GKB_core_ProjectKB_01753] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([GKB_core_ProjectKB_01754] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([GKB_core_ProjectKB_01755] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([GKB_core_ProjectKB_01756] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Selected Instances\"))\n\n([GKB_core_ProjectKB_01757] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"_displayName\")\n	(property_list [GKB_core_ProjectKB_01758])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 250)\n	(x 0)\n	(y 0))\n\n([GKB_core_ProjectKB_01758] of  Property_List\n)\n\n([GKB_core_ProjectKB_06062] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"__is_ghost\")\n	(property_list [GKB_core_ProjectKB_06063])\n	(widget_class_name \"edu.stanford.smi.protege.widget.CheckBoxWidget\")\n	(width 125)\n	(x 0)\n	(y 930))\n\n([GKB_core_ProjectKB_06063] of  Property_List\n)\n\n([GKB_test_ProjectKB_00050] of  Integer\n\n	(integer_value 373)\n	(name \"ClsesTab.left_right\"))\n\n([GKB_test_ProjectKB_00051] of  Integer\n\n	(integer_value 513)\n	(name \"ClsesTab.left.top_bottom\"))\n\n([GKB_test_ProjectKB_00052] of  Integer\n\n	(integer_value 247)\n	(name \"SlotsTab.left_right\"))\n\n([GKB_test_ProjectKB_00053] of  Integer\n\n	(integer_value 424)\n	(name \"SlotTab.left.top_bottom\"))\n\n([GKB_test_ProjectKB_00054] of  Integer\n\n	(integer_value 250)\n	(name \"FormsTab.left_right\"))\n\n([GKB_test_ProjectKB_00055] of  Integer\n\n	(integer_value 303)\n	(name \"InstancesTab.left_right\"))\n\n([GKB_test_ProjectKB_00056] of  Integer\n\n	(integer_value 312)\n	(name \"InstancesTab.right.left_right\"))\n\n([GKB_test_ProjectKB_00060] of  Map\n)\n\n([INSTANCE_ANNOTATION_FORM_WIDGET] of  Widget\n\n	(height 476)\n	(is_hidden FALSE)\n	(name \":INSTANCE-ANNOTATION\")\n	(property_list [KB_127379_Instance_33])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 603)\n	(x 0)\n	(y 0))\n\n([INSTANCES_TAB] of  Widget\n\n	(label \"Instances\")\n	(property_list [ProjectKB_4766_00048])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstancesTab\"))\n\n([KB_074785_Class0] of  Map\n)\n\n([KB_127379_Instance_0] of  Property_List\n\n	(name \"class widget properties\")\n	(properties\n		[KB_127379_Instance_1]\n		[KB_127379_Instance_2]\n		[KB_127379_Instance_3]\n		[KB_127379_Instance_4]\n		[KB_127379_Instance_5]\n		[KB_127379_Instance_6]\n		[KB_127379_Instance_7]\n		[KB_127379_Instance_8]\n		[KB_127379_Instance_9]\n		[KB_127379_Instance_10]))\n\n([KB_127379_Instance_1] of  Widget\n\n	(height 120)\n	(label \"Constraints\")\n	(name \":SLOT-CONSTRAINTS\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_45])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ConstraintsWidget\")\n	(width 200)\n	(x 400)\n	(y 0))\n\n([KB_127379_Instance_10] of  Property_List\n\n	(name \"layout properties\")\n	(properties [KB_127379_Instance_11]))\n\n([KB_127379_Instance_11] of  String\n\n	(name \"vertical_stretcher\")\n	(string_value \":DIRECT-TEMPLATE-SLOTS\"))\n\n([KB_127379_Instance_12] of  Property_List\n\n	(name \"slot widget properties\")\n	(properties\n		[KB_127379_Instance_13]\n		[KB_127379_Instance_14]\n		[KB_127379_Instance_15]\n		[KB_127379_Instance_16]\n		[KB_127379_Instance_17]\n		[KB_127379_Instance_18]\n		[KB_127379_Instance_19]\n		[KB_127379_Instance_20]\n		[KB_127379_Instance_21]\n		[KB_127379_Instance_22]\n		[KB_127379_Instance_23]\n		[KB_127379_Instance_24]\n		[KB_127379_Instance_25]\n		[KB_127379_Instance_26]\n		[KB_127379_Instance_27]\n		[KB_127379_Instance_28]))\n\n([KB_127379_Instance_13] of  Widget\n\n	(height 60)\n	(label \"Cardinality\")\n	(name \":SLOT-MINIMUM-CARDINALITY\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.MinimumCardinalityWidget\")\n	(width 200)\n	(x 200)\n	(y 120))\n\n([KB_127379_Instance_14] of  Widget\n\n	(height 35)\n	(name \":SLOT-MAXIMUM-CARDINALITY\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.MaximumCardinalityWidget\")\n	(width 200)\n	(x 200)\n	(y 180))\n\n([KB_127379_Instance_15] of  Widget\n\n	(name \":SLOT-CONSTRAINTS\"))\n\n([KB_127379_Instance_16] of  Widget\n\n	(name \":DIRECT-TYPE\"))\n\n([KB_127379_Instance_17] of  Widget\n\n	(height 95)\n	(label \"Domain\")\n	(name \":DIRECT-DOMAIN\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.DirectDomainWidget\")\n	(width 200)\n	(x 400)\n	(y 180))\n\n([KB_127379_Instance_18] of  Widget\n\n	(height 90)\n	(label \"Template Values\")\n	(name \":SLOT-VALUES\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.SlotValuesWidget\")\n	(width 200)\n	(x 400)\n	(y 0))\n\n([KB_127379_Instance_19] of  Widget\n\n	(name \":DIRECT-SUPERSLOTS\"))\n\n([KB_127379_Instance_2] of  Widget\n\n	(name \":DIRECT-INSTANCES\"))\n\n([KB_127379_Instance_20] of  Widget\n\n	(name \":DIRECT-SUBSLOTS\"))\n\n([KB_127379_Instance_21] of  Widget\n\n	(height 90)\n	(label \"Default\")\n	(name \":SLOT-DEFAULTS\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.DefaultValuesWidget\")\n	(width 200)\n	(x 400)\n	(y 90))\n\n([KB_127379_Instance_22] of  Widget\n\n	(height 120)\n	(label \"Documentation\")\n	(name \":DOCUMENTATION\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.DocumentationWidget\")\n	(width 200)\n	(x 200)\n	(y 0))\n\n([KB_127379_Instance_23] of  Widget\n\n	(height 60)\n	(label \"Maximum\")\n	(name \":SLOT-NUMERIC-MAXIMUM\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.NumericMaximumWidget\")\n	(width 100)\n	(x 100)\n	(y 215))\n\n([KB_127379_Instance_24] of  Widget\n\n	(height 60)\n	(label \"Minimum\")\n	(name \":SLOT-NUMERIC-MINIMUM\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.NumericMinimumWidget\")\n	(width 100)\n	(x 0)\n	(y 215))\n\n([KB_127379_Instance_25] of  Widget\n\n	(name \":ASSOCIATED-FACET\"))\n\n([KB_127379_Instance_26] of  Widget\n\n	(height 60)\n	(label \"Name\")\n	(name \":NAME\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceNameWidget\")\n	(width 200)\n	(x 0)\n	(y 0))\n\n([KB_127379_Instance_27] of  Widget\n\n	(height 60)\n	(label \"Inverse Slot\")\n	(name \":SLOT-INVERSE\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.InverseSlotWidget\")\n	(width 200)\n	(x 200)\n	(y 215))\n\n([KB_127379_Instance_28] of  Widget\n\n	(height 155)\n	(label \"Value Type\")\n	(name \":SLOT-VALUE-TYPE\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.ValueTypeWidget\")\n	(width 200)\n	(x 0)\n	(y 60))\n\n([KB_127379_Instance_29] of  Property_List\n\n	(name \"facet widget properties\")\n	(properties\n		[KB_127379_Instance_30]\n		[KB_127379_Instance_31]\n		[KB_127379_Instance_32]))\n\n([KB_127379_Instance_3] of  Widget\n\n	(name \":DIRECT-SUBCLASSES\"))\n\n([KB_127379_Instance_30] of  Widget\n\n	(height 60)\n	(label \"Name\")\n	(name \":NAME\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceNameWidget\")\n	(width 200)\n	(x 0)\n	(y 0))\n\n([KB_127379_Instance_31] of  Widget\n\n	(height 120)\n	(label \"Documentation\")\n	(name \":DOCUMENTATION\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.DocumentationWidget\")\n	(width 200)\n	(x 200)\n	(y 0))\n\n([KB_127379_Instance_32] of  Widget\n\n	(height 60)\n	(label \"Associated Slot\")\n	(name \":ASSOCIATED-SLOT\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceFieldWidget\")\n	(width 200)\n	(x 0)\n	(y 60))\n\n([KB_127379_Instance_33] of  Property_List\n\n	(properties\n		[KB_127379_Instance_34]\n		[KB_127379_Instance_35]\n		[KB_127379_Instance_36]\n		[KB_127379_Instance_37]))\n\n([KB_127379_Instance_34] of  Widget\n\n	(name \":ANNOTATED-INSTANCE\"))\n\n([KB_127379_Instance_35] of  Widget\n\n	(name \":CREATOR\"))\n\n([KB_127379_Instance_36] of  Widget\n\n	(name \":CREATION-TIMESTAMP\"))\n\n([KB_127379_Instance_37] of  Widget\n\n	(height 100)\n	(is_hidden FALSE)\n	(name \":ANNOTATION-TEXT\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.YellowStickyWidget\")\n	(width 200)\n	(x 0)\n	(y 0))\n\n([KB_127379_Instance_38] of  Property_List\n\n	(properties\n		[KB_127379_Instance_39]\n		[KB_127379_Instance_40]\n		[KB_127379_Instance_41]\n		[KB_127379_Instance_42]))\n\n([KB_127379_Instance_39] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(label \"Name\")\n	(name \":PAL-NAME\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextFieldWidget\")\n	(width 275)\n	(x 0)\n	(y 0))\n\n([KB_127379_Instance_4] of  Widget\n\n	(name \":DIRECT-SUPERCLASSES\"))\n\n([KB_127379_Instance_40] of  Widget\n\n	(height 180)\n	(is_hidden FALSE)\n	(label \"Range\")\n	(name \":PAL-RANGE\")\n	(widget_class_name \"edu.stanford.smi.protegex.widget.pal.constraint.PalRangeWidget\")\n	(width 250)\n	(x 275)\n	(y 180))\n\n([KB_127379_Instance_41] of  Widget\n\n	(height 180)\n	(is_hidden FALSE)\n	(label \"Description\")\n	(name \":PAL-DESCRIPTION\")\n	(widget_class_name \"edu.stanford.smi.protege.widget.TextAreaWidget\")\n	(width 250)\n	(x 275)\n	(y 0))\n\n([KB_127379_Instance_42] of  Widget\n\n	(height 300)\n	(is_hidden FALSE)\n	(label \"Statement\")\n	(name \":PAL-STATEMENT\")\n	(widget_class_name \"edu.stanford.smi.protegex.widget.pal.constraint.PalConstraintWidget\")\n	(width 275)\n	(x 0)\n	(y 60))\n\n([KB_127379_Instance_5] of  Widget\n\n	(height 120)\n	(label \"Documentation\")\n	(name \":DOCUMENTATION\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_48])\n	(widget_class_name \"edu.stanford.smi.protege.widget.DocumentationWidget\")\n	(width 200)\n	(x 200)\n	(y 0))\n\n([KB_127379_Instance_6] of  Widget\n\n	(height 60)\n	(label \"Name\")\n	(name \":NAME\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_49])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceNameWidget\")\n	(width 200)\n	(x 0)\n	(y 0))\n\n([KB_127379_Instance_7] of  Widget\n\n	(height 60)\n	(label \"Role\")\n	(name \":ROLE\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_53])\n	(widget_class_name \"edu.stanford.smi.protege.widget.RoleWidget\")\n	(width 200)\n	(x 0)\n	(y 60))\n\n([KB_127379_Instance_8] of  Widget\n\n	(name \":DIRECT-TYPE\"))\n\n([KB_127379_Instance_9] of  Widget\n\n	(height 150)\n	(label \"Template Slots\")\n	(name \":DIRECT-TEMPLATE-SLOTS\")\n	(property_list [Reactome_109607_19982_ProjectKB_Instance_50])\n	(widget_class_name \"edu.stanford.smi.protege.widget.TemplateSlotsWidget\")\n	(width 600)\n	(x 0)\n	(y 120))\n\n([PAL_FORM_WIDGET] of  Widget\n\n	(height 476)\n	(is_hidden FALSE)\n	(name \":PAL-CONSTRAINT\")\n	(property_list [KB_127379_Instance_38])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 603)\n	(x 0)\n	(y 0))\n\n([PROJECT] of  Project\n\n	(browser_slot_names [BROWSER_SLOT_NAMES])\n	(customized_instance_widgets\n		[GKB_core_ProjectKB_00254]\n		[INSTANCE_ANNOTATION_FORM_WIDGET]\n		[STANDARD_FACET_FORM_WIDGET]\n		[STANDARD_SLOT_FORM_WIDGET]\n		[GKB_core_ProjectKB_01234]\n		[PAL_FORM_WIDGET]\n		[STANDARD_CLASS_FORM_WIDGET]\n		[GKB_core_ProjectKB_00989])\n	(default_cls_metaclass \":STANDARD-CLASS\")\n	(default_facet_metaclass \":STANDARD-FACET\")\n	(default_instance_widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(default_slot_metaclass \":EXTENDED-SLOT\")\n	(next_frame_number 0)\n	(options [ProjectKB_4766_00003])\n	(property_map [GKB_test_ProjectKB_00060])\n	(sources [SOURCES])\n	(tabs\n		[CLSES_TAB]\n		[SLOTS_TAB]\n		[FORMS_TAB]\n		[INSTANCES_TAB]\n		[ProjectKB_4766_00012]\n		[ProjectKB_4766_00014]))\n\n([PROJECT_3770_ProjectKB_Instance_106] of  Property_List\n\n	(name \"layout properties\"))\n\n([PROJECT_3770_ProjectKB_Instance_59] of  Widget\n\n	(height 120)\n	(is_hidden FALSE)\n	(name \":DIRECT-DOMAIN\")\n	(property_list [PROJECT_3770_ProjectKB_Instance_60])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceListWidget\")\n	(width 250)\n	(x 0)\n	(y 390))\n\n([PROJECT_3770_ProjectKB_Instance_60] of  Property_List\n\n	(properties\n		[PROJECT_3770_ProjectKB_Instance_61]\n		[PROJECT_3770_ProjectKB_Instance_62]\n		[PROJECT_3770_ProjectKB_Instance_63]\n		[PROJECT_3770_ProjectKB_Instance_64]\n		[PROJECT_3770_ProjectKB_Instance_65]\n		[PROJECT_3770_ProjectKB_Instance_66]\n		[Reactome_109607_19982_ProjectKB_Instance_58]\n		[Reactome_109607_19982_ProjectKB_Instance_59]))\n\n([PROJECT_3770_ProjectKB_Instance_61] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Selected Instances\"))\n\n([PROJECT_3770_ProjectKB_Instance_62] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([PROJECT_3770_ProjectKB_Instance_63] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([PROJECT_3770_ProjectKB_Instance_64] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([PROJECT_3770_ProjectKB_Instance_65] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([PROJECT_3770_ProjectKB_Instance_66] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Selected Instances\"))\n\n([ProjectKB_4766_00000] of  String\n\n	(name \"classes_file_name\")\n	(string_value \"Reactome_109607_30201.pont\"))\n\n([ProjectKB_4766_00002] of  String\n\n	(name \"factory_class_name\")\n	(string_value \"edu.stanford.smi.protege.storage.clips.ClipsKnowledgeBaseFactory\"))\n\n([ProjectKB_4766_00003] of  Options\n\n	(confirm_on_remove FALSE)\n	(display_abstract_class_icon TRUE)\n	(display_hidden_classes TRUE)\n	(display_multi_parent_class_icon TRUE)\n	(is_readonly FALSE)\n	(update_modification_slots FALSE))\n\n([ProjectKB_4766_00012] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [ProjectKB_4766_00013])\n	(widget_class_name \"edu.stanford.smi.protege.widget.KAToolTab\"))\n\n([ProjectKB_4766_00013] of  Property_List\n)\n\n([ProjectKB_4766_00014] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [ProjectKB_4766_00015])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ClsesAndInstancesTab\"))\n\n([ProjectKB_4766_00015] of  Property_List\n)\n\n([ProjectKB_4766_00016] of  Property_List\n\n	(properties\n		[GKB_test_ProjectKB_00050]\n		[GKB_test_ProjectKB_00051]))\n\n([ProjectKB_4766_00033] of  Property_List\n\n	(properties\n		[GKB_test_ProjectKB_00052]\n		[GKB_test_ProjectKB_00053]))\n\n([ProjectKB_4766_00045] of  Property_List\n\n	(properties [GKB_test_ProjectKB_00054]))\n\n([ProjectKB_4766_00048] of  Property_List\n\n	(properties\n		[GKB_test_ProjectKB_00055]\n		[GKB_test_ProjectKB_00056]))\n\n([Reactome_109607_19982_ProjectKB_Instance_45] of  Property_List\n\n	(properties\n		[Reactome_109607_19982_ProjectKB_Instance_46]\n		[Reactome_109607_19982_ProjectKB_Instance_47]))\n\n([Reactome_109607_19982_ProjectKB_Instance_46] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-View References to Value\"))\n\n([Reactome_109607_19982_ProjectKB_Instance_47] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Instance\"))\n\n([Reactome_109607_19982_ProjectKB_Instance_48] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_49] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_50] of  Property_List\n\n	(properties\n		[Reactome_109607_19982_ProjectKB_Instance_51]\n		[Reactome_109607_19982_ProjectKB_Instance_52]))\n\n([Reactome_109607_19982_ProjectKB_Instance_51] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Move up\"))\n\n([Reactome_109607_19982_ProjectKB_Instance_52] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Move down\"))\n\n([Reactome_109607_19982_ProjectKB_Instance_53] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_54] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_55] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_56] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_57] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_58] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-View References to Value\"))\n\n([Reactome_109607_19982_ProjectKB_Instance_59] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Instance\"))\n\n([Reactome_109607_19982_ProjectKB_Instance_60] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_61] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_62] of  Property_List\n)\n\n([Reactome_109607_19982_ProjectKB_Instance_63] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class1] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class2])\n	(widget_class_name \"edu.stanford.smi.protegex.queries_tab.QueriesTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class10] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class105] of  String\n\n	(name \":INSTANCE-ANNOTATION\")\n	(string_value \"%3AANNOTATION-TEXT\"))\n\n([Reactome_109607_30201_ProjectKB_Class106] of  String\n\n	(name \":META-CLASS\")\n	(string_value \"%3ANAME\"))\n\n([Reactome_109607_30201_ProjectKB_Class107] of  String\n\n	(name \"DatabaseObject\")\n	(string_value \"_displayName\"))\n\n([Reactome_109607_30201_ProjectKB_Class108] of  String\n\n	(name \":PAL-CONSTRAINT\")\n	(string_value \"%3APAL-NAME\"))\n\n([Reactome_109607_30201_ProjectKB_Class109] of  String\n\n	(name \"Summation\")\n	(string_value \"title\"))\n\n([Reactome_109607_30201_ProjectKB_Class11] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class12])\n	(widget_class_name \"uk.ac.man.cs.mig.coode.debugger.test.DebuggerTestTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class12] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class13] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class14])\n	(widget_class_name \"ezpal.EZPalTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class14] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class15] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class16])\n	(widget_class_name \"edu.stanford.smi.protegex.server_changes.prompt.UsersTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class16] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class17] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class18])\n	(widget_class_name \"ca.uvic.csr.shrimp.jambalaya.JambalayaTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class18] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class19] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class20])\n	(widget_class_name \"edu.stanford.smi.protege.widget.instance_tree.InstanceTreeTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class2] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class20] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class21] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class22])\n	(widget_class_name \"edu.stanford.smi.protegex.widget.pal.PalConstraintsTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class22] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class23] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class24])\n	(widget_class_name \"edu.stanford.smi.RemoteKBTab.WordNetTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class24] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class25] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class26])\n	(widget_class_name \"uk.ac.man.ac.mig.coode.individuals.ui.OWLDLIndividualsTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class26] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class27] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class28])\n	(widget_class_name \"org.protege.owl.mm.protege3.MappingMasterTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class28] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class29] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class30])\n	(widget_class_name \"edu.stanford.smi.protegex.evaluation.MetaAnalysis\"))\n\n([Reactome_109607_30201_ProjectKB_Class3] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class4])\n	(widget_class_name \"se.liu.ida.JessTab.JessTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class30] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class31] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class32])\n	(widget_class_name \"uk.ac.man.cs.mig.coode.owlviz.ui.OWLVizTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class32] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class33] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class34])\n	(widget_class_name \"edu.stanford.smi.protege.query.LuceneQueryPlugin\"))\n\n([Reactome_109607_30201_ProjectKB_Class34] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class35] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class36])\n	(widget_class_name \"edu.stanford.smi.protegex.owl.ui.cls.OWLClassesTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class36] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class37] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class38])\n	(widget_class_name \"edu.stanford.smi.RemoteKBTab.UMLSTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class38] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class39] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class40])\n	(widget_class_name \"edu.stanford.smi.protegex.owl.ui.individuals.OWLIndividualsTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class4] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class40] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class41] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class42])\n	(widget_class_name \"edu.stanford.smi.protegex.changeanalysis.ChangeAnalysisTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class42] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class43] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class44])\n	(widget_class_name \"TGViztab.TGVizTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class44] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class45] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class46])\n	(widget_class_name \"org.algernon.kb.okbc.protege.plugins.AlgernonTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class46] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class47] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class48])\n	(widget_class_name \"edu.stanford.smi.protegex.changes.changesKBViewTab.ChangesKBViewTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class48] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class49] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class50])\n	(widget_class_name \"edu.stanford.smi.protegex.xml.tab.XMLTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class5] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class6])\n	(widget_class_name \"edu.stanford.smi.protegex.owl.ui.metadatatab.OWLMetadataTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class50] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class51] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class52])\n	(widget_class_name \"edu.stanford.smi.protegex.widget.pal.PalQueriesTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class52] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class53] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class54])\n	(widget_class_name \"edu.stanford.smi.protegex.chatPlugin.ChatTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class54] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class55] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class56])\n	(widget_class_name \"org.protege.owl.axiome.ui.AxiomeTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class56] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class57] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class58])\n	(widget_class_name \"edu.stanford.smi.protege.keywordsearch.StringSearch\"))\n\n([Reactome_109607_30201_ProjectKB_Class58] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class59] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class60])\n	(widget_class_name \"edu.stanford.smi.protegex.owl.ui.widget.OWLFormsTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class6] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class60] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class61] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class62])\n	(widget_class_name \"script.ProtegeScriptTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class62] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class63] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class64])\n	(widget_class_name \"edu.stanford.smi.protegex.changes.ChangesTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class64] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class65] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class66])\n	(widget_class_name \"edu.stanford.smi.protegex.owl.swrl.ui.tab.SWRLTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class66] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class67] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class68])\n	(widget_class_name \"edu.stanford.smi.protegex.fctab.FacetConstraintsTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class68] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class69] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class70])\n	(widget_class_name \"edu.stanford.smi.protege.widget.instance_tree.KnowledgeTreeTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class7] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class8])\n	(widget_class_name \"edu.stanford.smi.protegex.owl.ui.properties.OWLPropertiesTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class70] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class71] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class72])\n	(widget_class_name \"edu.stanford.smi.protegex.datamaster.DataMasterTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class72] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class73] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class74])\n	(widget_class_name \"dfki.protege.ontoviz_tab.OntovizTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class74] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class75] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class76])\n	(widget_class_name \"edu.stanford.smi.protegex.changes.stats.ChangeStatisticsTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class76] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class77] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class78])\n	(widget_class_name \"edu.stanford.smi.protegex.prompt.PromptTab\"))\n\n([Reactome_109607_30201_ProjectKB_Class78] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class8] of  Property_List\n)\n\n([Reactome_109607_30201_ProjectKB_Class9] of  Widget\n\n	(is_hidden TRUE)\n	(property_list [Reactome_109607_30201_ProjectKB_Class10])\n	(widget_class_name \"edu.stanford.smi.protege.widget.ProtegePropertiesTab\"))\n\n([Reactome_75244_8601_ProjectKB_Instance_203] of  Widget\n\n	(height 60)\n	(is_hidden FALSE)\n	(name \"stableIdentifier\")\n	(property_list [Reactome_75244_8601_ProjectKB_Instance_204])\n	(widget_class_name \"edu.stanford.smi.protege.widget.InstanceFieldWidget\")\n	(width 200)\n	(x 0)\n	(y 180))\n\n([Reactome_75244_8601_ProjectKB_Instance_204] of  Property_List\n\n	(properties\n		[Reactome_75244_8601_ProjectKB_Instance_205]\n		[Reactome_75244_8601_ProjectKB_Instance_206]\n		[Reactome_75244_8601_ProjectKB_Instance_207]\n		[Reactome_75244_8601_ProjectKB_Instance_208]\n		[Reactome_75244_8601_ProjectKB_Instance_209]\n		[Reactome_75244_8601_ProjectKB_Instance_210]))\n\n([Reactome_75244_8601_ProjectKB_Instance_205] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-View Instance\"))\n\n([Reactome_75244_8601_ProjectKB_Instance_206] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-References\"))\n\n([Reactome_75244_8601_ProjectKB_Instance_207] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Create Instance\"))\n\n([Reactome_75244_8601_ProjectKB_Instance_208] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Add\"))\n\n([Reactome_75244_8601_ProjectKB_Instance_209] of  Boolean\n\n	(boolean_value TRUE)\n	(name \"ButtonDisplayed-Remove\"))\n\n([Reactome_75244_8601_ProjectKB_Instance_210] of  Boolean\n\n	(boolean_value FALSE)\n	(name \"ButtonDisplayed-Delete Instance\"))\n\n([SLOTS_TAB] of  Widget\n\n	(label \"Slots\")\n	(property_list [ProjectKB_4766_00033])\n	(widget_class_name \"edu.stanford.smi.protege.widget.SlotsTab\"))\n\n([SOURCES] of  Property_List\n\n	(properties\n		[ProjectKB_4766_00000]\n		[ProjectKB_4766_00002]\n		[GKB_core_ProjectKB_00103]))\n\n([STANDARD_CLASS_FORM_WIDGET] of  Widget\n\n	(height 412)\n	(name \":STANDARD-CLASS\")\n	(property_list [KB_127379_Instance_0])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 838)\n	(x 0)\n	(y 0))\n\n([STANDARD_FACET_FORM_WIDGET] of  Widget\n\n	(name \":STANDARD-FACET\")\n	(property_list [KB_127379_Instance_29])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\"))\n\n([STANDARD_SLOT_FORM_WIDGET] of  Widget\n\n	(height 435)\n	(name \":STANDARD-SLOT\")\n	(property_list [KB_127379_Instance_12])\n	(widget_class_name \"edu.stanford.smi.protege.widget.FormWidget\")\n	(width 758)\n	(x 0)\n	(y 0))\n\0\0\0pprj_file_content\0\0\0debug\0áN; Wed Feb 05 17:58:55 PST 2014\n; \n;+ (version \"3.4.7\")\n;+ (build \"Build 620\")\n\n\n(defclass %3ACLIPS_TOP_LEVEL_SLOT_CLASS \"Fake class to save top-level slot information\"\n	(is-a USER)\n	(role abstract)\n	(multislot deletedInstanceDB_ID\n;+		(comment \"Contains the DB_IDs of the instances that have been deleted\")\n		(type INTEGER)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot containedInPathway\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n		(create-accessor read-write))\n	(single-slot modification\n		(type INSTANCE)\n;+		(allowed-classes ReferenceGroup PhysicalEntity)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot accession\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot locatedEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot isSequenceChanged\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot normalEntity\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(inverse-slot orthologousEvent)\n		(create-accessor read-write))\n	(single-slot volume\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot taxon\n		(type INSTANCE)\n;+		(allowed-classes Species)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot removingActivity\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot keyword\n;+		(comment \"KW lines from SPTREMBL records\")\n		(type STRING)\n		(create-accessor read-write))\n	(multislot hasModifiedResidue\n		(type INSTANCE)\n;+		(allowed-classes AbstractModifiedResidue)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot doi\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot referenceTranscript\n		(type INSTANCE)\n;+		(allowed-classes ReferenceRNASequence)\n		(create-accessor read-write))\n	(single-slot firstname\n		(type STRING)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot structuralVariant\n		(type INSTANCE)\n;+		(allowed-classes SequenceOntology)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot isChimeric\n;+		(comment \"this slot acts as a flag for \\\"artificial multi-species\\\" instances (typically used as source of inference for single-species instances where these don\'t have direct evidence)\")\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n		(default FALSE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species Taxon)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot year\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot pubMedIdentifier\n;+		(comment \"PubMed ID\")\n		(type INTEGER)\n		(create-accessor read-write))\n	(single-slot locationContext\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot targetX\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot targetY\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot referenceSequence\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot inferredTo\n;+		(comment \"reverse attribute of inferredFrom\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(single-slot url\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot hasComponent\n		(type INSTANCE)\n;+		(allowed-classes %3ATHING)\n		(create-accessor read-write))\n	(single-slot dateRevised\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot otherIdentifier\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot endCoordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot chapterTitle\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot maxCount\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot sourceVertex\n		(type INSTANCE)\n;+		(allowed-classes Vertex)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot project\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot address\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot functionalStatusType\n		(type INSTANCE)\n;+		(allowed-classes FunctionalStatusType)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot hasEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot requiredInputComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n		(create-accessor read-write))\n	(single-slot startCoordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot geneName\n;+		(comment \"GN lines from sequence record.\")\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot journal\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot alteredAminoAcidFragment\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot negativelyRegulate\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot regulate\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n		(create-accessor read-write))\n	(single-slot wrappedLabel\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(single-slot referenceGroup\n		(type INSTANCE)\n;+		(allowed-classes ReferenceGroup)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot width\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot hasPart\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot uniformResourceLocator\n		(type STRING)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot reverseReaction\n		(type INSTANCE)\n;+		(allowed-classes Reaction)\n;+		(cardinality 0 1)\n;+		(inverse-slot reverseReaction)\n		(create-accessor read-write))\n	(single-slot templateEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot concurrentEvents\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(single-slot y\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot variantIdentifier\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot x\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot publisher\n		(type INSTANCE)\n;+		(allowed-classes Affiliation)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot reviewer\n		(type INSTANCE)\n;+		(allowed-classes Person)\n		(create-accessor read-write))\n	(multislot isoformParent\n		(type INSTANCE)\n;+		(allowed-classes ReferenceGeneProduct)\n		(create-accessor read-write))\n	(multislot focusEntity\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot attributeValuesBeforeChange\n		(type INSTANCE)\n;+		(allowed-classes _AttributeValueBeforeChange)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(multislot functionalStatus\n		(type INSTANCE)\n;+		(allowed-classes FunctionalStatus)\n		(create-accessor read-write))\n	(multislot databaseIdentifier\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n		(create-accessor read-write))\n	(single-slot pointCoordinates\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot _applyToAllEditedInstances\n;+		(comment \"Set this attribute vaue to TRUE (by checking it) if you want to attch this InstenceEdit to all edited instances in your project.\")\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot editor\n		(type INSTANCE)\n;+		(allowed-classes Person)\n		(create-accessor read-write))\n	(single-slot releaseStatus\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot psiMod\n		(type INSTANCE)\n;+		(allowed-classes PsiMod)\n		(create-accessor read-write))\n	(single-slot minUnitCount\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot ecNumber\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot dateAccepted\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot hasMember\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot text\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot superTaxon\n		(type INSTANCE)\n;+		(allowed-classes Taxon)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot height\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot searchableTerm\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot maxUnitCount\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot activity\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot author\n		(type INSTANCE)\n;+		(allowed-classes Person)\n		(create-accessor read-write))\n	(multislot affiliation\n		(type INSTANCE)\n;+		(allowed-classes Affiliation)\n		(create-accessor read-write))\n	(multislot secondReferenceSequence\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n		(create-accessor read-write))\n	(single-slot maxX\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot maxY\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot providerCount\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot sequenceLength\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot Reactome_109607_14914_Slot_1\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot includedLocation\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n		(create-accessor read-write))\n	(single-slot regulator\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot storedATXML\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot _doRelease\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot databaseColumnSpecification\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(associated-facet _databaseColumnSpecification)\n		(create-accessor read-write))\n	(multislot referenceGroupCount\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot formula\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot isCanonical\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot releaseNumber\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot shortName\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot eventLocationContext\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot entityFunctionalStatus\n		(type INSTANCE)\n;+		(allowed-classes EntityFunctionalStatus)\n		(create-accessor read-write))\n	(single-slot ISBN\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot secondaryIdentifier\n;+		(comment \"This is for secondary/other identifiers WITHIN THE SAME DATABASE, e.g. secondary SPTREMBL accessions and SwissProt id. Use crossReference if the the identifier is from another db, e.g. for adding a RefSeq identifier to SPTREMBL SequenceDatabaseIdentifier.\")\n		(type STRING)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(create-accessor read-write))\n	(multislot creatingActivity\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot disease\n		(type INSTANCE)\n;+		(allowed-classes Disease)\n		(create-accessor read-write))\n	(single-slot classDisplayName\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot identityDefinedBy\n		(type SYMBOL)\n		(allowed-values ALL ANY none)\n;+		(cardinality 0 1)\n;+		(associated-facet _identityDefinedBy)\n		(create-accessor read-write))\n	(single-slot _displayName\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot identify\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot accessUrl\n;+		(comment \"Url for retrieving a record\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot previousValue\n		(type STRING)\n		(create-accessor read-write))\n	(multislot synonym\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot minCount\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot reason\n;+		(comment \"Reason for an action, drawn from a limited controlled vocabulary of terms\")\n		(type INSTANCE)\n;+		(allowed-classes ControlledVocabulary)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot edgeType\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot residue\n;+		(comment \"This is to hold the residue (like serine or tyrosine) which is modified. The value is instance of class CocnreteSimpleEntityk, i.e. an instance representig serine. I know, this not exactly correct since free amino acid (nucleotide) is different from the one in the chain but it\'s still pretty good I think.\")\n		(type INSTANCE)\n;+		(allowed-classes ReferenceMolecule)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot secondCoordinate\n		(type INTEGER)\n		(create-accessor read-write))\n	(single-slot changedInstance\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType RegulationType Regulation DatabaseIdentifier GO_BiologicalProcess FrontPage CatalystActivity Taxon AbstractModifiedResidue PhysicalEntity InstanceEdit Figure Person Summation LiteratureReference Affiliation ConcurrentEventSet Event ReferenceDatabase)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot initial\n		(type STRING)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot description\n;+		(comment \"DE lines from sequence record\")\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot pathwayDiagram\n		(type INSTANCE)\n;+		(allowed-classes PathwayDiagram)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot frontPageItem\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(single-slot hasFunctionalVariants\n		(type SYMBOL)\n;+		(allowed-parents PhysicalEntity)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot pages\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot hasSpecialisedForm\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot referenceEntity\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot startPositionInReferenceSequence\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot identifierVersion\n;+		(comment \"Version number for stable identifier\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot vertex\n		(type INSTANCE)\n;+		(allowed-classes Vertex)\n		(create-accessor read-write))\n	(multislot chain\n		(type STRING)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(multislot termProvider\n		(type INSTANCE)\n;+		(allowed-classes Event PhysicalEntity ReferenceEntity GO_MolecularFunction GO_BiologicalProcess DatabaseIdentifier)\n		(create-accessor read-write))\n	(single-slot _chainChangeLog\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot changedInstanceDB_ID\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot positivelyRegulate\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot stableIdentifier\n;+		(comment \"Contains stable ID and version number\")\n		(type INSTANCE)\n;+		(allowed-classes StableIdentifier)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot note\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot attributeCategory\n		(type SYMBOL)\n		(allowed-values MANDATORY REQUIRED OPTIONAL NOMANUALEDIT)\n;+		(cardinality 0 1)\n;+		(associated-facet _attributeCategory)\n		(create-accessor read-write))\n	(single-slot DB_ID\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot minY\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot instanceEdit\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot minX\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot componentOf\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot curatorComment\n;+		(comment \"A free-text comment that the curator may optionally provide to add extra explanation\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot dateSubmitted\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot changedAttribute\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot title\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot slotDisplayName\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(associated-facet _slotDisplayName)\n		(create-accessor read-write))\n	(single-slot containedInEntityVertex\n		(type INSTANCE)\n;+		(allowed-classes EntityVertex)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot vertexCount\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot hasInstance\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot normalReaction\n		(type INSTANCE)\n;+		(allowed-classes ReactionlikeEvent)\n		(create-accessor read-write))\n	(multislot chapterAuthors\n		(type INSTANCE)\n;+		(allowed-classes Person)\n		(create-accessor read-write))\n	(single-slot _Protege_id\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot output\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(multislot equivalentTo\n		(type INSTANCE)\n;+		(allowed-classes InterChainCrosslinkedResidue)\n		(create-accessor read-write))\n	(multislot referenceGene\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDNASequence)\n		(create-accessor read-write))\n	(multislot repeatedUnit\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot cellType\n		(type INSTANCE)\n;+		(allowed-classes CellType)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot dateTime\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n		(create-accessor read-write))\n	(single-slot checksum\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot endPositionInReferenceSequence\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot hasCandidate\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(multislot activeUnit\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n		(create-accessor read-write))\n	(single-slot regulatedEntity\n		(type INSTANCE)\n;+		(allowed-classes Event CatalystActivity)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot __is_ghost\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(create-accessor read-write))\n	(single-slot regulationType\n		(type INSTANCE)\n;+		(allowed-classes RegulationType)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot relatedSpecies\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot surname\n		(type STRING)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot representedPathway\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(single-slot fingerPrintString\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot representedInstance\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(multislot comment\n;+		(comment \"CC lines from sequence record.\")\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot physicalEntity\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot atomicConnectivity\n;+		(comment \"Contents of the mol file.\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot targetVertex\n		(type INSTANCE)\n;+		(allowed-classes Vertex)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot eMailAddress\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot _doNotRelease\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot replacementInstances\n;+		(comment \"The instance(s) (if any) that replace the deleted instance.  The allowed instance classes have been restricted to those for which stable IDs get generated.  If the classes for which stable IDs are generated is expanded, these need to be changed correspondingly.\")\n		(type INSTANCE)\n;+		(allowed-classes Regulation Event PhysicalEntity)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot entityOnOtherCell\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(single-slot sourceX\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot sourceY\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass %3AEXTENDED-CLASS\n	(is-a %3ASTANDARD-CLASS)\n	(role concrete)\n	(single-slot classDisplayName\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass %3AEXTENDED-SLOT\n	(is-a %3ASTANDARD-SLOT)\n	(role concrete)\n	(single-slot attributeCategory\n		(type SYMBOL)\n		(allowed-values MANDATORY REQUIRED OPTIONAL NOMANUALEDIT)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot slotDisplayName\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot databaseColumnSpecification\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot identityDefinedBy\n		(type SYMBOL)\n		(allowed-values ALL ANY none)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass DatabaseObject\n	(is-a USER)\n	(role abstract)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot _Protege_id\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _databaseColumnSpecification \"VARCHAR(255)\")\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot DB_ID\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot __is_ghost\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n;+		(user-facet _databaseColumnSpecification \"ENUM(\'TRUE\')\")\n		(create-accessor read-write))\n	(single-slot stableIdentifier\n;+		(comment \"Contains stable ID and version number\")\n		(type INSTANCE)\n;+		(allowed-classes StableIdentifier)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot _displayName\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write)))\n\n(defclass Event \"Presents some sort of event, be it chemical reaction or some complex multistep process.\\nThe identity of direct instances of this class is defined by the values of name and txon attributes.\\nUse this class to create a placeholder for a Reaction or Pathway to be fully defined later.\"\n	(is-a DatabaseObject)\n	(role abstract)\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot _doRelease\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _databaseColumnSpecification \"ENUM(\'TRUE\',\'FALSE\')\")\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n		(create-accessor read-write))\n	(multislot relatedSpecies\n		(type INSTANCE)\n;+		(allowed-classes Species)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseStatus\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n		(create-accessor read-write))\n	(multislot disease\n		(type INSTANCE)\n;+		(allowed-classes Disease)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write)))\n\n(defclass Pathway \"Multi-step Event. Pathway is recursive, i.e. can contain other Pathways and hence should be handled with care to prevent cycles.\\nUse sub-classes for creating appropriate instances.\\n(The identity of a Pathway is defined by the component Events.)\"\n	(is-a Event)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot isCanonical\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _databaseColumnSpecification \"ENUM(\'TRUE\',\'FALSE\')\")\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy none)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot doi\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _databaseColumnSpecification \"VARCHAR(64)\")\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass ReactionlikeEvent\n	(is-a Event)\n	(role abstract)\n	(multislot output\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(multislot entityOnOtherCell\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot requiredInputComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n		(create-accessor read-write))\n	(multislot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n		(create-accessor read-write))\n	(single-slot isChimeric\n;+		(comment \"this slot acts as a flag for \\\"artificial multi-species\\\" instances (typically used as source of inference for single-species instances where these don\'t have direct evidence)\")\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n		(default FALSE)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _databaseColumnSpecification \"ENUM(\'TRUE\',\'FALSE\')\")\n		(create-accessor read-write))\n	(multislot normalReaction\n		(type INSTANCE)\n;+		(allowed-classes ReactionlikeEvent)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot entityFunctionalStatus\n		(type INSTANCE)\n;+		(allowed-classes EntityFunctionalStatus)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write)))\n\n(defclass Reaction \"Single-step Event.\\nUse sub-classes for creating appropriate instances.\\n(Identity of a Reaction is defined by its input, output and catalystActivity.)\"\n	(is-a ReactionlikeEvent)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot output\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot reverseReaction\n		(type INSTANCE)\n;+		(allowed-classes Reaction)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy none)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot requiredInputComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass BlackBoxEvent\n	(is-a ReactionlikeEvent)\n	(role concrete)\n	(single-slot templateEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot output\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot hasEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot requiredInputComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass Polymerisation\n	(is-a ReactionlikeEvent)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot output\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot requiredInputComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass Depolymerisation\n	(is-a ReactionlikeEvent)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot orthologousEvent\n;+		(comment \"Holds \'equivalent\' Events in other organisms.\")\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot goBiologicalProcess\n;+		(comment \"Fill it with GO biological process which is roughly equal to this GK Event. I.e. For \\\"Human S-phase\\\" it should be \\\"S-phase\\\" and not, say, \\\"Mitotic cell cycle\\\".\")\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredFrom\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot evidenceType\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes Compartment)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot output\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot requiredInputComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot precedingEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass FailedReaction\n	(is-a ReactionlikeEvent)\n	(role concrete)\n	(multislot catalystActivity\n;+		(comment \"roughly the same as EcoCyc\'s EnzymaticReaction\")\n		(type INSTANCE)\n;+		(allowed-classes CatalystActivity)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot input\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass PhysicalEntity \"Thing which participates in Events as input, output or catalyst. Can be recursive,i.e. contain other PhysicalEntities. This allows easier creation of cascading complexes.\"\n	(is-a DatabaseObject)\n	(role abstract)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredTo\n;+		(comment \"reverse attribute of inferredFrom\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n		(create-accessor read-write))\n	(multislot disease\n		(type INSTANCE)\n;+		(allowed-classes Disease)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot cellType\n		(type INSTANCE)\n;+		(allowed-classes CellType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass SimpleEntity \"Indivisible entities, i.e. entities which do not have sub-units. Opposite to Complex\"\n	(is-a PhysicalEntity)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceEntity\n		(type INSTANCE)\n;+		(allowed-classes ReferenceMolecule)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass Complex \"Entities consisting of multiple independant subunits.\"\n	(is-a PhysicalEntity)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot entityOnOtherCell\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasComponent\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(cardinality 2 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot isChimeric\n;+		(comment \"this slot acts as a flag for \\\"artificial multi-species\\\" instances (typically used as source of inference for single-species instances where these don\'t have direct evidence)\")\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n		(default FALSE)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _databaseColumnSpecification \"ENUM(\'TRUE\',\'FALSE\')\")\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot includedLocation\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write)))\n\n(defclass GenomeEncodedEntity\n	(is-a PhysicalEntity)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot species\n		(type INSTANCE)\n;+		(allowed-classes Taxon Species)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass EntityWithAccessionedSequence\n	(is-a GenomeEncodedEntity)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot hasModifiedResidue\n		(type INSTANCE)\n;+		(allowed-classes AbstractModifiedResidue)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy none)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes SequenceDomain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceEntity\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot endCoordinate\n		(type INTEGER)\n		(default -1)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy none)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot startCoordinate\n		(type INTEGER)\n		(default 1)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass EntitySet\n	(is-a PhysicalEntity)\n	(role abstract)\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes Domain)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot hasMember\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(create-accessor read-write)))\n\n(defclass DefinedSet\n	(is-a EntitySet)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot hasMember\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass OpenSet\n	(is-a EntitySet)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceEntity\n		(type INSTANCE)\n;+		(allowed-classes ReferenceEntity)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot hasMember\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass CandidateSet\n	(is-a EntitySet)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot hasCandidate\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot hasMember\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass Polymer\n	(is-a PhysicalEntity)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot minUnitCount\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot maxUnitCount\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot repeatedUnit\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass OtherEntity\n	(is-a PhysicalEntity)\n	(role concrete)\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot inferredFrom\n;+		(comment \"reverse attribute of inferredTo\")\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot compartment\n		(type INSTANCE)\n;+		(allowed-classes EntityCompartment)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasDomain\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot goCellularComponent\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass Regulation\n	(is-a DatabaseObject)\n	(role abstract)\n	(single-slot regulator\n		(type INSTANCE)\n;+		(allowed-classes Event PhysicalEntity CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n		(create-accessor read-write))\n	(multislot containedInPathway\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n		(create-accessor read-write))\n	(single-slot regulatedEntity\n		(type INSTANCE)\n;+		(allowed-classes Event CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n		(create-accessor read-write))\n	(single-slot regulationType\n		(type INSTANCE)\n;+		(allowed-classes RegulationType)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass PositiveRegulation\n	(is-a Regulation)\n	(role concrete)\n	(single-slot regulator\n		(type INSTANCE)\n;+		(allowed-classes Event PhysicalEntity CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot regulatedEntity\n		(type INSTANCE)\n;+		(allowed-classes Event CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot regulationType\n		(type INSTANCE)\n;+		(allowed-classes RegulationType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass Requirement \"Extreme positive regulation, i.e. regulated entity absolutely requires regulator.\"\n	(is-a PositiveRegulation)\n	(role concrete)\n	(single-slot regulator\n		(type INSTANCE)\n;+		(allowed-classes Event PhysicalEntity CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot regulatedEntity\n		(type INSTANCE)\n;+		(allowed-classes Event CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot regulationType\n		(type INSTANCE)\n;+		(allowed-classes RegulationType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass NegativeRegulation\n	(is-a Regulation)\n	(role concrete)\n	(single-slot regulator\n		(type INSTANCE)\n;+		(allowed-classes Event PhysicalEntity CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot revised\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot authored\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot regulatedEntity\n		(type INSTANCE)\n;+		(allowed-classes Event CatalystActivity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot summation\n		(type INSTANCE)\n;+		(allowed-classes Summation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n;+		(user-facet _databaseColumnSpecification \"date\")\n		(create-accessor read-write))\n	(multislot reviewed\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot edited\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot regulationType\n		(type INSTANCE)\n;+		(allowed-classes RegulationType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass DatabaseIdentifier \"Placeholder for database identifiers. Name is something human readable, e.g. a GO term itself while identifier is the numeric id. (In the instances created I\'ve added the \\\"GO:\\\" prefix just for convenience really. When finally sticking the things in db these should be removed.\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"varchar(20)\")\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass Taxon \"Placeholder for organisms. All names go to aliases (easier to retrieve if we just need to \\\"scan\\\" one column rather than both \'name\' and \'alias\'). DatabaseIdentifier allows to stick on, say, NCBI taxonomy ID.\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot superTaxon\n		(type INSTANCE)\n;+		(allowed-classes Taxon)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"varchar(255)\")\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass Species\n	(is-a Taxon)\n	(role concrete)\n	(single-slot superTaxon\n		(type INSTANCE)\n;+		(allowed-classes Taxon)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass InstanceEdit\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot note\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot _applyToAllEditedInstances\n;+		(comment \"Set this attribute vaue to TRUE (by checking it) if you want to attch this InstenceEdit to all edited instances in your project.\")\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot author\n		(type INSTANCE)\n;+		(allowed-classes Person)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(single-slot dateTime\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _databaseColumnSpecification \"TIMESTAMP\")\n		(create-accessor read-write)))\n\n(defclass CatalystActivity \"Allows to denote one activity (out of many possible) of a part of molecule, molecule and complex. Should be read either as \\\"Entity has ACTIVITY activity\\\" or \\\"ACTIVITY activity of entity\\\". The latter is the case if used in a dependency statement.\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(multislot activeUnit\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity Domain)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot physicalEntity\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot activity\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass ReferenceDatabase \"Place to keep info about databases used so that the general stuff shouldn\'t be writen out again and again with each database identifier.\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot accessUrl\n;+		(comment \"Url for retrieving a record\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot url\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write)))\n\n(defclass GO_MolecularFunction \"Basically GO functions\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(multislot negativelyRegulate\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction GO_BiologicalProcess)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot positivelyRegulate\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction GO_BiologicalProcess)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot regulate\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction GO_BiologicalProcess)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(multislot ecNumber\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasPart\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction GO_BiologicalProcess)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot accession\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot componentOf\n		(type INSTANCE)\n;+		(allowed-classes GO_MolecularFunction GO_BiologicalProcess)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass AbstractModifiedResidue \"Placeholder for modified residues. Referenced by PhysicalEntity. Coordinate is the position of the modified residue in the VirtualMolecule. Virtual molecule is referenced in order to prevent situations where, say, residue 5 is phosphorylated in 2 diferent molecules. It woudl be logically wrong for these 2 real molecules to reference the same modified residue. Maybe an overkill (?).\"\n	(is-a DatabaseObject)\n	(role abstract)\n	(single-slot referenceSequence\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write)))\n\n(defclass TranslationalModification\n	(is-a AbstractModifiedResidue)\n	(role abstract)\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot psiMod\n		(type INSTANCE)\n;+		(allowed-classes PsiMod)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass GroupModifiedResidue\n	(is-a TranslationalModification)\n	(role concrete)\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot psiMod\n		(type INSTANCE)\n;+		(allowed-classes PsiMod)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot modification\n		(type INSTANCE)\n;+		(allowed-classes Polymer EntitySet ReferenceGroup)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass ModifiedResidue\n	(is-a TranslationalModification)\n	(role concrete)\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot psiMod\n		(type INSTANCE)\n;+		(allowed-classes PsiMod)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass CrosslinkedResidue\n	(is-a TranslationalModification)\n	(role abstract)\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot secondCoordinate\n		(type INTEGER)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot modification\n		(type INSTANCE)\n;+		(allowed-classes ReferenceGroup Polymer EntitySet)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass IntraChainCrosslinkedResidue\n	(is-a CrosslinkedResidue)\n	(role concrete)\n	(multislot secondCoordinate\n		(type INTEGER)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass InterChainCrosslinkedResidue\n	(is-a CrosslinkedResidue)\n	(role concrete)\n	(multislot equivalentTo\n		(type INSTANCE)\n;+		(allowed-classes InterChainCrosslinkedResidue)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot secondReferenceSequence\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass GeneticallyModifiedResidue\n	(is-a AbstractModifiedResidue)\n	(role abstract))\n\n(defclass ReplacedResidue\n	(is-a GeneticallyModifiedResidue)\n	(role concrete)\n	(single-slot referenceSequence\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot psiMod\n		(type INSTANCE)\n;+		(allowed-classes PsiMod)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass FragmentModification\n	(is-a GeneticallyModifiedResidue)\n	(role abstract)\n	(single-slot startPositionInReferenceSequence\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot endPositionInReferenceSequence\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass FragmentInsertionModification\n	(is-a FragmentModification)\n	(role concrete)\n	(single-slot coordinate\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass FragmentDeletionModification\n	(is-a FragmentModification)\n	(role concrete))\n\n(defclass FragmentReplacedModification\n	(is-a FragmentModification)\n	(role concrete)\n	(single-slot alteredAminoAcidFragment\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass Person\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot firstname\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot figure\n		(type INSTANCE)\n;+		(allowed-classes Figure)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot url\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot surname\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"varchar(255)\")\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot affiliation\n		(type INSTANCE)\n;+		(allowed-classes Affiliation)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot initial\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"varchar(10)\")\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot project\n		(type STRING)\n;+		(value \"FlyBase\")\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot eMailAddress\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _databaseColumnSpecification \"varchar(255)\")\n		(create-accessor read-write)))\n\n(defclass Affiliation\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot address\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write)))\n\n(defclass Figure\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot url\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass RegulationType\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes RegulationType)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _databaseColumnSpecification \"VARCHAR(255)\")\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write)))\n\n(defclass FrontPage\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot frontPageItem\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass Summation\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot literatureReference\n		(type INSTANCE)\n;+		(allowed-classes Publication)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot text\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass EvidenceType \"Instances of this class represent different evidence types\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes EvidenceType)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write)))\n\n(defclass ConcurrentEventSet\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot concurrentEvents\n		(type INSTANCE)\n;+		(allowed-classes Event)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot focusEntity\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write)))\n\n(defclass GO_BiologicalProcess\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot accession\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot hasPart\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess GO_MolecularFunction)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot negativelyRegulate\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess GO_MolecularFunction)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot positivelyRegulate\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess GO_MolecularFunction)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot regulate\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess GO_MolecularFunction)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot componentOf\n		(type INSTANCE)\n;+		(allowed-classes GO_BiologicalProcess GO_MolecularFunction)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass _AttributeValueBeforeChange\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot changedAttribute\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot previousValue\n		(type STRING)\n		(create-accessor read-write)))\n\n(defclass _InstanceBeforeChange\n	(is-a DatabaseObject)\n	(role concrete)\n	(multislot attributeValuesBeforeChange\n		(type INSTANCE)\n;+		(allowed-classes _AttributeValueBeforeChange)\n		(create-accessor read-write))\n	(single-slot changedInstanceDB_ID\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot instanceEdit\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass ReactionCoordinates\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot locatedEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot sourceX\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot locationContext\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot sourceY\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot targetX\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot targetY\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass GO_CellularComponent\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot accession\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot hasPart\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot componentOf\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass Compartment\n	(is-a GO_CellularComponent)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot accession\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot componentOf\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass EntityCompartment\n	(is-a Compartment)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot accession\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot componentOf\n		(type INSTANCE)\n;+		(allowed-classes GO_CellularComponent)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass Domain\n	(is-a DatabaseObject)\n	(role abstract)\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass SequenceDomain\n	(is-a Domain)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceEntity\n		(type INSTANCE)\n;+		(allowed-classes ReferenceSequence)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot endCoordinate\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot startCoordinate\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass ComplexDomain\n	(is-a Domain)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasComponent\n		(type INSTANCE)\n;+		(allowed-classes Domain PhysicalEntity)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass GenericDomain \"For generalising multiple domains in different entities so that GenericEntities can also have domains.\"\n	(is-a Domain)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot hasInstance\n		(type INSTANCE)\n;+		(allowed-classes ComplexDomain SequenceDomain)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass ReferenceEntity\n	(is-a DatabaseObject)\n	(role abstract)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot otherIdentifier\n		(type STRING)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n		(create-accessor read-write)))\n\n(defclass ReferenceMolecule\n	(is-a ReferenceEntity)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot formula\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot otherIdentifier\n		(type STRING)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot atomicConnectivity\n;+		(comment \"Contents of the mol file.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass ReferenceGroup\n	(is-a ReferenceEntity)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot formula\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot otherIdentifier\n		(type STRING)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot atomicConnectivity\n;+		(comment \"Contents of the mol file.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass ReferenceSequence\n	(is-a ReferenceEntity)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot keyword\n;+		(comment \"KW lines from SPTREMBL records\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot comment\n;+		(comment \"CC lines from sequence record.\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot checksum\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot secondaryIdentifier\n;+		(comment \"This is for secondary/other identifiers WITHIN THE SAME DATABASE, e.g. secondary SPTREMBL accessions and SwissProt id. Use crossReference if the the identifier is from another db, e.g. for adding a RefSeq identifier to SPTREMBL SequenceDatabaseIdentifier.\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot isSequenceChanged\n		(type SYMBOL)\n		(allowed-values FALSE TRUE)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot sequenceLength\n		(type INTEGER)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(multislot otherIdentifier\n		(type STRING)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot species\n		(type INSTANCE)\n;+		(allowed-classes Species)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot description\n;+		(comment \"DE lines from sequence record\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot geneName\n;+		(comment \"GN lines from sequence record.\")\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass ReferenceRNASequence\n	(is-a ReferenceSequence)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot referenceGene\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDNASequence)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass ReferenceDNASequence\n	(is-a ReferenceSequence)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot crossReference\n;+		(comment \"This slot is for holding references to the EQUIVALENT things in other databases. If the instance this slot is attached to represents an event the DatabaseIdentifier put into this slot must also point to some sort of event/processs/reaction/happening in some other db.\")\n		(type INSTANCE)\n;+		(allowed-classes DatabaseIdentifier)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass ReferenceGeneProduct\n	(is-a ReferenceSequence)\n	(role concrete)\n	(multislot referenceTranscript\n		(type INSTANCE)\n;+		(allowed-classes ReferenceRNASequence)\n		(create-accessor read-write))\n	(multislot referenceGene\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDNASequence)\n		(create-accessor read-write))\n	(single-slot _chainChangeLog\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot chain\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass ReferenceIsoform\n	(is-a ReferenceGeneProduct)\n	(role concrete)\n	(multislot isoformParent\n		(type INSTANCE)\n;+		(allowed-classes ReferenceGeneProduct)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot variantIdentifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass PathwayCoordinates\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot locatedEvent\n		(type INSTANCE)\n;+		(allowed-classes Event)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot minY\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot minX\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot maxX\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot maxY\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass StableIdentifier\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot identifierVersion\n;+		(comment \"Version number for stable identifier\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write)))\n\n(defclass _Deleted \"Stores information about a single deleted instance\"\n	(is-a DatabaseObject)\n	(role concrete)\n	(multislot replacementInstances\n;+		(comment \"The instance(s) (if any) that replace the deleted instance.  The allowed instance classes have been restricted to those for which stable IDs get generated.  If the classes for which stable IDs are generated is expanded, these need to be changed correspondingly.\")\n		(type INSTANCE)\n;+		(allowed-classes Regulation Event PhysicalEntity)\n		(create-accessor read-write))\n	(multislot deletedInstanceDB_ID\n;+		(comment \"Contains the DB_IDs of the instances that have been deleted\")\n		(type INTEGER)\n		(cardinality 1 ?VARIABLE)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot curatorComment\n;+		(comment \"A free-text comment that the curator may optionally provide to add extra explanation\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot reason\n;+		(comment \"Reason for deletion, drawn from a limited controlled vocabulary of terms\")\n		(type INSTANCE)\n;+		(allowed-classes DeletedControlledVocabulary)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass ControlledVocabulary \"Superclass for controlled vocabularies\"\n	(is-a DatabaseObject)\n	(role abstract)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"Short explanation.  Will appear on mouseover, if present.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"Maximum 5 word controlled vocabulary term\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass DeletedControlledVocabulary \"List of terms explaining reasons for deleting instances\"\n	(is-a ControlledVocabulary)\n	(role concrete))\n\n(defclass PathwayDiagramItem \"This is an abstract superclass purely fo grouping purposes.\"\n	(is-a DatabaseObject)\n	(role abstract))\n\n(defclass Vertex\n	(is-a PathwayDiagramItem)\n	(role concrete)\n	(single-slot pathwayDiagram\n		(type INSTANCE)\n;+		(allowed-classes PathwayDiagram)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot width\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot height\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot representedInstance\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot y\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot x\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write)))\n\n(defclass ReactionVertex\n	(is-a Vertex)\n	(role concrete)\n	(single-slot representedInstance\n		(type INSTANCE)\n;+		(allowed-classes ReactionlikeEvent)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot pointCoordinates\n;+		(comment \"Concatenated string of coordinate pairs defining the backbone of the reaction.\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass EntityVertex\n	(is-a Vertex)\n	(role concrete)\n	(single-slot containedInEntityVertex\n		(type INSTANCE)\n;+		(allowed-classes EntityVertex)\n;+		(cardinality 0 1)\n		(create-accessor read-write))\n	(single-slot representedInstance\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot y\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot x\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass PathwayVertex\n	(is-a Vertex)\n	(role concrete)\n	(single-slot representedInstance\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n;+		(cardinality 1 1)\n		(create-accessor read-write)))\n\n(defclass Edge\n	(is-a PathwayDiagramItem)\n	(role concrete)\n	(single-slot edgeType\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot pathwayDiagram\n		(type INSTANCE)\n;+		(allowed-classes PathwayDiagram)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot sourceVertex\n		(type INSTANCE)\n;+		(allowed-classes Vertex)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot targetVertex\n		(type INSTANCE)\n;+		(allowed-classes Vertex)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot pointCoordinates\n;+		(comment \"Concatenated coordinate pairs of the points specifying this Edge\")\n		(type STRING)\n;+		(cardinality 0 1)\n		(create-accessor read-write)))\n\n(defclass PathwayDiagram\n	(is-a PathwayDiagramItem)\n	(role concrete)\n	(single-slot width\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot height\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n		(create-accessor read-write))\n	(single-slot storedATXML\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _databaseColumnSpecification \"LONGBLOB\")\n		(create-accessor read-write))\n	(multislot representedPathway\n		(type INSTANCE)\n;+		(allowed-classes Pathway)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass VertexSearchableTerm\n	(is-a PathwayDiagramItem)\n	(role concrete)\n	(single-slot providerCount\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot searchableTerm\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _databaseColumnSpecification \"VARCHAR(255)\")\n		(create-accessor read-write))\n	(multislot termProvider\n		(type INSTANCE)\n;+		(allowed-classes Event PhysicalEntity ReferenceEntity GO_MolecularFunction GO_BiologicalProcess DatabaseIdentifier)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(multislot vertex\n		(type INSTANCE)\n;+		(allowed-classes Vertex)\n		(cardinality 1 ?VARIABLE)\n		(create-accessor read-write))\n	(single-slot species\n		(type INSTANCE)\n;+		(allowed-classes Species Taxon)\n;+		(cardinality 1 1)\n		(create-accessor read-write))\n	(single-slot vertexCount\n		(type INTEGER)\n;+		(cardinality 1 1)\n		(create-accessor read-write)))\n\n(defclass Publication\n	(is-a DatabaseObject)\n	(role abstract)\n	(single-slot title\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot author\n		(type INSTANCE)\n;+		(allowed-classes Person)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass LiteratureReference\n	(is-a Publication)\n	(role concrete)\n	(single-slot created\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot volume\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot journal\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _databaseColumnSpecification \"varchar(255)\")\n		(create-accessor read-write))\n	(single-slot pubMedIdentifier\n;+		(comment \"PubMed ID\")\n		(type INTEGER)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory REQUIRED)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot year\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot modified\n		(type INSTANCE)\n;+		(allowed-classes InstanceEdit)\n;+		(user-facet _attributeCategory NOMANUALEDIT)\n		(create-accessor read-write))\n	(single-slot pages\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass Book \"This class is used to describe a book or book chapter.\"\n	(is-a Publication)\n	(role concrete)\n	(multislot chapterAuthors\n		(type INSTANCE)\n;+		(allowed-classes Person)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot title\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(single-slot year\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(multislot author\n		(type INSTANCE)\n;+		(allowed-classes Person)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot chapterTitle\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(single-slot ISBN\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(single-slot publisher\n		(type INSTANCE)\n;+		(allowed-classes Affiliation)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot pages\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass URL\n	(is-a Publication)\n	(role concrete)\n	(single-slot uniformResourceLocator\n		(type STRING)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write)))\n\n(defclass _Release\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot releaseNumber\n		(type INTEGER)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot releaseDate\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass FunctionalStatus\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot functionalStatusType\n		(type INSTANCE)\n;+		(allowed-classes FunctionalStatusType)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(single-slot structuralVariant\n		(type INSTANCE)\n;+		(allowed-classes SequenceOntology)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass EntityFunctionalStatus\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot physicalEntity\n		(type INSTANCE)\n;+		(allowed-classes PhysicalEntity)\n;+		(cardinality 1 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot functionalStatus\n		(type INSTANCE)\n;+		(allowed-classes FunctionalStatus)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write)))\n\n(defclass FunctionalStatusType\n	(is-a DatabaseObject)\n	(role concrete)\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write)))\n\n(defclass ExternalOntology\n	(is-a DatabaseObject)\n	(role abstract)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot synonym\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass SequenceOntology\n	(is-a ExternalOntology)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ANY)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes SequenceOntology)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass Disease\n	(is-a ExternalOntology)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes Disease)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding GO definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot synonym\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Shoudl be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\n\n(defclass CellType\n	(is-a ExternalOntology)\n	(role concrete)\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes CellType)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write)))\n\n(defclass PsiMod \"to store PSI-MOD terms (ontology for protein modifications: http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MOD)\"\n	(is-a ExternalOntology)\n	(role concrete)\n	(single-slot identifier\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n;+		(user-facet _identityDefinedBy ALL)\n		(create-accessor read-write))\n	(multislot instanceOf\n		(type INSTANCE)\n;+		(allowed-classes PsiMod)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot definition\n;+		(comment \"This is for holding PSI-MOD definition field.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(multislot synonym\n		(type STRING)\n;+		(user-facet _attributeCategory OPTIONAL)\n		(create-accessor read-write))\n	(single-slot name_\n;+		(comment \"This is the name of the entity whatever that entity is. Should be unique. Usually this is going to be the browser name.\")\n		(type STRING)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write))\n	(single-slot referenceDatabase\n		(type INSTANCE)\n;+		(allowed-classes ReferenceDatabase)\n;+		(cardinality 0 1)\n;+		(user-facet _attributeCategory MANDATORY)\n		(create-accessor read-write)))\0\0\0pont_file_content\0\0\0\0\0\0\0\0\0\0:STANDARD-CLASS\0\0\0\0\0\0Ã\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0identityDefinedBy\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0ReferenceEntityÅ\0\0\0DatabaseIdentifierÅ\0\0\0EventÅ\0\0\0GO_MolecularFunctionÅ\0\0\0GO_BiologicalProcess\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0termProvider\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0removingActivity\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0dateTime\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0otherIdentifier\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0uniformResourceLocator\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0y\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0DB_ID\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0hasFunctionalVariants\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0formula\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0precedingEvent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0_Protege_id\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÅ\0\0\0multiple\0\0\0pubMedIdentifier\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0edited\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0doi\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninstanceOf\0\0\0\0\0\0Å\0\0\0ReferenceDatabase\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceDatabase\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rendCoordinate\0\0\0\0\0\0Å\0\0\0GO_BiologicalProcess\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goBiologicalProcess\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxCount\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0name\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0representedInstance\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewer\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0synonym\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondaryIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	regulator\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0title\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0curatorComment\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\ninferredTo\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0chapterAuthors\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0repeatedUnit\0\0\0\0\0\0Å\0\0\0Vertex\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0vertex\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0searchableTerm\0\0\0\0\0\0Å\0\0\0Figure\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0figure\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\n_doRelease\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0concurrentEvents\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0author\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0dateAccepted\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0regulate\0\0\0\0\0\0Å\0\0\0PathwayDiagram\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0pathwayDiagram\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minX\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0variantIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accessUrl\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0referenceGroupCount\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0edgeType\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0chapterTitle\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0sourceX\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0changedAttribute\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rfrontPageItem\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0databaseIdentifier\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0sequenceLength\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\n__is_ghost\0\0\0\0\0\0Å\0\0\0SpeciesÅ\0\0\0Taxon\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0species\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0height\0\0\0\0\0\0Å\0\0\0SequenceOntology\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0structuralVariant\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0targetX\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rproviderCount\0\0\0\0\0\0Å\0\0\0Vertex\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0targetVertex\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0representedPathway\0\0\0\0\0\0Å\0\0\0Taxon\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nsuperTaxon\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasSpecialisedForm\0\0\0\0\0\0Å\0\0\0InterChainCrosslinkedResidue\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0equivalentTo\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0isCanonical\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0locationContext\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxUnitCount\0\0\0\0\0\0Å\0\0\0GO_MolecularFunction\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0activity\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0wrappedLabel\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasMember\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nidentifier\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0creatingActivity\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0address\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0startCoordinate\0\0\0\0\0\0Å\0\0\0Affiliation\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0	publisher\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0atomicConnectivity\0\0\0\0\0\0Å\0\0\0AbstractModifiedResidue\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasModifiedResidue\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0_chainChangeLog\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0databaseColumnSpecification\0\0\0\0\0\0Å\0\0\0ReactionlikeEvent\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0normalReaction\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0inferredFrom\0\0\0\0\0\0Å\0\0\0ReferenceGroup\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceGroup\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0surname\0\0\0\0\0\0Å\0\0\0_AttributeValueBeforeChange\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0attributeValuesBeforeChange\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0physicalEntity\0\0\0\0\0\0Å\0\0\0Affiliation\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0affiliation\0\0\0\0\0\0Å\0\0\0Vertex\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0sourceVertex\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rreleaseStatus\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0project\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ndefinition\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasInstance\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0referenceSequence\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0instanceEdit\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxX\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0entityOnOtherCell\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0note\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÅ\0\0\0multiple\0\0\0deletedInstanceDB_ID\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0created\0\0\0\0\0\0Å\0\0\0ReferenceMolecule\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0residue\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0targetY\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\r_doNotRelease\0\0\0\0\0\0Å\0\0\0ReferenceSequence\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondReferenceSequence\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0ReferenceGroup\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0modification\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0containedInPathway\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0sourceY\0\0\0\0\0\0Å\0\0\0EvidenceType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0evidenceType\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minY\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0eMailAddress\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\nactiveUnit\0\0\0\0\0\0Å\0\0\0StableIdentifier\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0stableIdentifier\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0input\0\0\0\0\0\0Å\0\0\0	Summation\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	summation\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0classDisplayName\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasCandidate\0\0\0\0\0\0Å\0\0\0Publication\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0literatureReference\0\0\0\0\0\0Å\0\0\0Pathway\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0eventLocationContext\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0checksum\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0changedInstanceDB_ID\0\0\0\0\0\0Å\0\0\0Reaction\0\0\0allowed\nreverseReaction\0\0\0inverse-attribute\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0reverseReaction\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0Domain\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0requiredInputComponent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0url\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0dateRevised\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	shortName\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0EventÅ\0\0\0\nRegulation\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0replacementInstances\0\0\0\0\0\0Å\0\0\0ReferenceRNASequence\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0referenceTranscript\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0releaseDate\0\0\0\0\0\0Å\0\0\0DatabaseIdentifier\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0crossReference\0\0\0\0\0\0Å\0\0\0ControlledVocabulary\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0reason\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0output\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0description\0\0\0\0\0\0Å\0\0\0Disease\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0disease\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	accession\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0modified\0\0\0\0\0\0Å\0\0\0FunctionalStatus\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0functionalStatus\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0fingerPrintString\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0vertexCount\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0identify\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rtemplateEvent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0attributeCategory\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0pages\0\0\0\0\0\0Å\0\0\0EntityVertex\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0containedInEntityVertex\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0keyword\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0width\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0	hasDomain\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rdateSubmitted\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÅ\0\0\0multiple\0\0\0secondCoordinate\0\0\0\0\0\0Å\0\0\0EntityFunctionalStatus\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0entityFunctionalStatus\0\0\0\0\0\0Å\0\0\0FunctionalStatusType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0functionalStatusType\0\0\0\0\0\0Å\0\0\0Person\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0editor\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0storedATXML\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0ISBN\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0text\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0focusEntity\0\0\0\0\0\0Å\0\0\0PhysicalEntityÅ\0\0\0AffiliationÅ\0\0\0LiteratureReferenceÅ\0\0\0FigureÅ\0\0\0CatalystActivityÅ\0\0\0RegulationTypeÅ\0\0\0ReferenceDatabaseÅ\0\0\0PersonÅ\0\0\0\nRegulationÅ\0\0\0EvidenceTypeÅ\0\0\0GO_BiologicalProcessÅ\0\0\0	FrontPageÅ\0\0\0	SummationÅ\0\0\0InstanceEditÅ\0\0\0EventÅ\0\0\0DatabaseIdentifierÅ\0\0\0TaxonÅ\0\0\0AbstractModifiedResidueÅ\0\0\0ConcurrentEventSet\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0changedInstance\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0journal\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0alteredAminoAcidFragment\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0positivelyRegulate\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0_displayName\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0taxon\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasPart\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0endPositionInReferenceSequence\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0authored\0\0\0\0\0\0Å\0\0\0CatalystActivity\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0catalystActivity\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0year\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0	firstname\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0componentOf\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minCount\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasEvent\0\0\0\0\0\0Å\0\0\0Compartment\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0compartment\0\0\0\0\0\0Å\0\0\0EntityCompartment\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0includedLocation\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rpreviousValue\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0negativelyRegulate\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\rreleaseNumber\0\0\0\0\0\0Å\0\0\0ReferenceDNASequence\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\rreferenceGene\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0isSequenceChanged\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0pointCoordinates\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0geneName\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0chain\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0revised\0\0\0\0\0\0Å\0\0\0EventÅ\0\0\0CatalystActivity\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulatedEntity\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\northologousEvent\0\0\0inverse-attribute\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0orthologousEvent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0identifierVersion\0\0\0\0\0\0Å\0\0\0Event\0\0\0allowed\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0locatedEvent\0\0\0\0\0\0Å\0\0\0Species\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0relatedSpecies\0\0\0\0\0\0Å\0\0\0CellType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0cellType\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\nisChimeric\0\0\0\0\0\0Å\0\0\0InstanceEdit\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0reviewed\0\0\0\0\0\0Å\0\0\0PsiMod\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0psiMod\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0x\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0referenceEntity\0\0\0\0\0\0Å\0\0\0RegulationType\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0regulationType\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0\ncoordinate\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0 startPositionInReferenceSequence\0\0\0\0\0\0Å\0\0\0ReferenceGeneProduct\0\0\0allowed\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0\risoformParent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0minUnitCount\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0ecNumber\0\0\0\0\0\0Å\0\0\0GO_CellularComponent\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0goCellularComponent\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0volume\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0Reactome_109607_14914_Slot_1\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÅ\0\0\0multiple\0\0\0hasComponent\0\0\0\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÅ\0\0\0multiple\0\0\0comment\0\0\0\0\0\0Å\0\0\0PhysicalEntity\0\0\0allowed\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_instance_type\0\0\0typeÄ\0\0\0multiple\0\0\0normalEntity\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0slotDisplayName\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0initial\0\0\0\0\0\0max_cardinality\n1\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_integer_type\0\0\0typeÄ\0\0\0multiple\0\0\0maxY\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0\Z_applyToAllEditedInstances\0\0\0	attribute\0\0\0\0\0\0\0role\0\0\0:CLIPS_TOP_LEVEL_SLOT_CLASS\0\0\0\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:EXTENDED-CLASS\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0classDisplayName\0\0\0	attribute\0\0\0Å\0\0\0:STANDARD-CLASS\0\0\0parent\0\0\0:EXTENDED-CLASS\0\0\0\0\0\0\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:EXTENDED-SLOT\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0identityDefinedBy\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:EXTENDED-SLOT\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0databaseColumnSpecification\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:EXTENDED-SLOT\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0attributeCategory\0\0\0\0\0\0max_cardinality\n0\0\0\0min_cardinality\n:EXTENDED-SLOT\0\0\0origin\ndb_string_type\0\0\0typeÄ\0\0\0multiple\0\0\0slotDisplayName\0\0\0	attribute\0\0\0Å\0\0\0:STANDARD-SLOT\0\0\0parent\0\0\0:EXTENDED-SLOT\0\0\0\0\0\0\0:STANDARD-SLOT\0\0\0protege_classes\0yñ; Wed Feb 05 17:58:55 PST 2014\n; \n;+ (version \"3.4.7\")\n;+ (build \"Build 620\")\n\n([__is_ghost] of  %3AEXTENDED-SLOT\n)\n\n([_applyToAllEditedInstances] of  %3AEXTENDED-SLOT\n)\n\n([_attributeCategory] of  %3ASTANDARD-FACET\n\n	(%3AASSOCIATED-SLOT [attributeCategory]))\n\n([_AttributeValueBeforeChange] of  %3AEXTENDED-CLASS\n)\n\n([_chainChangeLog] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([_databaseColumnSpecification] of  %3ASTANDARD-FACET\n\n	(%3AASSOCIATED-SLOT [databaseColumnSpecification]))\n\n([_Deleted] of  %3AEXTENDED-CLASS\n)\n\n([_displayName] of  %3AEXTENDED-SLOT\n)\n\n([_doNotRelease] of  %3AEXTENDED-SLOT\n)\n\n([_doRelease] of  %3AEXTENDED-SLOT\n)\n\n([_identityDefinedBy] of  %3ASTANDARD-FACET\n\n	(%3AASSOCIATED-SLOT [identityDefinedBy]))\n\n([_InstanceBeforeChange] of  %3AEXTENDED-CLASS\n)\n\n([_Protege_id] of  %3AEXTENDED-SLOT\n)\n\n([_Release] of  %3AEXTENDED-CLASS\n)\n\n([_slotDisplayName] of  %3ASTANDARD-FACET\n\n	(%3AASSOCIATED-SLOT [slotDisplayName]))\n\n([AbstractModifiedResidue] of  %3AEXTENDED-CLASS\n)\n\n([accession] of  %3AEXTENDED-SLOT\n)\n\n([accessUrl] of  %3AEXTENDED-SLOT\n)\n\n([activeUnit] of  %3AEXTENDED-SLOT\n)\n\n([activity] of  %3AEXTENDED-SLOT\n)\n\n([address] of  %3AEXTENDED-SLOT\n)\n\n([affiliation] of  %3AEXTENDED-SLOT\n)\n\n([Affiliation] of  %3AEXTENDED-CLASS\n)\n\n([alteredAminoAcidFragment] of  %3AEXTENDED-SLOT\n)\n\n([atomicConnectivity] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([attributeCategory] of  %3AEXTENDED-SLOT\n\n	(%3AASSOCIATED-FACET [_attributeCategory]))\n\n([attributeValuesBeforeChange] of  %3AEXTENDED-SLOT\n)\n\n([author] of  %3AEXTENDED-SLOT\n)\n\n([authored] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([BlackBoxEvent] of  %3AEXTENDED-CLASS\n)\n\n([Book] of  %3AEXTENDED-CLASS\n)\n\n([CandidateSet] of  %3AEXTENDED-CLASS\n)\n\n([catalystActivity] of  %3AEXTENDED-SLOT\n)\n\n([CatalystActivity] of  %3AEXTENDED-CLASS\n)\n\n([cellType] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL)\n	(identityDefinedBy ALL))\n\n([CellType] of  %3AEXTENDED-CLASS\n)\n\n([chain] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([changedAttribute] of  %3AEXTENDED-SLOT\n\n	(databaseColumnSpecification \"VARCHAR(64)\"))\n\n([changedInstance] of  %3AEXTENDED-SLOT\n)\n\n([changedInstanceDB_ID] of  %3AEXTENDED-SLOT\n)\n\n([chapterAuthors] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([chapterTitle] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL)\n	(identityDefinedBy ANY))\n\n([checksum] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([classDisplayName] of  %3AEXTENDED-SLOT\n)\n\n([comment] of  %3AEXTENDED-SLOT\n)\n\n([Compartment] of  %3AEXTENDED-CLASS\n)\n\n([compartment] of  %3AEXTENDED-SLOT\n)\n\n([Complex] of  %3AEXTENDED-CLASS\n)\n\n([ComplexDomain] of  %3AEXTENDED-CLASS\n)\n\n([componentOf] of  %3AEXTENDED-SLOT\n)\n\n([concurrentEvents] of  %3AEXTENDED-SLOT\n\n	(identityDefinedBy ALL))\n\n([ConcurrentEventSet] of  %3AEXTENDED-CLASS\n)\n\n([containedInEntityVertex] of  %3AEXTENDED-SLOT\n)\n\n([containedInPathway] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([ControlledVocabulary] of  %3AEXTENDED-CLASS\n)\n\n([coordinate] of  %3AEXTENDED-SLOT\n\n	(attributeCategory REQUIRED))\n\n([created] of  %3AEXTENDED-SLOT\n)\n\n([creatingActivity] of  %3AEXTENDED-SLOT\n)\n\n([CrosslinkedResidue] of  %3AEXTENDED-CLASS\n)\n\n([crossReference] of  %3AEXTENDED-SLOT\n)\n\n([curatorComment] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([databaseColumnSpecification] of  %3AEXTENDED-SLOT\n\n	(%3AASSOCIATED-FACET [_databaseColumnSpecification]))\n\n([DatabaseIdentifier] of  %3AEXTENDED-CLASS\n)\n\n([databaseIdentifier] of  %3AEXTENDED-SLOT\n)\n\n([DatabaseObject] of  %3AEXTENDED-CLASS\n)\n\n([dateAccepted] of  %3AEXTENDED-SLOT\n)\n\n([dateRevised] of  %3AEXTENDED-SLOT\n)\n\n([dateSubmitted] of  %3AEXTENDED-SLOT\n)\n\n([dateTime] of  %3AEXTENDED-SLOT\n)\n\n([DB_ID] of  %3AEXTENDED-SLOT\n)\n\n([DefinedSet] of  %3AEXTENDED-CLASS\n)\n\n([definition] of  %3AEXTENDED-SLOT\n)\n\n([DeletedControlledVocabulary] of  %3AEXTENDED-CLASS\n)\n\n([deletedInstanceDB_ID] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([Depolymerisation] of  %3AEXTENDED-CLASS\n)\n\n([description] of  %3AEXTENDED-SLOT\n)\n\n([disease] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([Disease] of  %3AEXTENDED-CLASS\n)\n\n([doi] of  %3AEXTENDED-SLOT\n)\n\n([Domain] of  %3AEXTENDED-CLASS\n)\n\n([ecNumber] of  %3AEXTENDED-SLOT\n)\n\n([edgeType] of  %3AEXTENDED-SLOT\n)\n\n([edited] of  %3AEXTENDED-SLOT\n)\n\n([editor] of  %3AEXTENDED-SLOT\n)\n\n([eMailAddress] of  %3AEXTENDED-SLOT\n)\n\n([endCoordinate] of  %3AEXTENDED-SLOT\n)\n\n([endPositionInReferenceSequence] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([EntityCompartment] of  %3AEXTENDED-CLASS\n)\n\n([entityFunctionalStatus] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL)\n	(identityDefinedBy ALL))\n\n([EntityFunctionalStatus] of  %3AEXTENDED-CLASS\n)\n\n([entityOnOtherCell] of  %3AEXTENDED-SLOT\n)\n\n([EntitySet] of  %3AEXTENDED-CLASS\n)\n\n([EntityWithAccessionedSequence] of  %3AEXTENDED-CLASS\n)\n\n([equivalentTo] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([Event] of  %3AEXTENDED-CLASS\n)\n\n([eventLocationContext] of  %3AEXTENDED-SLOT\n)\n\n([evidenceType] of  %3AEXTENDED-SLOT\n)\n\n([ExternalOntology] of  %3AEXTENDED-CLASS\n)\n\n([FailedReaction] of  %3AEXTENDED-CLASS\n)\n\n([Figure] of  %3AEXTENDED-CLASS\n)\n\n([figure] of  %3AEXTENDED-SLOT\n)\n\n([fingerPrintString] of  %3AEXTENDED-SLOT\n)\n\n([firstname] of  %3AEXTENDED-SLOT\n)\n\n([focusEntity] of  %3AEXTENDED-SLOT\n\n	(identityDefinedBy ALL))\n\n([formula] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([FragmentDeletionModification] of  %3AEXTENDED-CLASS\n)\n\n([FragmentInsertionModification] of  %3AEXTENDED-CLASS\n)\n\n([FragmentModification] of  %3AEXTENDED-CLASS\n)\n\n([FragmentReplacedModification] of  %3AEXTENDED-CLASS\n)\n\n([FrontPage] of  %3AEXTENDED-CLASS\n)\n\n([frontPageItem] of  %3AEXTENDED-SLOT\n)\n\n([FunctionalStatus] of  %3AEXTENDED-CLASS\n)\n\n([functionalStatus] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY)\n	(identityDefinedBy ALL))\n\n([functionalStatusType] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY)\n	(databaseColumnSpecification \"ENUM(\'gain_of_function\', \'loss_of_function\', \'partial_loss_of_function\', \'decreased_translational_product_level\', \'increased_translational_product_level\')\")\n	(identityDefinedBy ANY))\n\n([FunctionalStatusType] of  %3AEXTENDED-CLASS\n)\n\n([geneName] of  %3AEXTENDED-SLOT\n)\n\n([GenericDomain] of  %3AEXTENDED-CLASS\n)\n\n([GeneticallyModifiedResidue] of  %3AEXTENDED-CLASS\n)\n\n([GenomeEncodedEntity] of  %3AEXTENDED-CLASS\n)\n\n([GK_1] of  ReferenceDatabase\n\n	(_displayName \"GO\")\n	(accessUrl \"http://www.ebi.ac.uk/ego/QuickGO?mode=display&entry=GO:###ID###\")\n	(DB_ID 1)\n	(name\n		\"GO\"\n		\"Gene Ontology\")\n	(url \"http://www.geneontology.org\"))\n\n([GK_1011570] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] perlecan [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 1011570)\n	(endCoordinate 4391)\n	(name\n		\"perlecan\"\n		\"HSPG2\")\n	(referenceEntity [GK_61584])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358429])\n	(startCoordinate 22))\n\n([GK_1042833] of  GO_CellularComponent\n\n	(_displayName \"cell periphery\")\n	(accession \"0071944\")\n	(DB_ID 1042833)\n	(definition \"The part of a cell encompassing the cell cortex, the plasma membrane, and any external encapsulating structures.\")\n	(instanceOf [GK_258147])\n	(name \"cell periphery\")\n	(referenceDatabase [GK_1]))\n\n([GK_1043016] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of cysteine-type endopeptidase activity\")\n	(accession \"2000116\")\n	(DB_ID 1043016)\n	(definition \"Any process that modulates the frequency, rate or extent of cysteine-type endopeptidase activity.\")\n	(instanceOf [GK_258859])\n	(name \"regulation of cysteine-type endopeptidase activity\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_4344]))\n\n([GK_106] of  GO_BiologicalProcess\n\n	(_displayName \"biological_process\")\n	(accession \"0008150\")\n	(DB_ID 106)\n	(definition \"Any process specifically pertinent to the functioning of integrated living units: cells, tissues, organs, and organisms. A process is a collection of molecular events with a defined beginning and end.\")\n	(name \"biological_process\")\n	(referenceDatabase [GK_1]))\n\n([GK_109477] of  Person\n\n	(_displayName \"Hellman, U\")\n	(DB_ID 109477)\n	(firstname \"Ulf\")\n	(initial \"U\")\n	(surname \"Hellman\"))\n\n([GK_109607] of  Pathway\n\n	(_displayName \"[Hsa] Extrinsic Pathway for Apoptosis\")\n	(_doRelease TRUE)\n	(authored [GK_139957])\n	(compartment [GK_356])\n	(DB_ID 109607)\n	(doi \"10.3180/REACT_1059.1\")\n	(edited [GK_165085])\n	(figure [GK_3209986])\n	(goBiologicalProcess [GK_2133497])\n	(hasEvent\n		[GK_73887]\n		[GK_140534]\n		[GK_3371378]\n		[GK_2672196])\n	(literatureReference [GK_139960])\n	(name \"Extrinsic Pathway for Apoptosis\")\n	(reviewed\n		[GK_165094]\n		[GK_3597630])\n	(revised [GK_3632873])\n	(species [GK_48887])\n	(stableIdentifier [GK_360111])\n	(summation [GK_139959]))\n\n([GK_1105998] of  StableIdentifier\n\n	(_displayName \"REACT_18708.1\")\n	(DB_ID 1105998)\n	(identifier \"REACT_18708\")\n	(identifierVersion \"1\"))\n\n([GK_111029] of  Person\n\n	(_displayName \"Camonis, JH\")\n	(DB_ID 111029)\n	(firstname \"J H\")\n	(initial \"JH\")\n	(surname \"Camonis\"))\n\n([GK_1110572] of  StableIdentifier\n\n	(_displayName \"REACT_24037.1\")\n	(DB_ID 1110572)\n	(identifier \"REACT_24037\")\n	(identifierVersion \"1\"))\n\n([GK_1111661] of  StableIdentifier\n\n	(_displayName \"REACT_18787.1\")\n	(DB_ID 1111661)\n	(identifier \"REACT_18787\")\n	(identifierVersion \"1\"))\n\n([GK_111168] of  Person\n\n	(_displayName \"Smith, CA\")\n	(DB_ID 111168)\n	(firstname \"C A\")\n	(initial \"CA\")\n	(surname \"Smith\"))\n\n([GK_113970] of  Person\n\n	(_displayName \"Bodmer, JL\")\n	(DB_ID 113970)\n	(firstname \"Jean-Luc\")\n	(initial \"JL\")\n	(surname \"Bodmer\"))\n\n([GK_114129] of  Person\n\n	(_displayName \"Zhang, X\")\n	(DB_ID 114129)\n	(firstname \"Xiuqin\")\n	(initial \"X\")\n	(surname \"Zhang\"))\n\n([GK_114197] of  Person\n\n	(_displayName \"Yamaguchi, Y\")\n	(DB_ID 114197)\n	(firstname \"Yuki\")\n	(initial \"Y\")\n	(surname \"Yamaguchi\"))\n\n([GK_114751] of  ReferenceMolecule\n\n	(_displayName \"Zinc [ChEBI:29105]\")\n	(crossReference\n		[GK_29427]\n		[GK_183308])\n	(DB_ID 114751)\n	(formula \"Zn\")\n	(identifier \"29105\")\n	(name\n		\"Zinc\"\n		\"Zn2+\"\n		\"Zn(II)\")\n	(referenceDatabase [GK_114984]))\n\n([GK_114768] of  ReferenceMolecule\n\n	(_displayName \"Calcium [ChEBI:29108]\")\n	(crossReference\n		[GK_29497]\n		[GK_183728])\n	(DB_ID 114768)\n	(identifier \"29108\")\n	(name\n		\"Calcium\"\n		\"Ca2+\")\n	(referenceDatabase [GK_114984]))\n\n([GK_114961] of  ReferenceGroup\n\n	(_displayName \"Double-stranded DNA\")\n	(crossReference [GK_30151])\n	(DB_ID 114961)\n	(identifier \"4705\")\n	(name \"Double-stranded DNA\")\n	(referenceDatabase [GK_114984]))\n\n([GK_114984] of  ReferenceDatabase\n\n	(_displayName \"ChEBI\")\n	(accessUrl \"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:###ID###\")\n	(DB_ID 114984)\n	(name \"ChEBI\")\n	(url \"http://www.ebi.ac.uk/chebi\"))\n\n([GK_1225400] of  StableIdentifier\n\n	(_displayName \"REACT_27636.3\")\n	(DB_ID 1225400)\n	(identifier \"REACT_27636\")\n	(identifierVersion \"3\"))\n\n([GK_1237360] of  Person\n\n	(_displayName \"Lu, M\")\n	(DB_ID 1237360)\n	(firstname \"Miao\")\n	(initial \"M\")\n	(surname \"Lu\"))\n\n([GK_1237368] of  Person\n\n	(_displayName \"Wu, H\")\n	(DB_ID 1237368)\n	(firstname \"Hao\")\n	(initial \"H\")\n	(surname \"Wu\"))\n\n([GK_1237372] of  Person\n\n	(_displayName \"Yin, Q\")\n	(DB_ID 1237372)\n	(firstname \"Qian\")\n	(initial \"Q\")\n	(surname \"Yin\"))\n\n([GK_1247556] of  Affiliation\n\n	(_displayName \"La Trobe University, Institute for Molecular Science\")\n	(address \"Victoria AU\")\n	(DB_ID 1247556)\n	(name \"La Trobe University, Institute for Molecular Science\"))\n\n([GK_1247689] of  ModifiedResidue\n\n	(_displayName \"L-proline residue at 30\")\n	(coordinate 30)\n	(DB_ID 1247689)\n	(psiMod [GK_611069])\n	(referenceSequence [GK_65462]))\n\n([GK_1247690] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Alpha-synuclein (A30P) [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 1247690)\n	(endCoordinate -1)\n	(hasModifiedResidue [GK_1247689])\n	(name\n		\"Alpha-synuclein (A30P)\"\n		\"SNCA\"\n		\"SYUA_HUMAN\"\n		\"SNCA\")\n	(referenceEntity [GK_65462])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357932])\n	(startCoordinate 1))\n\n([GK_1247691] of  LiteratureReference\n\n	(_displayName \"Amyloid fibril protein related to prealbumin in familial amyloidotic polyneuropathy\")\n	(author\n		[GK_1247754]\n		[GK_1247741]\n		[GK_1247799])\n	(DB_ID 1247691)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"4499-503\")\n	(pubMedIdentifier 279930)\n	(title \"Amyloid fibril protein related to prealbumin in familial amyloidotic polyneuropathy\")\n	(volume 75)\n	(year 1978))\n\n([GK_1247692] of  Person\n\n	(_displayName \"Canneva, F\")\n	(DB_ID 1247692)\n	(initial \"F\")\n	(surname \"Canneva\"))\n\n([GK_1247693] of  Person\n\n	(_displayName \"Gage, FH\")\n	(DB_ID 1247693)\n	(initial \"FH\")\n	(surname \"Gage\"))\n\n([GK_1247694] of  Person\n\n	(_displayName \"Jarrett, JT\")\n	(DB_ID 1247694)\n	(initial \"JT\")\n	(surname \"Jarrett\"))\n\n([GK_1247695] of  Person\n\n	(_displayName \"Soragni, A\")\n	(DB_ID 1247695)\n	(initial \"A\")\n	(surname \"Soragni\"))\n\n([GK_1247696] of  LiteratureReference\n\n	(_displayName \"Amyloid beta-protein assembly and Alzheimer disease\")\n	(author\n		[GK_1247749]\n		[GK_1247783]\n		[GK_1247785]\n		[GK_75542])\n	(DB_ID 1247696)\n	(journal \"J Biol Chem\")\n	(pages \"4749-53\")\n	(pubMedIdentifier 18845536)\n	(title \"Amyloid beta-protein assembly and Alzheimer disease\")\n	(volume 284)\n	(year 2009))\n\n([GK_1247697] of  LiteratureReference\n\n	(_displayName \"Serum levels of beta 2-microglobulin as a new form of amyloid protein in patients undergoing long-term hemodialysis\")\n	(author\n		[GK_977120]\n		[GK_1247744]\n		[GK_193525]\n		[GK_1247756])\n	(DB_ID 1247697)\n	(journal \"N Engl J Med\")\n	(pages \"585-6\")\n	(pubMedIdentifier 3080684)\n	(title \"Serum levels of beta 2-microglobulin as a new form of amyloid protein in patients undergoing long-term hemodialysis\")\n	(volume 314)\n	(year 1986))\n\n([GK_1247698] of  Person\n\n	(_displayName \"Scott, K\")\n	(DB_ID 1247698)\n	(initial \"K\")\n	(surname \"Scott\"))\n\n([GK_1247699] of  Person\n\n	(_displayName \"Hori, S\")\n	(DB_ID 1247699)\n	(initial \"S\")\n	(surname \"Hori\"))\n\n([GK_1247700] of  Person\n\n	(_displayName \"Sievers, SA\")\n	(DB_ID 1247700)\n	(initial \"SA\")\n	(surname \"Sievers\"))\n\n([GK_1247701] of  Person\n\n	(_displayName \"Stevens, FJ\")\n	(DB_ID 1247701)\n	(initial \"FJ\")\n	(surname \"Stevens\"))\n\n([GK_1247702] of  LiteratureReference\n\n	(_displayName \"Immunoglobulin heavy-chain-associated amyloidosis\")\n	(author\n		[GK_207287]\n		[GK_976922]\n		[GK_976858])\n	(DB_ID 1247702)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"6542-6\")\n	(pubMedIdentifier 2118650)\n	(title \"Immunoglobulin heavy-chain-associated amyloidosis\")\n	(volume 87)\n	(year 1990))\n\n([GK_1247703] of  LiteratureReference\n\n	(_displayName \"Tau oligomers and aggregation in Alzheimer\'s disease\")\n	(author\n		[GK_1247775]\n		[GK_1247731]\n		[GK_1247746]\n		[GK_1247710]\n		[GK_1247780])\n	(DB_ID 1247703)\n	(journal \"J Neurochem\")\n	(pages \"1353-67\")\n	(pubMedIdentifier 19943854)\n	(title \"Tau oligomers and aggregation in Alzheimer\'s disease\")\n	(volume 112)\n	(year 2010))\n\n([GK_1247704] of  URL\n\n	(_displayName \"http://www.alzforum.org/new/detailprint.asp?id=2276\")\n	(author\n		[GK_1247742]\n		[GK_1247705])\n	(DB_ID 1247704)\n	(title \"Alzheimer Research Forum\")\n	(uniformResourceLocator \"http://www.alzforum.org/new/detailprint.asp?id=2276\"))\n\n([GK_1247705] of  Person\n\n	(_displayName \"Strobel, G\")\n	(affiliation [GK_1247728])\n	(DB_ID 1247705)\n	(firstname \"Gabrielle\")\n	(initial \"G\")\n	(surname \"Strobel\"))\n\n([GK_1247706] of  Person\n\n	(_displayName \"Wallace, A\")\n	(DB_ID 1247706)\n	(initial \"A\")\n	(surname \"Wallace\"))\n\n([GK_1247707] of  DefinedSet\n\n	(_displayName \"[Hsa] Alpha-synuclein [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 1247707)\n	(hasMember\n		[GK_976954]\n		[GK_1247690])\n	(name \"Alpha-synuclein\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358302]))\n\n([GK_1247708] of  Person\n\n	(_displayName \"Riekel, C\")\n	(DB_ID 1247708)\n	(initial \"C\")\n	(surname \"Riekel\"))\n\n([GK_1247709] of  LiteratureReference\n\n	(_displayName \"Localized amyloidosis of the seminal vesicle: identification of lactoferrin immunoreactivity in the amyloid material\")\n	(author\n		[GK_375875]\n		[GK_1247752]\n		[GK_1247699])\n	(DB_ID 1247709)\n	(journal \"Pathol Int\")\n	(pages \"491-7\")\n	(pubMedIdentifier 8870004)\n	(title \"Localized amyloidosis of the seminal vesicle: identification of lactoferrin immunoreactivity in the amyloid material\")\n	(volume 46)\n	(year 1996))\n\n([GK_1247710] of  Person\n\n	(_displayName \"De Anda-Hern√É¬°ndez, MA\")\n	(DB_ID 1247710)\n	(initial \"MA\")\n	(surname \"De Anda-Hern√É¬°ndez\"))\n\n([GK_1247711] of  LiteratureReference\n\n	(_displayName \"Isolation and partial characterization of SAA-an amyloid-related protein from human serum\")\n	(author\n		[GK_1247750]\n		[GK_1247796]\n		[GK_976928]\n		[GK_1247764])\n	(DB_ID 1247711)\n	(journal \"J Immunol\")\n	(pages \"1415-8\")\n	(pubMedIdentifier 1270801)\n	(title \"Isolation and partial characterization of SAA-an amyloid-related protein from human serum\")\n	(volume 116)\n	(year 1976))\n\n([GK_1247712] of  Person\n\n	(_displayName \"Kennel, SJ\")\n	(DB_ID 1247712)\n	(initial \"SJ\")\n	(surname \"Kennel\"))\n\n([GK_1247713] of  Person\n\n	(_displayName \"James, S\")\n	(DB_ID 1247713)\n	(initial \"S\")\n	(surname \"James\"))\n\n([GK_1247714] of  Person\n\n	(_displayName \"Sambashivan, S\")\n	(DB_ID 1247714)\n	(initial \"S\")\n	(surname \"Sambashivan\"))\n\n([GK_1247715] of  LiteratureReference\n\n	(_displayName \"Transgenic mice as a model of pre-clinical Alzheimer\'s disease\")\n	(author\n		[GK_1247778]\n		[GK_1247769]\n		[GK_1247751]\n		[GK_1247692]\n		[GK_1247725]\n		[GK_1247723]\n		[GK_1247800]\n		[GK_1247727]\n		[GK_1247732]\n		[GK_1247738]\n		[GK_1247761]\n		[GK_212143]\n		[GK_1247789])\n	(DB_ID 1247715)\n	(journal \"Curr Alzheimer Res\")\n	(pages \"4-23\")\n	(pubMedIdentifier 21143159)\n	(title \"Transgenic mice as a model of pre-clinical Alzheimer\'s disease\")\n	(volume 8)\n	(year 2011))\n\n([GK_1247716] of  Person\n\n	(_displayName \"Kestler, DP\")\n	(DB_ID 1247716)\n	(initial \"DP\")\n	(surname \"Kestler\"))\n\n([GK_1247717] of  LiteratureReference\n\n	(_displayName \"Odontogenic ameloblast-associated protein nature of the amyloid found in calcifying epithelial odontogenic tumors and unerupted tooth follicles\")\n	(author\n		[GK_976849]\n		[GK_1247716]\n		[GK_1247798]\n		[GK_140991]\n		[GK_1247792]\n		[GK_1247712]\n		[GK_1247807]\n		[GK_517749]\n		[GK_976922]\n		[GK_976858])\n	(DB_ID 1247717)\n	(journal \"Amyloid\")\n	(pages \"89-95\")\n	(pubMedIdentifier 18484335)\n	(title \"Odontogenic ameloblast-associated protein nature of the amyloid found in calcifying epithelial odontogenic tumors and unerupted tooth follicles\")\n	(volume 15)\n	(year 2008))\n\n([GK_1247718] of  Person\n\n	(_displayName \"Myatt, EA\")\n	(DB_ID 1247718)\n	(initial \"EA\")\n	(surname \"Myatt\"))\n\n([GK_1247719] of  LiteratureReference\n\n	(_displayName \"In vivo demonstration that {alpha}-synuclein oligomers are toxic\")\n	(author\n		[GK_1247788]\n		[GK_1247790]\n		[GK_1247806]\n		[GK_1247765]\n		[GK_1247768]\n		[GK_437238]\n		[GK_1247804]\n		[GK_1247797]\n		[GK_1247726]\n		[GK_1247773]\n		[GK_1247743]\n		[GK_1247695]\n		[GK_1247747]\n		[GK_1247770]\n		[GK_1247767]\n		[GK_1247733]\n		[GK_1247794]\n		[GK_1247693]\n		[GK_1247772])\n	(DB_ID 1247719)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"4194-9\")\n	(pubMedIdentifier 21325059)\n	(title \"In vivo demonstration that {alpha}-synuclein oligomers are toxic\")\n	(volume 108)\n	(year 2011))\n\n([GK_1247720] of  Person\n\n	(_displayName \"Ivanova, MI\")\n	(DB_ID 1247720)\n	(initial \"MI\")\n	(surname \"Ivanova\"))\n\n([GK_1247721] of  Person\n\n	(_displayName \"Schiffer, M\")\n	(DB_ID 1247721)\n	(initial \"M\")\n	(surname \"Schiffer\"))\n\n([GK_1247722] of  LiteratureReference\n\n	(_displayName \"Amyloid fibril protein nomenclature: 2010 recommendations from the nomenclature committee of the International Society of Amyloidosis\")\n	(author\n		[GK_1247763]\n		[GK_976904]\n		[GK_1247816]\n		[GK_207482]\n		[GK_1247730]\n		[GK_976872]\n		[GK_976884])\n	(DB_ID 1247722)\n	(journal \"Amyloid\")\n	(pages \"101-4\")\n	(pubMedIdentifier 21039326)\n	(title \"Amyloid fibril protein nomenclature: 2010 recommendations from the nomenclature committee of the International Society of Amyloidosis\")\n	(volume 17)\n	(year 2010))\n\n([GK_1247723] of  Person\n\n	(_displayName \"Arvanitis, DN\")\n	(DB_ID 1247723)\n	(initial \"DN\")\n	(surname \"Arvanitis\"))\n\n([GK_1247724] of  LiteratureReference\n\n	(_displayName \"Leukocyte chemotactic factor 2: A novel renal amyloid protein\")\n	(author\n		[GK_976904]\n		[GK_1247713]\n		[GK_1247698]\n		[GK_976874]\n		[GK_976932])\n	(DB_ID 1247724)\n	(journal \"Kidney Int\")\n	(pages \"218-22\")\n	(pubMedIdentifier 18449172)\n	(title \"Leukocyte chemotactic factor 2: A novel renal amyloid protein\")\n	(volume 74)\n	(year 2008))\n\n([GK_1247725] of  Person\n\n	(_displayName \"Allard, S\")\n	(DB_ID 1247725)\n	(initial \"S\")\n	(surname \"Allard\"))\n\n([GK_1247726] of  Person\n\n	(_displayName \"Vilar, M\")\n	(DB_ID 1247726)\n	(initial \"M\")\n	(surname \"Vilar\"))\n\n([GK_1247727] of  Person\n\n	(_displayName \"Houle, D\")\n	(DB_ID 1247727)\n	(initial \"D\")\n	(surname \"Houle\"))\n\n([GK_1247728] of  Affiliation\n\n	(_displayName \"Alzheimer Research Foundation\")\n	(DB_ID 1247728)\n	(name \"Alzheimer Research Foundation\"))\n\n([GK_1247729] of  Person\n\n	(_displayName \"Arbustini, E\")\n	(DB_ID 1247729)\n	(initial \"E\")\n	(surname \"Arbustini\"))\n\n([GK_1247730] of  Person\n\n	(_displayName \"Merlini, G\")\n	(DB_ID 1247730)\n	(initial \"G\")\n	(surname \"Merlini\"))\n\n([GK_1247731] of  Person\n\n	(_displayName \"Lira-De Le√É¬≥n, KI\")\n	(DB_ID 1247731)\n	(initial \"KI\")\n	(surname \"Lira-De Le√É¬≥n\"))\n\n([GK_1247732] of  Person\n\n	(_displayName \"Ducatenzeiler, A\")\n	(DB_ID 1247732)\n	(initial \"A\")\n	(surname \"Ducatenzeiler\"))\n\n([GK_1247733] of  Person\n\n	(_displayName \"Pham, E\")\n	(DB_ID 1247733)\n	(initial \"E\")\n	(surname \"Pham\"))\n\n([GK_1247734] of  Affiliation\n\n	(_displayName \"National Association of Science Writers\")\n	(DB_ID 1247734)\n	(name \"National Association of Science Writers\"))\n\n([GK_1247735] of  Person\n\n	(_displayName \"Grasso, M\")\n	(DB_ID 1247735)\n	(initial \"M\")\n	(surname \"Grasso\"))\n\n([GK_1247736] of  Person\n\n	(_displayName \"Rubenstein, AH\")\n	(DB_ID 1247736)\n	(initial \"AH\")\n	(surname \"Rubenstein\"))\n\n([GK_1247737] of  Person\n\n	(_displayName \"Lansbury PT, Jr\")\n	(DB_ID 1247737)\n	(initial \"Jr\")\n	(surname \"Lansbury PT\"))\n\n([GK_1247738] of  Person\n\n	(_displayName \"Klein, WL\")\n	(DB_ID 1247738)\n	(initial \"WL\")\n	(surname \"Klein\"))\n\n([GK_1247739] of  Person\n\n	(_displayName \"Thompson, JL\")\n	(DB_ID 1247739)\n	(initial \"JL\")\n	(surname \"Thompson\"))\n\n([GK_1247740] of  Person\n\n	(_displayName \"Madsen, A√ÉÀú\")\n	(DB_ID 1247740)\n	(initial \"A√ÉÀú\")\n	(surname \"Madsen\"))\n\n([GK_1247741] of  Person\n\n	(_displayName \"Figueira, AS\")\n	(DB_ID 1247741)\n	(initial \"AS\")\n	(surname \"Figueira\"))\n\n([GK_1247742] of  Person\n\n	(_displayName \"Dance, A\")\n	(affiliation [GK_1247734])\n	(DB_ID 1247742)\n	(eMailAddress \"adance@nasw.org\")\n	(firstname \"Amber\")\n	(initial \"A\")\n	(surname \"Dance\"))\n\n([GK_1247743] of  Person\n\n	(_displayName \"Tzitzilonis, C\")\n	(DB_ID 1247743)\n	(initial \"C\")\n	(surname \"Tzitzilonis\"))\n\n([GK_1247744] of  Person\n\n	(_displayName \"Homma, N\")\n	(DB_ID 1247744)\n	(initial \"N\")\n	(surname \"Homma\"))\n\n([GK_1247745] of  LiteratureReference\n\n	(_displayName \"Structural and kinetic features of amyloid beta-protein fibrillogenesis\")\n	(author [GK_75542])\n	(DB_ID 1247745)\n	(journal \"Amyloid\")\n	(pages \"121-42\")\n	(pubMedIdentifier 9686307)\n	(title \"Structural and kinetic features of amyloid beta-protein fibrillogenesis\")\n	(volume 5)\n	(year 1998))\n\n([GK_1247746] of  Person\n\n	(_displayName \"Campos-Pe√É¬±a, V\")\n	(DB_ID 1247746)\n	(initial \"V\")\n	(surname \"Campos-Pe√É¬±a\"))\n\n([GK_1247747] of  Person\n\n	(_displayName \"Jessberger, S\")\n	(DB_ID 1247747)\n	(initial \"S\")\n	(surname \"Jessberger\"))\n\n([GK_1247748] of  Person\n\n	(_displayName \"Vigan√É¬≤, M\")\n	(DB_ID 1247748)\n	(initial \"M\")\n	(surname \"Vigan√É¬≤\"))\n\n([GK_1247749] of  Person\n\n	(_displayName \"Roychaudhuri, R\")\n	(DB_ID 1247749)\n	(initial \"R\")\n	(surname \"Roychaudhuri\"))\n\n([GK_1247750] of  Person\n\n	(_displayName \"Rosenthal, CJ\")\n	(DB_ID 1247750)\n	(initial \"CJ\")\n	(surname \"Rosenthal\"))\n\n([GK_1247751] of  Person\n\n	(_displayName \"Leon, WC\")\n	(DB_ID 1247751)\n	(initial \"WC\")\n	(surname \"Leon\"))\n\n([GK_1247752] of  Person\n\n	(_displayName \"Serizawa, A\")\n	(DB_ID 1247752)\n	(initial \"A\")\n	(surname \"Serizawa\"))\n\n([GK_1247753] of  Person\n\n	(_displayName \"McFarlane, HT\")\n	(DB_ID 1247753)\n	(initial \"HT\")\n	(surname \"McFarlane\"))\n\n([GK_1247754] of  Person\n\n	(_displayName \"Costa, PP\")\n	(DB_ID 1247754)\n	(initial \"PP\")\n	(surname \"Costa\"))\n\n([GK_1247755] of  Person\n\n	(_displayName \"Cotman, CW\")\n	(DB_ID 1247755)\n	(initial \"CW\")\n	(surname \"Cotman\"))\n\n([GK_1247756] of  Person\n\n	(_displayName \"Arakawa, M\")\n	(DB_ID 1247756)\n	(initial \"M\")\n	(surname \"Arakawa\"))\n\n([GK_1247757] of  LiteratureReference\n\n	(_displayName \"The carboxy terminus of the beta amyloid protein is critical for the seeding of amyloid formation: implications for the pathogenesis of Alzheimer\'s disease\")\n	(author\n		[GK_1247694]\n		[GK_1247814]\n		[GK_1247737])\n	(DB_ID 1247757)\n	(journal \"Biochemistry\")\n	(pages \"4693-7\")\n	(pubMedIdentifier 8490014)\n	(title \"The carboxy terminus of the beta amyloid protein is critical for the seeding of amyloid formation: implications for the pathogenesis of Alzheimer\'s disease\")\n	(volume 32)\n	(year 1993))\n\n([GK_1247758] of  Person\n\n	(_displayName \"Fournier, RE\")\n	(DB_ID 1247758)\n	(initial \"RE\")\n	(surname \"Fournier\"))\n\n([GK_1247759] of  Person\n\n	(_displayName \"Jakes, R\")\n	(DB_ID 1247759)\n	(initial \"R\")\n	(surname \"Jakes\"))\n\n([GK_1247760] of  Person\n\n	(_displayName \"McIntire, TM\")\n	(DB_ID 1247760)\n	(initial \"TM\")\n	(surname \"McIntire\"))\n\n([GK_1247761] of  Person\n\n	(_displayName \"Glabe, CG\")\n	(DB_ID 1247761)\n	(initial \"CG\")\n	(surname \"Glabe\"))\n\n([GK_1247762] of  LiteratureReference\n\n	(_displayName \"Atomic structures of amyloid cross-beta spines reveal varied steric zippers\")\n	(author\n		[GK_207642]\n		[GK_1247714]\n		[GK_977146]\n		[GK_1247720]\n		[GK_1247700]\n		[GK_1247779]\n		[GK_1247776]\n		[GK_1247793]\n		[GK_981768]\n		[GK_1247753]\n		[GK_1247740]\n		[GK_1247708]\n		[GK_977123])\n	(DB_ID 1247762)\n	(journal \"Nature\")\n	(pages \"453-7\")\n	(pubMedIdentifier 17468747)\n	(title \"Atomic structures of amyloid cross-beta spines reveal varied steric zippers\")\n	(volume 447)\n	(year 2007))\n\n([GK_1247763] of  Person\n\n	(_displayName \"Sipe, JD\")\n	(DB_ID 1247763)\n	(initial \"JD\")\n	(surname \"Sipe\"))\n\n([GK_1247764] of  Person\n\n	(_displayName \"Greenspan, J\")\n	(DB_ID 1247764)\n	(initial \"J\")\n	(surname \"Greenspan\"))\n\n([GK_1247765] of  Person\n\n	(_displayName \"Desplats, PA\")\n	(DB_ID 1247765)\n	(initial \"PA\")\n	(surname \"Desplats\"))\n\n([GK_1247766] of  LiteratureReference\n\n	(_displayName \"Islet amyloid polypeptide: pinpointing amino acid residues linked to amyloid fibril formation\")\n	(author\n		[GK_976884]\n		[GK_976986]\n		[GK_976995]\n		[GK_374884]\n		[GK_452587])\n	(DB_ID 1247766)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"5036-40\")\n	(pubMedIdentifier 2195544)\n	(title \"Islet amyloid polypeptide: pinpointing amino acid residues linked to amyloid fibril formation\")\n	(volume 87)\n	(year 1990))\n\n([GK_1247767] of  Person\n\n	(_displayName \"Consiglio, A\")\n	(DB_ID 1247767)\n	(initial \"A\")\n	(surname \"Consiglio\"))\n\n([GK_1247768] of  Person\n\n	(_displayName \"Boyer, L\")\n	(DB_ID 1247768)\n	(initial \"L\")\n	(surname \"Boyer\"))\n\n([GK_1247769] of  Person\n\n	(_displayName \"Partridge, V\")\n	(DB_ID 1247769)\n	(initial \"V\")\n	(surname \"Partridge\"))\n\n([GK_1247770] of  Person\n\n	(_displayName \"Mira, H\")\n	(DB_ID 1247770)\n	(initial \"H\")\n	(surname \"Mira\"))\n\n([GK_1247771] of  Person\n\n	(_displayName \"Wharton, J\")\n	(DB_ID 1247771)\n	(initial \"J\")\n	(surname \"Wharton\"))\n\n([GK_1247772] of  Person\n\n	(_displayName \"Riek, R\")\n	(DB_ID 1247772)\n	(initial \"R\")\n	(surname \"Riek\"))\n\n([GK_1247773] of  Person\n\n	(_displayName \"Campioni, S\")\n	(DB_ID 1247773)\n	(initial \"S\")\n	(surname \"Campioni\"))\n\n([GK_1247774] of  Person\n\n	(_displayName \"Needleman, P\")\n	(DB_ID 1247774)\n	(initial \"P\")\n	(surname \"Needleman\"))\n\n([GK_1247775] of  Person\n\n	(_displayName \"Meraz-R√É¬≠os, MA\")\n	(DB_ID 1247775)\n	(initial \"MA\")\n	(surname \"Meraz-R√É¬≠os\"))\n\n([GK_1247776] of  Person\n\n	(_displayName \"Thompson, MJ\")\n	(DB_ID 1247776)\n	(initial \"MJ\")\n	(surname \"Thompson\"))\n\n([GK_1247777] of  Person\n\n	(_displayName \"Milton, SC\")\n	(DB_ID 1247777)\n	(initial \"SC\")\n	(surname \"Milton\"))\n\n([GK_1247778] of  Person\n\n	(_displayName \"Ferretti, MT\")\n	(DB_ID 1247778)\n	(initial \"MT\")\n	(surname \"Ferretti\"))\n\n([GK_1247779] of  Person\n\n	(_displayName \"Apostol, MI\")\n	(DB_ID 1247779)\n	(initial \"MI\")\n	(surname \"Apostol\"))\n\n([GK_1247780] of  Person\n\n	(_displayName \"Mena-L√É¬≥pez, R\")\n	(DB_ID 1247780)\n	(initial \"R\")\n	(surname \"Mena-L√É¬≥pez\"))\n\n([GK_1247781] of  LiteratureReference\n\n	(_displayName \"Atrial amyloid deposits in the failing human heart display both atrial and brain natriuretic peptide-like immunoreactivity\")\n	(author\n		[GK_1247795]\n		[GK_1247771]\n		[GK_1247729]\n		[GK_1247735]\n		[GK_1247787]\n		[GK_1247774]\n		[GK_1247748]\n		[GK_388867])\n	(DB_ID 1247781)\n	(journal \"J Pathol\")\n	(pages \"235-41\")\n	(pubMedIdentifier 1837051)\n	(title \"Atrial amyloid deposits in the failing human heart display both atrial and brain natriuretic peptide-like immunoreactivity\")\n	(volume 165)\n	(year 1991))\n\n([GK_1247782] of  Person\n\n	(_displayName \"Cohn, VH\")\n	(DB_ID 1247782)\n	(initial \"VH\")\n	(surname \"Cohn\"))\n\n([GK_1247783] of  Person\n\n	(_displayName \"Yang, M\")\n	(DB_ID 1247783)\n	(initial \"M\")\n	(surname \"Yang\"))\n\n([GK_1247784] of  LiteratureReference\n\n	(_displayName \"Assignment of the human and mouse prion protein genes to homologous chromosomes\")\n	(author\n		[GK_75583]\n		[GK_75643]\n		[GK_1247782]\n		[GK_1247758]\n		[GK_75642]\n		[GK_75579]\n		[GK_480491]\n		[GK_1247810]\n		[GK_1247786]\n		[GK_75582])\n	(DB_ID 1247784)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"7358-62\")\n	(pubMedIdentifier 3094007)\n	(title \"Assignment of the human and mouse prion protein genes to homologous chromosomes\")\n	(volume 83)\n	(year 1986))\n\n([GK_1247785] of  Person\n\n	(_displayName \"Hoshi, MM\")\n	(DB_ID 1247785)\n	(initial \"MM\")\n	(surname \"Hoshi\"))\n\n([GK_1247786] of  Person\n\n	(_displayName \"Lucero, M\")\n	(DB_ID 1247786)\n	(initial \"M\")\n	(surname \"Lucero\"))\n\n([GK_1247787] of  Person\n\n	(_displayName \"Diegoli, M\")\n	(DB_ID 1247787)\n	(initial \"M\")\n	(surname \"Diegoli\"))\n\n([GK_1247788] of  Person\n\n	(_displayName \"Winner, B\")\n	(DB_ID 1247788)\n	(initial \"B\")\n	(surname \"Winner\"))\n\n([GK_1247789] of  Person\n\n	(_displayName \"Cuello, AC\")\n	(DB_ID 1247789)\n	(initial \"AC\")\n	(surname \"Cuello\"))\n\n([GK_1247790] of  Person\n\n	(_displayName \"Jappelli, R\")\n	(DB_ID 1247790)\n	(initial \"R\")\n	(surname \"Jappelli\"))\n\n([GK_1247791] of  LiteratureReference\n\n	(_displayName \"Pathogenic potential of human monoclonal immunoglobulin light chains: relationship of in vitro aggregation to in vivo organ deposition\")\n	(author\n		[GK_1247718]\n		[GK_1247803]\n		[GK_976922]\n		[GK_976858]\n		[GK_1247721]\n		[GK_1247701])\n	(DB_ID 1247791)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"3034-8\")\n	(pubMedIdentifier 8159701)\n	(title \"Pathogenic potential of human monoclonal immunoglobulin light chains: relationship of in vitro aggregation to in vivo organ deposition\")\n	(volume 91)\n	(year 1994))\n\n([GK_1247792] of  Person\n\n	(_displayName \"Macy, SD\")\n	(DB_ID 1247792)\n	(initial \"SD\")\n	(surname \"Macy\"))\n\n([GK_1247793] of  Person\n\n	(_displayName \"Balbirnie, M\")\n	(DB_ID 1247793)\n	(initial \"M\")\n	(surname \"Balbirnie\"))\n\n([GK_1247794] of  Person\n\n	(_displayName \"Masliah, E\")\n	(DB_ID 1247794)\n	(initial \"E\")\n	(surname \"Masliah\"))\n\n([GK_1247795] of  Person\n\n	(_displayName \"Pucci, A\")\n	(DB_ID 1247795)\n	(initial \"A\")\n	(surname \"Pucci\"))\n\n([GK_1247796] of  Person\n\n	(_displayName \"Franklin, EC\")\n	(DB_ID 1247796)\n	(initial \"EC\")\n	(surname \"Franklin\"))\n\n([GK_1247797] of  Person\n\n	(_displayName \"Loher, T\")\n	(DB_ID 1247797)\n	(initial \"T\")\n	(surname \"Loher\"))\n\n([GK_1247798] of  Person\n\n	(_displayName \"Foster, JS\")\n	(DB_ID 1247798)\n	(initial \"JS\")\n	(surname \"Foster\"))\n\n([GK_1247799] of  Person\n\n	(_displayName \"Bravo, FR\")\n	(DB_ID 1247799)\n	(initial \"FR\")\n	(surname \"Bravo\"))\n\n([GK_1247800] of  Person\n\n	(_displayName \"Vercauteren, F\")\n	(DB_ID 1247800)\n	(initial \"F\")\n	(surname \"Vercauteren\"))\n\n([GK_1247801] of  LiteratureReference\n\n	(_displayName \"An islet amyloid peptide is derived from an 89-amino acid precursor by proteolytic processing\")\n	(author\n		[GK_420066]\n		[GK_70121]\n		[GK_1247808]\n		[GK_1247736]\n		[GK_264348])\n	(DB_ID 1247801)\n	(journal \"J Biol Chem\")\n	(pages \"17243-6\")\n	(pubMedIdentifier 3053705)\n	(title \"An islet amyloid peptide is derived from an 89-amino acid precursor by proteolytic processing\")\n	(volume 263)\n	(year 1988))\n\n([GK_1247802] of  Person\n\n	(_displayName \"Kayed, R\")\n	(DB_ID 1247802)\n	(initial \"R\")\n	(surname \"Kayed\"))\n\n([GK_1247803] of  Person\n\n	(_displayName \"Westholm, FA\")\n	(DB_ID 1247803)\n	(initial \"FA\")\n	(surname \"Westholm\"))\n\n([GK_1247804] of  Person\n\n	(_displayName \"Hetzer, C\")\n	(DB_ID 1247804)\n	(initial \"C\")\n	(surname \"Hetzer\"))\n\n([GK_1247805] of  LiteratureReference\n\n	(_displayName \"Amyloid Oligomer Structures and Toxicity\")\n	(author [GK_1247761])\n	(DB_ID 1247805)\n	(journal \"Open Biol J\")\n	(pages \"222-227\")\n	(title \"Amyloid Oligomer Structures and Toxicity\")\n	(year 2009))\n\n([GK_1247806] of  Person\n\n	(_displayName \"Maji, SK\")\n	(DB_ID 1247806)\n	(initial \"SK\")\n	(surname \"Maji\"))\n\n([GK_1247807] of  Person\n\n	(_displayName \"Carlson, ER\")\n	(DB_ID 1247807)\n	(initial \"ER\")\n	(surname \"Carlson\"))\n\n([GK_1247808] of  Person\n\n	(_displayName \"Sample, C\")\n	(DB_ID 1247808)\n	(initial \"C\")\n	(surname \"Sample\"))\n\n([GK_1247809] of  LiteratureReference\n\n	(_displayName \"Effects of the mutations Ala30 to Pro and Ala53 to Thr on the physical and morphological properties of alpha-synuclein protein implicated in Parkinson\'s disease\")\n	(author\n		[GK_976894]\n		[GK_1247759]\n		[GK_1247811]\n		[GK_1247706])\n	(DB_ID 1247809)\n	(journal \"FEBS Lett\")\n	(pages \"67-70\")\n	(pubMedIdentifier 9862427)\n	(title \"Effects of the mutations Ala30 to Pro and Ala53 to Thr on the physical and morphological properties of alpha-synuclein protein implicated in Parkinson\'s disease\")\n	(volume 440)\n	(year 1998))\n\n([GK_1247810] of  Person\n\n	(_displayName \"Blatt, C\")\n	(DB_ID 1247810)\n	(initial \"C\")\n	(surname \"Blatt\"))\n\n([GK_1247811] of  Person\n\n	(_displayName \"Curran, MD\")\n	(DB_ID 1247811)\n	(initial \"MD\")\n	(surname \"Curran\"))\n\n([GK_1247812] of  LiteratureReference\n\n	(_displayName \"Common structure of soluble amyloid oligomers implies common mechanism of pathogenesis\")\n	(author\n		[GK_1247802]\n		[GK_1247815]\n		[GK_1247739]\n		[GK_1247760]\n		[GK_1247777]\n		[GK_1247755]\n		[GK_1247761])\n	(DB_ID 1247812)\n	(journal \"Science\")\n	(pages \"486-9\")\n	(pubMedIdentifier 12702875)\n	(title \"Common structure of soluble amyloid oligomers implies common mechanism of pathogenesis\")\n	(volume 300)\n	(year 2003))\n\n([GK_1247814] of  Person\n\n	(_displayName \"Berger, EP\")\n	(DB_ID 1247814)\n	(initial \"EP\")\n	(surname \"Berger\"))\n\n([GK_1247815] of  Person\n\n	(_displayName \"Head, E\")\n	(DB_ID 1247815)\n	(initial \"E\")\n	(surname \"Head\"))\n\n([GK_1247816] of  Person\n\n	(_displayName \"Buxbaum, JN\")\n	(DB_ID 1247816)\n	(initial \"JN\")\n	(surname \"Buxbaum\"))\n\n([GK_1247852] of  Polymer\n\n	(_displayName \"[Hsa] Alpha-synucein fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 1247852)\n	(name \"Alpha-synucein fibril\")\n	(repeatedUnit [GK_1247707])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358160]))\n\n([GK_1247882] of  InstanceEdit\n\n	(_displayName \"Perry, G, 2011-04-08\")\n	(author [GK_879404])\n	(dateTime \"2011-04-08 12:41:42\")\n	(DB_ID 1247882))\n\n([GK_1247883] of  Affiliation\n\n	(_displayName \"University of Texas at San Antonio\")\n	(DB_ID 1247883)\n	(name \"University of Texas at San Antonio\"))\n\n([GK_1247884] of  InstanceEdit\n\n	(_displayName \"Jupe, S, 2011-04-08\")\n	(author [GK_391309])\n	(dateTime \"2011-04-08 13:02:35\")\n	(DB_ID 1247884))\n\n([GK_1247963] of  Person\n\n	(_displayName \"Riedl, SJ\")\n	(DB_ID 1247963)\n	(initial \"SJ\")\n	(surname \"Riedl\"))\n\n([GK_1250148] of  Person\n\n	(_displayName \"Ashkenazi, A\")\n	(DB_ID 1250148)\n	(firstname \"Avi\")\n	(initial \"A\")\n	(surname \"Ashkenazi\"))\n\n([GK_1250230] of  Person\n\n	(_displayName \"Schr√É¬∂ter, M\")\n	(DB_ID 1250230)\n	(initial \"M\")\n	(surname \"Schr√É¬∂ter\"))\n\n([GK_1250231] of  LiteratureReference\n\n	(_displayName \"Inhibition of death receptor signals by cellular FLIP\")\n	(author\n		[GK_1250232]\n		[GK_202465]\n		[GK_1250241]\n		[GK_193908]\n		[GK_165955]\n		[GK_1250240]\n		[GK_113970]\n		[GK_1250230]\n		[GK_433999]\n		[GK_1250234]\n		[GK_1250233]\n		[GK_1250237]\n		[GK_77293])\n	(DB_ID 1250231)\n	(journal \"Nature\")\n	(pages \"190-5\")\n	(pubMedIdentifier 9217161)\n	(title \"Inhibition of death receptor signals by cellular FLIP\")\n	(volume 388)\n	(year 1997))\n\n([GK_1250232] of  Person\n\n	(_displayName \"Irmler, M\")\n	(DB_ID 1250232)\n	(initial \"M\")\n	(surname \"Irmler\"))\n\n([GK_1250233] of  Person\n\n	(_displayName \"Rimoldi, D\")\n	(DB_ID 1250233)\n	(initial \"D\")\n	(surname \"Rimoldi\"))\n\n([GK_1250234] of  Person\n\n	(_displayName \"Mattmann, C\")\n	(DB_ID 1250234)\n	(initial \"C\")\n	(surname \"Mattmann\"))\n\n([GK_1250237] of  Person\n\n	(_displayName \"French, LE\")\n	(DB_ID 1250237)\n	(initial \"LE\")\n	(surname \"French\"))\n\n([GK_1250240] of  Person\n\n	(_displayName \"Steiner, V\")\n	(DB_ID 1250240)\n	(initial \"V\")\n	(surname \"Steiner\"))\n\n([GK_1250241] of  Person\n\n	(_displayName \"Hahne, M\")\n	(DB_ID 1250241)\n	(initial \"M\")\n	(surname \"Hahne\"))\n\n([GK_1251963] of  Complex\n\n	(_displayName \"[Hsa] ADAM17 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 1251963)\n	(hasComponent\n		[GK_179878]\n		[GK_158417])\n	(isChimeric FALSE)\n	(name \"ADAM17\")\n	(species [GK_48887])\n	(stableIdentifier [GK_2016988]))\n\n([GK_1251983] of  CatalystActivity\n\n	(_displayName \"[Hsa] metalloendopeptidase activity of ADAM17 [plasma membrane]\")\n	(activity [GK_1791])\n	(DB_ID 1251983)\n	(physicalEntity [GK_1251963]))\n\n([GK_1267754] of  Person\n\n	(_displayName \"Denault, JB\")\n	(DB_ID 1267754)\n	(firstname \"Jean-Bernard\")\n	(initial \"JB\")\n	(surname \"Denault\"))\n\n([GK_1267755] of  Person\n\n	(_displayName \"Deis, C\")\n	(DB_ID 1267755)\n	(firstname \"Cristina\")\n	(initial \"C\")\n	(surname \"Deis\"))\n\n([GK_1267757] of  LiteratureReference\n\n	(_displayName \"Activation of caspases-8 and -10 by FLIP(L)\")\n	(author\n		[GK_141143]\n		[GK_1267755]\n		[GK_1267754]\n		[GK_879189]\n		[GK_141140])\n	(DB_ID 1267757)\n	(journal \"Biochem J\")\n	(pages \"651-7\")\n	(pubMedIdentifier 15209560)\n	(title \"Activation of caspases-8 and -10 by FLIP(L)\")\n	(volume 382)\n	(year 2004))\n\n([GK_1267777] of  Person\n\n	(_displayName \"Yang, X\")\n	(DB_ID 1267777)\n	(firstname \"Xiaolu\")\n	(initial \"X\")\n	(surname \"Yang\"))\n\n([GK_1267783] of  Person\n\n	(_displayName \"Pop, Cristina\")\n	(affiliation [GK_3597631])\n	(DB_ID 1267783)\n	(firstname \"Cristina\")\n	(initial \"C\")\n	(surname \"Pop\"))\n\n([GK_1267784] of  Person\n\n	(_displayName \"Van Raam, BJ\")\n	(DB_ID 1267784)\n	(firstname \"Bram J.\")\n	(initial \"BJ\")\n	(surname \"Van Raam\"))\n\n([GK_1267785] of  Person\n\n	(_displayName \"Oberst, A\")\n	(DB_ID 1267785)\n	(firstname \"Andrew\")\n	(initial \"A\")\n	(surname \"Oberst\"))\n\n([GK_1267786] of  LiteratureReference\n\n	(_displayName \"FLIP(L) induces caspase 8 activity in the absence of interdomain caspase 8 cleavage and alters substrate specificity\")\n	(author\n		[GK_1267783]\n		[GK_1267785]\n		[GK_1267788]\n		[GK_1267784]\n		[GK_1247963]\n		[GK_1267787]\n		[GK_141140])\n	(DB_ID 1267786)\n	(journal \"Biochem J\")\n	(pages \"447-57\")\n	(pubMedIdentifier 21235526)\n	(title \"FLIP(L) induces caspase 8 activity in the absence of interdomain caspase 8 cleavage and alters substrate specificity\")\n	(volume 433)\n	(year 2011))\n\n([GK_1267787] of  Person\n\n	(_displayName \"Green, DR\")\n	(DB_ID 1267787)\n	(firstname \"Douglas R\")\n	(initial \"DR\")\n	(surname \"Green\"))\n\n([GK_1267788] of  Person\n\n	(_displayName \"Drag, M\")\n	(DB_ID 1267788)\n	(firstname \"Marcin\")\n	(initial \"M\")\n	(surname \"Drag\"))\n\n([GK_1267799] of  Person\n\n	(_displayName \"Tremblay, AG\")\n	(DB_ID 1267799)\n	(firstname \"Alexandre G\")\n	(initial \"AG\")\n	(surname \"Tremblay\"))\n\n([GK_1267800] of  LiteratureReference\n\n	(_displayName \"Inducible dimerization and inducible cleavage reveal a requirement for both processes in caspase-8 activation\")\n	(author\n		[GK_1267785]\n		[GK_1267783]\n		[GK_1267799]\n		[GK_1267804]\n		[GK_1267754]\n		[GK_141140]\n		[GK_1267787])\n	(DB_ID 1267800)\n	(journal \"J Biol Chem\")\n	(pages \"16632-42\")\n	(pubMedIdentifier 20308068)\n	(title \"Inducible dimerization and inducible cleavage reveal a requirement for both processes in caspase-8 activation\")\n	(volume 285)\n	(year 2010))\n\n([GK_1267804] of  Person\n\n	(_displayName \"Blais, V\")\n	(DB_ID 1267804)\n	(firstname \"Veronique\")\n	(initial \"V\")\n	(surname \"Blais\"))\n\n([GK_1267806] of  Person\n\n	(_displayName \"Chang, DW\")\n	(DB_ID 1267806)\n	(firstname \"David W.\")\n	(initial \"DW\")\n	(surname \"Chang\"))\n\n([GK_1267807] of  LiteratureReference\n\n	(_displayName \"Interdimer processing mechanism of procaspase-8 activation\")\n	(author\n		[GK_1267806]\n		[GK_1267808]\n		[GK_1267809]\n		[GK_140502]\n		[GK_1267777])\n	(DB_ID 1267807)\n	(journal \"EMBO J\")\n	(pages \"4132-42\")\n	(pubMedIdentifier 12912912)\n	(title \"Interdimer processing mechanism of procaspase-8 activation\")\n	(volume 22)\n	(year 2003))\n\n([GK_1267808] of  Person\n\n	(_displayName \"Xing, Z\")\n	(DB_ID 1267808)\n	(firstname \"Zheng\")\n	(initial \"Z\")\n	(surname \"Xing\"))\n\n([GK_1267809] of  Person\n\n	(_displayName \"Capacio, VL\")\n	(DB_ID 1267809)\n	(firstname \"Vanessa L.\")\n	(initial \"VL\")\n	(surname \"Capacio\"))\n\n([GK_1267836] of  LiteratureReference\n\n	(_displayName \"Structural and biochemical studies on procaspase-8: new insights on initiator caspase activation\")\n	(author\n		[GK_1267840]\n		[GK_1267842]\n		[GK_1267839]\n		[GK_1267841])\n	(DB_ID 1267836)\n	(journal \"Structure\")\n	(pages \"438-48\")\n	(pubMedIdentifier 19278658)\n	(title \"Structural and biochemical studies on procaspase-8: new insights on initiator caspase activation\")\n	(volume 17)\n	(year 2009))\n\n([GK_1267838] of  Person\n\n	(_displayName \"Kataoka, T\")\n	(DB_ID 1267838)\n	(firstname \"Takao\")\n	(initial \"T\")\n	(surname \"Kataoka\"))\n\n([GK_1267839] of  Person\n\n	(_displayName \"Zerbe, O\")\n	(DB_ID 1267839)\n	(firstname \"Oliver\")\n	(initial \"O\")\n	(surname \"Zerbe\"))\n\n([GK_1267840] of  Person\n\n	(_displayName \"Keller, N\")\n	(DB_ID 1267840)\n	(firstname \"Nadine\")\n	(initial \"N\")\n	(surname \"Keller\"))\n\n([GK_1267841] of  Person\n\n	(_displayName \"Gr√É¬ºtter, MG\")\n	(DB_ID 1267841)\n	(firstname \"Markus G.\")\n	(initial \"MG\")\n	(surname \"Gr√É¬ºtter\"))\n\n([GK_1267842] of  Person\n\n	(_displayName \"Mares, J\")\n	(DB_ID 1267842)\n	(firstname \"Jiri\")\n	(initial \"J\")\n	(surname \"Mares\"))\n\n([GK_1302721] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Beta-2-microglobulin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 1302721)\n	(endCoordinate 119)\n	(name \"Beta-2-microglobulin\")\n	(referenceEntity [GK_50591])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357821])\n	(startCoordinate 21))\n\n([GK_1302722] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Kerato-epithelin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 1302722)\n	(endCoordinate 683)\n	(name\n		\"Kerato-epithelin\"\n		\"Transforming growth factor-beta induced protein IG-H3 precursor\"\n		\"BGH3_HUMAN\"\n		\"TGFBI\")\n	(referenceEntity [GK_50810])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358223])\n	(startCoordinate 24))\n\n([GK_1357590] of  StableIdentifier\n\n	(_displayName \"REACT_75858.1\")\n	(DB_ID 1357590)\n	(identifier \"REACT_75858\")\n	(identifierVersion \"1\"))\n\n([GK_1357608] of  StableIdentifier\n\n	(_displayName \"REACT_75876.1\")\n	(DB_ID 1357608)\n	(identifier \"REACT_75876\")\n	(identifierVersion \"1\"))\n\n([GK_1357610] of  StableIdentifier\n\n	(_displayName \"REACT_75878.1\")\n	(DB_ID 1357610)\n	(identifier \"REACT_75878\")\n	(identifierVersion \"1\"))\n\n([GK_1357634] of  StableIdentifier\n\n	(_displayName \"REACT_75902.1\")\n	(DB_ID 1357634)\n	(identifier \"REACT_75902\")\n	(identifierVersion \"1\"))\n\n([GK_1357637] of  StableIdentifier\n\n	(_displayName \"REACT_75905.1\")\n	(DB_ID 1357637)\n	(identifier \"REACT_75905\")\n	(identifierVersion \"1\"))\n\n([GK_1357657] of  StableIdentifier\n\n	(_displayName \"REACT_75925.1\")\n	(DB_ID 1357657)\n	(identifier \"REACT_75925\")\n	(identifierVersion \"1\"))\n\n([GK_1357672] of  StableIdentifier\n\n	(_displayName \"REACT_75940.1\")\n	(DB_ID 1357672)\n	(identifier \"REACT_75940\")\n	(identifierVersion \"1\"))\n\n([GK_1357676] of  StableIdentifier\n\n	(_displayName \"REACT_75944.1\")\n	(DB_ID 1357676)\n	(identifier \"REACT_75944\")\n	(identifierVersion \"1\"))\n\n([GK_1357692] of  StableIdentifier\n\n	(_displayName \"REACT_75960.1\")\n	(DB_ID 1357692)\n	(identifier \"REACT_75960\")\n	(identifierVersion \"1\"))\n\n([GK_1357699] of  StableIdentifier\n\n	(_displayName \"REACT_75967.1\")\n	(DB_ID 1357699)\n	(identifier \"REACT_75967\")\n	(identifierVersion \"1\"))\n\n([GK_1357703] of  StableIdentifier\n\n	(_displayName \"REACT_75971.1\")\n	(DB_ID 1357703)\n	(identifier \"REACT_75971\")\n	(identifierVersion \"1\"))\n\n([GK_1357707] of  StableIdentifier\n\n	(_displayName \"REACT_75975.1\")\n	(DB_ID 1357707)\n	(identifier \"REACT_75975\")\n	(identifierVersion \"1\"))\n\n([GK_1357725] of  StableIdentifier\n\n	(_displayName \"REACT_75993.1\")\n	(DB_ID 1357725)\n	(identifier \"REACT_75993\")\n	(identifierVersion \"1\"))\n\n([GK_1357743] of  StableIdentifier\n\n	(_displayName \"REACT_76011.1\")\n	(DB_ID 1357743)\n	(identifier \"REACT_76011\")\n	(identifierVersion \"1\"))\n\n([GK_1357748] of  StableIdentifier\n\n	(_displayName \"REACT_76016.1\")\n	(DB_ID 1357748)\n	(identifier \"REACT_76016\")\n	(identifierVersion \"1\"))\n\n([GK_1357751] of  StableIdentifier\n\n	(_displayName \"REACT_76019.1\")\n	(DB_ID 1357751)\n	(identifier \"REACT_76019\")\n	(identifierVersion \"1\"))\n\n([GK_1357757] of  StableIdentifier\n\n	(_displayName \"REACT_76025.1\")\n	(DB_ID 1357757)\n	(identifier \"REACT_76025\")\n	(identifierVersion \"1\"))\n\n([GK_1357767] of  StableIdentifier\n\n	(_displayName \"REACT_76035.1\")\n	(DB_ID 1357767)\n	(identifier \"REACT_76035\")\n	(identifierVersion \"1\"))\n\n([GK_1357778] of  StableIdentifier\n\n	(_displayName \"REACT_76046.1\")\n	(DB_ID 1357778)\n	(identifier \"REACT_76046\")\n	(identifierVersion \"1\"))\n\n([GK_1357799] of  StableIdentifier\n\n	(_displayName \"REACT_76067.1\")\n	(DB_ID 1357799)\n	(identifier \"REACT_76067\")\n	(identifierVersion \"1\"))\n\n([GK_1357802] of  StableIdentifier\n\n	(_displayName \"REACT_76070.1\")\n	(DB_ID 1357802)\n	(identifier \"REACT_76070\")\n	(identifierVersion \"1\"))\n\n([GK_1357805] of  StableIdentifier\n\n	(_displayName \"REACT_76073.1\")\n	(DB_ID 1357805)\n	(identifier \"REACT_76073\")\n	(identifierVersion \"1\"))\n\n([GK_1357816] of  StableIdentifier\n\n	(_displayName \"REACT_76084.1\")\n	(DB_ID 1357816)\n	(identifier \"REACT_76084\")\n	(identifierVersion \"1\"))\n\n([GK_1357821] of  StableIdentifier\n\n	(_displayName \"REACT_76089.1\")\n	(DB_ID 1357821)\n	(identifier \"REACT_76089\")\n	(identifierVersion \"1\"))\n\n([GK_1357824] of  StableIdentifier\n\n	(_displayName \"REACT_76092.1\")\n	(DB_ID 1357824)\n	(identifier \"REACT_76092\")\n	(identifierVersion \"1\"))\n\n([GK_1357834] of  StableIdentifier\n\n	(_displayName \"REACT_76102.1\")\n	(DB_ID 1357834)\n	(identifier \"REACT_76102\")\n	(identifierVersion \"1\"))\n\n([GK_1357839] of  StableIdentifier\n\n	(_displayName \"REACT_76107.1\")\n	(DB_ID 1357839)\n	(identifier \"REACT_76107\")\n	(identifierVersion \"1\"))\n\n([GK_1357842] of  StableIdentifier\n\n	(_displayName \"REACT_76110.1\")\n	(DB_ID 1357842)\n	(identifier \"REACT_76110\")\n	(identifierVersion \"1\"))\n\n([GK_1357844] of  StableIdentifier\n\n	(_displayName \"REACT_76112.1\")\n	(DB_ID 1357844)\n	(identifier \"REACT_76112\")\n	(identifierVersion \"1\"))\n\n([GK_1357848] of  StableIdentifier\n\n	(_displayName \"REACT_76116.1\")\n	(DB_ID 1357848)\n	(identifier \"REACT_76116\")\n	(identifierVersion \"1\"))\n\n([GK_1357872] of  StableIdentifier\n\n	(_displayName \"REACT_76140.1\")\n	(DB_ID 1357872)\n	(identifier \"REACT_76140\")\n	(identifierVersion \"1\"))\n\n([GK_1357875] of  StableIdentifier\n\n	(_displayName \"REACT_76143.1\")\n	(DB_ID 1357875)\n	(identifier \"REACT_76143\")\n	(identifierVersion \"1\"))\n\n([GK_1357931] of  StableIdentifier\n\n	(_displayName \"REACT_76199.1\")\n	(DB_ID 1357931)\n	(identifier \"REACT_76199\")\n	(identifierVersion \"1\"))\n\n([GK_1357932] of  StableIdentifier\n\n	(_displayName \"REACT_76200.1\")\n	(DB_ID 1357932)\n	(identifier \"REACT_76200\")\n	(identifierVersion \"1\"))\n\n([GK_1357950] of  StableIdentifier\n\n	(_displayName \"REACT_76218.1\")\n	(DB_ID 1357950)\n	(identifier \"REACT_76218\")\n	(identifierVersion \"1\"))\n\n([GK_1357954] of  StableIdentifier\n\n	(_displayName \"REACT_76222.1\")\n	(DB_ID 1357954)\n	(identifier \"REACT_76222\")\n	(identifierVersion \"1\"))\n\n([GK_1357967] of  StableIdentifier\n\n	(_displayName \"REACT_76235.1\")\n	(DB_ID 1357967)\n	(identifier \"REACT_76235\")\n	(identifierVersion \"1\"))\n\n([GK_1357973] of  StableIdentifier\n\n	(_displayName \"REACT_76241.1\")\n	(DB_ID 1357973)\n	(identifier \"REACT_76241\")\n	(identifierVersion \"1\"))\n\n([GK_1357978] of  StableIdentifier\n\n	(_displayName \"REACT_76246.1\")\n	(DB_ID 1357978)\n	(identifier \"REACT_76246\")\n	(identifierVersion \"1\"))\n\n([GK_1357980] of  StableIdentifier\n\n	(_displayName \"REACT_76248.1\")\n	(DB_ID 1357980)\n	(identifier \"REACT_76248\")\n	(identifierVersion \"1\"))\n\n([GK_1357984] of  StableIdentifier\n\n	(_displayName \"REACT_76252.1\")\n	(DB_ID 1357984)\n	(identifier \"REACT_76252\")\n	(identifierVersion \"1\"))\n\n([GK_1357996] of  StableIdentifier\n\n	(_displayName \"REACT_76264.1\")\n	(DB_ID 1357996)\n	(identifier \"REACT_76264\")\n	(identifierVersion \"1\"))\n\n([GK_1358001] of  StableIdentifier\n\n	(_displayName \"REACT_76269.1\")\n	(DB_ID 1358001)\n	(identifier \"REACT_76269\")\n	(identifierVersion \"1\"))\n\n([GK_1358008] of  StableIdentifier\n\n	(_displayName \"REACT_76276.1\")\n	(DB_ID 1358008)\n	(identifier \"REACT_76276\")\n	(identifierVersion \"1\"))\n\n([GK_1358010] of  StableIdentifier\n\n	(_displayName \"REACT_76278.1\")\n	(DB_ID 1358010)\n	(identifier \"REACT_76278\")\n	(identifierVersion \"1\"))\n\n([GK_1358027] of  StableIdentifier\n\n	(_displayName \"REACT_76295.1\")\n	(DB_ID 1358027)\n	(identifier \"REACT_76295\")\n	(identifierVersion \"1\"))\n\n([GK_1358045] of  StableIdentifier\n\n	(_displayName \"REACT_76313.1\")\n	(DB_ID 1358045)\n	(identifier \"REACT_76313\")\n	(identifierVersion \"1\"))\n\n([GK_1358054] of  StableIdentifier\n\n	(_displayName \"REACT_76322.1\")\n	(DB_ID 1358054)\n	(identifier \"REACT_76322\")\n	(identifierVersion \"1\"))\n\n([GK_1358060] of  StableIdentifier\n\n	(_displayName \"REACT_76328.1\")\n	(DB_ID 1358060)\n	(identifier \"REACT_76328\")\n	(identifierVersion \"1\"))\n\n([GK_1358061] of  StableIdentifier\n\n	(_displayName \"REACT_76329.1\")\n	(DB_ID 1358061)\n	(identifier \"REACT_76329\")\n	(identifierVersion \"1\"))\n\n([GK_1358063] of  StableIdentifier\n\n	(_displayName \"REACT_76331.1\")\n	(DB_ID 1358063)\n	(identifier \"REACT_76331\")\n	(identifierVersion \"1\"))\n\n([GK_1358067] of  StableIdentifier\n\n	(_displayName \"REACT_76335.1\")\n	(DB_ID 1358067)\n	(identifier \"REACT_76335\")\n	(identifierVersion \"1\"))\n\n([GK_1358072] of  StableIdentifier\n\n	(_displayName \"REACT_76340.1\")\n	(DB_ID 1358072)\n	(identifier \"REACT_76340\")\n	(identifierVersion \"1\"))\n\n([GK_1358076] of  StableIdentifier\n\n	(_displayName \"REACT_76344.1\")\n	(DB_ID 1358076)\n	(identifier \"REACT_76344\")\n	(identifierVersion \"1\"))\n\n([GK_1358084] of  StableIdentifier\n\n	(_displayName \"REACT_76352.1\")\n	(DB_ID 1358084)\n	(identifier \"REACT_76352\")\n	(identifierVersion \"1\"))\n\n([GK_1358096] of  StableIdentifier\n\n	(_displayName \"REACT_76364.1\")\n	(DB_ID 1358096)\n	(identifier \"REACT_76364\")\n	(identifierVersion \"1\"))\n\n([GK_1358102] of  StableIdentifier\n\n	(_displayName \"REACT_76370.1\")\n	(DB_ID 1358102)\n	(identifier \"REACT_76370\")\n	(identifierVersion \"1\"))\n\n([GK_1358114] of  StableIdentifier\n\n	(_displayName \"REACT_76382.1\")\n	(DB_ID 1358114)\n	(identifier \"REACT_76382\")\n	(identifierVersion \"1\"))\n\n([GK_1358143] of  StableIdentifier\n\n	(_displayName \"REACT_76411.1\")\n	(DB_ID 1358143)\n	(identifier \"REACT_76411\")\n	(identifierVersion \"1\"))\n\n([GK_1358155] of  StableIdentifier\n\n	(_displayName \"REACT_76423.1\")\n	(DB_ID 1358155)\n	(identifier \"REACT_76423\")\n	(identifierVersion \"1\"))\n\n([GK_1358160] of  StableIdentifier\n\n	(_displayName \"REACT_76428.1\")\n	(DB_ID 1358160)\n	(identifier \"REACT_76428\")\n	(identifierVersion \"1\"))\n\n([GK_1358170] of  StableIdentifier\n\n	(_displayName \"REACT_76438.1\")\n	(DB_ID 1358170)\n	(identifier \"REACT_76438\")\n	(identifierVersion \"1\"))\n\n([GK_1358175] of  StableIdentifier\n\n	(_displayName \"REACT_76443.1\")\n	(DB_ID 1358175)\n	(identifier \"REACT_76443\")\n	(identifierVersion \"1\"))\n\n([GK_1358177] of  StableIdentifier\n\n	(_displayName \"REACT_76445.1\")\n	(DB_ID 1358177)\n	(identifier \"REACT_76445\")\n	(identifierVersion \"1\"))\n\n([GK_1358185] of  StableIdentifier\n\n	(_displayName \"REACT_76453.1\")\n	(DB_ID 1358185)\n	(identifier \"REACT_76453\")\n	(identifierVersion \"1\"))\n\n([GK_1358187] of  StableIdentifier\n\n	(_displayName \"REACT_76455.1\")\n	(DB_ID 1358187)\n	(identifier \"REACT_76455\")\n	(identifierVersion \"1\"))\n\n([GK_1358194] of  StableIdentifier\n\n	(_displayName \"REACT_76462.1\")\n	(DB_ID 1358194)\n	(identifier \"REACT_76462\")\n	(identifierVersion \"1\"))\n\n([GK_1358195] of  StableIdentifier\n\n	(_displayName \"REACT_76463.1\")\n	(DB_ID 1358195)\n	(identifier \"REACT_76463\")\n	(identifierVersion \"1\"))\n\n([GK_1358199] of  StableIdentifier\n\n	(_displayName \"REACT_76467.1\")\n	(DB_ID 1358199)\n	(identifier \"REACT_76467\")\n	(identifierVersion \"1\"))\n\n([GK_1358212] of  StableIdentifier\n\n	(_displayName \"REACT_76480.1\")\n	(DB_ID 1358212)\n	(identifier \"REACT_76480\")\n	(identifierVersion \"1\"))\n\n([GK_1358223] of  StableIdentifier\n\n	(_displayName \"REACT_76491.1\")\n	(DB_ID 1358223)\n	(identifier \"REACT_76491\")\n	(identifierVersion \"1\"))\n\n([GK_1358224] of  StableIdentifier\n\n	(_displayName \"REACT_76492.1\")\n	(DB_ID 1358224)\n	(identifier \"REACT_76492\")\n	(identifierVersion \"1\"))\n\n([GK_1358225] of  StableIdentifier\n\n	(_displayName \"REACT_76493.1\")\n	(DB_ID 1358225)\n	(identifier \"REACT_76493\")\n	(identifierVersion \"1\"))\n\n([GK_1358234] of  StableIdentifier\n\n	(_displayName \"REACT_76502.1\")\n	(DB_ID 1358234)\n	(identifier \"REACT_76502\")\n	(identifierVersion \"1\"))\n\n([GK_1358239] of  StableIdentifier\n\n	(_displayName \"REACT_76507.1\")\n	(DB_ID 1358239)\n	(identifier \"REACT_76507\")\n	(identifierVersion \"1\"))\n\n([GK_1358244] of  StableIdentifier\n\n	(_displayName \"REACT_76512.1\")\n	(DB_ID 1358244)\n	(identifier \"REACT_76512\")\n	(identifierVersion \"1\"))\n\n([GK_1358246] of  StableIdentifier\n\n	(_displayName \"REACT_76514.1\")\n	(DB_ID 1358246)\n	(identifier \"REACT_76514\")\n	(identifierVersion \"1\"))\n\n([GK_1358248] of  StableIdentifier\n\n	(_displayName \"REACT_76516.1\")\n	(DB_ID 1358248)\n	(identifier \"REACT_76516\")\n	(identifierVersion \"1\"))\n\n([GK_1358276] of  StableIdentifier\n\n	(_displayName \"REACT_76544.1\")\n	(DB_ID 1358276)\n	(identifier \"REACT_76544\")\n	(identifierVersion \"1\"))\n\n([GK_1358281] of  StableIdentifier\n\n	(_displayName \"REACT_76549.1\")\n	(DB_ID 1358281)\n	(identifier \"REACT_76549\")\n	(identifierVersion \"1\"))\n\n([GK_1358284] of  StableIdentifier\n\n	(_displayName \"REACT_76552.1\")\n	(DB_ID 1358284)\n	(identifier \"REACT_76552\")\n	(identifierVersion \"1\"))\n\n([GK_1358302] of  StableIdentifier\n\n	(_displayName \"REACT_76570.1\")\n	(DB_ID 1358302)\n	(identifier \"REACT_76570\")\n	(identifierVersion \"1\"))\n\n([GK_1358324] of  StableIdentifier\n\n	(_displayName \"REACT_76592.1\")\n	(DB_ID 1358324)\n	(identifier \"REACT_76592\")\n	(identifierVersion \"1\"))\n\n([GK_1358325] of  StableIdentifier\n\n	(_displayName \"REACT_76593.1\")\n	(DB_ID 1358325)\n	(identifier \"REACT_76593\")\n	(identifierVersion \"1\"))\n\n([GK_1358327] of  StableIdentifier\n\n	(_displayName \"REACT_76595.1\")\n	(DB_ID 1358327)\n	(identifier \"REACT_76595\")\n	(identifierVersion \"1\"))\n\n([GK_1358336] of  StableIdentifier\n\n	(_displayName \"REACT_76604.1\")\n	(DB_ID 1358336)\n	(identifier \"REACT_76604\")\n	(identifierVersion \"1\"))\n\n([GK_1358358] of  StableIdentifier\n\n	(_displayName \"REACT_76626.1\")\n	(DB_ID 1358358)\n	(identifier \"REACT_76626\")\n	(identifierVersion \"1\"))\n\n([GK_1358359] of  StableIdentifier\n\n	(_displayName \"REACT_76627.1\")\n	(DB_ID 1358359)\n	(identifier \"REACT_76627\")\n	(identifierVersion \"1\"))\n\n([GK_1358370] of  StableIdentifier\n\n	(_displayName \"REACT_76638.1\")\n	(DB_ID 1358370)\n	(identifier \"REACT_76638\")\n	(identifierVersion \"1\"))\n\n([GK_1358373] of  StableIdentifier\n\n	(_displayName \"REACT_76641.1\")\n	(DB_ID 1358373)\n	(identifier \"REACT_76641\")\n	(identifierVersion \"1\"))\n\n([GK_1358376] of  StableIdentifier\n\n	(_displayName \"REACT_76644.1\")\n	(DB_ID 1358376)\n	(identifier \"REACT_76644\")\n	(identifierVersion \"1\"))\n\n([GK_1358387] of  StableIdentifier\n\n	(_displayName \"REACT_76655.1\")\n	(DB_ID 1358387)\n	(identifier \"REACT_76655\")\n	(identifierVersion \"1\"))\n\n([GK_1358393] of  StableIdentifier\n\n	(_displayName \"REACT_76661.1\")\n	(DB_ID 1358393)\n	(identifier \"REACT_76661\")\n	(identifierVersion \"1\"))\n\n([GK_1358399] of  StableIdentifier\n\n	(_displayName \"REACT_76667.1\")\n	(DB_ID 1358399)\n	(identifier \"REACT_76667\")\n	(identifierVersion \"1\"))\n\n([GK_1358413] of  StableIdentifier\n\n	(_displayName \"REACT_76681.1\")\n	(DB_ID 1358413)\n	(identifier \"REACT_76681\")\n	(identifierVersion \"1\"))\n\n([GK_1358419] of  StableIdentifier\n\n	(_displayName \"REACT_76687.1\")\n	(DB_ID 1358419)\n	(identifier \"REACT_76687\")\n	(identifierVersion \"1\"))\n\n([GK_1358429] of  StableIdentifier\n\n	(_displayName \"REACT_76697.1\")\n	(DB_ID 1358429)\n	(identifier \"REACT_76697\")\n	(identifierVersion \"1\"))\n\n([GK_1358435] of  StableIdentifier\n\n	(_displayName \"REACT_76703.1\")\n	(DB_ID 1358435)\n	(identifier \"REACT_76703\")\n	(identifierVersion \"1\"))\n\n([GK_1358465] of  StableIdentifier\n\n	(_displayName \"REACT_76733.1\")\n	(DB_ID 1358465)\n	(identifier \"REACT_76733\")\n	(identifierVersion \"1\"))\n\n([GK_1358509] of  StableIdentifier\n\n	(_displayName \"REACT_76777.1\")\n	(DB_ID 1358509)\n	(identifier \"REACT_76777\")\n	(identifierVersion \"1\"))\n\n([GK_1358519] of  StableIdentifier\n\n	(_displayName \"REACT_76787.1\")\n	(DB_ID 1358519)\n	(identifier \"REACT_76787\")\n	(identifierVersion \"1\"))\n\n([GK_1358524] of  StableIdentifier\n\n	(_displayName \"REACT_76792.1\")\n	(DB_ID 1358524)\n	(identifier \"REACT_76792\")\n	(identifierVersion \"1\"))\n\n([GK_1358546] of  StableIdentifier\n\n	(_displayName \"REACT_76814.1\")\n	(DB_ID 1358546)\n	(identifier \"REACT_76814\")\n	(identifierVersion \"1\"))\n\n([GK_1358552] of  StableIdentifier\n\n	(_displayName \"REACT_76820.1\")\n	(DB_ID 1358552)\n	(identifier \"REACT_76820\")\n	(identifierVersion \"1\"))\n\n([GK_1358566] of  StableIdentifier\n\n	(_displayName \"REACT_76834.1\")\n	(DB_ID 1358566)\n	(identifier \"REACT_76834\")\n	(identifierVersion \"1\"))\n\n([GK_1358567] of  StableIdentifier\n\n	(_displayName \"REACT_76835.1\")\n	(DB_ID 1358567)\n	(identifier \"REACT_76835\")\n	(identifierVersion \"1\"))\n\n([GK_1358575] of  StableIdentifier\n\n	(_displayName \"REACT_76843.1\")\n	(DB_ID 1358575)\n	(identifier \"REACT_76843\")\n	(identifierVersion \"1\"))\n\n([GK_1358579] of  StableIdentifier\n\n	(_displayName \"REACT_76847.1\")\n	(DB_ID 1358579)\n	(identifier \"REACT_76847\")\n	(identifierVersion \"1\"))\n\n([GK_1358592] of  StableIdentifier\n\n	(_displayName \"REACT_76860.1\")\n	(DB_ID 1358592)\n	(identifier \"REACT_76860\")\n	(identifierVersion \"1\"))\n\n([GK_1358602] of  StableIdentifier\n\n	(_displayName \"REACT_76870.1\")\n	(DB_ID 1358602)\n	(identifier \"REACT_76870\")\n	(identifierVersion \"1\"))\n\n([GK_1358620] of  StableIdentifier\n\n	(_displayName \"REACT_76888.1\")\n	(DB_ID 1358620)\n	(identifier \"REACT_76888\")\n	(identifierVersion \"1\"))\n\n([GK_1358637] of  StableIdentifier\n\n	(_displayName \"REACT_76905.1\")\n	(DB_ID 1358637)\n	(identifier \"REACT_76905\")\n	(identifierVersion \"1\"))\n\n([GK_1358639] of  StableIdentifier\n\n	(_displayName \"REACT_76907.1\")\n	(DB_ID 1358639)\n	(identifier \"REACT_76907\")\n	(identifierVersion \"1\"))\n\n([GK_139809] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of cellular process\")\n	(accession \"0050794\")\n	(DB_ID 139809)\n	(definition \"Any process that modulates the frequency, rate or extent of a cellular process, any of those that are carried out at the cellular level, but are not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level.\")\n	(instanceOf [GK_139810])\n	(name \"regulation of cellular process\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_143]))\n\n([GK_139810] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of biological process\")\n	(accession \"0050789\")\n	(DB_ID 139810)\n	(definition \"Any process that modulates the frequency, rate or extent of a biological process. Biological processes are regulated by many means; examples include the control of gene expression, protein modification or interaction with a protein or substrate molecule.\")\n	(instanceOf [GK_258123])\n	(name \"regulation of biological process\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_106]))\n\n([GK_139812] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of apoptotic process\")\n	(accession \"0042981\")\n	(DB_ID 139812)\n	(definition \"Any process that modulates the occurrence or rate of cell death by apoptotic process.\")\n	(instanceOf [GK_139813])\n	(name \"regulation of apoptotic process\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2273]))\n\n([GK_139813] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of programmed cell death\")\n	(accession \"0043067\")\n	(DB_ID 139813)\n	(definition \"Any process that modulates the frequency, rate or extent of programmed cell death, cell death resulting from activation of endogenous cellular processes.\")\n	(instanceOf [GK_422582])\n	(name \"regulation of programmed cell death\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2274]))\n\n([GK_139950] of  Complex\n\n	(_displayName \"[Hsa] Caspase-8 dimer [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 139950)\n	(hasComponent\n		[GK_75975]\n		[GK_76158])\n	(name \"Caspase-8 dimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_366061]))\n\n([GK_139951] of  CatalystActivity\n\n	(_displayName \"[Hsa] cysteine-type endopeptidase activity of active caspase-8  [cytosol]\")\n	(activity [GK_4344])\n	(DB_ID 139951)\n	(physicalEntity [GK_2562550]))\n\n([GK_139952] of  BlackBoxEvent\n\n	(_displayName \"[Hsa] Caspase-8 processing \")\n	(_doRelease TRUE)\n	(catalystActivity [GK_139951])\n	(compartment\n		[GK_70101]\n		[GK_876])\n	(DB_ID 139952)\n	(edited [GK_3508822])\n	(input [GK_2671813])\n	(literatureReference\n		[GK_1267800]\n		[GK_2584573]\n		[GK_1267836])\n	(name \"Caspase-8 processing \")\n	(output\n		[GK_2671814]\n		[GK_2562550]\n		[GK_2562550]\n		[GK_2562550])\n	(precedingEvent\n		[GK_141156]\n		[GK_141159]\n		[GK_73945])\n	(reviewed\n		[GK_2594924]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_358238])\n	(summation [GK_178769]))\n\n([GK_139957] of  InstanceEdit\n\n	(_displayName \"Gillespie, ME, 2004-08-10 16:59:18\")\n	(author [GK_73681])\n	(dateTime \"2004-08-10 16:59:18\")\n	(DB_ID 139957))\n\n([GK_139958] of  Summation\n\n	(_displayName \"The death receptors, all cell-surface receptors, begin the p...\")\n	(DB_ID 139958)\n	(text \"The death receptors, all cell-surface receptors, begin the process of caspase activation. The common feature of these type 1 transmembrane proteins is the \\\"death-domain\\\" a conserved cytoplasmic motif found on all of the three receptors (FAS/CD95, TNF-receptor, and TRAIL-receptor) that binds the Fas-associated protein with death domain (FADD). Ligand binding to death receptors (DR) results in receptor oligomerization and recruitment of adaptor proteins via homophilic interaction of death domains. This project describes an assembly of the death-inducing signaling complex (DISC) and shows protein composition and stoichiometry within the DISC. It also provides the appropriate literature references if available. However, the DISC signaling complex may vary in its components stoichiometry. DR signaling may trigger formation of higher order receptor structures or signaling through rearrangement of receptor chains, which is not reflected here.\"))\n\n([GK_139959] of  Summation\n\n	(_displayName \"Known as the \\\"death receptor pathway\\\" the extrinsic or caspa...\")\n	(DB_ID 139959)\n	(text \"Known as the \\\"death receptor pathway\\\" the extrinsic or caspase 8/10 dependent pathway is activated by ligand binding. The \\\"death receptors\\\" are specialized cell-surface receptors including Fas/CD95, tumor necrosis factor-alpha (TNF-alpha) receptor 1, and two receptors, DR4 and DR5, that bind to the TNF-alpha related apoptosis-inducing ligand (TRAIL). The extrinsic and intrinsic pathways unite in the activation of Caspase-3, though the two pathways communicate through the pro-apoptotic Bcl-2 family member Bid before uniting at the shared activation of Caspase-3.\"))\n\n([GK_139960] of  LiteratureReference\n\n	(_displayName \"The biochemistry of apoptosis.\")\n	(author [GK_140492])\n	(DB_ID 139960)\n	(journal \"Nature\")\n	(pages \"770-6\")\n	(pubMedIdentifier 11048727)\n	(title \"The biochemistry of apoptosis.\")\n	(volume 407)\n	(year 2000))\n\n([GK_140] of  GO_BiologicalProcess\n\n	(_displayName \"cell surface receptor signaling pathway\")\n	(accession \"0007166\")\n	(DB_ID 140)\n	(definition \"A series of molecular signals initiated by activation of a receptor on the surface of a cell. The pathway begins with binding of an extracellular ligand to a cell surface receptor, or for receptors that signal in the absence of a ligand, by ligand-withdrawal or the activity of a constitutively active receptor. The pathway ends with regulation of a downstream cellular process, e.g. transcription.\")\n	(instanceOf [GK_141])\n	(name \"cell surface receptor signaling pathway\")\n	(referenceDatabase [GK_1]))\n\n([GK_140480] of  InstanceEdit\n\n	(_displayName \"Gillespie, ME, 2004-08-18 21:05:31\")\n	(author [GK_73681])\n	(dateTime \"2004-08-18 21:05:31\")\n	(DB_ID 140480))\n\n([GK_140481] of  Summation\n\n	(_displayName \"The Fas family of cell surface receptors initiate the apotot...\")\n	(DB_ID 140481)\n	(literatureReference\n		[GK_140482]\n		[GK_140484]\n		[GK_140483])\n	(text \"The Fas family of cell surface receptors initiate the apototic pathway through interaction with the external ligand, FasL.  The cytoplasmic domain of Fas interacts with a number of molecules in the transduction of the external signal to the cytoplasmic side of the cell membrane. The most notable cytoplasmic domain is the Death Domain (DD) that is involved in recruiting the FAS-associating death domain-containing protein (FADD). This interaction drives downstream events.\"))\n\n([GK_140482] of  LiteratureReference\n\n	(_displayName \"The polypeptide encoded by the cDNA for human cell surface antigen Fas can mediate apoptosis\")\n	(author\n		[GK_141169]\n		[GK_141174]\n		[GK_141175]\n		[GK_141172]\n		[GK_141170]\n		[GK_141171]\n		[GK_141166]\n		[GK_76786]\n		[GK_141176])\n	(DB_ID 140482)\n	(journal \"Cell\")\n	(pages \"233-43\")\n	(pubMedIdentifier 1713127)\n	(title \"The polypeptide encoded by the cDNA for human cell surface antigen Fas can mediate apoptosis\")\n	(volume 66)\n	(year 1991))\n\n([GK_140483] of  LiteratureReference\n\n	(_displayName \"FADD, a novel death domain-containing protein, interacts with the death domain of Fas and initiates apoptosis\")\n	(author\n		[GK_141168]\n		[GK_141167]\n		[GK_141165]\n		[GK_141149])\n	(DB_ID 140483)\n	(journal \"Cell\")\n	(pages \"505-12\")\n	(pubMedIdentifier 7538907)\n	(title \"FADD, a novel death domain-containing protein, interacts with the death domain of Fas and initiates apoptosis\")\n	(volume 81)\n	(year 1995))\n\n([GK_140484] of  LiteratureReference\n\n	(_displayName \"A cell-killing monoclonal antibody (anti-Fas) to a cell surface antigen co-downregulated with the receptor of tumor necrosis factor\")\n	(author\n		[GK_141174]\n		[GK_141175]\n		[GK_141172])\n	(DB_ID 140484)\n	(journal \"J Exp Med\")\n	(pages \"1747-56\")\n	(pubMedIdentifier 2469768)\n	(title \"A cell-killing monoclonal antibody (anti-Fas) to a cell surface antigen co-downregulated with the receptor of tumor necrosis factor\")\n	(volume 169)\n	(year 1989))\n\n([GK_140492] of  Person\n\n	(_displayName \"Hengartner, MO\")\n	(DB_ID 140492)\n	(initial \"MO\")\n	(surname \"Hengartner\"))\n\n([GK_140494] of  Person\n\n	(_displayName \"Wallach, D\")\n	(DB_ID 140494)\n	(initial \"D\")\n	(surname \"Wallach\"))\n\n([GK_140495] of  LiteratureReference\n\n	(_displayName \"A novel protein that interacts with the death domain of Fas/APO1 contains a sequence motif related to the death domain\")\n	(author\n		[GK_140497]\n		[GK_140520]\n		[GK_140518]\n		[GK_140511]\n		[GK_111029]\n		[GK_140494])\n	(DB_ID 140495)\n	(journal \"J Biol Chem\")\n	(pages \"7795-8\")\n	(pubMedIdentifier 7536190)\n	(title \"A novel protein that interacts with the death domain of Fas/APO1 contains a sequence motif related to the death domain\")\n	(volume 270)\n	(year 1995))\n\n([GK_140496] of  Person\n\n	(_displayName \"Hellbardt, S\")\n	(DB_ID 140496)\n	(initial \"S\")\n	(surname \"Hellbardt\"))\n\n([GK_140497] of  Person\n\n	(_displayName \"Boldin, MP\")\n	(DB_ID 140497)\n	(initial \"MP\")\n	(surname \"Boldin\"))\n\n([GK_140498] of  Person\n\n	(_displayName \"Krammer, PH\")\n	(DB_ID 140498)\n	(initial \"PH\")\n	(surname \"Krammer\"))\n\n([GK_140499] of  Person\n\n	(_displayName \"Lynch, DH\")\n	(DB_ID 140499)\n	(initial \"DH\")\n	(surname \"Lynch\"))\n\n([GK_140500] of  Person\n\n	(_displayName \"Echeverri, F\")\n	(DB_ID 140500)\n	(initial \"F\")\n	(surname \"Echeverri\"))\n\n([GK_140501] of  Person\n\n	(_displayName \"Yoo, NJ\")\n	(DB_ID 140501)\n	(initial \"NJ\")\n	(surname \"Yoo\"))\n\n([GK_140502] of  Person\n\n	(_displayName \"Peter, ME\")\n	(DB_ID 140502)\n	(initial \"ME\")\n	(surname \"Peter\"))\n\n([GK_140504] of  Person\n\n	(_displayName \"Pawlita, M\")\n	(DB_ID 140504)\n	(initial \"M\")\n	(surname \"Pawlita\"))\n\n([GK_140505] of  LiteratureReference\n\n	(_displayName \"Cytotoxicity-dependent APO-1 (Fas/CD95)-associated proteins form a death-inducing signaling complex (DISC) with the receptor\")\n	(author\n		[GK_140512]\n		[GK_140496]\n		[GK_140507]\n		[GK_140517]\n		[GK_140504]\n		[GK_140498]\n		[GK_140502])\n	(DB_ID 140505)\n	(journal \"EMBO J\")\n	(pages \"5579-88\")\n	(pubMedIdentifier 8521815)\n	(title \"Cytotoxicity-dependent APO-1 (Fas/CD95)-associated proteins form a death-inducing signaling complex (DISC) with the receptor\")\n	(volume 14)\n	(year 1995))\n\n([GK_140506] of  Person\n\n	(_displayName \"Brunner, T\")\n	(DB_ID 140506)\n	(initial \"T\")\n	(surname \"Brunner\"))\n\n([GK_140507] of  Person\n\n	(_displayName \"Behrmann, I\")\n	(DB_ID 140507)\n	(initial \"I\")\n	(surname \"Behrmann\"))\n\n([GK_140509] of  Person\n\n	(_displayName \"Mahboubi, A\")\n	(DB_ID 140509)\n	(initial \"A\")\n	(surname \"Mahboubi\"))\n\n([GK_140510] of  Person\n\n	(_displayName \"LaFace, D\")\n	(DB_ID 140510)\n	(initial \"D\")\n	(surname \"LaFace\"))\n\n([GK_140511] of  Person\n\n	(_displayName \"Mett, IL\")\n	(DB_ID 140511)\n	(initial \"IL\")\n	(surname \"Mett\"))\n\n([GK_140512] of  Person\n\n	(_displayName \"Kischkel, FC\")\n	(DB_ID 140512)\n	(initial \"FC\")\n	(surname \"Kischkel\"))\n\n([GK_140513] of  LiteratureReference\n\n	(_displayName \"Cell-autonomous Fas (CD95)/Fas-ligand interaction mediates activation-induced apoptosis in T-cell hybridomas\")\n	(author\n		[GK_140506]\n		[GK_140515]\n		[GK_140510]\n		[GK_140501]\n		[GK_140509]\n		[GK_140500]\n		[GK_140516]\n		[GK_140514]\n		[GK_140499]\n		[GK_140519])\n	(DB_ID 140513)\n	(journal \"Nature\")\n	(pages \"441-4\")\n	(pubMedIdentifier 7530336)\n	(title \"Cell-autonomous Fas (CD95)/Fas-ligand interaction mediates activation-induced apoptosis in T-cell hybridomas\")\n	(volume 373)\n	(year 1995))\n\n([GK_140514] of  Person\n\n	(_displayName \"Force, WR\")\n	(DB_ID 140514)\n	(initial \"WR\")\n	(surname \"Force\"))\n\n([GK_140515] of  Person\n\n	(_displayName \"Mogil, RJ\")\n	(DB_ID 140515)\n	(initial \"RJ\")\n	(surname \"Mogil\"))\n\n([GK_140516] of  Person\n\n	(_displayName \"Martin, SJ\")\n	(DB_ID 140516)\n	(initial \"SJ\")\n	(surname \"Martin\"))\n\n([GK_140517] of  Person\n\n	(_displayName \"Germer, M\")\n	(DB_ID 140517)\n	(initial \"M\")\n	(surname \"Germer\"))\n\n([GK_140518] of  Person\n\n	(_displayName \"Pancer, Z\")\n	(DB_ID 140518)\n	(initial \"Z\")\n	(surname \"Pancer\"))\n\n([GK_140519] of  Person\n\n	(_displayName \"Ware, CF\")\n	(DB_ID 140519)\n	(initial \"CF\")\n	(surname \"Ware\"))\n\n([GK_140520] of  Person\n\n	(_displayName \"Varfolomeev, EE\")\n	(DB_ID 140520)\n	(initial \"EE\")\n	(surname \"Varfolomeev\"))\n\n([GK_140534] of  Pathway\n\n	(_displayName \"[Hsa] Caspase-8 activation\")\n	(_doRelease TRUE)\n	(compartment [GK_70101])\n	(DB_ID 140534)\n	(doi \"10.3180/REACT_1503.5\")\n	(edited [GK_3508822])\n	(goBiologicalProcess [GK_23175])\n	(hasEvent\n		[GK_69416]\n		[GK_139952])\n	(literatureReference\n		[GK_141160]\n		[GK_1267800]\n		[GK_879193]\n		[GK_2671825]\n		[GK_2584573])\n	(name \"Caspase-8 activation\")\n	(reviewed\n		[GK_2594924]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_362011])\n	(summation [GK_213088]))\n\n([GK_140935] of  Complex\n\n	(_displayName \"[Hsa] TRAF2:TRADD:RIPK1 [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 140935)\n	(goCellularComponent [GK_161217])\n	(hasComponent\n		[GK_66370]\n		[GK_66370]\n		[GK_66370]\n		[GK_66376]\n		[GK_66376]\n		[GK_66376]\n		[GK_168651])\n	(name \"TRAF2:TRADD:RIPK1\")\n	(species [GK_48887])\n	(stableIdentifier [GK_363774]))\n\n([GK_140936] of  InstanceEdit\n\n	(_displayName \"Gillespie, ME, 2004-08-24 21:32:45\")\n	(author [GK_73681])\n	(dateTime \"2004-08-24 21:32:45\")\n	(DB_ID 140936))\n\n([GK_140937] of  Person\n\n	(_displayName \"Hsu, H\")\n	(DB_ID 140937)\n	(initial \"H\")\n	(surname \"Hsu\"))\n\n([GK_140938] of  Person\n\n	(_displayName \"Aggarwal, BB\")\n	(DB_ID 140938)\n	(initial \"BB\")\n	(surname \"Aggarwal\"))\n\n([GK_140939] of  LiteratureReference\n\n	(_displayName \"TNF-R1 signaling: a beautiful pathway\")\n	(author\n		[GK_140944]\n		[GK_75697])\n	(DB_ID 140939)\n	(journal \"Science\")\n	(pages \"1634-5\")\n	(pubMedIdentifier 12040173)\n	(title \"TNF-R1 signaling: a beautiful pathway\")\n	(volume 296)\n	(year 2002))\n\n([GK_140940] of  Person\n\n	(_displayName \"Aiyer, RA\")\n	(DB_ID 140940)\n	(initial \"RA\")\n	(surname \"Aiyer\"))\n\n([GK_140941] of  LiteratureReference\n\n	(_displayName \"Involvement of MACH, a novel MORT1/FADD-interacting protease, in Fas/APO-1- and TNF receptor-induced cell death\")\n	(author\n		[GK_140497]\n		[GK_140949]\n		[GK_140947]\n		[GK_140494])\n	(DB_ID 140941)\n	(journal \"Cell\")\n	(pages \"803-15\")\n	(pubMedIdentifier 8681376)\n	(title \"Involvement of MACH, a novel MORT1/FADD-interacting protease, in Fas/APO-1- and TNF receptor-induced cell death\")\n	(volume 85)\n	(year 1996))\n\n([GK_140942] of  Summation\n\n	(_displayName \"The Tumor Necrosis Factor alpha (TNF-alpha) mediated apoptos...\")\n	(DB_ID 140942)\n	(text \"The Tumor Necrosis Factor alpha (TNF-alpha) mediated apoptosis pathway has been implicated in the pathogenesis of a number of diseases including sepsis, diabetes, cancer, osteoporosis, multiple sclerosis, rheumatoid arthritis, and inflammatory bowel diseases. The TNF signaling network provides extensive cross talk between the apoptotic pathway, and the other NF-B, and JNK pathways that also emanate from TNF-R.\"))\n\n([GK_140943] of  Person\n\n	(_displayName \"Stauber, GB\")\n	(DB_ID 140943)\n	(initial \"GB\")\n	(surname \"Stauber\"))\n\n([GK_140944] of  Person\n\n	(_displayName \"Chen, G\")\n	(DB_ID 140944)\n	(initial \"G\")\n	(surname \"Chen\"))\n\n([GK_140945] of  LiteratureReference\n\n	(_displayName \"Human tumor necrosis factor-alpha receptor. Purification by immunoaffinity chromatography and initial characterization.\")\n	(author\n		[GK_140943]\n		[GK_140940]\n		[GK_140938])\n	(DB_ID 140945)\n	(journal \"J Biol Chem\")\n	(pages \"19098-104\")\n	(pubMedIdentifier 2848815)\n	(title \"Human tumor necrosis factor-alpha receptor. Purification by immunoaffinity chromatography and initial characterization.\")\n	(volume 263)\n	(year 1988))\n\n([GK_140946] of  Complex\n\n	(_displayName \"[Hsa] TNF-alpha:TNF-R1:TRADD:RIPK1:TRAF2 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 140946)\n	(goCellularComponent [GK_161217])\n	(hasComponent\n		[GK_66376]\n		[GK_66376]\n		[GK_66376]\n		[GK_66370]\n		[GK_66370]\n		[GK_66370]\n		[GK_168651]\n		[GK_3371401])\n	(literatureReference\n		[GK_3465353]\n		[GK_3465415]\n		[GK_140979]\n		[GK_3465531])\n	(name \"TNF-alpha:TNF-R1:TRADD:RIPK1:TRAF2\")\n	(species [GK_48887])\n	(stableIdentifier [GK_371402]))\n\n([GK_140947] of  Person\n\n	(_displayName \"Goltsev, YV\")\n	(DB_ID 140947)\n	(initial \"YV\")\n	(surname \"Goltsev\"))\n\n([GK_140948] of  LiteratureReference\n\n	(_displayName \"The TNF receptor 1-associated protein TRADD signals cell death and NF-kappa B activation\")\n	(author\n		[GK_140937]\n		[GK_140950]\n		[GK_75697])\n	(DB_ID 140948)\n	(journal \"Cell\")\n	(pages \"495-504\")\n	(pubMedIdentifier 7758105)\n	(title \"The TNF receptor 1-associated protein TRADD signals cell death and NF-kappa B activation\")\n	(volume 81)\n	(year 1995))\n\n([GK_140949] of  Person\n\n	(_displayName \"Goncharov, TM\")\n	(DB_ID 140949)\n	(initial \"TM\")\n	(surname \"Goncharov\"))\n\n([GK_140950] of  Person\n\n	(_displayName \"Xiong, J\")\n	(DB_ID 140950)\n	(initial \"J\")\n	(surname \"Xiong\"))\n\n([GK_140975] of  Person\n\n	(_displayName \"Micheau, O\")\n	(DB_ID 140975)\n	(initial \"O\")\n	(surname \"Micheau\"))\n\n([GK_140976] of  Complex\n\n	(_displayName \"[Hsa] TRADD:TRAF2:RIP1:FADD:procaspase-8  [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 140976)\n	(goCellularComponent [GK_161217])\n	(hasComponent\n		[GK_140977]\n		[GK_57031])\n	(literatureReference [GK_2984174])\n	(name \"TRADD:TRAF2:RIP1:FADD:procaspase-8 \")\n	(species [GK_48887])\n	(stableIdentifier [GK_362216]))\n\n([GK_140977] of  Complex\n\n	(_displayName \"[Hsa] TRADD:TRAF2:RIP1:FADD [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 140977)\n	(goCellularComponent [GK_161217])\n	(hasComponent\n		[GK_140935]\n		[GK_54639]\n		[GK_54639])\n	(literatureReference\n		[GK_140979]\n		[GK_3465531])\n	(name \"TRADD:TRAF2:RIP1:FADD\")\n	(species [GK_48887])\n	(stableIdentifier [GK_367733]))\n\n([GK_140978] of  Reaction\n\n	(_displayName \"[Hsa] TRADD:TRAF2:RIP1 complex binds FADD\")\n	(_doRelease TRUE)\n	(compartment [GK_70101])\n	(DB_ID 140978)\n	(edited [GK_165085])\n	(input\n		[GK_140935]\n		[GK_54639]\n		[GK_54639])\n	(literatureReference [GK_140979])\n	(name \"TRADD:TRAF2:RIP1 complex binds FADD\")\n	(output [GK_140977])\n	(precedingEvent [GK_83582])\n	(reviewed\n		[GK_165094]\n		[GK_3597630])\n	(revised [GK_3604387])\n	(species [GK_48887])\n	(stableIdentifier [GK_358269])\n	(summation [GK_74353]))\n\n([GK_140979] of  LiteratureReference\n\n	(_displayName \"Induction of TNF receptor I-mediated apoptosis via two sequential signaling complexes\")\n	(author\n		[GK_140975]\n		[GK_77293])\n	(DB_ID 140979)\n	(journal \"Cell\")\n	(pages \"181-90\")\n	(pubMedIdentifier 12887920)\n	(title \"Induction of TNF receptor I-mediated apoptosis via two sequential signaling complexes\")\n	(volume 114)\n	(year 2003))\n\n([GK_140991] of  Person\n\n	(_displayName \"Wang, S\")\n	(DB_ID 140991)\n	(initial \"S\")\n	(surname \"Wang\"))\n\n([GK_140992] of  LiteratureReference\n\n	(_displayName \"TRAIL and apoptosis induction by TNF-family death receptors\")\n	(author\n		[GK_140991]\n		[GK_77151])\n	(DB_ID 140992)\n	(journal \"Oncogene\")\n	(pages \"8628-33\")\n	(pubMedIdentifier 14634624)\n	(title \"TRAIL and apoptosis induction by TNF-family death receptors\")\n	(volume 22)\n	(year 2003))\n\n([GK_141] of  GO_BiologicalProcess\n\n	(_displayName \"signal transduction\")\n	(accession \"0007165\")\n	(componentOf\n		[GK_142]\n		[GK_2587988])\n	(DB_ID 141)\n	(definition \"The cellular process in which a signal is conveyed to trigger a change in the activity or state of a cell. Signal transduction begins with reception of a signal (e.g. a ligand binding to a receptor or receptor activation by a stimulus such as light), or for signal transduction in the absence of ligand, signal-withdrawal or the activity of a constitutively active receptor. Signal transduction ends with regulation of a downstream cellular process, e.g. regulation of transcription or regulation of a metabolic process. Signal transduction covers signaling from receptors located on the surface of the cell and signaling via molecules located within the cell. For signaling between cells, signal transduction is restricted to events at and within the receiving cell.\")\n	(instanceOf\n		[GK_139809]\n		[GK_171571])\n	(name \"signal transduction\")\n	(referenceDatabase [GK_1]))\n\n([GK_141075] of  Person\n\n	(_displayName \"Ashkenazi, A\")\n	(DB_ID 141075)\n	(initial \"A\")\n	(surname \"Ashkenazi\"))\n\n([GK_141083] of  Summation\n\n	(_displayName \"Tumor necrosis factor-related apoptosis-inducing ligand or A...\")\n	(DB_ID 141083)\n	(literatureReference\n		[GK_140992]\n		[GK_3371373]\n		[GK_141314])\n	(text \"Tumor necrosis factor-related apoptosis-inducing ligand or Apo 2 ligand (TRAIL/Apo2L) is a member of the tumor necrosis factor (TNF) family. This group of apoptosis induction pathways all work through protein interactions mediated by the intracellular death domain (DD), encoded within the cytoplasmic domain of the receptor. TRAIL selectively induces apoptosis through its interaction with the Fas-associated death domain protein (FADD) and caspase-8/10 (Wang S & el-Deiry WS 2003; Sprick MR et al. 2002). TRAIL and its receptors, TRAIL-R1 and TRAIL-R2, were shown to be rapidly endocytosed via clathrin-dependent and -independent manner in human Burkitt\'s lymphoma B cells (BJAB) (Kohlhaas SL et al. 2007). However, FADD and caspase-8 were able to bind TRAIL-R1/R2 in TRAIL-stimulated BJAB cells at 4<sup>o</sup>C (at which membrane trafficking is inhibited), suggesting that the endocytosis was not required for an assembly of the functional TRAIL DISC complex. Moreover, blocking of clathrin-dependent endocytosis did not interfere with the capacity of TRAIL to promote apoptosis (Kohlhaas SL et al. 2007).<br>\"))\n\n([GK_141084] of  InstanceEdit\n\n	(_displayName \"Gillespie, ME, 2004-08-31 03:31:31\")\n	(author [GK_73681])\n	(dateTime \"2004-08-31 03:31:31\")\n	(DB_ID 141084))\n\n([GK_141118] of  Complex\n\n	(_displayName \"[Hsa] TRAIL receptor-2:TRAIL complex [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 141118)\n	(hasComponent\n		[GK_65505]\n		[GK_66186])\n	(name \"TRAIL receptor-2:TRAIL complex\")\n	(species [GK_48887])\n	(stableIdentifier [GK_368665]))\n\n([GK_141126] of  Complex\n\n	(_displayName \"[Hsa] TRAIL receptor-2:TRAIL Trimer [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 141126)\n	(hasComponent\n		[GK_141118]\n		[GK_141118]\n		[GK_141118])\n	(name \"TRAIL receptor-2:TRAIL Trimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_371770]))\n\n([GK_141128] of  LiteratureReference\n\n	(_displayName \"FADD/MORT1 and caspase-8 are recruited to TRAIL receptors 1 and 2 and are essential for apoptosis mediated by TRAIL receptor 2\")\n	(author\n		[GK_141131]\n		[GK_141129]\n		[GK_141136]\n		[GK_141130]\n		[GK_141135]\n		[GK_141134]\n		[GK_140498]\n		[GK_141138])\n	(DB_ID 141128)\n	(journal \"Immunity\")\n	(pages \"599-609\")\n	(pubMedIdentifier 10894160)\n	(title \"FADD/MORT1 and caspase-8 are recruited to TRAIL receptors 1 and 2 and are essential for apoptosis mediated by TRAIL receptor 2\")\n	(volume 12)\n	(year 2000))\n\n([GK_141129] of  Person\n\n	(_displayName \"Weigand, MA\")\n	(DB_ID 141129)\n	(initial \"MA\")\n	(surname \"Weigand\"))\n\n([GK_141130] of  Person\n\n	(_displayName \"Rauch, CT\")\n	(DB_ID 141130)\n	(initial \"CT\")\n	(surname \"Rauch\"))\n\n([GK_141131] of  Person\n\n	(_displayName \"Sprick, MR\")\n	(DB_ID 141131)\n	(initial \"MR\")\n	(surname \"Sprick\"))\n\n([GK_141133] of  Complex\n\n	(_displayName \"[Hsa] TRAIL:TRAIL receptor-2 Trimer:FADD:procaspase-8 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 141133)\n	(hasComponent\n		[GK_57031]\n		[GK_141137])\n	(literatureReference\n		[GK_141128]\n		[GK_3371373])\n	(name \"TRAIL:TRAIL receptor-2 Trimer:FADD:procaspase-8\")\n	(species [GK_48887])\n	(stableIdentifier [GK_364329]))\n\n([GK_141134] of  Person\n\n	(_displayName \"Blenis, J\")\n	(DB_ID 141134)\n	(initial \"J\")\n	(surname \"Blenis\"))\n\n([GK_141135] of  Person\n\n	(_displayName \"Juo, P\")\n	(DB_ID 141135)\n	(initial \"P\")\n	(surname \"Juo\"))\n\n([GK_141136] of  Person\n\n	(_displayName \"Rieser, E\")\n	(DB_ID 141136)\n	(initial \"E\")\n	(surname \"Rieser\"))\n\n([GK_141137] of  Complex\n\n	(_displayName \"[Hsa] TRAIL:TRAIL receptor-2:FADD [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 141137)\n	(hasComponent\n		[GK_141126]\n		[GK_54639]\n		[GK_54639])\n	(literatureReference\n		[GK_3465531]\n		[GK_141128])\n	(name \"TRAIL:TRAIL receptor-2:FADD\")\n	(species [GK_48887])\n	(stableIdentifier [GK_363728]))\n\n([GK_141138] of  Person\n\n	(_displayName \"Walczak, H\")\n	(DB_ID 141138)\n	(firstname \"Henning\")\n	(initial \"H\")\n	(surname \"Walczak\"))\n\n([GK_141139] of  Reaction\n\n	(_displayName \"[Hsa] Trimerization of TRAIL: TRAIL receptor-2 complex\")\n	(_doRelease TRUE)\n	(compartment [GK_876])\n	(DB_ID 141139)\n	(input\n		[GK_141118]\n		[GK_141118]\n		[GK_141118])\n	(literatureReference\n		[GK_141128]\n		[GK_3371348])\n	(name \"Trimerization of TRAIL: TRAIL receptor-2 complex\")\n	(output [GK_141126])\n	(precedingEvent [GK_75238])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_361815])\n	(summation [GK_179273]))\n\n([GK_141140] of  Person\n\n	(_displayName \"Salvesen, Guy S.\")\n	(affiliation [GK_3597631])\n	(DB_ID 141140)\n	(firstname \"Guy S.\")\n	(initial \"GS\")\n	(surname \"Salvesen\"))\n\n([GK_141141] of  Person\n\n	(_displayName \"Sutherland, GR\")\n	(DB_ID 141141)\n	(initial \"GR\")\n	(surname \"Sutherland\"))\n\n([GK_141142] of  Person\n\n	(_displayName \"Wiley, SR\")\n	(DB_ID 141142)\n	(initial \"SR\")\n	(surname \"Wiley\"))\n\n([GK_141143] of  Person\n\n	(_displayName \"Boatright, KM\")\n	(DB_ID 141143)\n	(initial \"KM\")\n	(surname \"Boatright\"))\n\n([GK_141147] of  LiteratureReference\n\n	(_displayName \"Identification and characterization of a new member of the TNF family that induces apoptosis\")\n	(author\n		[GK_141142]\n		[GK_141153]\n		[GK_141148]\n		[GK_141154]\n		[GK_141157]\n		[GK_141155]\n		[GK_141141]\n		[GK_141152]\n		[GK_141151]\n		[GK_111168])\n	(DB_ID 141147)\n	(journal \"Immunity\")\n	(pages \"673-82\")\n	(pubMedIdentifier 8777713)\n	(title \"Identification and characterization of a new member of the TNF family that induces apoptosis\")\n	(volume 3)\n	(year 1995))\n\n([GK_141148] of  Person\n\n	(_displayName \"Smolak, PJ\")\n	(DB_ID 141148)\n	(initial \"PJ\")\n	(surname \"Smolak\"))\n\n([GK_141149] of  Person\n\n	(_displayName \"Dixit, VM\")\n	(DB_ID 141149)\n	(initial \"VM\")\n	(surname \"Dixit\"))\n\n([GK_141151] of  Person\n\n	(_displayName \"Rauch, C\")\n	(DB_ID 141151)\n	(initial \"C\")\n	(surname \"Rauch\"))\n\n([GK_141152] of  Person\n\n	(_displayName \"Smith, TD\")\n	(DB_ID 141152)\n	(initial \"TD\")\n	(surname \"Smith\"))\n\n([GK_141153] of  Person\n\n	(_displayName \"Schooley, K\")\n	(DB_ID 141153)\n	(initial \"K\")\n	(surname \"Schooley\"))\n\n([GK_141154] of  Person\n\n	(_displayName \"Din, WS\")\n	(DB_ID 141154)\n	(initial \"WS\")\n	(surname \"Din\"))\n\n([GK_141155] of  Person\n\n	(_displayName \"Nicholl, JK\")\n	(DB_ID 141155)\n	(initial \"JK\")\n	(surname \"Nicholl\"))\n\n([GK_141156] of  Reaction\n\n	(_displayName \"[Hsa] TRAIL-mediated dimerization of procaspase-8\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 141156)\n	(edited [GK_3508822])\n	(input\n		[GK_141133]\n		[GK_57031])\n	(literatureReference\n		[GK_141128]\n		[GK_879193])\n	(name \"TRAIL-mediated dimerization of procaspase-8\")\n	(output [GK_2671816])\n	(precedingEvent [GK_83586])\n	(reviewed\n		[GK_2594924]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_361043])\n	(summation [GK_179244]))\n\n([GK_141157] of  Person\n\n	(_displayName \"Huang, CP\")\n	(DB_ID 141157)\n	(initial \"CP\")\n	(surname \"Huang\"))\n\n([GK_141158] of  LiteratureReference\n\n	(_displayName \"Death receptors: signaling and modulation\")\n	(author\n		[GK_141075]\n		[GK_141149])\n	(DB_ID 141158)\n	(journal \"Science\")\n	(pages \"1305-8\")\n	(pubMedIdentifier 9721089)\n	(title \"Death receptors: signaling and modulation\")\n	(volume 281)\n	(year 1998))\n\n([GK_141159] of  Reaction\n\n	(_displayName \"[Hsa] TNF-mediated dimerization of procaspase-8\")\n	(_doRelease TRUE)\n	(compartment [GK_70101])\n	(DB_ID 141159)\n	(edited [GK_3508822])\n	(input\n		[GK_140976]\n		[GK_57031])\n	(literatureReference\n		[GK_2984174]\n		[GK_140979]\n		[GK_2584573])\n	(name \"TNF-mediated dimerization of procaspase-8\")\n	(output [GK_2671812])\n	(precedingEvent [GK_75240])\n	(reviewed\n		[GK_2594924]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_359831])\n	(summation [GK_179239]))\n\n([GK_141160] of  LiteratureReference\n\n	(_displayName \"Mechanisms of caspase activation\")\n	(author\n		[GK_141143]\n		[GK_141140])\n	(DB_ID 141160)\n	(journal \"Curr Opin Cell Biol\")\n	(pages \"725-31\")\n	(pubMedIdentifier 14644197)\n	(title \"Mechanisms of caspase activation\")\n	(volume 15)\n	(year 2003))\n\n([GK_141165] of  Person\n\n	(_displayName \"Tewari, M\")\n	(DB_ID 141165)\n	(initial \"M\")\n	(surname \"Tewari\"))\n\n([GK_141166] of  Person\n\n	(_displayName \"Hase, A\")\n	(DB_ID 141166)\n	(initial \"A\")\n	(surname \"Hase\"))\n\n([GK_141167] of  Person\n\n	(_displayName \"O\'Rourke, K\")\n	(DB_ID 141167)\n	(initial \"K\")\n	(surname \"O\'Rourke\"))\n\n([GK_141168] of  Person\n\n	(_displayName \"Chinnaiyan, AM\")\n	(DB_ID 141168)\n	(initial \"AM\")\n	(surname \"Chinnaiyan\"))\n\n([GK_141169] of  Person\n\n	(_displayName \"Itoh, N\")\n	(DB_ID 141169)\n	(initial \"N\")\n	(surname \"Itoh\"))\n\n([GK_141170] of  Person\n\n	(_displayName \"Mizushima, S\")\n	(DB_ID 141170)\n	(initial \"S\")\n	(surname \"Mizushima\"))\n\n([GK_141171] of  Person\n\n	(_displayName \"Sameshima, M\")\n	(DB_ID 141171)\n	(initial \"M\")\n	(surname \"Sameshima\"))\n\n([GK_141172] of  Person\n\n	(_displayName \"Yonehara, M\")\n	(DB_ID 141172)\n	(initial \"M\")\n	(surname \"Yonehara\"))\n\n([GK_141174] of  Person\n\n	(_displayName \"Yonehara, S\")\n	(DB_ID 141174)\n	(initial \"S\")\n	(surname \"Yonehara\"))\n\n([GK_141175] of  Person\n\n	(_displayName \"Ishii, A\")\n	(DB_ID 141175)\n	(initial \"A\")\n	(surname \"Ishii\"))\n\n([GK_141176] of  Person\n\n	(_displayName \"Nagata, S\")\n	(DB_ID 141176)\n	(initial \"S\")\n	(surname \"Nagata\"))\n\n([GK_141226] of  Figure\n\n	(_displayName \"/figures/FasL_Fas_FADD.jpg\")\n	(DB_ID 141226)\n	(url \"/figures/FasL_Fas_FADD.jpg\"))\n\n([GK_141227] of  Figure\n\n	(_displayName \"/figures/FasL_Fas_FADD_Caspase8.jpg\")\n	(DB_ID 141227)\n	(url \"/figures/FasL_Fas_FADD_Caspase8.jpg\"))\n\n([GK_141228] of  Figure\n\n	(_displayName \"/figures/FasL_Fas_FADD_ProCaspase8.jpg\")\n	(DB_ID 141228)\n	(url \"/figures/FasL_Fas_FADD_ProCaspase8.jpg\"))\n\n([GK_141234] of  Figure\n\n	(_displayName \"/figures/FAS_Top.jpg\")\n	(DB_ID 141234)\n	(url \"/figures/FAS_Top.jpg\"))\n\n([GK_141237] of  Figure\n\n	(_displayName \"/figures/FasL_Fas.jpg\")\n	(DB_ID 141237)\n	(url \"/figures/FasL_Fas.jpg\"))\n\n([GK_141240] of  Figure\n\n	(_displayName \"/figures/FasL_Fas_FADD_ProCaspase10.jpg\")\n	(DB_ID 141240)\n	(url \"/figures/FasL_Fas_FADD_ProCaspase10.jpg\"))\n\n([GK_141277] of  Figure\n\n	(_displayName \"/figures/extrinsic_overview.jpeg\")\n	(DB_ID 141277)\n	(url \"/figures/extrinsic_overview.jpeg\"))\n\n([GK_141304] of  Complex\n\n	(_displayName \"[Hsa] TRAIL:TRAIL receptor-2 Trimer:FADD:Caspase-10 precursor complex [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 141304)\n	(hasComponent\n		[GK_57035]\n		[GK_141137])\n	(name \"TRAIL:TRAIL receptor-2 Trimer:FADD:Caspase-10 precursor complex\")\n	(species [GK_48887])\n	(stableIdentifier [GK_366444]))\n\n([GK_141305] of  Person\n\n	(_displayName \"Grosse-Wilde, A\")\n	(DB_ID 141305)\n	(initial \"A\")\n	(surname \"Grosse-Wilde\"))\n\n([GK_141306] of  LiteratureReference\n\n	(_displayName \"Caspase-10 is an initiator caspase in death receptor signaling\")\n	(author\n		[GK_84039]\n		[GK_141307]\n		[GK_141309]\n		[GK_141311]\n		[GK_141313])\n	(DB_ID 141306)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"13884-8\")\n	(pubMedIdentifier 11717445)\n	(title \"Caspase-10 is an initiator caspase in death receptor signaling\")\n	(volume 98)\n	(year 2001))\n\n([GK_141307] of  Person\n\n	(_displayName \"Chun, HJ\")\n	(DB_ID 141307)\n	(initial \"HJ\")\n	(surname \"Chun\"))\n\n([GK_141308] of  Person\n\n	(_displayName \"Stahl, H\")\n	(DB_ID 141308)\n	(initial \"H\")\n	(surname \"Stahl\"))\n\n([GK_141309] of  Person\n\n	(_displayName \"Wong, W\")\n	(DB_ID 141309)\n	(initial \"W\")\n	(surname \"Wong\"))\n\n([GK_141310] of  Reaction\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer:FADD complex binds pro-Caspase-10\")\n	(_doRelease TRUE)\n	(compartment [GK_876])\n	(DB_ID 141310)\n	(figure [GK_141240])\n	(input\n		[GK_43124]\n		[GK_57035])\n	(literatureReference\n		[GK_141306]\n		[GK_141314])\n	(name \"FASL:FAS Receptor Trimer:FADD complex binds pro-Caspase-10\")\n	(output [GK_141315])\n	(precedingEvent [GK_83650])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_359435])\n	(summation [GK_178763]))\n\n([GK_141311] of  Person\n\n	(_displayName \"Spencer, DM\")\n	(DB_ID 141311)\n	(initial \"DM\")\n	(surname \"Spencer\"))\n\n([GK_141313] of  Person\n\n	(_displayName \"Lenardo, MJ\")\n	(DB_ID 141313)\n	(initial \"MJ\")\n	(surname \"Lenardo\"))\n\n([GK_141314] of  LiteratureReference\n\n	(_displayName \"Caspase-10 is recruited to and activated at the native TRAIL and CD95 death-inducing signalling complexes in a FADD-dependent manner but can not functionally substitute caspase-8\")\n	(author\n		[GK_141131]\n		[GK_141136]\n		[GK_141308]\n		[GK_141305]\n		[GK_141129]\n		[GK_141138])\n	(DB_ID 141314)\n	(journal \"EMBO J\")\n	(pages \"4520-30\")\n	(pubMedIdentifier 12198154)\n	(title \"Caspase-10 is recruited to and activated at the native TRAIL and CD95 death-inducing signalling complexes in a FADD-dependent manner but can not functionally substitute caspase-8\")\n	(volume 21)\n	(year 2002))\n\n([GK_141315] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer:FADD:pro-Caspase-10 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 141315)\n	(hasComponent\n		[GK_43124]\n		[GK_57035])\n	(name \"FASL:FAS Receptor Trimer:FADD:pro-Caspase-10\")\n	(species [GK_48887])\n	(stableIdentifier [GK_370192]))\n\n([GK_141316] of  Reaction\n\n	(_displayName \"[Hsa] TRAIL:TRAIL-Receptor2 Trimer:FADD complex binds Caspase-10\")\n	(_doRelease TRUE)\n	(compartment [GK_876])\n	(DB_ID 141316)\n	(input\n		[GK_141137]\n		[GK_57035])\n	(literatureReference\n		[GK_141306]\n		[GK_141314])\n	(name \"TRAIL:TRAIL-Receptor2 Trimer:FADD complex binds Caspase-10\")\n	(output [GK_141304])\n	(precedingEvent [GK_75187])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_358282])\n	(summation [GK_179246]))\n\n([GK_141456] of  Person\n\n	(_displayName \"Vaux, DL\")\n	(affiliation [GK_1247556])\n	(DB_ID 141456)\n	(eMailAddress \"d.vaux@latrobe.edu.au\")\n	(firstname \"David L\")\n	(initial \"DL\")\n	(surname \"Vaux\"))\n\n([GK_141860] of  Person\n\n	(_displayName \"Park, YC\")\n	(DB_ID 141860)\n	(firstname \"Y C\")\n	(initial \"YC\")\n	(surname \"Park\"))\n\n([GK_141861] of  Person\n\n	(_displayName \"Rich, RL\")\n	(DB_ID 141861)\n	(firstname \"R L\")\n	(initial \"RL\")\n	(surname \"Rich\"))\n\n([GK_141862] of  Person\n\n	(_displayName \"Segal, D\")\n	(DB_ID 141862)\n	(firstname \"D\")\n	(initial \"D\")\n	(surname \"Segal\"))\n\n([GK_141863] of  Person\n\n	(_displayName \"Myszka, DG\")\n	(DB_ID 141863)\n	(firstname \"D G\")\n	(initial \"DG\")\n	(surname \"Myszka\"))\n\n([GK_141864] of  Person\n\n	(_displayName \"Wu, H\")\n	(DB_ID 141864)\n	(firstname \"H\")\n	(initial \"H\")\n	(surname \"Wu\"))\n\n([GK_141902] of  Person\n\n	(_displayName \"Vandenabeele, P\")\n	(DB_ID 141902)\n	(firstname \"Peter\")\n	(initial \"P\")\n	(surname \"Vandenabeele\"))\n\n([GK_141937] of  Person\n\n	(_displayName \"Green, DR\")\n	(DB_ID 141937)\n	(firstname \"D R\")\n	(initial \"DR\")\n	(surname \"Green\"))\n\n([GK_141951] of  Person\n\n	(_displayName \"Shi, Y\")\n	(DB_ID 141951)\n	(firstname \"Yigong\")\n	(initial \"Y\")\n	(surname \"Shi\"))\n\n([GK_142] of  GO_BiologicalProcess\n\n	(_displayName \"cell communication\")\n	(accession \"0007154\")\n	(DB_ID 142)\n	(definition \"Any process that mediates interactions between a cell and its surroundings. Encompasses interactions such as signaling or attachment between one cell and another cell, between a cell and an extracellular matrix, or between a cell and any other aspect of its environment.\")\n	(instanceOf [GK_3132308])\n	(name \"cell communication\")\n	(referenceDatabase [GK_1]))\n\n([GK_143] of  GO_BiologicalProcess\n\n	(_displayName \"cellular process\")\n	(accession \"0009987\")\n	(DB_ID 143)\n	(definition \"Any process that is carried out at the cellular level, but not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level.\")\n	(instanceOf [GK_106])\n	(name \"cellular process\")\n	(referenceDatabase [GK_1]))\n\n([GK_144921] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P01258 CALCA recommendedName: Calcitonin  component recommendedName: Calcitonin  /component component recommendedName: Katacalcin  alternativeName: Calcitonin carboxyl-terminal peptide shortName:CCP  alternativeName: PDN-21  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Calcitonin causes a rapid but short-lived drop in the level of calcium and phosphate in blood by promoting the incorporation of those ions in the bones.FUNCTION Katacalcin is a potent plasma calcium-lowering peptide.SIMILARITY Belongs to the calcitonin family.\")\n	(DB_ID 144921)\n	(description \"recommendedName: Calcitonin  component recommendedName: Calcitonin  /component component recommendedName: Katacalcin  alternativeName: Calcitonin carboxyl-terminal peptide shortName:CCP  alternativeName: PDN-21  /component\")\n	(geneName\n		\"CALCA\"\n		\"CALC1\")\n	(identifier \"P01258\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Amidation\"\n		\"Cleavage on pair of basic residues\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Hormone\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"CALCA\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CALC_HUMAN\"\n		\"B7ZL39\"\n		\"Q13935\"\n		\"Q13937\"\n		\"Q52LX7\")\n	(sequenceLength 141)\n	(species [GK_48887]))\n\n([GK_145326] of  ReferenceIsoform\n\n	(_displayName \"[Hsa] UniProt:O15519-2 CFLAR recommendedName: CASP8 and FADD-like apoptosis regulator  alternativeName: Caspase homolog shortName:CASH  alternativeName: Caspase-eight-related protein shortName:Casper  alternativeName: Caspase-like apoptosis regulatory protein shortName:CLARP  alternativeName: Cellular FLICE-like inhibitory protein shortName:c-FLIP  alternativeName: FADD-like antiapoptotic molecule 1 shortName:FLAME-1  alternativeName: Inhibitor of FLICE shortName:I-FLICE  alternativeName: MACH-related inducer of toxicity shortName:MRIT  alternativeName: Usurpin  component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p43  /component component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p12  /component\")\n	(checksum \"8C6D7E92AE1EB672\")\n	(comment \"FUNCTION Apoptosis regulator protein which may function as a crucial link between cell survival and cell death pathways in mammalian cells. Acts as an inhibitor of TNFRSF6 mediated apoptosis. A proteolytic fragment (p43) is likely retained in the death-inducing signaling complex (DISC) thereby blocking further recruitment and processing of caspase-8 at the complex. Full length and shorter isoforms have been shown either to induce apoptosis or to reduce TNFRSF-triggered apoptosis. Lacks enzymatic (caspase) activity.SUBUNIT TNFRSF6 stimulation triggers recruitment to the death-inducing signaling complex (DISC) formed by TNFRSF6, FADD and caspase-8. A proteolytic fragment (p43) stays associated with the DISC. Also interacts with caspase-10, caspase-3, TRAF1, TRAF2 and Bcl-X(L) (in vitro). Interacts with HBV protein X.TISSUE SPECIFICITY Widely expressed. Higher expression in skeletal muscle, pancreas, heart, kidney, placenta, and peripheral blood leukocytes. Also detected in diverse cell lines. Isoform 8 is predominantly expressed in testis and skeletal muscle.INDUCTION Repressed by IL2/interleukin-2 after TCR stimulation, during progression to the S phase of the cell cycle.DOMAIN The caspase domain lacks the active sites residues involved in catalysis.PTM Proteolytically processed; probably by caspase-8. Processing likely occurs at the DISC and generates subunit p43 and p12.SIMILARITY Belongs to the peptidase C14A family.SIMILARITY Contains 2 DED (death effector) domains.\")\n	(DB_ID 145326)\n	(description \"recommendedName: CASP8 and FADD-like apoptosis regulator  alternativeName: Caspase homolog shortName:CASH  alternativeName: Caspase-eight-related protein shortName:Casper  alternativeName: Caspase-like apoptosis regulatory protein shortName:CLARP  alternativeName: Cellular FLICE-like inhibitory protein shortName:c-FLIP  alternativeName: FADD-like antiapoptotic molecule 1 shortName:FLAME-1  alternativeName: Inhibitor of FLICE shortName:I-FLICE  alternativeName: MACH-related inducer of toxicity shortName:MRIT  alternativeName: Usurpin  component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p43  /component component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p12  /component\")\n	(geneName\n		\"CFLAR\"\n		\"CASH\"\n		\"CASP8AP1\"\n		\"CLARP\"\n		\"MRIT\")\n	(identifier \"O15519\")\n	(isoformParent [GK_51926])\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Host-virus interaction\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Repeat\")\n	(name \"CFLAR\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CFLAR_HUMAN\"\n		\"B4DJE0\"\n		\"B7Z9F9\"\n		\"O14673\"\n		\"O14674\"\n		\"O14675\"\n		\"O15137\"\n		\"O15138\"\n		\"O15356\"\n		\"O15510\"\n		\"O43618\"\n		\"O43619\"\n		\"O43620\"\n		\"O60458\"\n		\"O60459\"\n		\"Q53TS6\"\n		\"Q54AF1\"\n		\"Q96TE4\"\n		\"Q9UEW1\")\n	(sequenceLength 480)\n	(species [GK_48887])\n	(variantIdentifier \"O15519-2\"))\n\n([GK_1470062] of  EntityCompartment\n\n	(_displayName \"other organism cell\")\n	(accession \"0044216\")\n	(DB_ID 1470062)\n	(definition \"A cell of a secondary organism with which the first organism is interacting.\")\n	(instanceOf [GK_447577])\n	(name \"other organism cell\")\n	(referenceDatabase [GK_1]))\n\n([GK_148293] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:O60814 HIST1H2BK recommendedName: Histone H2B type 1-K shortName:H2B K  alternativeName: HIRA-interacting protein 1 \")\n	(checksum \"1\")\n	(comment \"SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 148293)\n	(description \"recommendedName: Histone H2B type 1-K shortName:H2B K  alternativeName: HIRA-interacting protein 1 \")\n	(geneName\n		\"HIST1H2BK\"\n		\"H2BFT\"\n		\"HIRIP1\")\n	(identifier \"O60814\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Antibiotic\"\n		\"Antimicrobial\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BK\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1K_HUMAN\"\n		\"A8K7P7\"\n		\"Q2VPI7\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_148294] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q96A08 HIST1H2BA recommendedName: Histone H2B type 1-A  alternativeName: Histone H2B, testis  alternativeName: Testis-specific histone H2B \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-122 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 148294)\n	(description \"recommendedName: Histone H2B type 1-A  alternativeName: Histone H2B, testis  alternativeName: Testis-specific histone H2B \")\n	(geneName\n		\"HIST1H2BA\"\n		\"TSH2B\")\n	(identifier \"Q96A08\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BA\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1A_HUMAN\"\n		\"B2R544\"\n		\"Q6NZ98\"\n		\"Q6NZA0\"\n		\"Q6NZA1\")\n	(sequenceLength 127)\n	(species [GK_48887]))\n\n([GK_148296] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q8N257 HIST3H2BB recommendedName: Histone H2B type 3-B  alternativeName: H2B type 12 \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 148296)\n	(description \"recommendedName: Histone H2B type 3-B  alternativeName: H2B type 12 \")\n	(geneName \"HIST3H2BB\")\n	(identifier \"Q8N257\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST3H2BB\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B3B_HUMAN\"\n		\"A4FU05\"\n		\"Q3ZCP6\"\n		\"Q5TA30\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_157508] of  Affiliation\n\n	(_displayName \"University of Lausanne\")\n	(address \"Department of Biochemistry <br>Chemin des Boveressses 155 <br>CH-1066 Epalinges <br>Switzerland\")\n	(DB_ID 157508)\n	(name \"University of Lausanne\"))\n\n([GK_158007] of  Affiliation\n\n	(_displayName \"Cold Spring Harbor Laboratory\")\n	(address \"1 Bungtown Road, Cold Spring Harbor, NY 11724\")\n	(DB_ID 158007)\n	(name\n		\"Cold Spring Harbor Laboratory\"\n		\"CSHL\"))\n\n([GK_158417] of  SimpleEntity\n\n	(_displayName \"Zn2+ [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_29427])\n	(DB_ID 158417)\n	(name\n		\"Zn2+\"\n		\"Zn++\"\n		\"Zinc ion\"\n		\"Zn(II)\")\n	(referenceEntity [GK_114751])\n	(stableIdentifier [GK_366839]))\n\n([GK_159968] of  GO_BiologicalProcess\n\n	(_displayName \"response to stimulus\")\n	(accession \"0050896\")\n	(DB_ID 159968)\n	(definition \"Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus. The process begins with detection of the stimulus and ends with a change in state or activity or the cell or organism.\")\n	(instanceOf [GK_106])\n	(name \"response to stimulus\")\n	(referenceDatabase [GK_1]))\n\n([GK_159979] of  GO_CellularComponent\n\n	(_displayName \"protein complex\")\n	(accession \"0043234\")\n	(DB_ID 159979)\n	(definition \"Any macromolecular complex composed of two or more polypeptide subunits, which may or may not be identical. Protein complexes may have other associated non-protein prosthetic groups, such as nucleotides, metal ions or other small molecules.\")\n	(instanceOf [GK_258170])\n	(name \"protein complex\")\n	(referenceDatabase [GK_1]))\n\n([GK_159990] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of biological process\")\n	(accession \"0048518\")\n	(DB_ID 159990)\n	(definition \"Any process that activates or increases the frequency, rate or extent of a biological process. Biological processes are regulated by many means; examples include the control of gene expression, protein modification or interaction with a protein or substrate molecule.\")\n	(instanceOf [GK_139810])\n	(name \"positive regulation of biological process\")\n	(positivelyRegulate [GK_106])\n	(referenceDatabase [GK_1]))\n\n([GK_160005] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of biological process\")\n	(accession \"0048519\")\n	(DB_ID 160005)\n	(definition \"Any process that stops, prevents, or reduces the frequency, rate or extent of a biological process. Biological processes are regulated by many means; examples include the control of gene expression, protein modification or interaction with a protein or substrate molecule.\")\n	(instanceOf [GK_139810])\n	(name \"negative regulation of biological process\")\n	(negativelyRegulate [GK_106])\n	(referenceDatabase [GK_1]))\n\n([GK_160064] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of cellular process\")\n	(accession \"0048522\")\n	(DB_ID 160064)\n	(definition \"Any process that activates or increases the frequency, rate or extent of a cellular process, any of those that are carried out at the cellular level, but are not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level.\")\n	(instanceOf\n		[GK_159990]\n		[GK_139809])\n	(name \"positive regulation of cellular process\")\n	(positivelyRegulate [GK_143])\n	(referenceDatabase [GK_1]))\n\n([GK_160114] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of catalytic activity\")\n	(accession \"0050790\")\n	(DB_ID 160114)\n	(definition \"Any process that modulates the activity of an enzyme.\")\n	(instanceOf\n		[GK_3320]\n		[GK_258234])\n	(name \"regulation of catalytic activity\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_97]))\n\n([GK_160116] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of catalytic activity\")\n	(accession \"0043085\")\n	(DB_ID 160116)\n	(definition \"Any process that activates or increases the activity of an enzyme.\")\n	(instanceOf\n		[GK_422587]\n		[GK_160114])\n	(name \"positive regulation of catalytic activity\")\n	(positivelyRegulate [GK_97])\n	(referenceDatabase [GK_1]))\n\n([GK_160200] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of cellular process\")\n	(accession \"0048523\")\n	(DB_ID 160200)\n	(definition \"Any process that stops, prevents, or reduces the frequency, rate or extent of a cellular process, any of those that are carried out at the cellular level, but are not necessarily restricted to a single cell. For example, cell communication occurs among more than one cell, but occurs at the cellular level.\")\n	(instanceOf\n		[GK_160005]\n		[GK_139809])\n	(name \"negative regulation of cellular process\")\n	(negativelyRegulate [GK_143])\n	(referenceDatabase [GK_1]))\n\n([GK_160227] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of programmed cell death\")\n	(accession \"0043068\")\n	(DB_ID 160227)\n	(definition \"Any process that activates or increases the frequency, rate or extent of programmed cell death, cell death resulting from activation of endogenous cellular processes.\")\n	(instanceOf\n		[GK_422583]\n		[GK_139813])\n	(name \"positive regulation of programmed cell death\")\n	(positivelyRegulate [GK_2274])\n	(referenceDatabase [GK_1]))\n\n([GK_160291] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of hydrolase activity\")\n	(accession \"0051336\")\n	(DB_ID 160291)\n	(definition \"Any process that modulates the frequency, rate or extent of hydrolase activity, the catalysis of the hydrolysis of various bonds, e.g. C-O, C-N, C-C, phosphoric anhydride bonds, etc. Hydrolase is the systematic name for any enzyme of EC class 3.\")\n	(instanceOf [GK_160114])\n	(name \"regulation of hydrolase activity\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_450]))\n\n([GK_160292] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of cysteine-type endopeptidase activity involved in apoptotic process\")\n	(accession \"0043281\")\n	(componentOf [GK_2273])\n	(DB_ID 160292)\n	(definition \"Any process that modulates the activity of a cysteine-type endopeptidase involved in apoptosis.\")\n	(instanceOf\n		[GK_139812]\n		[GK_1043016])\n	(name \"regulation of cysteine-type endopeptidase activity involved in apoptotic process\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2133460]))\n\n([GK_160293] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of hydrolase activity\")\n	(accession \"0051345\")\n	(DB_ID 160293)\n	(definition \"Any process that activates or increases the frequency, rate or extent of hydrolase activity, the catalysis of the hydrolysis of various bonds.\")\n	(instanceOf\n		[GK_160116]\n		[GK_160291])\n	(name \"positive regulation of hydrolase activity\")\n	(positivelyRegulate [GK_450])\n	(referenceDatabase [GK_1]))\n\n([GK_160294] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of cysteine-type endopeptidase activity involved in apoptotic process\")\n	(accession \"0043280\")\n	(componentOf [GK_2273])\n	(DB_ID 160294)\n	(definition \"Any process that activates or increases the activity of a cysteine-type endopeptidase involved in the apoptotic process.\")\n	(instanceOf\n		[GK_160292]\n		[GK_1996045])\n	(name \"positive regulation of cysteine-type endopeptidase activity involved in apoptotic process\")\n	(positivelyRegulate [GK_2133460])\n	(referenceDatabase [GK_1]))\n\n([GK_160483] of  GO_CellularComponent\n\n	(_displayName \"extraorganismal space\")\n	(accession \"0043245\")\n	(DB_ID 160483)\n	(definition \"The environmental space outside of an organism; this may be a host organism in the case of parasitic and symbiotic organisms.\")\n	(instanceOf [GK_258201])\n	(name \"extraorganismal space\")\n	(referenceDatabase [GK_1]))\n\n([GK_160565] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of apoptotic process\")\n	(accession \"0043065\")\n	(DB_ID 160565)\n	(definition \"Any process that activates or increases the frequency, rate or extent of cell death by apoptotic process.\")\n	(instanceOf\n		[GK_139812]\n		[GK_160227])\n	(name \"positive regulation of apoptotic process\")\n	(positivelyRegulate [GK_2273])\n	(referenceDatabase [GK_1]))\n\n([GK_161217] of  GO_CellularComponent\n\n	(_displayName \"death-inducing signaling complex\")\n	(accession \"0031264\")\n	(DB_ID 161217)\n	(definition \"A protein complex formed by the association of signaling proteins with a death receptor upon ligand binding. The complex includes procaspases and death domain-containing proteins in addition to the ligand-bound receptor.\")\n	(instanceOf\n		[GK_159979]\n		[GK_258155])\n	(name \"death-inducing signaling complex\")\n	(referenceDatabase [GK_1]))\n\n([GK_163570] of  Person\n\n	(_displayName \"Wernstedt, C\")\n	(DB_ID 163570)\n	(firstname \"Christier\")\n	(initial \"C\")\n	(surname \"Wernstedt\"))\n\n([GK_164487] of  Taxon\n\n	(_displayName \"root\")\n	(DB_ID 164487)\n	(name \"root\"))\n\n([GK_165085] of  InstanceEdit\n\n	(_displayName \"Gillespie, ME, 0000-00-00 00:00:00\")\n	(author [GK_73681])\n	(dateTime \"0000-00-00 00:00:00\")\n	(DB_ID 165085))\n\n([GK_165094] of  InstanceEdit\n\n	(_displayName \"Vaux, D, 0000-00-00 00:00:00\")\n	(author [GK_141456])\n	(dateTime \"0000-00-00 00:00:00\")\n	(DB_ID 165094))\n\n([GK_165955] of  Person\n\n	(_displayName \"Hofmann, K\")\n	(DB_ID 165955)\n	(initial \"K\")\n	(surname \"Hofmann\"))\n\n([GK_167743] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Beta-2-microglobulin [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 167743)\n	(endCoordinate 119)\n	(name \"Beta-2-microglobulin\")\n	(referenceEntity [GK_50591])\n	(species [GK_48887])\n	(stableIdentifier [GK_368685])\n	(startCoordinate 21))\n\n([GK_167908] of  Affiliation\n\n	(_displayName \"University of Texas Southwestern Medical Center at Dallas\")\n	(DB_ID 167908)\n	(name \"University of Texas Southwestern Medical Center at Dallas\"))\n\n([GK_167911] of  Person\n\n	(_displayName \"Luo, F\")\n	(affiliation [GK_167908])\n	(DB_ID 167911)\n	(firstname \"Feng\")\n	(initial \"F\")\n	(surname \"Luo\"))\n\n([GK_168344] of  InstanceEdit\n\n	(_displayName \"Luo, F, 2005-11-16 12:05:41\")\n	(author [GK_167911])\n	(dateTime \"2005-11-16 12:05:41\")\n	(DB_ID 168344))\n\n([GK_168651] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] RIPK1 [cytosol]\")\n	(authored [GK_168344])\n	(compartment [GK_70101])\n	(DB_ID 168651)\n	(endCoordinate 671)\n	(name\n		\"RIPK1\"\n		\"receptor interacting protein 1\")\n	(referenceEntity [GK_63119])\n	(species [GK_48887])\n	(stableIdentifier [GK_364408])\n	(startCoordinate 1))\n\n([GK_1687] of  GO_MolecularFunction\n\n	(_displayName \"peptidase activity\")\n	(accession \"0008233\")\n	(DB_ID 1687)\n	(definition \"Catalysis of the hydrolysis of a peptide bond. A peptide bond is a covalent bond formed when the carbon atom from the carboxyl group of one amino acid shares electrons with the nitrogen atom from the amino group of a second amino acid.\")\n	(ecNumber\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\"\n		\"3.4\")\n	(instanceOf [GK_450])\n	(name \"peptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_171090] of  Person\n\n	(_displayName \"Soutar, AK\")\n	(DB_ID 171090)\n	(firstname \"Anne K\")\n	(initial \"AK\")\n	(surname \"Soutar\"))\n\n([GK_171348] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of response to stimulus\")\n	(accession \"0048583\")\n	(DB_ID 171348)\n	(definition \"Any process that modulates the frequency, rate or extent of a response to a stimulus. Response to stimulus is a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus.\")\n	(instanceOf [GK_139810])\n	(name \"regulation of response to stimulus\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_159968]))\n\n([GK_171449] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of response to stimulus\")\n	(accession \"0048585\")\n	(DB_ID 171449)\n	(definition \"Any process that stops, prevents, or reduces the frequency, rate or extent of a response to a stimulus. Response to stimulus is a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus.\")\n	(instanceOf\n		[GK_160005]\n		[GK_171348])\n	(name \"negative regulation of response to stimulus\")\n	(negativelyRegulate [GK_159968])\n	(referenceDatabase [GK_1]))\n\n([GK_171571] of  GO_BiologicalProcess\n\n	(_displayName \"cellular response to stimulus\")\n	(accession \"0051716\")\n	(DB_ID 171571)\n	(definition \"Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus. The process begins with detection of the stimulus by a cell and ends with a change in state or activity or the cell.\")\n	(instanceOf\n		[GK_3132308]\n		[GK_159968])\n	(name \"cellular response to stimulus\")\n	(referenceDatabase [GK_1]))\n\n([GK_174719] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] perlecan [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 174719)\n	(endCoordinate 4391)\n	(name\n		\"perlecan\"\n		\"HSPG2\")\n	(referenceEntity [GK_61584])\n	(species [GK_48887])\n	(stableIdentifier [GK_366563])\n	(startCoordinate 22))\n\n([GK_178760] of  Summation\n\n	(_displayName \"Monomeric caspase-8 zymogens undergo dimerization and subseq...\")\n	(DB_ID 178760)\n	(text \"Monomeric caspase-8 zymogens undergo dimerization and subsequent conformational changes at the FASL:FAS Receptor Trimer:FADD:procaspase-8 receptor complex leading to the formation of the catalytically active form of procaspase-8.\"))\n\n([GK_178761] of  Summation\n\n	(_displayName \"FASL (FAS antigen ligand) binds FAS receptor (CD95) (Brunner...\")\n	(DB_ID 178761)\n	(literatureReference [GK_140513])\n	(text \"FASL (FAS antigen ligand) binds FAS receptor (CD95) (Brunner et al. 1995). \"))\n\n([GK_178762] of  Summation\n\n	(_displayName \"FADD (FAS-associating death domain-containing) protein binds...\")\n	(DB_ID 178762)\n	(literatureReference [GK_140495])\n	(text \"FADD (FAS-associating death domain-containing) protein binds FASL:FAS receptor trimer through interaction of death domains of FADD and FAS (Boldin et al. 1995).\"))\n\n([GK_178763] of  Summation\n\n	(_displayName \"FADD recruits caspase-10 precursor (pro-caspase-10) to FASL:...\")\n	(DB_ID 178763)\n	(literatureReference\n		[GK_141306]\n		[GK_141314])\n	(text \"FADD recruits caspase-10 precursor (pro-caspase-10) to FASL:FAS receptor trimer (Wang et al. 2001, Sprick et al. 2002). \"))\n\n([GK_178764] of  Summation\n\n	(_displayName \"Caspase-8 precursor (Pro-Caspase-8, also known as MACH) bind...\")\n	(DB_ID 178764)\n	(literatureReference [GK_140941])\n	(text \"Caspase-8 precursor (Pro-Caspase-8, also known as MACH) binds the complex of FADD and FASL:FAS receptor trimer (Boldin et al. 1996). \"))\n\n([GK_178769] of  Summation\n\n	(_displayName \"Caspase-8 zymogens are present in the cells as inactive mono...\")\n	(DB_ID 178769)\n	(text \"Caspase-8 zymogens are present in the cells as inactive monomers, containing a large N-terminal prodomain with two death effector domains (DED), and a C-terminal catalytic subunit composed of small and a large domains separated by a smaller linker region [Donepudi M et al 2003; Keller N et al 2009]. Dimerization is required for the caspase-8 activation [Donepudi M et al 2003]. Once dimerized, caspase-8 zymogen undergoes a series of autoproteolytic cleavage events at aspartic acid residues in their interdomain linker regions. A second cleavage event between the the N-terminal prodomain and the catalytic domain releases the active caspase from the activation complex into the cytosol. The resulting fully active enzyme is a homodimer of catalytic domains, where each domain is composed of a large p18 and a small p10 subunit [Keller N et al 2009; Oberst A et al 2010].\"))\n\n([GK_1791] of  GO_MolecularFunction\n\n	(_displayName \"metalloendopeptidase activity\")\n	(accession \"0004222\")\n	(DB_ID 1791)\n	(definition \"Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions.\")\n	(ecNumber\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\"\n		\"3.4.24\")\n	(instanceOf\n		[GK_1793]\n		[GK_1795])\n	(name \"metalloendopeptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_179238] of  Summation\n\n	(_displayName \"TNF-alpha (tumor necrosis factor-alpha, TNF) binds TNF-RI (t...\")\n	(DB_ID 179238)\n	(literatureReference [GK_140945])\n	(text \"TNF-alpha (tumor necrosis factor-alpha, TNF) binds TNF-RI (tumor necrosis factor receptor superfamily member 1A, TNFRSF1A) (Stauber et al. 1988).\"))\n\n([GK_179239] of  Summation\n\n	(_displayName \"Monomeric caspase-8 zymogens undergo dimerization and subseq...\")\n	(DB_ID 179239)\n	(text \"Monomeric caspase-8 zymogens undergo dimerization and subsequent conformational changes at the TRADD:TRAF2:RIP1:FADD:Capase-8 receptor complex leading to the formation of the catalytically active form of procaspase-8.\"))\n\n([GK_179240] of  Summation\n\n	(_displayName \"Caspase-8-precursor (pro-caspase-8) binds TRAF2:TRADD:RIP1:F...\")\n	(DB_ID 179240)\n	(literatureReference [GK_140979])\n	(text \"Caspase-8-precursor (pro-caspase-8) binds TRAF2:TRADD:RIP1:FADD complex (Micheau and Tschopp, 2003). \"))\n\n([GK_179243] of  Summation\n\n	(_displayName \"TRAIL (TNF-related apoptosis-inducing ligand) binds TRAIL re...\")\n	(DB_ID 179243)\n	(literatureReference [GK_141147])\n	(text \"TRAIL (TNF-related apoptosis-inducing ligand) binds TRAIL receptor-2 (TNFRSF10B) (Wiley et al. 1995). \"))\n\n([GK_179244] of  Summation\n\n	(_displayName \"Monomeric caspase-8 zymogens undergo dimerization and subseq...\")\n	(DB_ID 179244)\n	(text \"Monomeric caspase-8 zymogens undergo dimerization and subsequent conformational changes at the TRAIL:TRAIL receptor-2:FADD receptor complex leading to the formation of the catalytically active form of procaspase-8.\"))\n\n([GK_179245] of  Summation\n\n	(_displayName \"The trimeric complex of TRAIL and TRAIL receptor-2 (TRAIL:TR...\")\n	(DB_ID 179245)\n	(literatureReference [GK_141128])\n	(text \"The trimeric complex of TRAIL and TRAIL receptor-2 (TRAIL:TRAIL receptor-2) binds FADD (Sprick et al. 2000).\"))\n\n([GK_179246] of  Summation\n\n	(_displayName \"Caspase-10 precursor is recruited to the TRAIL:TRAIL recepto...\")\n	(DB_ID 179246)\n	(literatureReference\n		[GK_141306]\n		[GK_141314])\n	(text \"Caspase-10 precursor is recruited to the TRAIL:TRAIL receptor-2:FADD complex (Sprick et al. 2002) through interaction of death effector domains of caspase-10 and FADD (Wang et al. 2001). \"))\n\n([GK_179247] of  Summation\n\n	(_displayName \"FADD recruits caspase-8 precursor to trimeric complex of TRA...\")\n	(DB_ID 179247)\n	(literatureReference [GK_141128])\n	(text \"FADD recruits caspase-8 precursor to trimeric complex of TRAIL and TRAIL receptor 2 (Sprick et al. 2000).\"))\n\n([GK_179272] of  Summation\n\n	(_displayName \"The complex of FASL (FAS antigen ligand) and FAS receptor (C...\")\n	(DB_ID 179272)\n	(literatureReference [GK_140505])\n	(text \"The complex of FASL (FAS antigen ligand) and FAS receptor (CD95) trimerizes (Kischkel et al. 1995).\"))\n\n([GK_179273] of  Summation\n\n	(_displayName \"The complex of TRAIL and TRAIL receptor-2 trimerizes (Sprick...\")\n	(DB_ID 179273)\n	(literatureReference [GK_141128])\n	(text \"The complex of TRAIL and TRAIL receptor-2 trimerizes (Sprick et al. 2000). \"))\n\n([GK_1793] of  GO_MolecularFunction\n\n	(_displayName \"endopeptidase activity\")\n	(accession \"0004175\")\n	(DB_ID 1793)\n	(definition \"Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain.\")\n	(instanceOf [GK_376448])\n	(name \"endopeptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_1795] of  GO_MolecularFunction\n\n	(_displayName \"metallopeptidase activity\")\n	(accession \"0008237\")\n	(DB_ID 1795)\n	(definition \"Catalysis of the hydrolysis of peptide bonds by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions.\")\n	(instanceOf [GK_376448])\n	(name \"metallopeptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_179878] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] ADAM17 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 179878)\n	(endCoordinate 824)\n	(name\n		\"ADAM17\"\n		\"ADAM 17 metalloprotease\"\n		\"ADAM 17 sheddase\"\n		\"TACE\")\n	(referenceEntity [GK_49678])\n	(species [GK_48887])\n	(stableIdentifier [GK_363923])\n	(startCoordinate 215))\n\n([GK_183154] of  ReferenceDatabase\n\n	(_displayName \"PubChem Substance\")\n	(accessUrl \"http://pubchem.ncbi.nlm.nih.gov/summary/summary.cgi?sid=###ID###\")\n	(DB_ID 183154)\n	(name \"PubChem Substance\")\n	(url \"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pcsubstance\"))\n\n([GK_183308] of  DatabaseIdentifier\n\n	(_displayName \"PubChem Substance:11533263\")\n	(DB_ID 183308)\n	(identifier \"11533263\")\n	(referenceDatabase [GK_183154]))\n\n([GK_183728] of  DatabaseIdentifier\n\n	(_displayName \"PubChem Substance:8145461\")\n	(DB_ID 183728)\n	(identifier \"8145461\")\n	(referenceDatabase [GK_183154]))\n\n([GK_186930] of  Person\n\n	(_displayName \"Yamaguchi, I\")\n	(DB_ID 186930)\n	(firstname \"Isomaro\")\n	(initial \"I\")\n	(surname \"Yamaguchi\"))\n\n([GK_189563] of  Person\n\n	(_displayName \"Hasegawa, K\")\n	(DB_ID 189563)\n	(initial \"K\")\n	(surname \"Hasegawa\"))\n\n([GK_193525] of  Person\n\n	(_displayName \"Suzuki, Y\")\n	(DB_ID 193525)\n	(firstname \"Yasuyuki\")\n	(initial \"Y\")\n	(surname \"Suzuki\"))\n\n([GK_193908] of  Person\n\n	(_displayName \"Schneider, P\")\n	(DB_ID 193908)\n	(initial \"P\")\n	(surname \"Schneider\"))\n\n([GK_194488] of  Person\n\n	(_displayName \"Yamashita, T\")\n	(DB_ID 194488)\n	(initial \"T\")\n	(surname \"Yamashita\"))\n\n([GK_1996045] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of cysteine-type endopeptidase activity\")\n	(accession \"2001056\")\n	(DB_ID 1996045)\n	(definition \"Any process that activates or increases the frequency, rate or extent of cysteine-type endopeptidase activity.\")\n	(instanceOf\n		[GK_423068]\n		[GK_1043016])\n	(name \"positive regulation of cysteine-type endopeptidase activity\")\n	(positivelyRegulate [GK_4344])\n	(referenceDatabase [GK_1]))\n\n([GK_2] of  ReferenceDatabase\n\n	(_displayName \"UniProt\")\n	(accessUrl \"http://www.uniprot.org/entry/###ID###\")\n	(DB_ID 2)\n	(name\n		\"UniProt\"\n		\"SWALL\"\n		\"SPTR\"\n		\"SwissProt\"\n		\"SPTREMBL\")\n	(url \"http://www.uniprot.org\"))\n\n([GK_2016988] of  StableIdentifier\n\n	(_displayName \"REACT_116419.2\")\n	(DB_ID 2016988)\n	(identifier \"REACT_116419\")\n	(identifierVersion \"2\"))\n\n([GK_202465] of  Person\n\n	(_displayName \"Thome, M\")\n	(DB_ID 202465)\n	(initial \"M\")\n	(surname \"Thome\"))\n\n([GK_202881] of  Person\n\n	(_displayName \"Wang, S\")\n	(DB_ID 202881)\n	(initial \"S\")\n	(surname \"Wang\"))\n\n([GK_203163] of  Person\n\n	(_displayName \"Yang, W\")\n	(DB_ID 203163)\n	(initial \"W\")\n	(surname \"Yang\"))\n\n([GK_204598] of  Person\n\n	(_displayName \"Yamamoto, S\")\n	(DB_ID 204598)\n	(initial \"S\")\n	(surname \"Yamamoto\"))\n\n([GK_204605] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Kerato-epithelin [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 204605)\n	(endCoordinate 683)\n	(name\n		\"Kerato-epithelin\"\n		\"Transforming growth factor-beta induced protein IG-H3 precursor\"\n		\"BGH3_HUMAN\"\n		\"TGFBI\")\n	(referenceEntity [GK_50810])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358063])\n	(startCoordinate 24))\n\n([GK_205591] of  Person\n\n	(_displayName \"Williams, MG\")\n	(DB_ID 205591)\n	(initial \"MG\")\n	(project \"FlyBase\")\n	(surname \"Williams\"))\n\n([GK_207144] of  Person\n\n	(_displayName \"Lemaitre, B\")\n	(DB_ID 207144)\n	(initial \"B\")\n	(surname \"Lemaitre\"))\n\n([GK_207287] of  Person\n\n	(_displayName \"Eulitz, M\")\n	(DB_ID 207287)\n	(initial \"M\")\n	(surname \"Eulitz\"))\n\n([GK_207482] of  Person\n\n	(_displayName \"Ikeda, S\")\n	(DB_ID 207482)\n	(initial \"S\")\n	(surname \"Ikeda\"))\n\n([GK_207642] of  Person\n\n	(_displayName \"Sawaya, MR\")\n	(DB_ID 207642)\n	(initial \"MR\")\n	(surname \"Sawaya\"))\n\n([GK_207774] of  Person\n\n	(_displayName \"Silverman, N\")\n	(DB_ID 207774)\n	(initial \"N\")\n	(surname \"Silverman\"))\n\n([GK_212143] of  Person\n\n	(_displayName \"Szyf, M\")\n	(DB_ID 212143)\n	(initial \"M\")\n	(surname \"Szyf\"))\n\n([GK_213088] of  Summation\n\n	(_displayName \"Caspase-8 is synthesized as zymogen (procaspase-8) and is fo...\")\n	(DB_ID 213088)\n	(text \"Caspase-8 is synthesized as zymogen (procaspase-8) and is formed from procaspase-8 as a cleavage product. However, the cleavage itself appears not to be sufficient for the formation of an active caspase-8. Only the coordinated dimerization and cleavage of the zymogen produce efficient activation in vitro and apoptosis in cellular systems [Boatright KM and Salvesen GS 2003; Keller N et al 2010; Oberst A et al 2010].<p>The caspase-8 zymogens are present in the cells as inactive monomers, which are recruited to the death-inducing signaling complex (DISC) by homophilic interactions with the DED domain of FADD. The monomeric zymogens undergo dimerization and the subsequent conformational changes at the receptor complex, which results in the formation of catalytically active form of procaspase-8.[Boatright KM et al 2003; Donepudi M et al 2003; Keller N et al 2010; Oberst A et al 2010].\"))\n\n([GK_213104] of  Summation\n\n	(_displayName \"Procaspase-8 monomers undergo dimerization. The dimerization...\")\n	(DB_ID 213104)\n	(text \"Procaspase-8 monomers undergo dimerization. The dimerization event occurs at death-inducing signaling complex (DISC) and results in a reposition of the procaspase-8 inter-subunit linker to become accessible for intermolecular processing by the associated procaspase-8 molecule [Keller N et al 2010; Oberst A et al 2010].\"))\n\n([GK_2132740] of  GO_BiologicalProcess\n\n	(_displayName \"signal transduction in absence of ligand\")\n	(accession \"0038034\")\n	(DB_ID 2132740)\n	(definition \"A series of molecular signals initiated by the absence of a ligand or the withdrawal of a ligand from a receptor.\")\n	(instanceOf [GK_141])\n	(name \"signal transduction in absence of ligand\")\n	(referenceDatabase [GK_1]))\n\n([GK_2133460] of  GO_MolecularFunction\n\n	(_displayName \"cysteine-type endopeptidase activity involved in apoptotic process\")\n	(accession \"0097153\")\n	(componentOf [GK_2273])\n	(DB_ID 2133460)\n	(definition \"Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile, and contributing to the apoptotic process.\")\n	(instanceOf [GK_4344])\n	(name \"cysteine-type endopeptidase activity involved in apoptotic process\")\n	(referenceDatabase [GK_1]))\n\n([GK_2133497] of  GO_BiologicalProcess\n\n	(_displayName \"apoptotic signaling pathway\")\n	(accession \"0097190\")\n	(componentOf [GK_2273])\n	(DB_ID 2133497)\n	(definition \"A series of molecular signals which triggers the apoptotic death of a cell. The pathway starts with reception of a signal, and ends when the execution phase of apoptosis is triggered.\")\n	(instanceOf [GK_141])\n	(name \"apoptotic signaling pathway\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2133501]))\n\n([GK_2133498] of  GO_BiologicalProcess\n\n	(_displayName \"extrinsic apoptotic signaling pathway\")\n	(accession \"0097191\")\n	(DB_ID 2133498)\n	(definition \"A series of molecular signals in which a signal is conveyed from the cell surface to trigger the apoptotic death of a cell. The pathway starts with either a ligand binding to a cell surface receptor, or a ligand being withdrawn from a cell surface receptor (e.g. in the case of signaling by dependence receptors), and ends when the execution phase of apoptosis is triggered.\")\n	(instanceOf\n		[GK_140]\n		[GK_2133497])\n	(name \"extrinsic apoptotic signaling pathway\")\n	(referenceDatabase [GK_1]))\n\n([GK_2133499] of  GO_BiologicalProcess\n\n	(_displayName \"extrinsic apoptotic signaling pathway in absence of ligand\")\n	(accession \"0097192\")\n	(DB_ID 2133499)\n	(definition \"A series of molecular signals in which a signal is conveyed from the cell surface to trigger the apoptotic death of a cell. The pathway starts with withdrawal of a ligand from a cell surface receptor, and ends when the execution phase of apoptosis is triggered.\")\n	(instanceOf\n		[GK_2132740]\n		[GK_2133498])\n	(name \"extrinsic apoptotic signaling pathway in absence of ligand\")\n	(referenceDatabase [GK_1]))\n\n([GK_2133501] of  GO_BiologicalProcess\n\n	(_displayName \"execution phase of apoptosis\")\n	(accession \"0097194\")\n	(componentOf [GK_2273])\n	(DB_ID 2133501)\n	(definition \"A stage of the apoptotic process that starts with the controlled breakdown of the cell through the action of effector caspases or other effector molecules (e.g. cathepsins, calpains etc.). Key steps of the execution phase are rounding-up of the cell, retraction of pseudopodes, reduction of cellular volume (pyknosis), chromatin condensation, nuclear fragmentation (karyorrhexis), plasma membrane blebbing and fragmentation of the cell into apoptotic bodies. The execution phase ends when the cell has died.\")\n	(instanceOf [GK_3132308])\n	(name \"execution phase of apoptosis\")\n	(referenceDatabase [GK_1]))\n\n([GK_2133509] of  GO_BiologicalProcess\n\n	(_displayName \"activation of cysteine-type endopeptidase activity\")\n	(accession \"0097202\")\n	(DB_ID 2133509)\n	(definition \"Any process that initiates the activity of the inactive enzyme cysteine-type endopeptidase.\")\n	(instanceOf [GK_1996045])\n	(name \"activation of cysteine-type endopeptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_2133692] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of apoptotic signaling pathway\")\n	(accession \"2001233\")\n	(DB_ID 2133692)\n	(definition \"Any process that modulates the frequency, rate or extent of apoptotic signaling pathway.\")\n	(instanceOf\n		[GK_6192]\n		[GK_139812])\n	(name \"regulation of apoptotic signaling pathway\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2133497]))\n\n([GK_2133693] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of apoptotic signaling pathway\")\n	(accession \"2001234\")\n	(DB_ID 2133693)\n	(definition \"Any process that stops, prevents or reduces the frequency, rate or extent of apoptotic signaling pathway.\")\n	(instanceOf\n		[GK_6196]\n		[GK_2133692])\n	(name \"negative regulation of apoptotic signaling pathway\")\n	(negativelyRegulate [GK_2133497])\n	(referenceDatabase [GK_1]))\n\n([GK_2133695] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of extrinsic apoptotic signaling pathway\")\n	(accession \"2001236\")\n	(DB_ID 2133695)\n	(definition \"Any process that modulates the frequency, rate or extent of extrinsic apoptotic signaling pathway.\")\n	(instanceOf [GK_2133692])\n	(name \"regulation of extrinsic apoptotic signaling pathway\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2133498]))\n\n([GK_2133696] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of extrinsic apoptotic signaling pathway\")\n	(accession \"2001237\")\n	(DB_ID 2133696)\n	(definition \"Any process that stops, prevents or reduces the frequency, rate or extent of extrinsic apoptotic signaling pathway.\")\n	(instanceOf\n		[GK_2133693]\n		[GK_2133695])\n	(name \"negative regulation of extrinsic apoptotic signaling pathway\")\n	(negativelyRegulate [GK_2133498])\n	(referenceDatabase [GK_1]))\n\n([GK_2133698] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of extrinsic apoptotic signaling pathway in absence of ligand\")\n	(accession \"2001239\")\n	(DB_ID 2133698)\n	(definition \"Any process that modulates the frequency, rate or extent of extrinsic apoptotic signaling pathway in absence of ligand.\")\n	(instanceOf [GK_2133695])\n	(name \"regulation of extrinsic apoptotic signaling pathway in absence of ligand\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2133499]))\n\n([GK_216603] of  Person\n\n	(_displayName \"Munier, FL\")\n	(DB_ID 216603)\n	(initial \"FL\")\n	(surname \"Munier\"))\n\n([GK_216645] of  Person\n\n	(_displayName \"Schorderet, DF\")\n	(DB_ID 216645)\n	(initial \"DF\")\n	(surname \"Schorderet\"))\n\n([GK_217036] of  Person\n\n	(_displayName \"Lee, M\")\n	(DB_ID 217036)\n	(initial \"M\")\n	(surname \"Lee\"))\n\n([GK_217173] of  Person\n\n	(_displayName \"Seki, M\")\n	(DB_ID 217173)\n	(initial \"M\")\n	(surname \"Seki\"))\n\n([GK_2266716] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of execution phase of apoptosis\")\n	(accession \"1900117\")\n	(DB_ID 2266716)\n	(definition \"Any process that modulates the frequency, rate or extent of execution phase of apoptosis.\")\n	(instanceOf [GK_139812])\n	(name \"regulation of execution phase of apoptosis\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2133501]))\n\n([GK_2273] of  GO_BiologicalProcess\n\n	(_displayName \"apoptotic process\")\n	(accession \"0006915\")\n	(DB_ID 2273)\n	(definition \"A programmed cell death process which begins when a cell receives an internal (e.g. DNA damage) or external signal (e.g. an extracellular death ligand), and proceeds through a series of biochemical events (signaling pathways) which typically lead to rounding-up of the cell, retraction of pseudopodes, reduction of cellular volume (pyknosis), chromatin condensation, nuclear fragmentation (karyorrhexis), plasma membrane blebbing and fragmentation of the cell into apoptotic bodies. The process ends when the cell has died. The process is divided into a signaling pathway phase, and an execution phase, which is triggered by the former.\")\n	(instanceOf [GK_2274])\n	(name \"apoptotic process\")\n	(referenceDatabase [GK_1]))\n\n([GK_2274] of  GO_BiologicalProcess\n\n	(_displayName \"programmed cell death\")\n	(accession \"0012501\")\n	(DB_ID 2274)\n	(definition \"A process which begins when a cell receives an internal or external signal and activates a series of biochemical events (signaling pathway). The process ends with the death of the cell.\")\n	(instanceOf [GK_2275])\n	(name \"programmed cell death\")\n	(referenceDatabase [GK_1]))\n\n([GK_2275] of  GO_BiologicalProcess\n\n	(_displayName \"cell death\")\n	(accession \"0008219\")\n	(DB_ID 2275)\n	(definition \"Any biological process that results in permanent cessation of all vital functions of a cell. A cell should be considered dead when any one of the following molecular or morphological criteria is met: (1) the cell has lost the integrity of its plasma membrane; (2) the cell, including its nucleus, has undergone complete fragmentation into discrete bodies (frequently referred to as \\\"apoptotic bodies\\\"); and/or (3) its corpse (or its fragments) have been engulfed by an adjacent cell in vivo.\")\n	(instanceOf\n		[GK_2276]\n		[GK_3132308])\n	(name \"cell death\")\n	(referenceDatabase [GK_1]))\n\n([GK_2276] of  GO_BiologicalProcess\n\n	(_displayName \"death\")\n	(accession \"0016265\")\n	(DB_ID 2276)\n	(definition \"A permanent cessation of all vital functions: the end of life; can be applied to a whole organism or to a part of an organism.\")\n	(instanceOf [GK_2587987])\n	(name \"death\")\n	(referenceDatabase [GK_1]))\n\n([GK_231418] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q6FI13 HIST2H2AA3 recommendedName: Histone H2A type 2-A  alternativeName: Histone H2A.2  alternativeName: Histone H2A/o \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Deiminated on Arg-4 in granulocytes upon calcium entry.PTM Phosphorylation on Ser-2 is enhanced during mitosis. Phosphorylation on Ser-2 by RPS6KA5/MSK1 directly represses transcription. Acetylation of H3 inhibits Ser-2 phosphorylation by RPS6KA5/MSK1.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 231418)\n	(description \"recommendedName: Histone H2A type 2-A  alternativeName: Histone H2A.2  alternativeName: Histone H2A/o \")\n	(geneName\n		\"HIST2H2AA3\"\n		\"H2AFO\"\n		\"HIST2H2AA\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST2H2AA4\")\n	(identifier \"Q6FI13\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST2H2AA3\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2A2A_HUMAN\"\n		\"B2R5F0\"\n		\"P20670\")\n	(sequenceLength 130)\n	(species [GK_48887]))\n\n([GK_231427] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P0C5Y9 H2AFB1 recommendedName: Histone H2A-Bbd type 1  alternativeName: H2A Barr body-deficient shortName:H2A.Bbd  alternativeName: H2A Barr body-deficient shortName:H2A.Bbd \")\n	(checksum \"1\")\n	(comment \"SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. May be incorporated into a proportion of nucleosomes, replacing one or more H2A molecules.MISCELLANEOUS In contrast to other H2A histones, it does not contain the conserved residues that are the target of post-translational modifications.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 231427)\n	(description \"recommendedName: Histone H2A-Bbd type 1  alternativeName: H2A Barr body-deficient shortName:H2A.Bbd  alternativeName: H2A Barr body-deficient shortName:H2A.Bbd \")\n	(geneName \"H2AFB1\")\n	(identifier \"P0C5Y9\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Nucleosome core\"\n		\"Nucleus\")\n	(name \"H2AFB1\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2AB1_HUMAN\"\n		\"P98176\"\n		\"Q5TZB2\"\n		\"Q6FG78\"\n		\"Q96PR7\")\n	(sequenceLength 115)\n	(species [GK_48887]))\n\n([GK_231437] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P0C0S5 H2AFZ recommendedName: Histone H2A.Z shortName:H2A/z \")\n	(checksum \"1\")\n	(comment \"PTM Monoubiquitination of Lys-122 gives a specific tag for epigenetic transcriptional repression.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 231437)\n	(description \"recommendedName: Histone H2A.Z shortName:H2A/z \")\n	(geneName\n		\"H2AFZ\"\n		\"H2AZ\")\n	(identifier \"P0C0S5\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Ubl conjugation\")\n	(name \"H2AFZ\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2AZ_HUMAN\"\n		\"B2RD56\"\n		\"P17317\"\n		\"Q6I9U0\")\n	(sequenceLength 128)\n	(species [GK_48887]))\n\n([GK_23175] of  GO_BiologicalProcess\n\n	(_displayName \"activation of cysteine-type endopeptidase activity involved in apoptotic process\")\n	(accession \"0006919\")\n	(componentOf [GK_2273])\n	(DB_ID 23175)\n	(definition \"Any process that initiates the activity of the inactive enzyme cysteine-type endopeptidase in the context of an apoptotic process.\")\n	(instanceOf\n		[GK_160294]\n		[GK_2133509])\n	(name \"activation of cysteine-type endopeptidase activity involved in apoptotic process\")\n	(referenceDatabase [GK_1]))\n\n([GK_2318595] of  Person\n\n	(_displayName \"Wang, Y\")\n	(DB_ID 2318595)\n	(firstname \"Y\")\n	(initial \"Y\")\n	(surname \"Wang\"))\n\n([GK_2393943] of  Person\n\n	(_displayName \"Yamagata, Yuriko\")\n	(DB_ID 2393943)\n	(firstname \"Yuriko\")\n	(initial \"Y\")\n	(surname \"Yamagata\"))\n\n([GK_240941] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:A1E959 ODAM recommendedName: Odontogenic ameloblast-associated protein  alternativeName: Apin \")\n	(checksum \"1\")\n	(comment \"FUNCTION Tooth-associated epithelia protein that probably plays a role in odontogenesis, the complex process that results in the initiation and generation of the tooth. May be incorporated in the enamel matrix at the end of mineralization process.MISCELLANEOUS ODAM protein is the unique constituent of calcifying epithelial odontogenic tumors (CEOTs), also known as Pindborg tumors. CEOTs are benign but locally aggressive pathologic entities arising mainly in the mandible and commonly associated with an unerupted or embedded tooth. They are characterized by the presence of squamous-cell proliferation, calcification, and, notably, amyloid deposits.SIMILARITY Belongs to the ODAM family.\")\n	(DB_ID 240941)\n	(description \"recommendedName: Odontogenic ameloblast-associated protein  alternativeName: Apin \")\n	(geneName\n		\"ODAM\"\n		\"APIN\")\n	(identifier \"A1E959\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Biomineralization\"\n		\"Complete proteome\"\n		\"Glycoprotein\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"ODAM\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"ODAM_HUMAN\"\n		\"Q8WWE5\"\n		\"Q9NWZ9\")\n	(sequenceLength 279)\n	(species [GK_48887]))\n\n([GK_2534135] of  Person\n\n	(_displayName \"Salvesen, G S\")\n	(DB_ID 2534135)\n	(firstname \"G S\")\n	(initial \"GS\")\n	(surname \"Salvesen\"))\n\n([GK_25446] of  GO_BiologicalProcess\n\n	(_displayName \"induction of apoptosis by extracellular signals\")\n	(accession \"0008624\")\n	(DB_ID 25446)\n	(definition \"Any process induced by extracellular signals that directly activates any of the steps required for cell death by apoptosis.\")\n	(instanceOf [GK_6542])\n	(name \"induction of apoptosis by extracellular signals\")\n	(referenceDatabase [GK_1]))\n\n([GK_25449] of  GO_BiologicalProcess\n\n	(_displayName \"extrinsic apoptotic signaling pathway via death domain receptors\")\n	(accession \"0008625\")\n	(DB_ID 25449)\n	(definition \"A series of molecular signals in which a signal is conveyed from the cell surface to trigger the apoptotic death of a cell. The pathway starts with a ligand binding to a death domain receptor on the cell surface, and ends when the execution phase of apoptosis is triggered.\")\n	(instanceOf [GK_2133498])\n	(name \"extrinsic apoptotic signaling pathway via death domain receptors\")\n	(referenceDatabase [GK_1]))\n\n([GK_25482] of  GO_BiologicalProcess\n\n	(_displayName \"activation of pro-apoptotic gene products\")\n	(accession \"0008633\")\n	(componentOf [GK_2273])\n	(DB_ID 25482)\n	(definition \"The conversion of proteins that induce or sustain apoptosis to an active form.\")\n	(instanceOf [GK_160565])\n	(name \"activation of pro-apoptotic gene products\")\n	(referenceDatabase [GK_1]))\n\n([GK_2562543] of  Person\n\n	(_displayName \"Heinrikson, R L\")\n	(DB_ID 2562543)\n	(firstname \"R L\")\n	(initial \"RL\")\n	(surname \"Heinrikson\"))\n\n([GK_2562544] of  Person\n\n	(_displayName \"Micheau, O\")\n	(DB_ID 2562544)\n	(firstname \"O\")\n	(initial \"O\")\n	(surname \"Micheau\"))\n\n([GK_2562547] of  Person\n\n	(_displayName \"Mildner, A M\")\n	(DB_ID 2562547)\n	(firstname \"A M\")\n	(initial \"AM\")\n	(surname \"Mildner\"))\n\n([GK_2562548] of  Person\n\n	(_displayName \"MacFarlane, Marion\")\n	(DB_ID 2562548)\n	(firstname \"Marion\")\n	(initial \"M\")\n	(surname \"MacFarlane\"))\n\n([GK_2562550] of  Complex\n\n	(_displayName \"[Hsa] active caspase-8  [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2562550)\n	(hasComponent\n		[GK_139950]\n		[GK_139950])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_2562575]\n		[GK_2562554])\n	(name \"active caspase-8 \")\n	(species [GK_48887])\n	(stableIdentifier [GK_3203992]))\n\n([GK_2562552] of  Person\n\n	(_displayName \"Tomasselli, A G\")\n	(DB_ID 2562552)\n	(firstname \"A G\")\n	(initial \"AG\")\n	(surname \"Tomasselli\"))\n\n([GK_2562554] of  LiteratureReference\n\n	(_displayName \"The three-dimensional structure of caspase-8: an initiator enzyme in apoptosis\")\n	(author\n		[GK_2562561]\n		[GK_2562555]\n		[GK_2562588]\n		[GK_2562590]\n		[GK_2562576]\n		[GK_2562572]\n		[GK_2562557]\n		[GK_2562587])\n	(DB_ID 2562554)\n	(journal \"Structure\")\n	(pages \"1125-33\")\n	(pubMedIdentifier 10508784)\n	(title \"The three-dimensional structure of caspase-8: an initiator enzyme in apoptosis\")\n	(volume 7)\n	(year 1999))\n\n([GK_2562555] of  Person\n\n	(_displayName \"Kodandapani, L\")\n	(DB_ID 2562555)\n	(firstname \"L\")\n	(initial \"L\")\n	(surname \"Kodandapani\"))\n\n([GK_2562557] of  Person\n\n	(_displayName \"Tomaselli, K J\")\n	(DB_ID 2562557)\n	(firstname \"K J\")\n	(initial \"KJ\")\n	(surname \"Tomaselli\"))\n\n([GK_2562560] of  Person\n\n	(_displayName \"Cain, Kelvin\")\n	(DB_ID 2562560)\n	(firstname \"Kelvin\")\n	(initial \"K\")\n	(surname \"Cain\"))\n\n([GK_2562561] of  Person\n\n	(_displayName \"Blanchard, H\")\n	(DB_ID 2562561)\n	(firstname \"H\")\n	(initial \"H\")\n	(surname \"Blanchard\"))\n\n([GK_2562565] of  Person\n\n	(_displayName \"Watt, W\")\n	(DB_ID 2562565)\n	(firstname \"W\")\n	(initial \"W\")\n	(surname \"Watt\"))\n\n([GK_2562569] of  Person\n\n	(_displayName \"Watenpaugh, K D\")\n	(DB_ID 2562569)\n	(firstname \"K D\")\n	(initial \"KD\")\n	(surname \"Watenpaugh\"))\n\n([GK_2562572] of  Person\n\n	(_displayName \"Wu, J C\")\n	(DB_ID 2562572)\n	(firstname \"J C\")\n	(initial \"JC\")\n	(surname \"Wu\"))\n\n([GK_2562574] of  Person\n\n	(_displayName \"H√§cker, Georg\")\n	(DB_ID 2562574)\n	(firstname \"Georg\")\n	(initial \"G\")\n	(surname \"H√§cker\"))\n\n([GK_2562575] of  LiteratureReference\n\n	(_displayName \"The atomic-resolution structure of human caspase-8, a key activator of apoptosis\")\n	(author\n		[GK_2562565]\n		[GK_2562584]\n		[GK_2562547]\n		[GK_2562543]\n		[GK_2562552]\n		[GK_2562569])\n	(DB_ID 2562575)\n	(journal \"Structure\")\n	(pages \"1135-43\")\n	(pubMedIdentifier 10508785)\n	(title \"The atomic-resolution structure of human caspase-8, a key activator of apoptosis\")\n	(volume 7)\n	(year 1999))\n\n([GK_2562576] of  Person\n\n	(_displayName \"Krebs, J F\")\n	(DB_ID 2562576)\n	(firstname \"J F\")\n	(initial \"JF\")\n	(surname \"Krebs\"))\n\n([GK_2562581] of  Person\n\n	(_displayName \"Leverkus, Martin\")\n	(DB_ID 2562581)\n	(firstname \"Martin\")\n	(initial \"M\")\n	(surname \"Leverkus\"))\n\n([GK_2562584] of  Person\n\n	(_displayName \"Koeplinger, K A\")\n	(DB_ID 2562584)\n	(firstname \"K A\")\n	(initial \"KA\")\n	(surname \"Koeplinger\"))\n\n([GK_2562587] of  Person\n\n	(_displayName \"Gr√ºtter, M G\")\n	(DB_ID 2562587)\n	(firstname \"M G\")\n	(initial \"MG\")\n	(surname \"Gr√ºtter\"))\n\n([GK_2562588] of  Person\n\n	(_displayName \"Mittl, P R\")\n	(DB_ID 2562588)\n	(firstname \"P R\")\n	(initial \"PR\")\n	(surname \"Mittl\"))\n\n([GK_2562590] of  Person\n\n	(_displayName \"Marco, S D\")\n	(DB_ID 2562590)\n	(firstname \"S D\")\n	(initial \"SD\")\n	(surname \"Marco\"))\n\n([GK_258123] of  GO_BiologicalProcess\n\n	(_displayName \"biological regulation\")\n	(accession \"0065007\")\n	(DB_ID 258123)\n	(definition \"Any process that modulates a measurable attribute of any biological process, quality or function.\")\n	(instanceOf [GK_106])\n	(name \"biological regulation\")\n	(referenceDatabase [GK_1]))\n\n([GK_258147] of  GO_CellularComponent\n\n	(_displayName \"cell part\")\n	(accession \"0044464\")\n	(componentOf [GK_356])\n	(DB_ID 258147)\n	(definition \"Any constituent part of a cell, the basic structural and functional unit of all organisms.\")\n	(instanceOf [GK_357])\n	(name \"cell part\")\n	(referenceDatabase [GK_1]))\n\n([GK_258148] of  GO_CellularComponent\n\n	(_displayName \"intracellular part\")\n	(accession \"0044424\")\n	(componentOf [GK_354])\n	(DB_ID 258148)\n	(definition \"Any constituent part of the living contents of a cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm.\")\n	(instanceOf [GK_258147])\n	(name \"intracellular part\")\n	(referenceDatabase [GK_1]))\n\n([GK_258154] of  GO_CellularComponent\n\n	(_displayName \"membrane part\")\n	(accession \"0044425\")\n	(componentOf [GK_383187])\n	(DB_ID 258154)\n	(definition \"Any constituent part of a membrane, a double layer of lipid molecules that encloses all cells, and, in eukaryotes, many organelles; may be a single or double lipid bilayer; also includes associated proteins.\")\n	(instanceOf [GK_357])\n	(name \"membrane part\")\n	(referenceDatabase [GK_1]))\n\n([GK_258155] of  GO_CellularComponent\n\n	(_displayName \"plasma membrane part\")\n	(accession \"0044459\")\n	(componentOf [GK_876])\n	(DB_ID 258155)\n	(definition \"Any constituent part of the plasma membrane, the membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins.\")\n	(instanceOf\n		[GK_258154]\n		[GK_258147])\n	(name \"plasma membrane part\")\n	(referenceDatabase [GK_1]))\n\n([GK_258163] of  GO_CellularComponent\n\n	(_displayName \"cytoplasmic part\")\n	(accession \"0044444\")\n	(componentOf [GK_459])\n	(DB_ID 258163)\n	(definition \"Any constituent part of the cytoplasm, all of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.\")\n	(instanceOf [GK_258148])\n	(name \"cytoplasmic part\")\n	(referenceDatabase [GK_1]))\n\n([GK_258170] of  GO_CellularComponent\n\n	(_displayName \"macromolecular complex\")\n	(accession \"0032991\")\n	(DB_ID 258170)\n	(definition \"A stable assembly of two or more macromolecules, i.e. proteins, nucleic acids, carbohydrates or lipids, in which the constituent parts function together.\")\n	(instanceOf [GK_357])\n	(name \"macromolecular complex\")\n	(referenceDatabase [GK_1]))\n\n([GK_258201] of  GO_CellularComponent\n\n	(_displayName \"extracellular region part\")\n	(accession \"0044421\")\n	(componentOf [GK_984])\n	(DB_ID 258201)\n	(definition \"Any constituent part of the extracellular region, the space external to the outermost structure of a cell. For cells without external protective or external encapsulating structures this refers to space outside of the plasma membrane. This term covers constituent parts of the host cell environment outside an intracellular parasite.\")\n	(instanceOf [GK_357])\n	(name \"extracellular region part\")\n	(referenceDatabase [GK_1]))\n\n([GK_258234] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of molecular function\")\n	(accession \"0065009\")\n	(DB_ID 258234)\n	(definition \"Any process that modulates the frequency, rate or extent of a molecular function, an elemental biological activity occurring at the molecular level, such as catalysis or binding.\")\n	(instanceOf [GK_258123])\n	(name \"regulation of molecular function\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_43]))\n\n([GK_2584552] of  Person\n\n	(_displayName \"Wang, Xiaodong\")\n	(DB_ID 2584552)\n	(firstname \"Xiaodong\")\n	(initial \"X\")\n	(surname \"Wang\"))\n\n([GK_2584570] of  Person\n\n	(_displayName \"Gr√ºtter, Markus G\")\n	(DB_ID 2584570)\n	(firstname \"Markus G\")\n	(initial \"MG\")\n	(surname \"Gr√ºtter\"))\n\n([GK_2584571] of  Person\n\n	(_displayName \"Briand, Christophe\")\n	(DB_ID 2584571)\n	(firstname \"Christophe\")\n	(initial \"C\")\n	(surname \"Briand\"))\n\n([GK_2584573] of  LiteratureReference\n\n	(_displayName \"Insights into the regulatory mechanism for caspase-8 activation\")\n	(author\n		[GK_2584575]\n		[GK_2584574]\n		[GK_2584571]\n		[GK_2584570])\n	(DB_ID 2584573)\n	(journal \"Mol. Cell\")\n	(pages \"543-9\")\n	(pubMedIdentifier 12620240)\n	(title \"Insights into the regulatory mechanism for caspase-8 activation\")\n	(volume 11)\n	(year 2003))\n\n([GK_2584574] of  Person\n\n	(_displayName \"Mac Sweeney, Aengus\")\n	(DB_ID 2584574)\n	(firstname \"Aengus\")\n	(initial \"A\")\n	(surname \"Mac Sweeney\"))\n\n([GK_2584575] of  Person\n\n	(_displayName \"Donepudi, Mrudula\")\n	(DB_ID 2584575)\n	(firstname \"Mrudula\")\n	(initial \"M\")\n	(surname \"Donepudi\"))\n\n([GK_2587987] of  GO_BiologicalProcess\n\n	(_displayName \"single-organism process\")\n	(accession \"0044699\")\n	(DB_ID 2587987)\n	(definition \"A biological process that involves only one organism.\")\n	(instanceOf [GK_106])\n	(name \"single-organism process\")\n	(referenceDatabase [GK_1]))\n\n([GK_2587988] of  GO_BiologicalProcess\n\n	(_displayName \"single organism signaling\")\n	(accession \"0044700\")\n	(DB_ID 2587988)\n	(definition \"A signaling process occurring within a single organism.\")\n	(instanceOf\n		[GK_528244]\n		[GK_2587987])\n	(name \"single organism signaling\")\n	(referenceDatabase [GK_1]))\n\n([GK_258858] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of peptidase activity\")\n	(accession \"0052547\")\n	(DB_ID 258858)\n	(definition \"Any process that modulates the frequency, rate or extent of peptidase activity, the hydrolysis of peptide bonds within proteins.\")\n	(instanceOf [GK_160291])\n	(name \"regulation of peptidase activity\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_1687]))\n\n([GK_258859] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of endopeptidase activity\")\n	(accession \"0052548\")\n	(DB_ID 258859)\n	(definition \"Any process that modulates the frequency, rate or extent of endopeptidase activity, the endohydrolysis of peptide bonds within proteins.\")\n	(instanceOf [GK_258858])\n	(name \"regulation of endopeptidase activity\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_1793]))\n\n([GK_259431] of  GO_CellularComponent\n\n	(_displayName \"host cell\")\n	(accession \"0043657\")\n	(componentOf [GK_6486])\n	(DB_ID 259431)\n	(definition \"A cell within a host organism. Includes the host plasma membrane and any external encapsulating structures such as the host cell wall and cell envelope.\")\n	(instanceOf [GK_1470062])\n	(name \"host cell\")\n	(referenceDatabase [GK_1]))\n\n([GK_259432] of  GO_CellularComponent\n\n	(_displayName \"host cell part\")\n	(accession \"0033643\")\n	(componentOf [GK_259431])\n	(DB_ID 259432)\n	(definition \"Any constituent part of a host cell. The host is defined as the larger of the organisms involved in a symbiotic interaction.\")\n	(instanceOf [GK_447577])\n	(name \"host cell part\")\n	(referenceDatabase [GK_1]))\n\n([GK_259433] of  GO_CellularComponent\n\n	(_displayName \"intracellular region of host\")\n	(accession \"0043656\")\n	(DB_ID 259433)\n	(definition \"That space within the plasma membrane of a host cell.\")\n	(instanceOf [GK_259432])\n	(name \"intracellular region of host\")\n	(referenceDatabase [GK_1]))\n\n([GK_259434] of  GO_CellularComponent\n\n	(_displayName \"host intracellular part\")\n	(accession \"0033646\")\n	(componentOf [GK_259433])\n	(DB_ID 259434)\n	(definition \"Any constituent part of the living contents of a host cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm. The host is defined as the larger of the organisms involved in a symbiotic interaction.\")\n	(instanceOf [GK_259432])\n	(name \"host intracellular part\")\n	(referenceDatabase [GK_1]))\n\n([GK_259437] of  GO_CellularComponent\n\n	(_displayName \"host cell cytoplasm part\")\n	(accession \"0033655\")\n	(componentOf [GK_80155])\n	(DB_ID 259437)\n	(definition \"Any constituent part of the host cell cytoplasm, all of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures. The host is defined as the larger of the organisms involved in a symbiotic interaction.\")\n	(instanceOf [GK_259434])\n	(name \"host cell cytoplasm part\")\n	(referenceDatabase [GK_1]))\n\n([GK_2594924] of  InstanceEdit\n\n	(_displayName \"Gillespie, ME, 2012-11-19\")\n	(author [GK_73681])\n	(dateTime \"2012-11-19 14:31:40\")\n	(DB_ID 2594924))\n\n([GK_264348] of  Person\n\n	(_displayName \"Steiner, DF\")\n	(DB_ID 264348)\n	(initial \"DF\")\n	(surname \"Steiner\"))\n\n([GK_264874] of  ModifiedResidue\n\n	(_displayName \"half cystine at 109\")\n	(coordinate 109)\n	(DB_ID 264874)\n	(psiMod [GK_448182])\n	(referenceSequence [GK_57415]))\n\n([GK_264928] of  ModifiedResidue\n\n	(_displayName \"half cystine at 95\")\n	(coordinate 95)\n	(DB_ID 264928)\n	(psiMod [GK_448182])\n	(referenceSequence [GK_57415]))\n\n([GK_264936] of  ModifiedResidue\n\n	(_displayName \"half cystine at 96\")\n	(coordinate 96)\n	(DB_ID 264936)\n	(psiMod [GK_448182])\n	(referenceSequence [GK_57415]))\n\n([GK_265017] of  ModifiedResidue\n\n	(_displayName \"half cystine at 43\")\n	(coordinate 43)\n	(DB_ID 265017)\n	(psiMod [GK_448182])\n	(referenceSequence [GK_57415]))\n\n([GK_265035] of  ModifiedResidue\n\n	(_displayName \"half cystine at 100\")\n	(coordinate 100)\n	(DB_ID 265035)\n	(psiMod [GK_448182])\n	(referenceSequence [GK_57415]))\n\n([GK_265094] of  ModifiedResidue\n\n	(_displayName \"half cystine at 31\")\n	(coordinate 31)\n	(DB_ID 265094)\n	(psiMod [GK_448182])\n	(referenceSequence [GK_57415]))\n\n([GK_265170] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin A Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 265170)\n	(endCoordinate 110)\n	(hasModifiedResidue\n		[GK_265035]\n		[GK_264874]\n		[GK_264928]\n		[GK_264936])\n	(name \"Insulin A Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_387943])\n	(startCoordinate 90))\n\n([GK_2671784] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Vvi] SPI-2 [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2671784)\n	(endCoordinate 345)\n	(goCellularComponent [GK_436407])\n	(name\n		\"SPI-2\"\n		\"Serine proteinase inhibitor 2\"\n		\"B13R\")\n	(referenceEntity [GK_2671799])\n	(species [GK_2671785])\n	(startCoordinate 1))\n\n([GK_2671785] of  Species\n\n	(_displayName \"Vaccinia virus\")\n	(crossReference [GK_3159243])\n	(DB_ID 2671785)\n	(name\n		\"Vaccinia virus\"\n		\"vaccinia virus VV\")\n	(superTaxon [GK_3162966]))\n\n([GK_2671786] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Cvi] CRMA [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2671786)\n	(endCoordinate 341)\n	(goCellularComponent [GK_436407])\n	(name\n		\"CRMA\"\n		\"Serine proteinase inhibitor 2\"\n		\"Serpin-2\")\n	(referenceEntity [GK_2671797])\n	(species [GK_2671792])\n	(startCoordinate 1))\n\n([GK_2671792] of  Species\n\n	(_displayName \"Cowpox virus\")\n	(crossReference [GK_3159247])\n	(DB_ID 2671792)\n	(name\n		\"Cowpox virus\"\n		\"CPXV\")\n	(superTaxon [GK_3162966]))\n\n([GK_2671797] of  ReferenceGeneProduct\n\n	(_displayName \"[Cvi] UniProt:P07385 CRMA recommendedName: Serine proteinase inhibitor 2 shortName:Serp-2 shortName:Serpin-2  alternativeName: Cytokine response modifier protein A  alternativeName: Hemorrhage-inducing 38 kDa protein  alternativeName: ICE inhibitor \")\n	(checksum \"25AE4E0CB123E172\")\n	(comment \"FUNCTION Specific and potent inhibitor of the interleukin-1-beta converting enzyme (ICE) thereby suppressing an interleukin-1 beta response to infection. The inhibition of ICE by crmA is an example of a \\\"cross-class\\\" interaction, in which a serpin inhibits a non-serine proteinase. Also inhibits granzyme B.SIMILARITY Belongs to the serpin family. Poxviruses subfamily.\")\n	(DB_ID 2671797)\n	(description \"recommendedName: Serine proteinase inhibitor 2 shortName:Serp-2 shortName:Serpin-2  alternativeName: Cytokine response modifier protein A  alternativeName: Hemorrhage-inducing 38 kDa protein  alternativeName: ICE inhibitor \")\n	(geneName\n		\"CRMA\"\n		\"SPI-2\"\n		\"CPXV207\")\n	(identifier \"P07385\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Host cytoplasm\"\n		\"Host-virus interaction\"\n		\"Inhibition of host caspases by virus\"\n		\"Modulation of host cell apoptosis by virus\"\n		\"Protease inhibitor\"\n		\"Thiol protease inhibitor\")\n	(name \"CRMA\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier \"SPI2_CWPXB\")\n	(sequenceLength 341)\n	(species [GK_2671792]))\n\n([GK_2671799] of  ReferenceGeneProduct\n\n	(_displayName \"[Vvi] UniProt:P15059 SPI-2 recommendedName: Serine proteinase inhibitor 2 shortName:Serp-2 shortName:Serpin-2 \")\n	(checksum \"8A067A10B803F5EF\")\n	(comment \"FUNCTION Inhibits the proteolytic activity of interleukin 1-beta converting enzyme (ICE) and ICE-like enzymes. Can also block apoptosis through host tumor necrosis factor (TNF) receptor.INDUCTION Expressed in the early phase of the viral replicative cycle.SIMILARITY Belongs to the serpin family. Poxviruses subfamily.\")\n	(DB_ID 2671799)\n	(description \"recommendedName: Serine proteinase inhibitor 2 shortName:Serp-2 shortName:Serpin-2 \")\n	(geneName\n		\"SPI-2\"\n		\"VACWR195\"\n		\"B13R\")\n	(identifier \"P15059\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"Complete proteome\"\n		\"Early protein\"\n		\"Host cytoplasm\"\n		\"Host-virus interaction\"\n		\"Inhibition of host caspases by virus\"\n		\"Modulation of host cell apoptosis by virus\"\n		\"Protease inhibitor\"\n		\"Reference proteome\"\n		\"Serine protease inhibitor\")\n	(name \"SPI-2\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"SPI2_VACCW\"\n		\"Q76ZL0\")\n	(sequenceLength 345)\n	(species [GK_2671785]))\n\n([GK_2671811] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS receptor trimer:FADD:procaspase-8-dimer  [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 2671811)\n	(hasComponent\n		[GK_43124]\n		[GK_2671817])\n	(literatureReference [GK_2562554])\n	(name \"FASL:FAS receptor trimer:FADD:procaspase-8-dimer \")\n	(species [GK_48887])\n	(stableIdentifier [GK_3203544]))\n\n([GK_2671812] of  Complex\n\n	(_displayName \"[Hsa] TRAF2:TRADD:RIP1:FADD:procaspase-8-dimer [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2671812)\n	(goCellularComponent [GK_161217])\n	(hasComponent\n		[GK_140977]\n		[GK_2671817])\n	(isChimeric FALSE)\n	(name \"TRAF2:TRADD:RIP1:FADD:procaspase-8-dimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3203706]))\n\n([GK_2671813] of  DefinedSet\n\n	(_displayName \"[Hsa] DISC:procaspase-8-dimer [plasma membrane]\")\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 2671813)\n	(hasMember\n		[GK_2671811]\n		[GK_2671812]\n		[GK_2671816])\n	(name\n		\"DISC:procaspase-8-dimer\"\n		\"death-inducing signaling complex:procaspase-8-dimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3202731]))\n\n([GK_2671814] of  DefinedSet\n\n	(_displayName \"[Hsa] DISC [plasma membrane]\")\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 2671814)\n	(hasMember\n		[GK_43124]\n		[GK_140977]\n		[GK_141137])\n	(name\n		\"DISC\"\n		\"death-inducing signaling complex\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3202937]))\n\n([GK_2671816] of  Complex\n\n	(_displayName \"[Hsa] TRAIL:TRAIL receptor-2:FADD:procaspase-8-dimer [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 2671816)\n	(hasComponent\n		[GK_141137]\n		[GK_2671817])\n	(isChimeric FALSE)\n	(name \"TRAIL:TRAIL receptor-2:FADD:procaspase-8-dimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3201394]))\n\n([GK_2671817] of  Complex\n\n	(_displayName \"[Hsa] procaspase-8 dimer [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2671817)\n	(hasComponent\n		[GK_57031]\n		[GK_57031])\n	(isChimeric FALSE)\n	(name \"procaspase-8 dimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3200756]))\n\n([GK_2671823] of  Person\n\n	(_displayName \"Zerbe, O\")\n	(DB_ID 2671823)\n	(firstname \"O\")\n	(initial \"O\")\n	(surname \"Zerbe\"))\n\n([GK_2671824] of  Person\n\n	(_displayName \"Keller, N\")\n	(DB_ID 2671824)\n	(firstname \"N\")\n	(initial \"N\")\n	(surname \"Keller\"))\n\n([GK_2671825] of  LiteratureReference\n\n	(_displayName \"Studies of the molecular mechanism of caspase-8 activation by solution NMR\")\n	(author\n		[GK_2671824]\n		[GK_2562587]\n		[GK_2671823])\n	(DB_ID 2671825)\n	(journal \"Cell Death Differ.\")\n	(pages \"710-8\")\n	(pubMedIdentifier 19851329)\n	(title \"Studies of the molecular mechanism of caspase-8 activation by solution NMR\")\n	(volume 17)\n	(year 2010))\n\n([GK_2671826] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2012-11-26\")\n	(author [GK_429259])\n	(dateTime \"2012-11-26 04:21:07\")\n	(DB_ID 2671826))\n\n([GK_2672161] of  Person\n\n	(_displayName \"Jassoy, C\")\n	(DB_ID 2672161)\n	(firstname \"C\")\n	(initial \"C\")\n	(surname \"Jassoy\"))\n\n([GK_2672164] of  Person\n\n	(_displayName \"Jeffrey, Philip D\")\n	(DB_ID 2672164)\n	(firstname \"Philip D\")\n	(initial \"PD\")\n	(surname \"Jeffrey\"))\n\n([GK_2672165] of  LiteratureReference\n\n	(_displayName \"Target protease specificity of the viral serpin CrmA. Analysis of five caspases\")\n	(author\n		[GK_2672211]\n		[GK_2672179]\n		[GK_2672202]\n		[GK_2672176]\n		[GK_2672188]\n		[GK_2534135])\n	(DB_ID 2672165)\n	(journal \"J. Biol. Chem.\")\n	(pages \"7797-800\")\n	(pubMedIdentifier 9065443)\n	(title \"Target protease specificity of the viral serpin CrmA. Analysis of five caspases\")\n	(volume 272)\n	(year 1997))\n\n([GK_2672168] of  Person\n\n	(_displayName \"Miura, M\")\n	(DB_ID 2672168)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Miura\"))\n\n([GK_2672170] of  Person\n\n	(_displayName \"Burns, K\")\n	(DB_ID 2672170)\n	(firstname \"K\")\n	(initial \"K\")\n	(surname \"Burns\"))\n\n([GK_2672171] of  Person\n\n	(_displayName \"Smith, G L\")\n	(DB_ID 2672171)\n	(firstname \"G L\")\n	(initial \"GL\")\n	(surname \"Smith\"))\n\n([GK_2672173] of  LiteratureReference\n\n	(_displayName \"Tumor necrosis factor-induced apoptosis is mediated by a CrmA-sensitive cell death pathway\")\n	(author\n		[GK_2672168]\n		[GK_2672206]\n		[GK_2672200])\n	(DB_ID 2672173)\n	(journal \"Proc. Natl. Acad. Sci. U.S.A.\")\n	(pages \"8318-22\")\n	(pubMedIdentifier 7667287)\n	(title \"Tumor necrosis factor-induced apoptosis is mediated by a CrmA-sensitive cell death pathway\")\n	(volume 92)\n	(year 1995))\n\n([GK_2672174] of  Person\n\n	(_displayName \"Thome, M\")\n	(DB_ID 2672174)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Thome\"))\n\n([GK_2672175] of  Person\n\n	(_displayName \"Tschopp, J\")\n	(DB_ID 2672175)\n	(firstname \"J\")\n	(initial \"J\")\n	(surname \"Tschopp\"))\n\n([GK_2672176] of  Person\n\n	(_displayName \"Muzio, M\")\n	(DB_ID 2672176)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Muzio\"))\n\n([GK_2672179] of  Person\n\n	(_displayName \"Snipas, S\")\n	(DB_ID 2672179)\n	(firstname \"S\")\n	(initial \"S\")\n	(surname \"Snipas\"))\n\n([GK_2672185] of  LiteratureReference\n\n	(_displayName \"Fas- and tumor necrosis factor-induced apoptosis is inhibited by the poxvirus crmA gene product\")\n	(author\n		[GK_2672222]\n		[GK_2672188])\n	(DB_ID 2672185)\n	(journal \"J. Biol. Chem.\")\n	(pages \"3255-60\")\n	(pubMedIdentifier 7531702)\n	(title \"Fas- and tumor necrosis factor-induced apoptosis is inhibited by the poxvirus crmA gene product\")\n	(volume 270)\n	(year 1995))\n\n([GK_2672186] of  Person\n\n	(_displayName \"Ehret, R\")\n	(DB_ID 2672186)\n	(firstname \"R\")\n	(initial \"R\")\n	(surname \"Ehret\"))\n\n([GK_2672188] of  Person\n\n	(_displayName \"Dixit, V M\")\n	(DB_ID 2672188)\n	(firstname \"V M\")\n	(initial \"VM\")\n	(surname \"Dixit\"))\n\n([GK_2672191] of  Summation\n\n	(_displayName \"SPI-2/CrmA (cytokine response modifier A)  is a poxvirus gen...\")\n	(DB_ID 2672191)\n	(text \"SPI-2/CrmA (cytokine response modifier A)  is a poxvirus gene product with homology to members of the serpin (serine protease inhibitor) superfamily. Cowpox virus-derived and vaccinia virus-derived CrmA cDNAs transfected into cells can inhibit apoptosis induced by Fas-ligation and activation of the tumor necrosis factor receptor [Tewari M and Dixit VM 1995; Miura M et al 1995; Kettle S et al 1997]. Cowpox virus-derived CrmA was shown to selectively inhibit caspases in Fas-mediated apoptosis, showing the highest affinity for interleukin-1 beta-converting enzyme (ICE) and the second highest for the caspase-8 with Ki = 0.95 nM [Zhou Q et al 1997].\"))\n\n([GK_2672196] of  Reaction\n\n	(_displayName \"[Hsa] Viral serpin blocks caspase-8 activity\")\n	(_doRelease FALSE)\n	(authored [GK_3371406])\n	(compartment [GK_70101])\n	(DB_ID 2672196)\n	(input\n		[GK_2562550]\n		[GK_2672224])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_2672185]\n		[GK_2672213]\n		[GK_2672173]\n		[GK_2672165])\n	(name \"Viral serpin blocks caspase-8 activity\")\n	(output [GK_2672221])\n	(precedingEvent [GK_139952])\n	(species [GK_48887])\n	(summation [GK_2672191]))\n\n([GK_2672197] of  Person\n\n	(_displayName \"Peter, M E\")\n	(DB_ID 2672197)\n	(firstname \"M E\")\n	(initial \"ME\")\n	(surname \"Peter\"))\n\n([GK_2672198] of  Person\n\n	(_displayName \"Alcam√≠, A\")\n	(DB_ID 2672198)\n	(firstname \"A\")\n	(initial \"A\")\n	(surname \"Alcam√≠\"))\n\n([GK_2672199] of  Person\n\n	(_displayName \"Khanna, A\")\n	(DB_ID 2672199)\n	(firstname \"A\")\n	(initial \"A\")\n	(surname \"Khanna\"))\n\n([GK_2672200] of  Person\n\n	(_displayName \"Yuan, J\")\n	(DB_ID 2672200)\n	(firstname \"J\")\n	(initial \"J\")\n	(surname \"Yuan\"))\n\n([GK_2672201] of  Person\n\n	(_displayName \"Yu, Jong W\")\n	(DB_ID 2672201)\n	(firstname \"Jong W\")\n	(initial \"JW\")\n	(surname \"Yu\"))\n\n([GK_2672202] of  Person\n\n	(_displayName \"Orth, K\")\n	(DB_ID 2672202)\n	(firstname \"K\")\n	(initial \"K\")\n	(surname \"Orth\"))\n\n([GK_2672206] of  Person\n\n	(_displayName \"Friedlander, R M\")\n	(DB_ID 2672206)\n	(firstname \"R M\")\n	(initial \"RM\")\n	(surname \"Friedlander\"))\n\n([GK_2672208] of  Person\n\n	(_displayName \"Kettle, S\")\n	(DB_ID 2672208)\n	(firstname \"S\")\n	(initial \"S\")\n	(surname \"Kettle\"))\n\n([GK_2672211] of  Person\n\n	(_displayName \"Zhou, Q\")\n	(DB_ID 2672211)\n	(firstname \"Q\")\n	(initial \"Q\")\n	(surname \"Zhou\"))\n\n([GK_2672213] of  LiteratureReference\n\n	(_displayName \"Vaccinia virus serpin B13R (SPI-2) inhibits interleukin-1beta-converting enzyme and protects virus-infected cells from TNF- and Fas-mediated apoptosis, but does not prevent IL-1beta-induced fever\")\n	(author\n		[GK_2672208]\n		[GK_2672198]\n		[GK_2672199]\n		[GK_2672186]\n		[GK_2672161]\n		[GK_2672171])\n	(DB_ID 2672213)\n	(journal \"J. Gen. Virol.\")\n	(pages \"677-85\")\n	(pubMedIdentifier 9049422)\n	(title \"Vaccinia virus serpin B13R (SPI-2) inhibits interleukin-1beta-converting enzyme and protects virus-infected cells from TNF- and Fas-mediated apoptosis, but does not prevent IL-1beta-induced fever\")\n	(volume 78)\n	(year 1997))\n\n([GK_2672214] of  Person\n\n	(_displayName \"Krammer, P H\")\n	(DB_ID 2672214)\n	(firstname \"P H\")\n	(initial \"PH\")\n	(surname \"Krammer\"))\n\n([GK_2672219] of  Person\n\n	(_displayName \"Scaffidi, C\")\n	(DB_ID 2672219)\n	(firstname \"C\")\n	(initial \"C\")\n	(surname \"Scaffidi\"))\n\n([GK_2672221] of  Complex\n\n	(_displayName \"[Hsa] active caspase-8:viral CRMA/SPI-2 [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2672221)\n	(hasComponent\n		[GK_2562550]\n		[GK_2672224])\n	(isChimeric FALSE)\n	(name \"active caspase-8:viral CRMA/SPI-2\")\n	(species\n		[GK_48887]\n		[GK_2671792]\n		[GK_2671785]))\n\n([GK_2672222] of  Person\n\n	(_displayName \"Tewari, M\")\n	(DB_ID 2672222)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Tewari\"))\n\n([GK_2672224] of  DefinedSet\n\n	(_displayName \"[Cvi] viral CRMA/SPI-2 [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 2672224)\n	(hasMember\n		[GK_2671784]\n		[GK_2671786])\n	(name\n		\"viral CRMA/SPI-2\"\n		\"viral serpins\")\n	(species\n		[GK_2671792]\n		[GK_2671785]))\n\n([GK_2685159] of  Person\n\n	(_displayName \"Thome, Margot\")\n	(DB_ID 2685159)\n	(firstname \"Margot\")\n	(initial \"M\")\n	(surname \"Thome\"))\n\n([GK_2741] of  GO_MolecularFunction\n\n	(_displayName \"cysteine-type peptidase activity\")\n	(accession \"0008234\")\n	(DB_ID 2741)\n	(definition \"Catalysis of the hydrolysis of peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile.\")\n	(instanceOf [GK_376448])\n	(name \"cysteine-type peptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_29427] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:C00038\")\n	(DB_ID 29427)\n	(identifier \"C00038\")\n	(referenceDatabase [GK_6]))\n\n([GK_29497] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:C00076\")\n	(DB_ID 29497)\n	(identifier \"C00076\")\n	(referenceDatabase [GK_6]))\n\n([GK_2984174] of  LiteratureReference\n\n	(_displayName \"TNF-alpha induces two distinct caspase-8 activation pathways\")\n	(author\n		[GK_2984177]\n		[GK_2984180]\n		[GK_2584552])\n	(DB_ID 2984174)\n	(journal \"Cell\")\n	(pages \"693-703\")\n	(pubMedIdentifier 18485876)\n	(title \"TNF-alpha induces two distinct caspase-8 activation pathways\")\n	(volume 133)\n	(year 2008))\n\n([GK_2984175] of  Person\n\n	(_displayName \"Hughes, Michelle A\")\n	(DB_ID 2984175)\n	(firstname \"Michelle A\")\n	(initial \"MA\")\n	(surname \"Hughes\"))\n\n([GK_2984176] of  Person\n\n	(_displayName \"Butterworth, Michael\")\n	(DB_ID 2984176)\n	(firstname \"Michael\")\n	(initial \"M\")\n	(surname \"Butterworth\"))\n\n([GK_2984177] of  Person\n\n	(_displayName \"Wang, Lai\")\n	(DB_ID 2984177)\n	(firstname \"Lai\")\n	(initial \"L\")\n	(surname \"Wang\"))\n\n([GK_2984178] of  Person\n\n	(_displayName \"Cohen, Gerald M\")\n	(DB_ID 2984178)\n	(firstname \"Gerald M\")\n	(initial \"GM\")\n	(surname \"Cohen\"))\n\n([GK_2984180] of  Person\n\n	(_displayName \"Du, Fenghe\")\n	(DB_ID 2984180)\n	(firstname \"Fenghe\")\n	(initial \"F\")\n	(surname \"Du\"))\n\n([GK_2984181] of  Person\n\n	(_displayName \"Peter, Marcus E\")\n	(DB_ID 2984181)\n	(firstname \"Marcus E\")\n	(initial \"ME\")\n	(surname \"Peter\"))\n\n([GK_2984182] of  Person\n\n	(_displayName \"Harper, Nicholas\")\n	(DB_ID 2984182)\n	(firstname \"Nicholas\")\n	(initial \"N\")\n	(surname \"Harper\"))\n\n([GK_2984183] of  Person\n\n	(_displayName \"Chang, David W\")\n	(DB_ID 2984183)\n	(firstname \"David W\")\n	(initial \"DW\")\n	(surname \"Chang\"))\n\n([GK_2984185] of  LiteratureReference\n\n	(_displayName \"Reconstitution of the death-inducing signaling complex reveals a substrate switch that determines CD95-mediated death or survival\")\n	(author\n		[GK_2984175]\n		[GK_2984182]\n		[GK_2984176]\n		[GK_2562560]\n		[GK_2984178]\n		[GK_2562548])\n	(DB_ID 2984185)\n	(journal \"Mol. Cell\")\n	(pages \"265-79\")\n	(pubMedIdentifier 19683492)\n	(title \"Reconstitution of the death-inducing signaling complex reveals a substrate switch that determines CD95-mediated death or survival\")\n	(volume 35)\n	(year 2009))\n\n([GK_30151] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:C00434\")\n	(DB_ID 30151)\n	(identifier \"C00434\")\n	(referenceDatabase [GK_6]))\n\n([GK_3132308] of  GO_BiologicalProcess\n\n	(_displayName \"single-organism cellular process\")\n	(accession \"0044763\")\n	(DB_ID 3132308)\n	(definition \"Any process that is carried out at the cellular level, occurring within a single organism.\")\n	(instanceOf\n		[GK_143]\n		[GK_2587987])\n	(name \"single-organism cellular process\")\n	(referenceDatabase [GK_1]))\n\n([GK_3159243] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:10245\")\n	(DB_ID 3159243)\n	(identifier \"10245\")\n	(referenceDatabase [GK_72810]))\n\n([GK_3159247] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:10243\")\n	(DB_ID 3159247)\n	(identifier \"10243\")\n	(referenceDatabase [GK_72810]))\n\n([GK_3162959] of  Taxon\n\n	(_displayName \"Poxviridae\")\n	(crossReference [GK_3162963])\n	(DB_ID 3162959)\n	(name \"Poxviridae\")\n	(superTaxon [GK_73135]))\n\n([GK_3162960] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:10241\")\n	(DB_ID 3162960)\n	(identifier \"10241\")\n	(referenceDatabase [GK_72810]))\n\n([GK_3162963] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:10240\")\n	(DB_ID 3162963)\n	(identifier \"10240\")\n	(referenceDatabase [GK_72810]))\n\n([GK_3162964] of  Taxon\n\n	(_displayName \"Chordopoxvirinae\")\n	(crossReference [GK_3162960])\n	(DB_ID 3162964)\n	(name \"Chordopoxvirinae\")\n	(superTaxon [GK_3162959]))\n\n([GK_3162966] of  Taxon\n\n	(_displayName \"Orthopoxvirus\")\n	(crossReference [GK_3162967])\n	(DB_ID 3162966)\n	(name \"Orthopoxvirus\")\n	(superTaxon [GK_3162964]))\n\n([GK_3162967] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:10242\")\n	(DB_ID 3162967)\n	(identifier \"10242\")\n	(referenceDatabase [GK_72810]))\n\n([GK_3200756] of  StableIdentifier\n\n	(_displayName \"REACT_160329.1\")\n	(DB_ID 3200756)\n	(identifier \"REACT_160329\")\n	(identifierVersion \"1\"))\n\n([GK_3201394] of  StableIdentifier\n\n	(_displayName \"REACT_160570.1\")\n	(DB_ID 3201394)\n	(identifier \"REACT_160570\")\n	(identifierVersion \"1\"))\n\n([GK_3202731] of  StableIdentifier\n\n	(_displayName \"REACT_161085.1\")\n	(DB_ID 3202731)\n	(identifier \"REACT_161085\")\n	(identifierVersion \"1\"))\n\n([GK_3202937] of  StableIdentifier\n\n	(_displayName \"REACT_161157.1\")\n	(DB_ID 3202937)\n	(identifier \"REACT_161157\")\n	(identifierVersion \"1\"))\n\n([GK_3203544] of  StableIdentifier\n\n	(_displayName \"REACT_161393.1\")\n	(DB_ID 3203544)\n	(identifier \"REACT_161393\")\n	(identifierVersion \"1\"))\n\n([GK_3203706] of  StableIdentifier\n\n	(_displayName \"REACT_161446.2\")\n	(DB_ID 3203706)\n	(identifier \"REACT_161446\")\n	(identifierVersion \"2\"))\n\n([GK_3203992] of  StableIdentifier\n\n	(_displayName \"REACT_151128.1\")\n	(DB_ID 3203992)\n	(identifier \"REACT_151128\")\n	(identifierVersion \"1\"))\n\n([GK_3209986] of  Figure\n\n	(_displayName \"/figures/Pathway_Illustrations/Extrinsic_Apoptosis_72_oicr.png\")\n	(DB_ID 3209986)\n	(url \"/figures/Pathway_Illustrations/Extrinsic_Apoptosis_72_oicr.png\"))\n\n([GK_3214933] of  Person\n\n	(_displayName \"Hietakangas, Ville\")\n	(DB_ID 3214933)\n	(firstname \"Ville\")\n	(initial \"V\")\n	(surname \"Hietakangas\"))\n\n([GK_3214959] of  Person\n\n	(_displayName \"Sistonen, Lea\")\n	(DB_ID 3214959)\n	(firstname \"Lea\")\n	(initial \"L\")\n	(surname \"Sistonen\"))\n\n([GK_3320] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of metabolic process\")\n	(accession \"0019222\")\n	(DB_ID 3320)\n	(definition \"Any process that modulates the frequency, rate or extent of the chemical reactions and pathways within a cell or an organism.\")\n	(instanceOf [GK_139810])\n	(name \"regulation of metabolic process\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_504]))\n\n([GK_3371348] of  LiteratureReference\n\n	(_displayName \"Unraveling the binding mechanism of trivalent tumor necrosis factor ligands and their receptors\")\n	(author\n		[GK_3371384]\n		[GK_3371354]\n		[GK_3371403]\n		[GK_3371387])\n	(DB_ID 3371348)\n	(journal \"Mol. Cell Proteomics\")\n	(pages \"M110.002808\")\n	(pubMedIdentifier 20852190)\n	(title \"Unraveling the binding mechanism of trivalent tumor necrosis factor ligands and their receptors\")\n	(volume 10)\n	(year 2011))\n\n([GK_3371349] of  Person\n\n	(_displayName \"Kamada, Haruhiko\")\n	(DB_ID 3371349)\n	(firstname \"Haruhiko\")\n	(initial \"H\")\n	(surname \"Kamada\"))\n\n([GK_3371350] of  Person\n\n	(_displayName \"Tsutsumi, Yasuo\")\n	(DB_ID 3371350)\n	(firstname \"Yasuo\")\n	(initial \"Y\")\n	(surname \"Tsutsumi\"))\n\n([GK_3371351] of  Complex\n\n	(_displayName \"[Hsa] TNF-alpha trimer [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3371351)\n	(hasComponent\n		[GK_66220]\n		[GK_66220]\n		[GK_66220])\n	(isChimeric FALSE)\n	(literatureReference [GK_3371352])\n	(name \"TNF-alpha trimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655182]))\n\n([GK_3371352] of  LiteratureReference\n\n	(_displayName \"Human pro-tumor necrosis factor is a homotrimer\")\n	(author\n		[GK_3371400]\n		[GK_3371362]\n		[GK_3371388])\n	(DB_ID 3371352)\n	(journal \"Biochemistry\")\n	(pages \"8216-25\")\n	(pubMedIdentifier 8679576)\n	(title \"Human pro-tumor necrosis factor is a homotrimer\")\n	(volume 35)\n	(year 1996))\n\n([GK_3371353] of  Reaction\n\n	(_displayName \"[Hsa] Soluble TNF-alpha binds TNF-R1\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(compartment\n		[GK_876]\n		[GK_984])\n	(DB_ID 3371353)\n	(edited [GK_3632865])\n	(input\n		[GK_66344]\n		[GK_66344]\n		[GK_66344]\n		[GK_3371370])\n	(literatureReference [GK_140945])\n	(name \"Soluble TNF-alpha binds TNF-R1\")\n	(output [GK_3371397])\n	(precedingEvent [GK_3371385])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654232])\n	(summation [GK_3465562]))\n\n([GK_3371354] of  Person\n\n	(_displayName \"van Assen, Aart H G\")\n	(DB_ID 3371354)\n	(firstname \"Aart H G\")\n	(initial \"AH\")\n	(surname \"van Assen\"))\n\n([GK_3371355] of  Person\n\n	(_displayName \"Kayamuro, Hiroyuki\")\n	(DB_ID 3371355)\n	(firstname \"Hiroyuki\")\n	(initial \"H\")\n	(surname \"Kayamuro\"))\n\n([GK_3371356] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CFLAR(1-480) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 3371356)\n	(endCoordinate 480)\n	(name\n		\"CFLAR(1-480)\"\n		\"FLIP(L)\"\n		\"CLARP1\"\n		\"MRIT alpha-1\"\n		\"CASH alpha\"\n		\"I-FLICE 1\"\n		\"FLAME-1 gamma\"\n		\"CASP8 and FADD-like apoptosis regulator\"\n		\"FLIP long\")\n	(referenceEntity [GK_401351])\n	(species [GK_48887])\n	(stableIdentifier [GK_3655784])\n	(startCoordinate 1))\n\n([GK_3371357] of  Person\n\n	(_displayName \"Mayumi, Tadanori\")\n	(DB_ID 3371357)\n	(firstname \"Tadanori\")\n	(initial \"T\")\n	(surname \"Mayumi\"))\n\n([GK_3371358] of  Person\n\n	(_displayName \"Pinkoski, Michael J\")\n	(DB_ID 3371358)\n	(firstname \"Michael J\")\n	(initial \"MJ\")\n	(surname \"Pinkoski\"))\n\n([GK_3371359] of  Reaction\n\n	(_displayName \"[Hsa] FLIP(L) and procaspase-8 form heterodimer in TRAIL signaling\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 3371359)\n	(edited [GK_3508822])\n	(input\n		[GK_141133]\n		[GK_3371356])\n	(literatureReference\n		[GK_3465422]\n		[GK_3465385]\n		[GK_3465382]\n		[GK_3465460])\n	(name \"FLIP(L) and procaspase-8 form heterodimer in TRAIL signaling\")\n	(output [GK_3371390])\n	(precedingEvent [GK_75146])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654456])\n	(summation [GK_3465446]))\n\n([GK_3371360] of  Reaction\n\n	(_displayName \"[Hsa] FLIP(L) and procaspase-8 form heterodimer in TNF signaling\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(compartment [GK_70101])\n	(DB_ID 3371360)\n	(edited [GK_3508822])\n	(input\n		[GK_3371356]\n		[GK_140976])\n	(literatureReference\n		[GK_140979]\n		[GK_2584573]\n		[GK_3465454]\n		[GK_3465455]\n		[GK_3465494])\n	(name \"FLIP(L) and procaspase-8 form heterodimer in TNF signaling\")\n	(output [GK_3465532])\n	(precedingEvent [GK_75240])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654502])\n	(summation [GK_3465402]))\n\n([GK_3371361] of  Person\n\n	(_displayName \"Sun, Xiao-Ming\")\n	(DB_ID 3371361)\n	(firstname \"Xiao-Ming\")\n	(initial \"XM\")\n	(surname \"Sun\"))\n\n([GK_3371362] of  Person\n\n	(_displayName \"Hung MC, MC\")\n	(DB_ID 3371362)\n	(initial \"MC\")\n	(surname \"Hung MC\"))\n\n([GK_3371363] of  Person\n\n	(_displayName \"Hayakawa, Takao\")\n	(DB_ID 3371363)\n	(firstname \"Takao\")\n	(initial \"T\")\n	(surname \"Hayakawa\"))\n\n([GK_3371364] of  Person\n\n	(_displayName \"Ohkawa, Akiko\")\n	(DB_ID 3371364)\n	(firstname \"Akiko\")\n	(initial \"A\")\n	(surname \"Ohkawa\"))\n\n([GK_3371365] of  Person\n\n	(_displayName \"Minowa, Kyoko\")\n	(DB_ID 3371365)\n	(firstname \"Kyoko\")\n	(initial \"K\")\n	(surname \"Minowa\"))\n\n([GK_3371366] of  Person\n\n	(_displayName \"Shibata, Hiroko\")\n	(DB_ID 3371366)\n	(firstname \"Hiroko\")\n	(initial \"H\")\n	(surname \"Shibata\"))\n\n([GK_3371367] of  Person\n\n	(_displayName \"Nomura, Tetsuya\")\n	(DB_ID 3371367)\n	(firstname \"Tetsuya\")\n	(initial \"T\")\n	(surname \"Nomura\"))\n\n([GK_3371368] of  Person\n\n	(_displayName \"Tsunoda, Shin-ichi\")\n	(DB_ID 3371368)\n	(firstname \"Shin-ichi\")\n	(initial \"S\")\n	(surname \"Tsunoda\"))\n\n([GK_3371369] of  Person\n\n	(_displayName \"Ohta, Tsunetaka\")\n	(DB_ID 3371369)\n	(firstname \"Tsunetaka\")\n	(initial \"T\")\n	(surname \"Ohta\"))\n\n([GK_3371370] of  Complex\n\n	(_displayName \"[Hsa] TNF-alpha trimer [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 3371370)\n	(hasComponent\n		[GK_976821]\n		[GK_976821]\n		[GK_976821])\n	(isChimeric FALSE)\n	(literatureReference [GK_3371402])\n	(name\n		\"TNF-alpha trimer\"\n		\"TNF-alpha trimer, soluble form\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655921]))\n\n([GK_3371371] of  Person\n\n	(_displayName \"Sugita, Toshiki\")\n	(DB_ID 3371371)\n	(firstname \"Toshiki\")\n	(initial \"T\")\n	(surname \"Sugita\"))\n\n([GK_3371372] of  Person\n\n	(_displayName \"Abe, Yasuhiro\")\n	(DB_ID 3371372)\n	(firstname \"Yasuhiro\")\n	(initial \"Y\")\n	(surname \"Abe\"))\n\n([GK_3371373] of  LiteratureReference\n\n	(_displayName \"Receptor-mediated endocytosis is not required for tumor necrosis factor-related apoptosis-inducing ligand (TRAIL)-induced apoptosis\")\n	(author\n		[GK_3371375]\n		[GK_3371394]\n		[GK_3371361]\n		[GK_3371358]\n		[GK_2984178])\n	(DB_ID 3371373)\n	(journal \"J. Biol. Chem.\")\n	(pages \"12831-41\")\n	(pubMedIdentifier 17327223)\n	(title \"Receptor-mediated endocytosis is not required for tumor necrosis factor-related apoptosis-inducing ligand (TRAIL)-induced apoptosis\")\n	(volume 282)\n	(year 2007))\n\n([GK_3371374] of  Person\n\n	(_displayName \"Yoshikawa, Tomoaki\")\n	(DB_ID 3371374)\n	(firstname \"Tomoaki\")\n	(initial \"T\")\n	(surname \"Yoshikawa\"))\n\n([GK_3371375] of  Person\n\n	(_displayName \"Kohlhaas, Susan L\")\n	(DB_ID 3371375)\n	(firstname \"Susan L\")\n	(initial \"SL\")\n	(surname \"Kohlhaas\"))\n\n([GK_3371376] of  Person\n\n	(_displayName \"Aggarwal, Bharat B\")\n	(DB_ID 3371376)\n	(firstname \"Bharat B\")\n	(initial \"BB\")\n	(surname \"Aggarwal\"))\n\n([GK_3371377] of  Person\n\n	(_displayName \"Yoshioka, Yasuo\")\n	(DB_ID 3371377)\n	(firstname \"Yasuo\")\n	(initial \"Y\")\n	(surname \"Yoshioka\"))\n\n([GK_3371378] of  Pathway\n\n	(_displayName \"[Hsa] Regulation by c-FLIP\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(DB_ID 3371378)\n	(edited [GK_3508822])\n	(goBiologicalProcess [GK_2133698])\n	(hasEvent\n		[GK_3371360]\n		[GK_3371359]\n		[GK_3465459]\n		[GK_3465448]\n		[GK_3465429])\n	(literatureReference\n		[GK_1250231]\n		[GK_1267786]\n		[GK_3465386])\n	(name \"Regulation by c-FLIP\")\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654555])\n	(summation [GK_3465346]))\n\n([GK_3371379] of  Person\n\n	(_displayName \"Taniai, Madoka\")\n	(DB_ID 3371379)\n	(firstname \"Madoka\")\n	(initial \"M\")\n	(surname \"Taniai\"))\n\n([GK_3371380] of  LiteratureReference\n\n	(_displayName \"Creation and X-ray structure analysis of the tumor necrosis factor receptor-1-selective mutant of a tumor necrosis factor-alpha antagonist\")\n	(author\n		[GK_3371366]\n		[GK_3371377]\n		[GK_3371364]\n		[GK_3371365]\n		[GK_3371395]\n		[GK_3371372]\n		[GK_3371379]\n		[GK_3371367]\n		[GK_3371355]\n		[GK_3371398]\n		[GK_3371371]\n		[GK_3371382]\n		[GK_3371389]\n		[GK_3371374]\n		[GK_3371399]\n		[GK_3371391]\n		[GK_3371396]\n		[GK_3371369]\n		[GK_3371363]\n		[GK_3371357]\n		[GK_141902]\n		[GK_3371376]\n		[GK_3371393]\n		[GK_2393943]\n		[GK_3371368]\n		[GK_3371349]\n		[GK_3371350])\n	(DB_ID 3371380)\n	(journal \"J. Biol. Chem.\")\n	(pages \"998-1007\")\n	(pubMedIdentifier 18003610)\n	(title \"Creation and X-ray structure analysis of the tumor necrosis factor receptor-1-selective mutant of a tumor necrosis factor-alpha antagonist\")\n	(volume 283)\n	(year 2008))\n\n([GK_3371381] of  Complex\n\n	(_displayName \"[Hsa] DISC:procaspase-8:FLIP(L) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3371381)\n	(hasComponent\n		[GK_3371356]\n		[GK_3465443])\n	(isChimeric FALSE)\n	(name \"DISC:procaspase-8:FLIP(L)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655002]))\n\n([GK_3371382] of  Person\n\n	(_displayName \"Imai, Sunao\")\n	(DB_ID 3371382)\n	(firstname \"Sunao\")\n	(initial \"S\")\n	(surname \"Imai\"))\n\n([GK_3371384] of  Person\n\n	(_displayName \"Reis, Carlos R\")\n	(DB_ID 3371384)\n	(firstname \"Carlos R\")\n	(initial \"CR\")\n	(surname \"Reis\"))\n\n([GK_3371385] of  Reaction\n\n	(_displayName \"[Hsa] TNF-alpha cleavage by TACE\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(catalystActivity [GK_1251983])\n	(compartment\n		[GK_984]\n		[GK_876])\n	(DB_ID 3371385)\n	(edited [GK_3632865])\n	(input [GK_3371351])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_3465559]\n		[GK_3465568])\n	(name \"TNF-alpha cleavage by TACE\")\n	(output [GK_3371370])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654338])\n	(summation [GK_3465572]))\n\n([GK_3371386] of  Person\n\n	(_displayName \"Eck, M J\")\n	(DB_ID 3371386)\n	(firstname \"M J\")\n	(initial \"MJ\")\n	(surname \"Eck\"))\n\n([GK_3371387] of  Person\n\n	(_displayName \"Cool, Robbert H\")\n	(DB_ID 3371387)\n	(firstname \"Robbert H\")\n	(initial \"RH\")\n	(surname \"Cool\"))\n\n([GK_3371388] of  Person\n\n	(_displayName \"Klostergaard, J\")\n	(DB_ID 3371388)\n	(firstname \"J\")\n	(initial \"J\")\n	(surname \"Klostergaard\"))\n\n([GK_3371389] of  Person\n\n	(_displayName \"Nagano, Kazuya\")\n	(DB_ID 3371389)\n	(firstname \"Kazuya\")\n	(initial \"K\")\n	(surname \"Nagano\"))\n\n([GK_3371390] of  Complex\n\n	(_displayName \"[Hsa] TRAIL:TRAIL receptor-2 Trimer:FADD:procaspase-8:FLIP-L [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3371390)\n	(hasComponent\n		[GK_57031]\n		[GK_141137]\n		[GK_3371356])\n	(name \"TRAIL:TRAIL receptor-2 Trimer:FADD:procaspase-8:FLIP-L\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655740]))\n\n([GK_3371391] of  Person\n\n	(_displayName \"Nakagawa, Shinsaku\")\n	(DB_ID 3371391)\n	(firstname \"Shinsaku\")\n	(initial \"S\")\n	(surname \"Nakagawa\"))\n\n([GK_3371392] of  Person\n\n	(_displayName \"Sprang, S R\")\n	(DB_ID 3371392)\n	(firstname \"S R\")\n	(initial \"SR\")\n	(surname \"Sprang\"))\n\n([GK_3371393] of  Person\n\n	(_displayName \"Nakamura, Teruya\")\n	(DB_ID 3371393)\n	(firstname \"Teruya\")\n	(initial \"T\")\n	(surname \"Nakamura\"))\n\n([GK_3371394] of  Person\n\n	(_displayName \"Craxton, Andrew\")\n	(DB_ID 3371394)\n	(firstname \"Andrew\")\n	(initial \"A\")\n	(surname \"Craxton\"))\n\n([GK_3371395] of  Person\n\n	(_displayName \"Mukai, Yohei\")\n	(DB_ID 3371395)\n	(firstname \"Yohei\")\n	(initial \"Y\")\n	(surname \"Mukai\"))\n\n([GK_3371396] of  Person\n\n	(_displayName \"Yamamoto, Akira\")\n	(DB_ID 3371396)\n	(firstname \"Akira\")\n	(initial \"A\")\n	(surname \"Yamamoto\"))\n\n([GK_3371397] of  Complex\n\n	(_displayName \"[Hsa] soluble TNF-alpha:TNF-R1 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3371397)\n	(hasComponent\n		[GK_66344]\n		[GK_66344]\n		[GK_66344]\n		[GK_3371370])\n	(literatureReference [GK_3371380])\n	(name \"soluble TNF-alpha:TNF-R1\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655101]))\n\n([GK_3371398] of  Person\n\n	(_displayName \"Nabeshi, Hiromi\")\n	(DB_ID 3371398)\n	(firstname \"Hiromi\")\n	(initial \"H\")\n	(surname \"Nabeshi\"))\n\n([GK_3371399] of  Person\n\n	(_displayName \"Fujita, Takuya\")\n	(DB_ID 3371399)\n	(firstname \"Takuya\")\n	(initial \"T\")\n	(surname \"Fujita\"))\n\n([GK_3371400] of  Person\n\n	(_displayName \"Tang, P\")\n	(DB_ID 3371400)\n	(firstname \"P\")\n	(initial \"P\")\n	(surname \"Tang\"))\n\n([GK_3371401] of  DefinedSet\n\n	(_displayName \"[Hsa] TNF-alpha trimer:TNF-R1 trimer [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3371401)\n	(hasMember\n		[GK_74277]\n		[GK_3371397])\n	(name \"TNF-alpha trimer:TNF-R1 trimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655775]))\n\n([GK_3371402] of  LiteratureReference\n\n	(_displayName \"The structure of tumor necrosis factor-alpha at 2.6 A resolution. Implications for receptor binding\")\n	(author\n		[GK_3371386]\n		[GK_3371392])\n	(DB_ID 3371402)\n	(journal \"J. Biol. Chem.\")\n	(pages \"17595-605\")\n	(pubMedIdentifier 2551905)\n	(title \"The structure of tumor necrosis factor-alpha at 2.6 A resolution. Implications for receptor binding\")\n	(volume 264)\n	(year 1989))\n\n([GK_3371403] of  Person\n\n	(_displayName \"Quax, Wim J\")\n	(DB_ID 3371403)\n	(firstname \"Wim J\")\n	(initial \"WJ\")\n	(surname \"Quax\"))\n\n([GK_3371404] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-13\")\n	(author [GK_429259])\n	(dateTime \"2013-05-13 13:54:08\")\n	(DB_ID 3371404))\n\n([GK_3371406] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-13\")\n	(author [GK_429259])\n	(dateTime \"2013-05-13 13:56:05\")\n	(DB_ID 3371406))\n\n([GK_3446165] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of extrinsic apoptotic signaling pathway via death domain receptors\")\n	(accession \"1902041\")\n	(DB_ID 3446165)\n	(definition \"Any process that modulates the frequency, rate or extent of extrinsic apoptotic signaling pathway via death domain receptors.\")\n	(instanceOf [GK_2133695])\n	(name \"regulation of extrinsic apoptotic signaling pathway via death domain receptors\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_25449]))\n\n([GK_3446166] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of extrinsic apoptotic signaling pathway via death domain receptors\")\n	(accession \"1902042\")\n	(DB_ID 3446166)\n	(definition \"Any process that stops, prevents or reduces the frequency, rate or extent of extrinsic apoptotic signaling pathway via death domain receptors.\")\n	(instanceOf\n		[GK_3446165]\n		[GK_2133696])\n	(name \"negative regulation of extrinsic apoptotic signaling pathway via death domain receptors\")\n	(negativelyRegulate [GK_25449])\n	(referenceDatabase [GK_1]))\n\n([GK_3465328] of  Summation\n\n	(_displayName \"In the presence of FLIP(L), both caspase-8 and FLIP(L) are r...\")\n	(DB_ID 3465328)\n	(literatureReference\n		[GK_1250231]\n		[GK_3465478]\n		[GK_3465386]\n		[GK_1267786])\n	(text \"In the presence of FLIP(L), both caspase-8 and FLIP(L) are recruited and partially processed at the death-inducing signaling complex (DISC). The partially processed proteins stay bound to the DISC.<p>The long cellular FLIP (FLIP(L) or c-FLIPL) has two death effector domains (DED) and a caspase-like domain that lacks catalytic activity due to absence of a cysteine residue. Processing of FLIP(L) occurs at the DISC and depends on caspase-8 activity (zymogen and mature form). Upon activation FLIP(L) is cleaved to generate N-terminal FLIP(p43) and C-terminal FLIP(p12)(Irmler M et al. 1997; Jong WY et al. 2009). Processed FLIP(L) can enhance the proteolytic activity of procaspase-8 (Chang DW et al. 2002; Jong WY et al. 2009; Pop C et al. 2011). However, the increased FLIP(L) protein levels in cells have been found to limit caspase-8 activity and inhibit apoptotic signaling pathway,\"))\n\n([GK_3465329] of  Person\n\n	(_displayName \"Kovacsovics, M\")\n	(DB_ID 3465329)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Kovacsovics\"))\n\n([GK_3465330] of  LiteratureReference\n\n	(_displayName \"PKC-mediated phosphorylation regulates c-FLIP ubiquitylation and stability\")\n	(author\n		[GK_3465388]\n		[GK_3465434]\n		[GK_3465500]\n		[GK_3465458]\n		[GK_3465344]\n		[GK_3465498]\n		[GK_3465331])\n	(DB_ID 3465330)\n	(journal \"Cell Death Differ.\")\n	(pages \"1215-26\")\n	(pubMedIdentifier 19343040)\n	(title \"PKC-mediated phosphorylation regulates c-FLIP ubiquitylation and stability\")\n	(volume 16)\n	(year 2009))\n\n([GK_3465331] of  Person\n\n	(_displayName \"Eriksson, J E\")\n	(DB_ID 3465331)\n	(firstname \"J E\")\n	(initial \"JE\")\n	(surname \"Eriksson\"))\n\n([GK_3465333] of  Person\n\n	(_displayName \"Kennedy, N\")\n	(DB_ID 3465333)\n	(firstname \"N\")\n	(initial \"N\")\n	(surname \"Kennedy\"))\n\n([GK_3465334] of  LiteratureReference\n\n	(_displayName \"The caspase-8 inhibitor FLIP promotes activation of NF-kappaB and Erk signaling pathways\")\n	(author\n		[GK_3465376]\n		[GK_3465485]\n		[GK_3465398]\n		[GK_2672174]\n		[GK_3465452]\n		[GK_3465449]\n		[GK_2672170]\n		[GK_3465481]\n		[GK_3465333]\n		[GK_3465329]\n		[GK_2672175])\n	(DB_ID 3465334)\n	(journal \"Curr. Biol.\")\n	(pages \"640-8\")\n	(pubMedIdentifier 10837247)\n	(title \"The caspase-8 inhibitor FLIP promotes activation of NF-kappaB and Erk signaling pathways\")\n	(volume 10)\n	(year 2000))\n\n([GK_3465335] of  Person\n\n	(_displayName \"Eils, Roland\")\n	(DB_ID 3465335)\n	(firstname \"Roland\")\n	(initial \"R\")\n	(surname \"Eils\"))\n\n([GK_3465336] of  Person\n\n	(_displayName \"Vittori, Daniela\")\n	(DB_ID 3465336)\n	(firstname \"Daniela\")\n	(initial \"D\")\n	(surname \"Vittori\"))\n\n([GK_3465337] of  Person\n\n	(_displayName \"Langa, Susana\")\n	(DB_ID 3465337)\n	(firstname \"Susana\")\n	(initial \"S\")\n	(surname \"Langa\"))\n\n([GK_3465338] of  Person\n\n	(_displayName \"Haag, Christian\")\n	(DB_ID 3465338)\n	(firstname \"Christian\")\n	(initial \"C\")\n	(surname \"Haag\"))\n\n([GK_3465339] of  Person\n\n	(_displayName \"Poukkula, Minna\")\n	(DB_ID 3465339)\n	(firstname \"Minna\")\n	(initial \"M\")\n	(surname \"Poukkula\"))\n\n([GK_3465340] of  Person\n\n	(_displayName \"Richter, Petra\")\n	(DB_ID 3465340)\n	(firstname \"Petra\")\n	(initial \"P\")\n	(surname \"Richter\"))\n\n([GK_3465342] of  Person\n\n	(_displayName \"Meinander, Annika\")\n	(DB_ID 3465342)\n	(firstname \"Annika\")\n	(initial \"A\")\n	(surname \"Meinander\"))\n\n([GK_3465343] of  Person\n\n	(_displayName \"Schneider, Pascal\")\n	(DB_ID 3465343)\n	(firstname \"Pascal\")\n	(initial \"P\")\n	(surname \"Schneider\"))\n\n([GK_3465344] of  Person\n\n	(_displayName \"Poukkula, M\")\n	(DB_ID 3465344)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Poukkula\"))\n\n([GK_3465345] of  Person\n\n	(_displayName \"Kreuz, Sebastian\")\n	(DB_ID 3465345)\n	(firstname \"Sebastian\")\n	(initial \"S\")\n	(surname \"Kreuz\"))\n\n([GK_3465346] of  Summation\n\n	(_displayName \"c-FLIP proteins (CASP8 and FADD-like apoptosis regulators or...\")\n	(DB_ID 3465346)\n	(literatureReference\n		[GK_1250231]\n		[GK_3465444]\n		[GK_3465488]\n		[GK_3465397]\n		[GK_3465385]\n		[GK_3465361]\n		[GK_3465380]\n		[GK_3465456])\n	(text \"c-FLIP proteins (CASP8 and FADD-like apoptosis regulators or c-FLICE inhibitory proteins) are death effector domain (DED)-containing proteins that are recruited to the death-inducing signaling complex (DISC) to regulate activation of caspases-8. Three out of 13 distinct spliced variants of c-FLIP had been found to be expressed at the protein level, the 26 kDa short form FLIP(S), the 24 kDa form FLIP(R), and the 55 kDa long form FLIP(L) (Irmler M et al. 1997; Shu HB et al. 1997; Srinivasula SM et al. 1997; Scaffidi C et al. 1999; Golks A et al. 2005; Haag C et al. 2011)<p>All c-FLIP proteins carry two DEDs at their N termini, which can bind FADD and procaspase-8. In addition to two DEDs, FLIP(L) contains a large (p20) and a small (p12) caspase-like domain without catalytic activity. FLIP(S) and FLIP(R) consist of two DEDs and a small C terminus. Depending on its level of expression FLIP(L) may function as an anti-apoptotic or pro-apoptotic factor, while FLIP(S) and FLIP(R) protect cells from apoptosis by blocking the processing of caspase-8 at the receptor level (Scaffidi C et al. 1999; Golks A et al. 2005; Toivonen HT et al. 2011; Fricker N et al. 2010).\"))\n\n([GK_3465347] of  Person\n\n	(_displayName \"Siegmund, D\")\n	(DB_ID 3465347)\n	(firstname \"D\")\n	(initial \"D\")\n	(surname \"Siegmund\"))\n\n([GK_3465348] of  Person\n\n	(_displayName \"Krammer, Peter H\")\n	(DB_ID 3465348)\n	(firstname \"Peter H\")\n	(initial \"PH\")\n	(surname \"Krammer\"))\n\n([GK_3465349] of  Person\n\n	(_displayName \"Chung, Jee Y\")\n	(DB_ID 3465349)\n	(firstname \"Jee Y\")\n	(initial \"JY\")\n	(surname \"Chung\"))\n\n([GK_3465350] of  Person\n\n	(_displayName \"Callero, Mariana\")\n	(DB_ID 3465350)\n	(firstname \"Mariana\")\n	(initial \"M\")\n	(surname \"Callero\"))\n\n([GK_3465351] of  Person\n\n	(_displayName \"Johnson, Mark S\")\n	(DB_ID 3465351)\n	(firstname \"Mark S\")\n	(initial \"MS\")\n	(surname \"Johnson\"))\n\n([GK_3465352] of  Complex\n\n	(_displayName \"[Hsa] DISC:procaspase-8:FLIP(S) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3465352)\n	(hasComponent\n		[GK_3465443]\n		[GK_3465357])\n	(isChimeric FALSE)\n	(name \"DISC:procaspase-8:FLIP(S)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3654708]))\n\n([GK_3465353] of  LiteratureReference\n\n	(_displayName \"A novel mechanism of TRAF signaling revealed by structural and functional analyses of the TRADD-TRAF2 interaction\")\n	(author\n		[GK_141860]\n		[GK_3465399]\n		[GK_3465372]\n		[GK_141862]\n		[GK_141861]\n		[GK_3465419]\n		[GK_141863]\n		[GK_141864])\n	(DB_ID 3465353)\n	(journal \"Cell\")\n	(pages \"777-87\")\n	(pubMedIdentifier 10892748)\n	(title \"A novel mechanism of TRAF signaling revealed by structural and functional analyses of the TRADD-TRAF2 interaction\")\n	(volume 101)\n	(year 2000))\n\n([GK_3465354] of  Person\n\n	(_displayName \"Chamorro, Mar√≠a E\")\n	(DB_ID 3465354)\n	(firstname \"Mar√≠a E\")\n	(initial \"ME\")\n	(surname \"Chamorro\"))\n\n([GK_3465355] of  Person\n\n	(_displayName \"Alnemri, E S\")\n	(DB_ID 3465355)\n	(firstname \"E S\")\n	(initial \"ES\")\n	(surname \"Alnemri\"))\n\n([GK_3465356] of  Person\n\n	(_displayName \"Katajam√§ki, Tuire\")\n	(DB_ID 3465356)\n	(firstname \"Tuire\")\n	(initial \"T\")\n	(surname \"Katajam√§ki\"))\n\n([GK_3465357] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] FLIP(S) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 3465357)\n	(endCoordinate 221)\n	(name\n		\"FLIP(S)\"\n		\"CASP8 and FADD-like apoptosis regulator Isoform 2\"\n		\"CLARP2\"\n		\"MRIT beta-1\"\n		\"CASH beta\")\n	(referenceEntity [GK_145326])\n	(species [GK_48887])\n	(stableIdentifier [GK_3656025])\n	(startCoordinate 1))\n\n([GK_3465358] of  Person\n\n	(_displayName \"Vornlocher, Hans-Peter\")\n	(DB_ID 3465358)\n	(firstname \"Hans-Peter\")\n	(initial \"HP\")\n	(surname \"Vornlocher\"))\n\n([GK_3465359] of  Person\n\n	(_displayName \"Kim, Young-Youl\")\n	(DB_ID 3465359)\n	(firstname \"Young-Youl\")\n	(initial \"YY\")\n	(surname \"Kim\"))\n\n([GK_3465360] of  Person\n\n	(_displayName \"Vota, Daiana\")\n	(DB_ID 3465360)\n	(firstname \"Daiana\")\n	(initial \"D\")\n	(surname \"Vota\"))\n\n([GK_3465361] of  LiteratureReference\n\n	(_displayName \"Identification of c-FLIP(L) and c-FLIP(S) as critical regulators of death receptor-induced apoptosis in pancreatic cancer cells\")\n	(author\n		[GK_3465338]\n		[GK_3465447]\n		[GK_3465417]\n		[GK_3465439]\n		[GK_3465418]\n		[GK_3465421]\n		[GK_3465457])\n	(DB_ID 3465361)\n	(journal \"Gut\")\n	(pages \"225-37\")\n	(pubMedIdentifier 20876774)\n	(title \"Identification of c-FLIP(L) and c-FLIP(S) as critical regulators of death receptor-induced apoptosis in pancreatic cancer cells\")\n	(volume 60)\n	(year 2011))\n\n([GK_3465362] of  Person\n\n	(_displayName \"Palacios, Carmen\")\n	(DB_ID 3465362)\n	(firstname \"Carmen\")\n	(initial \"C\")\n	(surname \"Palacios\"))\n\n([GK_3465363] of  Person\n\n	(_displayName \"Reichwein, M\")\n	(DB_ID 3465363)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Reichwein\"))\n\n([GK_3465364] of  Person\n\n	(_displayName \"Barnhart, Bryan C\")\n	(DB_ID 3465364)\n	(firstname \"Bryan C\")\n	(initial \"BC\")\n	(surname \"Barnhart\"))\n\n([GK_3465365] of  Person\n\n	(_displayName \"Banks, S\")\n	(DB_ID 3465365)\n	(firstname \"S\")\n	(initial \"S\")\n	(surname \"Banks\"))\n\n([GK_3465366] of  Person\n\n	(_displayName \"Halpin, D R\")\n	(DB_ID 3465366)\n	(firstname \"D R\")\n	(initial \"DR\")\n	(surname \"Halpin\"))\n\n([GK_3465367] of  Person\n\n	(_displayName \"Lens, S\")\n	(DB_ID 3465367)\n	(firstname \"S\")\n	(initial \"S\")\n	(surname \"Lens\"))\n\n([GK_3465368] of  LiteratureReference\n\n	(_displayName \"Rapid turnover of c-FLIPshort is determined by its unique C-terminal tail\")\n	(author\n		[GK_3465339]\n		[GK_3465462]\n		[GK_3214933]\n		[GK_3465487]\n		[GK_3465356]\n		[GK_3465351]\n		[GK_3214959]\n		[GK_3465373])\n	(DB_ID 3465368)\n	(journal \"J. Biol. Chem.\")\n	(pages \"27345-55\")\n	(pubMedIdentifier 15886205)\n	(title \"Rapid turnover of c-FLIPshort is determined by its unique C-terminal tail\")\n	(volume 280)\n	(year 2005))\n\n([GK_3465369] of  Person\n\n	(_displayName \"Litwack, G\")\n	(DB_ID 3465369)\n	(firstname \"G\")\n	(initial \"G\")\n	(surname \"Litwack\"))\n\n([GK_3465370] of  Person\n\n	(_displayName \"Siegmund, Daniela\")\n	(DB_ID 3465370)\n	(firstname \"Daniela\")\n	(initial \"D\")\n	(surname \"Siegmund\"))\n\n([GK_3465371] of  Person\n\n	(_displayName \"Kracht, Michael\")\n	(DB_ID 3465371)\n	(firstname \"Michael\")\n	(initial \"M\")\n	(surname \"Kracht\"))\n\n([GK_3465372] of  Person\n\n	(_displayName \"Hsia, C\")\n	(DB_ID 3465372)\n	(firstname \"C\")\n	(initial \"C\")\n	(surname \"Hsia\"))\n\n([GK_3465373] of  Person\n\n	(_displayName \"Eriksson, John E\")\n	(DB_ID 3465373)\n	(firstname \"John E\")\n	(initial \"JE\")\n	(surname \"Eriksson\"))\n\n([GK_3465374] of  Person\n\n	(_displayName \"Westerlund, Mia\")\n	(DB_ID 3465374)\n	(firstname \"Mia\")\n	(initial \"M\")\n	(surname \"Westerlund\"))\n\n([GK_3465375] of  Person\n\n	(_displayName \"Taylor, Alison\")\n	(DB_ID 3465375)\n	(firstname \"Alison\")\n	(initial \"A\")\n	(surname \"Taylor\"))\n\n([GK_3465376] of  Person\n\n	(_displayName \"Kataoka, T\")\n	(DB_ID 3465376)\n	(firstname \"T\")\n	(initial \"T\")\n	(surname \"Kataoka\"))\n\n([GK_3465377] of  Person\n\n	(_displayName \"Inoue, Hidekazu\")\n	(DB_ID 3465377)\n	(firstname \"Hidekazu\")\n	(initial \"H\")\n	(surname \"Inoue\"))\n\n([GK_3465378] of  Person\n\n	(_displayName \"Asaoka, Tomoko\")\n	(DB_ID 3465378)\n	(firstname \"Tomoko\")\n	(initial \"T\")\n	(surname \"Asaoka\"))\n\n([GK_3465379] of  Person\n\n	(_displayName \"Srinivasula, S M\")\n	(DB_ID 3465379)\n	(firstname \"S M\")\n	(initial \"SM\")\n	(surname \"Srinivasula\"))\n\n([GK_3465380] of  LiteratureReference\n\n	(_displayName \"Modeling reveals that dynamic regulation of c-FLIP levels determines cell-to-cell distribution of CD95-mediated apoptosis\")\n	(author\n		[GK_3465384]\n		[GK_3465342]\n		[GK_3465378]\n		[GK_3465374]\n		[GK_3465433]\n		[GK_3465416]\n		[GK_3465373]\n		[GK_3465450])\n	(DB_ID 3465380)\n	(journal \"J. Biol. Chem.\")\n	(pages \"18375-82\")\n	(pubMedIdentifier 21324892)\n	(title \"Modeling reveals that dynamic regulation of c-FLIP levels determines cell-to-cell distribution of CD95-mediated apoptosis\")\n	(volume 286)\n	(year 2011))\n\n([GK_3465381] of  Person\n\n	(_displayName \"Kiem, Hans-Peter\")\n	(DB_ID 3465381)\n	(firstname \"Hans-Peter\")\n	(initial \"HP\")\n	(surname \"Kiem\"))\n\n([GK_3465382] of  LiteratureReference\n\n	(_displayName \"Fas-associated death domain protein (FADD) and caspase-8 mediate up-regulation of c-Fos by Fas ligand and tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) via a FLICE inhibitory protein (FLIP)-regulated pathway\")\n	(author\n		[GK_3465347]\n		[GK_3465414]\n		[GK_3465428]\n		[GK_3465469]\n		[GK_2672174]\n		[GK_3465363]\n		[GK_3465436]\n		[GK_3465420]\n		[GK_2672175]\n		[GK_3465508])\n	(DB_ID 3465382)\n	(journal \"J. Biol. Chem.\")\n	(pages \"32585-90\")\n	(pubMedIdentifier 11384965)\n	(title \"Fas-associated death domain protein (FADD) and caspase-8 mediate up-regulation of c-Fos by Fas ligand and tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) via a FLICE inhibitory protein (FLIP)-regulated pathway\")\n	(volume 276)\n	(year 2001))\n\n([GK_3465383] of  Person\n\n	(_displayName \"Tschopp, J√ºrg\")\n	(DB_ID 3465383)\n	(firstname \"J√ºrg\")\n	(initial \"J\")\n	(surname \"Tschopp\"))\n\n([GK_3465384] of  Person\n\n	(_displayName \"Toivonen, Hannu T\")\n	(DB_ID 3465384)\n	(firstname \"Hannu T\")\n	(initial \"HT\")\n	(surname \"Toivonen\"))\n\n([GK_3465385] of  LiteratureReference\n\n	(_displayName \"The role of c-FLIP in modulation of CD95-induced apoptosis\")\n	(author\n		[GK_2672219]\n		[GK_3465496]\n		[GK_2672214]\n		[GK_2672197])\n	(DB_ID 3465385)\n	(journal \"J. Biol. Chem.\")\n	(pages \"1541-8\")\n	(pubMedIdentifier 9880531)\n	(title \"The role of c-FLIP in modulation of CD95-induced apoptosis\")\n	(volume 274)\n	(year 1999))\n\n([GK_3465386] of  LiteratureReference\n\n	(_displayName \"Mechanism of procaspase-8 activation by c-FLIPL\")\n	(author\n		[GK_2672201]\n		[GK_2672164]\n		[GK_141951])\n	(DB_ID 3465386)\n	(journal \"Proc. Natl. Acad. Sci. U.S.A.\")\n	(pages \"8169-74\")\n	(pubMedIdentifier 19416807)\n	(title \"Mechanism of procaspase-8 activation by c-FLIPL\")\n	(volume 106)\n	(year 2009))\n\n([GK_3465387] of  Person\n\n	(_displayName \"Sharp, Darcie A\")\n	(DB_ID 3465387)\n	(firstname \"Darcie A\")\n	(initial \"DA\")\n	(surname \"Sharp\"))\n\n([GK_3465388] of  Person\n\n	(_displayName \"Kaunisto, A\")\n	(DB_ID 3465388)\n	(firstname \"A\")\n	(initial \"A\")\n	(surname \"Kaunisto\"))\n\n([GK_3465389] of  Person\n\n	(_displayName \"Nesse, Alcira\")\n	(DB_ID 3465389)\n	(firstname \"Alcira\")\n	(initial \"A\")\n	(surname \"Nesse\"))\n\n([GK_3465391] of  Person\n\n	(_displayName \"Scheurich, Peter\")\n	(DB_ID 3465391)\n	(firstname \"Peter\")\n	(initial \"P\")\n	(surname \"Scheurich\"))\n\n([GK_3465392] of  Person\n\n	(_displayName \"Lawrence, David A\")\n	(DB_ID 3465392)\n	(firstname \"David A\")\n	(initial \"DA\")\n	(surname \"Lawrence\"))\n\n([GK_3465393] of  LiteratureReference\n\n	(_displayName \"NF-kappaB inducers upregulate cFLIP, a cycloheximide-sensitive inhibitor of death receptor signaling\")\n	(author\n		[GK_3465445]\n		[GK_3465347]\n		[GK_3465420]\n		[GK_3465508])\n	(DB_ID 3465393)\n	(journal \"Mol. Cell. Biol.\")\n	(pages \"3964-73\")\n	(pubMedIdentifier 11359904)\n	(title \"NF-kappaB inducers upregulate cFLIP, a cycloheximide-sensitive inhibitor of death receptor signaling\")\n	(volume 21)\n	(year 2001))\n\n([GK_3465394] of  Person\n\n	(_displayName \"Holler, Nils\")\n	(DB_ID 3465394)\n	(firstname \"Nils\")\n	(initial \"N\")\n	(surname \"Holler\"))\n\n([GK_3465395] of  Summation\n\n	(_displayName \"The short form of cellular FLIP (FLIP(S) or c-FLIPS) has two...\")\n	(DB_ID 3465395)\n	(literatureReference\n		[GK_3465385]\n		[GK_3465368]\n		[GK_3465330]\n		[GK_3465503])\n	(text \"The short form of cellular FLIP (FLIP(S) or c-FLIPS) has two death effector domain DEDs, which can bind to FADD and caspase-8.  FLIP(S) protects the cells from apoptosis by inhibiting the processing of caspase-8 at the receptor level (Scaffidi C et al. 1999; Micheau O et al 2001)<p>FLIP(S) is a short-lived protein which is sensitive to ubiquitination and proteasomal degradation (Poukkula M et al. 2005;)  Protein kinase C (PKC)- mediated phosphorylation of FLIP(S) at Ser193 was shown to prolongs the half-lives of FLIP-S by inhibiting its polyubiquitination (Kaunisto A et al. 2009)\"))\n\n([GK_3465396] of  LiteratureReference\n\n	(_displayName \"Selective inhibition of FLICE-like inhibitory protein expression with small interfering RNA oligonucleotides is sufficient to sensitize tumor cells for TRAIL-induced apoptosis\")\n	(author\n		[GK_3465370]\n		[GK_3465507]\n		[GK_429711]\n		[GK_3465358]\n		[GK_3465486])\n	(DB_ID 3465396)\n	(journal \"Mol. Med.\")\n	(pages \"725-32\")\n	(pubMedIdentifier 12520089)\n	(title \"Selective inhibition of FLICE-like inhibitory protein expression with small interfering RNA oligonucleotides is sufficient to sensitize tumor cells for TRAIL-induced apoptosis\")\n	(volume 8)\n	(year 2002))\n\n([GK_3465397] of  LiteratureReference\n\n	(_displayName \"c-FLIPR, a new regulator of death receptor-induced apoptosis\")\n	(author\n		[GK_3465406]\n		[GK_3465475]\n		[GK_3465427]\n		[GK_3465348]\n		[GK_3465474])\n	(DB_ID 3465397)\n	(journal \"J. Biol. Chem.\")\n	(pages \"14507-13\")\n	(pubMedIdentifier 15701649)\n	(title \"c-FLIPR, a new regulator of death receptor-induced apoptosis\")\n	(volume 280)\n	(year 2005))\n\n([GK_3465398] of  Person\n\n	(_displayName \"Holler, N\")\n	(DB_ID 3465398)\n	(firstname \"N\")\n	(initial \"N\")\n	(surname \"Holler\"))\n\n([GK_3465399] of  Person\n\n	(_displayName \"Ye, H\")\n	(DB_ID 3465399)\n	(firstname \"H\")\n	(initial \"H\")\n	(surname \"Ye\"))\n\n([GK_3465400] of  Person\n\n	(_displayName \"Ohmori, Shigeru\")\n	(DB_ID 3465400)\n	(firstname \"Shigeru\")\n	(initial \"S\")\n	(surname \"Ohmori\"))\n\n([GK_3465401] of  Person\n\n	(_displayName \"Dittrich-Breiholz, Oliver\")\n	(DB_ID 3465401)\n	(firstname \"Oliver\")\n	(initial \"O\")\n	(surname \"Dittrich-Breiholz\"))\n\n([GK_3465402] of  Summation\n\n	(_displayName \"Following recruitment to the DISC, FLIP(L) forms a heterodim...\")\n	(DB_ID 3465402)\n	(literatureReference\n		[GK_1267757]\n		[GK_3465405]\n		[GK_3465422]\n		[GK_3465503]\n		[GK_3465393])\n	(text \"Following recruitment to the DISC, FLIP(L) forms a heterodimer with caspase-8 through both death effector domain (DED) and caspase-like domain (CLD). In addition, FLIP-L can also regulate TNF-R1 signaling via interaction with the DED of FADD. The regulatory function of FLIP(L) has been found to differ depending on its expression levels. c-FLIP was shown to inhibit death receptor (DR)-mediated apoptosis only when expressed at high levels, while low cell levels of FLIP(L) enhanced DR signaling to apoptosis (Boatright KM et al. 2004; Okano H et al. 2003; Yerbes R et al. 2011). The expression levels of c-FLIP proteins were shown to be regulated by NFkappaB signaling pathway (Micheau O et a. 2001; Kreuz S et al 2001).\"))\n\n([GK_3465403] of  Person\n\n	(_displayName \"Murata, Kazumoto\")\n	(DB_ID 3465403)\n	(firstname \"Kazumoto\")\n	(initial \"K\")\n	(surname \"Murata\"))\n\n([GK_3465404] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CASP8(1-374) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 3465404)\n	(endCoordinate 374)\n	(name\n		\"CASP8(1-374)\"\n		\"caspase-8 p43/41 fragment\"\n		\"ICE-like apoptotic protease 5\"\n		\"MORT1-associated CED-3 homolog\"\n		\"MACH\"\n		\"FADD-homologous ICE/CED-3- like protease\"\n		\"FADD-like ICE\"\n		\"FLICE\"\n		\"Apoptotic cysteine protease\"\n		\"Apoptotic protease Mch-5\"\n		\"CAP4\")\n	(referenceEntity [GK_57032])\n	(species [GK_48887])\n	(stableIdentifier [GK_3655311])\n	(startCoordinate 1))\n\n([GK_3465405] of  LiteratureReference\n\n	(_displayName \"Cellular FLICE/caspase-8-inhibitory protein as a principal regulator of cell death and survival in human hepatocellular carcinoma\")\n	(author\n		[GK_3465509]\n		[GK_3465479]\n		[GK_3465377]\n		[GK_3465409]\n		[GK_3465471]\n		[GK_3465407]\n		[GK_3465440]\n		[GK_3465426]\n		[GK_3465400]\n		[GK_3465477]\n		[GK_3465403]\n		[GK_3465465])\n	(DB_ID 3465405)\n	(journal \"Lab. Invest.\")\n	(pages \"1033-43\")\n	(pubMedIdentifier 12861043)\n	(title \"Cellular FLICE/caspase-8-inhibitory protein as a principal regulator of cell death and survival in human hepatocellular carcinoma\")\n	(volume 83)\n	(year 2003))\n\n([GK_3465406] of  Person\n\n	(_displayName \"Golks, Alexander\")\n	(DB_ID 3465406)\n	(firstname \"Alexander\")\n	(initial \"A\")\n	(surname \"Golks\"))\n\n([GK_3465407] of  Person\n\n	(_displayName \"Deguchi, Masatoshi\")\n	(DB_ID 3465407)\n	(firstname \"Masatoshi\")\n	(initial \"M\")\n	(surname \"Deguchi\"))\n\n([GK_3465408] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CFLAR(1-376) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 3465408)\n	(endCoordinate 376)\n	(name\n		\"CFLAR(1-376)\"\n		\"FLIP(L)(1-376)\"\n		\"FLIP(L) subunit p43\"\n		\"CASP8 and FADD-like apoptosis regulator Isoform 1 subunit p43\"\n		\"CLARP1\"\n		\"MRIT alpha-1\"\n		\"CASH alpha\"\n		\"I-FLICE 1\"\n		\"FLAME-1 gamma\"\n		\"CASP8 and FADD-like apoptosis regulator\")\n	(referenceEntity [GK_401351])\n	(species [GK_48887])\n	(stableIdentifier [GK_3655628])\n	(startCoordinate 1))\n\n([GK_3465409] of  Person\n\n	(_displayName \"Kawakita, Tomoyuki\")\n	(DB_ID 3465409)\n	(firstname \"Tomoyuki\")\n	(initial \"T\")\n	(surname \"Kawakita\"))\n\n([GK_3465411] of  Person\n\n	(_displayName \"Park, Chan\")\n	(DB_ID 3465411)\n	(firstname \"Chan\")\n	(initial \"C\")\n	(surname \"Park\"))\n\n([GK_3465412] of  LiteratureReference\n\n	(_displayName \"N-terminal fragment of c-FLIP(L) processed by caspase 8 specifically interacts with TRAF2 and induces activation of the NF-kappaB signaling pathway\")\n	(author\n		[GK_1267838]\n		[GK_3465383])\n	(DB_ID 3465412)\n	(journal \"Mol. Cell. Biol.\")\n	(pages \"2627-36\")\n	(pubMedIdentifier 15024054)\n	(title \"N-terminal fragment of c-FLIP(L) processed by caspase 8 specifically interacts with TRAF2 and induces activation of the NF-kappaB signaling pathway\")\n	(volume 24)\n	(year 2004))\n\n([GK_3465413] of  Person\n\n	(_displayName \"Reginato, Mauricio J\")\n	(DB_ID 3465413)\n	(firstname \"Mauricio J\")\n	(initial \"MJ\")\n	(surname \"Reginato\"))\n\n([GK_3465414] of  Person\n\n	(_displayName \"Mauri, D\")\n	(DB_ID 3465414)\n	(firstname \"D\")\n	(initial \"D\")\n	(surname \"Mauri\"))\n\n([GK_3465415] of  LiteratureReference\n\n	(_displayName \"Structural revelations of TRAF2 function in TNF receptor signaling pathway\")\n	(author\n		[GK_3465349]\n		[GK_1237360]\n		[GK_1237372]\n		[GK_1237368])\n	(DB_ID 3465415)\n	(journal \"Adv. Exp. Med. Biol.\")\n	(pages \"93-113\")\n	(pubMedIdentifier 17633020)\n	(title \"Structural revelations of TRAF2 function in TNF receptor signaling pathway\")\n	(volume 597)\n	(year 2007))\n\n([GK_3465416] of  Person\n\n	(_displayName \"Mikhailov, Andrey\")\n	(DB_ID 3465416)\n	(firstname \"Andrey\")\n	(initial \"A\")\n	(surname \"Mikhailov\"))\n\n([GK_3465417] of  Person\n\n	(_displayName \"Zhou, Shaoxia\")\n	(DB_ID 3465417)\n	(firstname \"Shaoxia\")\n	(initial \"S\")\n	(surname \"Zhou\"))\n\n([GK_3465418] of  Person\n\n	(_displayName \"M√∂ller, Peter\")\n	(DB_ID 3465418)\n	(firstname \"Peter\")\n	(initial \"P\")\n	(surname \"M√∂ller\"))\n\n([GK_3465419] of  Person\n\n	(_displayName \"Liou, H C\")\n	(DB_ID 3465419)\n	(firstname \"H C\")\n	(initial \"HC\")\n	(surname \"Liou\"))\n\n([GK_3465420] of  Person\n\n	(_displayName \"Scheurich, P\")\n	(DB_ID 3465420)\n	(firstname \"P\")\n	(initial \"P\")\n	(surname \"Scheurich\"))\n\n([GK_3465421] of  Person\n\n	(_displayName \"Debatin, Klaus-Michael\")\n	(DB_ID 3465421)\n	(firstname \"Klaus-Michael\")\n	(initial \"KM\")\n	(surname \"Debatin\"))\n\n([GK_3465422] of  LiteratureReference\n\n	(_displayName \"Cellular FLIP(L) plays a survival role and regulates morphogenesis in breast epithelial cells\")\n	(author\n		[GK_3465466]\n		[GK_3465362]\n		[GK_3465413]\n		[GK_3465461])\n	(DB_ID 3465422)\n	(journal \"Biochim. Biophys. Acta\")\n	(pages \"168-78\")\n	(pubMedIdentifier 20951169)\n	(title \"Cellular FLIP(L) plays a survival role and regulates morphogenesis in breast epithelial cells\")\n	(volume 1813)\n	(year 2011))\n\n([GK_3465423] of  LiteratureReference\n\n	(_displayName \"Association of TRAF2 with the short form of cellular FLICE-like inhibitory protein prevents TNFR1-mediated apoptosis\")\n	(author\n		[GK_3465442]\n		[GK_3465411]\n		[GK_3465491]\n		[GK_3465359])\n	(DB_ID 3465423)\n	(journal \"J Mol Signal\")\n	(pages \"2\")\n	(pubMedIdentifier 18190721)\n	(title \"Association of TRAF2 with the short form of cellular FLICE-like inhibitory protein prevents TNFR1-mediated apoptosis\")\n	(volume 3)\n	(year 2008))\n\n([GK_3465424] of  Person\n\n	(_displayName \"Spaulding, Emily Y\")\n	(DB_ID 3465424)\n	(firstname \"Emily Y\")\n	(initial \"EY\")\n	(surname \"Spaulding\"))\n\n([GK_3465425] of  LiteratureReference\n\n	(_displayName \"The long form of FLIP is an activator of caspase-8 at the Fas death-inducing signaling complex\")\n	(author\n		[GK_3465492]\n		[GK_2685159]\n		[GK_3465343]\n		[GK_3465394]\n		[GK_3465383]\n		[GK_3465502]\n		[GK_2584571]\n		[GK_2584570])\n	(DB_ID 3465425)\n	(journal \"J. Biol. Chem.\")\n	(pages \"45162-71\")\n	(pubMedIdentifier 12215447)\n	(title \"The long form of FLIP is an activator of caspase-8 at the Fas death-inducing signaling complex\")\n	(volume 277)\n	(year 2002))\n\n([GK_3465426] of  Person\n\n	(_displayName \"Sakai, Takahisa\")\n	(DB_ID 3465426)\n	(firstname \"Takahisa\")\n	(initial \"T\")\n	(surname \"Sakai\"))\n\n([GK_3465427] of  Person\n\n	(_displayName \"Fritsch, Cornelius\")\n	(DB_ID 3465427)\n	(firstname \"Cornelius\")\n	(initial \"C\")\n	(surname \"Fritsch\"))\n\n([GK_3465428] of  Person\n\n	(_displayName \"Peters, N\")\n	(DB_ID 3465428)\n	(firstname \"N\")\n	(initial \"N\")\n	(surname \"Peters\"))\n\n([GK_3465429] of  Reaction\n\n	(_displayName \"[Hsa] FLIP(S) and procaspase-8 form heterodimer\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(compartment\n		[GK_70101]\n		[GK_876])\n	(DB_ID 3465429)\n	(edited [GK_3508822])\n	(goBiologicalProcess [GK_3446166])\n	(input\n		[GK_3465443]\n		[GK_3465357])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_1267786]\n		[GK_3465423]\n		[GK_3465503]\n		[GK_3465385])\n	(name \"FLIP(S) and procaspase-8 form heterodimer\")\n	(output [GK_3465352])\n	(precedingEvent\n		[GK_83586]\n		[GK_75240]\n		[GK_75146])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654525])\n	(summation [GK_3465395]))\n\n([GK_3465430] of  Person\n\n	(_displayName \"Alevizopoulos, K\")\n	(DB_ID 3465430)\n	(firstname \"K\")\n	(initial \"K\")\n	(surname \"Alevizopoulos\"))\n\n([GK_3465431] of  Complex\n\n	(_displayName \"[Hsa] DISC:p43/p41CASP8:p43 FLIP(L) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3465431)\n	(hasComponent\n		[GK_3465408]\n		[GK_2671814]\n		[GK_3465404])\n	(isChimeric FALSE)\n	(name \"DISC:p43/p41CASP8:p43 FLIP(L)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3654957]))\n\n([GK_3465432] of  Summation\n\n	(_displayName \"Pro-caspase-8 and FLIP(L) are recruited to FAS/CD95 receptor...\")\n	(DB_ID 3465432)\n	(literatureReference\n		[GK_3465456]\n		[GK_3465380]\n		[GK_1267757]\n		[GK_3465405]\n		[GK_3465503]\n		[GK_3465393]\n		[GK_3465478])\n	(text \"Pro-caspase-8 and FLIP(L) are recruited to FAS/CD95 receptor complex where FLIP(L) forms a heterodimer with caspase-8 through both death effector domain (DED) and caspase-like domain (CLD). In addition, FLIP(L) can also regulate signaling via interaction with the DED of FADD. The regulatory function of FLIP(L) has been found to differ depending on its expression levels. FLIP(L) was shown to inhibit death receptor (DR)-mediated apoptosis only when expressed at high levels, while low cell levels of FLIP-L enhanced DR signaling to apoptosis (Chang DW et al. 2002; Fricker N et al. 2010; Toivonen HT et al. 2011; Boatright KM et al. 2004; Okano H et al. 2003). The expression levels of c-FLIP proteins were shown to be regulated by NFkappaB signaling pathway (Micheau O et a. 2001; Kreuz S et al 2001).\"))\n\n([GK_3465433] of  Person\n\n	(_displayName \"Pettersson, Frank\")\n	(DB_ID 3465433)\n	(firstname \"Frank\")\n	(initial \"F\")\n	(surname \"Pettersson\"))\n\n([GK_3465434] of  Person\n\n	(_displayName \"Kochin, V\")\n	(DB_ID 3465434)\n	(firstname \"V\")\n	(initial \"V\")\n	(surname \"Kochin\"))\n\n([GK_3465435] of  Person\n\n	(_displayName \"Yaish-Ohad, Shoshanit\")\n	(DB_ID 3465435)\n	(firstname \"Shoshanit\")\n	(initial \"S\")\n	(surname \"Yaish-Ohad\"))\n\n([GK_3465436] of  Person\n\n	(_displayName \"Blenis, J\")\n	(DB_ID 3465436)\n	(firstname \"J\")\n	(initial \"J\")\n	(surname \"Blenis\"))\n\n([GK_3465437] of  Person\n\n	(_displayName \"Rushworth, Stuart A\")\n	(DB_ID 3465437)\n	(firstname \"Stuart A\")\n	(initial \"SA\")\n	(surname \"Rushworth\"))\n\n([GK_3465439] of  Person\n\n	(_displayName \"Bachem, Max G\")\n	(DB_ID 3465439)\n	(firstname \"Max G\")\n	(initial \"MG\")\n	(surname \"Bachem\"))\n\n([GK_3465440] of  Person\n\n	(_displayName \"Sugimoto, Kazushi\")\n	(DB_ID 3465440)\n	(firstname \"Kazushi\")\n	(initial \"K\")\n	(surname \"Sugimoto\"))\n\n([GK_3465441] of  Person\n\n	(_displayName \"Fernandes-Alnemri, T\")\n	(DB_ID 3465441)\n	(firstname \"T\")\n	(initial \"T\")\n	(surname \"Fernandes-Alnemri\"))\n\n([GK_3465442] of  Person\n\n	(_displayName \"Kim, Dong-Joon\")\n	(DB_ID 3465442)\n	(firstname \"Dong-Joon\")\n	(initial \"DJ\")\n	(surname \"Kim\"))\n\n([GK_3465443] of  Complex\n\n	(_displayName \"[Hsa] DISC:procaspase-8 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3465443)\n	(hasComponent\n		[GK_57031]\n		[GK_2671814])\n	(name\n		\"DISC:procaspase-8\"\n		\"death-inducing signaling complex:procaspase-8\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655925]))\n\n([GK_3465444] of  LiteratureReference\n\n	(_displayName \"Casper is a FADD- and caspase-related inducer of apoptosis\")\n	(author\n		[GK_3465505]\n		[GK_3465366]\n		[GK_75697])\n	(DB_ID 3465444)\n	(journal \"Immunity\")\n	(pages \"751-63\")\n	(pubMedIdentifier 9208847)\n	(title \"Casper is a FADD- and caspase-related inducer of apoptosis\")\n	(volume 6)\n	(year 1997))\n\n([GK_3465445] of  Person\n\n	(_displayName \"Kreuz, S\")\n	(DB_ID 3465445)\n	(firstname \"S\")\n	(initial \"S\")\n	(surname \"Kreuz\"))\n\n([GK_3465446] of  Summation\n\n	(_displayName \"Pro-caspase-8 and FLIP(L) are recruited to the DISC. Followi...\")\n	(DB_ID 3465446)\n	(literatureReference\n		[GK_3465460]\n		[GK_3465396]\n		[GK_1267757]\n		[GK_3465405]\n		[GK_3465422]\n		[GK_3465503]\n		[GK_3465393])\n	(text \"Pro-caspase-8 and FLIP(L) are recruited to the DISC. Following recruitment to the DISC, FLIP-L forms a heterodimer with caspase-8 through both death effector domain (DED) and caspase-like domain (CLD). In addition, FLIP(L) can also regulate signaling via interaction with the DED of FADD. The regulatory function of FLIP(L) has been found to differ depending on its expression levels. FLIP(L) was shown to inhibit death receptor (DR)-mediated apoptosis only when expressed at high levels, while low cell levels of FLIP(L) enhanced DR signaling to apoptosis (Sharp DA et al. 2005; Siegmund D et al. 2002; Boatright KM et al. 2004; Okano H et al. 2003; Yerbes R et al. 2011). The expression levels of c-FLIP proteins were shown to be regulated by NFkappaB signaling pathway (Micheau O et a. 2001; Kreuz S et al 2001).\"))\n\n([GK_3465447] of  Person\n\n	(_displayName \"Stadel, Dominic\")\n	(DB_ID 3465447)\n	(firstname \"Dominic\")\n	(initial \"D\")\n	(surname \"Stadel\"))\n\n([GK_3465448] of  BlackBoxEvent\n\n	(_displayName \"[Hsa] Caspase-8 and FLIP(L) processing at DISC\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(catalystActivity [GK_139951])\n	(compartment [GK_70101])\n	(DB_ID 3465448)\n	(edited [GK_3508822])\n	(input [GK_3371381])\n	(literatureReference\n		[GK_1267786]\n		[GK_3465386]\n		[GK_3465412]\n		[GK_1267836]\n		[GK_3465456]\n		[GK_3465334])\n	(name \"Caspase-8 and FLIP(L) processing at DISC\")\n	(output\n		[GK_3465431]\n		[GK_3465483])\n	(precedingEvent\n		[GK_3465459]\n		[GK_3371360]\n		[GK_3371359])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654297])\n	(summation [GK_3465328]))\n\n([GK_3465449] of  Person\n\n	(_displayName \"Irmler, M\")\n	(DB_ID 3465449)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Irmler\"))\n\n([GK_3465450] of  Person\n\n	(_displayName \"Sax√©n, Henrik\")\n	(DB_ID 3465450)\n	(firstname \"Henrik\")\n	(initial \"H\")\n	(surname \"Sax√©n\"))\n\n([GK_3465451] of  Person\n\n	(_displayName \"Armstrong, R C\")\n	(DB_ID 3465451)\n	(firstname \"R C\")\n	(initial \"RC\")\n	(surname \"Armstrong\"))\n\n([GK_3465452] of  Person\n\n	(_displayName \"Martinon, F\")\n	(DB_ID 3465452)\n	(firstname \"F\")\n	(initial \"F\")\n	(surname \"Martinon\"))\n\n([GK_3465453] of  Person\n\n	(_displayName \"Bullrich, F\")\n	(DB_ID 3465453)\n	(firstname \"F\")\n	(initial \"F\")\n	(surname \"Bullrich\"))\n\n([GK_3465454] of  LiteratureReference\n\n	(_displayName \"c-FLIP is involved in erythropoietin-mediated protection of erythroid-differentiated cells from TNF-alpha-induced apoptosis\")\n	(author\n		[GK_3465336]\n		[GK_3465360]\n		[GK_3465350]\n		[GK_3465354]\n		[GK_3465389])\n	(DB_ID 3465454)\n	(journal \"Cell Biol. Int.\")\n	(pages \"621-30\")\n	(pubMedIdentifier 20218968)\n	(title \"c-FLIP is involved in erythropoietin-mediated protection of erythroid-differentiated cells from TNF-alpha-induced apoptosis\")\n	(volume 34)\n	(year 2010))\n\n([GK_3465455] of  LiteratureReference\n\n	(_displayName \"TNF signaling gets FLIPped off: TNF-induced regulation of FLIP\")\n	(author\n		[GK_3465437]\n		[GK_3465375]\n		[GK_3465337]\n		[GK_3465468])\n	(DB_ID 3465455)\n	(journal \"Cell Cycle\")\n	(pages \"194-9\")\n	(pubMedIdentifier 18256533)\n	(title \"TNF signaling gets FLIPped off: TNF-induced regulation of FLIP\")\n	(volume 7)\n	(year 2008))\n\n([GK_3465456] of  LiteratureReference\n\n	(_displayName \"Model-based dissection of CD95 signaling dynamics reveals both a pro- and antiapoptotic role of c-FLIPL\")\n	(author\n		[GK_3465472]\n		[GK_3465490]\n		[GK_3465340]\n		[GK_3465335]\n		[GK_3465348]\n		[GK_3465474])\n	(DB_ID 3465456)\n	(journal \"J. Cell Biol.\")\n	(pages \"377-89\")\n	(pubMedIdentifier 20696707)\n	(title \"Model-based dissection of CD95 signaling dynamics reveals both a pro- and antiapoptotic role of c-FLIPL\")\n	(volume 190)\n	(year 2010))\n\n([GK_3465457] of  Person\n\n	(_displayName \"Fulda, Simone\")\n	(DB_ID 3465457)\n	(firstname \"Simone\")\n	(initial \"S\")\n	(surname \"Fulda\"))\n\n([GK_3465458] of  Person\n\n	(_displayName \"Mikhailov, A\")\n	(DB_ID 3465458)\n	(firstname \"A\")\n	(initial \"A\")\n	(surname \"Mikhailov\"))\n\n([GK_3465459] of  Reaction\n\n	(_displayName \"[Hsa] FLIP(L) and procaspase-8 form heterodimer in FasL/CD95 signaling\")\n	(_doRelease TRUE)\n	(authored [GK_3371406])\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 3465459)\n	(edited [GK_3508822])\n	(figure [GK_141227])\n	(input\n		[GK_75114]\n		[GK_3371356])\n	(literatureReference\n		[GK_3465456]\n		[GK_3465425]\n		[GK_3465476]\n		[GK_3465382])\n	(name \"FLIP(L) and procaspase-8 form heterodimer in FasL/CD95 signaling\")\n	(output [GK_3465499])\n	(precedingEvent [GK_83586])\n	(releaseDate \"2013-06-11\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_3654514])\n	(summation [GK_3465432]))\n\n([GK_3465460] of  LiteratureReference\n\n	(_displayName \"Selective knockdown of the long variant of cellular FLICE inhibitory protein augments death receptor-mediated caspase-8 activation and apoptosis\")\n	(author\n		[GK_3465387]\n		[GK_3465392]\n		[GK_1250148])\n	(DB_ID 3465460)\n	(journal \"J. Biol. Chem.\")\n	(pages \"19401-9\")\n	(pubMedIdentifier 15760909)\n	(title \"Selective knockdown of the long variant of cellular FLICE inhibitory protein augments death receptor-mediated caspase-8 activation and apoptosis\")\n	(volume 280)\n	(year 2005))\n\n([GK_3465461] of  Person\n\n	(_displayName \"L√≥pez-Rivas, Abelardo\")\n	(DB_ID 3465461)\n	(firstname \"Abelardo\")\n	(initial \"A\")\n	(surname \"L√≥pez-Rivas\"))\n\n([GK_3465462] of  Person\n\n	(_displayName \"Kaunisto, Aura\")\n	(DB_ID 3465462)\n	(firstname \"Aura\")\n	(initial \"A\")\n	(surname \"Kaunisto\"))\n\n([GK_3465463] of  Person\n\n	(_displayName \"Samel, Dierk\")\n	(DB_ID 3465463)\n	(firstname \"Dierk\")\n	(initial \"D\")\n	(surname \"Samel\"))\n\n([GK_3465464] of  Person\n\n	(_displayName \"Ottilie, S\")\n	(DB_ID 3465464)\n	(firstname \"S\")\n	(initial \"S\")\n	(surname \"Ottilie\"))\n\n([GK_3465465] of  Person\n\n	(_displayName \"Nakano, Takeshi\")\n	(DB_ID 3465465)\n	(firstname \"Takeshi\")\n	(initial \"T\")\n	(surname \"Nakano\"))\n\n([GK_3465466] of  Person\n\n	(_displayName \"Yerbes, Rosario\")\n	(DB_ID 3465466)\n	(firstname \"Rosario\")\n	(initial \"R\")\n	(surname \"Yerbes\"))\n\n([GK_3465467] of  Person\n\n	(_displayName \"Gaide, O\")\n	(DB_ID 3465467)\n	(firstname \"O\")\n	(initial \"O\")\n	(surname \"Gaide\"))\n\n([GK_3465468] of  Person\n\n	(_displayName \"MacEwan, David J\")\n	(DB_ID 3465468)\n	(firstname \"David J\")\n	(initial \"DJ\")\n	(surname \"MacEwan\"))\n\n([GK_3465469] of  Person\n\n	(_displayName \"Juo, P\")\n	(DB_ID 3465469)\n	(firstname \"P\")\n	(initial \"P\")\n	(surname \"Juo\"))\n\n([GK_3465470] of  Person\n\n	(_displayName \"Pan, Yi\")\n	(DB_ID 3465470)\n	(firstname \"Yi\")\n	(initial \"Y\")\n	(surname \"Pan\"))\n\n([GK_3465471] of  Person\n\n	(_displayName \"Yamanaka, Takenari\")\n	(DB_ID 3465471)\n	(firstname \"Takenari\")\n	(initial \"T\")\n	(surname \"Yamanaka\"))\n\n([GK_3465472] of  Person\n\n	(_displayName \"Fricker, Nicolai\")\n	(DB_ID 3465472)\n	(firstname \"Nicolai\")\n	(initial \"N\")\n	(surname \"Fricker\"))\n\n([GK_3465473] of  Person\n\n	(_displayName \"Ahmad, M\")\n	(DB_ID 3465473)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Ahmad\"))\n\n([GK_3465474] of  Person\n\n	(_displayName \"Lavrik, Inna N\")\n	(DB_ID 3465474)\n	(firstname \"Inna N\")\n	(initial \"IN\")\n	(surname \"Lavrik\"))\n\n([GK_3465475] of  Person\n\n	(_displayName \"Brenner, Dirk\")\n	(DB_ID 3465475)\n	(firstname \"Dirk\")\n	(initial \"D\")\n	(surname \"Brenner\"))\n\n([GK_3465476] of  LiteratureReference\n\n	(_displayName \"NFkappaB activation by Fas is mediated through FADD, caspase-8, and RIP and is inhibited by FLIP\")\n	(author\n		[GK_3465345]\n		[GK_3465370]\n		[GK_3465511]\n		[GK_3465463]\n		[GK_2562581]\n		[GK_3465489]\n		[GK_2562574]\n		[GK_3465401]\n		[GK_3465371]\n		[GK_3465391]\n		[GK_3465486])\n	(DB_ID 3465476)\n	(journal \"J. Cell Biol.\")\n	(pages \"369-80\")\n	(pubMedIdentifier 15289496)\n	(title \"NFkappaB activation by Fas is mediated through FADD, caspase-8, and RIP and is inhibited by FLIP\")\n	(volume 166)\n	(year 2004))\n\n([GK_3465477] of  Person\n\n	(_displayName \"Fujikawa, Katsuhiko\")\n	(DB_ID 3465477)\n	(firstname \"Katsuhiko\")\n	(initial \"K\")\n	(surname \"Fujikawa\"))\n\n([GK_3465478] of  LiteratureReference\n\n	(_displayName \"c-FLIP(L) is a dual function regulator for caspase-8 activation and CD95-mediated apoptosis\")\n	(author\n		[GK_2984183]\n		[GK_1267808]\n		[GK_3465470]\n		[GK_3465482]\n		[GK_3465364]\n		[GK_3465435]\n		[GK_2984181]\n		[GK_1267777])\n	(DB_ID 3465478)\n	(journal \"EMBO J.\")\n	(pages \"3704-14\")\n	(pubMedIdentifier 12110583)\n	(title \"c-FLIP(L) is a dual function regulator for caspase-8 activation and CD95-mediated apoptosis\")\n	(volume 21)\n	(year 2002))\n\n([GK_3465479] of  Person\n\n	(_displayName \"Shiraki, Katsuya\")\n	(DB_ID 3465479)\n	(firstname \"Katsuya\")\n	(initial \"K\")\n	(surname \"Shiraki\"))\n\n([GK_3465481] of  Person\n\n	(_displayName \"Hahne, M\")\n	(DB_ID 3465481)\n	(firstname \"M\")\n	(initial \"M\")\n	(surname \"Hahne\"))\n\n([GK_3465482] of  Person\n\n	(_displayName \"Algeciras-Schimnich, Alicia\")\n	(DB_ID 3465482)\n	(firstname \"Alicia\")\n	(initial \"A\")\n	(surname \"Algeciras-Schimnich\"))\n\n([GK_3465483] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CFLAR(377-480) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 3465483)\n	(endCoordinate 480)\n	(name\n		\"CFLAR(377-480)\"\n		\"FLIP(L)(377-480)\"\n		\"FLIP(L) subunit p12\"\n		\"CASP8 and FADD-like apoptosis regulator Isoform 1 subunit p12\"\n		\"CLARP1\"\n		\"MRIT alpha-1\"\n		\"CASH alpha\"\n		\"I-FLICE 1\"\n		\"FLAME-1 gamma\"\n		\"CASP8 and FADD-like apoptosis regulator\")\n	(referenceEntity [GK_401351])\n	(species [GK_48887])\n	(stableIdentifier [GK_3655419])\n	(startCoordinate 377))\n\n([GK_3465484] of  Person\n\n	(_displayName \"Croce, C M\")\n	(DB_ID 3465484)\n	(firstname \"C M\")\n	(initial \"CM\")\n	(surname \"Croce\"))\n\n([GK_3465485] of  Person\n\n	(_displayName \"Budd, R C\")\n	(DB_ID 3465485)\n	(firstname \"R C\")\n	(initial \"RC\")\n	(surname \"Budd\"))\n\n([GK_3465486] of  Person\n\n	(_displayName \"Wajant, Harald\")\n	(DB_ID 3465486)\n	(firstname \"Harald\")\n	(initial \"H\")\n	(surname \"Wajant\"))\n\n([GK_3465487] of  Person\n\n	(_displayName \"Denessiouk, Konstantin\")\n	(DB_ID 3465487)\n	(firstname \"Konstantin\")\n	(initial \"K\")\n	(surname \"Denessiouk\"))\n\n([GK_3465488] of  LiteratureReference\n\n	(_displayName \"FLAME-1, a novel FADD-like anti-apoptotic molecule that regulates Fas/TNFR1-induced apoptosis\")\n	(author\n		[GK_3465379]\n		[GK_3465473]\n		[GK_3465464]\n		[GK_3465453]\n		[GK_3465365]\n		[GK_2318595]\n		[GK_3465441]\n		[GK_3465484]\n		[GK_3465369]\n		[GK_2562557]\n		[GK_3465451]\n		[GK_3465355])\n	(DB_ID 3465488)\n	(journal \"J. Biol. Chem.\")\n	(pages \"18542-5\")\n	(pubMedIdentifier 9228018)\n	(title \"FLAME-1, a novel FADD-like anti-apoptotic molecule that regulates Fas/TNFR1-induced apoptosis\")\n	(volume 272)\n	(year 1997))\n\n([GK_3465489] of  Person\n\n	(_displayName \"Janssen, Ottmar\")\n	(DB_ID 3465489)\n	(firstname \"Ottmar\")\n	(initial \"O\")\n	(surname \"Janssen\"))\n\n([GK_3465490] of  Person\n\n	(_displayName \"Beaudouin, Joel\")\n	(DB_ID 3465490)\n	(firstname \"Joel\")\n	(initial \"J\")\n	(surname \"Beaudouin\"))\n\n([GK_3465491] of  Person\n\n	(_displayName \"Oh, Bermseok\")\n	(DB_ID 3465491)\n	(firstname \"Bermseok\")\n	(initial \"B\")\n	(surname \"Oh\"))\n\n([GK_3465492] of  Person\n\n	(_displayName \"Micheau, Olivier\")\n	(DB_ID 3465492)\n	(firstname \"Olivier\")\n	(initial \"O\")\n	(surname \"Micheau\"))\n\n([GK_3465493] of  Person\n\n	(_displayName \"Hockenbery, David M\")\n	(DB_ID 3465493)\n	(firstname \"David M\")\n	(initial \"DM\")\n	(surname \"Hockenbery\"))\n\n([GK_3465494] of  LiteratureReference\n\n	(_displayName \"Differential responses of FLIPLong and FLIPShort-overexpressing human myeloid leukemia cells to TNF-alpha and TRAIL-initiated apoptotic signals\")\n	(author\n		[GK_3465510]\n		[GK_3465493]\n		[GK_3465424]\n		[GK_3465381]\n		[GK_3465506]\n		[GK_3465495])\n	(DB_ID 3465494)\n	(journal \"Exp. Hematol.\")\n	(pages \"1660-72\")\n	(pubMedIdentifier 18838202)\n	(title \"Differential responses of FLIPLong and FLIPShort-overexpressing human myeloid leukemia cells to TNF-alpha and TRAIL-initiated apoptotic signals\")\n	(volume 36)\n	(year 2008))\n\n([GK_3465495] of  Person\n\n	(_displayName \"Deeg, H Joachim\")\n	(DB_ID 3465495)\n	(firstname \"H Joachim\")\n	(initial \"HJ\")\n	(surname \"Deeg\"))\n\n([GK_3465496] of  Person\n\n	(_displayName \"Schmitz, I\")\n	(DB_ID 3465496)\n	(firstname \"I\")\n	(initial \"I\")\n	(surname \"Schmitz\"))\n\n([GK_3465498] of  Person\n\n	(_displayName \"Meinander, A\")\n	(DB_ID 3465498)\n	(firstname \"A\")\n	(initial \"A\")\n	(surname \"Meinander\"))\n\n([GK_3465499] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer:FADD:pro-Caspase-8:FLIP(L) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 3465499)\n	(hasComponent\n		[GK_43124]\n		[GK_57031]\n		[GK_3371356])\n	(name \"FASL:FAS Receptor Trimer:FADD:pro-Caspase-8:FLIP(L)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655991]))\n\n([GK_3465500] of  Person\n\n	(_displayName \"Asaoka, T\")\n	(DB_ID 3465500)\n	(firstname \"T\")\n	(initial \"T\")\n	(surname \"Asaoka\"))\n\n([GK_3465502] of  Person\n\n	(_displayName \"Nicholson, Donald W\")\n	(DB_ID 3465502)\n	(firstname \"Donald W\")\n	(initial \"DW\")\n	(surname \"Nicholson\"))\n\n([GK_3465503] of  LiteratureReference\n\n	(_displayName \"NF-kappaB signals induce the expression of c-FLIP\")\n	(author\n		[GK_2562544]\n		[GK_3465367]\n		[GK_3465467]\n		[GK_3465430]\n		[GK_2672175])\n	(DB_ID 3465503)\n	(journal \"Mol. Cell. Biol.\")\n	(pages \"5299-305\")\n	(pubMedIdentifier 11463813)\n	(title \"NF-kappaB signals induce the expression of c-FLIP\")\n	(volume 21)\n	(year 2001))\n\n([GK_3465505] of  Person\n\n	(_displayName \"Shu, H B\")\n	(DB_ID 3465505)\n	(firstname \"H B\")\n	(initial \"HB\")\n	(surname \"Shu\"))\n\n([GK_3465506] of  Person\n\n	(_displayName \"Abbassi, Nissa\")\n	(DB_ID 3465506)\n	(firstname \"Nissa\")\n	(initial \"N\")\n	(surname \"Abbassi\"))\n\n([GK_3465507] of  Person\n\n	(_displayName \"Hadwiger, Philipp\")\n	(DB_ID 3465507)\n	(firstname \"Philipp\")\n	(initial \"P\")\n	(surname \"Hadwiger\"))\n\n([GK_3465508] of  Person\n\n	(_displayName \"Wajant, H\")\n	(DB_ID 3465508)\n	(firstname \"H\")\n	(initial \"H\")\n	(surname \"Wajant\"))\n\n([GK_3465509] of  Person\n\n	(_displayName \"Okano, Hiroshi\")\n	(DB_ID 3465509)\n	(firstname \"Hiroshi\")\n	(initial \"H\")\n	(surname \"Okano\"))\n\n([GK_3465510] of  Person\n\n	(_displayName \"Seal, Sudeshna\")\n	(DB_ID 3465510)\n	(firstname \"Sudeshna\")\n	(initial \"S\")\n	(surname \"Seal\"))\n\n([GK_3465511] of  Person\n\n	(_displayName \"Rumpf, Jost-Julian\")\n	(DB_ID 3465511)\n	(firstname \"Jost-Julian\")\n	(initial \"JJ\")\n	(surname \"Rumpf\"))\n\n([GK_3465518] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-17\")\n	(author [GK_429259])\n	(dateTime \"2013-05-17 01:54:44\")\n	(DB_ID 3465518))\n\n([GK_3465522] of  LiteratureReference\n\n	(_displayName \"Ebselen sensitizes glioblastoma cells to Tumor Necrosis Factor (TNFalpha)-induced apoptosis through two distinct pathways involving NF-kappaB downregulation and Fas-mediated formation of death inducing signaling complex\")\n	(author\n		[GK_3465527]\n		[GK_3465523]\n		[GK_3465524]\n		[GK_3465526]\n		[GK_3465528])\n	(DB_ID 3465522)\n	(journal \"Int. J. Cancer\")\n	(pages \"2204-12\")\n	(pubMedIdentifier 18709644)\n	(title \"Ebselen sensitizes glioblastoma cells to Tumor Necrosis Factor (TNFalpha)-induced apoptosis through two distinct pathways involving NF-kappaB downregulation and Fas-mediated formation of death inducing signaling complex\")\n	(volume 123)\n	(year 2008))\n\n([GK_3465523] of  Person\n\n	(_displayName \"Tewari, Richa\")\n	(DB_ID 3465523)\n	(firstname \"Richa\")\n	(initial \"R\")\n	(surname \"Tewari\"))\n\n([GK_3465524] of  Person\n\n	(_displayName \"Sk, Ugir Hossain\")\n	(DB_ID 3465524)\n	(firstname \"Ugir Hossain\")\n	(initial \"UH\")\n	(surname \"Sk\"))\n\n([GK_3465526] of  Person\n\n	(_displayName \"Joseph, Christy\")\n	(DB_ID 3465526)\n	(firstname \"Christy\")\n	(initial \"C\")\n	(surname \"Joseph\"))\n\n([GK_3465527] of  Person\n\n	(_displayName \"Sharma, Vivek\")\n	(DB_ID 3465527)\n	(firstname \"Vivek\")\n	(initial \"V\")\n	(surname \"Sharma\"))\n\n([GK_3465528] of  Person\n\n	(_displayName \"Sen, Ellora\")\n	(DB_ID 3465528)\n	(firstname \"Ellora\")\n	(initial \"E\")\n	(surname \"Sen\"))\n\n([GK_3465529] of  Person\n\n	(_displayName \"Hughes, Michelle\")\n	(DB_ID 3465529)\n	(firstname \"Michelle\")\n	(initial \"M\")\n	(surname \"Hughes\"))\n\n([GK_3465531] of  LiteratureReference\n\n	(_displayName \"Fas-associated death domain protein and caspase-8 are not recruited to the tumor necrosis factor receptor 1 signaling complex during tumor necrosis factor-induced apoptosis\")\n	(author\n		[GK_2984182]\n		[GK_3465529]\n		[GK_2562548]\n		[GK_2984178])\n	(DB_ID 3465531)\n	(journal \"J. Biol. Chem.\")\n	(pages \"25534-41\")\n	(pubMedIdentifier 12721308)\n	(title \"Fas-associated death domain protein and caspase-8 are not recruited to the tumor necrosis factor receptor 1 signaling complex during tumor necrosis factor-induced apoptosis\")\n	(volume 278)\n	(year 2003))\n\n([GK_3465532] of  Complex\n\n	(_displayName \"[Hsa] TRADD:TRAF2:RIP1:FADD:procaspase-8:FLIP(L) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 3465532)\n	(goCellularComponent [GK_161217])\n	(hasComponent\n		[GK_140977]\n		[GK_57031]\n		[GK_3371356])\n	(literatureReference [GK_3465412])\n	(name \"TRADD:TRAF2:RIP1:FADD:procaspase-8:FLIP(L)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_3655914]))\n\n([GK_3465558] of  Person\n\n	(_displayName \"Brennan, F M\")\n	(DB_ID 3465558)\n	(firstname \"F M\")\n	(initial \"FM\")\n	(surname \"Brennan\"))\n\n([GK_3465559] of  LiteratureReference\n\n	(_displayName \"Release of tumour necrosis factor alpha (TNFalpha) by TNFalpha cleaving enzyme (TACE) in response to septic stimuli in vitro\")\n	(author\n		[GK_3465561]\n		[GK_3465558])\n	(DB_ID 3465559)\n	(journal \"Br J Anaesth\")\n	(pages \"222-8\")\n	(pubMedIdentifier 15556967)\n	(title \"Release of tumour necrosis factor alpha (TNFalpha) by TNFalpha cleaving enzyme (TACE) in response to septic stimuli in vitro\")\n	(volume 94)\n	(year 2005))\n\n([GK_3465561] of  Person\n\n	(_displayName \"Robertshaw, H J\")\n	(DB_ID 3465561)\n	(firstname \"H J\")\n	(initial \"HJ\")\n	(surname \"Robertshaw\"))\n\n([GK_3465562] of  Summation\n\n	(_displayName \"Soluble form of TNF-alpha is cleaved from membrane-anchored ...\")\n	(DB_ID 3465562)\n	(literatureReference [GK_140945])\n	(text \"Soluble form of TNF-alpha is cleaved from membrane-anchored TNF-alpha and bind to TNF receptors (TNF-R1 and -R2).\"))\n\n([GK_3465563] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-17\")\n	(author [GK_429259])\n	(dateTime \"2013-05-17 04:50:56\")\n	(DB_ID 3465563))\n\n([GK_3465564] of  Person\n\n	(_displayName \"Dumont, Serge\")\n	(DB_ID 3465564)\n	(firstname \"Serge\")\n	(initial \"S\")\n	(surname \"Dumont\"))\n\n([GK_3465565] of  Person\n\n	(_displayName \"Peluso, Jean\")\n	(DB_ID 3465565)\n	(firstname \"Jean\")\n	(initial \"J\")\n	(surname \"Peluso\"))\n\n([GK_3465566] of  Person\n\n	(_displayName \"Moreira-Tabaka, Helena\")\n	(DB_ID 3465566)\n	(firstname \"Helena\")\n	(initial \"H\")\n	(surname \"Moreira-Tabaka\"))\n\n([GK_3465567] of  Person\n\n	(_displayName \"Kessler, Pascal\")\n	(DB_ID 3465567)\n	(firstname \"Pascal\")\n	(initial \"P\")\n	(surname \"Kessler\"))\n\n([GK_3465568] of  LiteratureReference\n\n	(_displayName \"Unlike for human monocytes after LPS activation, release of TNF-? by THP-1 cells is produced by a TACE catalytically different from constitutive TACE\")\n	(author\n		[GK_3465566]\n		[GK_3465565]\n		[GK_3465573]\n		[GK_3465570]\n		[GK_3465567]\n		[GK_3465571]\n		[GK_3465564]\n		[GK_3465574])\n	(DB_ID 3465568)\n	(journal \"PLoS ONE\")\n	(pages \"e34184\")\n	(pubMedIdentifier 22479555)\n	(title \"Unlike for human monocytes after LPS activation, release of TNF-? by THP-1 cells is produced by a TACE catalytically different from constitutive TACE\")\n	(volume 7)\n	(year 2012))\n\n([GK_3465570] of  Person\n\n	(_displayName \"Hentsch, Didier\")\n	(DB_ID 3465570)\n	(firstname \"Didier\")\n	(initial \"D\")\n	(surname \"Hentsch\"))\n\n([GK_3465571] of  Person\n\n	(_displayName \"Reimund, Jean-Marie\")\n	(DB_ID 3465571)\n	(firstname \"Jean-Marie\")\n	(initial \"JM\")\n	(surname \"Reimund\"))\n\n([GK_3465572] of  Summation\n\n	(_displayName \"TNF-alpha is initially synthesized as a 26kDa transmembrane ...\")\n	(DB_ID 3465572)\n	(literatureReference\n		[GK_3371352]\n		[GK_3465559]\n		[GK_3465568])\n	(text \"TNF-alpha is initially synthesized as a 26kDa transmembrane precursor (pro-TNF-alpha), which is processed by proteolytic cleavage known as ectodomain shedding (Tang P et al. 1996). TNF-alpha-converting enzyme (TACE or ADAM17) has been shown to mediate the cleavage of TNF-alpha generating the soluble 17kDa mature form (Robertshaw HJ & Brennan FM 2005). Inhibition of TACE activity resulted in an accumulation of unprocessed TNF-alpha on the cell surface of human monocytic cells (THP1) (Tabaka HN et al. 2012). Both transmembrane and secreted forms of TNF-alpha are biologically active and may mediate distinct activities.\"))\n\n([GK_3465573] of  Person\n\n	(_displayName \"Vonesch, Jean-Luc\")\n	(DB_ID 3465573)\n	(firstname \"Jean-Luc\")\n	(initial \"JL\")\n	(surname \"Vonesch\"))\n\n([GK_3465574] of  Person\n\n	(_displayName \"Muller, Christian D\")\n	(DB_ID 3465574)\n	(firstname \"Christian D\")\n	(initial \"CD\")\n	(surname \"Muller\"))\n\n([GK_3465576] of  Summation\n\n	(_displayName \"Transmembrane TNF-alpha (tmTNF-alpha), a precursor of the so...\")\n	(DB_ID 3465576)\n	(text \"Transmembrane TNF-alpha (tmTNF-alpha), a precursor of the soluble form of TNF-alpha, is expressed in macrophages and lymphocytes as well as other cell types, tmTNF-alpha functions as a bipolar molecule that can transmit signals both as a ligand and as a receptor in a cell-to-cell contact fashion. Transmembrane TNF-alpha on TNF-alpha-producing cells binds to TNF-R1 and -R2, and transmits signals to the target cells as a ligand, whereas transmembrane TNF-alpha also acts as a receptor that transmits outside-to-inside (reverse) signals back to the cells after binding to its native receptors.\"))\n\n([GK_3508822] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-17\")\n	(author [GK_429259])\n	(dateTime \"2013-05-17 21:57:50\")\n	(DB_ID 3508822))\n\n([GK_354] of  GO_CellularComponent\n\n	(_displayName \"intracellular\")\n	(accession \"0005622\")\n	(DB_ID 354)\n	(definition \"The living contents of a cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm.\")\n	(instanceOf [GK_258147])\n	(name \"intracellular\")\n	(referenceDatabase [GK_1]))\n\n([GK_355446] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P68431 HIST1H3A recommendedName: Histone H3.1  alternativeName: Histone H3/a  alternativeName: Histone H3/b  alternativeName: Histone H3/c  alternativeName: Histone H3/d  alternativeName: Histone H3/f  alternativeName: Histone H3/h  alternativeName: Histone H3/i  alternativeName: Histone H3/j  alternativeName: Histone H3/k  alternativeName: Histone H3/l \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.DEVELOPMENTAL STAGE Expressed during S phase, then expression strongly decreases as cell division slows down during the process of differentiation.PTM Acetylation is generally linked to gene activation. Acetylation on Lys-10 (H3K9ac) impairs methylation at Arg-9 (H3R8me2s). Acetylation on Lys-19 (H3K18ac) and Lys-24 (H3K24ac) favors methylation at Arg-18 (H3R17me).PTM Citrullination at Arg-9 (H3R8ci) and/or Arg-18 (H3R17ci) by PADI4 impairs methylation and represses transcription.PTM Asymmetric dimethylation at Arg-18 (H3R17me2a) by CARM1 is linked to gene activation. Symmetric dimethylation at Arg-9 (H3R8me2s) by PRMT5 is linked to gene repression. Asymmetric dimethylation at Arg-3 (H3R2me2a) by PRMT6 is linked to gene repression and is mutually exclusive with H3 Lys-5 methylation (H3K4me2 and H3K4me3). H3R2me2a is present at the 3\' of genes regardless of their transcription state and is enriched on inactive promoters, while it is absent on active promoters.PTM Methylation at Lys-5 (H3K4me), Lys-37 (H3K36me) and Lys-80 (H3K79me) are linked to gene activation. Methylation at Lys-5 (H3K4me) facilitates subsequent acetylation of H3 and H4. Methylation at Lys-80 (H3K79me) is associated with DNA double-strand break (DSB) responses and is a specific target for TP53BP1. Methylation at Lys-10 (H3K9me) and Lys-28 (H3K27me) are linked to gene repression. Methylation at Lys-10 (H3K9me) is a specific target for HP1 proteins (CBX1, CBX3 and CBX5) and prevents subsequent phosphorylation at Ser-11 (H3S10ph) and acetylation of H3 and H4. Methylation at Lys-5 (H3K4me) and Lys-80 (H3K79me) require preliminary monoubiquitination of H2B at \'Lys-120\'. Methylation at Lys-10 (H3K9me) and Lys-28 (H3K27me) are enriched in inactive X chromosome chromatin.MISCELLANEOUS This histone is only present in mammals and is enriched in acetylation of Lys-15 and dimethylation of Lys-10 (H3K9me2).SIMILARITY Belongs to the histone H3 family.\")\n	(DB_ID 355446)\n	(description \"recommendedName: Histone H3.1  alternativeName: Histone H3/a  alternativeName: Histone H3/b  alternativeName: Histone H3/c  alternativeName: Histone H3/d  alternativeName: Histone H3/f  alternativeName: Histone H3/h  alternativeName: Histone H3/i  alternativeName: Histone H3/j  alternativeName: Histone H3/k  alternativeName: Histone H3/l \")\n	(geneName\n		\"HIST1H3A\"\n		\"H3FA\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3B\"\n		\"H3FL\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3C\"\n		\"H3FC\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3D\"\n		\"H3FB\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3E\"\n		\"H3FD\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3F\"\n		\"H3FI\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3G\"\n		\"H3FH\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3H\"\n		\"H3FK\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3I\"\n		\"H3FF\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H3J\"\n		\"H3FJ\")\n	(identifier \"P68431\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H3A\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H31_HUMAN\"\n		\"A0PJT7\"\n		\"A5PLR1\"\n		\"P02295\"\n		\"P02296\"\n		\"P16106\"\n		\"Q6ISV8\"\n		\"Q6NWP8\"\n		\"Q6NWP9\"\n		\"Q6NXU4\"\n		\"Q71DJ3\"\n		\"Q93081\")\n	(sequenceLength 136)\n	(species [GK_48887]))\n\n([GK_355447] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q71DI3 HIST2H3A recommendedName: Histone H3.2  alternativeName: Histone H3/m  alternativeName: Histone H3/o \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA. During nucleosome assembly the chaperone ASF1A interacts with the histone H3-H4 heterodimer.DEVELOPMENTAL STAGE Expressed during S phase, then expression strongly decreases as cell division slows down during the process of differentiation.PTM Acetylation is generally linked to gene activation. Acetylation on Lys-10 (H3K9ac) impairs methylation at Arg-9 (H3R8me2s). Acetylation on Lys-19 (H3K18ac) and Lys-24 (H3K24ac) favors methylation at Arg-18 (H3R17me).PTM Citrullination at Arg-9 (H3R8ci) and/or Arg-18 (H3R17ci) by PADI4 impairs methylation and represses transcription.PTM Asymmetric dimethylation at Arg-18 (H3R17me2a) by CARM1 is linked to gene activation. Symmetric dimethylation at Arg-9 (H3R8me2s) by PRMT5 is linked to gene repression. Asymmetric dimethylation at Arg-3 (H3R2me2a) by PRMT6 is linked to gene repression and is mutually exclusive with H3 Lys-5 methylation (H3K4me2 and H3K4me3). H3R2me2a is present at the 3\' of genes regardless of their transcription state and is enriched on inactive promoters, while it is absent on active promoters.PTM Methylation at Lys-5 (H3K4me), Lys-37 (H3K36me) and Lys-80 (H3K79me) are linked to gene activation. Methylation at Lys-5 (H3K4me) facilitates subsequent acetylation of H3 and H4. Methylation at Lys-80 (H3K79me) is associated with DNA double-strand break (DSB) responses and is a specific target for TP53BP1. Methylation at Lys-10 (H3K9me) and Lys-28 (H3K27me) are linked to gene repression. Methylation at Lys-10 (H3K9me) is a specific target for HP1 proteins (CBX1, CBX3 and CBX5) and prevents subsequent phosphorylation at Ser-11 (H3S10ph) and acetylation of H3 and H4. Methylation at Lys-5 (H3K4me) and Lys-80 (H3K79me) require preliminary monoubiquitination of H2B at \'Lys-120\'. Methylation at Lys-10 (H3K9me) and Lys-28 (H3K27me) are enriched in inactive X chromosome chromatin.SIMILARITY Belongs to the histone H3 family.\")\n	(DB_ID 355447)\n	(description \"recommendedName: Histone H3.2  alternativeName: Histone H3/m  alternativeName: Histone H3/o \")\n	(geneName\n		\"HIST2H3A\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST2H3C\"\n		\"H3F2\"\n		\"H3FM\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST2H3D\")\n	(identifier \"Q71DI3\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Ubl conjugation\")\n	(name \"HIST2H3A\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H32_HUMAN\"\n		\"A2BDF6\"\n		\"A6NFS4\"\n		\"Q6B053\")\n	(sequenceLength 136)\n	(species [GK_48887]))\n\n([GK_355448] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P84243 H3F3A recommendedName: Histone H3.3 \")\n	(checksum \"1\")\n	(comment \"DEVELOPMENTAL STAGE Expressed throughout the cell cycle independently of DNA synthesis.PTM Acetylation is generally linked to gene activation. Acetylation on Lys-10 (H3K9ac) impairs methylation at Arg-9 (H3R8me2s). Acetylation on Lys-19 (H3K18ac) and Lys-24 (H3K24ac) favors methylation at Arg-18 (H3R17me).PTM Citrullination at Arg-9 (H3R8ci) and/or Arg-18 (H3R17ci) by PADI4 impairs methylation and represses transcription.PTM Asymmetric dimethylation at Arg-18 (H3R17me2a) by CARM1 is linked to gene activation. Symmetric dimethylation at Arg-9 (H3R8me2s) by PRMT5 is linked to gene repression. Asymmetric dimethylation at Arg-3 (H3R2me2a) by PRMT6 is linked to gene repression and is mutually exclusive with H3 Lys-5 methylation (H3K4me2 and H3K4me3). H3R2me2a is present at the 3\' of genes regardless of their transcription state and is enriched on inactive promoters, while it is absent on active promoters.PTM Specifically enriched in modifications associated with active chromatin such as methylation at Lys-5 (H3K4me), Lys-37 and Lys-80. Methylation at Lys-5 (H3K4me) facilitates subsequent acetylation of H3 and H4. Methylation at Lys-80 (H3K79me) is associated with DNA double-strand break (DSB) responses and is a specific target for TP53BP1. Methylation at Lys-10 (H3K9me) and Lys-28 (H3K27me), which are linked to gene repression, are underrepresented. Methylation at Lys-10 (H3K9me) is a specific target for HP1 proteins (CBX1, CBX3 and CBX5) and prevents subsequent phosphorylation at Ser-11 (H3S10ph) and acetylation of H3 and H4. Methylation at Lys-5 (H3K4me) and Lys-80 (H3K79me) require preliminary monoubiquitination of H2B at \'Lys-120\'. Methylation at Lys-10 (H3K9me) and Lys-28 (H3K27me) are enriched in inactive X chromosome chromatin.SIMILARITY Belongs to the histone H3 family.\")\n	(DB_ID 355448)\n	(description \"recommendedName: Histone H3.3 \")\n	(geneName\n		\"H3F3A\"\n		\"H3.3A\"\n		\"H3F3\"\n		\"PP781\"\n		\"</gene>\"\n		\"<gene>\"\n		\"H3F3B\"\n		\"H3.3B\")\n	(identifier \"P84243\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"H3F3A\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H33_HUMAN\"\n		\"P06351\"\n		\"P33155\"\n		\"Q5VV55\"\n		\"Q5VV56\"\n		\"Q66I33\"\n		\"Q9V3W4\")\n	(sequenceLength 136)\n	(species [GK_48887]))\n\n([GK_355449] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P62805 HIST1H4A recommendedName: Histone H4 \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Acetylation at Lys-6 (H4K5ac), Lys-9 (H4K8ac), Lys-13 (H4K12ac) and Lys-17 (H4K16ac) occurs in coding regions of the genome but not in heterochromatin.PTM Citrullination at Arg-4 (H4R3ci) by PADI4 impairs methylation.PTM Monomethylation and asymmetric dimethylation at Arg-4 (H4R3me1 and H4R3me2a, respectively) by PRMT1 favors acetylation at Lys-9 (H4K8ac) and Lys-13 (H4K12ac). Demethylation is performed by JMJD6. Symmetric dimethylation on Arg-4 (H4R3me2s) by the PRDM1/PRMT5 complex may play a crucial role in the germ-cell lineage.SIMILARITY Belongs to the histone H4 family.\")\n	(DB_ID 355449)\n	(description \"recommendedName: Histone H4 \")\n	(geneName\n		\"HIST1H4A\"\n		\"H4/A\"\n		\"H4FA\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4B\"\n		\"H4/I\"\n		\"H4FI\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4C\"\n		\"H4/G\"\n		\"H4FG\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4D\"\n		\"H4/B\"\n		\"H4FB\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4E\"\n		\"H4/J\"\n		\"H4FJ\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4F\"\n		\"H4/C\"\n		\"H4FC\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4H\"\n		\"H4/H\"\n		\"H4FH\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4I\"\n		\"H4/M\"\n		\"H4FM\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4J\"\n		\"H4/E\"\n		\"H4FE\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4K\"\n		\"H4/D\"\n		\"H4FD\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H4L\"\n		\"H4/K\"\n		\"H4FK\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST2H4A\"\n		\"H4/N\"\n		\"H4F2\"\n		\"H4FN\"\n		\"HIST2H4\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST2H4B\"\n		\"H4/O\"\n		\"H4FO\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST4H4\")\n	(identifier \"P62805\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Ubl conjugation\")\n	(name \"HIST1H4A\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H4_HUMAN\"\n		\"A2VCL0\"\n		\"P02304\"\n		\"P02305\"\n		\"Q6DRA9\"\n		\"Q6FGB8\"\n		\"Q6NWP7\")\n	(sequenceLength 103)\n	(species [GK_48887]))\n\n([GK_356] of  GO_CellularComponent\n\n	(_displayName \"cell\")\n	(accession \"0005623\")\n	(DB_ID 356)\n	(definition \"The basic structural and functional unit of all organisms. Includes the plasma membrane and any external encapsulating structures such as the cell wall and cell envelope.\")\n	(instanceOf [GK_357])\n	(name \"cell\")\n	(referenceDatabase [GK_1]))\n\n([GK_357] of  GO_CellularComponent\n\n	(_displayName \"cellular_component\")\n	(accession \"0005575\")\n	(DB_ID 357)\n	(definition \"The part of a cell or its extracellular environment in which a gene product is located. A gene product may be located in one or more parts of a cell and its location may be as specific as a particular macromolecular complex, that is, a stable, persistent association of macromolecules that function together.\")\n	(name \"cellular_component\")\n	(referenceDatabase [GK_1]))\n\n([GK_358192] of  Person\n\n	(_displayName \"Robinson, CV\")\n	(DB_ID 358192)\n	(initial \"CV\")\n	(surname \"Robinson\"))\n\n([GK_358238] of  StableIdentifier\n\n	(_displayName \"REACT_1159.2\")\n	(DB_ID 358238)\n	(identifier \"REACT_1159\")\n	(identifierVersion \"4\"))\n\n([GK_358269] of  StableIdentifier\n\n	(_displayName \"REACT_1766.1\")\n	(DB_ID 358269)\n	(identifier \"REACT_1766\")\n	(identifierVersion \"3\"))\n\n([GK_358282] of  StableIdentifier\n\n	(_displayName \"REACT_1143.2\")\n	(DB_ID 358282)\n	(identifier \"REACT_1143\")\n	(identifierVersion \"4\"))\n\n([GK_358384] of  StableIdentifier\n\n	(_displayName \"REACT_832.2\")\n	(DB_ID 358384)\n	(identifier \"REACT_832\")\n	(identifierVersion \"4\"))\n\n([GK_358527] of  StableIdentifier\n\n	(_displayName \"REACT_2170.2\")\n	(DB_ID 358527)\n	(identifier \"REACT_2170\")\n	(identifierVersion \"4\"))\n\n([GK_358712] of  StableIdentifier\n\n	(_displayName \"REACT_1856.1\")\n	(DB_ID 358712)\n	(identifier \"REACT_1856\")\n	(identifierVersion \"3\"))\n\n([GK_358827] of  StableIdentifier\n\n	(_displayName \"REACT_402.1\")\n	(DB_ID 358827)\n	(identifier \"REACT_402\")\n	(identifierVersion \"3\"))\n\n([GK_359034] of  StableIdentifier\n\n	(_displayName \"REACT_1543.2\")\n	(DB_ID 359034)\n	(identifier \"REACT_1543\")\n	(identifierVersion \"4\"))\n\n([GK_359070] of  StableIdentifier\n\n	(_displayName \"REACT_279.2\")\n	(DB_ID 359070)\n	(identifier \"REACT_279\")\n	(identifierVersion \"4\"))\n\n([GK_359435] of  StableIdentifier\n\n	(_displayName \"REACT_1418.2\")\n	(DB_ID 359435)\n	(identifier \"REACT_1418\")\n	(identifierVersion \"4\"))\n\n([GK_359726] of  StableIdentifier\n\n	(_displayName \"REACT_606.2\")\n	(DB_ID 359726)\n	(identifier \"REACT_606\")\n	(identifierVersion \"4\"))\n\n([GK_3597630] of  InstanceEdit\n\n	(_displayName \"Pop, Cristina, Salvesen, Guy S., 2013-05-22\")\n	(author\n		[GK_1267783]\n		[GK_141140])\n	(dateTime \"2013-05-22 00:38:25\")\n	(DB_ID 3597630))\n\n([GK_3597631] of  Affiliation\n\n	(_displayName \"Sanford-Burnham Medical Research Institute\")\n	(address \"La Jolla, California 92037\")\n	(DB_ID 3597631)\n	(name \"Sanford-Burnham Medical Research Institute\"))\n\n([GK_359831] of  StableIdentifier\n\n	(_displayName \"REACT_663.2\")\n	(DB_ID 359831)\n	(identifier \"REACT_663\")\n	(identifierVersion \"4\"))\n\n([GK_360111] of  StableIdentifier\n\n	(_displayName \"REACT_1059.1\")\n	(DB_ID 360111)\n	(identifier \"REACT_1059\")\n	(identifierVersion \"5\"))\n\n([GK_360371] of  StableIdentifier\n\n	(_displayName \"REACT_737.2\")\n	(DB_ID 360371)\n	(identifier \"REACT_737\")\n	(identifierVersion \"4\"))\n\n([GK_3604387] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-23\")\n	(author [GK_429259])\n	(dateTime \"2013-05-23 00:12:17\")\n	(DB_ID 3604387))\n\n([GK_360533] of  StableIdentifier\n\n	(_displayName \"REACT_2096.2\")\n	(DB_ID 360533)\n	(identifier \"REACT_2096\")\n	(identifierVersion \"4\"))\n\n([GK_360537] of  StableIdentifier\n\n	(_displayName \"REACT_1038.2\")\n	(DB_ID 360537)\n	(identifier \"REACT_1038\")\n	(identifierVersion \"4\"))\n\n([GK_360947] of  StableIdentifier\n\n	(_displayName \"REACT_900.1\")\n	(DB_ID 360947)\n	(identifier \"REACT_900\")\n	(identifierVersion \"3\"))\n\n([GK_361043] of  StableIdentifier\n\n	(_displayName \"REACT_961.2\")\n	(DB_ID 361043)\n	(identifier \"REACT_961\")\n	(identifierVersion \"4\"))\n\n([GK_361068] of  StableIdentifier\n\n	(_displayName \"REACT_2175.2\")\n	(DB_ID 361068)\n	(identifier \"REACT_2175\")\n	(identifierVersion \"4\"))\n\n([GK_361140] of  StableIdentifier\n\n	(_displayName \"REACT_1619.1\")\n	(DB_ID 361140)\n	(identifier \"REACT_1619\")\n	(identifierVersion \"4\"))\n\n([GK_361228] of  StableIdentifier\n\n	(_displayName \"REACT_1432.1\")\n	(DB_ID 361228)\n	(identifier \"REACT_1432\")\n	(identifierVersion \"5\"))\n\n([GK_361795] of  StableIdentifier\n\n	(_displayName \"REACT_1426.2\")\n	(DB_ID 361795)\n	(identifier \"REACT_1426\")\n	(identifierVersion \"4\"))\n\n([GK_361815] of  StableIdentifier\n\n	(_displayName \"REACT_1360.2\")\n	(DB_ID 361815)\n	(identifier \"REACT_1360\")\n	(identifierVersion \"4\"))\n\n([GK_362003] of  StableIdentifier\n\n	(_displayName \"REACT_2191.3\")\n	(DB_ID 362003)\n	(identifier \"REACT_2191\")\n	(identifierVersion \"5\"))\n\n([GK_362011] of  StableIdentifier\n\n	(_displayName \"REACT_1503.2\")\n	(DB_ID 362011)\n	(identifier \"REACT_1503\")\n	(identifierVersion \"5\"))\n\n([GK_362019] of  StableIdentifier\n\n	(_displayName \"REACT_1721.2\")\n	(DB_ID 362019)\n	(identifier \"REACT_1721\")\n	(identifierVersion \"4\"))\n\n([GK_362216] of  StableIdentifier\n\n	(_displayName \"REACT_4646.1\")\n	(DB_ID 362216)\n	(identifier \"REACT_4646\")\n	(identifierVersion \"3\"))\n\n([GK_362769] of  StableIdentifier\n\n	(_displayName \"REACT_2430.1\")\n	(DB_ID 362769)\n	(identifier \"REACT_2430\")\n	(identifierVersion \"2\"))\n\n([GK_362980] of  StableIdentifier\n\n	(_displayName \"REACT_4998.1\")\n	(DB_ID 362980)\n	(identifier \"REACT_4998\")\n	(identifierVersion \"2\"))\n\n([GK_3632865] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-28\")\n	(author [GK_429259])\n	(dateTime \"2013-05-28 03:55:32\")\n	(DB_ID 3632865))\n\n([GK_3632873] of  InstanceEdit\n\n	(_displayName \"Shamovsky, V, 2013-05-28\")\n	(author [GK_429259])\n	(dateTime \"2013-05-28 07:40:49\")\n	(DB_ID 3632873))\n\n([GK_363728] of  StableIdentifier\n\n	(_displayName \"REACT_5875.1\")\n	(DB_ID 363728)\n	(identifier \"REACT_5875\")\n	(identifierVersion \"2\"))\n\n([GK_363774] of  StableIdentifier\n\n	(_displayName \"REACT_3424.1\")\n	(DB_ID 363774)\n	(identifier \"REACT_3424\")\n	(identifierVersion \"4\"))\n\n([GK_363887] of  StableIdentifier\n\n	(_displayName \"REACT_5668.1\")\n	(DB_ID 363887)\n	(identifier \"REACT_5668\")\n	(identifierVersion \"2\"))\n\n([GK_363923] of  StableIdentifier\n\n	(_displayName \"REACT_9539.1\")\n	(DB_ID 363923)\n	(identifier \"REACT_9539\")\n	(identifierVersion \"2\"))\n\n([GK_364006] of  StableIdentifier\n\n	(_displayName \"REACT_5339.1\")\n	(DB_ID 364006)\n	(identifier \"REACT_5339\")\n	(identifierVersion \"2\"))\n\n([GK_364329] of  StableIdentifier\n\n	(_displayName \"REACT_4420.1\")\n	(DB_ID 364329)\n	(identifier \"REACT_4420\")\n	(identifierVersion \"2\"))\n\n([GK_364408] of  StableIdentifier\n\n	(_displayName \"REACT_7501.1\")\n	(DB_ID 364408)\n	(identifier \"REACT_7501\")\n	(identifierVersion \"2\"))\n\n([GK_364838] of  StableIdentifier\n\n	(_displayName \"REACT_3983.1\")\n	(DB_ID 364838)\n	(identifier \"REACT_3983\")\n	(identifierVersion \"2\"))\n\n([GK_365034] of  StableIdentifier\n\n	(_displayName \"REACT_5415.1\")\n	(DB_ID 365034)\n	(identifier \"REACT_5415\")\n	(identifierVersion \"2\"))\n\n([GK_3654232] of  StableIdentifier\n\n	(_displayName \"REACT_163688.2\")\n	(DB_ID 3654232)\n	(identifier \"REACT_163688\")\n	(identifierVersion \"2\"))\n\n([GK_3654297] of  StableIdentifier\n\n	(_displayName \"REACT_163753.2\")\n	(DB_ID 3654297)\n	(identifier \"REACT_163753\")\n	(identifierVersion \"2\"))\n\n([GK_3654338] of  StableIdentifier\n\n	(_displayName \"REACT_163794.2\")\n	(DB_ID 3654338)\n	(identifier \"REACT_163794\")\n	(identifierVersion \"2\"))\n\n([GK_3654456] of  StableIdentifier\n\n	(_displayName \"REACT_163912.2\")\n	(DB_ID 3654456)\n	(identifier \"REACT_163912\")\n	(identifierVersion \"2\"))\n\n([GK_3654502] of  StableIdentifier\n\n	(_displayName \"REACT_163958.2\")\n	(DB_ID 3654502)\n	(identifier \"REACT_163958\")\n	(identifierVersion \"2\"))\n\n([GK_3654514] of  StableIdentifier\n\n	(_displayName \"REACT_163970.2\")\n	(DB_ID 3654514)\n	(identifier \"REACT_163970\")\n	(identifierVersion \"2\"))\n\n([GK_3654525] of  StableIdentifier\n\n	(_displayName \"REACT_163981.2\")\n	(DB_ID 3654525)\n	(identifier \"REACT_163981\")\n	(identifierVersion \"2\"))\n\n([GK_3654555] of  StableIdentifier\n\n	(_displayName \"REACT_164011.2\")\n	(DB_ID 3654555)\n	(identifier \"REACT_164011\")\n	(identifierVersion \"2\"))\n\n([GK_3654708] of  StableIdentifier\n\n	(_displayName \"REACT_164164.1\")\n	(DB_ID 3654708)\n	(identifier \"REACT_164164\")\n	(identifierVersion \"1\"))\n\n([GK_365494] of  StableIdentifier\n\n	(_displayName \"REACT_5404.1\")\n	(DB_ID 365494)\n	(identifier \"REACT_5404\")\n	(identifierVersion \"2\"))\n\n([GK_3654957] of  StableIdentifier\n\n	(_displayName \"REACT_164413.1\")\n	(DB_ID 3654957)\n	(identifier \"REACT_164413\")\n	(identifierVersion \"1\"))\n\n([GK_3655002] of  StableIdentifier\n\n	(_displayName \"REACT_164458.1\")\n	(DB_ID 3655002)\n	(identifier \"REACT_164458\")\n	(identifierVersion \"1\"))\n\n([GK_3655101] of  StableIdentifier\n\n	(_displayName \"REACT_164557.1\")\n	(DB_ID 3655101)\n	(identifier \"REACT_164557\")\n	(identifierVersion \"1\"))\n\n([GK_3655182] of  StableIdentifier\n\n	(_displayName \"REACT_164638.1\")\n	(DB_ID 3655182)\n	(identifier \"REACT_164638\")\n	(identifierVersion \"1\"))\n\n([GK_365522] of  StableIdentifier\n\n	(_displayName \"REACT_3459.1\")\n	(DB_ID 365522)\n	(identifier \"REACT_3459\")\n	(identifierVersion \"1\"))\n\n([GK_3655311] of  StableIdentifier\n\n	(_displayName \"REACT_164767.1\")\n	(DB_ID 3655311)\n	(identifier \"REACT_164767\")\n	(identifierVersion \"1\"))\n\n([GK_3655419] of  StableIdentifier\n\n	(_displayName \"REACT_164875.1\")\n	(DB_ID 3655419)\n	(identifier \"REACT_164875\")\n	(identifierVersion \"1\"))\n\n([GK_365546] of  StableIdentifier\n\n	(_displayName \"REACT_5449.2\")\n	(DB_ID 365546)\n	(identifier \"REACT_5449\")\n	(identifierVersion \"2\"))\n\n([GK_3655628] of  StableIdentifier\n\n	(_displayName \"REACT_165084.1\")\n	(DB_ID 3655628)\n	(identifier \"REACT_165084\")\n	(identifierVersion \"1\"))\n\n([GK_3655740] of  StableIdentifier\n\n	(_displayName \"REACT_165196.1\")\n	(DB_ID 3655740)\n	(identifier \"REACT_165196\")\n	(identifierVersion \"1\"))\n\n([GK_3655775] of  StableIdentifier\n\n	(_displayName \"REACT_165231.1\")\n	(DB_ID 3655775)\n	(identifier \"REACT_165231\")\n	(identifierVersion \"1\"))\n\n([GK_3655784] of  StableIdentifier\n\n	(_displayName \"REACT_165240.1\")\n	(DB_ID 3655784)\n	(identifier \"REACT_165240\")\n	(identifierVersion \"1\"))\n\n([GK_3655914] of  StableIdentifier\n\n	(_displayName \"REACT_165370.1\")\n	(DB_ID 3655914)\n	(identifier \"REACT_165370\")\n	(identifierVersion \"1\"))\n\n([GK_3655921] of  StableIdentifier\n\n	(_displayName \"REACT_165377.1\")\n	(DB_ID 3655921)\n	(identifier \"REACT_165377\")\n	(identifierVersion \"1\"))\n\n([GK_3655925] of  StableIdentifier\n\n	(_displayName \"REACT_165381.1\")\n	(DB_ID 3655925)\n	(identifier \"REACT_165381\")\n	(identifierVersion \"1\"))\n\n([GK_3655991] of  StableIdentifier\n\n	(_displayName \"REACT_165447.1\")\n	(DB_ID 3655991)\n	(identifier \"REACT_165447\")\n	(identifierVersion \"1\"))\n\n([GK_3656025] of  StableIdentifier\n\n	(_displayName \"REACT_165481.1\")\n	(DB_ID 3656025)\n	(identifier \"REACT_165481\")\n	(identifierVersion \"1\"))\n\n([GK_366061] of  StableIdentifier\n\n	(_displayName \"REACT_4185.1\")\n	(DB_ID 366061)\n	(identifier \"REACT_4185\")\n	(identifierVersion \"1\"))\n\n([GK_366444] of  StableIdentifier\n\n	(_displayName \"REACT_4656.1\")\n	(DB_ID 366444)\n	(identifier \"REACT_4656\")\n	(identifierVersion \"1\"))\n\n([GK_366491] of  StableIdentifier\n\n	(_displayName \"REACT_4500.1\")\n	(DB_ID 366491)\n	(identifier \"REACT_4500\")\n	(identifierVersion \"2\"))\n\n([GK_366563] of  StableIdentifier\n\n	(_displayName \"REACT_7535.1\")\n	(DB_ID 366563)\n	(identifier \"REACT_7535\")\n	(identifierVersion \"1\"))\n\n([GK_366839] of  StableIdentifier\n\n	(_displayName \"REACT_5883.2\")\n	(DB_ID 366839)\n	(identifier \"REACT_5883\")\n	(identifierVersion \"3\"))\n\n([GK_366902] of  StableIdentifier\n\n	(_displayName \"REACT_3803.1\")\n	(DB_ID 366902)\n	(identifier \"REACT_3803\")\n	(identifierVersion \"2\"))\n\n([GK_367393] of  StableIdentifier\n\n	(_displayName \"REACT_4773.1\")\n	(DB_ID 367393)\n	(identifier \"REACT_4773\")\n	(identifierVersion \"1\"))\n\n([GK_367488] of  StableIdentifier\n\n	(_displayName \"REACT_5140.1\")\n	(DB_ID 367488)\n	(identifier \"REACT_5140\")\n	(identifierVersion \"2\"))\n\n([GK_367733] of  StableIdentifier\n\n	(_displayName \"REACT_3905.1\")\n	(DB_ID 367733)\n	(identifier \"REACT_3905\")\n	(identifierVersion \"3\"))\n\n([GK_367761] of  StableIdentifier\n\n	(_displayName \"REACT_2698.1\")\n	(DB_ID 367761)\n	(identifier \"REACT_2698\")\n	(identifierVersion \"2\"))\n\n([GK_368490] of  StableIdentifier\n\n	(_displayName \"REACT_4632.1\")\n	(DB_ID 368490)\n	(identifier \"REACT_4632\")\n	(identifierVersion \"3\"))\n\n([GK_368665] of  StableIdentifier\n\n	(_displayName \"REACT_5556.1\")\n	(DB_ID 368665)\n	(identifier \"REACT_5556\")\n	(identifierVersion \"1\"))\n\n([GK_368685] of  StableIdentifier\n\n	(_displayName \"REACT_11797.1\")\n	(DB_ID 368685)\n	(identifier \"REACT_11797\")\n	(identifierVersion \"1\"))\n\n([GK_369343] of  StableIdentifier\n\n	(_displayName \"REACT_4499.1\")\n	(DB_ID 369343)\n	(identifier \"REACT_4499\")\n	(identifierVersion \"2\"))\n\n([GK_370163] of  StableIdentifier\n\n	(_displayName \"REACT_3855.1\")\n	(DB_ID 370163)\n	(identifier \"REACT_3855\")\n	(identifierVersion \"2\"))\n\n([GK_370192] of  StableIdentifier\n\n	(_displayName \"REACT_4528.1\")\n	(DB_ID 370192)\n	(identifier \"REACT_4528\")\n	(identifierVersion \"1\"))\n\n([GK_370748] of  StableIdentifier\n\n	(_displayName \"REACT_4114.1\")\n	(DB_ID 370748)\n	(identifier \"REACT_4114\")\n	(identifierVersion \"2\"))\n\n([GK_371020] of  StableIdentifier\n\n	(_displayName \"REACT_5688.1\")\n	(DB_ID 371020)\n	(identifier \"REACT_5688\")\n	(identifierVersion \"1\"))\n\n([GK_371402] of  StableIdentifier\n\n	(_displayName \"REACT_2659.1\")\n	(DB_ID 371402)\n	(identifier \"REACT_2659\")\n	(identifierVersion \"4\"))\n\n([GK_371450] of  StableIdentifier\n\n	(_displayName \"REACT_2540.1\")\n	(DB_ID 371450)\n	(identifier \"REACT_2540\")\n	(identifierVersion \"2\"))\n\n([GK_371770] of  StableIdentifier\n\n	(_displayName \"REACT_4466.1\")\n	(DB_ID 371770)\n	(identifier \"REACT_4466\")\n	(identifierVersion \"1\"))\n\n([GK_374884] of  Person\n\n	(_displayName \"Westermark, GT\")\n	(DB_ID 374884)\n	(initial \"GT\")\n	(surname \"Westermark\"))\n\n([GK_375875] of  Person\n\n	(_displayName \"Tsutsumi, Y\")\n	(DB_ID 375875)\n	(initial \"Y\")\n	(surname \"Tsutsumi\"))\n\n([GK_376448] of  GO_MolecularFunction\n\n	(_displayName \"peptidase activity, acting on L-amino acid peptides\")\n	(accession \"0070011\")\n	(DB_ID 376448)\n	(definition \"Catalysis of the hydrolysis of peptide bonds formed between L-amino acids.\")\n	(instanceOf [GK_1687])\n	(name \"peptidase activity, acting on L-amino acid peptides\")\n	(referenceDatabase [GK_1]))\n\n([GK_376482] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of cell communication\")\n	(accession \"0010646\")\n	(DB_ID 376482)\n	(definition \"Any process that modulates the frequency, rate or extent of cell communication. Cell communication is the process that mediates interactions between a cell and its surroundings. Encompasses interactions such as signaling or attachment between one cell and another cell, between a cell and an extracellular matrix, or between a cell and any other aspect of its environment.\")\n	(instanceOf [GK_139809])\n	(name \"regulation of cell communication\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_142]))\n\n([GK_376605] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of cell communication\")\n	(accession \"0010648\")\n	(DB_ID 376605)\n	(definition \"Any process that decreases the frequency, rate or extent of cell communication. Cell communication is the process that mediates interactions between a cell and its surroundings. Encompasses interactions such as signaling or attachment between one cell and another cell, between a cell and an extracellular matrix, or between a cell and any other aspect of its environment.\")\n	(instanceOf\n		[GK_376482]\n		[GK_160200])\n	(name \"negative regulation of cell communication\")\n	(negativelyRegulate [GK_142])\n	(referenceDatabase [GK_1]))\n\n([GK_383187] of  GO_CellularComponent\n\n	(_displayName \"membrane\")\n	(accession \"0016020\")\n	(DB_ID 383187)\n	(definition \"Double layer of lipid molecules that encloses all cells, and, in eukaryotes, many organelles; may be a single or double lipid bilayer; also includes associated proteins.\")\n	(instanceOf [GK_357])\n	(name \"membrane\")\n	(referenceDatabase [GK_1]))\n\n([GK_386948] of  InstanceEdit\n\n	(_displayName \"Williams, MG, 2007-07-11 16:49:12\")\n	(author [GK_205591])\n	(dateTime \"2007-07-11 16:49:12\")\n	(DB_ID 386948))\n\n([GK_386956] of  InstanceEdit\n\n	(_displayName \"Silverman, N, Lemaitre, B, 2008-06-20 10:21:55\")\n	(author\n		[GK_207774]\n		[GK_207144])\n	(dateTime \"2008-06-20 10:21:55\")\n	(DB_ID 386956))\n\n([GK_387943] of  StableIdentifier\n\n	(_displayName \"REACT_15682.1\")\n	(DB_ID 387943)\n	(identifier \"REACT_15682\")\n	(identifierVersion \"1\"))\n\n([GK_388867] of  Person\n\n	(_displayName \"Polak, JM\")\n	(DB_ID 388867)\n	(initial \"JM\")\n	(surname \"Polak\"))\n\n([GK_391309] of  Person\n\n	(_displayName \"Jupe, S\")\n	(DB_ID 391309)\n	(eMailAddress \"sjupe@ebi.ac.uk\")\n	(firstname \"Steve\")\n	(initial \"S\")\n	(surname \"Jupe\"))\n\n([GK_397172] of  StableIdentifier\n\n	(_displayName \"REACT_16281.1\")\n	(DB_ID 397172)\n	(identifier \"REACT_16281\")\n	(identifierVersion \"1\"))\n\n([GK_397296] of  StableIdentifier\n\n	(_displayName \"REACT_16405.1\")\n	(DB_ID 397296)\n	(identifier \"REACT_16405\")\n	(identifierVersion \"1\"))\n\n([GK_397681] of  StableIdentifier\n\n	(_displayName \"REACT_16790.1\")\n	(DB_ID 397681)\n	(identifier \"REACT_16790\")\n	(identifierVersion \"1\"))\n\n([GK_397759] of  StableIdentifier\n\n	(_displayName \"REACT_16868.1\")\n	(DB_ID 397759)\n	(identifier \"REACT_16868\")\n	(identifierVersion \"1\"))\n\n([GK_401351] of  ReferenceIsoform\n\n	(_displayName \"[Hsa] UniProt:O15519-1 CFLAR recommendedName: CASP8 and FADD-like apoptosis regulator  alternativeName: Caspase homolog shortName:CASH  alternativeName: Caspase-eight-related protein shortName:Casper  alternativeName: Caspase-like apoptosis regulatory protein shortName:CLARP  alternativeName: Cellular FLICE-like inhibitory protein shortName:c-FLIP  alternativeName: FADD-like antiapoptotic molecule 1 shortName:FLAME-1  alternativeName: Inhibitor of FLICE shortName:I-FLICE  alternativeName: MACH-related inducer of toxicity shortName:MRIT  alternativeName: Usurpin  component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p43  /component component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p12  /component\")\n	(checksum \"8C6D7E92AE1EB672\")\n	(comment \"FUNCTION Apoptosis regulator protein which may function as a crucial link between cell survival and cell death pathways in mammalian cells. Acts as an inhibitor of TNFRSF6 mediated apoptosis. A proteolytic fragment (p43) is likely retained in the death-inducing signaling complex (DISC) thereby blocking further recruitment and processing of caspase-8 at the complex. Full length and shorter isoforms have been shown either to induce apoptosis or to reduce TNFRSF-triggered apoptosis. Lacks enzymatic (caspase) activity.SUBUNIT TNFRSF6 stimulation triggers recruitment to the death-inducing signaling complex (DISC) formed by TNFRSF6, FADD and caspase-8. A proteolytic fragment (p43) stays associated with the DISC. Also interacts with caspase-10, caspase-3, TRAF1, TRAF2 and Bcl-X(L) (in vitro). Interacts with HBV protein X.TISSUE SPECIFICITY Widely expressed. Higher expression in skeletal muscle, pancreas, heart, kidney, placenta, and peripheral blood leukocytes. Also detected in diverse cell lines. Isoform 8 is predominantly expressed in testis and skeletal muscle.INDUCTION Repressed by IL2/interleukin-2 after TCR stimulation, during progression to the S phase of the cell cycle.DOMAIN The caspase domain lacks the active sites residues involved in catalysis.PTM Proteolytically processed; probably by caspase-8. Processing likely occurs at the DISC and generates subunit p43 and p12.SIMILARITY Belongs to the peptidase C14A family.SIMILARITY Contains 2 DED (death effector) domains.\")\n	(DB_ID 401351)\n	(description \"recommendedName: CASP8 and FADD-like apoptosis regulator  alternativeName: Caspase homolog shortName:CASH  alternativeName: Caspase-eight-related protein shortName:Casper  alternativeName: Caspase-like apoptosis regulatory protein shortName:CLARP  alternativeName: Cellular FLICE-like inhibitory protein shortName:c-FLIP  alternativeName: FADD-like antiapoptotic molecule 1 shortName:FLAME-1  alternativeName: Inhibitor of FLICE shortName:I-FLICE  alternativeName: MACH-related inducer of toxicity shortName:MRIT  alternativeName: Usurpin  component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p43  /component component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p12  /component\")\n	(geneName\n		\"CFLAR\"\n		\"CASH\"\n		\"CASP8AP1\"\n		\"CLARP\"\n		\"MRIT\")\n	(identifier \"O15519\")\n	(isoformParent [GK_51926])\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Host-virus interaction\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Repeat\")\n	(name \"CFLAR\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CFLAR_HUMAN\"\n		\"B4DJE0\"\n		\"B7Z9F9\"\n		\"O14673\"\n		\"O14674\"\n		\"O14675\"\n		\"O15137\"\n		\"O15138\"\n		\"O15356\"\n		\"O15510\"\n		\"O43618\"\n		\"O43619\"\n		\"O43620\"\n		\"O60458\"\n		\"O60459\"\n		\"Q53TS6\"\n		\"Q54AF1\"\n		\"Q96TE4\"\n		\"Q9UEW1\")\n	(sequenceLength 480)\n	(species [GK_48887])\n	(variantIdentifier \"O15519-1\"))\n\n([GK_419781] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Calcitonin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 419781)\n	(endCoordinate 116)\n	(name\n		\"Calcitonin\"\n		\"CALCA\"\n		\"CALC_HUMAN\"\n		\"CALCA\")\n	(referenceEntity [GK_144921])\n	(species [GK_48887])\n	(stableIdentifier [GK_1105998])\n	(startCoordinate 85))\n\n([GK_419837] of  Person\n\n	(_displayName \"Lorenz, B\")\n	(DB_ID 419837)\n	(initial \"B\")\n	(surname \"Lorenz\"))\n\n([GK_420066] of  Person\n\n	(_displayName \"Sanke, T\")\n	(DB_ID 420066)\n	(initial \"T\")\n	(surname \"Sanke\"))\n\n([GK_420076] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Amylin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 420076)\n	(endCoordinate 70)\n	(name\n		\"Amylin\"\n		\"Islet amyloid polypeptide\"\n		\"IAPP\"\n		\"IAPP_HUMAN\"\n		\"IAPP\")\n	(referenceEntity [GK_56978])\n	(species [GK_48887])\n	(stableIdentifier [GK_1111661])\n	(startCoordinate 34))\n\n([GK_422582] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of cell death\")\n	(accession \"0010941\")\n	(DB_ID 422582)\n	(definition \"Any process that modulates the rate or frequency of cell death. Cell death is the specific activation or halting of processes within a cell so that its vital functions markedly cease, rather than simply deteriorating gradually over time, which culminates in cell death.\")\n	(instanceOf [GK_139809])\n	(name \"regulation of cell death\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2275]))\n\n([GK_422583] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of cell death\")\n	(accession \"0010942\")\n	(DB_ID 422583)\n	(definition \"Any process that increases the rate or frequency of cell death. Cell death is the specific activation or halting of processes within a cell so that its vital functions markedly cease, rather than simply deteriorating gradually over time, which culminates in cell death.\")\n	(instanceOf\n		[GK_422582]\n		[GK_160064])\n	(name \"positive regulation of cell death\")\n	(positivelyRegulate [GK_2275])\n	(referenceDatabase [GK_1]))\n\n([GK_422587] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of molecular function\")\n	(accession \"0044093\")\n	(DB_ID 422587)\n	(definition \"Any process that activates or increases the rate or extent of a molecular function, an elemental biological activity occurring at the molecular level, such as catalysis or binding.\")\n	(instanceOf [GK_258234])\n	(name \"positive regulation of molecular function\")\n	(positivelyRegulate [GK_43])\n	(referenceDatabase [GK_1]))\n\n([GK_422633] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of peptidase activity\")\n	(accession \"0010952\")\n	(DB_ID 422633)\n	(definition \"Any process that increases the frequency, rate or extent of peptidase activity, the hydrolysis of peptide bonds within proteins.\")\n	(instanceOf\n		[GK_160293]\n		[GK_258858])\n	(name \"positive regulation of peptidase activity\")\n	(positivelyRegulate [GK_1687])\n	(referenceDatabase [GK_1]))\n\n([GK_423068] of  GO_BiologicalProcess\n\n	(_displayName \"positive regulation of endopeptidase activity\")\n	(accession \"0010950\")\n	(DB_ID 423068)\n	(definition \"Any process that increases the frequency, rate or extent of endopeptidase activity, the endohydrolysis of peptide bonds within proteins.\")\n	(instanceOf\n		[GK_422633]\n		[GK_258859])\n	(name \"positive regulation of endopeptidase activity\")\n	(positivelyRegulate [GK_1793])\n	(referenceDatabase [GK_1]))\n\n([GK_428598] of  Person\n\n	(_displayName \"Paul, SM\")\n	(DB_ID 428598)\n	(initial \"SM\")\n	(surname \"Paul\"))\n\n([GK_429259] of  Person\n\n	(_displayName \"Shamovsky, V\")\n	(affiliation [GK_69324])\n	(DB_ID 429259)\n	(firstname \"Veronica\")\n	(initial \"V\")\n	(surname \"Shamovsky\"))\n\n([GK_429341] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:D03250\")\n	(DB_ID 429341)\n	(identifier \"D03250\")\n	(referenceDatabase [GK_6]))\n\n([GK_429343] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin_Detemir B Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429343)\n	(endCoordinate 53)\n	(hasModifiedResidue\n		[GK_265094]\n		[GK_265017]\n		[GK_429398])\n	(name \"Insulin_Detemir B Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358276])\n	(startCoordinate 25))\n\n([GK_429344] of  ReplacedResidue\n\n	(_displayName \"L-lysine 53 replaced with L-proline\")\n	(coordinate 53)\n	(DB_ID 429344)\n	(psiMod\n		[GK_611065]\n		[GK_611069])\n	(referenceSequence [GK_57415]))\n\n([GK_429345] of  Person\n\n	(_displayName \"Hirsch, IB\")\n	(DB_ID 429345)\n	(initial \"IB\")\n	(surname \"Hirsch\"))\n\n([GK_429347] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin Glargine B Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429347)\n	(endCoordinate 56)\n	(hasModifiedResidue\n		[GK_265094]\n		[GK_265017]\n		[GK_429376])\n	(name \"Insulin Glargine B Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358234])\n	(startCoordinate 25))\n\n([GK_429348] of  Complex\n\n	(_displayName \"[Hsa] Insulin_Glargine [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_429341])\n	(DB_ID 429348)\n	(hasComponent\n		[GK_429347]\n		[GK_429369])\n	(name \"Insulin_Glargine\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358602]))\n\n([GK_429349] of  Complex\n\n	(_displayName \"[Hsa] Insulin_Detemir(Levemir) [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_429388])\n	(DB_ID 429349)\n	(hasComponent\n		[GK_265170]\n		[GK_429343])\n	(name \"Insulin_Detemir(Levemir)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358413]))\n\n([GK_429352] of  Complex\n\n	(_displayName \"[Hsa] Insulin Aspart (Novolog) [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_429381])\n	(DB_ID 429352)\n	(hasComponent\n		[GK_265170]\n		[GK_429393])\n	(name \"Insulin Aspart (Novolog)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1357816]))\n\n([GK_429361] of  ReplacedResidue\n\n	(_displayName \"L-proline 52 replaced with L-aspartic acid\")\n	(coordinate 52)\n	(DB_ID 429361)\n	(psiMod\n		[GK_611072]\n		[GK_611070])\n	(referenceSequence [GK_57415]))\n\n([GK_429362] of  ReplacedResidue\n\n	(_displayName \"L-asparagine 110 replaced with glycine\")\n	(coordinate 110)\n	(DB_ID 429362)\n	(psiMod\n		[GK_611064]\n		[GK_611068])\n	(referenceSequence [GK_57415]))\n\n([GK_429369] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin Glargine A Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429369)\n	(endCoordinate 110)\n	(hasModifiedResidue\n		[GK_265035]\n		[GK_264874]\n		[GK_264928]\n		[GK_264936]\n		[GK_429362])\n	(name \"Insulin Glargine A Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357824])\n	(startCoordinate 90))\n\n([GK_429372] of  LiteratureReference\n\n	(_displayName \"Insulin analogues\")\n	(author [GK_429345])\n	(DB_ID 429372)\n	(journal \"N Engl J Med\")\n	(pages \"174-83\")\n	(pubMedIdentifier 15647580)\n	(title \"Insulin analogues\")\n	(volume 352)\n	(year 2005))\n\n([GK_429374] of  ReplacedResidue\n\n	(_displayName \"L-proline 52 replaced with L-lysine\")\n	(coordinate 52)\n	(DB_ID 429374)\n	(psiMod\n		[GK_611072]\n		[GK_611062])\n	(referenceSequence [GK_57415]))\n\n([GK_429376] of  ReplacedResidue\n\n	(_displayName \"L-threonine 54 replaced with L-threonine - L-arginine - L-arginine\")\n	(coordinate 54)\n	(DB_ID 429376)\n	(psiMod\n		[GK_611087]\n		[GK_611082]\n		[GK_611066]\n		[GK_611066])\n	(referenceSequence [GK_57415]))\n\n([GK_429381] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:D04475\")\n	(DB_ID 429381)\n	(identifier \"D04475\")\n	(referenceDatabase [GK_6]))\n\n([GK_429384] of  Complex\n\n	(_displayName \"[Hsa] Insulin_Humalog(Lispro) [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_429391])\n	(DB_ID 429384)\n	(hasComponent\n		[GK_265170]\n		[GK_429397])\n	(name \"Insulin_Humalog(Lispro)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358519]))\n\n([GK_429388] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:D04539\")\n	(DB_ID 429388)\n	(identifier \"D04539\")\n	(referenceDatabase [GK_6]))\n\n([GK_429391] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:D04477\")\n	(DB_ID 429391)\n	(identifier \"D04477\")\n	(referenceDatabase [GK_6]))\n\n([GK_429393] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin Aspart B Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429393)\n	(endCoordinate 54)\n	(hasModifiedResidue\n		[GK_265094]\n		[GK_265017]\n		[GK_429361])\n	(name \"Insulin Aspart B Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357978])\n	(startCoordinate 25))\n\n([GK_429397] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin Humalog B Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429397)\n	(endCoordinate 54)\n	(hasModifiedResidue\n		[GK_265094]\n		[GK_265017]\n		[GK_429374]\n		[GK_429344])\n	(name \"Insulin Humalog B Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357751])\n	(startCoordinate 25))\n\n([GK_429398] of  ModifiedResidue\n\n	(_displayName \"N6-myristoyl-L-lysine at 53\")\n	(coordinate 53)\n	(DB_ID 429398)\n	(psiMod [GK_448179])\n	(referenceSequence [GK_57415]))\n\n([GK_429407] of  DefinedSet\n\n	(_displayName \"[Hsa] Human Insulin Analogues [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429407)\n	(hasMember\n		[GK_429348]\n		[GK_429384]\n		[GK_429349]\n		[GK_429352]\n		[GK_429472])\n	(literatureReference [GK_429372])\n	(name \"Human Insulin Analogues\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358336]))\n\n([GK_429461] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:D04540\")\n	(DB_ID 429461)\n	(identifier \"D04540\")\n	(referenceDatabase [GK_6]))\n\n([GK_429462] of  ReplacedResidue\n\n	(_displayName \"L-asparagine 27 replaced with L-lysine\")\n	(coordinate 27)\n	(DB_ID 429462)\n	(psiMod\n		[GK_611064]\n		[GK_611062])\n	(referenceSequence [GK_57415]))\n\n([GK_429463] of  ReplacedResidue\n\n	(_displayName \"L-lysine 53 replaced with L-glutamic acid\")\n	(coordinate 53)\n	(DB_ID 429463)\n	(psiMod\n		[GK_611065]\n		[GK_611067])\n	(referenceSequence [GK_57415]))\n\n([GK_429464] of  Person\n\n	(_displayName \"Becker, RH\")\n	(DB_ID 429464)\n	(initial \"RH\")\n	(surname \"Becker\"))\n\n([GK_429465] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Insulin Glulisine B Chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 429465)\n	(endCoordinate 54)\n	(hasModifiedResidue\n		[GK_265094]\n		[GK_265017]\n		[GK_429463]\n		[GK_429462])\n	(name \"Insulin Glulisine B Chain\")\n	(referenceEntity [GK_57415])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358212])\n	(startCoordinate 25))\n\n([GK_429466] of  Person\n\n	(_displayName \"Plosker, GL\")\n	(DB_ID 429466)\n	(initial \"GL\")\n	(surname \"Plosker\"))\n\n([GK_429467] of  Person\n\n	(_displayName \"Frick, AD\")\n	(DB_ID 429467)\n	(initial \"AD\")\n	(surname \"Frick\"))\n\n([GK_429468] of  Summation\n\n	(_displayName \"Insulin Glulisine, a fast acting analogue of human insulin, ...\")\n	(DB_ID 429468)\n	(text \"Insulin Glulisine, a fast acting analogue of human insulin, that has a faster onset of action and shorter duration of action than regular human insulin (RHI) in patients with type 1 or 2 diabetes mellitus\"))\n\n([GK_429470] of  LiteratureReference\n\n	(_displayName \"Clinical pharmacokinetics and pharmacodynamics of insulin glulisine\")\n	(author\n		[GK_429464]\n		[GK_429467])\n	(DB_ID 429470)\n	(journal \"Clin Pharmacokinet\")\n	(pages \"7-20\")\n	(pubMedIdentifier 18076215)\n	(title \"Clinical pharmacokinetics and pharmacodynamics of insulin glulisine\")\n	(volume 47)\n	(year 2008))\n\n([GK_429471] of  Person\n\n	(_displayName \"Garnock-Jones, KP\")\n	(DB_ID 429471)\n	(initial \"KP\")\n	(surname \"Garnock-Jones\"))\n\n([GK_429472] of  Complex\n\n	(_displayName \"[Hsa] Insulin Glulisine (Apidra) [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_429461])\n	(DB_ID 429472)\n	(hasComponent\n		[GK_265170]\n		[GK_429465])\n	(literatureReference\n		[GK_429470]\n		[GK_429473])\n	(name \"Insulin Glulisine (Apidra)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358370])\n	(summation [GK_429468]))\n\n([GK_429473] of  LiteratureReference\n\n	(_displayName \"Insulin glulisine: a review of its use in the management of diabetes mellitus\")\n	(author\n		[GK_429471]\n		[GK_429466])\n	(DB_ID 429473)\n	(journal \"Drugs\")\n	(pages \"1035-57\")\n	(pubMedIdentifier 19496630)\n	(title \"Insulin glulisine: a review of its use in the management of diabetes mellitus\")\n	(volume 69)\n	(year 2009))\n\n([GK_429711] of  Person\n\n	(_displayName \"Pfizenmaier, K\")\n	(DB_ID 429711)\n	(firstname \"Klaus\")\n	(initial \"K\")\n	(surname \"Pfizenmaier\"))\n\n([GK_43] of  GO_MolecularFunction\n\n	(_displayName \"molecular_function\")\n	(accession \"0003674\")\n	(DB_ID 43)\n	(definition \"Elemental activities, such as catalysis or binding, describing the actions of a gene product at the molecular level. A given gene product may exhibit one or more molecular functions.\")\n	(name \"molecular_function\")\n	(referenceDatabase [GK_1]))\n\n([GK_43124] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer:FADD [plasma membrane]\")\n	(compartment [GK_876])\n	(crossReference [GK_43125])\n	(DB_ID 43124)\n	(hasComponent\n		[GK_76195]\n		[GK_54639]\n		[GK_54639])\n	(literatureReference\n		[GK_3465522]\n		[GK_3465531])\n	(name \"FASL:FAS Receptor Trimer:FADD\")\n	(species [GK_48887])\n	(stableIdentifier [GK_366491]))\n\n([GK_43125] of  DatabaseIdentifier\n\n	(_displayName \"COMPOUND:C08698\")\n	(DB_ID 43125)\n	(identifier \"C08698\")\n	(referenceDatabase [GK_6]))\n\n([GK_433999] of  Person\n\n	(_displayName \"Burns, K\")\n	(DB_ID 433999)\n	(initial \"K\")\n	(surname \"Burns\"))\n\n([GK_4344] of  GO_MolecularFunction\n\n	(_displayName \"cysteine-type endopeptidase activity\")\n	(accession \"0004197\")\n	(DB_ID 4344)\n	(definition \"Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which the sulfhydryl group of a cysteine residue at the active center acts as a nucleophile.\")\n	(ecNumber\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\"\n		\"3.4.22\")\n	(instanceOf\n		[GK_1793]\n		[GK_2741])\n	(name \"cysteine-type endopeptidase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_436407] of  EntityCompartment\n\n	(_displayName \"host cell cytosol\")\n	(accession \"0044164\")\n	(DB_ID 436407)\n	(definition \"The part of the host cell cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.\")\n	(instanceOf [GK_259437])\n	(name \"host cell cytosol\")\n	(referenceDatabase [GK_1]))\n\n([GK_437238] of  Person\n\n	(_displayName \"Aigner, S\")\n	(DB_ID 437238)\n	(initial \"S\")\n	(surname \"Aigner\"))\n\n([GK_437354] of  ReferenceDatabase\n\n	(_displayName \"MOD\")\n	(accessUrl \"http://www.ebi.ac.uk/ontology-lookup/?termId=MOD:###ID###\")\n	(DB_ID 437354)\n	(name\n		\"MOD\"\n		\"PSI-MOD\")\n	(url \"http://www.ebi.ac.uk/ontology-lookup/browse.do?ontName=MOD\"))\n\n([GK_445564] of  Person\n\n	(_displayName \"Takahashi, N\")\n	(DB_ID 445564)\n	(initial \"N\")\n	(surname \"Takahashi\"))\n\n([GK_447576] of  GO_CellularComponent\n\n	(_displayName \"other organism\")\n	(accession \"0044215\")\n	(DB_ID 447576)\n	(definition \"A secondary organism with which the first organism is interacting.\")\n	(instanceOf [GK_160483])\n	(name \"other organism\")\n	(referenceDatabase [GK_1]))\n\n([GK_447577] of  GO_CellularComponent\n\n	(_displayName \"other organism part\")\n	(accession \"0044217\")\n	(componentOf [GK_447576])\n	(DB_ID 447577)\n	(definition \"Any constituent part of a secondary organism with which the first organism is interacting.\")\n	(instanceOf [GK_258201])\n	(name \"other organism part\")\n	(referenceDatabase [GK_1]))\n\n([GK_448179] of  PsiMod\n\n	(_displayName \"N6-myristoyl-L-lysine [MOD:00087]\")\n	(DB_ID 448179)\n	(definition \"A protein modification that effectively converts an L-lysine residue to N6-myristoyl-L-lysine.\")\n	(identifier \"00087\")\n	(name \"N6-myristoyl-L-lysine\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-amino-6-(tetradecanoylamino)hexanoic acid\"\n		\"N6-myristoyl lysine\"\n		\"N6MyrLys\"\n		\"N6-myristoylated L-lysine\"\n		\"Myristoylation\"\n		\"Myristoyl\"))\n\n([GK_448182] of  PsiMod\n\n	(_displayName \"half cystine [MOD:00798]\")\n	(DB_ID 448182)\n	(definition \"A protein modification that can be regarded as effectively either one half of a cystine cross-link, or a cysteine residue with one hydrogen atom or proton removed.\")\n	(identifier \"00798\")\n	(name \"half cystine\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"Half of a disulfide bridge\"\n		\"Dehydro\"))\n\n([GK_450] of  GO_MolecularFunction\n\n	(_displayName \"hydrolase activity\")\n	(accession \"0016787\")\n	(DB_ID 450)\n	(definition \"Catalysis of the hydrolysis of various bonds, e.g. C-O, C-N, C-C, phosphoric anhydride bonds, etc. Hydrolase is the systematic name for any enzyme of EC class 3.\")\n	(ecNumber\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\"\n		\"3\")\n	(instanceOf [GK_97])\n	(name \"hydrolase activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_452043] of  Person\n\n	(_displayName \"Shin, H\")\n	(DB_ID 452043)\n	(firstname \"Hyunshun\")\n	(initial \"H\")\n	(surname \"Shin\"))\n\n([GK_452587] of  Person\n\n	(_displayName \"Betsholtz, C\")\n	(DB_ID 452587)\n	(initial \"C\")\n	(surname \"Betsholtz\"))\n\n([GK_459] of  EntityCompartment\n\n	(_displayName \"cytoplasm\")\n	(accession \"0005737\")\n	(DB_ID 459)\n	(definition \"All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.\")\n	(instanceOf [GK_258148])\n	(name \"cytoplasm\")\n	(referenceDatabase [GK_1]))\n\n([GK_469669] of  Person\n\n	(_displayName \"Gilbey, SG\")\n	(DB_ID 469669)\n	(initial \"SG\")\n	(surname \"Gilbey\"))\n\n([GK_480491] of  Person\n\n	(_displayName \"Heinzmann, C\")\n	(DB_ID 480491)\n	(initial \"C\")\n	(surname \"Heinzmann\"))\n\n([GK_48887] of  Species\n\n	(_displayName \"Homo sapiens\")\n	(crossReference [GK_72938])\n	(DB_ID 48887)\n	(name\n		\"Homo sapiens\"\n		\"H. sapiens\"\n		\"Hs\"\n		\"human\"\n		\"man\")\n	(superTaxon [GK_72939]))\n\n([GK_49678] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P78536 ADAM17 recommendedName: Disintegrin and metalloproteinase domain-containing protein 17 shortName:ADAM 17 ecNumber3.4.24.86/ecNumber  alternativeName: Snake venom-like protease  alternativeName: TNF-alpha convertase  alternativeName: TNF-alpha-converting enzyme  cdAntigenNameCD156b/cdAntigenName\")\n	(checksum \"5B1032F6B88A837F\")\n	(comment \"CATALYTIC ACTIVITY Narrow endopeptidase specificity. Cleaves Pro-Leu-Ala-Gln-Ala-|-Val-Arg-Ser-Ser-Ser in the membrane-bound, 26-kDa form of tumor necrosis factor alpha (TNF-alpha). Similarly cleaves other membrane-anchored, cell-surface proteins to \'shed\' the extracellular domains.COFACTOR Binds 1 zinc ion per subunit.SUBUNIT Interacts with MAD2L1, MAPK14 and MUC1.TISSUE SPECIFICITY Ubiquitously expressed. Expressed at highest levels in adult heart, placenta, skeletal muscle, pancreas, spleen, thymus, prostate, testes, ovary and small intestine, and in fetal brain, lung, liver and kidney.INDUCTION In arthritis-affected cartilage.DOMAIN The conserved cysteine present in the cysteine-switch motif binds the catalytic zinc ion, thus inhibiting the enzyme. The dissociation of the cysteine from the zinc ion upon the activation-peptide release activates the enzyme.PTM Phosphorylated. Stimulation by growth factor or phorbol 12-myristate 13-acetate induces phosphorylation of Ser-819 but decreases phosphorylation of Ser-791. Phosphorylation at THR-735 by MAPK14 is required for ADAM17-mediated ectodomain shedding.SIMILARITY Contains 1 disintegrin domain.SIMILARITY Contains 1 peptidase M12B domain.\")\n	(DB_ID 49678)\n	(description \"recommendedName: Disintegrin and metalloproteinase domain-containing protein 17 shortName:ADAM 17 ecNumber3.4.24.86/ecNumber  alternativeName: Snake venom-like protease  alternativeName: TNF-alpha convertase  alternativeName: TNF-alpha-converting enzyme  cdAntigenNameCD156b/cdAntigenName\")\n	(geneName\n		\"ADAM17\"\n		\"CSVP\"\n		\"TACE\")\n	(identifier \"P78536\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Cleavage on pair of basic residues\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Hydrolase\"\n		\"Membrane\"\n		\"Metal-binding\"\n		\"Metalloprotease\"\n		\"Notch signaling pathway\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Protease\"\n		\"Reference proteome\"\n		\"SH3-binding\"\n		\"Signal\"\n		\"Transmembrane\"\n		\"Transmembrane helix\"\n		\"Zinc\"\n		\"Zymogen\")\n	(name \"ADAM17\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"ADA17_HUMAN\"\n		\"O60226\")\n	(sequenceLength 824)\n	(species [GK_48887]))\n\n([GK_49984] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P01160 NPPA recommendedName: Natriuretic peptides A  alternativeName: CDD-ANF  alternativeName: Prepronatriodilatin  component recommendedName: Cardiodilatin-related peptide shortName:CDP  /component component recommendedName: Atrial natriuretic factor shortName:ANF  alternativeName: Atrial natriuretic peptide shortName:ANP  /component\")\n	(checksum \"1\")\n	(comment \"PTM Cleaved by CORIN upon secretion to produce the functional hormone.POLYMORPHISM There are two different prepronatriodilatin alleles. One codes for 2 Arg residues at the C-terminus that are cleaved to form the mature peptide, while the other ends in a termination codon immediately after the last codon of the mature peptide.SIMILARITY Belongs to the natriuretic peptide family.\")\n	(DB_ID 49984)\n	(description \"recommendedName: Natriuretic peptides A  alternativeName: CDD-ANF  alternativeName: Prepronatriodilatin  component recommendedName: Cardiodilatin-related peptide shortName:CDP  /component component recommendedName: Atrial natriuretic factor shortName:ANF  alternativeName: Atrial natriuretic peptide shortName:ANP  /component\")\n	(geneName\n		\"NPPA\"\n		\"ANP\"\n		\"PND\")\n	(identifier \"P01160\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Atrial fibrillation\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Hormone\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\"\n		\"Vasoactive\")\n	(name \"NPPA\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"ANF_HUMAN\"\n		\"Q13766\"\n		\"Q5JZE1\")\n	(sequenceLength 153)\n	(species [GK_48887]))\n\n([GK_500144] of  Person\n\n	(_displayName \"Baker, D\")\n	(DB_ID 500144)\n	(firstname \"David\")\n	(initial \"D\")\n	(surname \"Baker\"))\n\n([GK_500361] of  Person\n\n	(_displayName \"Peng, S\")\n	(DB_ID 500361)\n	(initial \"S\")\n	(surname \"Peng\"))\n\n([GK_50094] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P02647 APOA1 recommendedName: Apolipoprotein A-I shortName:Apo-AI shortName:ApoA-I  alternativeName: Apolipoprotein A1  component recommendedName: Apolipoprotein A-I(1-242)  /component\")\n	(checksum \"1\")\n	(comment \"PTM Phosphorylation sites are present in the extracelllular medium.SIMILARITY Belongs to the apolipoprotein A1/A4/E family.\")\n	(DB_ID 50094)\n	(description \"recommendedName: Apolipoprotein A-I shortName:Apo-AI shortName:ApoA-I  alternativeName: Apolipoprotein A1  component recommendedName: Apolipoprotein A-I(1-242)  /component\")\n	(geneName \"APOA1\")\n	(identifier \"P02647\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Atherosclerosis\"\n		\"Cholesterol metabolism\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Glycation\"\n		\"Glycoprotein\"\n		\"HDL\"\n		\"Lipid metabolism\"\n		\"Lipid transport\"\n		\"Lipoprotein\"\n		\"Neuropathy\"\n		\"Palmitate\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\"\n		\"Steroid metabolism\"\n		\"Transport\")\n	(name \"APOA1\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"APOA1_HUMAN\"\n		\"A8K866\"\n		\"Q6LDN9\"\n		\"Q6Q785\"\n		\"Q9UCS8\"\n		\"Q9UCT8\")\n	(sequenceLength 267)\n	(species [GK_48887]))\n\n([GK_50098] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P06727 APOA4 recommendedName: Apolipoprotein A-IV shortName:Apo-AIV shortName:ApoA-IV  alternativeName: Apolipoprotein A4 \")\n	(checksum \"1\")\n	(comment \"FUNCTION May have a role in chylomicrons and VLDL secretion and catabolism. Required for efficient activation of lipoprotein lipase by ApoC-II; potent activator of LCAT. Apoa-IV is a major component of HDL and chylomicrons.DOMAIN Nine of the thirteen 22-amino acid tandem repeats (each 22-mer is actually a tandem array of two, A and B, related 11-mers) occurring in this sequence are predicted to be highly alpha-helical, and many of these helices are amphipathic. They may therefore serve as lipid-binding domains with lecithin:cholesterol acyltransferase (LCAT) activating abilities.PTM Phosphorylation sites are present in the extracelllular medium.POLYMORPHISM Eight alleles have been characterized (APOA-IV*0 to APOA-IV*7). APOA-IV*1 is the major allele (90%), APOA-IV*2 is also common (8%), the others are rare alleles. The sequence shown represents allele APOA-IV*1.SIMILARITY Belongs to the apolipoprotein A1/A4/E family.\")\n	(DB_ID 50098)\n	(description \"recommendedName: Apolipoprotein A-IV shortName:Apo-AIV shortName:ApoA-IV  alternativeName: Apolipoprotein A4 \")\n	(geneName \"APOA4\")\n	(identifier \"P06727\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Chylomicron\"\n		\"Complete proteome\"\n		\"HDL\"\n		\"Lipid transport\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\"\n		\"Transport\")\n	(name \"APOA4\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"APOA4_HUMAN\"\n		\"A8MSL6\"\n		\"Q14CW8\"\n		\"Q6Q787\")\n	(sequenceLength 396)\n	(species [GK_48887]))\n\n([GK_504] of  GO_BiologicalProcess\n\n	(_displayName \"metabolic process\")\n	(accession \"0008152\")\n	(DB_ID 504)\n	(definition \"The chemical reactions and pathways, including anabolism and catabolism, by which living organisms transform chemical substances. Metabolic processes typically transform small molecules, but also include macromolecular processes such as DNA repair and replication, and protein synthesis and degradation.\")\n	(instanceOf [GK_106])\n	(name \"metabolic process\")\n	(referenceDatabase [GK_1]))\n\n([GK_50591] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P61769 B2M recommendedName: Beta-2-microglobulin  component recommendedName: Beta-2-microglobulin form pI 5.3  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Component of the class I major histocompatibility complex (MHC). Involved in the presentation of peptide antigens to the immune system.PTM Glycation of Ile-21 is observed in long-term hemodialysis patients.SIMILARITY Belongs to the beta-2-microglobulin family.SIMILARITY Contains 1 Ig-like C1-type (immunoglobulin-like) domain.\")\n	(DB_ID 50591)\n	(description \"recommendedName: Beta-2-microglobulin  component recommendedName: Beta-2-microglobulin form pI 5.3  /component\")\n	(geneName\n		\"B2M\"\n		\"CDABP0092\"\n		\"HDCMA22P\")\n	(identifier \"P61769\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycation\"\n		\"Glycoprotein\"\n		\"Immunity\"\n		\"Immunoglobulin domain\"\n		\"MHC I\"\n		\"Pyrrolidone carboxylic acid\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"B2M\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"B2MG_HUMAN\"\n		\"P01884\"\n		\"Q540F8\"\n		\"Q6IAT8\"\n		\"Q9UCK0\"\n		\"Q9UD48\"\n		\"Q9UDF4\")\n	(sequenceLength 119)\n	(species [GK_48887]))\n\n([GK_50810] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q15582 TGFBI recommendedName: Transforming growth factor-beta-induced protein ig-h3 shortName:Beta ig-h3  alternativeName: Kerato-epithelin  alternativeName: RGD-containing collagen-associated protein shortName:RGD-CAP  alternativeName: RGD-containing collagen-associated protein shortName:RGD-CAP \")\n	(checksum \"1\")\n	(comment \"FUNCTION Binds to type I, II, and IV collagens. This adhesion protein may play an important role in cell-collagen interactions. In cartilage, may be involved in endochondral bone formation.INDUCTION By TGFB1.DISEASE Defects in TGFBI are the cause of corneal dystrophy lattice type 1 (CDL1) [MIM:122200]. Inheritance is autosomal dominant.DISEASE Defects in TGFBI are a cause of corneal dystrophy Thiel-Behnke type (CDTB) [MIM:602082]; also known as corneal dystrophy of Bowman layer type 2 (CDB2).DISEASE Defects in TGFBI are the cause of Avellino corneal dystrophy (ACD) [MIM:607541]. ACD could be considered a variant of granular dystrophy with a significant amyloidogenic tendency. Inheritance is autosomal dominant.SIMILARITY Contains 1 EMI domain.SIMILARITY Contains 4 FAS1 domains.\")\n	(DB_ID 50810)\n	(description \"recommendedName: Transforming growth factor-beta-induced protein ig-h3 shortName:Beta ig-h3  alternativeName: Kerato-epithelin  alternativeName: RGD-containing collagen-associated protein shortName:RGD-CAP  alternativeName: RGD-containing collagen-associated protein shortName:RGD-CAP \")\n	(geneName\n		\"TGFBI\"\n		\"BIGH3\")\n	(identifier \"Q15582\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Amyloid\"\n		\"Cell adhesion\"\n		\"Complete proteome\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Extracellular matrix\"\n		\"Gamma-carboxyglutamic acid\"\n		\"Polymorphism\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Sensory transduction\"\n		\"Signal\"\n		\"Vision\")\n	(name \"TGFBI\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"BGH3_HUMAN\"\n		\"D3DQB1\"\n		\"O14471\"\n		\"O14472\"\n		\"O14476\"\n		\"O43216\"\n		\"O43217\"\n		\"O43218\"\n		\"O43219\"\n		\"Q53XM1\")\n	(sequenceLength 683)\n	(species [GK_48887]))\n\n([GK_517749] of  Person\n\n	(_displayName \"Hudson, J\")\n	(DB_ID 517749)\n	(initial \"J\")\n	(surname \"Hudson\"))\n\n([GK_51926] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:O15519 CFLAR recommendedName: CASP8 and FADD-like apoptosis regulator  alternativeName: Caspase homolog shortName:CASH  alternativeName: Caspase-eight-related protein shortName:Casper  alternativeName: Caspase-like apoptosis regulatory protein shortName:CLARP  alternativeName: Cellular FLICE-like inhibitory protein shortName:c-FLIP  alternativeName: FADD-like antiapoptotic molecule 1 shortName:FLAME-1  alternativeName: Inhibitor of FLICE shortName:I-FLICE  alternativeName: MACH-related inducer of toxicity shortName:MRIT  alternativeName: Usurpin  component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p43  /component component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p12  /component\")\n	(checksum \"8C6D7E92AE1EB672\")\n	(comment \"FUNCTION Apoptosis regulator protein which may function as a crucial link between cell survival and cell death pathways in mammalian cells. Acts as an inhibitor of TNFRSF6 mediated apoptosis. A proteolytic fragment (p43) is likely retained in the death-inducing signaling complex (DISC) thereby blocking further recruitment and processing of caspase-8 at the complex. Full length and shorter isoforms have been shown either to induce apoptosis or to reduce TNFRSF-triggered apoptosis. Lacks enzymatic (caspase) activity.SUBUNIT TNFRSF6 stimulation triggers recruitment to the death-inducing signaling complex (DISC) formed by TNFRSF6, FADD and caspase-8. A proteolytic fragment (p43) stays associated with the DISC. Also interacts with caspase-10, caspase-3, TRAF1, TRAF2 and Bcl-X(L) (in vitro). Interacts with HBV protein X.TISSUE SPECIFICITY Widely expressed. Higher expression in skeletal muscle, pancreas, heart, kidney, placenta, and peripheral blood leukocytes. Also detected in diverse cell lines. Isoform 8 is predominantly expressed in testis and skeletal muscle.INDUCTION Repressed by IL2/interleukin-2 after TCR stimulation, during progression to the S phase of the cell cycle.DOMAIN The caspase domain lacks the active sites residues involved in catalysis.PTM Proteolytically processed; probably by caspase-8. Processing likely occurs at the DISC and generates subunit p43 and p12.SIMILARITY Belongs to the peptidase C14A family.SIMILARITY Contains 2 DED (death effector) domains.\")\n	(DB_ID 51926)\n	(description \"recommendedName: CASP8 and FADD-like apoptosis regulator  alternativeName: Caspase homolog shortName:CASH  alternativeName: Caspase-eight-related protein shortName:Casper  alternativeName: Caspase-like apoptosis regulatory protein shortName:CLARP  alternativeName: Cellular FLICE-like inhibitory protein shortName:c-FLIP  alternativeName: FADD-like antiapoptotic molecule 1 shortName:FLAME-1  alternativeName: Inhibitor of FLICE shortName:I-FLICE  alternativeName: MACH-related inducer of toxicity shortName:MRIT  alternativeName: Usurpin  component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p43  /component component recommendedName: CASP8 and FADD-like apoptosis regulator subunit p12  /component\")\n	(geneName\n		\"CFLAR\"\n		\"CASH\"\n		\"CASP8AP1\"\n		\"CLARP\"\n		\"MRIT\")\n	(identifier \"O15519\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Host-virus interaction\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Repeat\")\n	(name \"CFLAR\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CFLAR_HUMAN\"\n		\"B4DJE0\"\n		\"B7Z9F9\"\n		\"O14673\"\n		\"O14674\"\n		\"O14675\"\n		\"O15137\"\n		\"O15138\"\n		\"O15356\"\n		\"O15510\"\n		\"O43618\"\n		\"O43619\"\n		\"O43620\"\n		\"O60458\"\n		\"O60459\"\n		\"Q53TS6\"\n		\"Q54AF1\"\n		\"Q96TE4\"\n		\"Q9UEW1\")\n	(sequenceLength 480)\n	(species [GK_48887]))\n\n([GK_528243] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of signaling\")\n	(accession \"0023051\")\n	(DB_ID 528243)\n	(definition \"Any process that modulates the frequency, rate or extent of a signaling process.\")\n	(instanceOf [GK_139810])\n	(name \"regulation of signaling\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_2587988]))\n\n([GK_528244] of  GO_BiologicalProcess\n\n	(_displayName \"signaling\")\n	(accession \"0023052\")\n	(DB_ID 528244)\n	(definition \"The entirety of a process in which information is transmitted within a biological system. This process begins with an active signal and ends when a cellular response has been triggered.\")\n	(instanceOf [GK_106])\n	(name \"signaling\")\n	(referenceDatabase [GK_1]))\n\n([GK_528249] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of signaling\")\n	(accession \"0023057\")\n	(DB_ID 528249)\n	(definition \"Any process that stops, prevents, or reduces the frequency, rate or extent of a signaling process.\")\n	(instanceOf\n		[GK_528243]\n		[GK_160005])\n	(name \"negative regulation of signaling\")\n	(negativelyRegulate [GK_2587988])\n	(referenceDatabase [GK_1]))\n\n([GK_53331] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P01034 CST3 recommendedName: Cystatin-C  alternativeName: Cystatin-3  alternativeName: Gamma-trace  alternativeName: Neuroendocrine basic polypeptide  alternativeName: Post-gamma-globulin \")\n	(checksum \"1\")\n	(comment \"FUNCTION As an inhibitor of cysteine proteinases, this protein is thought to serve an important physiological role as a local regulator of this enzyme activity.SUBUNIT Homodimer.MISCELLANEOUS Potential cerebrospinal fluid marker for the diagnosis of Creutzfeldt-Jakob disease.SIMILARITY Belongs to the cystatin family.\")\n	(DB_ID 53331)\n	(description \"recommendedName: Cystatin-C  alternativeName: Cystatin-3  alternativeName: Gamma-trace  alternativeName: Neuroendocrine basic polypeptide  alternativeName: Post-gamma-globulin \")\n	(geneName \"CST3\")\n	(identifier \"P01034\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Age-related macular degeneration\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Polymorphism\"\n		\"Protease inhibitor\"\n		\"Secreted\"\n		\"Signal\"\n		\"Thiol protease inhibitor\")\n	(name \"CST3\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CYTC_HUMAN\"\n		\"B2R5J9\"\n		\"D3DW42\"\n		\"Q6FGW9\")\n	(sequenceLength 146)\n	(species [GK_48887]))\n\n([GK_538696] of  Person\n\n	(_displayName \"Enghild, JJ\")\n	(DB_ID 538696)\n	(initial \"JJ\")\n	(surname \"Enghild\"))\n\n([GK_54639] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] FADD [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 54639)\n	(endCoordinate 208)\n	(literatureReference [GK_140483])\n	(name\n		\"FADD\"\n		\"FADD protein (FAS-associating death domain-containing protein) (Mediator of receptor induced toxicity)\"\n		\"FADD protein\"\n		\"FAS-associating death domain-containing protein\"\n		\"Mediator of receptor induced toxicity\")\n	(referenceEntity [GK_54640])\n	(species [GK_48887])\n	(stableIdentifier [GK_365546])\n	(startCoordinate 1))\n\n([GK_54640] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q13158 FADD recommendedName: Protein FADD  alternativeName: FAS-associated death domain protein  alternativeName: FAS-associating death domain-containing protein  alternativeName: Growth-inhibiting gene 3 protein  alternativeName: Mediator of receptor induced toxicity \")\n	(checksum \"0E65E2F852E83507\")\n	(comment \"FUNCTION Apoptotic adaptor molecule that recruits caspase-8 or caspase-10 to the activated Fas (CD95) or TNFR-1 receptors. The resulting aggregate called the death-inducing signaling complex (DISC) performs caspase-8 proteolytic activation. Active caspase-8 initiates the subsequent cascade of caspases mediating apoptosis. Involved in interferon-mediated antiviral immune response, playing a role in the positive regulation of interferon signaling.SUBUNIT Can self-associate. Interacts with CFLAR, PEA15 and MBD4. When phosphorylated, part of a complex containing HIPK3 and FAS. May interact with MAVS/IPS1. Interacts with MOCV v-CFLAR protein and LRDD. Interacts (via death domain) with FAS (via death domain). Interacts with CASP8.TISSUE SPECIFICITY Expressed in a wide variety of tissues, except for peripheral blood mononuclear leukocytes.DOMAIN Contains a death domain involved in the binding of the corresponding domain within Fas receptor.DOMAIN The interaction between the FAS and FADD death domains is crucial for the formation of the death-inducing signaling complex (DISC).SIMILARITY Contains 1 death domain.SIMILARITY Contains 1 DED (death effector) domain.\")\n	(DB_ID 54640)\n	(description \"recommendedName: Protein FADD  alternativeName: FAS-associated death domain protein  alternativeName: FAS-associating death domain-containing protein  alternativeName: Growth-inhibiting gene 3 protein  alternativeName: Mediator of receptor induced toxicity \")\n	(geneName\n		\"FADD\"\n		\"MORT1\"\n		\"GIG3\")\n	(identifier \"Q13158\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Disease mutation\"\n		\"Host-virus interaction\"\n		\"Immunity\"\n		\"Innate immunity\"\n		\"Phosphoprotein\"\n		\"Reference proteome\")\n	(name \"FADD\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"FADD_HUMAN\"\n		\"Q14866\"\n		\"Q6IBR4\")\n	(sequenceLength 208)\n	(species [GK_48887]))\n\n([GK_54840] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P02671 FGA recommendedName: Fibrinogen alpha chain  component recommendedName: Fibrinopeptide A  /component component recommendedName: Fibrinogen alpha chain  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Fibrinogen has a double function: yielding monomers that polymerize into fibrin and acting as a cofactor in platelet aggregation.DOMAIN A long coiled coil structure formed by 3 polypeptide chains connects the central nodule to the C-terminal domains (distal nodules). The long C-terminal ends of the alpha chains fold back, contributing a fourth strand to the coiled coil structure.PTM Forms F13A-mediated cross-links between a glutamine and the epsilon-amino group of a lysine residue, forming fibronectin-fibrinogen heteropolymers.PTM Conversion of fibrinogen to fibrin is triggered by thrombin, which cleaves fibrinopeptides A and B from alpha and beta chains, and thus exposes the N-terminal polymerization sites responsible for the formation of the soft clot. The soft clot is converted into the hard clot by factor XIIIA which catalyzes the epsilon-(gamma-glutamyl)lysine cross-linking between gamma chains (stronger) and between alpha chains (weaker) of different monomers.PTM Phosphorylation sites are present in the extracelllular medium.DISEASE Defects in FGA are a cause of congenital afibrinogenemia (CAFBN) [MIM:202400]. This is a rare autosomal recessive disorder characterized by bleeding that varies from mild to severe and by complete absence or extremely low levels of plasma and platelet fibrinogen. Note=The majority of cases of afibrinogenemia are due to truncating mutations. Variations in position Arg-35 (the site of cleavage of fibrinopeptide a by thrombin) leads to alpha-dysfibrinogenemias.SIMILARITY Contains 1 fibrinogen C-terminal domain.\")\n	(DB_ID 54840)\n	(description \"recommendedName: Fibrinogen alpha chain  component recommendedName: Fibrinopeptide A  /component component recommendedName: Fibrinogen alpha chain  /component\")\n	(geneName \"FGA\")\n	(identifier \"P02671\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Blood coagulation\"\n		\"Coiled coil\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Isopeptide bond\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"FGA\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"FIBA_HUMAN\"\n		\"D3DP14\"\n		\"D3DP15\"\n		\"Q4QQH7\"\n		\"Q9BX62\"\n		\"Q9UCH2\")\n	(sequenceLength 866)\n	(species [GK_48887]))\n\n([GK_55564] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P06396 GSN recommendedName: Gelsolin  alternativeName: AGEL  alternativeName: Actin-depolymerizing factor shortName:ADF  alternativeName: Actin-depolymerizing factor shortName:ADF  alternativeName: Brevin \")\n	(checksum \"1\")\n	(comment \"TISSUE SPECIFICITY Phagocytic cells, platelets, fibroblasts, nonmuscle cells, smooth and skeletal muscle cells.PTM Phosphorylation on Tyr-86, Tyr-409, Tyr-465, Tyr-603 and Tyr-651 in vitro is induced in presence of phospholipids.SIMILARITY Belongs to the villin/gelsolin family.SIMILARITY Contains 6 gelsolin-like repeats.\")\n	(DB_ID 55564)\n	(description \"recommendedName: Gelsolin  alternativeName: AGEL  alternativeName: Actin-depolymerizing factor shortName:ADF  alternativeName: Actin-depolymerizing factor shortName:ADF  alternativeName: Brevin \")\n	(geneName \"GSN\")\n	(identifier \"P06396\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Actin capping\"\n		\"Actin-binding\"\n		\"Alternative initiation\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Calcium\"\n		\"Cilium biogenesis/degradation\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Cytoskeleton\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"GSN\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"GELS_HUMAN\"\n		\"Q8WVV7\")\n	(sequenceLength 782)\n	(species [GK_48887]))\n\n([GK_56134] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q99878 HIST1H2AJ recommendedName: Histone H2A type 1-J  alternativeName: Histone H2A/e \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Deiminated on Arg-4 in granulocytes upon calcium entry.PTM Phosphorylation on Ser-2 is enhanced during mitosis. Phosphorylation on Ser-2 by RPS6KA5/MSK1 directly represses transcription. Acetylation of H3 inhibits Ser-2 phosphorylation by RPS6KA5/MSK1.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 56134)\n	(description \"recommendedName: Histone H2A type 1-J  alternativeName: Histone H2A/e \")\n	(geneName\n		\"HIST1H2AJ\"\n		\"H2AFE\")\n	(identifier \"Q99878\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2AJ\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2A1J_HUMAN\"\n		\"A2RUU6\"\n		\"Q5JXQ5\")\n	(sequenceLength 128)\n	(species [GK_48887]))\n\n([GK_56136] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P20671 HIST1H2AD recommendedName: Histone H2A type 1-D  alternativeName: Histone H2A.3  alternativeName: Histone H2A/g \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Deiminated on Arg-4 in granulocytes upon calcium entry.PTM Phosphorylation on Ser-2 is enhanced during mitosis. Phosphorylation on Ser-2 by RPS6KA5/MSK1 directly represses transcription. Acetylation of H3 inhibits Ser-2 phosphorylation by RPS6KA5/MSK1.SIMILARITY Belongs to the histone H2A family.CAUTION Was originally (PubMed:2587222) thought to originate from mouse.\")\n	(DB_ID 56136)\n	(description \"recommendedName: Histone H2A type 1-D  alternativeName: Histone H2A.3  alternativeName: Histone H2A/g \")\n	(geneName\n		\"HIST1H2AD\"\n		\"H2AFG\")\n	(identifier \"P20671\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2AD\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2A1D_HUMAN\"\n		\"A0PK91\"\n		\"P57754\"\n		\"Q6FGY6\")\n	(sequenceLength 130)\n	(species [GK_48887]))\n\n([GK_56140] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q93077 HIST1H2AC recommendedName: Histone H2A type 1-C  alternativeName: Histone H2A/l \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Deiminated on Arg-4 in granulocytes upon calcium entry.PTM Phosphorylation on Ser-2 is enhanced during mitosis. Phosphorylation on Ser-2 by RPS6KA5/MSK1 directly represses transcription. Acetylation of H3 inhibits Ser-2 phosphorylation by RPS6KA5/MSK1.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 56140)\n	(description \"recommendedName: Histone H2A type 1-C  alternativeName: Histone H2A/l \")\n	(geneName\n		\"HIST1H2AC\"\n		\"H2AFL\")\n	(identifier \"Q93077\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2AC\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2A1C_HUMAN\"\n		\"B2R4F7\"\n		\"O00775\"\n		\"O00776\"\n		\"O00777\"\n		\"O00778\"\n		\"Q540R1\")\n	(sequenceLength 130)\n	(species [GK_48887]))\n\n([GK_56142] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P04908 HIST1H2AB recommendedName: Histone H2A type 1-B/E  alternativeName: Histone H2A.2  alternativeName: Histone H2A/a  alternativeName: Histone H2A/m \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Deiminated on Arg-4 in granulocytes upon calcium entry.PTM Phosphorylation on Ser-2 is enhanced during mitosis. Phosphorylation on Ser-2 by RPS6KA5/MSK1 directly represses transcription. Acetylation of H3 inhibits Ser-2 phosphorylation by RPS6KA5/MSK1.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 56142)\n	(description \"recommendedName: Histone H2A type 1-B/E  alternativeName: Histone H2A.2  alternativeName: Histone H2A/a  alternativeName: Histone H2A/m \")\n	(geneName\n		\"HIST1H2AB\"\n		\"H2AFM\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H2AE\"\n		\"H2AFA\")\n	(identifier \"P04908\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2AB\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2A1B_HUMAN\"\n		\"P28001\"\n		\"Q76P63\")\n	(sequenceLength 130)\n	(species [GK_48887]))\n\n([GK_56148] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q16777 HIST2H2AC recommendedName: Histone H2A type 2-C  alternativeName: Histone H2A-GL101  alternativeName: Histone H2A/q \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Deiminated on Arg-4 in granulocytes upon calcium entry.PTM Phosphorylation on Ser-2 is enhanced during mitosis. Phosphorylation on Ser-2 by RPS6KA5/MSK1 directly represses transcription. Acetylation of H3 inhibits Ser-2 phosphorylation by RPS6KA5/MSK1.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 56148)\n	(description \"recommendedName: Histone H2A type 2-C  alternativeName: Histone H2A-GL101  alternativeName: Histone H2A/q \")\n	(geneName\n		\"HIST2H2AC\"\n		\"H2AFQ\")\n	(identifier \"Q16777\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Citrullination\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST2H2AC\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2A2C_HUMAN\"\n		\"Q6DRA7\"\n		\"Q8IUE5\")\n	(sequenceLength 129)\n	(species [GK_48887]))\n\n([GK_56152] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P16104 H2AFX recommendedName: Histone H2A.x shortName:H2a/x \")\n	(checksum \"1\")\n	(comment \"DEVELOPMENTAL STAGE Synthesized in G1 as well as in S-phase.DOMAIN The [ST]-Q motif constitutes a recognition sequence for kinases from the PI3/PI4-kinase family.PTM Monoubiquitination of Lys-120 (H2AXK119ub) by RING1 and RNF2/RING2 complex gives a specific tag for epigenetic transcriptional repression. Following DNA double-strand breaks (DSBs), it is ubiquitinated through \'Lys-63\' linkage of ubiquitin moieties by the E2 ligase UBE2N and the E3 ligases RNF8 and RNF168, leading to the recruitment of repair proteins to sites of DNA damage. Monoubiquitination and ionizing radiation-induced \'Lys-63\'-linked ubiquitination are distinct events.SIMILARITY Belongs to the histone H2A family.\")\n	(DB_ID 56152)\n	(description \"recommendedName: Histone H2A.x shortName:H2a/x \")\n	(geneName\n		\"H2AFX\"\n		\"H2AX\")\n	(identifier \"P16104\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Cell cycle\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"DNA damage\"\n		\"DNA recombination\"\n		\"DNA repair\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Meiosis\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"H2AFX\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2AX_HUMAN\"\n		\"Q4ZGJ7\"\n		\"Q6IAS5\")\n	(sequenceLength 143)\n	(species [GK_48887]))\n\n([GK_56164] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P62807 HIST1H2BC recommendedName: Histone H2B type 1-C/E/F/G/I  alternativeName: Histone H2B.1 A  alternativeName: Histone H2B.a shortName:H2B/a  alternativeName: Histone H2B.a shortName:H2B/a  alternativeName: Histone H2B.g shortName:H2B/g  alternativeName: Histone H2B.g shortName:H2B/g  alternativeName: Histone H2B.h shortName:H2B/h  alternativeName: Histone H2B.h shortName:H2B/h  alternativeName: Histone H2B.k shortName:H2B/k  alternativeName: Histone H2B.k shortName:H2B/k  alternativeName: Histone H2B.l shortName:H2B/l  alternativeName: Histone H2B.l shortName:H2B/l \")\n	(checksum \"1\")\n	(comment \"SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56164)\n	(description \"recommendedName: Histone H2B type 1-C/E/F/G/I  alternativeName: Histone H2B.1 A  alternativeName: Histone H2B.a shortName:H2B/a  alternativeName: Histone H2B.a shortName:H2B/a  alternativeName: Histone H2B.g shortName:H2B/g  alternativeName: Histone H2B.g shortName:H2B/g  alternativeName: Histone H2B.h shortName:H2B/h  alternativeName: Histone H2B.h shortName:H2B/h  alternativeName: Histone H2B.k shortName:H2B/k  alternativeName: Histone H2B.k shortName:H2B/k  alternativeName: Histone H2B.l shortName:H2B/l  alternativeName: Histone H2B.l shortName:H2B/l \")\n	(geneName\n		\"HIST1H2BC\"\n		\"H2BFL\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H2BE\"\n		\"H2BFH\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H2BF\"\n		\"H2BFG\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H2BG\"\n		\"H2BFA\"\n		\"</gene>\"\n		\"<gene>\"\n		\"HIST1H2BI\"\n		\"H2BFK\")\n	(identifier \"P62807\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Antibiotic\"\n		\"Antimicrobial\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BC\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1C_HUMAN\"\n		\"P02278\"\n		\"Q3B872\"\n		\"Q4VB69\"\n		\"Q93078\"\n		\"Q93080\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56168] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P58876 HIST1H2BD recommendedName: Histone H2B type 1-D  alternativeName: HIRA-interacting protein 2  alternativeName: Histone H2B.1 B  alternativeName: Histone H2B.b shortName:H2B/b  alternativeName: Histone H2B.b shortName:H2B/b \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.MISCELLANEOUS The mouse orthologous protein seems not to exist.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56168)\n	(description \"recommendedName: Histone H2B type 1-D  alternativeName: HIRA-interacting protein 2  alternativeName: Histone H2B.1 B  alternativeName: Histone H2B.b shortName:H2B/b  alternativeName: Histone H2B.b shortName:H2B/b \")\n	(geneName\n		\"HIST1H2BD\"\n		\"H2BFB\"\n		\"HIRIP2\")\n	(identifier \"P58876\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BD\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier \"H2B1D_HUMAN\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56170] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q99880 HIST1H2BL recommendedName: Histone H2B type 1-L  alternativeName: Histone H2B.c shortName:H2B/c  alternativeName: Histone H2B.c shortName:H2B/c \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56170)\n	(description \"recommendedName: Histone H2B type 1-L  alternativeName: Histone H2B.c shortName:H2B/c  alternativeName: Histone H2B.c shortName:H2B/c \")\n	(geneName\n		\"HIST1H2BL\"\n		\"H2BFC\")\n	(identifier \"Q99880\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BL\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1L_HUMAN\"\n		\"B2R5A3\"\n		\"Q52LW9\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56172] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q99877 HIST1H2BN recommendedName: Histone H2B type 1-N  alternativeName: Histone H2B.d shortName:H2B/d  alternativeName: Histone H2B.d shortName:H2B/d \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56172)\n	(description \"recommendedName: Histone H2B type 1-N  alternativeName: Histone H2B.d shortName:H2B/d  alternativeName: Histone H2B.d shortName:H2B/d \")\n	(geneName\n		\"HIST1H2BN\"\n		\"H2BFD\")\n	(identifier \"Q99877\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BN\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1N_HUMAN\"\n		\"B2R5L4\"\n		\"Q494S8\"\n		\"Q96FB7\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56174] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q99879 HIST1H2BM recommendedName: Histone H2B type 1-M  alternativeName: Histone H2B.e shortName:H2B/e  alternativeName: Histone H2B.e shortName:H2B/e \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56174)\n	(description \"recommendedName: Histone H2B type 1-M  alternativeName: Histone H2B.e shortName:H2B/e  alternativeName: Histone H2B.e shortName:H2B/e \")\n	(geneName\n		\"HIST1H2BM\"\n		\"H2BFE\")\n	(identifier \"Q99879\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BM\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1M_HUMAN\"\n		\"Q6NWQ3\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56176] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P33778 HIST1H2BB recommendedName: Histone H2B type 1-B  alternativeName: Histone H2B.1  alternativeName: Histone H2B.f shortName:H2B/f  alternativeName: Histone H2B.f shortName:H2B/f \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56176)\n	(description \"recommendedName: Histone H2B type 1-B  alternativeName: Histone H2B.1  alternativeName: Histone H2B.f shortName:H2B/f  alternativeName: Histone H2B.f shortName:H2B/f \")\n	(geneName\n		\"HIST1H2BB\"\n		\"H2BFF\")\n	(identifier \"P33778\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BB\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1B_HUMAN\"\n		\"Q4KN36\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56180] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q93079 HIST1H2BH recommendedName: Histone H2B type 1-H  alternativeName: Histone H2B.j shortName:H2B/j  alternativeName: Histone H2B.j shortName:H2B/j \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56180)\n	(description \"recommendedName: Histone H2B type 1-H  alternativeName: Histone H2B.j shortName:H2B/j  alternativeName: Histone H2B.j shortName:H2B/j \")\n	(geneName\n		\"HIST1H2BH\"\n		\"H2BFJ\")\n	(identifier \"Q93079\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BH\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1H_HUMAN\"\n		\"B2R541\"\n		\"Q4VB74\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56184] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P23527 HIST1H2BO recommendedName: Histone H2B type 1-O  alternativeName: Histone H2B.2  alternativeName: Histone H2B.n shortName:H2B/n  alternativeName: Histone H2B.n shortName:H2B/n \")\n	(checksum \"1\")\n	(comment \"FUNCTION Core component of nucleosome. Nucleosomes wrap and compact DNA into chromatin, limiting DNA accessibility to the cellular machineries which require DNA as a template. Histones thereby play a central role in transcription regulation, DNA repair, DNA replication and chromosomal stability. DNA accessibility is regulated via a complex set of post-translational modifications of histones, also called histone code, and nucleosome remodeling.SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.MISCELLANEOUS The mouse orthologous protein seems not to exist.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56184)\n	(description \"recommendedName: Histone H2B type 1-O  alternativeName: Histone H2B.2  alternativeName: Histone H2B.n shortName:H2B/n  alternativeName: Histone H2B.n shortName:H2B/n \")\n	(geneName\n		\"HIST1H2BO\"\n		\"H2BFH\"\n		\"H2BFN\")\n	(identifier \"P23527\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BO\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1O_HUMAN\"\n		\"Q3KPI7\"\n		\"Q8TCV6\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56186] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q16778 HIST2H2BE recommendedName: Histone H2B type 2-E  alternativeName: Histone H2B-GL105  alternativeName: Histone H2B.q shortName:H2B/q  alternativeName: Histone H2B.q shortName:H2B/q \")\n	(checksum \"1\")\n	(comment \"SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56186)\n	(description \"recommendedName: Histone H2B type 2-E  alternativeName: Histone H2B-GL105  alternativeName: Histone H2B.q shortName:H2B/q  alternativeName: Histone H2B.q shortName:H2B/q \")\n	(geneName\n		\"HIST2H2BE\"\n		\"H2BFQ\")\n	(identifier \"Q16778\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Antibiotic\"\n		\"Antimicrobial\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST2H2BE\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B2E_HUMAN\"\n		\"A3KMC7\"\n		\"A8K110\"\n		\"Q4KMY1\"\n		\"Q5QNX0\"\n		\"Q9UE88\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56188] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P06899 HIST1H2BJ recommendedName: Histone H2B type 1-J  alternativeName: Histone H2B.1  alternativeName: Histone H2B.r shortName:H2B/r  alternativeName: Histone H2B.r shortName:H2B/r \")\n	(checksum \"1\")\n	(comment \"SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56188)\n	(description \"recommendedName: Histone H2B type 1-J  alternativeName: Histone H2B.1  alternativeName: Histone H2B.r shortName:H2B/r  alternativeName: Histone H2B.r shortName:H2B/r \")\n	(geneName\n		\"HIST1H2BJ\"\n		\"H2BFR\")\n	(identifier \"P06899\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Antibiotic\"\n		\"Antimicrobial\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"HIST1H2BJ\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2B1J_HUMAN\"\n		\"B2R4J4\"\n		\"O60816\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56190] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P57053 H2BFS recommendedName: Histone H2B type F-S  alternativeName: Histone H2B.s shortName:H2B/s  alternativeName: Histone H2B.s shortName:H2B/s \")\n	(checksum \"1\")\n	(comment \"SUBUNIT The nucleosome is a histone octamer containing two molecules each of H2A, H2B, H3 and H4 assembled in one H3-H4 heterotetramer and two H2A-H2B heterodimers. The octamer wraps approximately 147 bp of DNA.PTM Monoubiquitination of Lys-121 by the RNF20/40 complex gives a specific tag for epigenetic transcriptional activation and is also prerequisite for histone H3 \'Lys-4\' and \'Lys-79\' methylation. It also functions cooperatively with the FACT dimer to stimulate elongation by RNA polymerase II.SIMILARITY Belongs to the histone H2B family.\")\n	(DB_ID 56190)\n	(description \"recommendedName: Histone H2B type F-S  alternativeName: Histone H2B.s shortName:H2B/s  alternativeName: Histone H2B.s shortName:H2B/s \")\n	(geneName \"H2BFS\")\n	(identifier \"P57053\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acetylation\"\n		\"Antibiotic\"\n		\"Antimicrobial\"\n		\"Chromosome\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"DNA-binding\"\n		\"Isopeptide bond\"\n		\"Methylation\"\n		\"Nucleosome core\"\n		\"Nucleus\"\n		\"Phosphoprotein\"\n		\"Ubl conjugation\")\n	(name \"H2BFS\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"H2BFS_HUMAN\"\n		\"A0AVF9\"\n		\"Q5R2W0\")\n	(sequenceLength 126)\n	(species [GK_48887]))\n\n([GK_56978] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P10997 IAPP recommendedName: Islet amyloid polypeptide  alternativeName: Amylin  alternativeName: Diabetes-associated peptide shortName:DAP  alternativeName: Diabetes-associated peptide shortName:DAP  alternativeName: Insulinoma amyloid peptide \")\n	(checksum \"1\")\n	(comment \"FUNCTION Selectively inhibits insulin-stimulated glucose utilization and glycogen deposition in muscle, while not affecting adipocyte glucose metabolism.PTM Amyloid fibrils are degraded by IDE.MISCELLANEOUS IAPP is the peptide subunit of amyloid found in pancreatic islets of type 2 diabetic patients and in insulinomas.SIMILARITY Belongs to the calcitonin family.\")\n	(DB_ID 56978)\n	(description \"recommendedName: Islet amyloid polypeptide  alternativeName: Amylin  alternativeName: Diabetes-associated peptide shortName:DAP  alternativeName: Diabetes-associated peptide shortName:DAP  alternativeName: Insulinoma amyloid peptide \")\n	(geneName \"IAPP\")\n	(identifier \"P10997\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Amidation\"\n		\"Amyloid\"\n		\"Cleavage on pair of basic residues\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Hormone\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"IAPP\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"IAPP_HUMAN\"\n		\"Q0ZD87\"\n		\"Q14598\")\n	(sequenceLength 89)\n	(species [GK_48887]))\n\n([GK_57031] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CASP8(1-479) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 57031)\n	(endCoordinate 479)\n	(name\n		\"CASP8(1-479)\"\n		\"Caspase-8 precursor \"\n		\"ICE-like apoptotic protease 5\"\n		\"MORT1-associated CED-3 homolog\"\n		\"MACH\"\n		\"FADD-homologous ICE/CED-3- like protease\"\n		\"FADD-like ICE\"\n		\"FLICE\"\n		\"Apoptotic cysteine protease\"\n		\"Apoptotic protease Mch-5\"\n		\"CAP4\")\n	(referenceEntity [GK_57032])\n	(species [GK_48887])\n	(stableIdentifier [GK_362980])\n	(startCoordinate 1))\n\n([GK_57032] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q14790 CASP8 recommendedName: Caspase-8 shortName:CASP-8 ecNumber3.4.22.61/ecNumber  alternativeName: Apoptotic cysteine protease  alternativeName: Apoptotic protease Mch-5  alternativeName: CAP4  alternativeName: FADD-homologous ICE/ced-3-like protease  alternativeName: FADD-like ICE shortName:FLICE  alternativeName: ICE-like apoptotic protease 5  alternativeName: MORT1-associated ced-3 homolog shortName:MACH  component recommendedName: Caspase-8 subunit p18  /component component recommendedName: Caspase-8 subunit p10  /component\")\n	(checksum \"7A5FEAA6B39B582F\")\n	(comment \"FUNCTION Most upstream protease of the activation cascade of caspases responsible for the TNFRSF6/FAS mediated and TNFRSF1A induced cell death. Binding to the adapter molecule FADD recruits it to either receptor. The resulting aggregate called death-inducing signaling complex (DISC) performs CASP8 proteolytic activation. The active dimeric enzyme is then liberated from the DISC and free to activate downstream apoptotic proteases. Proteolytic fragments of the N-terminal propeptide (termed CAP3, CAP5 and CAP6) are likely retained in the DISC. Cleaves and activates CASP3, CASP4, CASP6, CASP7, CASP9 and CASP10. May participate in the GZMB apoptotic pathways. Cleaves ADPRT. Hydrolyzes the small-molecule substrate, Ac-Asp-Glu-Val-Asp-|-AMC. Likely target for the cowpox virus CRMA death inhibitory protein. Isoform 5, isoform 6, isoform 7 and isoform 8 lack the catalytic site and may interfere with the pro-apoptotic activity of the complex.CATALYTIC ACTIVITY Strict requirement for Asp at position P1 and has a preferred cleavage sequence of (Leu/Asp/Val)-Glu-Thr-Asp-|-(Gly/Ser/Ala).ENZYME REGULATION Inhibited by the effector protein NleF that is produced by pathogenic E.coli; this inhibits apoptosis.SUBUNIT Heterotetramer that consists of two anti-parallel arranged heterodimers, each one formed by a 18 kDa (p18) and a 10 kDa (p10) subunit. Interacts with FADD, CFLAR and PEA15. Isoform 9 interacts at the endoplasmic reticulum with a complex containing BCAP31, BAP29, BCL2 and/or BCL2L1. Interacts with TNFAIP8L2 (By similarity). Interacts with CASP8AP2. Interacts with human cytomegalovirus/HHV-5 protein vICA/UL36; this interaction inhibits CASP8 activation. Interacts with NleF from pathogenic E.coli.TISSUE SPECIFICITY Isoform 1, isoform 5 and isoform 7 are expressed in a wide variety of tissues. Highest expression in peripheral blood leukocytes, spleen, thymus and liver. Barely detectable in brain, testis and skeletal muscle.DOMAIN Isoform 9 contains a N-terminal extension that is required for interaction with the BCAP31 complex.PTM Generation of the subunits requires association with the death-inducing signaling complex (DISC), whereas additional processing is likely due to the autocatalytic activity of the activated protease. GZMB and CASP10 can be involved in these processing events.PTM Phosphorylation on Ser-387 during mitosis by CDK1 inhibits activation by proteolysis and prevents apoptosis. This phosphorylation occurs in cancer cell lines, as well as in primary breast tissues and lymphocytes.POLYMORPHISM Genetic variations in CASP8 are associated with reduced risk of lung cancer [MIM:211980] in a population of Han Chinese subjects. Genetic variations are also associated with decreased risk of cancer of various other forms including esophageal, gastric, colorectal, cervical, and breast, acting in an allele dose-dependent manner.SIMILARITY Belongs to the peptidase C14A family.SIMILARITY Contains 2 DED (death effector) domains.\")\n	(DB_ID 57032)\n	(description \"recommendedName: Caspase-8 shortName:CASP-8 ecNumber3.4.22.61/ecNumber  alternativeName: Apoptotic cysteine protease  alternativeName: Apoptotic protease Mch-5  alternativeName: CAP4  alternativeName: FADD-homologous ICE/ced-3-like protease  alternativeName: FADD-like ICE shortName:FLICE  alternativeName: ICE-like apoptotic protease 5  alternativeName: MORT1-associated ced-3 homolog shortName:MACH  component recommendedName: Caspase-8 subunit p18  /component component recommendedName: Caspase-8 subunit p10  /component\")\n	(geneName\n		\"CASP8\"\n		\"MCH5\")\n	(identifier \"Q14790\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Host-virus interaction\"\n		\"Hydrolase\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Protease\"\n		\"Reference proteome\"\n		\"Repeat\"\n		\"Thiol protease\"\n		\"Zymogen\")\n	(name \"CASP8\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CASP8_HUMAN\"\n		\"O14676\"\n		\"Q14791\"\n		\"Q14792\"\n		\"Q14793\"\n		\"Q14794\"\n		\"Q14795\"\n		\"Q14796\"\n		\"Q15780\"\n		\"Q15806\"\n		\"Q53TT5\"\n		\"Q8TDI1\"\n		\"Q8TDI2\"\n		\"Q8TDI3\"\n		\"Q8TDI4\"\n		\"Q8TDI5\"\n		\"Q96T22\"\n		\"Q9C0K4\"\n		\"Q9UQ81\")\n	(sequenceLength 479)\n	(species [GK_48887]))\n\n([GK_57035] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CASP10(220-415) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 57035)\n	(endCoordinate 415)\n	(name\n		\"CASP10(220-415)\"\n		\"Caspase-10 precursor \"\n		\"ICE-like apoptotic protease 4\"\n		\"Apoptotic protease Mch-4\"\n		\"FAS-associated death domain protein interleukin-1B-converting enzyme 2\"\n		\"FLICE2\")\n	(referenceEntity [GK_57036])\n	(species [GK_48887])\n	(stableIdentifier [GK_370163])\n	(startCoordinate 220))\n\n([GK_57036] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q92851 CASP10 recommendedName: Caspase-10 shortName:CASP-10 ecNumber3.4.22.63/ecNumber  alternativeName: Apoptotic protease Mch-4  alternativeName: FAS-associated death domain protein interleukin-1B-converting enzyme 2 shortName:FLICE2  alternativeName: ICE-like apoptotic protease 4  component recommendedName: Caspase-10 subunit p23/17  /component component recommendedName: Caspase-10 subunit p12  /component\")\n	(checksum \"840348AE602B8243\")\n	(comment \"FUNCTION Involved in the activation cascade of caspases responsible for apoptosis execution. Recruited to both Fas- and TNFR-1 receptors in a FADD dependent manner. May participate in the granzyme B apoptotic pathways. Cleaves and activates caspase-3, -4, -6, -7, -8, and -9. Hydrolyzes the small- molecule substrates, Tyr-Val-Ala-Asp-|-AMC and Asp-Glu-Val-Asp-|-AMC.FUNCTION Isoform C is proteolytically inactive.CATALYTIC ACTIVITY Strict requirement for Asp at position P1 and has a preferred cleavage sequence of Leu-Gln-Thr-Asp-|-Gly.SUBUNIT Heterotetramer that consists of two anti-parallel arranged heterodimers, each one formed by a 23/17 kDa (p23/17) (depending on the splicing events) and a 12 kDa (p12) subunit (By similarity). Self-associates. Interacts with FADD and CASP8. Found in a Fas signaling complex consisting of FAS, FADD, CASP8 and CASP10.TISSUE SPECIFICITY Detectable in most tissues. Lowest expression is seen in brain, kidney, prostate, testis and colon.PTM Cleavage by granzyme B and autocatalytic activity generate the two active subunits.SIMILARITY Belongs to the peptidase C14A family.SIMILARITY Contains 2 DED (death effector) domains.\")\n	(DB_ID 57036)\n	(description \"recommendedName: Caspase-10 shortName:CASP-10 ecNumber3.4.22.63/ecNumber  alternativeName: Apoptotic protease Mch-4  alternativeName: FAS-associated death domain protein interleukin-1B-converting enzyme 2 shortName:FLICE2  alternativeName: ICE-like apoptotic protease 4  component recommendedName: Caspase-10 subunit p23/17  /component component recommendedName: Caspase-10 subunit p12  /component\")\n	(geneName\n		\"CASP10\"\n		\"MCH4\")\n	(identifier \"Q92851\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Hydrolase\"\n		\"Polymorphism\"\n		\"Protease\"\n		\"Reference proteome\"\n		\"Repeat\"\n		\"Thiol protease\"\n		\"Zymogen\")\n	(name \"CASP10\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"CASPA_HUMAN\"\n		\"Q68HC0\"\n		\"Q6KF62\"\n		\"Q6KF63\"\n		\"Q8IUP5\"\n		\"Q8WYQ8\"\n		\"Q99845\"\n		\"Q9Y2U6\"\n		\"Q9Y2U7\")\n	(sequenceLength 521)\n	(species [GK_48887]))\n\n([GK_571213] of  Person\n\n	(_displayName \"Zhou, H\")\n	(DB_ID 571213)\n	(firstname \"Huaijun\")\n	(initial \"H\")\n	(surname \"Zhou\"))\n\n([GK_57415] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P01308 INS recommendedName: Insulin  component recommendedName: Insulin B chain  /component component recommendedName: Insulin A chain  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Insulin decreases blood glucose concentration. It increases cell permeability to monosaccharides, amino acids and fatty acids. It accelerates glycolysis, the pentose phosphate cycle, and glycogen synthesis in liver.SUBUNIT Heterodimer of a B chain and an A chain linked by two disulfide bonds.PHARMACEUTICAL Available under the names Humulin or Humalog (Eli Lilly) and Novolin (Novo Nordisk). Used in the treatment of diabetes. Humalog is an insulin analog with 52-Lys-Pro-53 instead of 52-Pro-Lys-53.SIMILARITY Belongs to the insulin family.\")\n	(DB_ID 57415)\n	(description \"recommendedName: Insulin  component recommendedName: Insulin B chain  /component component recommendedName: Insulin A chain  /component\")\n	(geneName \"INS\")\n	(identifier \"P01308\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Carbohydrate metabolism\"\n		\"Cleavage on pair of basic residues\"\n		\"Complete proteome\"\n		\"Diabetes mellitus\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glucose metabolism\"\n		\"Hormone\"\n		\"Pharmaceutical\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"INS\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"INS_HUMAN\"\n		\"Q5EEX2\")\n	(sequenceLength 110)\n	(species [GK_48887]))\n\n([GK_57627] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q9Y287 ITM2B recommendedName: Integral membrane protein 2B  alternativeName: Protein E25B  alternativeName: Transmembrane protein BRI  component recommendedName: ABri/ADan amyloid peptide  /component\")\n	(checksum \"1\")\n	(comment \"TISSUE SPECIFICITY Expressed in brain and in other tissues.DISEASE Defects in ITM2B are a cause of cerebral amyloid angiopathy ITM2B-related type 2 (CAA-ITM2B2) [MIM:117300]; also known as heredopathia ophthalmo-oto-encephalica. A disorder characterized by amyloid deposition in the walls of the blood vessels of the cerebrum, choroid plexus, cerebellum, spinal cord and retina. Plaques and neurofibrillary tangles are observed in the hippocampus. Clinical features include progressive ataxia, dementia, cataracts and deafness.SIMILARITY Belongs to the ITM2 family.SIMILARITY Contains 1 BRICHOS domain.\")\n	(DB_ID 57627)\n	(description \"recommendedName: Integral membrane protein 2B  alternativeName: Protein E25B  alternativeName: Transmembrane protein BRI  component recommendedName: ABri/ADan amyloid peptide  /component\")\n	(geneName\n		\"ITM2B\"\n		\"BRI\"\n		\"BRI2\")\n	(identifier \"Q9Y287\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Cell membrane\"\n		\"Cleavage on pair of basic residues\"\n		\"Complete proteome\"\n		\"Deafness\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Golgi apparatus\"\n		\"Membrane\"\n		\"Signal-anchor\"\n		\"Transmembrane\"\n		\"Transmembrane helix\")\n	(name \"ITM2B\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"ITM2B_HUMAN\"\n		\"Q9NYH1\")\n	(sequenceLength 266)\n	(species [GK_48887]))\n\n([GK_58841] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P61626 LYZ  Lysozyme C  alternativeName: 1,4-beta-N-acetylmuramidase C \")\n	(checksum \"1\")\n	(comment \"FUNCTION Lysozymes have primarily a bacteriolytic function; those in tissues and body fluids are associated with the monocyte-macrophage system and enhance the activity of immunoagents.CATALYTIC ACTIVITY Hydrolysis of (1-&gt;4)-beta-linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues in a peptidoglycan and between N-acetyl-D-glucosamine residues in chitodextrins.SUBUNIT Monomer.MISCELLANEOUS Lysozyme C is capable of both hydrolysis and transglycosylation; it shows also a slight esterase activity. It acts rapidly on both peptide-substituted and unsubstituted peptidoglycan, and slowly on chitin oligosaccharides.SIMILARITY Belongs to the glycosyl hydrolase 22 family.\")\n	(DB_ID 58841)\n	(description \" Lysozyme C  alternativeName: 1,4-beta-N-acetylmuramidase C \")\n	(geneName\n		\"LYZ\"\n		\"LZM\")\n	(identifier \"P61626\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Antimicrobial\"\n		\"Bacteriolytic enzyme\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycosidase\"\n		\"Hydrolase\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"LYZ\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"LYSC_HUMAN\"\n		\"P00695\"\n		\"Q13170\"\n		\"Q9UCF8\")\n	(sequenceLength 148)\n	(species [GK_48887]))\n\n([GK_59185] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q08431 MFGE8 recommendedName: Lactadherin  alternativeName: Breast epithelial antigen BA46  alternativeName: HMFG  alternativeName: MFGM  alternativeName: Milk fat globule-EGF factor 8 shortName:MFG-E8  alternativeName: Milk fat globule-EGF factor 8 shortName:MFG-E8  alternativeName: SED1  component recommendedName: Lactadherin short form  /component component recommendedName: Medin  /component\")\n	(checksum \"1\")\n	(comment \"TISSUE SPECIFICITY Mammary epithelial cell surfaces and aortic media. Overexpressed in several carcinomas.PTM Medin has a ragged N-terminus with minor species starting at Pro-264 and Gly-273.SIMILARITY Contains 1 EGF-like domain.SIMILARITY Contains 2 F5/8 type C domains.\")\n	(DB_ID 59185)\n	(description \"recommendedName: Lactadherin  alternativeName: Breast epithelial antigen BA46  alternativeName: HMFG  alternativeName: MFGM  alternativeName: Milk fat globule-EGF factor 8 shortName:MFG-E8  alternativeName: Milk fat globule-EGF factor 8 shortName:MFG-E8  alternativeName: SED1  component recommendedName: Lactadherin short form  /component component recommendedName: Medin  /component\")\n	(geneName \"MFGE8\")\n	(identifier \"Q08431\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Alternative splicing\"\n		\"Amyloid\"\n		\"Angiogenesis\"\n		\"Cell adhesion\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"EGF-like domain\"\n		\"Fertilization\"\n		\"Glycoprotein\"\n		\"Host-virus interaction\"\n		\"Membrane\"\n		\"Polymorphism\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"MFGE8\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"MFGM_HUMAN\"\n		\"B2R6M7\"\n		\"Q53FU9\"\n		\"Q7Z3D2\"\n		\"Q9BTL9\")\n	(sequenceLength 387)\n	(species [GK_48887]))\n\n([GK_6] of  ReferenceDatabase\n\n	(_displayName \"COMPOUND\")\n	(accessUrl \"http://www.genome.ad.jp/dbget-bin/www_bget?cpd:###ID###\")\n	(DB_ID 6)\n	(name \"COMPOUND\")\n	(url \"http://www.genome.ad.jp/kegg/\"))\n\n([GK_611062] of  PsiMod\n\n	(_displayName \"L-lysine residue [MOD:00021]\")\n	(DB_ID 611062)\n	(definition \"A protein modification that effectively converts a source amino acid residue to L-lysine.\")\n	(identifier \"00021\")\n	(name \"L-lysine residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2,6-diaminohexanoic acid\"\n		\"Lys\"\n		\"alpha,epsilon-diaminocaproic acid\"))\n\n([GK_611064] of  PsiMod\n\n	(_displayName \"L-asparagine removal [MOD:01633]\")\n	(DB_ID 611064)\n	(definition \"A protein modification that effectively removes or replaces an L-asparagine.\")\n	(identifier \"01633\")\n	(name \"L-asparagine removal\")\n	(referenceDatabase [GK_437354]))\n\n([GK_611065] of  PsiMod\n\n	(_displayName \"L-lysine removal [MOD:01642]\")\n	(DB_ID 611065)\n	(definition \"A protein modification that effectively removes or replaces an L-lysine.\")\n	(identifier \"01642\")\n	(name \"L-lysine removal\")\n	(referenceDatabase [GK_437354]))\n\n([GK_611066] of  PsiMod\n\n	(_displayName \"L-arginine residue [MOD:00011]\")\n	(DB_ID 611066)\n	(definition \"A protein modification that effectively converts a source amino acid residue to an L-arginine.\")\n	(identifier \"00011\")\n	(name \"L-arginine residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-amino-5-(carbamimidamido)pentanoic acid\"\n		\"Arg\"))\n\n([GK_611067] of  PsiMod\n\n	(_displayName \"L-glutamic acid residue [MOD:00015]\")\n	(DB_ID 611067)\n	(definition \"A protein modification that effectively converts a source amino acid residue to an L-glutamic acid.\")\n	(identifier \"00015\")\n	(name \"L-glutamic acid residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-aminopentanedioic acid\"\n		\"Glu\"))\n\n([GK_611068] of  PsiMod\n\n	(_displayName \"glycine residue [MOD:00017]\")\n	(DB_ID 611068)\n	(definition \"A protein modification that effectively converts a source amino acid residue to a glycine.\")\n	(identifier \"00017\")\n	(name \"glycine residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"aminoethanoic acid\"\n		\"Gly\"))\n\n([GK_611069] of  PsiMod\n\n	(_displayName \"L-proline residue [MOD:00024]\")\n	(DB_ID 611069)\n	(definition \"A protein modification that effectively converts a source amino acid residue to L-proline.\")\n	(identifier \"00024\")\n	(name \"L-proline residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-pyrrolidinecarboxylic acid\"\n		\"Pro\"\n		\"pyrrolidine-2-carboxylic acid\"))\n\n([GK_611070] of  PsiMod\n\n	(_displayName \"L-aspartic acid residue [MOD:00013]\")\n	(DB_ID 611070)\n	(definition \"A protein modification that effectively converts a source amino acid residue to an L-aspartic acid.\")\n	(identifier \"00013\")\n	(name \"L-aspartic acid residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-aminobutanedioic acid\"\n		\"Asp\"\n		\"aminosuccinic acid\"))\n\n([GK_611072] of  PsiMod\n\n	(_displayName \"L-proline removal [MOD:01645]\")\n	(DB_ID 611072)\n	(definition \"A protein modification that effectively removes or replaces an L-proline.\")\n	(identifier \"01645\")\n	(name \"L-proline removal\")\n	(referenceDatabase [GK_437354]))\n\n([GK_611082] of  PsiMod\n\n	(_displayName \"L-threonine residue [MOD:00026]\")\n	(DB_ID 611082)\n	(definition \"A protein modification that effectively converts a source amino acid residue to L-threonine.\")\n	(identifier \"00026\")\n	(name \"L-threonine residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(2S,3R)-2-amino-3-hydroxybutanoic acid\"\n		\"Thr\"))\n\n([GK_611087] of  PsiMod\n\n	(_displayName \"L-threonine removal [MOD:01647]\")\n	(DB_ID 611087)\n	(definition \"A protein modification that effectively removes or replaces an L-threonine.\")\n	(identifier \"01647\")\n	(name \"L-threonine removal\")\n	(referenceDatabase [GK_437354]))\n\n([GK_611088] of  PsiMod\n\n	(_displayName \"L-glutamine residue [MOD:00016]\")\n	(DB_ID 611088)\n	(definition \"A protein modification that effectively converts a source amino acid residue to an L-glutamine.\")\n	(identifier \"00016\")\n	(name \"L-glutamine residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-amino-5-pentanediamic acid\"\n		\"Gln\"))\n\n([GK_611093] of  PsiMod\n\n	(_displayName \"L-leucine residue [MOD:00020]\")\n	(DB_ID 611093)\n	(definition \"A protein modification that effectively converts a source amino acid residue to an L-leucine.\")\n	(identifier \"00020\")\n	(name \"L-leucine residue\")\n	(referenceDatabase [GK_437354])\n	(synonym\n		\"(S)-2-amino-4-methylpentanoic acid\"\n		\"Leu\"))\n\n([GK_61584] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P98160 HSPG2 recommendedName: Basement membrane-specific heparan sulfate proteoglycan core protein shortName:HSPG  alternativeName: Perlecan shortName:PLC  alternativeName: Perlecan shortName:PLC  component recommendedName: Endorepellin  /component component recommendedName: LG3 peptide  /component\")\n	(checksum \"1\")\n	(comment \"TISSUE SPECIFICITY Found in the basement membranes.PTM Proteolytic processing produces the C-terminal angiogenic peptide, endorepellin. This peptide can be further processed to produce the LG3 peptide.MISCELLANEOUS The LG3 peptide has been found in the urine of patients with end-stage renal disease and in the amniotic fluid of pregnant women with premature rupture of fetal membranes.SIMILARITY Contains 4 EGF-like domains.SIMILARITY Contains 22 Ig-like C2-type (immunoglobulin-like) domains.SIMILARITY Contains 11 laminin EGF-like domains.SIMILARITY Contains 3 laminin G-like domains.SIMILARITY Contains 3 laminin IV type A domains.SIMILARITY Contains 4 LDL-receptor class A domains.SIMILARITY Contains 1 SEA domain.\")\n	(DB_ID 61584)\n	(description \"recommendedName: Basement membrane-specific heparan sulfate proteoglycan core protein shortName:HSPG  alternativeName: Perlecan shortName:PLC  alternativeName: Perlecan shortName:PLC  component recommendedName: Endorepellin  /component component recommendedName: LG3 peptide  /component\")\n	(geneName \"HSPG2\")\n	(identifier \"P98160\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Angiogenesis\"\n		\"Basement membrane\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"EGF-like domain\"\n		\"Extracellular matrix\"\n		\"Glycoprotein\"\n		\"Heparan sulfate\"\n		\"Immunoglobulin domain\"\n		\"Laminin EGF-like domain\"\n		\"Polymorphism\"\n		\"Proteoglycan\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"HSPG2\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"PGBM_HUMAN\"\n		\"Q16287\"\n		\"Q5SZI3\"\n		\"Q9H3V5\")\n	(sequenceLength 4391)\n	(species [GK_48887]))\n\n([GK_6192] of  GO_BiologicalProcess\n\n	(_displayName \"regulation of signal transduction\")\n	(accession \"0009966\")\n	(DB_ID 6192)\n	(definition \"Any process that modulates the frequency, rate or extent of signal transduction.\")\n	(instanceOf\n		[GK_376482]\n		[GK_528243]\n		[GK_171348])\n	(name \"regulation of signal transduction\")\n	(referenceDatabase [GK_1])\n	(regulate [GK_141]))\n\n([GK_6196] of  GO_BiologicalProcess\n\n	(_displayName \"negative regulation of signal transduction\")\n	(accession \"0009968\")\n	(DB_ID 6196)\n	(definition \"Any process that stops, prevents, or reduces the frequency, rate or extent of signal transduction.\")\n	(instanceOf\n		[GK_6192]\n		[GK_376605]\n		[GK_528249]\n		[GK_171449])\n	(name \"negative regulation of signal transduction\")\n	(negativelyRegulate [GK_141])\n	(referenceDatabase [GK_1]))\n\n([GK_62178] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P01236 PRL recommendedName: Prolactin shortName:PRL \")\n	(checksum \"1\")\n	(comment \"FUNCTION Prolactin acts primarily on the mammary gland by promoting lactation.SIMILARITY Belongs to the somatotropin/prolactin family.\")\n	(DB_ID 62178)\n	(description \"recommendedName: Prolactin shortName:PRL \")\n	(geneName \"PRL\")\n	(identifier \"P01236\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Hormone\"\n		\"Lactation\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"PRL\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"PRL_HUMAN\"\n		\"Q15199\"\n		\"Q92996\")\n	(sequenceLength 227)\n	(species [GK_48887]))\n\n([GK_63118] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] RIP [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 63118)\n	(endCoordinate -1)\n	(name\n		\"RIP\"\n		\"Receptor-interacting serine/threonine protein kinase 2 (EC 2.7.1.-) (Serine/threonine protein kinase RIP) (Cell death protein RIP) (Receptor interacting protein)\"\n		\"Receptor-interacting serine/threonine protein kinase 2 \"\n		\"Serine/threonine protein kinase RIP\"\n		\"Cell death protein RIP\"\n		\"Receptor interacting protein\")\n	(referenceEntity [GK_63119])\n	(species [GK_48887])\n	(stableIdentifier [GK_367393])\n	(startCoordinate 1))\n\n([GK_63119] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q13546 RIPK1 recommendedName: Receptor-interacting serine/threonine-protein kinase 1 ecNumber2.7.11.1/ecNumber  alternativeName: Cell death protein RIP  alternativeName: Receptor-interacting protein 1 shortName:RIP-1  alternativeName: Serine/threonine-protein kinase RIP \")\n	(checksum \"976E2428D525A9B2\")\n	(comment \"FUNCTION Serine-threonine kinase which transduces inflammatory and cell-death signals (programmed necrosis) following death receptors ligation, activation of pathogen recognition receptors (PRRs), and DNA damage. Upon activation of TNFR1 by the TNF-alpha family cytokines, TRADD and TRAF2 are recruited to the receptor. Phosphorylates DAB2IP at \'Ser-728\' in a TNF-alpha-dependent manner, and thereby activates the MAP3K5-JNK apoptotic cascade. Ubiquitination by TRAF2 via \'Lys-63\'-link chains acts as a critical enhancer of communication with downstream signal transducers in the mitogen-activated protein kinase pathway and the NF-kappa-B pathway, which in turn mediate downstream events including the activation of genes encoding inflammatory molecules. Polyubiquitinated protein binds to IKBKG/NEMO, the regulatory subunit of the IKK complex, a critical event for NF-kappa-B activation. Interaction with other cellular RHIM-containing adapters initiates gene activation and cell death. RIPK1 and RIPK3 association, in particular, forms a necrosis-inducing complex.CATALYTIC ACTIVITY ATP + a protein = ADP + a phosphoprotein.ENZYME REGULATION Inhibited by necrostatins, including necrostatin-1, necrostatin-3 and necrostatin-4.SUBUNIT Interacts (via RIP homotypic interaction motif) with RIPK3 (via RIP homotypic interaction motif). Upon TNF-induced necrosis, the RIPK1-RIPK3 dimer further interacts with PGAM5 and MLKL; the formation of this complex leads to PGAM5 phosphorylation and increase in PGAM5 phosphatase activity. Interacts (via the death domain) with TNFRSF6 (via the death domain) and TRADD (via the death domain). Is recruited by TRADD to TNFRSF1A in a TNF-dependent process. Binds RNF216, EGFR, IKBKG, TRAF1, TRAF2 and TRAF3. Interacts with BNLF1. Interacts with SQSTM1 upon TNF-alpha stimulation. May interact with MAVS/IPS1. Interacts with ZFAND5. Interacts with RBCK1 (By similarity). Interacts with BIRC2/c-IAP1, BIRC3/c-IAP2 and XIAP/BIRC4. Upon TNF-induced necrosis, forms in complex with PGAM5, RIPK3 and MLKL. Interacts (via kinase domain) with DAB2IP (via Ras-GAP domain); the interaction occurs in a TNF-alpha-dependent manner.DOMAIN Contains a C-terminal death domain (DD) that engages other DD-containing proteins as well as a central (intermediate) region important for NF-kB activation and RHIM-dependent signaling.PTM Proteolytically cleaved by caspase-8 during TNF-induced apoptosis. Cleavage abolishes NF-kappa-B activation and enhances pro-apoptotic signaling through the TRADD-FADD interaction.PTM RIPK1 and RIPK3 undergo reciprocal auto- and trans-phosphorylation. Phosphorylation of Ser-161 by RIPK3 is necessary for the formation of the necroptosis-inducing complex.PTM Ubiquitinated by \'Lys-11\'-, \'Lys-48\'-, \'Lys-63\'- and linear-linked type ubiquitin. Polyubiquitination with \'Lys-63\'-linked chains by TRAF2 induces association with the IKK complex. Deubiquitination of \'Lys-63\'-linked chains and polyubiquitination with \'Lys-48\'-linked chains by TNFAIP3 leads to RIPK1 proteasomal degradation and consequently down-regulates TNF-alpha-induced NFkappa-B signaling. Linear polyubiquitinated; the head-to-tail polyubiquitination is mediated by the LUBAC complex. LPS-mediated activation of NF-kappa-B. Also ubiquitinated with \'Lys-11\'-linked chains. Polyubiquitinated with \'Lys-48\' and \'Lys-63\'-linked chains by BIRC2/c-IAP1 and BIRC3/c-IAP2, leading to activation of NF-kappa-B.SIMILARITY Belongs to the protein kinase superfamily. TKL Ser/Thr protein kinase family.SIMILARITY Contains 1 death domain.SIMILARITY Contains 1 protein kinase domain.\")\n	(DB_ID 63119)\n	(description \"recommendedName: Receptor-interacting serine/threonine-protein kinase 1 ecNumber2.7.11.1/ecNumber  alternativeName: Cell death protein RIP  alternativeName: Receptor-interacting protein 1 shortName:RIP-1  alternativeName: Serine/threonine-protein kinase RIP \")\n	(geneName\n		\"RIPK1\"\n		\"RIP\"\n		\"RIP1\")\n	(identifier \"Q13546\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"ATP-binding\"\n		\"Cell membrane\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Isopeptide bond\"\n		\"Kinase\"\n		\"Membrane\"\n		\"Necrosis\"\n		\"Nucleotide-binding\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Serine/threonine-protein kinase\"\n		\"Transferase\"\n		\"Ubl conjugation\")\n	(name \"RIPK1\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"RIPK1_HUMAN\"\n		\"A0AV89\"\n		\"B2RAG1\"\n		\"B4E3F9\"\n		\"Q13180\"\n		\"Q59H33\")\n	(sequenceLength 671)\n	(species [GK_48887]))\n\n([GK_64108] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P02735 SAA1 recommendedName: Serum amyloid A protein shortName:SAA  component recommendedName: Amyloid protein A  alternativeName: Amyloid fibril protein AA  /component component recommendedName: Serum amyloid protein A(2-104)  /component component recommendedName: Serum amyloid protein A(3-104)  /component component recommendedName: Serum amyloid protein A(2-103)  /component component recommendedName: Serum amyloid protein A(2-102)  /component component recommendedName: Serum amyloid protein A(4-101)  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Major acute phase reactant. Apolipoprotein of the HDL complex.INDUCTION Upon cytokine stimulation.PTM This protein is the precursor of amyloid protein A, which is formed by the removal of approximately 24 residues from the C-terminal end.POLYMORPHISM Both SAA1 and SAA2 have a number of alleles. We use here the nomenclature of PubMed:1546977. The sequence shown is that of 1-alpha.DISEASE Note=Reactive, secondary amyloidosis is characterized by the extracellular accumulation in various tissues of the SAA protein. These deposits are highly insoluble and resistant to proteolysis; they disrupt tissue structure and compromise function.DISEASE Note=Elevated serum SSA protein levels may be associated with lung cancer.SIMILARITY Belongs to the SAA family.\")\n	(DB_ID 64108)\n	(description \"recommendedName: Serum amyloid A protein shortName:SAA  component recommendedName: Amyloid protein A  alternativeName: Amyloid fibril protein AA  /component component recommendedName: Serum amyloid protein A(2-104)  /component component recommendedName: Serum amyloid protein A(3-104)  /component component recommendedName: Serum amyloid protein A(2-103)  /component component recommendedName: Serum amyloid protein A(2-102)  /component component recommendedName: Serum amyloid protein A(4-101)  /component\")\n	(geneName\n		\"SAA1\"\n		\"</gene>\"\n		\"<gene>\"\n		\"SAA2\")\n	(identifier \"P02735\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Acute phase\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"HDL\"\n		\"Methylation\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"SAA1\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"SAA_HUMAN\"\n		\"P02736\"\n		\"P02737\"\n		\"Q16730\"\n		\"Q16834\"\n		\"Q16835\"\n		\"Q16879\"\n		\"Q3KRB3\"\n		\"Q6FG67\"\n		\"Q96QN0\"\n		\"Q9UCK9\"\n		\"Q9UCL0\")\n	(sequenceLength 122)\n	(species [GK_48887]))\n\n([GK_64128] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P02743 APCS recommendedName: Serum amyloid P-component shortName:SAP  alternativeName: 9.5S alpha-1-glycoprotein  component recommendedName: Serum amyloid P-component(1-203)  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Can interact with DNA and histones and may scavenge nuclear material released from damaged circulating cells. May also function as a calcium-dependent lectin.COFACTOR Binds 2 calcium ions per subunit.SUBUNIT Homopentamer. Pentaxin (or pentraxin) have a discoid arrangement of 5 non-covalently bound subunits.DISEASE Note=SAP is a precursor of amyloid component P which is found in basement membrane and associated with amyloid deposits.SIMILARITY Belongs to the pentaxin family.SIMILARITY Contains 1 pentaxin domain.\")\n	(DB_ID 64128)\n	(description \"recommendedName: Serum amyloid P-component shortName:SAP  alternativeName: 9.5S alpha-1-glycoprotein  component recommendedName: Serum amyloid P-component(1-203)  /component\")\n	(geneName\n		\"APCS\"\n		\"PTX2\")\n	(identifier \"P02743\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Amyloid\"\n		\"Calcium\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Lectin\"\n		\"Metal-binding\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"APCS\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier \"SAMP_HUMAN\")\n	(sequenceLength 223)\n	(species [GK_48887]))\n\n([GK_64296] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P04279 SEMG1 recommendedName: Semenogelin-1  alternativeName: Semenogelin I shortName:SGI  alternativeName: Semenogelin I shortName:SGI  component recommendedName: Alpha-inhibin-92  /component component recommendedName: Alpha-inhibin-31  /component component recommendedName: Seminal basic protein  /component\")\n	(checksum \"1\")\n	(comment \"FUNCTION Predominant protein in semen. It participates in the formation of a gel matrix entrapping the accessory gland secretions and ejaculated spermatozoa. Fragments of semenogelin and/or fragments of the related proteins may contribute to the activation of progressive sperm movements as the gel-forming proteins are fragmented by KLK3/PSA.FUNCTION Alpha-inhibin-92 and alpha-inhibin-31, derived from the proteolytic degradation of semenogelin, inhibit the secretion of pituitary follicle-stimulating hormone.SUBUNIT Occurs in disulfide-linked complexes which may also contain two less abundant 71- and 76-kDa semenogelin-related polypeptides.TISSUE SPECIFICITY Seminal vesicle.PTM Transglutaminase substrate.PTM Rapidly cleaved after ejaculation by KLK3/PSA, resulting in liquefaction of the semen coagulum and the progressive release of motile spermatozoa.SIMILARITY Belongs to the semenogelin family.\")\n	(DB_ID 64296)\n	(description \"recommendedName: Semenogelin-1  alternativeName: Semenogelin I shortName:SGI  alternativeName: Semenogelin I shortName:SGI  component recommendedName: Alpha-inhibin-92  /component component recommendedName: Alpha-inhibin-31  /component component recommendedName: Seminal basic protein  /component\")\n	(geneName\n		\"SEMG1\"\n		\"SEMG\")\n	(identifier \"P04279\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"Alternative splicing\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Polymorphism\"\n		\"Pyrrolidone carboxylic acid\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\")\n	(name \"SEMG1\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"SEMG1_HUMAN\"\n		\"Q53ZV0\"\n		\"Q53ZV1\"\n		\"Q53ZV2\"\n		\"Q6X4I9\"\n		\"Q6Y809\"\n		\"Q6Y822\"\n		\"Q6Y823\"\n		\"Q86U64\"\n		\"Q96QM3\")\n	(sequenceLength 462)\n	(species [GK_48887]))\n\n([GK_6486] of  GO_CellularComponent\n\n	(_displayName \"host\")\n	(accession \"0018995\")\n	(DB_ID 6486)\n	(definition \"Any organism in which another organism, especially a parasite or symbiont, spends part or all of its life cycle and from which it obtains nourishment and/or protection.\")\n	(instanceOf [GK_447576])\n	(name \"host\")\n	(referenceDatabase [GK_1]))\n\n([GK_6542] of  GO_BiologicalProcess\n\n	(_displayName \"induction of apoptosis\")\n	(accession \"0006917\")\n	(DB_ID 6542)\n	(definition \"A process that directly activates any of the steps required for cell death by apoptosis.\")\n	(instanceOf\n		[GK_6543]\n		[GK_160565])\n	(name \"induction of apoptosis\")\n	(referenceDatabase [GK_1]))\n\n([GK_6543] of  GO_BiologicalProcess\n\n	(_displayName \"induction of programmed cell death\")\n	(accession \"0012502\")\n	(DB_ID 6543)\n	(definition \"A process which directly activates any of the steps required for programmed cell death.\")\n	(instanceOf [GK_160227])\n	(name \"induction of programmed cell death\")\n	(referenceDatabase [GK_1]))\n\n([GK_65462] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P37840 SNCA recommendedName: Alpha-synuclein  alternativeName: Non-A beta component of AD amyloid  alternativeName: Non-A4 component of amyloid precursor shortName:NACP  alternativeName: Non-A4 component of amyloid precursor shortName:NACP \")\n	(checksum \"1\")\n	(comment \"FUNCTION May be involved in the regulation of dopamine release and transport. Induces fibrillization of microtubule-associated protein tau. Reduces neuronal responsiveness to various apoptotic stimuli, leading to a decreased caspase-3 activation.TISSUE SPECIFICITY Expressed principally in brain but is also expressed in low concentrations in all tissues examined except in liver. Concentrated in presynaptic nerve terminals.PTM Hallmark lesions of neurodegenerative synucleinopathies contain alpha-synuclein that is modified by nitration of tyrosine residues and possibly by dityrosine cross-linking to generated stable oligomers.DISEASE Note=Genetic alterations of SNCA resulting in aberrant polymerization into fibrils, are associated with several neurodegenerative diseases (synucleinopathies). SNCA fibrillar aggregates represent the major non A-beta component of Alzheimer disease amyloid plaque, and a major component of Lewy body inclusions. They are also found within Lewy body (LB)-like intraneuronal inclusions, glial inclusions and axonal spheroids in neurodegeneration with brain iron accumulation type 1.DISEASE Defects in SNCA are the cause of Parkinson disease type 4 (PARK4) [MIM:605543]. A complex neurodegenerative disorder with manifestations ranging from typical Parkinson disease to dementia with Lewy bodies. Clinical features include parkinsonian symptoms (tremor, rigidity, postural instability and bradykinesia), dementia, diffuse Lewy body pathology, autonomic dysfunction, hallucinations and paranoia.DISEASE Defects in SNCA are the cause of dementia Lewy body (DLB) [MIM:127750]. A neurodegenerative disorder clinically characterized by mental impairment leading to dementia, parkinsonism, often with fluctuating cognitive function, visual hallucinations, falls, syncopal episodes, and sensitivity to neuroleptic medication. Brainstem or cortical intraneuronal accumulations of aggregated proteins (Lewy bodies) are the only essential pathologic features. Patients may also have hippocampal and neocortical senile plaques, sometimes in sufficient number to fulfill the diagnostic criteria for Alzheimer disease.SIMILARITY Belongs to the synuclein family.\")\n	(DB_ID 65462)\n	(description \"recommendedName: Alpha-synuclein  alternativeName: Non-A beta component of AD amyloid  alternativeName: Non-A4 component of amyloid precursor shortName:NACP  alternativeName: Non-A4 component of amyloid precursor shortName:NACP \")\n	(geneName\n		\"SNCA\"\n		\"NACP\"\n		\"PARK1\")\n	(identifier \"P37840\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Alzheimer disease\"\n		\"Amyloid\"\n		\"Cell junction\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Membrane\"\n		\"Neurodegeneration\"\n		\"Nucleus\"\n		\"Parkinson disease\"\n		\"Parkinsonism\"\n		\"Phosphoprotein\"\n		\"Repeat\"\n		\"Synapse\"\n		\"Ubl conjugation\")\n	(name \"SNCA\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"SYUA_HUMAN\"\n		\"A8K2A4\"\n		\"Q13701\"\n		\"Q4JHI3\"\n		\"Q6IAU6\")\n	(sequenceLength 140)\n	(species [GK_48887]))\n\n([GK_65505] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TNFRSF10B [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 65505)\n	(endCoordinate 440)\n	(name\n		\"TNFRSF10B\"\n		\"TRAIL receptor-2\"\n		\"Tumor necrosis factor receptor superfamily member 10B precursor (Death receptor 5) (TNF-related apoptosis-inducing ligand receptor 2) (TRAIL receptor-2) (TRAIL-R2)\"\n		\"Tumor necrosis factor receptor superfamily member 10B precursor\"\n		\"Death receptor 5\"\n		\"TNF-related apoptosis-inducing ligand receptor 2\"\n		\"TRAIL-R2\")\n	(referenceEntity [GK_65506])\n	(species [GK_48887])\n	(stableIdentifier [GK_371450])\n	(startCoordinate 56))\n\n([GK_65506] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:O14763 TNFRSF10B recommendedName: Tumor necrosis factor receptor superfamily member 10B  alternativeName: Death receptor 5  alternativeName: TNF-related apoptosis-inducing ligand receptor 2 shortName:TRAIL receptor 2 shortName:TRAIL-R2  cdAntigenNameCD262/cdAntigenName\")\n	(checksum \"60358EAF2A835870\")\n	(comment \"FUNCTION Receptor for the cytotoxic ligand TNFSF10/TRAIL. The adapter molecule FADD recruits caspase-8 to the activated receptor. The resulting death-inducing signaling complex (DISC) performs caspase-8 proteolytic activation which initiates the subsequent cascade of caspases (aspartate-specific cysteine proteases) mediating apoptosis. Promotes the activation of NF-kappa-B. Essential for ER stress-induced apoptosis.SUBUNIT Homotrimer. Can interact with TRADD and RIPK1.TISSUE SPECIFICITY Widely expressed in adult and fetal tissues; very highly expressed in tumor cell lines such as HeLaS3, K-562, HL-60, SW480, A-549 and G-361; highly expressed in heart, peripheral blood lymphocytes, liver, pancreas, spleen, thymus, prostate, ovary, uterus, placenta, testis, esophagus, stomach and throughout the intestinal tract; not detectable in brain.INDUCTION By ER stress. Regulated by p53/TP53.SIMILARITY Contains 1 death domain.SIMILARITY Contains 3 TNFR-Cys repeats.\")\n	(DB_ID 65506)\n	(description \"recommendedName: Tumor necrosis factor receptor superfamily member 10B  alternativeName: Death receptor 5  alternativeName: TNF-related apoptosis-inducing ligand receptor 2 shortName:TRAIL receptor 2 shortName:TRAIL-R2  cdAntigenNameCD262/cdAntigenName\")\n	(geneName\n		\"TNFRSF10B\"\n		\"DR5\"\n		\"KILLER\"\n		\"TRAILR2\"\n		\"TRICK2\"\n		\"ZTNFR9\"\n		\"UNQ160/PRO186\")\n	(identifier \"O14763\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Membrane\"\n		\"Polymorphism\"\n		\"Receptor\"\n		\"Reference proteome\"\n		\"Repeat\"\n		\"Signal\"\n		\"Transmembrane\"\n		\"Transmembrane helix\")\n	(name \"TNFRSF10B\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TR10B_HUMAN\"\n		\"O14720\"\n		\"O15508\"\n		\"O15517\"\n		\"O15531\"\n		\"Q6UXM8\"\n		\"Q7Z360\"\n		\"Q9BVE0\")\n	(sequenceLength 440)\n	(species [GK_48887]))\n\n([GK_66186] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TNFSF10 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 66186)\n	(endCoordinate 281)\n	(name\n		\"TNFSF10\"\n		\"TRAIL\"\n		\"Tumor necrosis factor ligand superfamily member 10 (TNF-related apoptosis inducing ligand) (TRAIL protein) (Apo-2 ligand) (Apo-2L)\"\n		\"Tumor necrosis factor ligand superfamily member 10\"\n		\"TNF-related apoptosis inducing ligand\"\n		\"TRAIL protein\"\n		\"Apo-2 ligand\"\n		\"Apo-2L\")\n	(referenceEntity [GK_66187])\n	(species [GK_48887])\n	(stableIdentifier [GK_366902])\n	(startCoordinate 1))\n\n([GK_66187] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P50591 TNFSF10 recommendedName: Tumor necrosis factor ligand superfamily member 10  alternativeName: Apo-2 ligand shortName:Apo-2L  alternativeName: TNF-related apoptosis-inducing ligand shortName:Protein TRAIL  cdAntigenNameCD253/cdAntigenName\")\n	(checksum \"DDAAAF78DAAB2F6D\")\n	(comment \"FUNCTION Cytokine that binds to TNFRSF10A/TRAILR1, TNFRSF10B/TRAILR2, TNFRSF10C/TRAILR3, TNFRSF10D/TRAILR4 and possibly also to TNFRSF11B/OPG. Induces apoptosis. Its activity may be modulated by binding to the decoy receptors TNFRSF10C/TRAILR3, TNFRSF10D/TRAILR4 and TNFRSF11B/OPG that cannot induce apoptosis.COFACTOR Binds 1 zinc ion per trimer.SUBUNIT Homotrimer.TISSUE SPECIFICITY Widespread; most predominant in spleen, lung and prostate.SIMILARITY Belongs to the tumor necrosis factor family.\")\n	(DB_ID 66187)\n	(description \"recommendedName: Tumor necrosis factor ligand superfamily member 10  alternativeName: Apo-2 ligand shortName:Apo-2L  alternativeName: TNF-related apoptosis-inducing ligand shortName:Protein TRAIL  cdAntigenNameCD253/cdAntigenName\")\n	(geneName\n		\"TNFSF10\"\n		\"APO2L\"\n		\"TRAIL\")\n	(identifier \"P50591\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Cytokine\"\n		\"Membrane\"\n		\"Metal-binding\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Signal-anchor\"\n		\"Transmembrane\"\n		\"Transmembrane helix\"\n		\"Zinc\")\n	(name \"TNFSF10\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TNF10_HUMAN\"\n		\"A1Y9B3\")\n	(sequenceLength 281)\n	(species [GK_48887]))\n\n([GK_66212] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] FASLG(1-281) [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 66212)\n	(endCoordinate 281)\n	(name\n		\"FASLG(1-281)\"\n		\"FASL\"\n		\"Tumor necrosis factor ligand superfamily member 6 (FAS antigen ligand) (Apoptosis antigen ligand) (APTL) (CD178 antigen)\"\n		\"Tumor necrosis factor ligand superfamily member 6\"\n		\"FAS antigen ligand\"\n		\"Apoptosis antigen ligand\"\n		\"APTL\"\n		\"CD178 antigen\")\n	(referenceEntity [GK_66213])\n	(species [GK_48887])\n	(stableIdentifier [GK_364838])\n	(startCoordinate 1))\n\n([GK_66213] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P48023 FASLG recommendedName: Tumor necrosis factor ligand superfamily member 6  alternativeName: Apoptosis antigen ligand shortName:APTL  alternativeName: CD95 ligand shortName:CD95-L  alternativeName: Fas antigen ligand shortName:Fas ligand shortName:FasL  cdAntigenNameCD178/cdAntigenName component recommendedName: Tumor necrosis factor ligand superfamily member 6, membrane form  /component component recommendedName: Tumor necrosis factor ligand superfamily member 6, soluble form  alternativeName: Receptor-binding FasL ectodomain  alternativeName: Soluble Fas ligand shortName:sFasL  /component component recommendedName: ADAM10-processed FasL form shortName:APL  /component component recommendedName: FasL intracellular domain shortName:FasL ICD  alternativeName: SPPL2A-processed FasL form shortName:SPA  /component\")\n	(checksum \"A8A6EB358246E9BB\")\n	(comment \"FUNCTION Cytokine that binds to TNFRSF6/FAS, a receptor that transduces the apoptotic signal into cells. May be involved in cytotoxic T-cell mediated apoptosis and in T-cell development. TNFRSF6/FAS-mediated apoptosis may have a role in the induction of peripheral tolerance, in the antigen-stimulated suicide of mature T-cells, or both. Binding to the decoy receptor TNFRSF6B/DcR3 modulates its effects.FUNCTION The FasL intracellular domain (FasL ICD) cytoplasmic form induces gene transcription inhibition.SUBUNIT Homotrimer (Probable). Interacts with ARHGAP9, BAIAP2L1, BTK, CACNB3, CACNB4, CRK, DLG2, DNMBP, DOCK4, EPS8L3, FGR, FYB, FYN, HCK, ITK, ITSN2, KALRN, LYN, MACC1, MIA, MPP4, MYO15A, NCF1, NCK1, NCK2, NCKIPSD, OSTF1, PIK3R1, PSTPIP1, RIMBP3C, SAMSN1, SH3GL3, SH3PXD2B, SH3PXD2A, SH3RF2, SKAP2, SNX33, SNX9, SORBS3, SPTA1, SRC, SRGAP1, SRGAP2, SRGAP3, TEC, TJP3 and YES1.PTM The soluble form derives from the membrane form by proteolytic processing. The membrane-bound form undergoes two successive intramembrane proteolytic cleavages. The first one is processed by ADAM10 producing an N-terminal fragment, which lacks the receptor-binding extracellular domain. This ADAM10-processed FasL (FasL APL) remnant form is still membrane anchored and further processed by SPPL2A that liberates the FasL intracellular domain (FasL ICD). FasL shedding by ADAM10 is a prerequisite for subsequent intramembrane cleavage by SPPL2A in T-cells.PTM N-glycosylated.PTM Phosphorylated by FGR on tyrosine residues; this is required for ubiquitination and subsequent internalization.PTM Monoubiquitinated.SIMILARITY Belongs to the tumor necrosis factor family.\")\n	(DB_ID 66213)\n	(description \"recommendedName: Tumor necrosis factor ligand superfamily member 6  alternativeName: Apoptosis antigen ligand shortName:APTL  alternativeName: CD95 ligand shortName:CD95-L  alternativeName: Fas antigen ligand shortName:Fas ligand shortName:FasL  cdAntigenNameCD178/cdAntigenName component recommendedName: Tumor necrosis factor ligand superfamily member 6, membrane form  /component component recommendedName: Tumor necrosis factor ligand superfamily member 6, soluble form  alternativeName: Receptor-binding FasL ectodomain  alternativeName: Soluble Fas ligand shortName:sFasL  /component component recommendedName: ADAM10-processed FasL form shortName:APL  /component component recommendedName: FasL intracellular domain shortName:FasL ICD  alternativeName: SPPL2A-processed FasL form shortName:SPA  /component\")\n	(geneName\n		\"FASLG\"\n		\"APT1LG1\"\n		\"CD95L\"\n		\"FASL\"\n		\"TNFSF6\")\n	(identifier \"P48023\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Cell membrane\"\n		\"Complete proteome\"\n		\"Cytokine\"\n		\"Cytoplasmic vesicle\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Lysosome\"\n		\"Membrane\"\n		\"Nucleus\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Repressor\"\n		\"Secreted\"\n		\"Signal-anchor\"\n		\"Transcription\"\n		\"Transcription regulation\"\n		\"Transmembrane\"\n		\"Transmembrane helix\"\n		\"Ubl conjugation\")\n	(name \"FASLG\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TNFL6_HUMAN\"\n		\"Q9BZP9\")\n	(sequenceLength 281)\n	(species [GK_48887]))\n\n([GK_66220] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TNF(1-233) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 66220)\n	(endCoordinate 233)\n	(name\n		\"TNF(1-233)\"\n		\"TNF-alpha\"\n		\"Tumor necrosis factor precursor (TNF-alpha) (Tumor necrosis factor ligand superfamily member 2) (Cachectin)\"\n		\"Tumor necrosis factor precursor\"\n		\"Tumor necrosis factor ligand superfamily member 2\"\n		\"TNF-a\"\n		\"Cachectin\")\n	(referenceEntity [GK_66221])\n	(species [GK_48887])\n	(stableIdentifier [GK_370748])\n	(startCoordinate 1))\n\n([GK_66221] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P01375 TNF recommendedName: Tumor necrosis factor  alternativeName: Cachectin  alternativeName: TNF-alpha  alternativeName: Tumor necrosis factor ligand superfamily member 2 shortName:TNF-a  component recommendedName: Tumor necrosis factor, membrane form  alternativeName: N-terminal fragment shortName:NTF  /component component recommendedName: Intracellular domain 1 shortName:ICD1  /component component recommendedName: Intracellular domain 2 shortName:ICD2  /component component recommendedName: C-domain 1  /component component recommendedName: C-domain 2  /component component recommendedName: Tumor necrosis factor, soluble form  /component\")\n	(checksum \"3DF90F96C9031FFE\")\n	(comment \"FUNCTION Cytokine that binds to TNFRSF1A/TNFR1 and TNFRSF1B/TNFBR. It is mainly secreted by macrophages and can induce cell death of certain tumor cell lines. It is potent pyrogen causing fever by direct action or by stimulation of interleukin-1 secretion and is implicated in the induction of cachexia, Under certain conditions it can stimulate cell proliferation and induce cell differentiation.FUNCTION The TNF intracellular domain (ICD) form induces IL12 production in dendritic cells.SUBUNIT Homotrimer. Interacts with SPPL2B.PTM The soluble form derives from the membrane form by proteolytic processing. The membrane-bound form is further proteolytically processed by SPPL2A or SPPL2B through regulated intramembrane proteolysis producing TNF intracellular domains (ICD1 and ICD2) released in the cytosol and TNF C-domain 1 and C-domain 2 secreted into the extracellular space.PTM The membrane form, but not the soluble form, is phosphorylated on serine residues. Dephosphorylation of the membrane form occurs by binding to soluble TNFRSF1A/TNFR1.PTM O-glycosylated; glycans contain galactose, N-acetylgalactosamine and N-acetylneuraminic acid.POLYMORPHISM Genetic variations in TNF influence susceptibility to hepatitis B virus (HBV) infection [MIM:610424].POLYMORPHISM Genetic variations in TNF are involved in susceptibility to malaria [MIM:611162].SIMILARITY Belongs to the tumor necrosis factor family.\")\n	(DB_ID 66221)\n	(description \"recommendedName: Tumor necrosis factor  alternativeName: Cachectin  alternativeName: TNF-alpha  alternativeName: Tumor necrosis factor ligand superfamily member 2 shortName:TNF-a  component recommendedName: Tumor necrosis factor, membrane form  alternativeName: N-terminal fragment shortName:NTF  /component component recommendedName: Intracellular domain 1 shortName:ICD1  /component component recommendedName: Intracellular domain 2 shortName:ICD2  /component component recommendedName: C-domain 1  /component component recommendedName: C-domain 2  /component component recommendedName: Tumor necrosis factor, soluble form  /component\")\n	(geneName\n		\"TNF\"\n		\"TNFA\"\n		\"TNFSF2\")\n	(identifier \"P01375\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Cell membrane\"\n		\"Complete proteome\"\n		\"Cytokine\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Lipoprotein\"\n		\"Membrane\"\n		\"Myristate\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Reference proteome\"\n		\"Secreted\"\n		\"Signal-anchor\"\n		\"Transmembrane\"\n		\"Transmembrane helix\")\n	(name \"TNF\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TNFA_HUMAN\"\n		\"O43647\"\n		\"Q9P1Q2\"\n		\"Q9UIV3\")\n	(sequenceLength 233)\n	(species [GK_48887]))\n\n([GK_66248] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] FAS [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 66248)\n	(endCoordinate 335)\n	(name\n		\"FAS\"\n		\"FAS Receptor\"\n		\"Tumor necrosis factor receptor superfamily member 6 precursor (FASL receptor) (Apoptosis-mediating surface antigen FAS) (Apo-1 antigen) (CD95)\"\n		\"Tumor necrosis factor receptor superfamily member 6 precursor\"\n		\"FASL receptor\"\n		\"Apoptosis-mediating surface antigen FAS\"\n		\"Apo-1 antigen\"\n		\"CD95\")\n	(referenceEntity [GK_66249])\n	(species [GK_48887])\n	(stableIdentifier [GK_364006])\n	(startCoordinate 26))\n\n([GK_66249] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P25445 FAS recommendedName: Tumor necrosis factor receptor superfamily member 6  alternativeName: Apo-1 antigen  alternativeName: Apoptosis-mediating surface antigen FAS  alternativeName: FASLG receptor  cdAntigenNameCD95/cdAntigenName\")\n	(checksum \"0139942535111410\")\n	(comment \"FUNCTION Receptor for TNFSF6/FASLG. The adapter molecule FADD recruits caspase-8 to the activated receptor. The resulting death-inducing signaling complex (DISC) performs caspase-8 proteolytic activation which initiates the subsequent cascade of caspases (aspartate-specific cysteine proteases) mediating apoptosis. FAS-mediated apoptosis may have a role in the induction of peripheral tolerance, in the antigen-stimulated suicide of mature T-cells, or both. The secreted isoforms 2 to 6 block apoptosis (in vitro).SUBUNIT Binds DAXX. Interacts with HIPK3. Part of a complex containing HIPK3 and FADD (By similarity). Binds RIPK1 and FAIM2. Interacts with BRE and FEM1B. Interacts with FADD.TISSUE SPECIFICITY Isoform 1 and isoform 6 are expressed at equal levels in resting peripheral blood mononuclear cells. After activation there is an increase in isoform 1 and decrease in the levels of isoform 6.DOMAIN Contains a death domain involved in the binding of FADD, and maybe to other cytosolic adapter proteins.PTM N- and O-glycosylated. O-glycosylated with core 1 or possibly core 8 glycans.SIMILARITY Contains 1 death domain.SIMILARITY Contains 3 TNFR-Cys repeats.\")\n	(DB_ID 66249)\n	(description \"recommendedName: Tumor necrosis factor receptor superfamily member 6  alternativeName: Apo-1 antigen  alternativeName: Apoptosis-mediating surface antigen FAS  alternativeName: FASLG receptor  cdAntigenNameCD95/cdAntigenName\")\n	(geneName\n		\"FAS\"\n		\"APT1\"\n		\"FAS1\"\n		\"TNFRSF6\")\n	(identifier \"P25445\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Cell membrane\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Membrane\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Receptor\"\n		\"Reference proteome\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\"\n		\"Transmembrane\"\n		\"Transmembrane helix\")\n	(name \"FAS\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TNR6_HUMAN\"\n		\"A9UJX4\"\n		\"B6VNV4\"\n		\"Q14292\"\n		\"Q14293\"\n		\"Q14294\"\n		\"Q14295\"\n		\"Q16652\"\n		\"Q5T9P1\"\n		\"Q5T9P2\"\n		\"Q5T9P3\"\n		\"Q6SSE9\")\n	(sequenceLength 335)\n	(species [GK_48887]))\n\n([GK_66344] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TNFRSF1A(22-455) [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 66344)\n	(endCoordinate 455)\n	(name\n		\"TNFRSF1A(22-455)\"\n		\"TNF-RI\"\n		\"Tumor necrosis factor receptor superfamily member 1A precursor (p60) (TNF-R1) (TNF-RI) (p55) (CD120a) (Contains: Tumor necrosis factor binding protein 1 (TBPI))\"\n		\"Tumor necrosis factor receptor superfamily member 1A precursor\"\n		\"p60\"\n		\"TNF-R1\"\n		\"p55\"\n		\"CD120a\"\n		\"Tumor necrosis factor binding protein 1\"\n		\"TBPI\")\n	(referenceEntity [GK_66345])\n	(species [GK_48887])\n	(stableIdentifier [GK_368490])\n	(startCoordinate 22))\n\n([GK_66345] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P19438 TNFRSF1A recommendedName: Tumor necrosis factor receptor superfamily member 1A  alternativeName: Tumor necrosis factor receptor 1 shortName:TNF-R1  alternativeName: Tumor necrosis factor receptor type I shortName:TNF-RI shortName:TNFR-I  alternativeName: p55  alternativeName: p60  cdAntigenNameCD120a/cdAntigenName component recommendedName: Tumor necrosis factor receptor superfamily member 1A, membrane form  /component component recommendedName: Tumor necrosis factor-binding protein 1 shortName:TBPI  /component\")\n	(checksum \"4CEFBA96D03B8225\")\n	(comment \"FUNCTION Receptor for TNFSF2/TNF-alpha and homotrimeric TNFSF1/lymphotoxin-alpha. The adapter molecule FADD recruits caspase-8 to the activated receptor. The resulting death-inducing signaling complex (DISC) performs caspase-8 proteolytic activation which initiates the subsequent cascade of caspases (aspartate-specific cysteine proteases) mediating apoptosis. Contributes to the induction of non-cytocidal TNF effects including anti-viral state and activation of the acid sphingomyelinase.SUBUNIT Binding of TNF to the extracellular domain leads to homotrimerization. The aggregated death domains provide a novel molecular interface that interacts specifically with the death domain of TRADD. Various TRADD-interacting proteins such as TRAFS, RIPK1 and possibly FADD, are recruited to the complex by their association with TRADD. This complex activates at least two distinct signaling cascades, apoptosis and NF-kappa-B signaling. Interacts with BAG4, BRE, FEM1B, GRB2, SQSTM1 and TRPC4AP. Interacts with HCV core protein. Interacts with human cytomegalovirus/HHV-5 protein UL138.DOMAIN The domain that induces A-SMASE is probably identical to the death domain. The N-SMASE activation domain (NSD) is both necessary and sufficient for activation of N-SMASE.PTM The soluble form is produced from the membrane form by proteolytic processing.SIMILARITY Contains 1 death domain.SIMILARITY Contains 4 TNFR-Cys repeats.\")\n	(DB_ID 66345)\n	(description \"recommendedName: Tumor necrosis factor receptor superfamily member 1A  alternativeName: Tumor necrosis factor receptor 1 shortName:TNF-R1  alternativeName: Tumor necrosis factor receptor type I shortName:TNF-RI shortName:TNFR-I  alternativeName: p55  alternativeName: p60  cdAntigenNameCD120a/cdAntigenName component recommendedName: Tumor necrosis factor receptor superfamily member 1A, membrane form  /component component recommendedName: Tumor necrosis factor-binding protein 1 shortName:TBPI  /component\")\n	(geneName\n		\"TNFRSF1A\"\n		\"TNFAR\"\n		\"TNFR1\")\n	(identifier \"P19438\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Amyloidosis\"\n		\"Apoptosis\"\n		\"Cell membrane\"\n		\"Cleavage on pair of basic residues\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Golgi apparatus\"\n		\"Host-virus interaction\"\n		\"Membrane\"\n		\"Polymorphism\"\n		\"Receptor\"\n		\"Reference proteome\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Signal\"\n		\"Transmembrane\"\n		\"Transmembrane helix\")\n	(name \"TNFRSF1A\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TNR1A_HUMAN\"\n		\"A8K4X3\"\n		\"B2RDE4\"\n		\"B3KPQ1\"\n		\"B4DQB7\"\n		\"B4E309\"\n		\"B5M0B5\"\n		\"D3DUR1\"\n		\"Q9UCA4\")\n	(sequenceLength 455)\n	(species [GK_48887]))\n\n([GK_66370] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TRAF2 [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 66370)\n	(endCoordinate 501)\n	(name\n		\"TRAF2\"\n		\"TNF receptor associated factor 2\"\n		\"Tumor necrosis factor type 2 receptor associated protein 3\")\n	(referenceEntity [GK_66371])\n	(species [GK_48887])\n	(stableIdentifier [GK_369343])\n	(startCoordinate 2))\n\n([GK_66371] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q12933 TRAF2 recommendedName: TNF receptor-associated factor 2 ecNumber6.3.2.-/ecNumber  alternativeName: E3 ubiquitin-protein ligase TRAF2  alternativeName: Tumor necrosis factor type 2 receptor-associated protein 3 \")\n	(checksum \"C508BE185B783B20\")\n	(comment \"FUNCTION Regulates activation of NF-kappa-B and JNK and plays a central role in the regulation of cell survival and apoptosis. Required for normal antibody isotype switching from IgM to IgG. Has E3 ubiquitin-protein ligase activity and promotes \'Lys-63\'-linked ubiquitination of target proteins, such as BIRC3, RIPK1 and TICAM1. Is an essential constituent of several E3 ubiquitin-protein ligase complexes, where it promotes the ubiquitination of target proteins by bringing them into contact with other E3 ubiquitin ligases. Regulates BIRC2 and BIRC3 protein levels by inhibiting their autoubiquitination and subsequent degradation; this does not depend on the TRAF2 RING-type zinc finger domain. Plays a role in mediating activation of NF-kappa-B by EIF2AK2/PKR.ENZYME REGULATION Has very low E3 ubiquitin ligase activity in the absence of sphingosine-1-phosphate. E3 ubiquitin ligase activity is strongly activated by cytoplasmic sphingosine-1-phosphate.PATHWAY Protein modification; protein ubiquitination.SUBUNIT Homotrimer, and heterotrimer with TRAF1 and TRAF3 (via TRAF domain). The domain containing the RING-type and the first TRAF-type zinc finger can also form homodimers (in vitro). Interacts with TNFRSF1B/TNFR2, TNFRSF4, TNFRSF5/CD40, CD27/TNFRSF7, TNFRSF8/CD30, TNFRSF9/CD137, TNFRSF11A/RANK, TNFRSF13B/TACI, TNFRSF14, TNFRSF16/NGFR, TNFRSF17/BCMA, TNFRSF18/AITR, TNFRSF19/TROY, TNFRSF19L/RELT, XEDAR, EDAR, Epstein-Barr virus BNFL1/LMP-1 and IL15RA. Interacts with CDK9, CSK, MAP3K1, MAP3K5, MAP3K11, MAP3K14, MAP4K2, RIPK1, RIPK2, TNIK, TBK1, SPHK1, TRADD, TRAFD1, TRAIP, TANK/ITRAF, TNFAIP3, TDP2, MAVS/IPS1, TICAM1 and TRPC4AP. Interacts with CASP8AP2, NFATC2IP, PEG3 and HIVEP3. Interacts with ERN1; the interaction requires DAB2IP (By similarity). Interacts with BIRC2 and BIRC3 N-terminus; a single BIRC2 or BIRC3 molecule interacts with a heterotrimer formed by TRAF1 and TRAF2, or a TRAF2 homotrimer. Identified in a complex composed of TRAF2, TRAF3, BIRC2 and BIRC3. Interaction with BIRC2 and/or BIRC3 is essential for degradation of NFKBIA and activation of NF-kappa-B. Interacts with CYLD, USP48, IKKA and IKKB. Identified in a complex with TNFRSF1A, RIPK1 and IKKB. Interacts (via \'Lys-63\'-linked polyubiquitin chains) with TAB2 and TAB3. Interacts with ERN1 and TAOK3. Interaction with TAOK3 is facilitated under ER stress conditions, such as treatment with tunicamycin, and may promote TRAF2 phosphorylation. Interacts (via zinc fingers) with DAB2IP (via C-terminus PER domain); the interaction occurs in a TNF-alpha-dependent manner. Interacts (via C-terminus) with EIF2AK2/PKR (via the kinase catalytic domain). Interacts with CARD14. Interacts with PTPN2; probably involved in tumor necrosis factor-mediated signaling.DOMAIN The coiled coil domain mediates homo- and hetero-oligomerization.DOMAIN The MATH/TRAF domain binds to receptor cytoplasmic domains.DOMAIN The RING-type zinc finger domain is essential for E3 ubiquitin-protein ligase activity. It is not essential for the stabilization of BIRC2, or for the ubiquitination of RIPK1 in response to TNFR1 signaling.PTM Phosphorylated at several serine residues within the first 128 amino acid residues. Phosphorylated at Thr-117 in response to signaling via TNF and TNFRSF1A. Phosphorylation at Thr-117 is required for \'Lys-63\'-linked polyubiquitination, but not for \'Lys-48\'-linked polyubiquitination. Phosphorylation at Thr-117 is important for interaction with IKKA and IKKB, activation of IKK and subsequent activation of NF-kappa-B.PTM Undergoes both \'Lys-48\'-linked and \'Lys-63\'-linked polyubiquitination. Polyubiquitinated via \'Lys-63\'-linked ubiquitin in response to TNF signaling; this requires prior phosphorylation at Thr-117. \'Lys-63\'-linked polyubiquitination promotes TRAF2-mediated activation of NF-kappa-B. Can be polyubiquitinated at several Lys residues via \'Lys-48\'-linked ubiquitin chains in response to TNF signaling, leading to proteasomal degradation. Autoubiquitinated, leading to its subsequent proteasomal degradation. Polyubiquitinated by BIRC2 and SIAH2, leading to its subsequent proteasomal degradation. Deubiquitinated by CYLD, a protease that specifically cleaves \'Lys-63\'-linked polyubiquitin chains.SIMILARITY Belongs to the TNF receptor-associated factor family. A subfamily.SIMILARITY Contains 1 MATH domain.SIMILARITY Contains 1 RING-type zinc finger.SIMILARITY Contains 2 TRAF-type zinc fingers.\")\n	(DB_ID 66371)\n	(description \"recommendedName: TNF receptor-associated factor 2 ecNumber6.3.2.-/ecNumber  alternativeName: E3 ubiquitin-protein ligase TRAF2  alternativeName: Tumor necrosis factor type 2 receptor-associated protein 3 \")\n	(geneName\n		\"TRAF2\"\n		\"TRAP3\")\n	(identifier \"Q12933\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Acetylation\"\n		\"Alternative splicing\"\n		\"Apoptosis\"\n		\"Coiled coil\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Isopeptide bond\"\n		\"Ligase\"\n		\"Lipid-binding\"\n		\"Metal-binding\"\n		\"Phosphoprotein\"\n		\"Reference proteome\"\n		\"Repeat\"\n		\"Ubl conjugation\"\n		\"Ubl conjugation pathway\"\n		\"Zinc\"\n		\"Zinc-finger\")\n	(name \"TRAF2\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TRAF2_HUMAN\"\n		\"A8K107\"\n		\"B4DPJ7\"\n		\"Q7Z337\"\n		\"Q96NT2\")\n	(sequenceLength 501)\n	(species [GK_48887]))\n\n([GK_66376] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TRADD [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 66376)\n	(endCoordinate 312)\n	(name\n		\"TRADD\"\n		\"TNFR1-associated DEATH domain protein\"\n		\"Tumor necrosis factor receptor type 1 associated DEATH domain protein\"\n		\"TNFRSF1A-associated via death domain\")\n	(referenceEntity [GK_66377])\n	(species [GK_48887])\n	(stableIdentifier [GK_365494])\n	(startCoordinate 1))\n\n([GK_66377] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:Q15628 TRADD recommendedName: Tumor necrosis factor receptor type 1-associated DEATH domain protein shortName:TNFR1-associated DEATH domain protein  alternativeName: TNFRSF1A-associated via death domain \")\n	(checksum \"5645D7E63E5FF05A\")\n	(comment \"FUNCTION The nuclear form acts as a tumor suppressor by preventing ubiquitination and degradation of isoform p19ARF/ARF of CDKN2A by TRIP12: acts by interacting with TRIP12, leading to disrupt interaction between TRIP12 and isoform p19ARF/ARF of CDKN2A (By similarity). Adapter molecule for TNFRSF1A/TNFR1 that specifically associates with the cytoplasmic domain of activated TNFRSF1A/TNFR1 mediating its interaction with FADD. Overexpression of TRADD leads to two major TNF-induced responses, apoptosis and activation of NF-kappa-B.SUBUNIT Interacts with TRIP12 (By similarity). Heterodimer with TNFRSF1A/TNFR1. Interacts with DAB2IP, FADD, HIPK2, KRT14, KRT16, KRT17, KRT18, RIPK1, SQSTM1, TRAF1, TRAF2 and TRPC4AP.TISSUE SPECIFICITY Found in all examined tissues.DOMAIN Requires the intact death domain to associate with TNFRSF1A/TNFR1.SIMILARITY Contains 1 death domain.\")\n	(DB_ID 66377)\n	(description \"recommendedName: Tumor necrosis factor receptor type 1-associated DEATH domain protein shortName:TNFR1-associated DEATH domain protein  alternativeName: TNFRSF1A-associated via death domain \")\n	(geneName \"TRADD\")\n	(identifier \"Q15628\")\n	(isSequenceChanged FALSE)\n	(keyword\n		\"3D-structure\"\n		\"Apoptosis\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Cytoskeleton\"\n		\"Nucleus\"\n		\"Reference proteome\")\n	(name \"TRADD\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TRADD_HUMAN\"\n		\"B2RDS3\"\n		\"Q52NZ1\")\n	(sequenceLength 312)\n	(species [GK_48887]))\n\n([GK_66401] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P02788 LTF  Lactotransferrin shortName:Lactoferrin  alternativeName: Talalactoferrin  component recommendedName: Kaliocin-1  /component component recommendedName: Lactoferroxin-A  /component component recommendedName: Lactoferroxin-B  /component component recommendedName: Lactoferroxin-C  /component\")\n	(checksum \"1\")\n	(comment \"CATALYTIC ACTIVITY Preferential at -Arg-Ser-Arg-Arg-|- and -Arg-Arg-Ser-Arg-|-, and of Z-Phe-Arg-|-aminomethylcoumarin.SUBUNIT Monomer.SIMILARITY Belongs to the transferrin family.SIMILARITY Contains 2 transferrin-like domains.\")\n	(DB_ID 66401)\n	(description \" Lactotransferrin shortName:Lactoferrin  alternativeName: Talalactoferrin  component recommendedName: Kaliocin-1  /component component recommendedName: Lactoferroxin-A  /component component recommendedName: Lactoferroxin-B  /component component recommendedName: Lactoferroxin-C  /component\")\n	(geneName\n		\"LTF\"\n		\"LF\")\n	(identifier \"P02788\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Antibiotic\"\n		\"Antimicrobial\"\n		\"Complete proteome\"\n		\"Direct protein sequencing\"\n		\"Disulfide bond\"\n		\"Glycoprotein\"\n		\"Hydrolase\"\n		\"Ion transport\"\n		\"Iron\"\n		\"Iron transport\"\n		\"Metal-binding\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Protease\"\n		\"Repeat\"\n		\"Secreted\"\n		\"Serine protease\"\n		\"Signal\"\n		\"Transport\")\n	(name \"LTF\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TRFL_HUMAN\"\n		\"O00756\"\n		\"Q16780\"\n		\"Q16785\"\n		\"Q16786\"\n		\"Q16789\"\n		\"Q8IU92\"\n		\"Q8IZH6\"\n		\"Q8TCD2\"\n		\"Q96KZ4\"\n		\"Q96KZ5\"\n		\"Q9H1Z3\"\n		\"Q9UCY5\")\n	(sequenceLength 710)\n	(species [GK_48887]))\n\n([GK_66545] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P02766 TTR recommendedName: Transthyretin  alternativeName: ATTR  alternativeName: Prealbumin  alternativeName: TBPA \")\n	(checksum \"1\")\n	(comment \"DOMAIN Each monomer has two 4-stranded beta sheets and the shape of a prolate ellipsoid. Antiparallel beta-sheet interactions link monomers into dimers. A short loop from each monomer forms the main dimer-dimer interaction. These two pairs of loops separate the opposed, convex beta-sheets of the dimers to form an internal channel.MISCELLANEOUS Tetramer dissociation and partial unfolding leads to the formation of aggregates and amyloid fibrils. Small molecules that occupy at least one of the thyroid hormone binding sites stabilize the tetramer, and thereby stabilize the native state and protect against misfolding and the formation of amyloid fibrils.MISCELLANEOUS Two binding sites for thyroxine are located in the channel. Less than 1% of plasma prealbumin molecules are normally involved in thyroxine transport. L-thyroxine binds to the transthyretin by an order of magnitude stronger than does the triiodo-L-thyronine. Thyroxine-binding globulin is the major carrier protein for thyroid hormones in man.MISCELLANEOUS About 40% of plasma transthyretin circulates in a tight protein-protein complex with the plasma retinol-binding protein (RBP). The formation of the complex with RBP stabilizes the binding of retinol to RBP and decreases the glomerular filtration and renal catabolism of the relatively small RBP molecule. There is evidence for 2 binding sites for RBP, one possibly being a region that includes Ile-104, located on the outer surface of the transthyretin molecule.SIMILARITY Belongs to the transthyretin family.\")\n	(DB_ID 66545)\n	(description \"recommendedName: Transthyretin  alternativeName: ATTR  alternativeName: Prealbumin  alternativeName: TBPA \")\n	(geneName\n		\"TTR\"\n		\"PALB\")\n	(identifier \"P02766\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Complete proteome\"\n		\"Cytoplasm\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Gamma-carboxyglutamic acid\"\n		\"Glycoprotein\"\n		\"Hormone\"\n		\"Neuropathy\"\n		\"Polymorphism\"\n		\"Secreted\"\n		\"Signal\"\n		\"Thyroid hormone\"\n		\"Transport\")\n	(name \"TTR\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"TTHY_HUMAN\"\n		\"Q549C7\"\n		\"Q6IB96\"\n		\"Q9UBZ6\"\n		\"Q9UCM9\")\n	(sequenceLength 147)\n	(species [GK_48887]))\n\n([GK_69324] of  Affiliation\n\n	(_displayName \"NYU School of Medicine, Department of Biochemistry\")\n	(address \"New York NY USA\")\n	(DB_ID 69324)\n	(name \"NYU School of Medicine, Department of Biochemistry\"))\n\n([GK_69416] of  Pathway\n\n	(_displayName \"[Hsa] Dimerization of procaspase-8\")\n	(_doRelease TRUE)\n	(compartment [GK_356])\n	(DB_ID 69416)\n	(edited [GK_3508822])\n	(goBiologicalProcess [GK_23175])\n	(hasEvent\n		[GK_73945]\n		[GK_141156]\n		[GK_141159])\n	(literatureReference\n		[GK_1267800]\n		[GK_2584573]\n		[GK_2671825])\n	(name \"Dimerization of procaspase-8\")\n	(reviewed\n		[GK_2594924]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_358384])\n	(summation [GK_213104]))\n\n([GK_69781] of  ReferenceGeneProduct\n\n	(_displayName \"[Hsa] UniProt:P05067 APP recommendedName: Amyloid beta A4 protein  alternativeName: ABPP  alternativeName: APPI shortName:APP  alternativeName: APPI shortName:APP  alternativeName: Alzheimer disease amyloid protein  alternativeName: Cerebral vascular amyloid peptide shortName:CVAP  alternativeName: Cerebral vascular amyloid peptide shortName:CVAP  alternativeName: PreA4  alternativeName: Protease nexin-II shortName:PN-II  alternativeName: Protease nexin-II shortName:PN-II  component recommendedName: N-APP  /component component recommendedName: Soluble APP-alpha shortName:S-APP-alpha  /component component recommendedName: Soluble APP-beta shortName:S-APP-beta  /component component recommendedName: C99  /component component recommendedName: Beta-amyloid protein 42  alternativeName: Beta-APP42  /component component recommendedName: Beta-amyloid protein 40  alternativeName: Beta-APP40  /component component recommendedName: C83  /component component recommendedName: P3(42)  /component component recommendedName: P3(40)  /component component recommendedName: C80  /component component recommendedName: Gamma-secretase C-terminal fragment 59  alternativeName: Amyloid intracellular domain 59 shortName:AICD-59 shortName:AID(59)  alternativeName: Gamma-CTF(59)  /component component recommendedName: Gamma-secretase C-terminal fragment 57  alternativeName: Amyloid intracellular domain 57 shortName:AICD-57 shortName:AID(57)  alternativeName: Gamma-CTF(57)  /component component recommendedName: Gamma-secretase C-terminal fragment 50  alternativeName: Amyloid intracellular domain 50 shortName:AICD-50 shortName:AID(50)  alternativeName: Gamma-CTF(50)  /component component recommendedName: C31  /component\")\n	(checksum \"1\")\n	(comment \"INDUCTION Increased levels during neuronal differentiation.PTM Proteolytically processed under normal cellular conditions. Cleavage either by alpha-secretase, beta-secretase or theta-secretase leads to generation and extracellular release of soluble APP peptides, S-APP-alpha and S-APP-beta, and the retention of corresponding membrane-anchored C-terminal fragments, C80, C83 and C99. Subsequent processing of C80 and C83 by gamma-secretase yields P3 peptides. This is the major secretory pathway and is non-amyloidogenic. Alternatively, presenilin/nicastrin-mediated gamma-secretase processing of C99 releases the amyloid beta proteins, amyloid-beta 40 (Abeta40) and amyloid-beta 42 (Abeta42), major components of amyloid plaques, and the cytotoxic C-terminal fragments, gamma-CTF(50), gamma-CTF(57) and gamma-CTF(59).PTM Extracellular binding and reduction of copper, results in a corresponding oxidation of Cys-144 and Cys-158, and the formation of a disulfide bond. In vitro, the APP-Cu(+) complex in the presence of hydrogen peroxide results in an increased production of beta-amyloid-containing peptides.PTM Beta-amyloid peptides are degraded by IDE.MISCELLANEOUS Chelation of metal ions, notably copper, iron and zinc, can induce histidine-bridging between beta-amyloid molecules resulting in beta-amyloid-metal aggregates. The affinity for copper is much higher than for other transient metals and is increased under acidic conditions. Extracellular zinc-binding increases binding of heparin to APP and inhibits collagen-binding.SIMILARITY Belongs to the APP family.SIMILARITY Contains 1 BPTI/Kunitz inhibitor domain.\")\n	(DB_ID 69781)\n	(description \"recommendedName: Amyloid beta A4 protein  alternativeName: ABPP  alternativeName: APPI shortName:APP  alternativeName: APPI shortName:APP  alternativeName: Alzheimer disease amyloid protein  alternativeName: Cerebral vascular amyloid peptide shortName:CVAP  alternativeName: Cerebral vascular amyloid peptide shortName:CVAP  alternativeName: PreA4  alternativeName: Protease nexin-II shortName:PN-II  alternativeName: Protease nexin-II shortName:PN-II  component recommendedName: N-APP  /component component recommendedName: Soluble APP-alpha shortName:S-APP-alpha  /component component recommendedName: Soluble APP-beta shortName:S-APP-beta  /component component recommendedName: C99  /component component recommendedName: Beta-amyloid protein 42  alternativeName: Beta-APP42  /component component recommendedName: Beta-amyloid protein 40  alternativeName: Beta-APP40  /component component recommendedName: C83  /component component recommendedName: P3(42)  /component component recommendedName: P3(40)  /component component recommendedName: C80  /component component recommendedName: Gamma-secretase C-terminal fragment 59  alternativeName: Amyloid intracellular domain 59 shortName:AICD-59 shortName:AID(59)  alternativeName: Gamma-CTF(59)  /component component recommendedName: Gamma-secretase C-terminal fragment 57  alternativeName: Amyloid intracellular domain 57 shortName:AICD-57 shortName:AID(57)  alternativeName: Gamma-CTF(57)  /component component recommendedName: Gamma-secretase C-terminal fragment 50  alternativeName: Amyloid intracellular domain 50 shortName:AICD-50 shortName:AID(50)  alternativeName: Gamma-CTF(50)  /component component recommendedName: C31  /component\")\n	(geneName\n		\"APP\"\n		\"A4\"\n		\"AD1\")\n	(identifier \"P05067\")\n	(isSequenceChanged TRUE)\n	(keyword\n		\"3D-structure\"\n		\"Alternative splicing\"\n		\"Alzheimer disease\"\n		\"Amyloid\"\n		\"Amyloidosis\"\n		\"Apoptosis\"\n		\"Cell adhesion\"\n		\"Coated pit\"\n		\"Complete proteome\"\n		\"Copper\"\n		\"Direct protein sequencing\"\n		\"Disease mutation\"\n		\"Disulfide bond\"\n		\"Endocytosis\"\n		\"Glycoprotein\"\n		\"Heparin-binding\"\n		\"Iron\"\n		\"Membrane\"\n		\"Metal-binding\"\n		\"Neurodegeneration\"\n		\"Notch signaling pathway\"\n		\"Phosphoprotein\"\n		\"Polymorphism\"\n		\"Protease inhibitor\"\n		\"Proteoglycan\"\n		\"Serine protease inhibitor\"\n		\"Signal\"\n		\"Transmembrane\"\n		\"Transmembrane helix\"\n		\"Zinc\")\n	(name \"APP\")\n	(referenceDatabase [GK_2])\n	(secondaryIdentifier\n		\"A4_HUMAN\"\n		\"B2R5V1\"\n		\"D3DSD1\"\n		\"D3DSD2\"\n		\"D3DSD3\"\n		\"P09000\"\n		\"P78438\"\n		\"Q13764\"\n		\"Q13778\"\n		\"Q13793\"\n		\"Q16011\"\n		\"Q16014\"\n		\"Q16019\"\n		\"Q16020\"\n		\"Q6GSC0\"\n		\"Q8WZ99\"\n		\"Q9BT38\"\n		\"Q9UC33\"\n		\"Q9UCA9\"\n		\"Q9UCB6\"\n		\"Q9UCC8\"\n		\"Q9UCD1\"\n		\"Q9UQ58\")\n	(sequenceLength 770)\n	(species [GK_48887]))\n\n([GK_70101] of  EntityCompartment\n\n	(_displayName \"cytosol\")\n	(accession \"0005829\")\n	(DB_ID 70101)\n	(definition \"The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.\")\n	(instanceOf [GK_258163])\n	(name \"cytosol\")\n	(referenceDatabase [GK_1]))\n\n([GK_70121] of  Person\n\n	(_displayName \"Bell, GI\")\n	(DB_ID 70121)\n	(firstname \"Graeme I\")\n	(initial \"GI\")\n	(surname \"Bell\"))\n\n([GK_70650] of  Person\n\n	(_displayName \"Hohenester, E\")\n	(DB_ID 70650)\n	(firstname \"Erhard\")\n	(initial \"E\")\n	(surname \"Hohenester\"))\n\n([GK_71050] of  Reaction\n\n	(_displayName \"[Hsa] Trimerization of the FASL:FAS receptor complex\")\n	(_doRelease TRUE)\n	(compartment [GK_876])\n	(DB_ID 71050)\n	(figure [GK_141237])\n	(input\n		[GK_76564]\n		[GK_76564]\n		[GK_76564])\n	(literatureReference [GK_140505])\n	(name \"Trimerization of the FASL:FAS receptor complex\")\n	(output [GK_76195])\n	(precedingEvent [GK_75244])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_359726])\n	(summation [GK_179272]))\n\n([GK_72803] of  Taxon\n\n	(_displayName \"Coelomata\")\n	(crossReference [GK_72818])\n	(DB_ID 72803)\n	(name \"Coelomata\")\n	(superTaxon [GK_72804]))\n\n([GK_72804] of  Taxon\n\n	(_displayName \"Bilateria\")\n	(crossReference [GK_72817])\n	(DB_ID 72804)\n	(name \"Bilateria\")\n	(superTaxon [GK_72805]))\n\n([GK_72805] of  Taxon\n\n	(_displayName \"Eumetazoa\")\n	(crossReference [GK_72816])\n	(DB_ID 72805)\n	(name \"Eumetazoa\")\n	(superTaxon [GK_72806]))\n\n([GK_72806] of  Taxon\n\n	(_displayName \"Metazoa\")\n	(crossReference [GK_72815])\n	(DB_ID 72806)\n	(name\n		\"Metazoa\"\n		\"Animalia\"\n		\"animals\"\n		\"multicellular animals\")\n	(superTaxon [GK_72807]))\n\n([GK_72807] of  Taxon\n\n	(_displayName \"Fungi/Metazoa group\")\n	(crossReference [GK_72814])\n	(DB_ID 72807)\n	(name \"Fungi/Metazoa group\")\n	(superTaxon [GK_72808]))\n\n([GK_72808] of  Taxon\n\n	(_displayName \"Eukaryota\")\n	(crossReference [GK_72813])\n	(DB_ID 72808)\n	(name\n		\"Eukaryota\"\n		\"eucaryotes\")\n	(superTaxon [GK_72809]))\n\n([GK_72809] of  Taxon\n\n	(_displayName \"cellular_organisms\")\n	(crossReference [GK_72812])\n	(DB_ID 72809)\n	(name \"cellular_organisms\")\n	(superTaxon [GK_164487]))\n\n([GK_72810] of  ReferenceDatabase\n\n	(_displayName \"NCBI Taxonomy\")\n	(accessUrl \"http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=###ID###\")\n	(DB_ID 72810)\n	(name\n		\"NCBI Taxonomy\"\n		\"NCBI_taxonomy\")\n	(url \"http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html\"))\n\n([GK_72812] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:131567\")\n	(DB_ID 72812)\n	(identifier \"131567\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72813] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:2759\")\n	(DB_ID 72813)\n	(identifier \"2759\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72814] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:33154\")\n	(DB_ID 72814)\n	(identifier \"33154\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72815] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:33208\")\n	(DB_ID 72815)\n	(identifier \"33208\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72816] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:6072\")\n	(DB_ID 72816)\n	(identifier \"6072\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72817] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:33213\")\n	(DB_ID 72817)\n	(identifier \"33213\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72818] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:33316\")\n	(DB_ID 72818)\n	(identifier \"33316\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72862] of  Taxon\n\n	(_displayName \"Eutheria\")\n	(crossReference [GK_72887])\n	(DB_ID 72862)\n	(name \"Eutheria\")\n	(superTaxon [GK_72863]))\n\n([GK_72863] of  Taxon\n\n	(_displayName \"Theria\")\n	(crossReference [GK_72886])\n	(DB_ID 72863)\n	(name \"Theria\")\n	(superTaxon [GK_72864]))\n\n([GK_72864] of  Taxon\n\n	(_displayName \"Mammalia\")\n	(crossReference [GK_72885])\n	(DB_ID 72864)\n	(name \"Mammalia\")\n	(superTaxon [GK_72865]))\n\n([GK_72865] of  Taxon\n\n	(_displayName \"Amniota\")\n	(crossReference [GK_72884])\n	(DB_ID 72865)\n	(name \"Amniota\")\n	(superTaxon [GK_72866]))\n\n([GK_72866] of  Taxon\n\n	(_displayName \"Tetrapoda\")\n	(crossReference [GK_72883])\n	(DB_ID 72866)\n	(name \"Tetrapoda\")\n	(superTaxon [GK_72867]))\n\n([GK_72867] of  Taxon\n\n	(_displayName \"Sarcopterygii\")\n	(crossReference [GK_72882])\n	(DB_ID 72867)\n	(name \"Sarcopterygii\")\n	(superTaxon [GK_72868]))\n\n([GK_72868] of  Taxon\n\n	(_displayName \"Euteleostomi\")\n	(crossReference [GK_72881])\n	(DB_ID 72868)\n	(name \"Euteleostomi\")\n	(superTaxon [GK_72869]))\n\n([GK_72869] of  Taxon\n\n	(_displayName \"Teleostomi\")\n	(crossReference [GK_72880])\n	(DB_ID 72869)\n	(name \"Teleostomi\")\n	(superTaxon [GK_72870]))\n\n([GK_72870] of  Taxon\n\n	(_displayName \"Gnathostomata\")\n	(crossReference [GK_72879])\n	(DB_ID 72870)\n	(name \"Gnathostomata\")\n	(superTaxon [GK_72871]))\n\n([GK_72871] of  Taxon\n\n	(_displayName \"Vertebrata\")\n	(crossReference [GK_72878])\n	(DB_ID 72871)\n	(name\n		\"Vertebrata\"\n		\"vertebrates\")\n	(superTaxon [GK_72872]))\n\n([GK_72872] of  Taxon\n\n	(_displayName \"Craniata\")\n	(crossReference [GK_72877])\n	(DB_ID 72872)\n	(name \"Craniata\")\n	(superTaxon [GK_72873]))\n\n([GK_72873] of  Taxon\n\n	(_displayName \"Chordata\")\n	(crossReference [GK_72876])\n	(DB_ID 72873)\n	(name \"Chordata\")\n	(superTaxon [GK_72874]))\n\n([GK_72874] of  Taxon\n\n	(_displayName \"Deuterostomia\")\n	(crossReference [GK_72875])\n	(DB_ID 72874)\n	(name \"Deuterostomia\")\n	(superTaxon [GK_72803]))\n\n([GK_72875] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:33511\")\n	(DB_ID 72875)\n	(identifier \"33511\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72876] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:7711\")\n	(DB_ID 72876)\n	(identifier \"7711\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72877] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:89593\")\n	(DB_ID 72877)\n	(identifier \"89593\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72878] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:7742\")\n	(DB_ID 72878)\n	(identifier \"7742\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72879] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:7776\")\n	(DB_ID 72879)\n	(identifier \"7776\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72880] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:117570\")\n	(DB_ID 72880)\n	(identifier \"117570\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72881] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:117571\")\n	(DB_ID 72881)\n	(identifier \"117571\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72882] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:8287\")\n	(DB_ID 72882)\n	(identifier \"8287\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72883] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:32523\")\n	(DB_ID 72883)\n	(identifier \"32523\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72884] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:32524\")\n	(DB_ID 72884)\n	(identifier \"32524\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72885] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:40674\")\n	(DB_ID 72885)\n	(identifier \"40674\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72886] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:32525\")\n	(DB_ID 72886)\n	(identifier \"32525\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72887] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:9347\")\n	(DB_ID 72887)\n	(identifier \"9347\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72938] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:9606\")\n	(DB_ID 72938)\n	(identifier \"9606\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72939] of  Taxon\n\n	(_displayName \"Homo\")\n	(crossReference [GK_72948])\n	(DB_ID 72939)\n	(name \"Homo\")\n	(superTaxon [GK_72940]))\n\n([GK_72940] of  Taxon\n\n	(_displayName \"Homo/Pan/Gorilla group\")\n	(crossReference [GK_72947])\n	(DB_ID 72940)\n	(name \"Homo/Pan/Gorilla group\")\n	(superTaxon [GK_72941]))\n\n([GK_72941] of  Taxon\n\n	(_displayName \"Hominidae\")\n	(crossReference [GK_72946])\n	(DB_ID 72941)\n	(name \"Hominidae\")\n	(superTaxon [GK_72942]))\n\n([GK_72942] of  Taxon\n\n	(_displayName \"Catarrhini\")\n	(crossReference [GK_72945])\n	(DB_ID 72942)\n	(name \"Catarrhini\")\n	(superTaxon [GK_72943]))\n\n([GK_72943] of  Taxon\n\n	(_displayName \"Primates\")\n	(crossReference [GK_72944])\n	(DB_ID 72943)\n	(name \"Primates\")\n	(superTaxon [GK_72862]))\n\n([GK_72944] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:9443\")\n	(DB_ID 72944)\n	(identifier \"9443\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72945] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:9526\")\n	(DB_ID 72945)\n	(identifier \"9526\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72946] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:9604\")\n	(DB_ID 72946)\n	(identifier \"9604\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72947] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:207598\")\n	(DB_ID 72947)\n	(identifier \"207598\")\n	(referenceDatabase [GK_72810]))\n\n([GK_72948] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:9605\")\n	(DB_ID 72948)\n	(identifier \"9605\")\n	(referenceDatabase [GK_72810]))\n\n([GK_73093] of  Taxon\n\n	(_displayName \"Viruses\")\n	(crossReference [GK_73094])\n	(DB_ID 73093)\n	(name \"Viruses\")\n	(superTaxon [GK_164487]))\n\n([GK_73094] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:10239\")\n	(DB_ID 73094)\n	(identifier \"10239\")\n	(referenceDatabase [GK_72810]))\n\n([GK_73135] of  Taxon\n\n	(_displayName \"dsDNA viruses, no RNA stage\")\n	(crossReference [GK_73137])\n	(DB_ID 73135)\n	(name \"dsDNA viruses, no RNA stage\")\n	(superTaxon [GK_73093]))\n\n([GK_73137] of  DatabaseIdentifier\n\n	(_displayName \"NCBI_taxonomy:35237\")\n	(DB_ID 73137)\n	(identifier \"35237\")\n	(referenceDatabase [GK_72810]))\n\n([GK_73681] of  Person\n\n	(_displayName \"Gillespie, Marc E\")\n	(affiliation\n		[GK_77249]\n		[GK_158007]\n		[GK_69324])\n	(DB_ID 73681)\n	(eMailAddress \"gillespm@cshl.org\")\n	(firstname \"Marc E\")\n	(initial \"ME\")\n	(surname \"Gillespie\"))\n\n([GK_73887] of  Pathway\n\n	(_displayName \"[Hsa] Death Receptor  Signalling\")\n	(_doRelease TRUE)\n	(authored [GK_139957])\n	(compartment [GK_356])\n	(DB_ID 73887)\n	(edited [GK_165085])\n	(goBiologicalProcess [GK_2133497])\n	(hasEvent\n		[GK_75157]\n		[GK_75893]\n		[GK_75158])\n	(name \"Death Receptor  Signalling\")\n	(reviewed\n		[GK_165094]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_361140])\n	(summation [GK_139958]))\n\n([GK_73945] of  Reaction\n\n	(_displayName \"[Hsa] FAS-mediated dimerization of procaspase-8\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 73945)\n	(edited [GK_3508822])\n	(figure [GK_141227])\n	(input\n		[GK_75114]\n		[GK_57031])\n	(literatureReference\n		[GK_141158]\n		[GK_2984185]\n		[GK_1267807])\n	(name \"FAS-mediated dimerization of procaspase-8\")\n	(output [GK_2671811])\n	(precedingEvent [GK_83586])\n	(reviewed\n		[GK_2594924]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_359070])\n	(summation [GK_178760]))\n\n([GK_74112] of  SimpleEntity\n\n	(_displayName \"Ca2+ [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_29497])\n	(DB_ID 74112)\n	(name\n		\"Ca2+\"\n		\"Calcium\")\n	(referenceEntity [GK_114768])\n	(stableIdentifier [GK_363887]))\n\n([GK_74277] of  Complex\n\n	(_displayName \"[Hsa] TNF-alpha:TNF-R1 [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 74277)\n	(hasComponent\n		[GK_66344]\n		[GK_66344]\n		[GK_66344]\n		[GK_3371351])\n	(literatureReference [GK_3371380])\n	(name \"TNF-alpha:TNF-R1\")\n	(species [GK_48887])\n	(stableIdentifier [GK_367488]))\n\n([GK_74302] of  Summation\n\n	(_displayName \"Once the TNF-aplha:TNF-R1:TRADD complex is formed the two TN...\")\n	(DB_ID 74302)\n	(text \"Once the TNF-aplha:TNF-R1:TRADD complex is formed the two TNF-alpha mediated pathways are possible. The variable is the recruitment of FADD to the larger complex formed by the TNF-aplha:TNF-R1 platform via the interaction of the Death Domains. The steps leading to the Jun, NF kappaB, or apoptotic pathways are rife with modulation.\"))\n\n([GK_74353] of  Summation\n\n	(_displayName \"Once formed the TRADD:TRAF2:RIP1 complex may dissociate from...\")\n	(DB_ID 74353)\n	(text \"Once formed the TRADD:TRAF2:RIP1 complex may dissociate from the TNF:TNF-R1 platform and become cytosolic. If this complex recruits FADD then the cell will be pushed along the apoptotic pathway.\"))\n\n([GK_75114] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer:FADD:pro-Caspase-8 DISC [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 75114)\n	(hasComponent\n		[GK_43124]\n		[GK_57031])\n	(literatureReference\n		[GK_3465531]\n		[GK_3465522])\n	(name \"FASL:FAS Receptor Trimer:FADD:pro-Caspase-8 DISC\")\n	(species [GK_48887])\n	(stableIdentifier [GK_362769]))\n\n([GK_75146] of  Reaction\n\n	(_displayName \"[Hsa] TRAIL:TRAIL-Receptor2 Trimer:FADD complex binds Caspase-8\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 75146)\n	(input\n		[GK_57031]\n		[GK_141137])\n	(literatureReference [GK_141128])\n	(name \"TRAIL:TRAIL-Receptor2 Trimer:FADD complex binds Caspase-8\")\n	(output [GK_141133])\n	(precedingEvent [GK_75187])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_359034])\n	(summation [GK_179247]))\n\n([GK_75157] of  Pathway\n\n	(_displayName \"[Hsa] FasL/ CD95L signaling\")\n	(_doRelease TRUE)\n	(authored [GK_140480])\n	(compartment [GK_356])\n	(DB_ID 75157)\n	(edited [GK_165085])\n	(figure [GK_141234])\n	(goBiologicalProcess [GK_2133497])\n	(hasEvent\n		[GK_75244]\n		[GK_71050]\n		[GK_83650]\n		[GK_83586]\n		[GK_141310])\n	(literatureReference [GK_139960])\n	(name \"FasL/ CD95L signaling\")\n	(reviewed\n		[GK_165094]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_360947])\n	(summation [GK_140481]))\n\n([GK_75158] of  Pathway\n\n	(_displayName \"[Hsa] TRAIL  signaling\")\n	(_doRelease TRUE)\n	(authored [GK_141084])\n	(compartment [GK_356])\n	(DB_ID 75158)\n	(edited [GK_165085])\n	(hasEvent\n		[GK_75238]\n		[GK_141139]\n		[GK_75187]\n		[GK_75146]\n		[GK_141316])\n	(literatureReference [GK_140992])\n	(name \"TRAIL  signaling\")\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_358827])\n	(summation [GK_141083]))\n\n([GK_75187] of  Reaction\n\n	(_displayName \"[Hsa] TRAIL:TRAIL receptor-2 Trimer Binds FADD\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 75187)\n	(input\n		[GK_141126]\n		[GK_54639]\n		[GK_54639])\n	(literatureReference\n		[GK_141128]\n		[GK_3371373])\n	(name \"TRAIL:TRAIL receptor-2 Trimer Binds FADD\")\n	(output [GK_141137])\n	(precedingEvent [GK_75238])\n	(reviewed [GK_3597630])\n	(revised [GK_3508822])\n	(species [GK_48887])\n	(stableIdentifier [GK_362019])\n	(summation [GK_179245]))\n\n([GK_75238] of  Reaction\n\n	(_displayName \"[Hsa] TRAIL Binds TRAIL-Receptor2\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_984])\n	(DB_ID 75238)\n	(input\n		[GK_66186]\n		[GK_65505])\n	(literatureReference [GK_141147])\n	(name \"TRAIL Binds TRAIL-Receptor2\")\n	(output [GK_141118])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_358527])\n	(summation [GK_179243]))\n\n([GK_75240] of  Reaction\n\n	(_displayName \"[Hsa] TRADD:TRAF2:RIP1:FADD complex binds Pro-Caspase 8  \")\n	(_doRelease TRUE)\n	(compartment [GK_70101])\n	(DB_ID 75240)\n	(input\n		[GK_57031]\n		[GK_140977])\n	(literatureReference [GK_140979])\n	(name \"TRADD:TRAF2:RIP1:FADD complex binds Pro-Caspase 8  \")\n	(output [GK_140976])\n	(precedingEvent [GK_140978])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_360371])\n	(summation [GK_179240]))\n\n([GK_75244] of  Reaction\n\n	(_displayName \"[Hsa] FASL binds FAS Receptor\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_984])\n	(DB_ID 75244)\n	(figure [GK_141237])\n	(goBiologicalProcess [GK_2133497])\n	(input\n		[GK_66212]\n		[GK_66248])\n	(literatureReference [GK_140513])\n	(name \"FASL binds FAS Receptor\")\n	(output [GK_76564])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_361795])\n	(summation [GK_178761]))\n\n([GK_75542] of  Person\n\n	(_displayName \"Teplow, DB\")\n	(DB_ID 75542)\n	(firstname \"D B\")\n	(initial \"DB\")\n	(surname \"Teplow\"))\n\n([GK_75579] of  Person\n\n	(_displayName \"Klisak, I\")\n	(DB_ID 75579)\n	(firstname \"I\")\n	(initial \"I\")\n	(surname \"Klisak\"))\n\n([GK_75582] of  Person\n\n	(_displayName \"Mohandas, T\")\n	(DB_ID 75582)\n	(firstname \"T\")\n	(initial \"T\")\n	(surname \"Mohandas\"))\n\n([GK_75583] of  Person\n\n	(_displayName \"Sparkes, RS\")\n	(DB_ID 75583)\n	(firstname \"R S\")\n	(initial \"RS\")\n	(surname \"Sparkes\"))\n\n([GK_75642] of  Person\n\n	(_displayName \"Lem, J\")\n	(DB_ID 75642)\n	(firstname \"Janis\")\n	(initial \"J\")\n	(surname \"Lem\"))\n\n([GK_75643] of  Person\n\n	(_displayName \"Simon, M\")\n	(DB_ID 75643)\n	(firstname \"Melvin\")\n	(initial \"M\")\n	(surname \"Simon\"))\n\n([GK_75697] of  Person\n\n	(_displayName \"Goeddel, DV\")\n	(DB_ID 75697)\n	(firstname \"D V\")\n	(initial \"DV\")\n	(surname \"Goeddel\"))\n\n([GK_75893] of  Pathway\n\n	(_displayName \"[Hsa] TNF signaling\")\n	(_doRelease TRUE)\n	(authored [GK_140936])\n	(compartment [GK_356])\n	(DB_ID 75893)\n	(edited [GK_3371404])\n	(hasEvent\n		[GK_3371385]\n		[GK_3371353]\n		[GK_83660]\n		[GK_83656]\n		[GK_83582]\n		[GK_140978]\n		[GK_75240])\n	(literatureReference [GK_140939])\n	(name \"TNF signaling\")\n	(reviewed [GK_3597630])\n	(revised [GK_3632873])\n	(species [GK_48887])\n	(stableIdentifier [GK_361228])\n	(summation [GK_140942]))\n\n([GK_75975] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CASP8(217-374) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 75975)\n	(endCoordinate 374)\n	(name\n		\"CASP8(217-374)\"\n		\"CASP8\"\n		\"p18 subunit of Caspase 8\")\n	(referenceEntity [GK_57032])\n	(species [GK_48887])\n	(stableIdentifier [GK_365034])\n	(startCoordinate 217))\n\n([GK_76158] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] CASP8(385-479) [cytosol]\")\n	(compartment [GK_70101])\n	(DB_ID 76158)\n	(endCoordinate 479)\n	(name\n		\"CASP8(385-479)\"\n		\"p10 subunit of Caspase 8\")\n	(referenceEntity [GK_57032])\n	(species [GK_48887])\n	(stableIdentifier [GK_367761])\n	(startCoordinate 385))\n\n([GK_76195] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 76195)\n	(hasComponent\n		[GK_76564]\n		[GK_76564]\n		[GK_76564])\n	(name \"FASL:FAS Receptor Trimer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_371020]))\n\n([GK_76564] of  Complex\n\n	(_displayName \"[Hsa] FASL:FAS Receptor monomer [plasma membrane]\")\n	(compartment [GK_876])\n	(DB_ID 76564)\n	(hasComponent\n		[GK_66212]\n		[GK_66248])\n	(name \"FASL:FAS Receptor monomer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_365522]))\n\n([GK_76786] of  Person\n\n	(_displayName \"Seto, Y\")\n	(DB_ID 76786)\n	(firstname \"Y\")\n	(initial \"Y\")\n	(surname \"Seto\"))\n\n([GK_77151] of  Person\n\n	(_displayName \"el-Deiry, WS\")\n	(DB_ID 77151)\n	(firstname \"W S\")\n	(initial \"WS\")\n	(surname \"el-Deiry\"))\n\n([GK_77249] of  Affiliation\n\n	(_displayName \"St. John\'s University\")\n	(address \"8000 Utopia Parkway\")\n	(DB_ID 77249)\n	(name \"St. John\'s University\"))\n\n([GK_77293] of  Person\n\n	(_displayName \"Tschopp, J\")\n	(affiliation [GK_157508])\n	(DB_ID 77293)\n	(eMailAddress \"jtschopp@mail-eliot.unil.ch\")\n	(firstname \"Jurg\")\n	(initial \"J\")\n	(surname \"Tschopp\"))\n\n([GK_80155] of  GO_CellularComponent\n\n	(_displayName \"host cell cytoplasm\")\n	(accession \"0030430\")\n	(DB_ID 80155)\n	(definition \"The cytoplasm of a host cell.\")\n	(instanceOf [GK_259434])\n	(name \"host cell cytoplasm\")\n	(referenceDatabase [GK_1]))\n\n([GK_83582] of  Reaction\n\n	(_displayName \"[Hsa] TRADD:TRAF2:RIP1 complex dissociates from the TNF-alpha:TNF-R1 complex.\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 83582)\n	(edited [GK_3508822])\n	(input [GK_140946])\n	(literatureReference\n		[GK_140979]\n		[GK_3465531])\n	(name \"TRADD:TRAF2:RIP1 complex dissociates from the TNF-alpha:TNF-R1 complex.\")\n	(output\n		[GK_140935]\n		[GK_3371401])\n	(precedingEvent [GK_83656])\n	(reviewed\n		[GK_165094]\n		[GK_3597630])\n	(revised [GK_3604387])\n	(species [GK_48887])\n	(stableIdentifier [GK_358712])\n	(summation [GK_74353]))\n\n([GK_83586] of  Reaction\n\n	(_displayName \"[Hsa] FASL:FAS Receptor Trimer:FADD complex binds pro-Caspase-8\")\n	(_doRelease TRUE)\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 83586)\n	(figure [GK_141228])\n	(input\n		[GK_43124]\n		[GK_57031])\n	(literatureReference [GK_140941])\n	(name \"FASL:FAS Receptor Trimer:FADD complex binds pro-Caspase-8\")\n	(output [GK_75114])\n	(precedingEvent [GK_83650])\n	(reviewed [GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_360537])\n	(summation [GK_178764]))\n\n([GK_83650] of  Reaction\n\n	(_displayName \"[Hsa] FasL:Fas binds FADD\")\n	(_doRelease TRUE)\n	(authored [GK_386948])\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 83650)\n	(edited [GK_3508822])\n	(figure [GK_141226])\n	(input\n		[GK_76195]\n		[GK_54639]\n		[GK_54639])\n	(literatureReference [GK_140495])\n	(name \"FasL:Fas binds FADD\")\n	(output [GK_43124])\n	(precedingEvent [GK_71050])\n	(reviewed\n		[GK_386956]\n		[GK_3597630])\n	(species [GK_48887])\n	(stableIdentifier [GK_362003])\n	(summation [GK_178762]))\n\n([GK_83656] of  Reaction\n\n	(_displayName \"[Hsa] TNF:TNF-R1 binds TRADD, TRAF2 and RIP \")\n	(_doRelease TRUE)\n	(authored [GK_386948])\n	(compartment\n		[GK_876]\n		[GK_70101])\n	(DB_ID 83656)\n	(edited [GK_3371404])\n	(input\n		[GK_66376]\n		[GK_168651]\n		[GK_66370]\n		[GK_3371401]\n		[GK_66376]\n		[GK_66376]\n		[GK_66370]\n		[GK_66370])\n	(literatureReference [GK_140948])\n	(name \"TNF:TNF-R1 binds TRADD, TRAF2 and RIP \")\n	(output [GK_140946])\n	(precedingEvent\n		[GK_83660]\n		[GK_3371353])\n	(reviewed\n		[GK_386956]\n		[GK_3597630])\n	(revised [GK_3465563])\n	(species [GK_48887])\n	(stableIdentifier [GK_361068])\n	(summation [GK_74302]))\n\n([GK_83660] of  Reaction\n\n	(_displayName \"[Hsa] Membrane-anchored TNF-alpha binds TNF-R1\")\n	(_doRelease TRUE)\n	(compartment [GK_876])\n	(DB_ID 83660)\n	(edited [GK_3371404])\n	(input\n		[GK_66344]\n		[GK_3371351]\n		[GK_66344]\n		[GK_66344])\n	(literatureReference\n		[GK_140945]\n		[GK_3371352])\n	(name \"Membrane-anchored TNF-alpha binds TNF-R1\")\n	(output [GK_74277])\n	(reviewed [GK_3597630])\n	(revised [GK_3465518])\n	(species [GK_48887])\n	(stableIdentifier [GK_360533])\n	(summation [GK_3465576]))\n\n([GK_84039] of  Person\n\n	(_displayName \"Wang, J\")\n	(DB_ID 84039)\n	(firstname \"J\")\n	(initial \"J\")\n	(surname \"Wang\"))\n\n([GK_876] of  EntityCompartment\n\n	(_displayName \"plasma membrane\")\n	(accession \"0005886\")\n	(componentOf [GK_1042833])\n	(DB_ID 876)\n	(definition \"The membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins.\")\n	(instanceOf\n		[GK_383187]\n		[GK_258147])\n	(name \"plasma membrane\")\n	(referenceDatabase [GK_1]))\n\n([GK_879187] of  Person\n\n	(_displayName \"Sperandio, S\")\n	(DB_ID 879187)\n	(initial \"S\")\n	(surname \"Sperandio\"))\n\n([GK_879189] of  Person\n\n	(_displayName \"Sutherlin, DP\")\n	(DB_ID 879189)\n	(firstname \"Daniel P\")\n	(initial \"DP\")\n	(surname \"Sutherlin\"))\n\n([GK_879190] of  Person\n\n	(_displayName \"Ricci, JE\")\n	(DB_ID 879190)\n	(initial \"JE\")\n	(surname \"Ricci\"))\n\n([GK_879191] of  Person\n\n	(_displayName \"Scott, FL\")\n	(DB_ID 879191)\n	(initial \"FL\")\n	(surname \"Scott\"))\n\n([GK_879193] of  LiteratureReference\n\n	(_displayName \"A unified model for apical caspase activation\")\n	(author\n		[GK_141143]\n		[GK_879204]\n		[GK_879191]\n		[GK_879187]\n		[GK_452043]\n		[GK_879196]\n		[GK_879190]\n		[GK_879224]\n		[GK_879189]\n		[GK_141937]\n		[GK_141140])\n	(DB_ID 879193)\n	(journal \"Mol Cell\")\n	(pages \"529-41\")\n	(pubMedIdentifier 12620239)\n	(title \"A unified model for apical caspase activation\")\n	(volume 11)\n	(year 2003))\n\n([GK_879196] of  Person\n\n	(_displayName \"Pedersen, IM\")\n	(DB_ID 879196)\n	(initial \"IM\")\n	(surname \"Pedersen\"))\n\n([GK_879204] of  Person\n\n	(_displayName \"Renatus, M\")\n	(DB_ID 879204)\n	(initial \"M\")\n	(surname \"Renatus\"))\n\n([GK_879224] of  Person\n\n	(_displayName \"Edris, WA\")\n	(DB_ID 879224)\n	(initial \"WA\")\n	(surname \"Edris\"))\n\n([GK_879340] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Beta-amyloid protein 42 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 879340)\n	(endCoordinate 713)\n	(name\n		\"Beta-amyloid protein 42\"\n		\"Amyloid beta A4 protein\"\n		\"APP\"\n		\"A4_HUMAN\")\n	(referenceEntity [GK_69781])\n	(species [GK_48887])\n	(stableIdentifier [GK_1110572])\n	(startCoordinate 672))\n\n([GK_879404] of  Person\n\n	(_displayName \"Perry, G\")\n	(affiliation [GK_1247883])\n	(DB_ID 879404)\n	(firstname \"George\")\n	(initial \"G\")\n	(surname \"Perry\"))\n\n([GK_893611] of  Person\n\n	(_displayName \"Gibson, G\")\n	(DB_ID 893611)\n	(firstname \"Gordon\")\n	(initial \"G\")\n	(surname \"Gibson\"))\n\n([GK_913920] of  Person\n\n	(_displayName \"Aebi, U\")\n	(DB_ID 913920)\n	(initial \"U\")\n	(surname \"Aebi\"))\n\n([GK_936419] of  Person\n\n	(_displayName \"Wright, A\")\n	(DB_ID 936419)\n	(initial \"A\")\n	(surname \"Wright\"))\n\n([GK_947995] of  Person\n\n	(_displayName \"Nichols, WC\")\n	(DB_ID 947995)\n	(initial \"WC\")\n	(surname \"Nichols\"))\n\n([GK_97] of  GO_MolecularFunction\n\n	(_displayName \"catalytic activity\")\n	(accession \"0003824\")\n	(componentOf [GK_504])\n	(DB_ID 97)\n	(definition \"Catalysis of a biochemical reaction at physiological temperatures. In biologically catalyzed reactions, the reactants are known as substrates, and the catalysts are naturally occurring macromolecular substances known as enzymes. Enzymes possess specific binding sites for substrates, and are usually composed wholly or largely of protein, but RNA that has catalytic activity (ribozyme) is often also regarded as enzymatic.\")\n	(instanceOf [GK_43])\n	(name \"catalytic activity\")\n	(referenceDatabase [GK_1]))\n\n([GK_976720] of  Summation\n\n	(_displayName \"Serum amyloid P component (SAP) is a member of the pentraxin...\")\n	(DB_ID 976720)\n	(literatureReference [GK_976803])\n	(text \"Serum amyloid P component (SAP) is a member of the pentraxin family, characterized by the formation of pentameric ring structures. Each member of the ring has two associated calcium ions. SAP is an acute phase reactant, highly induced by IL-6. It has 50% homology with the related C-reactive peptide. \"))\n\n([GK_976721] of  Person\n\n	(_displayName \"Butler, PJ\")\n	(DB_ID 976721)\n	(initial \"PJ\")\n	(surname \"Butler\"))\n\n([GK_976723] of  Reaction\n\n	(_displayName \"[Hsa] Serum amyloid P-component forms homopentamers\")\n	(_doRelease TRUE)\n	(authored [GK_977142])\n	(compartment [GK_984])\n	(DB_ID 976723)\n	(edited [GK_1247884])\n	(input\n		[GK_976737]\n		[GK_976737]\n		[GK_976737]\n		[GK_976737]\n		[GK_976737]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112])\n	(isChimeric FALSE)\n	(literatureReference [GK_976803])\n	(name \"Serum amyloid P-component forms homopentamers\")\n	(output [GK_976776])\n	(releaseDate \"2011-06-14\")\n	(reviewed [GK_1247882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357634])\n	(summation [GK_976720]))\n\n([GK_976725] of  Person\n\n	(_displayName \"White, HE\")\n	(DB_ID 976725)\n	(initial \"HE\")\n	(surname \"White\"))\n\n([GK_976727] of  Person\n\n	(_displayName \"O\'Hara, BP\")\n	(DB_ID 976727)\n	(initial \"BP\")\n	(surname \"O\'Hara\"))\n\n([GK_976731] of  Person\n\n	(_displayName \"Wood, SP\")\n	(DB_ID 976731)\n	(initial \"SP\")\n	(surname \"Wood\"))\n\n([GK_976733] of  Person\n\n	(_displayName \"Tickle, IJ\")\n	(DB_ID 976733)\n	(initial \"IJ\")\n	(surname \"Tickle\"))\n\n([GK_976734] of  Reaction\n\n	(_displayName \"[Hsa] Amyloid fibrils have additional components\")\n	(_doRelease TRUE)\n	(authored [GK_977142])\n	(compartment [GK_984])\n	(DB_ID 976734)\n	(edited [GK_1247884])\n	(input\n		[GK_977144]\n		[GK_977180]\n		[GK_976776]\n		[GK_1011570])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_976742]\n		[GK_977126]\n		[GK_977109]\n		[GK_977094]\n		[GK_977121])\n	(name \"Amyloid fibrils have additional components\")\n	(output [GK_977084])\n	(precedingEvent\n		[GK_976723]\n		[GK_977136])\n	(releaseDate \"2011-06-14\")\n	(reviewed [GK_1247882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357637])\n	(summation [GK_977174]))\n\n([GK_976737] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Serum amyloid P-component [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976737)\n	(endCoordinate 223)\n	(name\n		\"Serum amyloid P-component\"\n		\"APCS\"\n		\"SAMP_HUMAN\"\n		\"APCS\")\n	(referenceEntity [GK_64128])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358325])\n	(startCoordinate 20))\n\n([GK_976739] of  Person\n\n	(_displayName \"Pepys, MB\")\n	(DB_ID 976739)\n	(initial \"MB\")\n	(surname \"Pepys\"))\n\n([GK_976740] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Beta-amyloid protein 40 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976740)\n	(endCoordinate 711)\n	(name\n		\"Beta-amyloid protein 40\"\n		\"APP\"\n		\"A4_HUMAN\"\n		\"APP\")\n	(referenceEntity [GK_69781])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358524])\n	(startCoordinate 672))\n\n([GK_976741] of  Person\n\n	(_displayName \"Dyck, RF\")\n	(DB_ID 976741)\n	(initial \"RF\")\n	(surname \"Dyck\"))\n\n([GK_976742] of  LiteratureReference\n\n	(_displayName \"Binding of serum amyloid P-component (SAP) by amyloid fibrils\")\n	(author\n		[GK_976739]\n		[GK_976741]\n		[GK_976780]\n		[GK_976771]\n		[GK_976747])\n	(DB_ID 976742)\n	(journal \"Clin Exp Immunol\")\n	(pages \"284-93\")\n	(pubMedIdentifier 118839)\n	(title \"Binding of serum amyloid P-component (SAP) by amyloid fibrils\")\n	(volume 38)\n	(year 1979))\n\n([GK_976746] of  Person\n\n	(_displayName \"Srinivasan, N\")\n	(DB_ID 976746)\n	(initial \"N\")\n	(surname \"Srinivasan\"))\n\n([GK_976747] of  Person\n\n	(_displayName \"Cohen, AS\")\n	(DB_ID 976747)\n	(initial \"AS\")\n	(surname \"Cohen\"))\n\n([GK_976748] of  Polymer\n\n	(_displayName \"[Hsa] Beta amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976748)\n	(name \"Beta amyloid fibril\")\n	(repeatedUnit [GK_976800])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357699]))\n\n([GK_976760] of  DefinedSet\n\n	(_displayName \"[Hsa] Systemic amyloid fibril main peptide chains [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976760)\n	(hasMember\n		[GK_976898]\n		[GK_976889]\n		[GK_976945]\n		[GK_976929]\n		[GK_976893]\n		[GK_976916]\n		[GK_976919]\n		[GK_976860]\n		[GK_976911]\n		[GK_976935]\n		[GK_976871])\n	(literatureReference [GK_976859])\n	(name \"Systemic amyloid fibril main peptide chains\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358393])\n	(summation [GK_976920]))\n\n([GK_976771] of  Person\n\n	(_displayName \"Skinner, M\")\n	(DB_ID 976771)\n	(initial \"M\")\n	(surname \"Skinner\"))\n\n([GK_976776] of  Complex\n\n	(_displayName \"[Hsa] Serum amyloid P-component homopentamer [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976776)\n	(hasComponent\n		[GK_976737]\n		[GK_976737]\n		[GK_976737]\n		[GK_976737]\n		[GK_976737]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112]\n		[GK_74112])\n	(isChimeric FALSE)\n	(name \"Serum amyloid P-component homopentamer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358060]))\n\n([GK_976780] of  Person\n\n	(_displayName \"de Beer, FC\")\n	(DB_ID 976780)\n	(initial \"FC\")\n	(surname \"de Beer\"))\n\n([GK_976786] of  Person\n\n	(_displayName \"Oliva, G\")\n	(DB_ID 976786)\n	(initial \"G\")\n	(surname \"Oliva\"))\n\n([GK_976787] of  Complex\n\n	(_displayName \"[Hsa] Serum amyloid P decamer [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976787)\n	(hasComponent\n		[GK_976776]\n		[GK_976776])\n	(isChimeric FALSE)\n	(name \"Serum amyloid P decamer\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358195]))\n\n([GK_976790] of  LiteratureReference\n\n	(_displayName \"Serum amyloid P component is the major calcium-dependent specific DNA binding protein of the serum\")\n	(author\n		[GK_976739]\n		[GK_976721])\n	(DB_ID 976790)\n	(journal \"Biochem Biophys Res Commun\")\n	(pages \"308-13\")\n	(pubMedIdentifier 3675579)\n	(title \"Serum amyloid P component is the major calcium-dependent specific DNA binding protein of the serum\")\n	(volume 148)\n	(year 1987))\n\n([GK_976796] of  OpenSet\n\n	(_displayName \"Double-stranded DNA [extracellular region]\")\n	(compartment [GK_984])\n	(crossReference [GK_30151])\n	(DB_ID 976796)\n	(name \"Double-stranded DNA\")\n	(referenceEntity [GK_114961])\n	(stableIdentifier [GK_1358566]))\n\n([GK_976798] of  Person\n\n	(_displayName \"Emsley, J\")\n	(DB_ID 976798)\n	(initial \"J\")\n	(surname \"Emsley\"))\n\n([GK_976800] of  DefinedSet\n\n	(_displayName \"[Hsa] Beta-amyloid [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976800)\n	(hasMember\n		[GK_976740]\n		[GK_879340])\n	(name \"Beta-amyloid\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358579]))\n\n([GK_976803] of  LiteratureReference\n\n	(_displayName \"Structure of pentameric human serum amyloid P component\")\n	(author\n		[GK_976798]\n		[GK_976725]\n		[GK_976727]\n		[GK_976786]\n		[GK_976746]\n		[GK_976733]\n		[GK_976811]\n		[GK_976739]\n		[GK_976731])\n	(DB_ID 976803)\n	(journal \"Nature\")\n	(pages \"338-45\")\n	(pubMedIdentifier 8114934)\n	(title \"Structure of pentameric human serum amyloid P component\")\n	(volume 367)\n	(year 1994))\n\n([GK_976806] of  Person\n\n	(_displayName \"Hirschfield, GM\")\n	(DB_ID 976806)\n	(initial \"GM\")\n	(surname \"Hirschfield\"))\n\n([GK_976811] of  Person\n\n	(_displayName \"Blundell, TL\")\n	(DB_ID 976811)\n	(initial \"TL\")\n	(surname \"Blundell\"))\n\n([GK_976815] of  Summation\n\n	(_displayName \"At physiological pH serum amyloid P component is a decamer o...\")\n	(DB_ID 976815)\n	(literatureReference [GK_976803])\n	(text \"At physiological pH serum amyloid P component is a decamer of two pentameric rings lying face to face. This non-covalent interaction is readily dissociated by reducing the pH.\"))\n\n([GK_976817] of  Reaction\n\n	(_displayName \"[Hsa] Formation of serum amyloid P decamer\")\n	(_doRelease TRUE)\n	(authored [GK_977142])\n	(compartment [GK_984])\n	(DB_ID 976817)\n	(edited [GK_1247884])\n	(input\n		[GK_976776]\n		[GK_976776])\n	(isChimeric FALSE)\n	(literatureReference [GK_976803])\n	(name \"Formation of serum amyloid P decamer\")\n	(output [GK_976787])\n	(precedingEvent\n		[GK_976723]\n		[GK_976723])\n	(releaseDate \"2011-06-14\")\n	(reviewed [GK_1247882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357610])\n	(summation [GK_976815]))\n\n([GK_976821] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] TNF(77-233) [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976821)\n	(endCoordinate 233)\n	(name\n		\"TNF(77-233)\"\n		\"TNF-alpha\"\n		\"Tumor necrosis factor\"\n		\"TNF\"\n		\"TNFA_HUMAN\"\n		\"Cachectin\")\n	(referenceEntity [GK_66221])\n	(species [GK_48887])\n	(stableIdentifier [GK_1225400])\n	(startCoordinate 77))\n\n([GK_976849] of  Person\n\n	(_displayName \"Murphy, CL\")\n	(DB_ID 976849)\n	(initial \"CL\")\n	(surname \"Murphy\"))\n\n([GK_976850] of  LiteratureReference\n\n	(_displayName \"Serum amyloid A and protein AA: molecular mechanisms of a transmissible amyloidosis\")\n	(author\n		[GK_374884]\n		[GK_976884])\n	(DB_ID 976850)\n	(journal \"FEBS Lett\")\n	(pages \"2685-90\")\n	(pubMedIdentifier 19393650)\n	(title \"Serum amyloid A and protein AA: molecular mechanisms of a transmissible amyloidosis\")\n	(volume 583)\n	(year 2009))\n\n([GK_976851] of  Person\n\n	(_displayName \"Terry, CJ\")\n	(DB_ID 976851)\n	(initial \"CJ\")\n	(surname \"Terry\"))\n\n([GK_976852] of  LiteratureReference\n\n	(_displayName \"Gelsolin-related amyloidosis. Identification of the amyloid protein in Finnish hereditary amyloidosis as a fragment of variant gelsolin\")\n	(author [GK_976854])\n	(DB_ID 976852)\n	(journal \"J Clin Invest\")\n	(pages \"1195-9\")\n	(pubMedIdentifier 1849145)\n	(title \"Gelsolin-related amyloidosis. Identification of the amyloid protein in Finnish hereditary amyloidosis as a fragment of variant gelsolin\")\n	(volume 87)\n	(year 1991))\n\n([GK_976853] of  Person\n\n	(_displayName \"Dwulet, FE\")\n	(DB_ID 976853)\n	(initial \"FE\")\n	(surname \"Dwulet\"))\n\n([GK_976854] of  Person\n\n	(_displayName \"Maury, CP\")\n	(DB_ID 976854)\n	(initial \"CP\")\n	(surname \"Maury\"))\n\n([GK_976855] of  Person\n\n	(_displayName \"Abrahamson, M\")\n	(DB_ID 976855)\n	(initial \"M\")\n	(surname \"Abrahamson\"))\n\n([GK_976856] of  Person\n\n	(_displayName \"Totty, N\")\n	(DB_ID 976856)\n	(initial \"N\")\n	(surname \"Totty\"))\n\n([GK_976857] of  Person\n\n	(_displayName \"Gozu, M\")\n	(DB_ID 976857)\n	(initial \"M\")\n	(surname \"Gozu\"))\n\n([GK_976858] of  Person\n\n	(_displayName \"Solomon, A\")\n	(DB_ID 976858)\n	(initial \"A\")\n	(surname \"Solomon\"))\n\n([GK_976859] of  LiteratureReference\n\n	(_displayName \"Aspects on human amyloid forms and their fibril polypeptides\")\n	(author [GK_976884])\n	(DB_ID 976859)\n	(journal \"FEBS J\")\n	(pages \"5942-9\")\n	(pubMedIdentifier 16302959)\n	(title \"Aspects on human amyloid forms and their fibril polypeptides\")\n	(volume 272)\n	(year 2005))\n\n([GK_976860] of  Polymer\n\n	(_displayName \"[Hsa] Variant lysozyme C fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976860)\n	(literatureReference [GK_976877])\n	(name \"Variant lysozyme C fibril\")\n	(repeatedUnit [GK_976944])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358239]))\n\n([GK_976861] of  Person\n\n	(_displayName \"Guenther, B\")\n	(DB_ID 976861)\n	(initial \"B\")\n	(surname \"Guenther\"))\n\n([GK_976862] of  LiteratureReference\n\n	(_displayName \"Polymorphism in the intermediates and products of amyloid assembly\")\n	(author\n		[GK_976866]\n		[GK_976917])\n	(DB_ID 976862)\n	(journal \"Curr Opin Struct Biol\")\n	(pages \"48-57\")\n	(pubMedIdentifier 17251001)\n	(title \"Polymorphism in the intermediates and products of amyloid assembly\")\n	(volume 17)\n	(year 2007))\n\n([GK_976863] of  ReplacedResidue\n\n	(_displayName \"L-glutamine residue 94 replaced with unknown\")\n	(coordinate 94)\n	(DB_ID 976863)\n	(psiMod [GK_611088])\n	(referenceSequence [GK_53331]))\n\n([GK_976864] of  Person\n\n	(_displayName \"Liepnieks, J\")\n	(DB_ID 976864)\n	(initial \"J\")\n	(surname \"Liepnieks\"))\n\n([GK_976865] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Variant apolipoprotein AI [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976865)\n	(endCoordinate 107)\n	(hasModifiedResidue [GK_976875])\n	(literatureReference [GK_976941])\n	(name\n		\"Variant apolipoprotein AI\"\n		\"APOA1\"\n		\"APOA1_HUMAN\"\n		\"APOA1\")\n	(referenceEntity [GK_50094])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358084])\n	(startCoordinate 25))\n\n([GK_976866] of  Person\n\n	(_displayName \"Kodali, R\")\n	(DB_ID 976866)\n	(initial \"R\")\n	(surname \"Kodali\"))\n\n([GK_976867] of  Summation\n\n	(_displayName \"This protein is formed when the normal stop codon is skipped...\")\n	(DB_ID 976867)\n	(literatureReference [GK_976942])\n	(text \"This protein is formed when the normal stop codon is skipped, adding an extra 21 residues to the normal C terminus of ApoAII. An almost identical variant was reported by Benson et al. (2001).\"))\n\n([GK_976868] of  Person\n\n	(_displayName \"Barats, MS\")\n	(DB_ID 976868)\n	(initial \"MS\")\n	(surname \"Barats\"))\n\n([GK_976869] of  Person\n\n	(_displayName \"Hawkins, PN\")\n	(DB_ID 976869)\n	(initial \"PN\")\n	(surname \"Hawkins\"))\n\n([GK_976870] of  GenomeEncodedEntity\n\n	(_displayName \"[Hsa] Variant apolipoprotein AII [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976870)\n	(literatureReference [GK_976913])\n	(name \"Variant apolipoprotein AII\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1357707])\n	(summation [GK_976867]))\n\n([GK_976871] of  Polymer\n\n	(_displayName \"[Hsa] ABri/ADan amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976871)\n	(literatureReference\n		[GK_976939]\n		[GK_976873])\n	(name \"ABri/ADan amyloid fibril\")\n	(repeatedUnit [GK_976878])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357802]))\n\n([GK_976872] of  Person\n\n	(_displayName \"Saraiva, MJ\")\n	(DB_ID 976872)\n	(initial \"MJ\")\n	(surname \"Saraiva\"))\n\n([GK_976873] of  LiteratureReference\n\n	(_displayName \"Oligomerization and neurotoxicity of the amyloid ADan peptide implicated in familial Danish dementia\")\n	(author\n		[GK_893611]\n		[GK_976903]\n		[GK_217036]\n		[GK_976948]\n		[GK_976894]\n		[GK_936419]\n		[GK_976890])\n	(DB_ID 976873)\n	(journal \"J Neurochem\")\n	(pages \"281-90\")\n	(pubMedIdentifier 14690516)\n	(title \"Oligomerization and neurotoxicity of the amyloid ADan peptide implicated in familial Danish dementia\")\n	(volume 88)\n	(year 2004))\n\n([GK_976874] of  Person\n\n	(_displayName \"Liepnieks, JJ\")\n	(DB_ID 976874)\n	(initial \"JJ\")\n	(surname \"Liepnieks\"))\n\n([GK_976875] of  ReplacedResidue\n\n	(_displayName \"L-arginine residue 50 replaced with unknown\")\n	(coordinate 50)\n	(DB_ID 976875)\n	(psiMod [GK_611066])\n	(referenceSequence [GK_50094]))\n\n([GK_976876] of  Person\n\n	(_displayName \"Ghiso, J\")\n	(DB_ID 976876)\n	(initial \"J\")\n	(surname \"Ghiso\"))\n\n([GK_976877] of  LiteratureReference\n\n	(_displayName \"Human lysozyme gene mutations cause hereditary systemic amyloidosis\")\n	(author\n		[GK_976739]\n		[GK_976869]\n		[GK_976907]\n		[GK_976936]\n		[GK_976947]\n		[GK_171090]\n		[GK_976856]\n		[GK_976899]\n		[GK_976934]\n		[GK_976851])\n	(DB_ID 976877)\n	(journal \"Nature\")\n	(pages \"553-7\")\n	(pubMedIdentifier 8464497)\n	(title \"Human lysozyme gene mutations cause hereditary systemic amyloidosis\")\n	(volume 362)\n	(year 1993))\n\n([GK_976878] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] ABri/ADan amyloid peptide [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976878)\n	(endCoordinate 266)\n	(name\n		\"ABri/ADan amyloid peptide\"\n		\"Integral membrane protein 2B\"\n		\"ITM2B_HUMAN\"\n		\"ITM2B\")\n	(referenceEntity [GK_57627])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358324])\n	(startCoordinate 244)\n	(summation [GK_976914]))\n\n([GK_976879] of  LiteratureReference\n\n	(_displayName \"Mutation in cystatin C gene causes hereditary brain haemorrhage\")\n	(author\n		[GK_976933]\n		[GK_976855]\n		[GK_976909]\n		[GK_976943]\n		[GK_976888]\n		[GK_976938]\n		[GK_976887])\n	(DB_ID 976879)\n	(journal \"Lancet\")\n	(pages \"603-4\")\n	(pubMedIdentifier 2900981)\n	(title \"Mutation in cystatin C gene causes hereditary brain haemorrhage\")\n	(volume 2)\n	(year 1988))\n\n([GK_976880] of  Person\n\n	(_displayName \"Goto, Y\")\n	(DB_ID 976880)\n	(initial \"Y\")\n	(surname \"Goto\"))\n\n([GK_976881] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Amyloid protein A [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976881)\n	(endCoordinate 94)\n	(name\n		\"Amyloid protein A\"\n		\"SAA1\"\n		\"SAA_HUMAN\"\n		\"SAA1\")\n	(referenceEntity [GK_64108])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358246])\n	(startCoordinate 19))\n\n([GK_976882] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Apolipoprotein A-IV amyloid fragment [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976882)\n	(endCoordinate 94)\n	(name\n		\"Apolipoprotein A-IV amyloid fragment\"\n		\"APOA4\"\n		\"APOA4_HUMAN\"\n		\"APOA4\")\n	(referenceEntity [GK_50098])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357778])\n	(startCoordinate 21))\n\n([GK_976883] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Transthyretin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976883)\n	(endCoordinate 147)\n	(name\n		\"Transthyretin\"\n		\"TTR\"\n		\"TTHY_HUMAN\"\n		\"TTR\")\n	(referenceEntity [GK_66545])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358281])\n	(startCoordinate 21))\n\n([GK_976884] of  Person\n\n	(_displayName \"Westermark, P\")\n	(DB_ID 976884)\n	(initial \"P\")\n	(surname \"Westermark\"))\n\n([GK_976885] of  LiteratureReference\n\n	(_displayName \"Two different types of amyloid deposits--apolipoprotein A-IV and transthyretin--in a patient with systemic amyloidosis\")\n	(author\n		[GK_976946]\n		[GK_976849]\n		[GK_976922]\n		[GK_976858]\n		[GK_976906]\n		[GK_109477]\n		[GK_976884])\n	(DB_ID 976885)\n	(journal \"Lab Invest\")\n	(pages \"981-8\")\n	(pubMedIdentifier 15146166)\n	(title \"Two different types of amyloid deposits--apolipoprotein A-IV and transthyretin--in a patient with systemic amyloidosis\")\n	(volume 84)\n	(year 2004))\n\n([GK_976886] of  Person\n\n	(_displayName \"Yazaki, M\")\n	(DB_ID 976886)\n	(initial \"M\")\n	(surname \"Yazaki\"))\n\n([GK_976887] of  Person\n\n	(_displayName \"Jensson, O\")\n	(DB_ID 976887)\n	(initial \"O\")\n	(surname \"Jensson\"))\n\n([GK_976888] of  Person\n\n	(_displayName \"Olafsson, I\")\n	(DB_ID 976888)\n	(initial \"I\")\n	(surname \"Olafsson\"))\n\n([GK_976889] of  Polymer\n\n	(_displayName \"[Hsa] Apolipoprotein A-IV fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976889)\n	(literatureReference [GK_976885])\n	(name \"Apolipoprotein A-IV fibril\")\n	(repeatedUnit [GK_976882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358185]))\n\n([GK_976890] of  Person\n\n	(_displayName \"Austen, B\")\n	(DB_ID 976890)\n	(initial \"B\")\n	(surname \"Austen\"))\n\n([GK_976891] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Variant cystatin-C [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976891)\n	(endCoordinate 146)\n	(hasModifiedResidue [GK_976863])\n	(literatureReference [GK_976879])\n	(name\n		\"Variant cystatin-C\"\n		\"CST3\"\n		\"CYTC_HUMAN\"\n		\"CST3\")\n	(referenceEntity [GK_53331])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357725])\n	(startCoordinate 27))\n\n([GK_976892] of  Person\n\n	(_displayName \"Naiki, H\")\n	(DB_ID 976892)\n	(initial \"H\")\n	(surname \"Naiki\"))\n\n([GK_976893] of  Polymer\n\n	(_displayName \"[Hsa] Variant apolipoprotein AI fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976893)\n	(literatureReference [GK_976941])\n	(name \"Variant apolipoprotein AI fibril\")\n	(repeatedUnit [GK_976865])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357799]))\n\n([GK_976894] of  Person\n\n	(_displayName \"El-Agnaf, OM\")\n	(DB_ID 976894)\n	(initial \"OM\")\n	(surname \"El-Agnaf\"))\n\n([GK_976895] of  Person\n\n	(_displayName \"Mead, S\")\n	(DB_ID 976895)\n	(initial \"S\")\n	(surname \"Mead\"))\n\n([GK_976896] of  LiteratureReference\n\n	(_displayName \"Conformation of beta 2-microglobulin amyloid fibrils analyzed by reduction of the disulfide bond\")\n	(author\n		[GK_976918]\n		[GK_976857]\n		[GK_189563]\n		[GK_976892]\n		[GK_976880])\n	(DB_ID 976896)\n	(journal \"J Biol Chem\")\n	(pages \"21554-60\")\n	(pubMedIdentifier 11943769)\n	(title \"Conformation of beta 2-microglobulin amyloid fibrils analyzed by reduction of the disulfide bond\")\n	(volume 277)\n	(year 2002))\n\n([GK_976897] of  Person\n\n	(_displayName \"Rostagno, A\")\n	(DB_ID 976897)\n	(initial \"A\")\n	(surname \"Rostagno\"))\n\n([GK_976898] of  Polymer\n\n	(_displayName \"[Hsa] Amyloid protein A fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976898)\n	(literatureReference\n		[GK_976908]\n		[GK_976850])\n	(name \"Amyloid protein A fibril\")\n	(repeatedUnit [GK_976881])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358419]))\n\n([GK_976899] of  Person\n\n	(_displayName \"Nguyen, O\")\n	(DB_ID 976899)\n	(initial \"O\")\n	(surname \"Nguyen\"))\n\n([GK_976900] of  Person\n\n	(_displayName \"Chatani, E\")\n	(DB_ID 976900)\n	(initial \"E\")\n	(surname \"Chatani\"))\n\n([GK_976901] of  Person\n\n	(_displayName \"Vidal, R\")\n	(DB_ID 976901)\n	(initial \"R\")\n	(surname \"Vidal\"))\n\n([GK_976902] of  ReplacedResidue\n\n	(_displayName \"L-leucine residue 573 replaced with unknown\")\n	(coordinate 573)\n	(DB_ID 976902)\n	(psiMod [GK_611093])\n	(referenceSequence [GK_54840]))\n\n([GK_976903] of  Person\n\n	(_displayName \"Gunasekera, N\")\n	(DB_ID 976903)\n	(initial \"N\")\n	(surname \"Gunasekera\"))\n\n([GK_976904] of  Person\n\n	(_displayName \"Benson, MD\")\n	(DB_ID 976904)\n	(initial \"MD\")\n	(surname \"Benson\"))\n\n([GK_976905] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Variant fibrinogen alpha chain [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976905)\n	(endCoordinate 866)\n	(hasModifiedResidue [GK_976902])\n	(literatureReference [GK_976921])\n	(name \"Variant fibrinogen alpha chain\")\n	(referenceEntity [GK_54840])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358175])\n	(startCoordinate 20))\n\n([GK_976906] of  Person\n\n	(_displayName \"Sletten, K\")\n	(DB_ID 976906)\n	(initial \"K\")\n	(surname \"Sletten\"))\n\n([GK_976907] of  Person\n\n	(_displayName \"Booth, DR\")\n	(DB_ID 976907)\n	(initial \"DR\")\n	(surname \"Booth\"))\n\n([GK_976908] of  LiteratureReference\n\n	(_displayName \"Serum amyloid A protein in amyloidosis, rheumatic, and enoplastic diseases\")\n	(author\n		[GK_976904]\n		[GK_976747])\n	(DB_ID 976908)\n	(journal \"Arthritis Rheum\")\n	(pages \"36-42\")\n	(pubMedIdentifier 103558)\n	(title \"Serum amyloid A protein in amyloidosis, rheumatic, and enoplastic diseases\")\n	(volume 22)\n	(year 1979))\n\n([GK_976909] of  Person\n\n	(_displayName \"Thorsteinsson, L\")\n	(DB_ID 976909)\n	(initial \"L\")\n	(surname \"Thorsteinsson\"))\n\n([GK_976910] of  Person\n\n	(_displayName \"R√É¬©v√É¬©sz, T\")\n	(DB_ID 976910)\n	(initial \"T\")\n	(surname \"R√É¬©v√É¬©sz\"))\n\n([GK_976911] of  Polymer\n\n	(_displayName \"[Hsa] Variant fibrinogen alpha chain fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976911)\n	(literatureReference [GK_976921])\n	(name \"Variant fibrinogen alpha chain fibril\")\n	(repeatedUnit [GK_976905])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357842]))\n\n([GK_976912] of  Person\n\n	(_displayName \"Wheeler, G\")\n	(DB_ID 976912)\n	(initial \"G\")\n	(surname \"Wheeler\"))\n\n([GK_976913] of  LiteratureReference\n\n	(_displayName \"Hereditary systemic amyloidosis associated with a new apolipoprotein AII stop codon mutation Stop78Arg\")\n	(author\n		[GK_976886]\n		[GK_976874]\n		[GK_976868]\n		[GK_976923]\n		[GK_976904])\n	(DB_ID 976913)\n	(journal \"Kidney Int\")\n	(pages \"11-6\")\n	(pubMedIdentifier 12787390)\n	(title \"Hereditary systemic amyloidosis associated with a new apolipoprotein AII stop codon mutation Stop78Arg\")\n	(volume 64)\n	(year 2003))\n\n([GK_976914] of  Summation\n\n	(_displayName \"The ABri and ADan peptides are extensions of the normal C te...\")\n	(DB_ID 976914)\n	(text \"The ABri and ADan peptides are extensions of the normal C terminus of ITM2B (BRI). The sequence of ABri results from mutatin of the normal stop codon to R, resulting in an additional 11 residues. The ABri peptide is the last 34 residues of this mutated peptide -  EASNCFAIRHFENKFAVETLICSRTVKKNIIEEN.\\nThe ADan peptide is the result of a 10 nucleotide insertion between the codons for residue 265 and 266, introducing a frameshift that adds a different extension to the normal peptide. The ADan peptide is the last 34 residues of this mutated peptide, identical to ABri for the first 22 residues.\\n\\nEASNCFAIRHFENKFAVETLICFNLFLNSQEKHY \"))\n\n([GK_976915] of  Person\n\n	(_displayName \"Plant, G\")\n	(DB_ID 976915)\n	(initial \"G\")\n	(surname \"Plant\"))\n\n([GK_976916] of  Polymer\n\n	(_displayName \"[Hsa] Variant apolipoprotein AII fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976916)\n	(literatureReference [GK_976913])\n	(name \"Variant apolipoprotein AII fibril\")\n	(repeatedUnit [GK_976870])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358575]))\n\n([GK_976917] of  Person\n\n	(_displayName \"Wetzel, R\")\n	(DB_ID 976917)\n	(initial \"R\")\n	(surname \"Wetzel\"))\n\n([GK_976918] of  Person\n\n	(_displayName \"Hong, DP\")\n	(DB_ID 976918)\n	(initial \"DP\")\n	(surname \"Hong\"))\n\n([GK_976919] of  Polymer\n\n	(_displayName \"[Hsa] Gelsolin amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976919)\n	(literatureReference [GK_976852])\n	(name \"Gelsolin amyloid fibril\")\n	(repeatedUnit [GK_976940])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358387]))\n\n([GK_976920] of  Summation\n\n	(_displayName \"Amyloid is a term used to describe typically extracellular d...\")\n	(DB_ID 976920)\n	(literatureReference\n		[GK_977172]\n		[GK_1247722]\n		[GK_1247762]\n		[GK_1247745]\n		[GK_1247696]\n		[GK_1247719]\n		[GK_1247704]\n		[GK_1247703]\n		[GK_1247812]\n		[GK_1247715]\n		[GK_1247805])\n	(text \"Amyloid is a term used to describe typically extracellular deposits of aggregated proteins, sometimes known as plaques. Abnormal accumulation of amyloid is amyloidosis, a term associated with diseased organs and tissues, particularly neurodegenerative diseases such as Alzheimer\'s, Parkinson\'s and Huntingdon\'s. Amyloid deposits consist predominantly of amyloid fibrils, rigid, non-branching structures that form ordered assemblies, characteristically with a cross beta-sheet structure where the sheets run parallel to the direction of the fibril (Sawaya et al. 2007). Often the fibril has a left-handed twist (Nelson & Eisenberg 2006). At least 27 human proteins form amyloid fibrils (Sipe et al. 2010). Many of these proteins have non-pathological functions; the trigger that leads to abnormal aggregations differs between proteins and is not well understood but in many cases the peptides are abnormal fragments or mutant forms arising from polymorphisms, suggesting that the initial event may be aggregation of misfolded or unfolded peptides. Early studies of Amyloid-Beta assembly led to a widely accepted model that assembly was a nucleation-dependent polymerization reaction (Teplow 1998) but it is now understood to be more complex, with multiple \'off-pathway\' events leading to a variety of oligomeric structures in addition to fibrils (Roychaudhuri et al. 2008). An increasing body of evidence suggests that these oligomeric forms are primarily responsible for the neurotoxic effects of Amyloid-beta (Roychaudhuri et al. 2008), alpha-synuclein (Winner et al. 2011) and tau (Dance & Strobel 2009, Meraz-Rios et al. 2010). Amyloid oligomers are believed to have a common structural motif that is independent of the protein involved and not present in fibrils (Kayed et al. 2003). Conformation dependent, aggregation specific antibodies suggest that there are 3 general classes of amyloid oligomer structures (Glabe 2009) including annular structures which may be responsible for the widely reported membrane permeabilization effect of amyloid oligomers. Toxicity of amyloid oligomers preceeds the appearance of plaques in mouse models (Ferretti et al. 2011). \\nFibrils are often associated with other molecules, notably heparan sulfate proteoglycans and Serum Amyloid P-component, which are universally associated and seem to stabilize fibrils, possibly by protecting them from degradation.\"))\n\n([GK_976921] of  LiteratureReference\n\n	(_displayName \"Hereditary renal amyloidosis associated with a mutant fibrinogen alpha-chain\")\n	(author\n		[GK_976904]\n		[GK_976864]\n		[GK_976927]\n		[GK_976912]\n		[GK_976930])\n	(DB_ID 976921)\n	(journal \"Nat Genet\")\n	(pages \"252-5\")\n	(pubMedIdentifier 8097946)\n	(title \"Hereditary renal amyloidosis associated with a mutant fibrinogen alpha-chain\")\n	(volume 3)\n	(year 1993))\n\n([GK_976922] of  Person\n\n	(_displayName \"Weiss, DT\")\n	(DB_ID 976922)\n	(initial \"DT\")\n	(surname \"Weiss\"))\n\n([GK_976923] of  Person\n\n	(_displayName \"Cohen, AH\")\n	(DB_ID 976923)\n	(initial \"AH\")\n	(surname \"Cohen\"))\n\n([GK_976924] of  LiteratureReference\n\n	(_displayName \"Hereditary transthyretin amyloidosis: molecular basis and therapeutical strategies\")\n	(author [GK_976872])\n	(DB_ID 976924)\n	(journal \"Expert Rev Mol Med\")\n	(pages \"1-11\")\n	(pubMedIdentifier 14987380)\n	(title \"Hereditary transthyretin amyloidosis: molecular basis and therapeutical strategies\")\n	(volume 4)\n	(year 2002))\n\n([GK_976926] of  ReplacedResidue\n\n	(_displayName \"L-threonine residue 74 replaced with unknown\")\n	(coordinate 74)\n	(DB_ID 976926)\n	(psiMod [GK_611082])\n	(referenceSequence [GK_58841]))\n\n([GK_976927] of  Person\n\n	(_displayName \"Uemichi, T\")\n	(DB_ID 976927)\n	(initial \"T\")\n	(surname \"Uemichi\"))\n\n([GK_976928] of  Person\n\n	(_displayName \"Frangione, B\")\n	(DB_ID 976928)\n	(initial \"B\")\n	(surname \"Frangione\"))\n\n([GK_976929] of  Polymer\n\n	(_displayName \"[Hsa] Transthyretin fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976929)\n	(literatureReference [GK_976924])\n	(name \"Transthyretin fibril\")\n	(repeatedUnit [GK_976883])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358552]))\n\n([GK_976930] of  Person\n\n	(_displayName \"Correa, R\")\n	(DB_ID 976930)\n	(initial \"R\")\n	(surname \"Correa\"))\n\n([GK_976931] of  Person\n\n	(_displayName \"Hamidi Asl, K\")\n	(DB_ID 976931)\n	(initial \"K\")\n	(surname \"Hamidi Asl\"))\n\n([GK_976932] of  Person\n\n	(_displayName \"Kluve-Beckerman, B\")\n	(DB_ID 976932)\n	(initial \"B\")\n	(surname \"Kluve-Beckerman\"))\n\n([GK_976933] of  Person\n\n	(_displayName \"Palsdottir, A\")\n	(DB_ID 976933)\n	(initial \"A\")\n	(surname \"Palsdottir\"))\n\n([GK_976934] of  Person\n\n	(_displayName \"Blake, CC\")\n	(DB_ID 976934)\n	(initial \"CC\")\n	(surname \"Blake\"))\n\n([GK_976935] of  Polymer\n\n	(_displayName \"[Hsa] Variant cystatin-C fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976935)\n	(name \"Variant cystatin-C fibril\")\n	(repeatedUnit [GK_976891])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357844]))\n\n([GK_976936] of  Person\n\n	(_displayName \"Vigushin, DM\")\n	(DB_ID 976936)\n	(initial \"DM\")\n	(surname \"Vigushin\"))\n\n([GK_976937] of  LiteratureReference\n\n	(_displayName \"Structural stability of amyloid fibrils of beta(2)-microglobulin in comparison with its native fold\")\n	(author\n		[GK_976900]\n		[GK_976880])\n	(DB_ID 976937)\n	(journal \"Biochim Biophys Acta\")\n	(pages \"64-75\")\n	(pubMedIdentifier 16213801)\n	(title \"Structural stability of amyloid fibrils of beta(2)-microglobulin in comparison with its native fold\")\n	(volume 1753)\n	(year 2005))\n\n([GK_976938] of  Person\n\n	(_displayName \"Grubb, A\")\n	(DB_ID 976938)\n	(initial \"A\")\n	(surname \"Grubb\"))\n\n([GK_976939] of  LiteratureReference\n\n	(_displayName \"A stop-codon mutation in the BRI gene associated with familial British dementia\")\n	(author\n		[GK_976901]\n		[GK_976928]\n		[GK_976897]\n		[GK_976895]\n		[GK_976910]\n		[GK_976915]\n		[GK_976876])\n	(DB_ID 976939)\n	(journal \"Nature\")\n	(pages \"776-81\")\n	(pubMedIdentifier 10391242)\n	(title \"A stop-codon mutation in the BRI gene associated with familial British dementia\")\n	(volume 399)\n	(year 1999))\n\n([GK_976940] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Gelsolin amyloid fragment [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976940)\n	(endCoordinate 270)\n	(name \"Gelsolin amyloid fragment\")\n	(referenceEntity [GK_55564])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358177])\n	(startCoordinate 200))\n\n([GK_976941] of  LiteratureReference\n\n	(_displayName \"Variant apolipoprotein AI as a major constituent of a human hereditary amyloid\")\n	(author\n		[GK_947995]\n		[GK_976853]\n		[GK_976864]\n		[GK_976904])\n	(DB_ID 976941)\n	(journal \"Biochem Biophys Res Commun\")\n	(pages \"762-8\")\n	(pubMedIdentifier 3142462)\n	(title \"Variant apolipoprotein AI as a major constituent of a human hereditary amyloid\")\n	(volume 156)\n	(year 1988))\n\n([GK_976942] of  LiteratureReference\n\n	(_displayName \"A new human hereditary amyloidosis: the result of a stop-codon mutation in the apolipoprotein AII gene\")\n	(author\n		[GK_976904]\n		[GK_976874]\n		[GK_976886]\n		[GK_194488]\n		[GK_976931]\n		[GK_976861]\n		[GK_976932])\n	(DB_ID 976942)\n	(journal \"Genomics\")\n	(pages \"272-7\")\n	(pubMedIdentifier 11401442)\n	(title \"A new human hereditary amyloidosis: the result of a stop-codon mutation in the apolipoprotein AII gene\")\n	(volume 72)\n	(year 2001))\n\n([GK_976943] of  Person\n\n	(_displayName \"Arnason, A\")\n	(DB_ID 976943)\n	(initial \"A\")\n	(surname \"Arnason\"))\n\n([GK_976944] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Variant lysozyme C [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976944)\n	(endCoordinate 148)\n	(hasModifiedResidue [GK_976926])\n	(literatureReference [GK_976877])\n	(name\n		\"Variant lysozyme C\"\n		\"LYZ\"\n		\"LYSC_HUMAN\"\n		\"LYZ\")\n	(referenceEntity [GK_58841])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357692])\n	(startCoordinate 19))\n\n([GK_976945] of  Polymer\n\n	(_displayName \"[Hsa] Beta2-microglobulin fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976945)\n	(literatureReference\n		[GK_976896]\n		[GK_976937])\n	(name \"Beta2-microglobulin fibril\")\n	(repeatedUnit [GK_167743])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358224]))\n\n([GK_976946] of  Person\n\n	(_displayName \"Bergstr√É¬∂m, J\")\n	(DB_ID 976946)\n	(initial \"J\")\n	(surname \"Bergstr√É¬∂m\"))\n\n([GK_976947] of  Person\n\n	(_displayName \"Tennent, GA\")\n	(DB_ID 976947)\n	(initial \"GA\")\n	(surname \"Tennent\"))\n\n([GK_976948] of  Person\n\n	(_displayName \"Lelyveld, V\")\n	(DB_ID 976948)\n	(initial \"V\")\n	(surname \"Lelyveld\"))\n\n([GK_976949] of  LiteratureReference\n\n	(_displayName \"Codeposition of apolipoprotein A-IV and transthyretin in senile systemic (ATTR) amyloidosis\")\n	(author\n		[GK_976946]\n		[GK_976953]\n		[GK_207287]\n		[GK_976922]\n		[GK_374884]\n		[GK_976858]\n		[GK_976884])\n	(DB_ID 976949)\n	(journal \"Biochem Biophys Res Commun\")\n	(pages \"903-8\")\n	(pubMedIdentifier 11467836)\n	(title \"Codeposition of apolipoprotein A-IV and transthyretin in senile systemic (ATTR) amyloidosis\")\n	(volume 285)\n	(year 2001))\n\n([GK_976950] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Atrial natriuretic factor [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976950)\n	(endCoordinate -1)\n	(name\n		\"Atrial natriuretic factor\"\n		\"NPPA\"\n		\"ANF_HUMAN\"\n		\"NPPA\")\n	(referenceEntity [GK_49984])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358061])\n	(startCoordinate 1))\n\n([GK_976952] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Medin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976952)\n	(endCoordinate 317)\n	(name\n		\"Medin\"\n		\"MFGE8\"\n		\"MFGM_HUMAN\"\n		\"MFGE8\")\n	(referenceEntity [GK_59185])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357984])\n	(startCoordinate 268))\n\n([GK_976953] of  Person\n\n	(_displayName \"Murphy, C\")\n	(DB_ID 976953)\n	(initial \"C\")\n	(surname \"Murphy\"))\n\n([GK_976954] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Alpha-synuclein [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976954)\n	(endCoordinate -1)\n	(name\n		\"Alpha-synuclein\"\n		\"SNCA\"\n		\"SYUA_HUMAN\"\n		\"SNCA\")\n	(referenceEntity [GK_65462])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358359])\n	(startCoordinate 1))\n\n([GK_976957] of  Person\n\n	(_displayName \"Agarwal, A\")\n	(DB_ID 976957)\n	(initial \"A\")\n	(surname \"Agarwal\"))\n\n([GK_976958] of  Person\n\n	(_displayName \"Hayden, DW\")\n	(DB_ID 976958)\n	(initial \"DW\")\n	(surname \"Hayden\"))\n\n([GK_976959] of  LiteratureReference\n\n	(_displayName \"Prolactin-derived amyloid in the aging pituitary gland\")\n	(author\n		[GK_976884]\n		[GK_976962]\n		[GK_976986]\n		[GK_976965]\n		[GK_976906])\n	(DB_ID 976959)\n	(journal \"Am J Pathol\")\n	(pages \"67-73\")\n	(pubMedIdentifier 9006323)\n	(title \"Prolactin-derived amyloid in the aging pituitary gland\")\n	(volume 150)\n	(year 1997))\n\n([GK_976960] of  Person\n\n	(_displayName \"Slavin, G\")\n	(DB_ID 976960)\n	(initial \"G\")\n	(surname \"Slavin\"))\n\n([GK_976961] of  Person\n\n	(_displayName \"O\'Brien, TD\")\n	(DB_ID 976961)\n	(initial \"TD\")\n	(surname \"O\'Brien\"))\n\n([GK_976962] of  Person\n\n	(_displayName \"Eriksson, L\")\n	(DB_ID 976962)\n	(initial \"L\")\n	(surname \"Eriksson\"))\n\n([GK_976963] of  DefinedSet\n\n	(_displayName \"[Hsa] Localized amyloid fibril main peptide chains [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976963)\n	(hasMember\n		[GK_976748]\n		[GK_976978]\n		[GK_976981]\n		[GK_976987]\n		[GK_976974]\n		[GK_977219]\n		[GK_977096]\n		[GK_977188]\n		[GK_977101]\n		[GK_977079]\n		[GK_977087])\n	(literatureReference [GK_976859])\n	(name \"Localized amyloid fibril main peptide chains\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358199])\n	(summation [GK_976920]))\n\n([GK_976965] of  Person\n\n	(_displayName \"Enestr√É¬∂m, S\")\n	(DB_ID 976965)\n	(initial \"S\")\n	(surname \"Enestr√É¬∂m\"))\n\n([GK_976967] of  LiteratureReference\n\n	(_displayName \"Isolated atrial amyloid contains atrial natriuretic peptide: a report of six cases\")\n	(author\n		[GK_976980]\n		[GK_976994]\n		[GK_976973]\n		[GK_976975]\n		[GK_976985]\n		[GK_976960])\n	(DB_ID 976967)\n	(journal \"Br Heart J\")\n	(pages \"317-20\")\n	(pubMedIdentifier 2945573)\n	(title \"Isolated atrial amyloid contains atrial natriuretic peptide: a report of six cases\")\n	(volume 56)\n	(year 1986))\n\n([GK_976968] of  LiteratureReference\n\n	(_displayName \"Unraveling the amyloid associated with human medullary thyroid carcinoma\")\n	(author\n		[GK_976988]\n		[GK_976957]\n		[GK_976972]\n		[GK_976969]\n		[GK_976971]\n		[GK_976989]\n		[GK_976992])\n	(DB_ID 976968)\n	(journal \"Endocrinology\")\n	(pages \"5465-70\")\n	(pubMedIdentifier 15459123)\n	(title \"Unraveling the amyloid associated with human medullary thyroid carcinoma\")\n	(volume 145)\n	(year 2004))\n\n([GK_976969] of  Person\n\n	(_displayName \"Verma, N\")\n	(DB_ID 976969)\n	(initial \"N\")\n	(surname \"Verma\"))\n\n([GK_976970] of  LiteratureReference\n\n	(_displayName \"Amyloid fibrils in human insulinoma and islets of Langerhans of the diabetic cat are derived from a neuropeptide-like protein also present in normal islet cells\")\n	(author\n		[GK_976884]\n		[GK_163570]\n		[GK_976979]\n		[GK_976958]\n		[GK_976961]\n		[GK_976995])\n	(DB_ID 976970)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"3881-5\")\n	(pubMedIdentifier 3035556)\n	(title \"Amyloid fibrils in human insulinoma and islets of Langerhans of the diabetic cat are derived from a neuropeptide-like protein also present in normal islet cells\")\n	(volume 84)\n	(year 1987))\n\n([GK_976971] of  Person\n\n	(_displayName \"Sharma, AK\")\n	(DB_ID 976971)\n	(initial \"AK\")\n	(surname \"Sharma\"))\n\n([GK_976972] of  Person\n\n	(_displayName \"Bajpai, VK\")\n	(DB_ID 976972)\n	(initial \"VK\")\n	(surname \"Bajpai\"))\n\n([GK_976973] of  Person\n\n	(_displayName \"d\'Ardenne, AJ\")\n	(DB_ID 976973)\n	(initial \"AJ\")\n	(surname \"d\'Ardenne\"))\n\n([GK_976974] of  Polymer\n\n	(_displayName \"[Hsa] Prolactin amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976974)\n	(literatureReference [GK_976959])\n	(name \"Prolactin amyloid fibril\")\n	(repeatedUnit [GK_976984])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357875]))\n\n([GK_976975] of  Person\n\n	(_displayName \"Edmondson, SJ\")\n	(DB_ID 976975)\n	(initial \"SJ\")\n	(surname \"Edmondson\"))\n\n([GK_976978] of  Polymer\n\n	(_displayName \"[Hsa] Calcitonin amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976978)\n	(literatureReference [GK_976968])\n	(name \"Calcitonin amyloid fibril\")\n	(repeatedUnit [GK_419781])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358284]))\n\n([GK_976979] of  Person\n\n	(_displayName \"Wilander, E\")\n	(DB_ID 976979)\n	(initial \"E\")\n	(surname \"Wilander\"))\n\n([GK_976980] of  Person\n\n	(_displayName \"Kaye, GC\")\n	(DB_ID 976980)\n	(initial \"GC\")\n	(surname \"Kaye\"))\n\n([GK_976981] of  Polymer\n\n	(_displayName \"[Hsa] Islet amyloid polypeptide fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976981)\n	(literatureReference [GK_976970])\n	(name \"Islet amyloid polypeptide fibril\")\n	(repeatedUnit [GK_420076])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357672]))\n\n([GK_976984] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Prolactin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976984)\n	(endCoordinate 227)\n	(name\n		\"Prolactin\"\n		\"PRL\"\n		\"PRL_HUMAN\"\n		\"PRL\")\n	(referenceEntity [GK_62178])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358225])\n	(startCoordinate 29))\n\n([GK_976985] of  Person\n\n	(_displayName \"Camm, AJ\")\n	(DB_ID 976985)\n	(initial \"AJ\")\n	(surname \"Camm\"))\n\n([GK_976986] of  Person\n\n	(_displayName \"Engstr√É¬∂m, U\")\n	(DB_ID 976986)\n	(initial \"U\")\n	(surname \"Engstr√É¬∂m\"))\n\n([GK_976987] of  Polymer\n\n	(_displayName \"[Hsa] Atrial natriuretic factor amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 976987)\n	(literatureReference\n		[GK_976967]\n		[GK_976993])\n	(name \"Atrial natriuretic factor amyloid fibril\")\n	(repeatedUnit [GK_976950])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357872]))\n\n([GK_976988] of  Person\n\n	(_displayName \"Khurana, R\")\n	(DB_ID 976988)\n	(initial \"R\")\n	(surname \"Khurana\"))\n\n([GK_976989] of  Person\n\n	(_displayName \"Gupta, RP\")\n	(DB_ID 976989)\n	(initial \"RP\")\n	(surname \"Gupta\"))\n\n([GK_976990] of  Person\n\n	(_displayName \"Johansson, B\")\n	(DB_ID 976990)\n	(initial \"B\")\n	(surname \"Johansson\"))\n\n([GK_976992] of  Person\n\n	(_displayName \"Madhusudan, KP\")\n	(DB_ID 976992)\n	(initial \"KP\")\n	(surname \"Madhusudan\"))\n\n([GK_976993] of  LiteratureReference\n\n	(_displayName \"The relation of atrial natriuretic factor to isolated atrial amyloid\")\n	(author\n		[GK_976990]\n		[GK_976884])\n	(DB_ID 976993)\n	(journal \"Exp Mol Pathol\")\n	(pages \"266-78\")\n	(pubMedIdentifier 2142465)\n	(title \"The relation of atrial natriuretic factor to isolated atrial amyloid\")\n	(volume 52)\n	(year 1990))\n\n([GK_976994] of  Person\n\n	(_displayName \"Butler, MG\")\n	(DB_ID 976994)\n	(initial \"MG\")\n	(surname \"Butler\"))\n\n([GK_976995] of  Person\n\n	(_displayName \"Johnson, KH\")\n	(DB_ID 976995)\n	(initial \"KH\")\n	(surname \"Johnson\"))\n\n([GK_977077] of  LiteratureReference\n\n	(_displayName \"Glycosaminoglycan and proteoglycan inhibit the depolymerization of beta2-microglobulin amyloid fibrils in vitro\")\n	(author\n		[GK_186930]\n		[GK_977108]\n		[GK_977193]\n		[GK_977107]\n		[GK_217173]\n		[GK_114197]\n		[GK_189563]\n		[GK_445564]\n		[GK_204598]\n		[GK_977120]\n		[GK_976892])\n	(DB_ID 977077)\n	(journal \"Kidney Int\")\n	(pages \"1080-8\")\n	(pubMedIdentifier 12911560)\n	(title \"Glycosaminoglycan and proteoglycan inhibit the depolymerization of beta2-microglobulin amyloid fibrils in vitro\")\n	(volume 64)\n	(year 2003))\n\n([GK_977078] of  Person\n\n	(_displayName \"Kisilevsky, R\")\n	(DB_ID 977078)\n	(initial \"R\")\n	(surname \"Kisilevsky\"))\n\n([GK_977079] of  Polymer\n\n	(_displayName \"[Hsa] Semenogelin-1 amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977079)\n	(literatureReference [GK_977189])\n	(name \"Semenogelin-1 amyloid fibril\")\n	(repeatedUnit [GK_977100])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357980]))\n\n([GK_977080] of  Person\n\n	(_displayName \"Klintworth, GK\")\n	(DB_ID 977080)\n	(initial \"GK\")\n	(surname \"Klintworth\"))\n\n([GK_977081] of  LiteratureReference\n\n	(_displayName \"Apolipoprotein E, amyloid, and Alzheimer disease\")\n	(author\n		[GK_977160]\n		[GK_977124]\n		[GK_977132]\n		[GK_977178]\n		[GK_428598])\n	(DB_ID 977081)\n	(journal \"Mol Interv\")\n	(pages \"363-75, 339\")\n	(pubMedIdentifier 14993413)\n	(title \"Apolipoprotein E, amyloid, and Alzheimer disease\")\n	(volume 2)\n	(year 2002))\n\n([GK_977082] of  Person\n\n	(_displayName \"Jim√É¬©nez, JL\")\n	(DB_ID 977082)\n	(initial \"JL\")\n	(surname \"Jim√É¬©nez\"))\n\n([GK_977083] of  Person\n\n	(_displayName \"Donnell, RL\")\n	(DB_ID 977083)\n	(initial \"RL\")\n	(surname \"Donnell\"))\n\n([GK_977084] of  Complex\n\n	(_displayName \"[Hsa] Amyloid fibrils [extracellular region]\")\n	(compartment\n		[GK_984]\n		[GK_876])\n	(DB_ID 977084)\n	(hasComponent\n		[GK_977144]\n		[GK_976776]\n		[GK_174719]\n		[GK_977180])\n	(isChimeric FALSE)\n	(name \"Amyloid fibrils\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358358]))\n\n([GK_977085] of  LiteratureReference\n\n	(_displayName \"Multiple assembly pathways underlie amyloid-beta fibril polymorphisms\")\n	(author\n		[GK_977214]\n		[GK_977095]\n		[GK_977204]\n		[GK_913920]\n		[GK_977129])\n	(DB_ID 977085)\n	(journal \"J Mol Biol\")\n	(pages \"282-98\")\n	(pubMedIdentifier 16095615)\n	(title \"Multiple assembly pathways underlie amyloid-beta fibril polymorphisms\")\n	(volume 352)\n	(year 2005))\n\n([GK_977086] of  Person\n\n	(_displayName \"Chiti, F\")\n	(DB_ID 977086)\n	(initial \"F\")\n	(surname \"Chiti\"))\n\n([GK_977087] of  Polymer\n\n	(_displayName \"[Hsa] Odontogenic ameloblast-associated protein amyloid fibrils [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977087)\n	(literatureReference [GK_977213])\n	(name \"Odontogenic ameloblast-associated protein amyloid fibrils\")\n	(repeatedUnit [GK_977149])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357973]))\n\n([GK_977088] of  Person\n\n	(_displayName \"Brunden, KR\")\n	(DB_ID 977088)\n	(initial \"KR\")\n	(surname \"Brunden\"))\n\n([GK_977089] of  Person\n\n	(_displayName \"Purvis, A\")\n	(DB_ID 977089)\n	(initial \"A\")\n	(surname \"Purvis\"))\n\n([GK_977090] of  Person\n\n	(_displayName \"Baratz, KH\")\n	(DB_ID 977090)\n	(initial \"KH\")\n	(surname \"Baratz\"))\n\n([GK_977091] of  LiteratureReference\n\n	(_displayName \"Insulin as an amyloid-fibril protein at sites of repeated insulin injections in a diabetic patient\")\n	(author\n		[GK_977133]\n		[GK_163570]\n		[GK_374884]\n		[GK_976884]\n		[GK_976739]\n		[GK_977159]\n		[GK_469669]\n		[GK_977200])\n	(DB_ID 977091)\n	(journal \"Diabetologia\")\n	(pages \"158-61\")\n	(pubMedIdentifier 3286343)\n	(title \"Insulin as an amyloid-fibril protein at sites of repeated insulin injections in a diabetic patient\")\n	(volume 31)\n	(year 1988))\n\n([GK_977092] of  Person\n\n	(_displayName \"Rocken, C\")\n	(DB_ID 977092)\n	(initial \"C\")\n	(surname \"Rocken\"))\n\n([GK_977093] of  Person\n\n	(_displayName \"Glennert, J\")\n	(DB_ID 977093)\n	(initial \"J\")\n	(surname \"Glennert\"))\n\n([GK_977094] of  LiteratureReference\n\n	(_displayName \"Temporal relationship between glycosaminoglycan accumulation and amyloid deposition during experimental amyloidosis. A histochemical study\")\n	(author\n		[GK_977167]\n		[GK_977078])\n	(DB_ID 977094)\n	(journal \"Lab Invest\")\n	(pages \"37-44\")\n	(pubMedIdentifier 2409350)\n	(title \"Temporal relationship between glycosaminoglycan accumulation and amyloid deposition during experimental amyloidosis. A histochemical study\")\n	(volume 53)\n	(year 1985))\n\n([GK_977095] of  Person\n\n	(_displayName \"Frey, P\")\n	(DB_ID 977095)\n	(initial \"P\")\n	(surname \"Frey\"))\n\n([GK_977096] of  Polymer\n\n	(_displayName \"[Hsa] Medin amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977096)\n	(literatureReference\n		[GK_977154]\n		[GK_977187])\n	(name \"Medin amyloid fibril\")\n	(repeatedUnit [GK_976952])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358567]))\n\n([GK_977097] of  Person\n\n	(_displayName \"Dobson, CM\")\n	(DB_ID 977097)\n	(initial \"CM\")\n	(surname \"Dobson\"))\n\n([GK_977098] of  Person\n\n	(_displayName \"Nordstedt, C\")\n	(DB_ID 977098)\n	(initial \"C\")\n	(surname \"Nordstedt\"))\n\n([GK_977099] of  LiteratureReference\n\n	(_displayName \"The protofilament structure of insulin amyloid fibrils\")\n	(author\n		[GK_977082]\n		[GK_977113]\n		[GK_977116]\n		[GK_358192]\n		[GK_977097]\n		[GK_977210])\n	(DB_ID 977099)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"9196-201\")\n	(pubMedIdentifier 12093917)\n	(title \"The protofilament structure of insulin amyloid fibrils\")\n	(volume 99)\n	(year 2002))\n\n([GK_977100] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Semenogelin-1 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977100)\n	(endCoordinate -1)\n	(name\n		\"Semenogelin-1\"\n		\"SEMG1\"\n		\"SEMG1_HUMAN\"\n		\"SEMG1\")\n	(referenceEntity [GK_64296])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358399])\n	(startCoordinate 1))\n\n([GK_977101] of  Polymer\n\n	(_displayName \"[Hsa] Lactotransferrin amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977101)\n	(literatureReference [GK_977118])\n	(name\n		\"Lactotransferrin amyloid fibril\"\n		\"Lactoferrin amyloid fibril\")\n	(repeatedUnit [GK_977171])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358001]))\n\n([GK_977102] of  Person\n\n	(_displayName \"Blasi, MA\")\n	(DB_ID 977102)\n	(initial \"MA\")\n	(surname \"Blasi\"))\n\n([GK_977103] of  Person\n\n	(_displayName \"Escoto, R\")\n	(DB_ID 977103)\n	(initial \"R\")\n	(surname \"Escoto\"))\n\n([GK_977104] of  Person\n\n	(_displayName \"Bernstein, SL\")\n	(DB_ID 977104)\n	(initial \"SL\")\n	(surname \"Bernstein\"))\n\n([GK_977105] of  DefinedSet\n\n	(_displayName \"[Hsa] Systemic amyloid fibrim monomers [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977105)\n	(hasMember\n		[GK_976881]\n		[GK_976882]\n		[GK_1302721]\n		[GK_976883]\n		[GK_976865]\n		[GK_976870]\n		[GK_976940]\n		[GK_976944]\n		[GK_976891]\n		[GK_976905]\n		[GK_976878])\n	(name \"Systemic amyloid fibrim monomers\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1357748]))\n\n([GK_977106] of  Person\n\n	(_displayName \"Alexandrescu, AT\")\n	(DB_ID 977106)\n	(initial \"AT\")\n	(surname \"Alexandrescu\"))\n\n([GK_977107] of  Person\n\n	(_displayName \"Seto, K\")\n	(DB_ID 977107)\n	(initial \"K\")\n	(surname \"Seto\"))\n\n([GK_977108] of  Person\n\n	(_displayName \"Suda, H\")\n	(DB_ID 977108)\n	(initial \"H\")\n	(surname \"Suda\"))\n\n([GK_977109] of  LiteratureReference\n\n	(_displayName \"Heparan sulfate proteoglycans in amyloidosis\")\n	(author\n		[GK_114129]\n		[GK_977158])\n	(DB_ID 977109)\n	(journal \"Prog Mol Biol Transl Sci\")\n	(pages \"309-34\")\n	(pubMedIdentifier 20807650)\n	(title \"Heparan sulfate proteoglycans in amyloidosis\")\n	(volume 93)\n	(year 2010))\n\n([GK_977110] of  Person\n\n	(_displayName \"Cousin, P\")\n	(DB_ID 977110)\n	(initial \"P\")\n	(surname \"Cousin\"))\n\n([GK_977111] of  Summation\n\n	(_displayName \"Amyloid fibril formation is associated with a wide range of ...\")\n	(DB_ID 977111)\n	(literatureReference\n		[GK_977131]\n		[GK_1247805]\n		[GK_977199]\n		[GK_976862]\n		[GK_977195]\n		[GK_977085])\n	(text \"Amyloid fibril formation is associated with a wide range of diseases (Chiti & Dobson 2006), thoughthe accumulation and deposition of fibrillar material does not correlate well with disease pathogenesis and it is now widely believed that oligomeric amyloid forms are  largely responsible for the cytotoxic effects of amyloid (Glabe 2009). Fibrils have been described as more like crystalline polymer structures than the protein monomers they are derived from (Wetzel et al. 2007). Fibril formation is usually preceded by the association of monomers into oligomeric structures (Kodali & Wetzel 2007). For Beta-amyloid, these are spherical structures with around 12 units (Bernstein et al. 2005). Larger structures called protofibrils are also observed, non-spherical filamentous structures lacking a periodic substructure (Goldsbury 2005). \"))\n\n([GK_977112] of  Person\n\n	(_displayName \"Hansen, JB\")\n	(DB_ID 977112)\n	(initial \"JB\")\n	(surname \"Hansen\"))\n\n([GK_977113] of  Person\n\n	(_displayName \"Nettleton, EJ\")\n	(DB_ID 977113)\n	(initial \"EJ\")\n	(surname \"Nettleton\"))\n\n([GK_977114] of  Person\n\n	(_displayName \"Ngo, C\")\n	(DB_ID 977114)\n	(initial \"C\")\n	(surname \"Ngo\"))\n\n([GK_977115] of  LiteratureReference\n\n	(_displayName \"Proteoglycans in the pathogenesis of Alzheimer\'s disease and other amyloidoses\")\n	(author\n		[GK_977167]\n		[GK_977185])\n	(DB_ID 977115)\n	(journal \"Neurobiol Aging\")\n	(pages \"481-97\")\n	(pubMedIdentifier 2682326)\n	(title \"Proteoglycans in the pathogenesis of Alzheimer\'s disease and other amyloidoses\")\n	(volume 10)\n	(year 1989))\n\n([GK_977116] of  Person\n\n	(_displayName \"Bouchard, M\")\n	(DB_ID 977116)\n	(initial \"M\")\n	(surname \"Bouchard\"))\n\n([GK_977117] of  Person\n\n	(_displayName \"Hrncic, R\")\n	(DB_ID 977117)\n	(initial \"R\")\n	(surname \"Hrncic\"))\n\n([GK_977118] of  LiteratureReference\n\n	(_displayName \"Familial subepithelial corneal amyloidosis--a lactoferrin-related amyloidosis\")\n	(author\n		[GK_977080]\n		[GK_977208]\n		[GK_977134]\n		[GK_977090]\n		[GK_977201]\n		[GK_538696])\n	(DB_ID 977118)\n	(journal \"Invest Ophthalmol Vis Sci\")\n	(pages \"2756-63\")\n	(pubMedIdentifier 9418728)\n	(title \"Familial subepithelial corneal amyloidosis--a lactoferrin-related amyloidosis\")\n	(volume 38)\n	(year 1997))\n\n([GK_977119] of  Person\n\n	(_displayName \"Linke, RP\")\n	(DB_ID 977119)\n	(initial \"RP\")\n	(surname \"Linke\"))\n\n([GK_977120] of  Person\n\n	(_displayName \"Gejyo, F\")\n	(DB_ID 977120)\n	(initial \"F\")\n	(surname \"Gejyo\"))\n\n([GK_977121] of  LiteratureReference\n\n	(_displayName \"Interactions of Alzheimer amyloid-beta peptides with glycosaminoglycans effects on fibril nucleation and growth\")\n	(author\n		[GK_977202]\n		[GK_977211]\n		[GK_114129]\n		[GK_977186]\n		[GK_977130])\n	(DB_ID 977121)\n	(journal \"Eur J Biochem\")\n	(pages \"1101-10\")\n	(pubMedIdentifier 10583407)\n	(title \"Interactions of Alzheimer amyloid-beta peptides with glycosaminoglycans effects on fibril nucleation and growth\")\n	(volume 266)\n	(year 1999))\n\n([GK_977122] of  Person\n\n	(_displayName \"Shea, JE\")\n	(DB_ID 977122)\n	(initial \"JE\")\n	(surname \"Shea\"))\n\n([GK_977123] of  Person\n\n	(_displayName \"Eisenberg, D\")\n	(DB_ID 977123)\n	(initial \"D\")\n	(surname \"Eisenberg\"))\n\n([GK_977124] of  Person\n\n	(_displayName \"Dodart, JC\")\n	(DB_ID 977124)\n	(initial \"JC\")\n	(surname \"Dodart\"))\n\n([GK_977125] of  Person\n\n	(_displayName \"Othenin-Girard, P\")\n	(DB_ID 977125)\n	(initial \"P\")\n	(surname \"Othenin-Girard\"))\n\n([GK_977126] of  LiteratureReference\n\n	(_displayName \"Amyloid P component. A critical review\")\n	(author\n		[GK_976739]\n		[GK_976907]\n		[GK_977153]\n		[GK_977145]\n		[GK_977170]\n		[GK_70650])\n	(DB_ID 977126)\n	(journal \"Amyloid\")\n	(title \"Amyloid P component. A critical review\")\n	(volume 4)\n	(year 1997))\n\n([GK_977127] of  LiteratureReference\n\n	(_displayName \"BIGH3 mutation spectrum in corneal dystrophies\")\n	(author\n		[GK_216603]\n		[GK_977212]\n		[GK_977125]\n		[GK_977150]\n		[GK_977110]\n		[GK_977143]\n		[GK_977137]\n		[GK_977217]\n		[GK_977102]\n		[GK_977128]\n		[GK_419837]\n		[GK_977103]\n		[GK_977215]\n		[GK_977194]\n		[GK_977164]\n		[GK_977179]\n		[GK_977176]\n		[GK_977209]\n		[GK_977135]\n		[GK_216645])\n	(DB_ID 977127)\n	(journal \"Invest Ophthalmol Vis Sci\")\n	(pages \"949-54\")\n	(pubMedIdentifier 11923233)\n	(title \"BIGH3 mutation spectrum in corneal dystrophies\")\n	(volume 43)\n	(year 2002))\n\n([GK_977128] of  Person\n\n	(_displayName \"Balestrazzi, E\")\n	(DB_ID 977128)\n	(initial \"E\")\n	(surname \"Balestrazzi\"))\n\n([GK_977129] of  Person\n\n	(_displayName \"M√É¬ºller, SA\")\n	(DB_ID 977129)\n	(initial \"SA\")\n	(surname \"M√É¬ºller\"))\n\n([GK_977130] of  Person\n\n	(_displayName \"Fraser, PE\")\n	(DB_ID 977130)\n	(initial \"PE\")\n	(surname \"Fraser\"))\n\n([GK_977131] of  LiteratureReference\n\n	(_displayName \"Protein misfolding, functional amyloid, and human disease\")\n	(author\n		[GK_977086]\n		[GK_977097])\n	(DB_ID 977131)\n	(journal \"Annu Rev Biochem\")\n	(pages \"333-66\")\n	(pubMedIdentifier 16756495)\n	(title \"Protein misfolding, functional amyloid, and human disease\")\n	(volume 75)\n	(year 2006))\n\n([GK_977132] of  Person\n\n	(_displayName \"DeMattos, RB\")\n	(DB_ID 977132)\n	(initial \"RB\")\n	(surname \"DeMattos\"))\n\n([GK_977133] of  Person\n\n	(_displayName \"Dische, FE\")\n	(DB_ID 977133)\n	(initial \"FE\")\n	(surname \"Dische\"))\n\n([GK_977134] of  Person\n\n	(_displayName \"Kielar, RA\")\n	(DB_ID 977134)\n	(initial \"RA\")\n	(surname \"Kielar\"))\n\n([GK_977135] of  Person\n\n	(_displayName \"Zografos, L\")\n	(DB_ID 977135)\n	(initial \"L\")\n	(surname \"Zografos\"))\n\n([GK_977136] of  Polymerisation\n\n	(_displayName \"[Hsa] Amyloid precursor proteins form ordered fibrils\")\n	(_doRelease TRUE)\n	(authored [GK_977142])\n	(compartment [GK_984])\n	(DB_ID 977136)\n	(edited [GK_1247884])\n	(input [GK_977175])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_1247791]\n		[GK_1247809]\n		[GK_1247702]\n		[GK_1247697]\n		[GK_1247691]\n		[GK_1247711]\n		[GK_976941]\n		[GK_976942]\n		[GK_976949]\n		[GK_976852]\n		[GK_976877]\n		[GK_976921]\n		[GK_976879]\n		[GK_976939]\n		[GK_1247724]\n		[GK_1247757]\n		[GK_1247784]\n		[GK_976970]\n		[GK_1247766]\n		[GK_1247781]\n		[GK_976959]\n		[GK_1247801]\n		[GK_977154]\n		[GK_977166]\n		[GK_1247709]\n		[GK_1247717]\n		[GK_977189])\n	(name \"Amyloid precursor proteins form ordered fibrils\")\n	(output [GK_977144])\n	(releaseDate \"2011-06-14\")\n	(reviewed [GK_1247882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357608])\n	(summation [GK_977111]))\n\n([GK_977137] of  Person\n\n	(_displayName \"H√É¬©on, E\")\n	(DB_ID 977137)\n	(initial \"E\")\n	(surname \"H√É¬©on\"))\n\n([GK_977138] of  Person\n\n	(_displayName \"Rimvall, K\")\n	(DB_ID 977138)\n	(initial \"K\")\n	(surname \"Rimvall\"))\n\n([GK_977139] of  LiteratureReference\n\n	(_displayName \"Molecular chaperone properties of serum amyloid P component\")\n	(author\n		[GK_977196]\n		[GK_977089]\n		[GK_500144]\n		[GK_976739]\n		[GK_976731])\n	(DB_ID 977139)\n	(journal \"FEBS Lett\")\n	(pages \"199-202\")\n	(pubMedIdentifier 10812074)\n	(title \"Molecular chaperone properties of serum amyloid P component\")\n	(volume 473)\n	(year 2000))\n\n([GK_977140] of  Person\n\n	(_displayName \"Tjernberg, LO\")\n	(DB_ID 977140)\n	(initial \"LO\")\n	(surname \"Tjernberg\"))\n\n([GK_977141] of  Person\n\n	(_displayName \"Djema√É¬Ø, A\")\n	(DB_ID 977141)\n	(initial \"A\")\n	(surname \"Djema√É¬Ø\"))\n\n([GK_977142] of  InstanceEdit\n\n	(_displayName \"Jupe, S, 2010-10-15\")\n	(author [GK_391309])\n	(dateTime \"2010-10-15 13:19:12\")\n	(DB_ID 977142))\n\n([GK_977143] of  Person\n\n	(_displayName \"Wang, MX\")\n	(DB_ID 977143)\n	(initial \"MX\")\n	(surname \"Wang\"))\n\n([GK_977144] of  DefinedSet\n\n	(_displayName \"[Hsa] Amyloid fibril main peptide chains [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977144)\n	(hasMember\n		[GK_976963]\n		[GK_976760]\n		[GK_1247852])\n	(name \"Amyloid fibril main peptide chains\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1357743]))\n\n([GK_977145] of  Person\n\n	(_displayName \"Gallimore, J\")\n	(DB_ID 977145)\n	(initial \"J\")\n	(surname \"Gallimore\"))\n\n([GK_977146] of  Person\n\n	(_displayName \"Nelson, R\")\n	(DB_ID 977146)\n	(initial \"R\")\n	(surname \"Nelson\"))\n\n([GK_977147] of  Person\n\n	(_displayName \"Cummings, J\")\n	(DB_ID 977147)\n	(initial \"J\")\n	(surname \"Cummings\"))\n\n([GK_977148] of  LiteratureReference\n\n	(_displayName \"Amyloidosis: new strategies for treatment\")\n	(author\n		[GK_976806]\n		[GK_976869])\n	(DB_ID 977148)\n	(journal \"Int J Biochem Cell Biol\")\n	(pages \"1608-13\")\n	(pubMedIdentifier 12962700)\n	(title \"Amyloidosis: new strategies for treatment\")\n	(volume 35)\n	(year 2003))\n\n([GK_977149] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Odontogenic ameloblast-associated protein [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977149)\n	(endCoordinate -1)\n	(name\n		\"Odontogenic ameloblast-associated protein\"\n		\"Apin\"\n		\"ODAM_HUMAN\"\n		\"ODAM\")\n	(referenceEntity [GK_240941])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358376])\n	(startCoordinate 1))\n\n([GK_977150] of  Person\n\n	(_displayName \"Uffer, S\")\n	(DB_ID 977150)\n	(initial \"S\")\n	(surname \"Uffer\"))\n\n([GK_977151] of  Person\n\n	(_displayName \"Korvatska, E\")\n	(DB_ID 977151)\n	(initial \"E\")\n	(surname \"Korvatska\"))\n\n([GK_977152] of  Person\n\n	(_displayName \"Le Paslier, D\")\n	(DB_ID 977152)\n	(initial \"D\")\n	(surname \"Le Paslier\"))\n\n([GK_977153] of  Person\n\n	(_displayName \"Hutchinson, WL\")\n	(DB_ID 977153)\n	(initial \"WL\")\n	(surname \"Hutchinson\"))\n\n([GK_977154] of  LiteratureReference\n\n	(_displayName \"Medin: an integral fragment of aortic smooth muscle cell-produced lactadherin forms the most common human amyloid\")\n	(author\n		[GK_977173]\n		[GK_977218]\n		[GK_976906]\n		[GK_374884]\n		[GK_977198]\n		[GK_977140]\n		[GK_977098]\n		[GK_976986]\n		[GK_976884])\n	(DB_ID 977154)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"8669-74\")\n	(pubMedIdentifier 10411933)\n	(title \"Medin: an integral fragment of aortic smooth muscle cell-produced lactadherin forms the most common human amyloid\")\n	(volume 96)\n	(year 1999))\n\n([GK_977155] of  Person\n\n	(_displayName \"Bowers, MT\")\n	(DB_ID 977155)\n	(initial \"MT\")\n	(surname \"Bowers\"))\n\n([GK_977156] of  Person\n\n	(_displayName \"Frederickson, RC\")\n	(DB_ID 977156)\n	(initial \"RC\")\n	(surname \"Frederickson\"))\n\n([GK_977157] of  Person\n\n	(_displayName \"Baumketner, A\")\n	(DB_ID 977157)\n	(initial \"A\")\n	(surname \"Baumketner\"))\n\n([GK_977158] of  Person\n\n	(_displayName \"Li, JP\")\n	(DB_ID 977158)\n	(initial \"JP\")\n	(surname \"Li\"))\n\n([GK_977159] of  Person\n\n	(_displayName \"Rennie, JA\")\n	(DB_ID 977159)\n	(initial \"JA\")\n	(surname \"Rennie\"))\n\n([GK_977160] of  Person\n\n	(_displayName \"Bales, KR\")\n	(DB_ID 977160)\n	(initial \"KR\")\n	(surname \"Bales\"))\n\n([GK_977161] of  Person\n\n	(_displayName \"Shivaprasad, S\")\n	(DB_ID 977161)\n	(initial \"S\")\n	(surname \"Shivaprasad\"))\n\n([GK_977162] of  LiteratureReference\n\n	(_displayName \"Sulfate content and specific glycosaminoglycan backbone of perlecan are critical for perlecan\'s enhancement of islet amyloid polypeptide (amylin) fibril formation\")\n	(author\n		[GK_977169]\n		[GK_977197]\n		[GK_203163]\n		[GK_977191]\n		[GK_977207]\n		[GK_977138]\n		[GK_977112]\n		[GK_977167])\n	(DB_ID 977162)\n	(journal \"Diabetes\")\n	(pages \"612-20\")\n	(pubMedIdentifier 9568695)\n	(title \"Sulfate content and specific glycosaminoglycan backbone of perlecan are critical for perlecan\'s enhancement of islet amyloid polypeptide (amylin) fibril formation\")\n	(volume 47)\n	(year 1998))\n\n([GK_977163] of  LiteratureReference\n\n	(_displayName \"Serum amyloid P component prevents proteolysis of the amyloid fibrils of Alzheimer disease and systemic amyloidosis\")\n	(author\n		[GK_976947]\n		[GK_977182]\n		[GK_976739])\n	(DB_ID 977163)\n	(journal \"Proc Natl Acad Sci U S A\")\n	(pages \"4299-303\")\n	(pubMedIdentifier 7753801)\n	(title \"Serum amyloid P component prevents proteolysis of the amyloid fibrils of Alzheimer disease and systemic amyloidosis\")\n	(volume 92)\n	(year 1995))\n\n([GK_977164] of  Person\n\n	(_displayName \"Gloor, B\")\n	(DB_ID 977164)\n	(initial \"B\")\n	(surname \"Gloor\"))\n\n([GK_977165] of  Person\n\n	(_displayName \"Wyttenbach, T\")\n	(DB_ID 977165)\n	(initial \"T\")\n	(surname \"Wyttenbach\"))\n\n([GK_977166] of  LiteratureReference\n\n	(_displayName \"Kerato-epithelin mutations in four 5q31-linked corneal dystrophies\")\n	(author\n		[GK_216603]\n		[GK_977151]\n		[GK_977141]\n		[GK_977152]\n		[GK_977135]\n		[GK_977184]\n		[GK_216645])\n	(DB_ID 977166)\n	(journal \"Nat Genet\")\n	(pages \"247-51\")\n	(pubMedIdentifier 9054935)\n	(title \"Kerato-epithelin mutations in four 5q31-linked corneal dystrophies\")\n	(volume 15)\n	(year 1997))\n\n([GK_977167] of  Person\n\n	(_displayName \"Snow, AD\")\n	(DB_ID 977167)\n	(initial \"AD\")\n	(surname \"Snow\"))\n\n([GK_977168] of  Person\n\n	(_displayName \"Williams, AD\")\n	(DB_ID 977168)\n	(initial \"AD\")\n	(surname \"Williams\"))\n\n([GK_977169] of  Person\n\n	(_displayName \"Castillo, GM\")\n	(DB_ID 977169)\n	(initial \"GM\")\n	(surname \"Castillo\"))\n\n([GK_977170] of  Person\n\n	(_displayName \"Collins, PM\")\n	(DB_ID 977170)\n	(initial \"PM\")\n	(surname \"Collins\"))\n\n([GK_977171] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Lactotransferrin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977171)\n	(endCoordinate 710)\n	(name\n		\"Lactotransferrin\"\n		\"Lactoferrin\"\n		\"TRFL_HUMAN\"\n		\"LTF\")\n	(referenceEntity [GK_66401])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357954])\n	(startCoordinate 20))\n\n([GK_977172] of  LiteratureReference\n\n	(_displayName \"Structural models of amyloid-like fibrils\")\n	(author\n		[GK_977146]\n		[GK_977123])\n	(DB_ID 977172)\n	(journal \"Adv Protein Chem\")\n	(pages \"235-82\")\n	(pubMedIdentifier 17190616)\n	(title \"Structural models of amyloid-like fibrils\")\n	(volume 73)\n	(year 2006))\n\n([GK_977173] of  Person\n\n	(_displayName \"H√É¬§ggqvist, B\")\n	(DB_ID 977173)\n	(initial \"B\")\n	(surname \"H√É¬§ggqvist\"))\n\n([GK_977174] of  Summation\n\n	(_displayName \"In addition to the main fibril peptide, mature amyloid fibri...\")\n	(DB_ID 977174)\n	(literatureReference\n		[GK_977163]\n		[GK_976859]\n		[GK_977139]\n		[GK_977177]\n		[GK_977148]\n		[GK_977077]\n		[GK_977190]\n		[GK_977115]\n		[GK_977162]\n		[GK_977183]\n		[GK_977081])\n	(text \"In addition to the main fibril peptide, mature amyloid fibrils have additional components. Serum amyloid P component (SAP) binds to all types of amyloid fibrils and is a universal constituent of amyloid deposits. SAP binding protects amyloid fibrils from proteolytic degradation (Tennent et al. 1995, Westermark 2005). SAP may function as a chaperone for amyloid formation (Coker et al. 2000).\\n\\nGlycosaminoglycans (GAGs) and proteoglycans are found associated with all types of amyloid deposits (Alexandrescu 2005). Of the different types of GAG heparan sulfate and dermatan sulfate are the most prominent in amyloid deposits (Hirschfield & Hawkins, 2003). GAGs have been implicated in the nucleation of fibrils, they can also stabilize mature fibrils against dissociation (Yamaguchi et al. 2003) and proteolytic degradation (Gupta-Bansal et al. 1995).\\n\\nPerlecan coimmunolocalizes with all types of amyloids (Snow & Wright 1989), accelerating fibril formation (Castillo et al. 1998), stabilizing them once formed (Castillo et al. 1997), and protecting them from proteolytic degradation (Gupta-Bansal et al. 1995).\\n\\nApoE tightly binds to soluble ABeta peptide forming complexes that resist dissociation; it also binds to ABeta in its fibril form (Bales et al. 2002).  \"))\n\n([GK_977175] of  DefinedSet\n\n	(_displayName \"[Hsa] Amyloid fibril monomers [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977175)\n	(hasMember\n		[GK_977181]\n		[GK_977105]\n		[GK_1247707])\n	(name \"Amyloid fibril monomers\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358010]))\n\n([GK_977176] of  Person\n\n	(_displayName \"Singh, AD\")\n	(DB_ID 977176)\n	(initial \"AD\")\n	(surname \"Singh\"))\n\n([GK_977177] of  LiteratureReference\n\n	(_displayName \"Amyloid accomplices and enforcers\")\n	(author [GK_977106])\n	(DB_ID 977177)\n	(journal \"Protein Sci\")\n	(pages \"1-12\")\n	(pubMedIdentifier 15576561)\n	(title \"Amyloid accomplices and enforcers\")\n	(volume 14)\n	(year 2005))\n\n([GK_977178] of  Person\n\n	(_displayName \"Holtzman, DM\")\n	(DB_ID 977178)\n	(initial \"DM\")\n	(surname \"Holtzman\"))\n\n([GK_977179] of  Person\n\n	(_displayName \"Fossarello, M\")\n	(DB_ID 977179)\n	(initial \"M\")\n	(surname \"Fossarello\"))\n\n([GK_977180] of  SimpleEntity\n\n	(_displayName \"Glycosaminoglycan [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977180)\n	(name \"Glycosaminoglycan\")\n	(referenceEntity [GK_977192])\n	(stableIdentifier [GK_1358187]))\n\n([GK_977181] of  DefinedSet\n\n	(_displayName \"[Hsa] Localized amyloid fibril monomers [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977181)\n	(hasMember\n		[GK_976800]\n		[GK_419781]\n		[GK_420076]\n		[GK_976950]\n		[GK_976984]\n		[GK_429407]\n		[GK_976952]\n		[GK_1302722]\n		[GK_977171]\n		[GK_977100]\n		[GK_977149])\n	(name \"Localized amyloid fibril monomers\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358592]))\n\n([GK_977182] of  Person\n\n	(_displayName \"Lovat, LB\")\n	(DB_ID 977182)\n	(initial \"LB\")\n	(surname \"Lovat\"))\n\n([GK_977183] of  LiteratureReference\n\n	(_displayName \"Perlecan binds to the beta-amyloid proteins (A beta) of Alzheimer\'s disease, accelerates A beta fibril formation, and maintains A beta fibril stability\")\n	(author\n		[GK_977169]\n		[GK_977114]\n		[GK_977147]\n		[GK_977185]\n		[GK_977167])\n	(DB_ID 977183)\n	(journal \"J Neurochem\")\n	(pages \"2452-65\")\n	(pubMedIdentifier 9375678)\n	(title \"Perlecan binds to the beta-amyloid proteins (A beta) of Alzheimer\'s disease, accelerates A beta fibril formation, and maintains A beta fibril stability\")\n	(volume 69)\n	(year 1997))\n\n([GK_977184] of  Person\n\n	(_displayName \"Pescia, G\")\n	(DB_ID 977184)\n	(initial \"G\")\n	(surname \"Pescia\"))\n\n([GK_977185] of  Person\n\n	(_displayName \"Wight, TN\")\n	(DB_ID 977185)\n	(initial \"TN\")\n	(surname \"Wight\"))\n\n([GK_977186] of  Person\n\n	(_displayName \"Deng, J\")\n	(DB_ID 977186)\n	(initial \"J\")\n	(surname \"Deng\"))\n\n([GK_977187] of  LiteratureReference\n\n	(_displayName \"Medin-amyloid: a recently characterized age-associated arterial amyloid form affects mainly arteries in the upper part of the body\")\n	(author\n		[GK_500361]\n		[GK_977093]\n		[GK_976884])\n	(DB_ID 977187)\n	(journal \"Amyloid\")\n	(pages \"96-102\")\n	(pubMedIdentifier 16011985)\n	(title \"Medin-amyloid: a recently characterized age-associated arterial amyloid form affects mainly arteries in the upper part of the body\")\n	(volume 12)\n	(year 2005))\n\n([GK_977188] of  Polymer\n\n	(_displayName \"[Hsa] Kerato-epithelin amyloid fibril [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977188)\n	(literatureReference\n		[GK_977127]\n		[GK_977166])\n	(name \"Kerato-epithelin amyloid fibril\")\n	(repeatedUnit [GK_204605])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358373]))\n\n([GK_977189] of  LiteratureReference\n\n	(_displayName \"Senile seminal vesicle amyloid is derived from semenogelin I\")\n	(author\n		[GK_977119]\n		[GK_977206]\n		[GK_976849]\n		[GK_202881]\n		[GK_571213]\n		[GK_977220]\n		[GK_977092]\n		[GK_976884]\n		[GK_976922]\n		[GK_976858])\n	(DB_ID 977189)\n	(journal \"J Lab Clin Med\")\n	(pages \"187-93\")\n	(pubMedIdentifier 15962837)\n	(title \"Senile seminal vesicle amyloid is derived from semenogelin I\")\n	(volume 145)\n	(year 2005))\n\n([GK_977190] of  LiteratureReference\n\n	(_displayName \"Proteoglycan-mediated inhibition of A beta proteolysis. A potential cause of senile plaque accumulation\")\n	(author\n		[GK_977221]\n		[GK_977156]\n		[GK_977088])\n	(DB_ID 977190)\n	(journal \"J Biol Chem\")\n	(pages \"18666-71\")\n	(pubMedIdentifier 7629198)\n	(title \"Proteoglycan-mediated inhibition of A beta proteolysis. A potential cause of senile plaque accumulation\")\n	(volume 270)\n	(year 1995))\n\n([GK_977191] of  Person\n\n	(_displayName \"Judge, ME\")\n	(DB_ID 977191)\n	(initial \"ME\")\n	(surname \"Judge\"))\n\n([GK_977192] of  ReferenceMolecule\n\n	(_displayName \"ChEBI:18085 Glycosaminoglycan\")\n	(DB_ID 977192)\n	(identifier \"18085\")\n	(name \"Glycosaminoglycan\")\n	(referenceDatabase [GK_114984]))\n\n([GK_977193] of  Person\n\n	(_displayName \"Tsuzuike, N\")\n	(DB_ID 977193)\n	(initial \"N\")\n	(surname \"Tsuzuike\"))\n\n([GK_977194] of  Person\n\n	(_displayName \"Hoeltzenbein, M\")\n	(DB_ID 977194)\n	(initial \"M\")\n	(surname \"Hoeltzenbein\"))\n\n([GK_977195] of  LiteratureReference\n\n	(_displayName \"Amyloid beta-protein: monomer structure and early aggregation states of Abeta42 and its Pro19 alloform\")\n	(author\n		[GK_977104]\n		[GK_977165]\n		[GK_977157]\n		[GK_977122]\n		[GK_977205]\n		[GK_75542]\n		[GK_977155])\n	(DB_ID 977195)\n	(journal \"J Am Chem Soc\")\n	(pages \"2075-84\")\n	(pubMedIdentifier 15713083)\n	(title \"Amyloid beta-protein: monomer structure and early aggregation states of Abeta42 and its Pro19 alloform\")\n	(volume 127)\n	(year 2005))\n\n([GK_977196] of  Person\n\n	(_displayName \"Coker, AR\")\n	(DB_ID 977196)\n	(initial \"AR\")\n	(surname \"Coker\"))\n\n([GK_977197] of  Person\n\n	(_displayName \"Cummings, JA\")\n	(DB_ID 977197)\n	(initial \"JA\")\n	(surname \"Cummings\"))\n\n([GK_977198] of  Person\n\n	(_displayName \"Mucchiano, G\")\n	(DB_ID 977198)\n	(initial \"G\")\n	(surname \"Mucchiano\"))\n\n([GK_977199] of  LiteratureReference\n\n	(_displayName \"Plasticity of amyloid fibrils\")\n	(author\n		[GK_976917]\n		[GK_977161]\n		[GK_977168])\n	(DB_ID 977199)\n	(journal \"Biochemistry\")\n	(pages \"1-10\")\n	(pubMedIdentifier 17198370)\n	(title \"Plasticity of amyloid fibrils\")\n	(volume 46)\n	(year 2007))\n\n([GK_977200] of  Person\n\n	(_displayName \"Watkins, PJ\")\n	(DB_ID 977200)\n	(initial \"PJ\")\n	(surname \"Watkins\"))\n\n([GK_977201] of  Person\n\n	(_displayName \"Campbell, RJ\")\n	(DB_ID 977201)\n	(initial \"RJ\")\n	(surname \"Campbell\"))\n\n([GK_977202] of  Person\n\n	(_displayName \"McLaurin, J\")\n	(DB_ID 977202)\n	(initial \"J\")\n	(surname \"McLaurin\"))\n\n([GK_977203] of  Person\n\n	(_displayName \"Weaver, K\")\n	(DB_ID 977203)\n	(initial \"K\")\n	(surname \"Weaver\"))\n\n([GK_977204] of  Person\n\n	(_displayName \"Olivieri, V\")\n	(DB_ID 977204)\n	(initial \"V\")\n	(surname \"Olivieri\"))\n\n([GK_977205] of  Person\n\n	(_displayName \"Bitan, G\")\n	(DB_ID 977205)\n	(initial \"G\")\n	(surname \"Bitan\"))\n\n([GK_977206] of  Person\n\n	(_displayName \"Joswig, R\")\n	(DB_ID 977206)\n	(initial \"R\")\n	(surname \"Joswig\"))\n\n([GK_977207] of  Person\n\n	(_displayName \"Sheardown, MJ\")\n	(DB_ID 977207)\n	(initial \"MJ\")\n	(surname \"Sheardown\"))\n\n([GK_977208] of  Person\n\n	(_displayName \"Valnickova, Z\")\n	(DB_ID 977208)\n	(initial \"Z\")\n	(surname \"Valnickova\"))\n\n([GK_977209] of  Person\n\n	(_displayName \"Arsenijevic, Y\")\n	(DB_ID 977209)\n	(initial \"Y\")\n	(surname \"Arsenijevic\"))\n\n([GK_977210] of  Person\n\n	(_displayName \"Saibil, HR\")\n	(DB_ID 977210)\n	(initial \"HR\")\n	(surname \"Saibil\"))\n\n([GK_977211] of  Person\n\n	(_displayName \"Franklin, T\")\n	(DB_ID 977211)\n	(initial \"T\")\n	(surname \"Franklin\"))\n\n([GK_977212] of  Person\n\n	(_displayName \"Frueh, BE\")\n	(DB_ID 977212)\n	(initial \"BE\")\n	(surname \"Frueh\"))\n\n([GK_977213] of  LiteratureReference\n\n	(_displayName \"Calcifying epithelial odontogenic (Pindborg) tumor-associated amyloid consists of a novel human protein\")\n	(author\n		[GK_976858]\n		[GK_976849]\n		[GK_977203]\n		[GK_976922]\n		[GK_977117]\n		[GK_207287]\n		[GK_977083]\n		[GK_976906]\n		[GK_977216]\n		[GK_976884])\n	(DB_ID 977213)\n	(journal \"J Lab Clin Med\")\n	(pages \"348-55\")\n	(pubMedIdentifier 14647039)\n	(title \"Calcifying epithelial odontogenic (Pindborg) tumor-associated amyloid consists of a novel human protein\")\n	(volume 142)\n	(year 2003))\n\n([GK_977214] of  Person\n\n	(_displayName \"Goldsbury, C\")\n	(DB_ID 977214)\n	(initial \"C\")\n	(surname \"Goldsbury\"))\n\n([GK_977215] of  Person\n\n	(_displayName \"Barraquer, R\")\n	(DB_ID 977215)\n	(initial \"R\")\n	(surname \"Barraquer\"))\n\n([GK_977216] of  Person\n\n	(_displayName \"Westermark, G\")\n	(DB_ID 977216)\n	(initial \"G\")\n	(surname \"Westermark\"))\n\n([GK_977217] of  Person\n\n	(_displayName \"Black, GC\")\n	(DB_ID 977217)\n	(initial \"GC\")\n	(surname \"Black\"))\n\n([GK_977218] of  Person\n\n	(_displayName \"N√É¬§slund, J\")\n	(DB_ID 977218)\n	(initial \"J\")\n	(surname \"N√É¬§slund\"))\n\n([GK_977219] of  Polymer\n\n	(_displayName \"[Hsa] Human insulin analogue amyloid fibrils [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977219)\n	(literatureReference\n		[GK_977091]\n		[GK_977099])\n	(name \"Human insulin analogue amyloid fibrils\")\n	(repeatedUnit [GK_429407])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357931]))\n\n([GK_977220] of  Person\n\n	(_displayName \"Gross, U\")\n	(DB_ID 977220)\n	(initial \"U\")\n	(surname \"Gross\"))\n\n([GK_977221] of  Person\n\n	(_displayName \"Gupta-Bansal, R\")\n	(DB_ID 977221)\n	(initial \"R\")\n	(surname \"Gupta-Bansal\"))\n\n([GK_977223] of  Complex\n\n	(_displayName \"[Hsa] Serum amyloid P-component pentamer:Double-stranded DNA [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977223)\n	(hasComponent\n		[GK_976776]\n		[GK_977589])\n	(isChimeric FALSE)\n	(name \"Serum amyloid P-component pentamer:Double-stranded DNA\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358194]))\n\n([GK_977224] of  Reaction\n\n	(_displayName \"[Hsa] Serum amyloid P binds DNA and chromatin\")\n	(_doRelease TRUE)\n	(authored [GK_977142])\n	(compartment [GK_984])\n	(DB_ID 977224)\n	(edited [GK_1247884])\n	(input\n		[GK_976776]\n		[GK_977589])\n	(isChimeric FALSE)\n	(literatureReference\n		[GK_976790]\n		[GK_977414])\n	(name \"Serum amyloid P binds DNA and chromatin\")\n	(output [GK_977223])\n	(precedingEvent [GK_976723])\n	(releaseDate \"2011-06-14\")\n	(reviewed [GK_1247882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357590])\n	(summation [GK_977425]))\n\n([GK_977225] of  Pathway\n\n	(_displayName \"[Hsa] Amyloids\")\n	(_doRelease TRUE)\n	(authored [GK_977142])\n	(compartment [GK_984])\n	(DB_ID 977225)\n	(edited [GK_1247884])\n	(hasEvent\n		[GK_976723]\n		[GK_977136]\n		[GK_976734]\n		[GK_976817]\n		[GK_977224])\n	(literatureReference [GK_976859])\n	(name \"Amyloids\")\n	(releaseDate \"2011-06-14\")\n	(reviewed [GK_1247882])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357657])\n	(summation [GK_976920]))\n\n([GK_977412] of  Person\n\n	(_displayName \"Walport, MJ\")\n	(DB_ID 977412)\n	(initial \"MJ\")\n	(surname \"Walport\"))\n\n([GK_977413] of  LiteratureReference\n\n	(_displayName \"Serum amyloid P component controls chromatin degradation and prevents antinuclear autoimmunity\")\n	(author\n		[GK_977419]\n		[GK_977423]\n		[GK_977153]\n		[GK_977421]\n		[GK_976947]\n		[GK_977415]\n		[GK_977418]\n		[GK_977417]\n		[GK_976721]\n		[GK_977412]\n		[GK_976739])\n	(DB_ID 977413)\n	(journal \"Nat Med\")\n	(pages \"694-7\")\n	(pubMedIdentifier 10371509)\n	(title \"Serum amyloid P component controls chromatin degradation and prevents antinuclear autoimmunity\")\n	(volume 5)\n	(year 1999))\n\n([GK_977414] of  LiteratureReference\n\n	(_displayName \"Binding of pentraxins to different nuclear structures: C-reactive protein binds to small nuclear ribonucleoprotein particles, serum amyloid P component binds to chromatin and nucleoli\")\n	(author\n		[GK_976739]\n		[GK_977427]\n		[GK_976947]\n		[GK_976721]\n		[GK_977424])\n	(DB_ID 977414)\n	(journal \"Clin Exp Immunol\")\n	(pages \"152-7\")\n	(pubMedIdentifier 8033412)\n	(title \"Binding of pentraxins to different nuclear structures: C-reactive protein binds to small nuclear ribonucleoprotein particles, serum amyloid P component binds to chromatin and nucleoli\")\n	(volume 97)\n	(year 1994))\n\n([GK_977415] of  Person\n\n	(_displayName \"Bybee, A\")\n	(DB_ID 977415)\n	(initial \"A\")\n	(surname \"Bybee\"))\n\n([GK_977416] of  LiteratureReference\n\n	(_displayName \"Pentraxin-chromatin interactions: serum amyloid P component specifically displaces H1-type histones and solubilizes native long chromatin\")\n	(author\n		[GK_976721]\n		[GK_976947]\n		[GK_976739])\n	(DB_ID 977416)\n	(journal \"J Exp Med\")\n	(pages \"13-8\")\n	(pubMedIdentifier 2358775)\n	(title \"Pentraxin-chromatin interactions: serum amyloid P component specifically displaces H1-type histones and solubilizes native long chromatin\")\n	(volume 172)\n	(year 1990))\n\n([GK_977417] of  Person\n\n	(_displayName \"Cook, HT\")\n	(DB_ID 977417)\n	(initial \"HT\")\n	(surname \"Cook\"))\n\n([GK_977418] of  Person\n\n	(_displayName \"Mitchell, DA\")\n	(DB_ID 977418)\n	(initial \"DA\")\n	(surname \"Mitchell\"))\n\n([GK_977419] of  Person\n\n	(_displayName \"Bickerstaff, MC\")\n	(DB_ID 977419)\n	(initial \"MC\")\n	(surname \"Bickerstaff\"))\n\n([GK_977421] of  Person\n\n	(_displayName \"Herbert, J\")\n	(DB_ID 977421)\n	(initial \"J\")\n	(surname \"Herbert\"))\n\n([GK_977423] of  Person\n\n	(_displayName \"Botto, M\")\n	(DB_ID 977423)\n	(initial \"M\")\n	(surname \"Botto\"))\n\n([GK_977424] of  Person\n\n	(_displayName \"Williams, DG\")\n	(DB_ID 977424)\n	(initial \"DG\")\n	(surname \"Williams\"))\n\n([GK_977425] of  Summation\n\n	(_displayName \"Serum amyloid P component (SAP) binds DNA and chromatin in a...\")\n	(DB_ID 977425)\n	(literatureReference\n		[GK_976790]\n		[GK_977416]\n		[GK_977413])\n	(text \"Serum amyloid P component (SAP) binds DNA and chromatin in a calcium dependent manner in physiological conditions (Pepys et al. 1987). This binding displaces H1-type histones (Butler et al. 1990), solubilizing chromatin which is otherwise insoluble in extracellular fluids. SAP may therefore participate in the in vivo handling of chromatin exposed by cell death. SAP knockout mice spontaneously develop antinuclear autoimmunity and severe glomerulonephritis, a phenotype resembling human systemic lupus erythematosus, a serious autoimmune disease, suggesting that SAP binding may play a role in reducing the immunogenicity of chromatin and preventing autoimmunity (Bickerstaff et al. 1999). \"))\n\n([GK_977427] of  Person\n\n	(_displayName \"Booth, SE\")\n	(DB_ID 977427)\n	(initial \"SE\")\n	(surname \"Booth\"))\n\n([GK_977554] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.g [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977554)\n	(endCoordinate -1)\n	(name \"Histone H2A.g\")\n	(referenceEntity [GK_56136])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357676])\n	(startCoordinate 1))\n\n([GK_977555] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.a [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977555)\n	(endCoordinate -1)\n	(name \"Histone H2A.a\")\n	(referenceEntity [GK_56142])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358244])\n	(startCoordinate 1))\n\n([GK_977556] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A-Bbd [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977556)\n	(endCoordinate -1)\n	(name \"Histone H2A-Bbd\")\n	(referenceEntity [GK_231427])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358637])\n	(startCoordinate 1))\n\n([GK_977557] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.n [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977557)\n	(endCoordinate -1)\n	(name \"Histone H2B.n\")\n	(referenceEntity [GK_56184])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358045])\n	(startCoordinate 1))\n\n([GK_977558] of  CandidateSet\n\n	(_displayName \"[Hsa] Histone H2A [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977558)\n	(hasCandidate\n		[GK_977556]\n		[GK_977555]\n		[GK_977573]\n		[GK_977554]\n		[GK_977560]\n		[GK_977577]\n		[GK_977587]\n		[GK_977563]\n		[GK_977583]\n		[GK_977575])\n	(name \"Histone H2A\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1357839]))\n\n([GK_977559] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.d [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977559)\n	(endCoordinate -1)\n	(name \"Histone H2B.d\")\n	(referenceEntity [GK_56172])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358639])\n	(startCoordinate 1))\n\n([GK_977560] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.l [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977560)\n	(endCoordinate -1)\n	(name \"Histone H2A.l\")\n	(referenceEntity [GK_56140])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358509])\n	(startCoordinate 1))\n\n([GK_977561] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.b [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977561)\n	(endCoordinate -1)\n	(name \"Histone H2B.b\")\n	(referenceEntity [GK_56168])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358546])\n	(startCoordinate 1))\n\n([GK_977562] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.e [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977562)\n	(endCoordinate -1)\n	(name \"Histone H2B.e\")\n	(referenceEntity [GK_56174])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358096])\n	(startCoordinate 1))\n\n([GK_977563] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.q [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977563)\n	(endCoordinate -1)\n	(name \"Histone H2A.q\")\n	(referenceEntity [GK_56148])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358143])\n	(startCoordinate 1))\n\n([GK_977564] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.a/g/h/k/l [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977564)\n	(endCoordinate -1)\n	(name \"Histone H2B.a/g/h/k/l\")\n	(referenceEntity [GK_56164])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358054])\n	(startCoordinate 1))\n\n([GK_977565] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.r [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977565)\n	(endCoordinate -1)\n	(name \"Histone H2B.r\")\n	(referenceEntity [GK_56188])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357996])\n	(startCoordinate 1))\n\n([GK_977566] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.f [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977566)\n	(endCoordinate -1)\n	(name \"Histone H2B.f\")\n	(referenceEntity [GK_56176])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358170])\n	(startCoordinate 1))\n\n([GK_977567] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H3.3 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977567)\n	(endCoordinate 136)\n	(name \"Histone H3.3\")\n	(referenceEntity [GK_355448])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357834])\n	(startCoordinate 2))\n\n([GK_977569] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H4 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977569)\n	(endCoordinate -1)\n	(name \"Histone H4\")\n	(referenceEntity [GK_355449])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358435])\n	(startCoordinate 1))\n\n([GK_977570] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H3.1 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977570)\n	(endCoordinate 136)\n	(name \"Histone H3.1\")\n	(referenceEntity [GK_355446])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357967])\n	(startCoordinate 2))\n\n([GK_977571] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.q [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977571)\n	(endCoordinate -1)\n	(name \"Histone H2B.q\")\n	(referenceEntity [GK_56186])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358327])\n	(startCoordinate 1))\n\n([GK_977572] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B type 12 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977572)\n	(endCoordinate -1)\n	(name \"Histone H2B type 12\")\n	(referenceEntity [GK_148296])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357950])\n	(startCoordinate 1))\n\n([GK_977573] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.e [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977573)\n	(endCoordinate -1)\n	(name \"Histone H2A.e\")\n	(referenceEntity [GK_56134])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358008])\n	(startCoordinate 1))\n\n([GK_977574] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B K [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977574)\n	(endCoordinate -1)\n	(name \"Histone H2B K\")\n	(referenceEntity [GK_148293])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358155])\n	(startCoordinate 1))\n\n([GK_977575] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.z [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977575)\n	(endCoordinate -1)\n	(name \"Histone H2A.z\")\n	(referenceEntity [GK_231437])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358027])\n	(startCoordinate 1))\n\n([GK_977576] of  CandidateSet\n\n	(_displayName \"[Hsa] Histone H2B [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977576)\n	(hasCandidate\n		[GK_977574]\n		[GK_977572]\n		[GK_977582]\n		[GK_977564]\n		[GK_977561]\n		[GK_977579]\n		[GK_977559]\n		[GK_977562]\n		[GK_977566]\n		[GK_977580]\n		[GK_977557]\n		[GK_977571]\n		[GK_977565]\n		[GK_977588])\n	(name \"Histone H2B\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358114]))\n\n([GK_977577] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.m [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977577)\n	(endCoordinate -1)\n	(name \"Histone H2A.m\")\n	(referenceEntity [GK_56142])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357757])\n	(startCoordinate 1))\n\n([GK_977578] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H3.2 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977578)\n	(endCoordinate 136)\n	(name \"Histone H3.2\")\n	(referenceEntity [GK_355447])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358248])\n	(startCoordinate 2))\n\n([GK_977579] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.c [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977579)\n	(endCoordinate -1)\n	(name \"Histone H2B.c\")\n	(referenceEntity [GK_56170])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357703])\n	(startCoordinate 1))\n\n([GK_977580] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.j [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977580)\n	(endCoordinate -1)\n	(name \"Histone H2B.j\")\n	(referenceEntity [GK_56180])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358076])\n	(startCoordinate 1))\n\n([GK_977582] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B, testis [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977582)\n	(endCoordinate -1)\n	(name \"Histone H2B, testis\")\n	(referenceEntity [GK_148294])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358072])\n	(startCoordinate 1))\n\n([GK_977583] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.x [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977583)\n	(endCoordinate -1)\n	(name\n		\"Histone H2A.x\"\n		\"H2a/x\")\n	(referenceEntity [GK_56152])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357805])\n	(startCoordinate 1))\n\n([GK_977584] of  Complex\n\n	(_displayName \"[Hsa] Chromatin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977584)\n	(hasComponent\n		[GK_977585]\n		[GK_976796])\n	(isChimeric FALSE)\n	(name \"Chromatin\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358067]))\n\n([GK_977585] of  Complex\n\n	(_displayName \"[Hsa] Nucleosome (Deacetylated) [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977585)\n	(hasComponent\n		[GK_977576]\n		[GK_977586]\n		[GK_977586]\n		[GK_977558]\n		[GK_977558]\n		[GK_977576]\n		[GK_977569]\n		[GK_977569])\n	(name \"Nucleosome (Deacetylated)\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1357767]))\n\n([GK_977586] of  DefinedSet\n\n	(_displayName \"[Hsa] Histone H3 [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977586)\n	(hasMember\n		[GK_977570]\n		[GK_977578]\n		[GK_977567])\n	(name \"Histone H3\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358465]))\n\n([GK_977587] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2A.o [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977587)\n	(endCoordinate -1)\n	(name \"Histone H2A.o\")\n	(referenceEntity [GK_231418])\n	(species [GK_48887])\n	(stableIdentifier [GK_1357848])\n	(startCoordinate 1))\n\n([GK_977588] of  EntityWithAccessionedSequence\n\n	(_displayName \"[Hsa] Histone H2B.s [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977588)\n	(endCoordinate -1)\n	(name \"Histone H2B.s\")\n	(referenceEntity [GK_56190])\n	(species [GK_48887])\n	(stableIdentifier [GK_1358620])\n	(startCoordinate 1))\n\n([GK_977589] of  DefinedSet\n\n	(_displayName \"[Hsa] Double-stranded DNA and chromatin [extracellular region]\")\n	(compartment [GK_984])\n	(DB_ID 977589)\n	(hasMember\n		[GK_976796]\n		[GK_977584])\n	(name \"Double-stranded DNA and chromatin\")\n	(species [GK_48887])\n	(stableIdentifier [GK_1358102]))\n\n([GK_981768] of  Person\n\n	(_displayName \"Wiltzius, JJ\")\n	(DB_ID 981768)\n	(initial \"JJ\")\n	(surname \"Wiltzius\"))\n\n([GK_984] of  EntityCompartment\n\n	(_displayName \"extracellular region\")\n	(accession \"0005576\")\n	(DB_ID 984)\n	(definition \"The space external to the outermost structure of a cell. For cells without external protective or external encapsulating structures this refers to space outside of the plasma membrane. This term covers the host cell environment outside an intracellular parasite.\")\n	(instanceOf [GK_357])\n	(name \"extracellular region\")\n	(referenceDatabase [GK_1]))\n\n([GKB_core_00388] of  %3ASTANDARD-FACET\n)\n\n([GO_BiologicalProcess] of  %3AEXTENDED-CLASS\n)\n\n([GO_CellularComponent] of  %3AEXTENDED-CLASS\n)\n\n([GO_MolecularFunction] of  %3AEXTENDED-CLASS\n)\n\n([goBiologicalProcess] of  %3AEXTENDED-SLOT\n)\n\n([goCellularComponent] of  %3AEXTENDED-SLOT\n)\n\n([GroupModifiedResidue] of  %3AEXTENDED-CLASS\n)\n\n([hasCandidate] of  %3AEXTENDED-SLOT\n)\n\n([hasComponent] of  %3AEXTENDED-SLOT\n)\n\n([hasDomain] of  %3AEXTENDED-SLOT\n)\n\n([hasEvent] of  %3AEXTENDED-SLOT\n)\n\n([hasFunctionalVariants] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([hasInstance] of  %3AEXTENDED-SLOT\n)\n\n([hasMember] of  %3AEXTENDED-SLOT\n)\n\n([hasModifiedResidue] of  %3AEXTENDED-SLOT\n)\n\n([hasPart] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([hasSpecialisedForm] of  %3AEXTENDED-SLOT\n)\n\n([height] of  %3AEXTENDED-SLOT\n)\n\n([identifier] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([identifierVersion] of  %3AEXTENDED-SLOT\n)\n\n([identify] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([identityDefinedBy] of  %3AEXTENDED-SLOT\n\n	(%3AASSOCIATED-FACET [_identityDefinedBy]))\n\n([includedLocation] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([inferredFrom] of  %3AEXTENDED-SLOT\n)\n\n([inferredTo] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([initial] of  %3AEXTENDED-SLOT\n)\n\n([input] of  %3AEXTENDED-SLOT\n)\n\n([InstanceEdit] of  %3AEXTENDED-CLASS\n)\n\n([instanceEdit] of  %3AEXTENDED-SLOT\n)\n\n([instanceOf] of  %3AEXTENDED-SLOT\n)\n\n([InterChainCrosslinkedResidue] of  %3AEXTENDED-CLASS\n)\n\n([IntraChainCrosslinkedResidue] of  %3AEXTENDED-CLASS\n)\n\n([ISBN] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL)\n	(identityDefinedBy ANY))\n\n([isCanonical] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([isChimeric] of  %3AEXTENDED-SLOT\n)\n\n([isoformParent] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([isSequenceChanged] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([journal] of  %3AEXTENDED-SLOT\n)\n\n([keyword] of  %3AEXTENDED-SLOT\n)\n\n([LiteratureReference] of  %3AEXTENDED-CLASS\n)\n\n([literatureReference] of  %3AEXTENDED-SLOT\n)\n\n([locatedEvent] of  %3AEXTENDED-SLOT\n\n	(identityDefinedBy ALL))\n\n([locationContext] of  %3AEXTENDED-SLOT\n)\n\n([maxCount] of  %3AEXTENDED-SLOT\n)\n\n([maxUnitCount] of  %3AEXTENDED-SLOT\n)\n\n([maxX] of  %3AEXTENDED-SLOT\n)\n\n([maxY] of  %3AEXTENDED-SLOT\n)\n\n([minCount] of  %3AEXTENDED-SLOT\n)\n\n([minUnitCount] of  %3AEXTENDED-SLOT\n)\n\n([minX] of  %3AEXTENDED-SLOT\n)\n\n([minY] of  %3AEXTENDED-SLOT\n)\n\n([modification] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY)\n	(identityDefinedBy ALL))\n\n([modified] of  %3AEXTENDED-SLOT\n)\n\n([ModifiedResidue] of  %3AEXTENDED-CLASS\n)\n\n([name] of  %3AEXTENDED-SLOT\n)\n\n([negativelyRegulate] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([NegativeRegulation] of  %3AEXTENDED-CLASS\n)\n\n([normalEntity] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([normalReaction] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([note] of  %3AEXTENDED-SLOT\n)\n\n([OpenSet] of  %3AEXTENDED-CLASS\n)\n\n([orthologousEvent] of  %3AEXTENDED-SLOT\n)\n\n([OtherEntity] of  %3AEXTENDED-CLASS\n)\n\n([otherIdentifier] of  %3AEXTENDED-SLOT\n)\n\n([output] of  %3AEXTENDED-SLOT\n)\n\n([pages] of  %3AEXTENDED-SLOT\n)\n\n([Pathway] of  %3AEXTENDED-CLASS\n)\n\n([PathwayCoordinates] of  %3AEXTENDED-CLASS\n)\n\n([PathwayDiagram] of  %3AEXTENDED-CLASS\n)\n\n([pathwayDiagram] of  %3AEXTENDED-SLOT\n)\n\n([PathwayDiagramItem] of  %3AEXTENDED-CLASS\n)\n\n([PathwayVertex] of  %3AEXTENDED-CLASS\n)\n\n([Person] of  %3AEXTENDED-CLASS\n)\n\n([PhysicalEntity] of  %3AEXTENDED-CLASS\n)\n\n([physicalEntity] of  %3AEXTENDED-SLOT\n)\n\n([pointCoordinates] of  %3AEXTENDED-SLOT\n)\n\n([Polymer] of  %3AEXTENDED-CLASS\n)\n\n([Polymerisation] of  %3AEXTENDED-CLASS\n)\n\n([positivelyRegulate] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([PositiveRegulation] of  %3AEXTENDED-CLASS\n)\n\n([precedingEvent] of  %3AEXTENDED-SLOT\n)\n\n([previousValue] of  %3AEXTENDED-SLOT\n\n	(databaseColumnSpecification \"TEXT\"))\n\n([project] of  %3AEXTENDED-SLOT\n)\n\n([providerCount] of  %3AEXTENDED-SLOT\n)\n\n([PsiMod] of  %3AEXTENDED-CLASS\n)\n\n([psiMod] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY)\n	(identityDefinedBy ALL))\n\n([Publication] of  %3AEXTENDED-CLASS\n)\n\n([publisher] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([pubMedIdentifier] of  %3AEXTENDED-SLOT\n)\n\n([Reaction] of  %3AEXTENDED-CLASS\n)\n\n([ReactionCoordinates] of  %3AEXTENDED-CLASS\n)\n\n([ReactionlikeEvent] of  %3AEXTENDED-CLASS\n)\n\n([Reactome_109607_14914_Slot_1] of  %3AEXTENDED-SLOT\n)\n\n([reason] of  %3AEXTENDED-SLOT\n\n	(attributeCategory REQUIRED))\n\n([referenceDatabase] of  %3AEXTENDED-SLOT\n)\n\n([ReferenceDatabase] of  %3AEXTENDED-CLASS\n)\n\n([ReferenceDNASequence] of  %3AEXTENDED-CLASS\n)\n\n([referenceEntity] of  %3AEXTENDED-SLOT\n)\n\n([ReferenceEntity] of  %3AEXTENDED-CLASS\n)\n\n([referenceGene] of  %3AEXTENDED-SLOT\n)\n\n([ReferenceGeneProduct] of  %3AEXTENDED-CLASS\n)\n\n([ReferenceGroup] of  %3AEXTENDED-CLASS\n)\n\n([referenceGroup] of  %3AEXTENDED-SLOT\n)\n\n([referenceGroupCount] of  %3AEXTENDED-SLOT\n)\n\n([ReferenceIsoform] of  %3AEXTENDED-CLASS\n)\n\n([ReferenceMolecule] of  %3AEXTENDED-CLASS\n)\n\n([ReferenceRNASequence] of  %3AEXTENDED-CLASS\n)\n\n([referenceSequence] of  %3AEXTENDED-SLOT\n)\n\n([ReferenceSequence] of  %3AEXTENDED-CLASS\n)\n\n([referenceTranscript] of  %3AEXTENDED-SLOT\n)\n\n([regulate] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([regulatedEntity] of  %3AEXTENDED-SLOT\n)\n\n([Regulation] of  %3AEXTENDED-CLASS\n)\n\n([RegulationType] of  %3AEXTENDED-CLASS\n)\n\n([regulationType] of  %3AEXTENDED-SLOT\n)\n\n([regulator] of  %3AEXTENDED-SLOT\n)\n\n([relatedSpecies] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL)\n	(identityDefinedBy ALL))\n\n([releaseDate] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([releaseNumber] of  %3AEXTENDED-SLOT\n\n	(attributeCategory REQUIRED))\n\n([releaseStatus] of  %3AEXTENDED-SLOT\n\n	(attributeCategory NOMANUALEDIT))\n\n([removingActivity] of  %3AEXTENDED-SLOT\n)\n\n([repeatedUnit] of  %3AEXTENDED-SLOT\n)\n\n([ReplacedResidue] of  %3AEXTENDED-CLASS\n)\n\n([replacementInstances] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([representedInstance] of  %3AEXTENDED-SLOT\n)\n\n([representedPathway] of  %3AEXTENDED-SLOT\n)\n\n([requiredInputComponent] of  %3AEXTENDED-SLOT\n)\n\n([Requirement] of  %3AEXTENDED-CLASS\n)\n\n([residue] of  %3AEXTENDED-SLOT\n)\n\n([reverseReaction] of  %3AEXTENDED-SLOT\n)\n\n([reviewed] of  %3AEXTENDED-SLOT\n)\n\n([reviewer] of  %3AEXTENDED-SLOT\n)\n\n([revised] of  %3AEXTENDED-SLOT\n)\n\n([searchableTerm] of  %3AEXTENDED-SLOT\n)\n\n([secondaryIdentifier] of  %3AEXTENDED-SLOT\n)\n\n([secondCoordinate] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([secondReferenceSequence] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([SequenceDomain] of  %3AEXTENDED-CLASS\n)\n\n([sequenceLength] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([SequenceOntology] of  %3AEXTENDED-CLASS\n)\n\n([shortName] of  %3AEXTENDED-SLOT\n)\n\n([SimpleEntity] of  %3AEXTENDED-CLASS\n)\n\n([slotDisplayName] of  %3AEXTENDED-SLOT\n\n	(%3AASSOCIATED-FACET [_slotDisplayName]))\n\n([sourceVertex] of  %3AEXTENDED-SLOT\n)\n\n([sourceX] of  %3AEXTENDED-SLOT\n)\n\n([sourceY] of  %3AEXTENDED-SLOT\n)\n\n([species] of  %3AEXTENDED-SLOT\n)\n\n([Species] of  %3AEXTENDED-CLASS\n)\n\n([stableIdentifier] of  %3AEXTENDED-SLOT\n)\n\n([StableIdentifier] of  %3AEXTENDED-CLASS\n)\n\n([startCoordinate] of  %3AEXTENDED-SLOT\n)\n\n([startPositionInReferenceSequence] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY))\n\n([storedATXML] of  %3AEXTENDED-SLOT\n)\n\n([structuralVariant] of  %3AEXTENDED-SLOT\n)\n\n([Summation] of  %3AEXTENDED-CLASS\n)\n\n([summation] of  %3AEXTENDED-SLOT\n)\n\n([superTaxon] of  %3AEXTENDED-SLOT\n)\n\n([surname] of  %3AEXTENDED-SLOT\n)\n\n([synonym] of  %3AEXTENDED-SLOT\n\n	(attributeCategory OPTIONAL))\n\n([targetVertex] of  %3AEXTENDED-SLOT\n)\n\n([targetX] of  %3AEXTENDED-SLOT\n)\n\n([targetY] of  %3AEXTENDED-SLOT\n)\n\n([Taxon] of  %3AEXTENDED-CLASS\n)\n\n([taxon] of  %3AEXTENDED-SLOT\n)\n\n([templateEvent] of  %3AEXTENDED-SLOT\n)\n\n([termProvider] of  %3AEXTENDED-SLOT\n)\n\n([text] of  %3AEXTENDED-SLOT\n)\n\n([title] of  %3AEXTENDED-SLOT\n)\n\n([TranslationalModification] of  %3AEXTENDED-CLASS\n)\n\n([uniformResourceLocator] of  %3AEXTENDED-SLOT\n\n	(attributeCategory MANDATORY)\n	(identityDefinedBy ANY))\n\n([update_200405_intermediate_Instance_7] of  %3APAL-CONSTRAINT\n)\n\n([url] of  %3AEXTENDED-SLOT\n)\n\n([URL] of  %3AEXTENDED-CLASS\n)\n\n([variantIdentifier] of  %3AEXTENDED-SLOT\n)\n\n([vertex] of  %3AEXTENDED-SLOT\n)\n\n([Vertex] of  %3AEXTENDED-CLASS\n)\n\n([vertexCount] of  %3AEXTENDED-SLOT\n)\n\n([VertexSearchableTerm] of  %3AEXTENDED-CLASS\n)\n\n([volume] of  %3AEXTENDED-SLOT\n)\n\n([width] of  %3AEXTENDED-SLOT\n)\n\n([wrappedLabel] of  %3AEXTENDED-SLOT\n)\n\n([x] of  %3AEXTENDED-SLOT\n)\n\n([y] of  %3AEXTENDED-SLOT\n)\n\n([year] of  %3AEXTENDED-SLOT\n)\n\0\0\0pins_file_stub\nDatabaseObject\0\0\0\nroot_class');
/*!40000 ALTER TABLE `Ontology` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2014-05-28 16:59:59
